{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "QqyaLIqh4NlE"
      },
      "source": [
        "## <center>**Information Retrieval**</center>\n",
        "## <center>**Coursework 3 - Search Engine Implementation**</center><br>\n",
        "_**Name: Kweku E. Acquaye<br>\n",
        "Group: 68 (single member group)**_<br> \n",
        "\n",
        "### **<center> Table of Contents </center>**\n",
        "\n",
        "1. Section 1: Introduction\n",
        "  * Section 1.1: Outline\n",
        "  * Section 1.2: Model Search Engine\n",
        "2. Section 2: Building the Model\n",
        "  * Section 2.1: Loading the data\n",
        "  * Section 2.2: Creating the index\n",
        "3. Section 3: Creating a BM25 Model \n",
        "  * Section 3.1: Creating retrieval function\n",
        "  * Section 3.2: Searching and ranking documents for queries\n",
        "4. Section 4: Evaluating BM25 model\n",
        "  * Section 4.1: Evaluation by Harmonic Mean (F1-score)\n",
        "  * Section 4.2: Evaluation by Normalized Discounted Cumulative Gain (nDCG)\n",
        "5. Section 5: Discussion and Conclusions"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "TgI6MsLCP8Gr"
      },
      "source": [
        "### **Section 1: Introduction**<br> \n",
        "#### **Section 1.1: Outline**\n",
        "This report uses modern data science methods to build, implement, and evaluate the performance of a model search engine designed earlier in Coursework 2. The aim is to fully understand and appreciate how a search engine works by actually building the search engine. This notebook is to be submitted together with a pdf description summarising theory, and a video presentation demostrating the model's ability to receive a query (via a command line) and then yield an output (in a file or on screen) of a ranked list of documents.<br>  \n",
        "\n",
        "This work constitutes Coursework 3a of Information Retrieval module.\n",
        "\n",
        "**Declaration:** Some of the code used in this assignment has been adapted and customized from www.docs.python.org/, www.pandas.pydata.org/docs, www.pytorch.org/tutorials/, www.matplotlib.org/stable/, www.stackoverflow.com/questions/, www.geeksforgeeks.org/fundamentals-of-algorithms/, www.machinelearningmind.com/, www.kaggle.com/, www.scikit-learn.org, www.numpy.org/doc/stable/user/, www.github.com/, www.ethen8181.github.io/machine-learning/search/bm25_intro.html, and IR Lab, Tutorial, and Lecture Notes.<br> "
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ifvTiK5lBY3F"
      },
      "source": [
        "#### **Section 1.2: Model Search Engine**\n",
        "In **_Coursework 2 - Design a Model Search Engine_**, the following design architecture was created:<br> \n",
        "\n",
        "<center>\n",
        "<div>\n",
        "<img src= \"https://drive.google.com/uc?id=1zk40emIkynMDI1IWKMPpovoNs1s96_4h\" alt= \"star schema\" width=600/>\n",
        "</div>\n",
        "</center>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "PdKIDR0InfxG"
      },
      "source": [
        "This design architecture is now implemented in this notebook, with two minor differences - the dataset has had to be changed. The reason for this is that in early trials of implementation, execution time for the 30 GB CORD-19 dataset (Wang _et al_ 2020), even in the simplest of operations, proved too long to be of use as a **_model_** search engine.<br> \n",
        "\n",
        "It has therefore been substituted with a fairly wide dataset of Wikipedia articles on Covid-19. Instead of local or cloud storage of the dataset, data would be accessed and parsed directly by utilising Wikipedia API for Python version 1.4 (online reference 1).<br> \n",
        "\n",
        "Also, instead of determining the Mean Average Precision (MAP) as the 2nd method of evaluation, the normalized Discounted Cumulative Gain (nDCG) method is used."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "w0srNCYbQGZk"
      },
      "source": [
        "### **Section 2: Building the Model**\n",
        "The following series of steps instantiates, builds and implements the network architecture and pipeline:\n",
        "\n",
        "#### **Section 2.1: Loading the data**"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "GAvnzoqYkodG",
        "outputId": "526f9320-e65a-4a0b-92bf-0796b0fc3c45"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: wikipedia in /usr/local/lib/python3.7/dist-packages (1.4.0)\n",
            "Requirement already satisfied: beautifulsoup4 in /usr/local/lib/python3.7/dist-packages (from wikipedia) (4.6.3)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.0.0 in /usr/local/lib/python3.7/dist-packages (from wikipedia) (2.23.0)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.0.0->wikipedia) (2021.10.8)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.0.0->wikipedia) (1.24.3)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.0.0->wikipedia) (3.0.4)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests<3.0.0,>=2.0.0->wikipedia) (2.10)\n",
            "Requirement already satisfied: statsmodels in /usr/local/lib/python3.7/dist-packages (0.13.2)\n",
            "Requirement already satisfied: packaging>=21.3 in /usr/local/lib/python3.7/dist-packages (from statsmodels) (21.3)\n",
            "Requirement already satisfied: patsy>=0.5.2 in /usr/local/lib/python3.7/dist-packages (from statsmodels) (0.5.2)\n",
            "Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.7/dist-packages (from statsmodels) (1.21.5)\n",
            "Requirement already satisfied: pandas>=0.25 in /usr/local/lib/python3.7/dist-packages (from statsmodels) (1.3.5)\n",
            "Requirement already satisfied: scipy>=1.3 in /usr/local/lib/python3.7/dist-packages (from statsmodels) (1.4.1)\n",
            "Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from packaging>=21.3->statsmodels) (3.0.8)\n",
            "Requirement already satisfied: python-dateutil>=2.7.3 in /usr/local/lib/python3.7/dist-packages (from pandas>=0.25->statsmodels) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2017.3 in /usr/local/lib/python3.7/dist-packages (from pandas>=0.25->statsmodels) (2018.9)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.7/dist-packages (from patsy>=0.5.2->statsmodels) (1.15.0)\n"
          ]
        }
      ],
      "source": [
        "# Importing libraries\n",
        "import pandas as pd\n",
        "import numpy as np\n",
        "from sklearn.feature_extraction.text import CountVectorizer\n",
        "import os\n",
        "import re\n",
        "\n",
        "# Installing and upgrading wikipedia and statsmodels packages\n",
        "!pip install wikipedia\n",
        "!pip install statsmodels --upgrade\n",
        "\n",
        "import wikipedia"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "638SxjAnU9go"
      },
      "source": [
        "For the purpose of this coursework, the entire content of each article is cast as a document.<br> \n",
        "This next step defines and loads the 10-megabyte 30-document corpus:"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yoSOVmYYo93V",
        "outputId": "2b0b3a43-c7e1-47fa-d2b4-e7dbe4d43b47"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "loading content:  COVID-19\n",
            "loading content:  COVID-19 vaccine\n",
            "loading content:  Long COVID\n",
            "loading content:  COVID-19 testing\n",
            "loading content:  COVID-19 pandemic cases\n",
            "loading content:  Symptoms of COVID-19\n",
            "loading content:  Variants of SARS-CoV-2\n",
            "loading content:  COVID-19 pandemic in North America\n",
            "loading content:  COVID-19 pandemic in Europe\n",
            "loading content:  COVID-19 pandemic in Asia\n",
            "loading content:  COVID-19 pandemic in Africa\n",
            "loading content:  COVID-19 pandemic in Australia\n",
            "loading content:  COVID-19 pandemic by country and territory\n",
            "loading content:  COVID-19 lockdowns\n",
            "loading content:  COVID-19 pandemic in South America\n",
            "loading content:  Impact of the COVID-19 pandemic on education\n",
            "loading content:  SARS-CoV-2 Delta variant\n",
            "loading content:  SARS-CoV-2 Gamma variant\n",
            "loading content:  SARS-CoV-2 Omicron variant\n",
            "loading content:  COVID-19 vaccine clinical research\n",
            "loading content:  Coronavirus\n",
            "loading content:  Economic impact of the COVID-19 pandemic\n",
            "loading content:  COVID-19 pandemic in Antarctica\n",
            "loading content:  Virus\n",
            "loading content:  History of COVID-19 vaccine development\n",
            "loading content:  Impact of the COVID-19 pandemic on religion\n",
            "loading content:  Political impact of the COVID-19 pandemic\n",
            "loading content:  Severe acute respiratory syndrome coronavirus 2\n",
            "loading content:  Investigations into the origin of COVID-19\n",
            "loading content:  Angiotensin-converting enzyme 2\n"
          ]
        }
      ],
      "source": [
        "# Defining dataset of documents\n",
        "articles=['COVID-19','COVID-19 vaccine','Long COVID','COVID-19 testing','COVID-19 pandemic cases','Symptoms of COVID-19','Variants of SARS-CoV-2',\n",
        "          'COVID-19 pandemic in North America', 'COVID-19 pandemic in Europe', 'COVID-19 pandemic in Asia', 'COVID-19 pandemic in Africa',\n",
        "          'COVID-19 pandemic in Australia', 'COVID-19 pandemic by country and territory', 'COVID-19 lockdowns', 'COVID-19 pandemic in South America', \n",
        "          'Impact of the COVID-19 pandemic on education', 'SARS-CoV-2 Delta variant', 'SARS-CoV-2 Gamma variant', 'SARS-CoV-2 Omicron variant', \n",
        "          'COVID-19 vaccine clinical research', 'Coronavirus', 'Economic impact of the COVID-19 pandemic', 'COVID-19 pandemic in Antarctica', 'Virus',\n",
        "          'History of COVID-19 vaccine development', 'Impact of the COVID-19 pandemic on religion', 'Political impact of the COVID-19 pandemic',\n",
        "          'Severe acute respiratory syndrome coronavirus 2', 'Investigations into the origin of COVID-19', 'Angiotensin-converting enzyme 2', \n",
        "          ]\n",
        "documents=[]\n",
        "title=[]\n",
        "\n",
        "# Loading wikipedia articles\n",
        "for article in articles:\n",
        "   print(\"loading content: \",article)\n",
        "   documents.append(wikipedia.page(article,auto_suggest=False).content)\n",
        "   title.append(article)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "HJSuJUWlpFIs",
        "outputId": "628bc48a-fd3b-4ffc-86c7-4ec6172f1761"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Coronavirus disease 2019 (COVID-19) is a contagious disease caused by a virus, the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The first known case was identified in Wuhan, China, in December 2019. The disease has since spread worldwide, leading to the ongoing COVID-19 pandemic.Symptoms of COVID‑19 are variable, but often include fever, cough, headache, fatigue, breathing difficulties, loss of smell, and loss of taste. Symptoms may begin one to fourteen days after exposure to the virus. At least a third of people who are infected do not develop noticeable symptoms. Of those people who develop symptoms noticeable enough to be classed as patients, most (81%) develop mild to moderate symptoms (up to mild pneumonia), while 14% develop severe symptoms (dyspnea, hypoxia, or more than 50% lung involvement on imaging), and 5% suffer critical symptoms (respiratory failure, shock, or multiorgan dysfunction). Older people are at a higher risk of developing severe symptoms. Some people continue to experience a range of effects (long COVID) for months after recovery, and damage to organs has been observed. Multi-year studies are underway to further investigate the long-term effects of the disease.COVID‑19 transmits when people breathe in air contaminated by droplets and small airborne particles containing the virus. The risk of breathing these in is highest when people are in close proximity, but they can be inhaled over longer distances, particularly indoors. Transmission can also occur if splashed or sprayed with contaminated fluids in the eyes, nose or mouth, and, rarely, via contaminated surfaces. People remain contagious for up to 20 days, and can spread the virus even if they do not develop symptoms.Several COVID-19 testing methods have been developed to diagnose the disease. The standard diagnostic method is by detection of the virus's nucleic acid by real-time reverse transcription polymerase chain reaction (rRT-PCR), transcription-mediated amplification (TMA), or by reverse transcription loop-mediated isothermal amplification (RT-LAMP) from a nasopharyngeal swab.\n",
            "Several COVID-19 vaccines have been approved and distributed in various countries, which have initiated mass vaccination campaigns. Other preventive measures include physical or social distancing, quarantining, ventilation of indoor spaces, covering coughs and sneezes, hand washing, and keeping unwashed hands away from the face. The use of face masks or coverings has been recommended in public settings to minimize the risk of transmission. While work is underway to develop drugs that inhibit the virus, the primary treatment is symptomatic. Management involves the treatment of symptoms, supportive care, isolation, and experimental measures.\n",
            "\n",
            "\n",
            "== Nomenclature ==\n",
            "\n",
            "During the initial outbreak in Wuhan, the virus and disease were commonly referred to as \"coronavirus\" and \"Wuhan coronavirus\", with the disease sometimes called \"Wuhan pneumonia\". In the past, many diseases have been named after geographical locations, such as the Spanish flu, Middle East respiratory syndrome, and Zika virus. In January 2020, the World Health Organization (WHO) recommended 2019-nCoV and 2019-nCoV acute respiratory disease as interim names for the virus and disease per 2015 guidance and international guidelines against using geographical locations or groups of people in disease and virus names to prevent social stigma. The official names COVID‑19 and SARS-CoV-2 were issued by the WHO on 11 February 2020. The Director-General, Tedros Adhanom explained that CO stands for corona, VI for virus, D for disease, and 19 for 2019, the year in which the outbreak was first identified. The WHO additionally uses \"the COVID‑19 virus\" and \"the virus responsible for COVID‑19\" in public communications.\n",
            "\n",
            "\n",
            "== Signs and symptoms ==\n",
            "\n",
            "\n",
            "== Cause ==\n",
            "COVID‑19 is caused by infection with a strain of coronavirus known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).\n",
            "\n",
            "\n",
            "=== Transmission ===\n",
            "\n",
            "\n",
            "=== Virology ===\n",
            "\n",
            "Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a novel severe acute respiratory syndrome coronavirus. It was first isolated from three people with pneumonia connected to the cluster of acute respiratory illness cases in Wuhan. All structural features of the novel SARS-CoV-2 virus particle occur in related coronaviruses in nature.Outside the human body, the virus is destroyed by household soap, which bursts its protective bubble.SARS-CoV-2 is closely related to the original SARS-CoV. It is thought to have an animal (zoonotic) origin. Genetic analysis has revealed that the coronavirus genetically clusters with the genus Betacoronavirus, in subgenus Sarbecovirus (lineage B) together with two bat-derived strains. It is 96% identical at the whole genome level to other bat coronavirus samples (BatCov RaTG13). The structural proteins of SARS-CoV-2 include membrane glycoprotein (M), envelope protein (E), nucleocapsid protein (N), and the spike protein (S). The M protein of SARS-CoV-2 is about 98% similar to the M protein of bat SARS-CoV, maintains around 98% homology with pangolin SARS-CoV, and has 90% homology with the M protein of SARS-CoV; whereas, the similarity is only around 38% with the M protein of MERS-CoV.\n",
            "\n",
            "\n",
            "=== SARS-CoV-2 variants ===\n",
            "\n",
            "The many thousands of SARS-CoV-2 variants are grouped into either clades or lineages. The WHO, in collaboration with partners, expert networks, national authorities, institutions and researchers, have established nomenclature systems for naming and tracking SARS-CoV-2 genetic lineages by GISAID, Nextstrain and Pango. At the present time, the expert group convened by WHO has recommended the labeling of variants using letters of the Greek Alphabet, for example, Alpha, Beta, Delta, and Gamma, giving the justification that they \"will be easier and more practical to discussed by non-scientific audiences.\" Nextstrain divides the variants into five clades (19A, 19B, 20A, 20B, and 20C), while GISAID divides them into seven (L, O, V, S, G, GH, and GR). The Pango tool groups variants into lineages, with many circulating lineages being classed under the B.1 lineage.Several notable variants of SARS-CoV-2 emerged throughout 2020. Cluster 5 emerged among minks and mink farmers in Denmark. After strict quarantines and a mink euthanasia campaign, the cluster was assessed to no longer be circulating among humans in Denmark as of 1 February 2021.As of December 2021, there are five dominant variants of SARS-CoV-2 spreading among global populations: the Alpha variant (B.1.1.7, formerly called the UK variant), first found in London and Kent, the Beta variant (B.1.351, formerly called the South Africa variant), the Gamma variant (P.1, formerly called the Brazil variant), the Delta variant (B.1.617.2, formerly called the India variant), and the Omicron variant (B.1.1.529), which had spread to 57 countries as of 7 December.\n",
            "\n",
            "\n",
            "== Pathophysiology ==\n",
            "\n",
            "The SARS-CoV-2 virus can infect a wide range of cells and systems of the body. COVID‑19 is most known for affecting the upper respiratory tract (sinuses, nose, and throat) and the lower respiratory tract (windpipe and lungs). The lungs are the organs most affected by COVID‑19 because the virus accesses host cells via the receptor for the enzyme angiotensin-converting enzyme 2 (ACE2), which is most abundant on the surface of type II alveolar cells of the lungs. The virus uses a special surface glycoprotein called a \"spike\" to connect to the ACE2 receptor and enter the host cell.\n",
            "\n",
            "\n",
            "=== Respiratory tract ===\n",
            "Following viral entry, COVID‑19 infects the ciliated epithelium of the nasopharynx and upper airways.\n",
            "\n",
            "\n",
            "=== Nervous system ===\n",
            "One common symptom, loss of smell, results from infection of the support cells of the olfactory epithelium, with subsequent damage to the olfactory neurons. The involvement of both the central and peripheral nervous system in COVID‑19 has been reported in many medical publications. It is clear that many people with COVID-19 exhibit neurological or mental health issues. The virus is not detected in the CNS of the majority of COVID-19 patients with neurological issues. However, SARS-CoV-2 has been detected at low levels in the brains of those who have died from COVID‑19, but these results need to be confirmed. While virus has been detected in cerebrospinal fluid of autopsies, the exact mechanism by which it invades the CNS remains unclear and may first involve invasion of peripheral nerves given the low levels of ACE2 in the brain. The virus may also enter the bloodstream from the lungs and cross the blood-brain barrier to gain access to the CNS, possibly within an infected white blood cell.\n",
            "  Research conducted when Alpha was the dominant variant has suggested COVID-19 May cause brain damage. It is currently unknown if such damage is temporary or permanent, and whether Omicron has similar effects.  Observed individuals infected with COVID-19 (most with mild cases) experienced an additional 0.2% to 2% of brain tissue lost in regions of the brain connected to the sense of smell compared with uninfected individuals, and the overall effect on the brain was equivalent on average to at least one extra year of normal aging; infected individuals also scored lower on several cognitive tests.  All effects were more pronounced among older ages.\n",
            "\n",
            "\n",
            "=== Gastrointestinal tract ===\n",
            "The virus also affects gastrointestinal organs as ACE2 is abundantly expressed in the glandular cells of gastric, duodenal and rectal epithelium as well as endothelial cells and enterocytes of the small intestine.\n",
            "\n",
            "\n",
            "=== Cardiovascular system ===\n",
            "The virus can cause acute myocardial injury and chronic damage to the cardiovascular system. An acute cardiac injury was found in 12% of infected people admitted to the hospital in Wuhan, China, and is more frequent in severe disease. Rates of cardiovascular symptoms are high, owing to the systemic inflammatory response and immune system disorders during disease progression, but acute myocardial injuries may also be related to ACE2 receptors in the heart. ACE2 receptors are highly expressed in the heart and are involved in heart function. A high incidence of thrombosis and venous thromboembolism have been found in people transferred to Intensive care units (ICU) with COVID‑19 infections, and may be related to poor prognosis. Blood vessel dysfunction and clot formation (as suggested by high D-dimer levels caused by blood clots) are thought to play a significant role in mortality, incidences of clots leading to pulmonary embolisms, and ischaemic events within the brain have been noted as complications leading to death in people infected with SARS-CoV-2. Infection appears to set off a chain of vasoconstrictive responses within the body, constriction of blood vessels within the pulmonary circulation has also been posited as a mechanism in which oxygenation decreases alongside the presentation of viral pneumonia. Furthermore, microvascular (arterioles and capillaries) blood vessel damage has been reported in a small number of tissue samples of the brains – without detected SARS-CoV-2 – and the olfactory bulbs from those who have died from COVID‑19. COVID‑19 was also found to cause substantial – including morphological and mechanical – changes to blood cells – such as increased sizes – sometimes persisting for months after hospital discharge.\n",
            "\n",
            "\n",
            "=== Other organs ===\n",
            "Another common cause of death is complications related to the kidneys. Early reports show that up to 30% of hospitalized patients both in China and in New York have experienced some injury to their kidneys, including some persons with no previous kidney problems.Autopsies of people who died of COVID‑19 have found diffuse alveolar damage, and lymphocyte-containing inflammatory infiltrates within the lung.\n",
            "\n",
            "\n",
            "=== Immunopathology ===\n",
            "\n",
            "Although SARS-CoV-2 has a tropism for ACE2-expressing epithelial cells of the respiratory tract, people with severe COVID‑19 have symptoms of systemic hyperinflammation. Clinical laboratory findings of elevated IL‑2, IL‑7, IL‑6, granulocyte-macrophage colony-stimulating factor (GM‑CSF), interferon gamma-induced protein 10 (IP‑10), monocyte chemoattractant protein 1 (MCP1), macrophage inflammatory protein 1‑alpha (MIP‑1‑alpha), and tumour necrosis factor (TNF‑α) indicative of cytokine release syndrome (CRS) suggest an underlying immunopathology.Additionally, people with COVID‑19 and acute respiratory distress syndrome (ARDS) have classical serum biomarkers of CRS, including elevated C-reactive protein (CRP), lactate dehydrogenase (LDH), D-dimer, and ferritin.Systemic inflammation results in vasodilation, allowing inflammatory lymphocytic and monocytic infiltration of the lung and the heart. In particular, pathogenic GM-CSF-secreting T cells were shown to correlate with the recruitment of inflammatory IL-6-secreting monocytes and severe lung pathology in people with COVID‑19. Lymphocytic infiltrates have also been reported at autopsy.\n",
            "\n",
            "\n",
            "=== Viral and host factors ===\n",
            "\n",
            "\n",
            "==== Virus proteins ====\n",
            "\n",
            "Multiple viral and host factors affect the pathogenesis of the virus. The S-protein, otherwise known as the spike protein, is the viral component that attaches to the host receptor via the ACE2 receptors. It includes two subunits: S1 and S2. S1 determines the virus-host range and cellular tropism via the receptor-binding domain. S2 mediates the membrane fusion of the virus to its potential cell host via the H1 and HR2, which are heptad repeat regions. Studies have shown that S1 domain induced IgG and IgA antibody levels at a much higher capacity. It is the focus spike proteins expression that are involved in many effective COVID‑19 vaccines.The M protein is the viral protein responsible for the transmembrane transport of nutrients. It is the cause of the bud release and the formation of the viral envelope. The N and E protein are accessory proteins that interfere with the host's immune response.\n",
            "\n",
            "\n",
            "==== Host factors ====\n",
            "Human angiotensin converting enzyme 2 (hACE2) is the host factor that SARS-COV2 virus targets causing COVID‑19. Theoretically, the usage of angiotensin receptor blockers (ARB) and ACE inhibitors upregulating ACE2 expression might increase morbidity with COVID‑19, though animal data suggest some potential protective effect of ARB; however no clinical studies have proven susceptibility or outcomes. Until further data is available, guidelines and recommendations for hypertensive patients remain.The effect of the virus on ACE2 cell surfaces leads to leukocytic infiltration, increased blood vessel permeability, alveolar wall permeability, as well as decreased secretion of lung surfactants. These effects cause the majority of the respiratory symptoms. However, the aggravation of local inflammation causes a cytokine storm eventually leading to a systemic inflammatory response syndrome.Among healthy adults not exposed to SARS-CoV-2, about 35% have CD4+ T cells that recognize the SARS-CoV-2 S protein (particularly the S2 subunit) and about 50% react to other proteins of the virus, suggesting cross-reactivity from previous common colds caused by other coronaviruses.It is unknown whether different persons use similar antibody genes in response to COVID‑19.\n",
            "\n",
            "\n",
            "=== Host cytokine response ===\n",
            "\n",
            "The severity of the inflammation can be attributed to the severity of what is known as the cytokine storm. Levels of interleukin 1B, interferon-gamma, interferon-inducible protein 10, and monocyte chemoattractant protein 1 were all associated with COVID‑19 disease severity. Treatment has been proposed to combat the cytokine storm as it remains to be one of the leading causes of morbidity and mortality in COVID‑19 disease.A cytokine storm is due to an acute hyperinflammatory response that is responsible for clinical illness in an array of diseases but in COVID‑19, it is related to worse prognosis and increased fatality. The storm causes acute respiratory distress syndrome, blood clotting events such as strokes, myocardial infarction, encephalitis, acute kidney injury, and vasculitis. The production of IL-1, IL-2, IL-6, TNF-alpha, and interferon-gamma, all crucial components of normal immune responses, inadvertently become the causes of a cytokine storm. The cells of the central nervous system, the microglia, neurons, and astrocytes, are also involved in the release of pro-inflammatory cytokines affecting the nervous system, and effects of cytokine storms toward the CNS are not uncommon.\n",
            "\n",
            "\n",
            "=== Pregnancy response ===\n",
            "There are many unknowns for pregnant women during the COVID‑19 pandemic. Given that they are prone to suffering from complications and severe disease infection with other types of coronaviruses, they have been identified as a vulnerable group and advised to take supplementary preventive measures.Physiological responses to pregnancy can include:\n",
            "\n",
            "Immunological: The immunological response to COVID-19, like other viruses, depends on a working immune system. It adapts during pregnancy to allow the development of the fetus whose genetic load is only partially shared with their mother, leading to a different immunological reaction to infections during the course of pregnancy.\n",
            "Respiratory: Many factors can make pregnant women more vulnerable to hard respiratory infections. One of them is the total reduction of the lungs' capacity and inability to clear secretions.\n",
            "Coagulation: During pregnancy, there are higher levels of circulating coagulation factors, and the pathogenesis of SARS-CoV-2 infection can be implicated. The thromboembolic events with associated mortality are a risk for pregnant women.However, from the evidence base, it is difficult to conclude whether pregnant women are at increased risk of grave consequences of this virus.In addition to the above, other clinical studies have proved that SARS-CoV-2 can affect the period of pregnancy in different ways. On the one hand, there is little evidence of its impact up to 12 weeks gestation. On the other hand, COVID-19 infection may cause increased rates of unfavorable outcomes in the course of the pregnancy. Some examples of these could be fetal growth restriction, preterm birth, and perinatal mortality, which refers to the fetal death past 22 or 28 completed weeks of pregnancy as well as the death among live-born children up to seven completed days of life.Unvaccinated women in later stages of pregnancy with COVID-19 are more likely than other patients to need very intensive care. Babies born to mothers with COVID-19 are more likely to have breathing problems. Pregnant women are strongly encouraged to get vaccinated.\n",
            "\n",
            "\n",
            "== Diagnosis ==\n",
            "\n",
            "COVID‑19 can provisionally be diagnosed on the basis of symptoms and confirmed using reverse transcription polymerase chain reaction (RT-PCR) or other nucleic acid testing of infected secretions. Along with laboratory testing, chest CT scans may be helpful to diagnose COVID‑19 in individuals with a high clinical suspicion of infection. Detection of a past infection is possible with serological tests, which detect antibodies produced by the body in response to the infection.\n",
            "\n",
            "\n",
            "=== Viral testing ===\n",
            "\n",
            "The standard methods of testing for presence of SARS-CoV-2 are nucleic acid tests, which detects the presence of viral RNA fragments. As these tests detect RNA but not infectious virus, its \"ability to determine duration of infectivity of patients is limited.\" The test is typically done on respiratory samples obtained by a nasopharyngeal swab; however, a nasal swab or sputum sample may also be used. Results are generally available within hours. The WHO has published several testing protocols for the disease.Several laboratories and companies have developed serological tests, which detect antibodies produced by the body in response to infection. Several have been evaluated by Public Health England and approved for use in the UK.The University of Oxford's CEBM has pointed to mounting evidence that \"a good proportion of 'new' mild cases and people re-testing positives after quarantine or discharge from hospital are not infectious, but are simply clearing harmless virus particles which their immune system has efficiently dealt with\" and have called for \"an international effort to standardize and periodically calibrate testing\" On 7 September, the UK government issued \"guidance for procedures to be implemented in laboratories to provide assurance of positive SARS-CoV-2 RNA results during periods of low prevalence, when there is a reduction in the predictive value of positive test results\".\n",
            "\n",
            "\n",
            "=== Imaging ===\n",
            "\n",
            "Chest CT scans may be helpful to diagnose COVID‑19 in individuals with a high clinical suspicion of infection but are not recommended for routine screening. Bilateral multilobar ground-glass opacities with a peripheral, asymmetric, and posterior distribution are common in early infection. Subpleural dominance, crazy paving (lobular septal thickening with variable alveolar filling), and consolidation may appear as the disease progresses. Characteristic imaging features on chest radiographs and computed tomography (CT) of people who are symptomatic include asymmetric peripheral ground-glass opacities without pleural effusions.Many groups have created COVID‑19 datasets that include imagery such as the Italian Radiological Society which has compiled an international online database of imaging findings for confirmed cases. Due to overlap with other infections such as adenovirus, imaging without confirmation by rRT-PCR is of limited specificity in identifying COVID‑19. A large study in China compared chest CT results to PCR and demonstrated that though imaging is less specific for the infection, it is faster and more sensitive.\n",
            "\n",
            "\n",
            "=== Coding ===\n",
            "In late 2019, the WHO assigned emergency ICD-10 disease codes U07.1 for deaths from lab-confirmed SARS-CoV-2 infection and U07.2 for deaths from clinically or epidemiologically diagnosed COVID‑19 without lab-confirmed SARS-CoV-2 infection.\n",
            "\n",
            "\n",
            "=== Pathology ===\n",
            "The main pathological findings at autopsy are:\n",
            "\n",
            "Macroscopy: pericarditis, lung consolidation and pulmonary oedema\n",
            "Lung findings:\n",
            "minor serous exudation, minor fibrin exudation\n",
            "pulmonary oedema, pneumocyte hyperplasia, large atypical pneumocytes, interstitial inflammation with lymphocytic infiltration and multinucleated giant cell formation\n",
            "diffuse alveolar damage (DAD) with diffuse alveolar exudates. DAD is the cause of acute respiratory distress syndrome (ARDS) and severe hypoxemia.\n",
            "organisation of exudates in alveolar cavities and pulmonary interstitial fibrosis\n",
            "plasmocytosis in BAL\n",
            "Blood and vessels: disseminated intravascular coagulation (DIC); leukoerythroblastic reaction, endotheliitis, hemophagocytosis\n",
            "Heart: cardiac muscle cell necrosis\n",
            "Liver: microvesicular steatosis\n",
            "Nose: shedding of olfactory epithelium\n",
            "Brain: infarction\n",
            "Kidneys: acute tubular damage.\n",
            "Spleen: white pulp depletion.\n",
            "\n",
            "\n",
            "== Prevention ==\n",
            "\n",
            "Preventive measures to reduce the chances of infection include getting vaccinated, staying at home, wearing a mask in public, avoiding crowded places, keeping distance from others, ventilating indoor spaces, managing potential exposure durations, washing hands with soap and water often and for at least twenty seconds, practising good respiratory hygiene, and avoiding touching the eyes, nose, or mouth with unwashed hands.Those diagnosed with COVID‑19 or who believe they may be infected are advised by the CDC to stay home except to get medical care, call ahead before visiting a healthcare provider, wear a face mask before entering the healthcare provider's office and when in any room or vehicle with another person, cover coughs and sneezes with a tissue, regularly wash hands with soap and water and avoid sharing personal household items.The first COVID‑19 vaccine was granted regulatory approval on 2 December 2020 by the UK medicines regulator MHRA. It was evaluated for emergency use authorization (EUA) status by the US FDA, and in several other countries. Initially, the US National Institutes of Health guidelines do not recommend any medication for prevention of COVID‑19, before or after exposure to the SARS-CoV-2 virus, outside the setting of a clinical trial. Without a vaccine, other prophylactic measures, or effective treatments, a key part of managing COVID‑19 is trying to decrease and delay the epidemic peak, known as \"flattening the curve\". This is done by slowing the infection rate to decrease the risk of health services being overwhelmed, allowing for better treatment of active cases, and delaying additional cases until effective treatments or a vaccine become available.\n",
            "\n",
            "\n",
            "=== Vaccine ===\n",
            "\n",
            "\n",
            "=== Face masks and respiratory hygiene ===\n",
            "\n",
            "The WHO and the US CDC recommend individuals wear non-medical face coverings in public settings where there is an increased risk of transmission and where social distancing measures are difficult to maintain. This recommendation is meant to reduce the spread of the disease by asymptomatic and pre-symptomatic individuals and is complementary to established preventive measures such as social distancing. Face coverings limit the volume and travel distance of expiratory droplets dispersed when talking, breathing, and coughing. A face covering without vents or holes will also filter out particles containing the virus from inhaled and exhaled air, reducing the chances of infection. But, if the mask include an exhalation valve, a wearer that is infected (maybe without having noticed that, and asymptomatic) would transmit the virus outwards through it, despite any certification they can have. So the masks with exhalation valve are not for the infected wearers, and are not reliable to stop the pandemic in a large scale. Many countries and local jurisdictions encourage or mandate the use of face masks or cloth face coverings by members of the public to limit the spread of the virus.Masks are also strongly recommended for those who may have been infected and those taking care of someone who may have the disease. When not wearing a mask, the CDC recommends covering the mouth and nose with a tissue when coughing or sneezing and recommends using the inside of the elbow if no tissue is available. Proper hand hygiene after any cough or sneeze is encouraged. Healthcare professionals interacting directly with people who have COVID‑19 are advised to use respirators at least as protective as NIOSH-certified N95 or equivalent, in addition to other personal protective equipment.\n",
            "\n",
            "\n",
            "=== Indoor ventilation and avoiding crowded indoor spaces ===\n",
            "The CDC recommends that crowded indoor spaces should be avoided. When indoors, increasing the rate of air change, decreasing recirculation of air and increasing the use of outdoor air can reduce transmission. The WHO recommends ventilation and air filtration in public spaces to help clear out infectious aerosols.Exhaled respiratory particles can build-up within enclosed spaces with inadequate ventilation. The risk of COVID‑19 infection increases especially in spaces where people engage in physical exertion or raise their voice (e.g., exercising, shouting, singing) as this increases exhalation of respiratory droplets. Prolonged exposure to these conditions, typically more than 15 minutes, leads to higher risk of infection.Displacement ventilation with large natural inlets can move stale air directly to the exhaust in laminar flow while significantly reducing the concentration of droplets and particles. Passive ventilation reduces energy consumption and maintenance costs but may lack controllability and heat recovery. Displacement ventilation can also be achieved mechanically with higher energy and maintenance costs. The use of large ducts and openings helps to prevent mixing in closed environments. Recirculation and mixing should be avoided because recirculation prevents dilution of harmful particles and redistributes possibly contaminated air, and mixing increases the concentration and range of infectious particles and keeps larger particles in the air.\n",
            "\n",
            "\n",
            "=== Hand-washing and hygiene ===\n",
            "\n",
            "Thorough hand hygiene after any cough or sneeze is required. The WHO also recommends that individuals wash hands often with soap and water for at least twenty seconds, especially after going to the toilet or when hands are visibly dirty, before eating and after blowing one's nose. When soap and water are not available, the CDC recommends using an alcohol-based hand sanitiser with at least 60% alcohol. For areas where commercial hand sanitisers are not readily available, the WHO provides two formulations for local production. In these formulations, the antimicrobial activity arises from ethanol or isopropanol. Hydrogen peroxide is used to help eliminate bacterial spores in the alcohol; it is \"not an active substance for hand antisepsis.\" Glycerol is added as a humectant.\n",
            "\n",
            "\n",
            "=== Social distancing ===\n",
            "\n",
            "Social distancing (also known as physical distancing) includes infection control actions intended to slow the spread of the disease by minimising close contact between individuals. Methods include quarantines; travel restrictions; and the closing of schools, workplaces, stadiums, theatres, or shopping centres. Individuals may apply social distancing methods by staying at home, limiting travel, avoiding crowded areas, using no-contact greetings, and physically distancing themselves from others. Many governments are now mandating or recommending social distancing in regions affected by the outbreak.Outbreaks have occurred in prisons due to crowding and an inability to enforce adequate social distancing. In the United States, the prisoner population is aging and many of them are at high risk for poor outcomes from COVID‑19 due to high rates of coexisting heart and lung disease, and poor access to high-quality healthcare.\n",
            "\n",
            "\n",
            "=== Surface cleaning ===\n",
            "After being expelled from the body, coronaviruses can survive on surfaces for hours to days. If a person touches the dirty surface, they may deposit the virus at the eyes, nose, or mouth where it can enter the body and cause infection. Evidence indicates that contact with infected surfaces is not the main driver of COVID‑19, leading to recommendations for optimised disinfection procedures to avoid issues such as the increase of antimicrobial resistance through the use of inappropriate cleaning products and processes. Deep cleaning and other surface sanitation has been criticized as hygiene theater, giving a false sense of security against something primarily spread through the air.The amount of time that the virus can survive depends significantly on the type of surface, the temperature, and the humidity. Coronaviruses die very quickly when exposed to the UV light in sunlight. Like other enveloped viruses, SARS-CoV-2 survives longest when the temperature is at room temperature or lower, and when the relative humidity is low (<50%).On many surfaces, including glass, some types of plastic, stainless steel, and skin, the virus can remain infective for several days indoors at room temperature, or even about a week under ideal conditions. On some surfaces, including cotton fabric and copper, the virus usually dies after a few hours. The virus dies faster on porous surfaces than on non-porous surfaces due to capillary action within pores and faster aerosol droplet evaporation. However, of the many surfaces tested, two with the longest survival times are N95 respirator masks and surgical masks, both of which are considered porous surfaces.The CDC says that in most situations, cleaning surfaces with soap or detergent, not disinfecting, is enough to reduce risk of transmission. The CDC recommends that if a COVID‑19 case is suspected or confirmed at a facility such as an office or day care, all areas such as offices, bathrooms, common areas, shared electronic equipment like tablets, touch screens, keyboards, remote controls, and ATM machines used by the ill persons should be disinfected. Surfaces may be decontaminated with 62–71 percent ethanol, 50–100 percent isopropanol, 0.1 percent sodium hypochlorite, 0.5 percent hydrogen peroxide, and 0.2–7.5 percent povidone-iodine. Other solutions, such as benzalkonium chloride and chlorhexidine gluconate, are less effective. Ultraviolet germicidal irradiation may also be used. A datasheet comprising the authorised substances to disinfection in the food industry (including suspension or surface tested, kind of surface, use dilution, disinfectant and inocuylum volumes) can be seen in the supplementary material of.\n",
            "\n",
            "\n",
            "=== Self-isolation ===\n",
            "Self-isolation at home has been recommended for those diagnosed with COVID‑19 and those who suspect they have been infected. Health agencies have issued detailed instructions for proper self-isolation. Many governments have mandated or recommended self-quarantine for entire populations. The strongest self-quarantine instructions have been issued to those in high-risk groups. Those who may have been exposed to someone with COVID‑19 and those who have recently travelled to a country or region with the widespread transmission have been advised to self-quarantine for 14 days from the time of last possible exposure.\n",
            "\n",
            "\n",
            "=== Healthy diet and lifestyle ===\n",
            "The Harvard T.H. Chan School of Public Health recommends a healthy diet, being physically active, managing psychological stress, and getting enough sleep.Consistently meeting scientific guidelines of 150+ minutes per week of exercise or similar physical activity was shown to be associated with a smaller risk of hospitalisation and death due to COVID‑19, even when considering likely risk factors such as elevated Body mass index (BMI).A meta-analysis, published online in October 2021, concluded that \"Vitamin D supplementation in SARS-CoV-2 positive patients has the potential to positively impact patients with both mild and severe symptoms.\" The largest observational study on the subject using online questionnaires, with over 6,000 participants and a dosage regime near the RDI, is set to conclude in July 2021. One of the collaborators in the study is Synergy Biologics Ltd, a manufacturer of vitamin D3 supplements.\n",
            "\n",
            "\n",
            "=== International travel-related control measures ===\n",
            "A 2021 Cochrane rapid review found that based upon low-certainty evidence, international travel-related control measures such as restricting cross-border travel may help to contain the spread of COVID‑19. Additionally, symptom/exposure-based screening measures at borders may miss many positive cases. While test-based border screening measures may be more effective, it could also miss many positive cases if only conducted upon arrival without follow-up. The review concluded that a minimum 10-day quarantine may be beneficial in preventing the spread of COVID‑19 and may be more effective if combined with an additional control measure like border screening.\n",
            "\n",
            "\n",
            "== Treatment ==\n",
            "\n",
            "\n",
            "== Prognosis ==\n",
            "\n",
            "The severity of COVID‑19 varies. The disease may take a mild course with few or no symptoms, resembling other common upper respiratory diseases such as the common cold. In 3–4% of cases (7.4% for those over age 65) symptoms are severe enough to cause hospitalization. Mild cases typically recover within two weeks, while those with severe or critical diseases may take three to six weeks to recover. Among those who have died, the time from symptom onset to death has ranged from two to eight weeks. The Italian Istituto Superiore di Sanità reported that the median time between the onset of symptoms and death was twelve days, with seven being hospitalised. However, people transferred to an ICU had a median time of ten days between hospitalisation and death. Prolonged prothrombin time and elevated C-reactive protein levels on admission to the hospital are associated with severe course of COVID‑19 and with a transfer to ICU.Some early studies suggest 10% to 20% of people with COVID‑19 will experience symptoms lasting longer than a month. A majority of those who were admitted to hospital with severe disease report long-term problems including fatigue and shortness of breath. On 30 October 2020, WHO chief Tedros Adhanom warned that \"to a significant number of people, the COVID virus poses a range of serious long-term effects.\" He has described the vast spectrum of COVID‑19 symptoms that fluctuate over time as \"really concerning\". They range from fatigue, a cough and shortness of breath, to inflammation and injury of major organs – including the lungs and heart, and also neurological and psychologic effects. Symptoms often overlap and can affect any system in the body. Infected people have reported cyclical bouts of fatigue, headaches, months of complete exhaustion, mood swings, and other symptoms. Tedros therefore concluded that a strategy of achieving herd immunity by infection, rather than vaccination, is \"morally unconscionable and unfeasible\".In terms of hospital readmissions about 9% of 106,000 individuals had to return for hospital treatment within two months of discharge. The average to readmit was eight days since first hospital visit. There are several risk factors that have been identified as being a cause of multiple admissions to a hospital facility. Among these are advanced age (above 65 years of age) and presence of a chronic condition such as diabetes, COPD, heart failure or chronic kidney disease.According to scientific reviews smokers are more likely to require intensive care or die compared to non-smokers. Acting on the same ACE2 pulmonary receptors affected by smoking, air pollution has been correlated with the disease. Short term and chronic exposure to air pollution seems to enhance morbidity and mortality from COVID‑19. Pre-existing heart and lung diseases and also obesity, especially in conjunction with fatty liver disease, contributes to an increased health risk of COVID‑19.It is also assumed that those that are immunocompromised are at higher risk of getting severely sick from SARS-CoV-2. One research that looked into the COVID‑19 infections in hospitalized kidney transplant recipients found a mortality rate of 11%.Genetics also plays an important role in the ability to fight off the disease. For instance, those that do not produce detectable type I interferons or produce auto-antibodies against these may get much sicker from COVID‑19. Genetic screening is able to detect interferon effector genes.\n",
            "\n",
            "\n",
            "=== Children ===\n",
            "\n",
            "While very young children have experienced lower rates of infection, older children have a rate of infection that is similar to the population as a whole. Children are likely to have milder symptoms and are at lower risk of severe disease than adults. The CDC reports that in the US roughly a third of hospitalized children were admitted to the ICU, while a European multinational study of hospitalized children from June 2020 found that about 8% of children admitted to a hospital needed intensive care. Four of the 582 children (0.7%) in the European study died, but the actual mortality rate may be \"substantially lower\" since milder cases that did not seek medical help were not included in the study.\n",
            "\n",
            "\n",
            "=== Complications ===\n",
            "\n",
            "Complications may include pneumonia, acute respiratory distress syndrome (ARDS), multi-organ failure, septic shock, and death. Cardiovascular complications may include heart failure, arrhythmias (including atrial fibrillation), heart inflammation, and thrombosis, particularly venous thromboembolism. Approximately 20–30% of people who present with COVID‑19 have elevated liver enzymes, reflecting liver injury.Neurologic manifestations include seizure, stroke, encephalitis, and Guillain–Barré syndrome (which includes loss of motor functions). Following the infection, children may develop paediatric multisystem inflammatory syndrome, which has symptoms similar to Kawasaki disease, which can be fatal. In very rare cases, acute encephalopathy can occur, and it can be considered in those who have been diagnosed with COVID‑19 and have an altered mental status.In the case of pregnant women, it is important to note that, according to the Centers for Disease Control and Prevention, pregnant women are at increased risk of becoming seriously ill from COVID‑19. This is because pregnant women with COVID‑19 appear to be more likely to develop respiratory and obstetric complications that can lead to miscarriage, premature delivery and intrauterine growth restriction.Fungal infections such as aspergillosis, candidiasis, cryptococcosis and mucormycosis have been recorded in patients recovering from COVID‑19.\n",
            "\n",
            "\n",
            "=== Longer-term effects ===\n",
            "\n",
            "Some early studies suggest that 10–20% of people with COVID‑19 will experience symptoms lasting longer than a month. A majority of those who were admitted to hospital with severe disease report long-term problems, including fatigue and shortness of breath. About 5–10% of patients admitted to hospital progress to severe or critical disease, including pneumonia and acute respiratory failure.By a variety of mechanisms, the lungs are the organs most affected in COVID‑19. In people requiring hospital admission, up to 98% of CT scans performed show lung abnormalities after 28 days of illness even if they had clinically improved.People with advanced age, severe disease, prolonged ICU stays, or who smoke are more likely to have long-lasting effects, including pulmonary fibrosis. Overall, approximately one-third of those investigated after four weeks will have findings of pulmonary fibrosis or reduced lung function as measured by DLCO, even in asymptomatic people, but with the suggestion of continuing improvement with the passing of more time.\n",
            "\n",
            "\n",
            "=== Immunity ===\n",
            "\n",
            "The immune response by humans to SARS-CoV-2 virus occurs as a combination of the cell-mediated immunity and antibody production, just as with most other infections. B cells interact with T cells and begin dividing before selection into the plasma cell, partly on the basis of their affinity for antigen. Since SARS-CoV-2 has been in the human population only since December 2019, it remains unknown if the immunity is long-lasting in people who recover from the disease. The presence of neutralizing antibodies in blood strongly correlates with protection from infection, but the level of neutralizing antibody declines with time. Those with asymptomatic or mild disease had undetectable levels of neutralizing antibody two months after infection. In another study, the level of neutralizing antibodies fell four-fold one to four months after the onset of symptoms. However, the lack of antibodies in the blood does not mean antibodies will not be rapidly produced upon reexposure to SARS-CoV-2. Memory B cells specific for the spike and nucleocapsid proteins of SARS-CoV-2 last for at least six months after the appearance of symptoms.Reinfection with COVID‑19 is possible but uncommon. The first case of reinfection was documented in August 2020. A systematic review found 17 cases of confirmed reinfection in medical literature as of May 2021.\n",
            "\n",
            "\n",
            "== Mortality ==\n",
            "\n",
            "Several measures are commonly used to quantify mortality. These numbers vary by region and over time and are influenced by the volume of testing, healthcare system quality, treatment options, time since the initial outbreak, and population characteristics such as age, sex, and overall health.The mortality rate reflects the number of deaths within a specific demographic group divided by the population of that demographic group. Consequently, the mortality rate reflects the prevalence as well as the severity of the disease within a given population. Mortality rates are highly correlated to age, with relatively low rates for young people and relatively high rates among the elderly. In fact, one relevant factor of mortality rates is the age structure of the countries' populations. For example, the case fatality rate for COVID‑19 is lower in India than in the US since India's younger population represents a larger percentage than in the US.\n",
            "\n",
            "\n",
            "=== Case fatality rate ===\n",
            "The case fatality rate (CFR) reflects the number of deaths divided by the number of diagnosed cases within a given time interval. Based on Johns Hopkins University statistics, the global death-to-case ratio is 1.24% (6,183,197/500,280,791) as of 12 April 2022. The number varies by region.\n",
            "\n",
            "\t\t\n",
            "\t\t\n",
            "\t\t\n",
            "\n",
            "\n",
            "=== Infection fatality rate ===\n",
            "A key metric in gauging the severity of COVID‑19 is the infection fatality rate (IFR), also referred to as the infection fatality ratio or infection fatality risk. This metric is calculated by dividing the total number of deaths from the disease by the total number of infected individuals; hence, in contrast to the CFR, the IFR incorporates asymptomatic and undiagnosed infections as well as reported cases.\n",
            "\n",
            "\n",
            "==== Estimates ====\n",
            "\n",
            "A December 2020 systematic review and meta-analysis estimated that population IFR during the first wave of the pandemic was about 0.5% to 1% in many locations (including France, Netherlands, New Zealand, and Portugal), 1% to 2% in other locations (Australia, England, Lithuania, and Spain), and exceeded 2% in Italy. That study also found that most of these differences in IFR reflected corresponding differences in the age composition of the population and age-specific infection rates; in particular, the metaregression estimate of IFR is very low for children and younger adults (e.g., 0.002% at age 10 and 0.01% at age 25) but increases progressively to 0.4% at age 55, 1.4% at age 65, 4.6% at age 75, and 15% at age 85. These results were also highlighted in a December 2020 report issued by the WHO.\n",
            "An analysis of those IFR rates indicates that COVID‑19 is hazardous not only for the elderly but also for middle-aged adults, for whom the infection fatality rate of COVID-19 is two orders of magnitude greater than the annualized risk of a fatal automobile accident and far more dangerous than seasonal influenza.\n",
            "\n",
            "\n",
            "==== Earlier estimates of IFR ====\n",
            "At an early stage of the pandemic, the World Health Organization reported estimates of IFR between 0.3% and 1%. On 2 July, The WHO's chief scientist reported that the average IFR estimate presented at a two-day WHO expert forum was about 0.6%. In August, the WHO found that studies incorporating data from broad serology testing in Europe showed IFR estimates converging at approximately 0.5–1%. Firm lower limits of IFRs have been established in a number of locations such as New York City and Bergamo in Italy since the IFR cannot be less than the population fatality rate. (After sufficient time however, people can get reinfected). As of 10 July, in New York City, with a population of 8.4 million, 23,377 individuals (18,758 confirmed and 4,619 probable) have died with COVID‑19 (0.3% of the population). Antibody testing in New York City suggested an IFR of ~0.9%, and ~1.4%. In Bergamo province, 0.6% of the population has died. In September 2020, the U.S. Centers for Disease Control and Prevention (CDC) reported preliminary estimates of age-specific IFRs for public health planning purposes.\n",
            "\n",
            "\n",
            "=== Sex differences ===\n",
            "\n",
            "COVID‑19 case fatality rates are higher among men than women in most countries. However, in a few countries like India, Nepal, Vietnam, and Slovenia the fatality cases are higher in women than men. Globally, men are more likely to be admitted to the ICU and more likely to die. One meta-analysis found that globally, men were more likely to get COVID‑19 than women; there were approximately 55 men and 45 women per 100 infections (CI: 51.43–56.58).The Chinese Center for Disease Control and Prevention reported the death rate was 2.8% for men and 1.7% for women. Later reviews in June 2020 indicated that there is no significant difference in susceptibility or in CFR between genders. One review acknowledges the different mortality rates in Chinese men, suggesting that it may be attributable to lifestyle choices such as smoking and drinking alcohol rather than genetic factors. Smoking, which in some countries like China is mainly a male activity, is a habit that contributes to increasing significantly the case fatality rates among men. Sex-based immunological differences, lesser prevalence of smoking in women and men developing co-morbid conditions such as hypertension at a younger age than women could have contributed to the higher mortality in men. In Europe as of February 2020, 57% of the infected people were men and 72% of those died with COVID‑19 were men. As of April 2020, the US government is not tracking sex-related data of COVID‑19 infections. Research has shown that viral illnesses like Ebola, HIV, influenza and SARS affect men and women differently.\n",
            "\n",
            "\n",
            "=== Ethnic differences ===\n",
            "In the US, a greater proportion of deaths due to COVID‑19 have occurred among African Americans and other minority groups. Structural factors that prevent them from practicing social distancing include their concentration in crowded substandard housing and in \"essential\" occupations such as retail grocery workers, public transit employees, health-care workers and custodial staff. Greater prevalence of lacking health insurance and care of underlying conditions such as diabetes, hypertension, and heart disease also increase their risk of death. Similar issues affect Native American and Latino communities. On the one hand, in the Dominican Republic there is a clear example of both gender and ethnic inequality. In this Latin American territory, there is great inequality and precariousness that especially affects Dominican women, with greater emphasis on those of Haitian descent. According to a US health policy non-profit, 34% of American Indian and Alaska Native People (AIAN) non-elderly adults are at risk of serious illness compared to 21% of white non-elderly adults. The source attributes it to disproportionately high rates of many health conditions that may put them at higher risk as well as living conditions like lack of access to clean water.Leaders have called for efforts to research and address the disparities. In the UK, a greater proportion of deaths due to COVID‑19 have occurred in those of a Black, Asian, and other ethnic minority background. More severe impacts upon victims including the relative incidence of the necessity of hospitalization requirements, and vulnerability to the disease has been associated via DNA analysis to be expressed in genetic variants at chromosomal region 3, features that are associated with European Neanderthal heritage. That structure imposes greater risks that those affected will develop a more severe form of the disease. The findings are from Professor Svante Pääbo and researchers he leads at the Max Planck Institute for Evolutionary Anthropology and the Karolinska Institutet. This admixture of modern human and Neanderthal genes is estimated to have occurred roughly between 50,000 and 60,000 years ago in Southern Europe.\n",
            "\n",
            "\n",
            "=== Comorbidities ===\n",
            "Biological factors (immune response) and the general behaviour (habits) can strongly determine the consequences of COVID‑19. Most of those who die of COVID‑19 have pre-existing (underlying) conditions, including hypertension, diabetes mellitus, and cardiovascular disease. According to March data from the United States, 89% of those hospitalised had preexisting conditions. The Italian Istituto Superiore di Sanità reported that out of 8.8% of deaths where medical charts were available, 96.1% of people had at least one comorbidity with the average person having 3.4 diseases. According to this report the most common comorbidities are hypertension (66% of deaths), type 2 diabetes (29.8% of deaths), Ischemic Heart Disease (27.6% of deaths), atrial fibrillation (23.1% of deaths) and chronic renal failure (20.2% of deaths).\n",
            "Most critical respiratory comorbidities according to the Centers for Disease Control and Prevention (CDC), are: moderate or severe asthma, pre-existing COPD, pulmonary fibrosis, cystic fibrosis. Evidence stemming from meta-analysis of several smaller research papers also suggests that smoking can be associated with worse outcomes. When someone with existing respiratory problems is infected with COVID‑19, they might be at greater risk for severe symptoms. COVID‑19 also poses a greater risk to people who misuse opioids and methamphetamines, insofar as their drug use may have caused lung damage.In August 2020, the CDC issued a caution that tuberculosis (TB) infections could increase the risk of severe illness or death. The WHO recommended that people with respiratory symptoms be screened for both diseases, as testing positive for COVID‑19 could not rule out co-infections. Some projections have estimated that reduced TB detection due to the pandemic could result in 6.3 million additional TB cases and 1.4 million TB-related deaths by 2025.\n",
            "\n",
            "\n",
            "== History ==\n",
            "\n",
            "The virus is thought to be of natural animal origin, most likely through spillover infection. There are several theories about where the index case originated and investigations into the origin of the pandemic are ongoing. Phylogenetics estimates that SARS-CoV-2 arose in October or November 2019. A phylogenetic algorithm analysis suggested that the virus may have been circulating in Guangdong before Wuhan. Evidence suggests that it descends from a coronavirus that infects wild bats, and spread to humans through an intermediary wildlife host. The possibility that the virus was accidentally released from a laboratory is also under increasingly active consideration. U.S intelligence agencies found that the virus was not developed as a biological weapon and that it is unlikely for it to have been genetically engineered.The first confirmed human infections were in Wuhan, Hubei, China. A study of the first 41 cases of confirmed COVID‑19, published in January 2020 in The Lancet, reported the earliest date of onset of symptoms as 1 December 2019. Official publications from the WHO reported the earliest onset of symptoms as 8 December 2019. Human-to-human transmission was confirmed by the WHO and Chinese authorities by 20 January 2020. According to official Chinese sources, these were mostly linked to the Huanan Seafood Wholesale Market, which also sold live animals. In May 2020, George Gao, the director of the CDC, said animal samples collected from the seafood market had tested negative for the virus, indicating that the market was the site of an early superspreading event, but that it was not the site of the initial outbreak. Traces of the virus have been found in wastewater samples that were collected in Milan and Turin, Italy, on 18 December 2019.By December 2019, the spread of infection was almost entirely driven by human-to-human transmission. The number of COVID-19 cases in Hubei gradually increased, reaching sixty by 20 December, and at least 266 by 31 December. On 24 December, Wuhan Central Hospital sent a bronchoalveolar lavage fluid (BAL) sample from an unresolved clinical case to sequencing company Vision Medicals. On 27 and 28 December, Vision Medicals informed the Wuhan Central Hospital and the Chinese CDC of the results of the test, showing a new coronavirus. A pneumonia cluster of unknown cause was observed on 26 December and treated by the doctor Zhang Jixian in Hubei Provincial Hospital, who informed the Wuhan Jianghan CDC on 27 December. On 30 December, a test report addressed to Wuhan Central Hospital, from company CapitalBio Medlab, stated an erroneous positive result for SARS, causing a group of doctors at Wuhan Central Hospital to alert their colleagues and relevant hospital authorities of the result. The Wuhan Municipal Health Commission issued a notice to various medical institutions on \"the treatment of pneumonia of unknown cause\" that same evening. Eight of these doctors, including Li Wenliang (punished on 3 January), were later admonished by the police for spreading false rumours and another, Ai Fen, was reprimanded by her superiors for raising the alarm.The Wuhan Municipal Health Commission made the first public announcement of a pneumonia outbreak of unknown cause on 31 December, confirming 27 cases – enough to trigger an investigation.During the early stages of the outbreak, the number of cases doubled approximately every seven and a half days. In early and mid-January 2020, the virus spread to other Chinese provinces, helped by the Chinese New Year migration and Wuhan being a transport hub and major rail interchange. On 20 January, China reported nearly 140 new cases in one day, including two people in Beijing and one in Shenzhen. Later official data shows 6,174 people had already developed symptoms by then, and more may have been infected. A report in The Lancet on 24 January indicated human transmission, strongly recommended personal protective equipment for health workers, and said testing for the virus was essential due to its \"pandemic potential\". On 30 January, the WHO declared COVID-19 a Public Health Emergency of International Concern. By this time, the outbreak spread by a factor of 100 to 200 times.Italy had its first confirmed cases on 31 January 2020, two tourists from China. Italy overtook China as the country with the most deaths on 19 March 2020. By 26 March the United States had overtaken China and Italy with the highest number of confirmed cases in the world. Research on coronavirus genomes indicates the majority of COVID-19 cases in New York came from European travellers, rather than directly from China or any other Asian country. Retesting of prior samples found a person in France who had the virus on 27 December 2019, and a person in the United States who died from the disease on 6 February 2020.RT-PCR testing of untreated wastewater samples from Brazil and Italy have suggested detection of SARS-CoV-2 as early as November and December 2019, respectively, but the methods of such sewage studies have not been optimised, many have not been peer-reviewed, details are often missing, and there is a risk of false positives due to contamination or if only one gene target is detected. A September 2020 review journal article said, \"The possibility that the COVID‑19 infection had already spread to Europe at the end of last year is now indicated by abundant, even if partially circumstantial, evidence,\" including pneumonia case numbers and radiology in France and Italy in November and December.As of 1 October 2021, Reuters reported that it had estimated the worldwide total number of deaths due to COVID‑19 to have exceeded five million.\n",
            "\n",
            "\n",
            "== Misinformation ==\n",
            "\n",
            "After the initial outbreak of COVID‑19, misinformation and disinformation regarding the origin, scale, prevention, treatment, and other aspects of the disease rapidly spread online.In September 2020, the US Centers for Disease Control and Prevention (CDC) published preliminary estimates of the risk of death by age groups in the United States, but those estimates were widely misreported and misunderstood.\n",
            "\n",
            "\n",
            "== Other species ==\n",
            "\n",
            "Humans appear to be capable of spreading the virus to some other animals, a type of disease transmission referred to as zooanthroponosis.\n",
            "Some pets, especially cats and ferrets, can catch this virus from infected humans. Symptoms in cats include respiratory (such as a cough) and digestive symptoms. Cats can spread the virus to other cats, and may be able to spread the virus to humans, but cat-to-human transmission of SARS-CoV-2 has not been proven. Compared to cats, dogs are less susceptible to this infection. Behaviors which increase the risk of transmission include kissing, licking, and petting the animal.The virus does not appear to be able to infect pigs, ducks, or chickens at all. Mice, rats, and rabbits, if they can be infected at all, are unlikely to be involved in spreading the virus.Tigers and lions in zoos have become infected as a result of contact with infected humans. As expected, monkeys and great ape species such as orangutans can also be infected with the COVID‑19 virus.Minks, which are in the same family as ferrets, have been infected. Minks may be asymptomatic, and can also spread the virus to humans. Multiple countries have identified infected animals in mink farms. Denmark, a major producer of mink pelts, ordered the slaughter of all minks over fears of viral mutations, following an outbreak referred to as Cluster 5. A vaccine for mink and other animals is being researched.\n",
            "\n",
            "\n",
            "== Research ==\n",
            "\n",
            "International research on vaccines and medicines in COVID‑19 is underway by government organisations, academic groups, and industry researchers. The CDC has classified it to require a BSL3 grade laboratory. There has been a great deal of COVID‑19 research, involving accelerated research processes and publishing shortcuts to meet the global demand.As of December 2020, hundreds of clinical trials have been undertaken, with research happening on every continent except Antarctica. As of November 2020, more than 200 possible treatments had been studied in humans so far.\n",
            "\n",
            "\n",
            "=== Transmission and prevention research ===\n",
            "\n",
            "Modelling research has been conducted with several objectives, including predictions of the dynamics of transmission, diagnosis and prognosis of infection, estimation of the impact of interventions, or allocation of resources. Modelling studies are mostly based on compartmental models in epidemiology, estimating the number of infected people over time under given conditions. Several other types of models have been developed and used during the COVID‑19 including computational fluid dynamics models to study the flow physics of COVID‑19, retrofits of crowd movement models to study occupant exposure, mobility-data based models to investigate transmission, or the use of macroeconomic models to assess the economic impact of the pandemic. Further, conceptual frameworks from crisis management research have been applied to better understand the effects of COVID‑19 on organizations worldwide.\n",
            "\n",
            "\n",
            "=== Treatment-related research ===\n",
            "\n",
            "Repurposed antiviral drugs make up most of the research into COVID‑19 treatments. Other candidates in trials include vasodilators, corticosteroids, immune therapies, lipoic acid, bevacizumab, and recombinant angiotensin-converting enzyme 2.In March 2020, the World Health Organization (WHO) initiated the Solidarity trial to assess the treatment effects of some promising drugs: an experimental drug called remdesivir; anti-malarial drugs chloroquine and hydroxychloroquine; two anti-HIV drugs, lopinavir/ritonavir; and interferon-beta. More than 300 active clinical trials are underway as of April 2020.Research on the antimalarial drugs hydroxychloroquine and chloroquine showed that they were ineffective at best, and that they may reduce the antiviral activity of remdesivir. By May 2020, France, Italy, and Belgium had banned the use of hydroxychloroquine as a COVID‑19 treatment.In June, initial results from the randomised RECOVERY Trial in the United Kingdom showed that dexamethasone reduced mortality by one third for people who are critically ill on ventilators and one fifth for those receiving supplemental oxygen. Because this is a well-tested and widely available treatment, it was welcomed by the WHO, which is in the process of updating treatment guidelines to include dexamethasone and other steroids. Based on those preliminary results, dexamethasone treatment has been recommended by the NIH for patients with COVID‑19 who are mechanically ventilated or who require supplemental oxygen but not in patients with COVID‑19 who do not require supplemental oxygen.In September 2020, the WHO released updated guidance on using corticosteroids for COVID‑19. The WHO recommends systemic corticosteroids rather than no systemic corticosteroids for the treatment of people with severe and critical COVID‑19 (strong recommendation, based on moderate certainty evidence). The WHO suggests not to use corticosteroids in the treatment of people with non-severe COVID‑19 (conditional recommendation, based on low certainty evidence). The updated guidance was based on a meta-analysis of clinical trials of critically ill COVID‑19 patients.In September 2020, the European Medicines Agency (EMA) endorsed the use of dexamethasone in adults and adolescents from twelve years of age and weighing at least 40 kilograms (88 lb) who require supplemental oxygen therapy. Dexamethasone can be taken by mouth or given as an injection or infusion (drip) into a vein.In November 2020, the US Food and Drug Administration (FDA) issued an emergency use authorization for the investigational monoclonal antibody therapy bamlanivimab for the treatment of mild-to-moderate COVID‑19. Bamlanivimab is authorized for people with positive results of direct SARS-CoV-2 viral testing who are twelve years of age and older weighing at least 40 kilograms (88 lb), and who are at high risk for progressing to severe COVID‑19 or hospitalization. This includes those who are 65 years of age or older, or who have chronic medical conditions.In February 2021, the FDA issued an emergency use authorization (EUA) for bamlanivimab and etesevimab administered together for the treatment of mild to moderate COVID‑19 in people twelve years of age or older weighing at least 40 kilograms (88 lb) who test positive for SARS‑CoV‑2 and who are at high risk for progressing to severe COVID‑19. The authorized use includes treatment for those who are 65 years of age or older or who have certain chronic medical conditions.In April 2021, the FDA revoked the emergency use authorization (EUA) that allowed for the investigational monoclonal antibody therapy bamlanivimab, when administered alone, to be used for the treatment of mild-to-moderate COVID‑19 in adults and certain pediatric patients.\n",
            "\n",
            "\n",
            "==== Cytokine storm ====\n",
            "\n",
            "A cytokine storm can be a complication in the later stages of severe COVID‑19. A cytokine storm is a potentially deadly immune reaction where a large amount of pro-inflammatory cytokines and chemokines are released too quickly. A cytokine storm can lead to ARDS and multiple organ failure. Data collected from Jin Yin-tan Hospital in Wuhan, China indicates that patients who had more severe responses to COVID‑19 had greater amounts of pro-inflammatory cytokines and chemokines in their system than patients who had milder responses. These high levels of pro-inflammatory cytokines and chemokines indicate presence of a cytokine storm.Tocilizumab has been included in treatment guidelines by China's National Health Commission after a small study was completed. It is undergoing a Phase II non-randomised trial at the national level in Italy after showing positive results in people with severe disease. Combined with a serum ferritin blood test to identify a cytokine storm (also called cytokine storm syndrome, not to be confused with cytokine release syndrome), it is meant to counter such developments, which are thought to be the cause of death in some affected people. The interleukin-6 receptor (IL-6R) antagonist was approved by the FDA to undergo a Phase III clinical trial assessing its effectiveness on COVID‑19 based on retrospective case studies for the treatment of steroid-refractory cytokine release syndrome induced by a different cause, CAR T cell therapy, in 2017. There is no randomised, controlled evidence that tocilizumab is an efficacious treatment for CRS. Prophylactic tocilizumab has been shown to increase serum IL-6 levels by saturating the IL-6R, driving IL-6 across the blood-brain barrier, and exacerbating neurotoxicity while having no effect on the incidence of CRS.Lenzilumab, an anti-GM-CSF monoclonal antibody, is protective in murine models for CAR T cell-induced CRS and neurotoxicity and is a viable therapeutic option due to the observed increase of pathogenic GM-CSF secreting T cells in hospitalised patients with COVID‑19.\n",
            "\n",
            "\n",
            "==== Passive antibodies ====\n",
            "\n",
            "Transferring purified and concentrated antibodies produced by the immune systems of those who have recovered from COVID‑19 to people who need them is being investigated as a non-vaccine method of passive immunisation. Viral neutralization is the anticipated mechanism of action by which passive antibody therapy can mediate defence against SARS-CoV-2. The spike protein of SARS-CoV-2 is the primary target for neutralizing antibodies. As of 8 August 2020, eight neutralizing antibodies targeting the spike protein of SARS-CoV-2 have entered clinical studies. It has been proposed that selection of broad-neutralizing antibodies against SARS-CoV-2 and SARS-CoV might be useful for treating not only COVID‑19 but also future SARS-related CoV infections. Other mechanisms, however, such as antibody-dependent cellular cytotoxicity or phagocytosis, may be possible. Other forms of passive antibody therapy, for example, using manufactured monoclonal antibodies, are in development.The use of passive antibodies to treat people with active COVID‑19 is also being studied. This involves the production of convalescent serum, which consists of the liquid portion of the blood from people who recovered from the infection and contains antibodies specific to this virus, which is then administered to active patients. This strategy was tried for SARS with inconclusive results. An updated Cochrane review in May 2021 found high certainty evidence that, for the treatment of people with moderate to severe COVID‑19, convalescent plasma did not reduce mortality or bring about symptom improvement. There continues to be uncertainty about the safety of convalescent plasma administration to people with COVID‑19 and differing outcomes measured in different studies limits their use in determining efficacy.\n",
            "\n",
            "\n",
            "=== Bioethics ===\n",
            "Since the outbreak of the COVID‑19 pandemic, scholars have explored the bioethics, normative economics, and political theories of healthcare policies related to the public health crisis. Academics have pointed to the moral distress of healthcare workers, ethics of distributing scarce healthcare resources such as ventilators, and the global justice of vaccine diplomacies. The socio-economic inequalities between genders, races, groups with disabilities, communities, regions, countries, and continents have also drawn attention in academia and the general public.\n",
            "\n",
            "\n",
            "== Effect on other diseases and the pharmacy trade ==\n",
            "There was a report on 3 March 2021, that social distancing and common wearing of surgical masks and similar as a common precaution against COVID‑19 caused a drop in the spread rate of the common cold and flu.\n",
            "\n",
            "\n",
            "== See also ==\n",
            "Coronavirus diseases, a group of closely related syndromes\n",
            "Disease X, a WHO term\n",
            "Law of declining virulence – Claim by epidemiologist Theobald Smith\n",
            "Theory of virulence – Theory by biologist Paul W. Ewald\n",
            "\n",
            "\n",
            "== References ==\n",
            "\n",
            "\n",
            "== Further reading ==\n",
            "\n",
            "\n",
            "== External links ==\n",
            "\n",
            "\n",
            "=== Health agencies ===\n",
            "Coronavirus disease (COVID‑19) Facts by the World Health Organization (WHO)\n",
            "Coronavirus 2019 (COVID-19) by the US Centers for Disease Control and Prevention (CDC)\n",
            "Coronavirus (COVID‑19) by the UK National Health Service (NHS)\n",
            "\n",
            "\n",
            "=== Directories ===\n",
            "Coronavirus Resource Center at the Center for Inquiry\n",
            "COVID-19 at Curlie\n",
            "COVID‑19 Resource Directory on OpenMD\n",
            "COVID‑19 Information on FireMountain.net\n",
            "\n",
            "\n",
            "=== Medical journals ===\n",
            "Coronavirus Disease 2019 (COVID‑19) by JAMA\n",
            "BMJ's Coronavirus (covid‑19) Hub by the BMJ\n",
            "Novel Coronavirus Information Center by Elsevier\n",
            "COVID‑19 Resource Centre by The Lancet\n",
            "Coronavirus (COVID‑19) Research Highlights by Springer Nature\n",
            "Coronavirus (Covid‑19) by The New England Journal of Medicine\n",
            "Covid‑19: Novel Coronavirus by Wiley Publishing\n",
            "\n",
            "\n",
            "=== Treatment guidelines ===\n",
            "\"JHMI Clinical Recommendations for Available Pharmacologic Therapies for COVID-19\" (PDF). Johns Hopkins Medicine.\n",
            "\"Bouncing Back From COVID-19: Your Guide to Restoring Movement\" (PDF). Johns Hopkins Medicine.\n",
            "\"Guidelines on the Treatment and Management of Patients with COVID-19\". Infectious Diseases Society of America.\n",
            "\"Coronavirus Disease 2019 (COVID-19) Treatment Guidelines\" (PDF). National Institutes of Health.\n",
            "World Health Organization (2022). Therapeutics and COVID-19: living guideline, 14 January 2022 (Report). hdl:10665/351006. WHO/2019-nCoV/therapeutics/2022.1.\n",
            "NHS England and NHS Improvement. National Guidance for post-COVID syndrome assessment clinics (Report).\n"
          ]
        }
      ],
      "source": [
        "# Outputting 1st document\n",
        "print(documents[0])"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "11S_SNBATnkO"
      },
      "source": [
        "#### **Section 2.2: Creating the index**<br> \n",
        "The next series of steps creates an index (i.e. documents index) by obtaining requisite quantifiers such as term frequencies _(tf)_, document frequencies _(df)_, term-document pair _(t-d)_ dictionary, and inverse document frequencies _(idf)_."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "iCtZQLRP-ene"
      },
      "outputs": [],
      "source": [
        "# Creating document terms\n",
        "document_terms = [doc.split(' ') for doc in documents]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "r4_IGZ5KfbEu"
      },
      "outputs": [],
      "source": [
        "# Removing stop words and vectorizing\n",
        "vectorizer = CountVectorizer(stop_words= {'00', '000',\t'00011',\t'001',\t'002',\t'006',\t'009',\t'00906',\t'00937', 'δ156',\t'δ211',\t'δ31',\t'δ69',\t\n",
        "                             '01',\t'010',\t'011',\t'012',\t'013',\t'015',\t'016',\t'017',\t'019',\t'δfvi',\t'δh69',\t'δv70',\t'κορώνη',\t'ṣaḥn',\t'ἰός', \n",
        "                             'áder',\t'áñez',\t'édouard',\t'état',\t'être',\t'óscar',\t'δ105',\t'δ1265',\t'δ143', '02',\t'020',\t'021',\t'02140',\t'023',\n",
        "                             '025',\t'0257',\t'026',\t'028',\t'english'})\n",
        "documents_vectorized = vectorizer.fit_transform(documents)\n",
        "vocabulary = vectorizer.get_feature_names_out()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "kpBxNoFZfvG-",
        "outputId": "a14b4a06-6e9e-4734-9aa4-8e6508041e5e"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "array(['03', '030', '035', ..., 'zulia', 'zycov', 'zürich'], dtype=object)"
            ]
          },
          "execution_count": 42,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Peeking vocabulary data type \n",
        "vocabulary"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "QFS-7d1Cf_7z",
        "outputId": "1200fecf-3d08-48dd-ab7d-80a61ba8f9d7"
      },
      "outputs": [
        {
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-3417e680-d74e-46bb-b3e4-76d6abdca450\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>03</th>\n",
              "      <th>030</th>\n",
              "      <th>035</th>\n",
              "      <th>03614</th>\n",
              "      <th>037</th>\n",
              "      <th>04</th>\n",
              "      <th>041</th>\n",
              "      <th>042</th>\n",
              "      <th>043</th>\n",
              "      <th>045</th>\n",
              "      <th>...</th>\n",
              "      <th>zones</th>\n",
              "      <th>zooanthroponosis</th>\n",
              "      <th>zoom</th>\n",
              "      <th>zoonosis</th>\n",
              "      <th>zoonotic</th>\n",
              "      <th>zoos</th>\n",
              "      <th>zoster</th>\n",
              "      <th>zulia</th>\n",
              "      <th>zycov</th>\n",
              "      <th>zürich</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>10</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>12</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>13</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>14</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>15</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>6</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>16</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>17</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>18</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>19</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>20</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>21</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>22</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>23</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>24</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>25</th>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>26</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>27</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>28</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>11</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>29</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>30 rows × 13249 columns</p>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-3417e680-d74e-46bb-b3e4-76d6abdca450')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-3417e680-d74e-46bb-b3e4-76d6abdca450 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-3417e680-d74e-46bb-b3e4-76d6abdca450');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "    03  030  035  03614  037  04  041  042  043  045  ...  zones  \\\n",
              "0    0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "1    0    0    0      0    0   0    1    0    0    0  ...      0   \n",
              "2    0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "3    0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "4    0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "5    0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "6    2    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "7    0    0    0      0    0   0    0    0    1    0  ...      0   \n",
              "8    0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "9    0    0    0      0    1   0    0    1    0    0  ...      0   \n",
              "10   0    0    0      0    0   0    0    0    0    1  ...      0   \n",
              "11   0    0    1      0    0   0    0    0    0    0  ...      0   \n",
              "12   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "13   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "14   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "15   0    0    0      0    0   0    0    0    0    0  ...      1   \n",
              "16   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "17   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "18   0    0    0      1    0   0    0    0    0    0  ...      0   \n",
              "19   0    0    0      0    0   1    0    0    0    0  ...      0   \n",
              "20   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "21   0    0    0      0    0   1    0    0    0    0  ...      0   \n",
              "22   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "23   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "24   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "25   0    1    0      0    0   0    0    0    0    0  ...      0   \n",
              "26   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "27   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "28   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "29   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "\n",
              "    zooanthroponosis  zoom  zoonosis  zoonotic  zoos  zoster  zulia  zycov  \\\n",
              "0                  1     0         0         1     1       0      0      0   \n",
              "1                  0     0         0         0     0       0      0      0   \n",
              "2                  0     0         0         0     0       0      0      0   \n",
              "3                  0     0         0         0     0       0      0      0   \n",
              "4                  0     0         0         0     0       0      0      0   \n",
              "5                  0     0         0         0     0       0      0      0   \n",
              "6                  0     0         1         0     0       0      0      0   \n",
              "7                  0     0         0         0     0       0      0      0   \n",
              "8                  0     0         0         0     0       0      0      0   \n",
              "9                  0     0         0         0     0       0      0      0   \n",
              "10                 0     0         0         0     0       0      0      0   \n",
              "11                 0     0         0         0     1       0      0      0   \n",
              "12                 0     0         0         0     0       0      0      0   \n",
              "13                 0     0         0         0     0       0      0      0   \n",
              "14                 0     0         0         0     0       0      1      0   \n",
              "15                 0     6         0         0     0       0      0      0   \n",
              "16                 0     0         0         0     0       0      0      0   \n",
              "17                 0     0         0         0     0       0      0      0   \n",
              "18                 0     0         0         0     0       0      0      0   \n",
              "19                 0     0         0         0     0       0      0      1   \n",
              "20                 0     0         1         1     0       0      0      0   \n",
              "21                 0     0         0         0     0       0      0      0   \n",
              "22                 0     0         0         0     0       0      0      0   \n",
              "23                 0     0         0         0     0       2      0      0   \n",
              "24                 0     0         0         0     0       0      0      0   \n",
              "25                 0     0         0         0     0       0      0      0   \n",
              "26                 0     0         0         0     0       0      0      0   \n",
              "27                 0     0         0         2     0       0      0      0   \n",
              "28                 0     0         0        11     0       0      0      0   \n",
              "29                 0     0         0         0     0       0      0      0   \n",
              "\n",
              "    zürich  \n",
              "0        0  \n",
              "1        0  \n",
              "2        0  \n",
              "3        0  \n",
              "4        0  \n",
              "5        0  \n",
              "6        0  \n",
              "7        0  \n",
              "8        1  \n",
              "9        0  \n",
              "10       0  \n",
              "11       0  \n",
              "12       0  \n",
              "13       0  \n",
              "14       0  \n",
              "15       0  \n",
              "16       0  \n",
              "17       0  \n",
              "18       0  \n",
              "19       0  \n",
              "20       0  \n",
              "21       0  \n",
              "22       0  \n",
              "23       0  \n",
              "24       0  \n",
              "25       0  \n",
              "26       0  \n",
              "27       0  \n",
              "28       0  \n",
              "29       0  \n",
              "\n",
              "[30 rows x 13249 columns]"
            ]
          },
          "execution_count": 43,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Outputting data term frequency matrix\n",
        "dataframe = pd.DataFrame(documents_vectorized.toarray(), columns=vocabulary)\n",
        "dataframe"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "aAQEw_vu7s_J"
      },
      "source": [
        "There are 13,248 terms in the vocabulary."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "La5QGQWui7EN"
      },
      "outputs": [],
      "source": [
        "# Creating term-document pairs dictionary\n",
        "from collections import defaultdict\n",
        "\n",
        "term_occurence_dict = defaultdict(list)\n",
        "\n",
        "for doc_id, doc in enumerate(document_terms):\n",
        "  for term in doc:\n",
        "    term_occurence_dict[term].append(doc_id)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VSXB09bW8Wa2",
        "outputId": "2e2f7ff7-b355-4335-9108-02db3810200b"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "['SARS-CoV-2 Gamma variant',\n",
              " 'Variants of SARS-CoV-2',\n",
              " 'SARS-CoV-2 Delta variant',\n",
              " 'Coronavirus',\n",
              " 'Investigations into the origin of COVID-19',\n",
              " 'Severe acute respiratory syndrome coronavirus 2',\n",
              " 'COVID-19',\n",
              " 'Angiotensin-converting enzyme 2',\n",
              " 'SARS-CoV-2 Omicron variant',\n",
              " 'Virus']"
            ]
          },
          "execution_count": 45,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Checking success of term-document dictionary\n",
        "terms = ['genome', 'etiology']                   # find all documents with the word \"genome\" or \"etiology\"\n",
        "result_atcls = []\n",
        "\n",
        "for term in terms:\n",
        "  doc_ids = term_occurence_dict[term]\n",
        "  for doc_id in doc_ids:\n",
        "    result_atcls.append(articles[doc_id])\n",
        "\n",
        "result_atcls = list(set(result_atcls))\n",
        "result_atcls                                     # return document titles"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "B_bH7Vnhkb7a",
        "outputId": "ed9b4f54-6cb6-496a-c2ee-7cf4ff0b68e4"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "['Coronavirus disease 2019 (COVID-19) is a contagious disease caused by a virus, the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The first known case was identified in Wuhan, China, in December 2019. The disease has since spread worldwide, leading to the ongoing COVID-19 pandemic.Symptoms of COVID‑19 are variable, but often include fever, cough, headache, fatigue, breathing difficulties, loss of smell, and loss of taste. Symptoms may begin one to fourteen days after exposure to the virus. At least a third of people who are infected do not develop noticeable symptoms. Of those people who develop symptoms noticeable enough to be classed as patients, most (81%) develop mild to moderate symptoms (up to mild pneumonia), while 14% develop severe symptoms (dyspnea, hypoxia, or more than 50% lung involvement on imaging), and 5% suffer critical symptoms (respiratory failure, shock, or multiorgan dysfunction). Older people are at a higher risk of developing severe symptoms. Some people continue to experience a range of effects (long COVID) for months after recovery, and damage to organs has been observed. Multi-year studies are underway to further investigate the long-term effects of the disease.COVID‑19 transmits when people breathe in air contaminated by droplets and small airborne particles containing the virus. The risk of breathing these in is highest when people are in close proximity, but they can be inhaled over longer distances, particularly indoors. Transmission can also occur if splashed or sprayed with contaminated fluids in the eyes, nose or mouth, and, rarely, via contaminated surfaces. People remain contagious for up to 20 days, and can spread the virus even if they do not develop symptoms.Several COVID-19 testing methods have been developed to diagnose the disease. The standard diagnostic method is by detection of the virus\\'s nucleic acid by real-time reverse transcription polymerase chain reaction (rRT-PCR), transcription-mediated amplification (TMA), or by reverse transcription loop-mediated isothermal amplification (RT-LAMP) from a nasopharyngeal swab.\\nSeveral COVID-19 vaccines have been approved and distributed in various countries, which have initiated mass vaccination campaigns. Other preventive measures include physical or social distancing, quarantining, ventilation of indoor spaces, covering coughs and sneezes, hand washing, and keeping unwashed hands away from the face. The use of face masks or coverings has been recommended in public settings to minimize the risk of transmission. While work is underway to develop drugs that inhibit the virus, the primary treatment is symptomatic. Management involves the treatment of symptoms, supportive care, isolation, and experimental measures.\\n\\n\\n== Nomenclature ==\\n\\nDuring the initial outbreak in Wuhan, the virus and disease were commonly referred to as \"coronavirus\" and \"Wuhan coronavirus\", with the disease sometimes called \"Wuhan pneumonia\". In the past, many diseases have been named after geographical locations, such as the Spanish flu, Middle East respiratory syndrome, and Zika virus. In January 2020, the World Health Organization (WHO) recommended 2019-nCoV and 2019-nCoV acute respiratory disease as interim names for the virus and disease per 2015 guidance and international guidelines against using geographical locations or groups of people in disease and virus names to prevent social stigma. The official names COVID‑19 and SARS-CoV-2 were issued by the WHO on 11 February 2020. The Director-General, Tedros Adhanom explained that CO stands for corona, VI for virus, D for disease, and 19 for 2019, the year in which the outbreak was first identified. The WHO additionally uses \"the COVID‑19 virus\" and \"the virus responsible for COVID‑19\" in public communications.\\n\\n\\n== Signs and symptoms ==\\n\\n\\n== Cause ==\\nCOVID‑19 is caused by infection with a strain of coronavirus known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).\\n\\n\\n=== Transmission ===\\n\\n\\n=== Virology ===\\n\\nSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a novel severe acute respiratory syndrome coronavirus. It was first isolated from three people with pneumonia connected to the cluster of acute respiratory illness cases in Wuhan. All structural features of the novel SARS-CoV-2 virus particle occur in related coronaviruses in nature.Outside the human body, the virus is destroyed by household soap, which bursts its protective bubble.SARS-CoV-2 is closely related to the original SARS-CoV. It is thought to have an animal (zoonotic) origin. Genetic analysis has revealed that the coronavirus genetically clusters with the genus Betacoronavirus, in subgenus Sarbecovirus (lineage B) together with two bat-derived strains. It is 96% identical at the whole genome level to other bat coronavirus samples (BatCov RaTG13). The structural proteins of SARS-CoV-2 include membrane glycoprotein (M), envelope protein (E), nucleocapsid protein (N), and the spike protein (S). The M protein of SARS-CoV-2 is about 98% similar to the M protein of bat SARS-CoV, maintains around 98% homology with pangolin SARS-CoV, and has 90% homology with the M protein of SARS-CoV; whereas, the similarity is only around 38% with the M protein of MERS-CoV.\\n\\n\\n=== SARS-CoV-2 variants ===\\n\\nThe many thousands of SARS-CoV-2 variants are grouped into either clades or lineages. The WHO, in collaboration with partners, expert networks, national authorities, institutions and researchers, have established nomenclature systems for naming and tracking SARS-CoV-2 genetic lineages by GISAID, Nextstrain and Pango. At the present time, the expert group convened by WHO has recommended the labeling of variants using letters of the Greek Alphabet, for example, Alpha, Beta, Delta, and Gamma, giving the justification that they \"will be easier and more practical to discussed by non-scientific audiences.\" Nextstrain divides the variants into five clades (19A, 19B, 20A, 20B, and 20C), while GISAID divides them into seven (L, O, V, S, G, GH, and GR). The Pango tool groups variants into lineages, with many circulating lineages being classed under the B.1 lineage.Several notable variants of SARS-CoV-2 emerged throughout 2020. Cluster 5 emerged among minks and mink farmers in Denmark. After strict quarantines and a mink euthanasia campaign, the cluster was assessed to no longer be circulating among humans in Denmark as of 1 February 2021.As of December 2021, there are five dominant variants of SARS-CoV-2 spreading among global populations: the Alpha variant (B.1.1.7, formerly called the UK variant), first found in London and Kent, the Beta variant (B.1.351, formerly called the South Africa variant), the Gamma variant (P.1, formerly called the Brazil variant), the Delta variant (B.1.617.2, formerly called the India variant), and the Omicron variant (B.1.1.529), which had spread to 57 countries as of 7 December.\\n\\n\\n== Pathophysiology ==\\n\\nThe SARS-CoV-2 virus can infect a wide range of cells and systems of the body. COVID‑19 is most known for affecting the upper respiratory tract (sinuses, nose, and throat) and the lower respiratory tract (windpipe and lungs). The lungs are the organs most affected by COVID‑19 because the virus accesses host cells via the receptor for the enzyme angiotensin-converting enzyme 2 (ACE2), which is most abundant on the surface of type II alveolar cells of the lungs. The virus uses a special surface glycoprotein called a \"spike\" to connect to the ACE2 receptor and enter the host cell.\\n\\n\\n=== Respiratory tract ===\\nFollowing viral entry, COVID‑19 infects the ciliated epithelium of the nasopharynx and upper airways.\\n\\n\\n=== Nervous system ===\\nOne common symptom, loss of smell, results from infection of the support cells of the olfactory epithelium, with subsequent damage to the olfactory neurons. The involvement of both the central and peripheral nervous system in COVID‑19 has been reported in many medical publications. It is clear that many people with COVID-19 exhibit neurological or mental health issues. The virus is not detected in the CNS of the majority of COVID-19 patients with neurological issues. However, SARS-CoV-2 has been detected at low levels in the brains of those who have died from COVID‑19, but these results need to be confirmed. While virus has been detected in cerebrospinal fluid of autopsies, the exact mechanism by which it invades the CNS remains unclear and may first involve invasion of peripheral nerves given the low levels of ACE2 in the brain. The virus may also enter the bloodstream from the lungs and cross the blood-brain barrier to gain access to the CNS, possibly within an infected white blood cell.\\n  Research conducted when Alpha was the dominant variant has suggested COVID-19 May cause brain damage. It is currently unknown if such damage is temporary or permanent, and whether Omicron has similar effects.  Observed individuals infected with COVID-19 (most with mild cases) experienced an additional 0.2% to 2% of brain tissue lost in regions of the brain connected to the sense of smell compared with uninfected individuals, and the overall effect on the brain was equivalent on average to at least one extra year of normal aging; infected individuals also scored lower on several cognitive tests.  All effects were more pronounced among older ages.\\n\\n\\n=== Gastrointestinal tract ===\\nThe virus also affects gastrointestinal organs as ACE2 is abundantly expressed in the glandular cells of gastric, duodenal and rectal epithelium as well as endothelial cells and enterocytes of the small intestine.\\n\\n\\n=== Cardiovascular system ===\\nThe virus can cause acute myocardial injury and chronic damage to the cardiovascular system. An acute cardiac injury was found in 12% of infected people admitted to the hospital in Wuhan, China, and is more frequent in severe disease. Rates of cardiovascular symptoms are high, owing to the systemic inflammatory response and immune system disorders during disease progression, but acute myocardial injuries may also be related to ACE2 receptors in the heart. ACE2 receptors are highly expressed in the heart and are involved in heart function. A high incidence of thrombosis and venous thromboembolism have been found in people transferred to Intensive care units (ICU) with COVID‑19 infections, and may be related to poor prognosis. Blood vessel dysfunction and clot formation (as suggested by high D-dimer levels caused by blood clots) are thought to play a significant role in mortality, incidences of clots leading to pulmonary embolisms, and ischaemic events within the brain have been noted as complications leading to death in people infected with SARS-CoV-2. Infection appears to set off a chain of vasoconstrictive responses within the body, constriction of blood vessels within the pulmonary circulation has also been posited as a mechanism in which oxygenation decreases alongside the presentation of viral pneumonia. Furthermore, microvascular (arterioles and capillaries) blood vessel damage has been reported in a small number of tissue samples of the brains – without detected SARS-CoV-2 – and the olfactory bulbs from those who have died from COVID‑19. COVID‑19 was also found to cause substantial – including morphological and mechanical – changes to blood cells – such as increased sizes – sometimes persisting for months after hospital discharge.\\n\\n\\n=== Other organs ===\\nAnother common cause of death is complications related to the kidneys. Early reports show that up to 30% of hospitalized patients both in China and in New York have experienced some injury to their kidneys, including some persons with no previous kidney problems.Autopsies of people who died of COVID‑19 have found diffuse alveolar damage, and lymphocyte-containing inflammatory infiltrates within the lung.\\n\\n\\n=== Immunopathology ===\\n\\nAlthough SARS-CoV-2 has a tropism for ACE2-expressing epithelial cells of the respiratory tract, people with severe COVID‑19 have symptoms of systemic hyperinflammation. Clinical laboratory findings of elevated IL‑2, IL‑7, IL‑6, granulocyte-macrophage colony-stimulating factor (GM‑CSF), interferon gamma-induced protein 10 (IP‑10), monocyte chemoattractant protein 1 (MCP1), macrophage inflammatory protein 1‑alpha (MIP‑1‑alpha), and tumour necrosis factor (TNF‑α) indicative of cytokine release syndrome (CRS) suggest an underlying immunopathology.Additionally, people with COVID‑19 and acute respiratory distress syndrome (ARDS) have classical serum biomarkers of CRS, including elevated C-reactive protein (CRP), lactate dehydrogenase (LDH), D-dimer, and ferritin.Systemic inflammation results in vasodilation, allowing inflammatory lymphocytic and monocytic infiltration of the lung and the heart. In particular, pathogenic GM-CSF-secreting T cells were shown to correlate with the recruitment of inflammatory IL-6-secreting monocytes and severe lung pathology in people with COVID‑19. Lymphocytic infiltrates have also been reported at autopsy.\\n\\n\\n=== Viral and host factors ===\\n\\n\\n==== Virus proteins ====\\n\\nMultiple viral and host factors affect the pathogenesis of the virus. The S-protein, otherwise known as the spike protein, is the viral component that attaches to the host receptor via the ACE2 receptors. It includes two subunits: S1 and S2. S1 determines the virus-host range and cellular tropism via the receptor-binding domain. S2 mediates the membrane fusion of the virus to its potential cell host via the H1 and HR2, which are heptad repeat regions. Studies have shown that S1 domain induced IgG and IgA antibody levels at a much higher capacity. It is the focus spike proteins expression that are involved in many effective COVID‑19 vaccines.The M protein is the viral protein responsible for the transmembrane transport of nutrients. It is the cause of the bud release and the formation of the viral envelope. The N and E protein are accessory proteins that interfere with the host\\'s immune response.\\n\\n\\n==== Host factors ====\\nHuman angiotensin converting enzyme 2 (hACE2) is the host factor that SARS-COV2 virus targets causing COVID‑19. Theoretically, the usage of angiotensin receptor blockers (ARB) and ACE inhibitors upregulating ACE2 expression might increase morbidity with COVID‑19, though animal data suggest some potential protective effect of ARB; however no clinical studies have proven susceptibility or outcomes. Until further data is available, guidelines and recommendations for hypertensive patients remain.The effect of the virus on ACE2 cell surfaces leads to leukocytic infiltration, increased blood vessel permeability, alveolar wall permeability, as well as decreased secretion of lung surfactants. These effects cause the majority of the respiratory symptoms. However, the aggravation of local inflammation causes a cytokine storm eventually leading to a systemic inflammatory response syndrome.Among healthy adults not exposed to SARS-CoV-2, about 35% have CD4+ T cells that recognize the SARS-CoV-2 S protein (particularly the S2 subunit) and about 50% react to other proteins of the virus, suggesting cross-reactivity from previous common colds caused by other coronaviruses.It is unknown whether different persons use similar antibody genes in response to COVID‑19.\\n\\n\\n=== Host cytokine response ===\\n\\nThe severity of the inflammation can be attributed to the severity of what is known as the cytokine storm. Levels of interleukin 1B, interferon-gamma, interferon-inducible protein 10, and monocyte chemoattractant protein 1 were all associated with COVID‑19 disease severity. Treatment has been proposed to combat the cytokine storm as it remains to be one of the leading causes of morbidity and mortality in COVID‑19 disease.A cytokine storm is due to an acute hyperinflammatory response that is responsible for clinical illness in an array of diseases but in COVID‑19, it is related to worse prognosis and increased fatality. The storm causes acute respiratory distress syndrome, blood clotting events such as strokes, myocardial infarction, encephalitis, acute kidney injury, and vasculitis. The production of IL-1, IL-2, IL-6, TNF-alpha, and interferon-gamma, all crucial components of normal immune responses, inadvertently become the causes of a cytokine storm. The cells of the central nervous system, the microglia, neurons, and astrocytes, are also involved in the release of pro-inflammatory cytokines affecting the nervous system, and effects of cytokine storms toward the CNS are not uncommon.\\n\\n\\n=== Pregnancy response ===\\nThere are many unknowns for pregnant women during the COVID‑19 pandemic. Given that they are prone to suffering from complications and severe disease infection with other types of coronaviruses, they have been identified as a vulnerable group and advised to take supplementary preventive measures.Physiological responses to pregnancy can include:\\n\\nImmunological: The immunological response to COVID-19, like other viruses, depends on a working immune system. It adapts during pregnancy to allow the development of the fetus whose genetic load is only partially shared with their mother, leading to a different immunological reaction to infections during the course of pregnancy.\\nRespiratory: Many factors can make pregnant women more vulnerable to hard respiratory infections. One of them is the total reduction of the lungs\\' capacity and inability to clear secretions.\\nCoagulation: During pregnancy, there are higher levels of circulating coagulation factors, and the pathogenesis of SARS-CoV-2 infection can be implicated. The thromboembolic events with associated mortality are a risk for pregnant women.However, from the evidence base, it is difficult to conclude whether pregnant women are at increased risk of grave consequences of this virus.In addition to the above, other clinical studies have proved that SARS-CoV-2 can affect the period of pregnancy in different ways. On the one hand, there is little evidence of its impact up to 12 weeks gestation. On the other hand, COVID-19 infection may cause increased rates of unfavorable outcomes in the course of the pregnancy. Some examples of these could be fetal growth restriction, preterm birth, and perinatal mortality, which refers to the fetal death past 22 or 28 completed weeks of pregnancy as well as the death among live-born children up to seven completed days of life.Unvaccinated women in later stages of pregnancy with COVID-19 are more likely than other patients to need very intensive care. Babies born to mothers with COVID-19 are more likely to have breathing problems. Pregnant women are strongly encouraged to get vaccinated.\\n\\n\\n== Diagnosis ==\\n\\nCOVID‑19 can provisionally be diagnosed on the basis of symptoms and confirmed using reverse transcription polymerase chain reaction (RT-PCR) or other nucleic acid testing of infected secretions. Along with laboratory testing, chest CT scans may be helpful to diagnose COVID‑19 in individuals with a high clinical suspicion of infection. Detection of a past infection is possible with serological tests, which detect antibodies produced by the body in response to the infection.\\n\\n\\n=== Viral testing ===\\n\\nThe standard methods of testing for presence of SARS-CoV-2 are nucleic acid tests, which detects the presence of viral RNA fragments. As these tests detect RNA but not infectious virus, its \"ability to determine duration of infectivity of patients is limited.\" The test is typically done on respiratory samples obtained by a nasopharyngeal swab; however, a nasal swab or sputum sample may also be used. Results are generally available within hours. The WHO has published several testing protocols for the disease.Several laboratories and companies have developed serological tests, which detect antibodies produced by the body in response to infection. Several have been evaluated by Public Health England and approved for use in the UK.The University of Oxford\\'s CEBM has pointed to mounting evidence that \"a good proportion of \\'new\\' mild cases and people re-testing positives after quarantine or discharge from hospital are not infectious, but are simply clearing harmless virus particles which their immune system has efficiently dealt with\" and have called for \"an international effort to standardize and periodically calibrate testing\" On 7 September, the UK government issued \"guidance for procedures to be implemented in laboratories to provide assurance of positive SARS-CoV-2 RNA results during periods of low prevalence, when there is a reduction in the predictive value of positive test results\".\\n\\n\\n=== Imaging ===\\n\\nChest CT scans may be helpful to diagnose COVID‑19 in individuals with a high clinical suspicion of infection but are not recommended for routine screening. Bilateral multilobar ground-glass opacities with a peripheral, asymmetric, and posterior distribution are common in early infection. Subpleural dominance, crazy paving (lobular septal thickening with variable alveolar filling), and consolidation may appear as the disease progresses. Characteristic imaging features on chest radiographs and computed tomography (CT) of people who are symptomatic include asymmetric peripheral ground-glass opacities without pleural effusions.Many groups have created COVID‑19 datasets that include imagery such as the Italian Radiological Society which has compiled an international online database of imaging findings for confirmed cases. Due to overlap with other infections such as adenovirus, imaging without confirmation by rRT-PCR is of limited specificity in identifying COVID‑19. A large study in China compared chest CT results to PCR and demonstrated that though imaging is less specific for the infection, it is faster and more sensitive.\\n\\n\\n=== Coding ===\\nIn late 2019, the WHO assigned emergency ICD-10 disease codes U07.1 for deaths from lab-confirmed SARS-CoV-2 infection and U07.2 for deaths from clinically or epidemiologically diagnosed COVID‑19 without lab-confirmed SARS-CoV-2 infection.\\n\\n\\n=== Pathology ===\\nThe main pathological findings at autopsy are:\\n\\nMacroscopy: pericarditis, lung consolidation and pulmonary oedema\\nLung findings:\\nminor serous exudation, minor fibrin exudation\\npulmonary oedema, pneumocyte hyperplasia, large atypical pneumocytes, interstitial inflammation with lymphocytic infiltration and multinucleated giant cell formation\\ndiffuse alveolar damage (DAD) with diffuse alveolar exudates. DAD is the cause of acute respiratory distress syndrome (ARDS) and severe hypoxemia.\\norganisation of exudates in alveolar cavities and pulmonary interstitial fibrosis\\nplasmocytosis in BAL\\nBlood and vessels: disseminated intravascular coagulation (DIC); leukoerythroblastic reaction, endotheliitis, hemophagocytosis\\nHeart: cardiac muscle cell necrosis\\nLiver: microvesicular steatosis\\nNose: shedding of olfactory epithelium\\nBrain: infarction\\nKidneys: acute tubular damage.\\nSpleen: white pulp depletion.\\n\\n\\n== Prevention ==\\n\\nPreventive measures to reduce the chances of infection include getting vaccinated, staying at home, wearing a mask in public, avoiding crowded places, keeping distance from others, ventilating indoor spaces, managing potential exposure durations, washing hands with soap and water often and for at least twenty seconds, practising good respiratory hygiene, and avoiding touching the eyes, nose, or mouth with unwashed hands.Those diagnosed with COVID‑19 or who believe they may be infected are advised by the CDC to stay home except to get medical care, call ahead before visiting a healthcare provider, wear a face mask before entering the healthcare provider\\'s office and when in any room or vehicle with another person, cover coughs and sneezes with a tissue, regularly wash hands with soap and water and avoid sharing personal household items.The first COVID‑19 vaccine was granted regulatory approval on 2 December 2020 by the UK medicines regulator MHRA. It was evaluated for emergency use authorization (EUA) status by the US FDA, and in several other countries. Initially, the US National Institutes of Health guidelines do not recommend any medication for prevention of COVID‑19, before or after exposure to the SARS-CoV-2 virus, outside the setting of a clinical trial. Without a vaccine, other prophylactic measures, or effective treatments, a key part of managing COVID‑19 is trying to decrease and delay the epidemic peak, known as \"flattening the curve\". This is done by slowing the infection rate to decrease the risk of health services being overwhelmed, allowing for better treatment of active cases, and delaying additional cases until effective treatments or a vaccine become available.\\n\\n\\n=== Vaccine ===\\n\\n\\n=== Face masks and respiratory hygiene ===\\n\\nThe WHO and the US CDC recommend individuals wear non-medical face coverings in public settings where there is an increased risk of transmission and where social distancing measures are difficult to maintain. This recommendation is meant to reduce the spread of the disease by asymptomatic and pre-symptomatic individuals and is complementary to established preventive measures such as social distancing. Face coverings limit the volume and travel distance of expiratory droplets dispersed when talking, breathing, and coughing. A face covering without vents or holes will also filter out particles containing the virus from inhaled and exhaled air, reducing the chances of infection. But, if the mask include an exhalation valve, a wearer that is infected (maybe without having noticed that, and asymptomatic) would transmit the virus outwards through it, despite any certification they can have. So the masks with exhalation valve are not for the infected wearers, and are not reliable to stop the pandemic in a large scale. Many countries and local jurisdictions encourage or mandate the use of face masks or cloth face coverings by members of the public to limit the spread of the virus.Masks are also strongly recommended for those who may have been infected and those taking care of someone who may have the disease. When not wearing a mask, the CDC recommends covering the mouth and nose with a tissue when coughing or sneezing and recommends using the inside of the elbow if no tissue is available. Proper hand hygiene after any cough or sneeze is encouraged. Healthcare professionals interacting directly with people who have COVID‑19 are advised to use respirators at least as protective as NIOSH-certified N95 or equivalent, in addition to other personal protective equipment.\\n\\n\\n=== Indoor ventilation and avoiding crowded indoor spaces ===\\nThe CDC recommends that crowded indoor spaces should be avoided. When indoors, increasing the rate of air change, decreasing recirculation of air and increasing the use of outdoor air can reduce transmission. The WHO recommends ventilation and air filtration in public spaces to help clear out infectious aerosols.Exhaled respiratory particles can build-up within enclosed spaces with inadequate ventilation. The risk of COVID‑19 infection increases especially in spaces where people engage in physical exertion or raise their voice (e.g., exercising, shouting, singing) as this increases exhalation of respiratory droplets. Prolonged exposure to these conditions, typically more than 15 minutes, leads to higher risk of infection.Displacement ventilation with large natural inlets can move stale air directly to the exhaust in laminar flow while significantly reducing the concentration of droplets and particles. Passive ventilation reduces energy consumption and maintenance costs but may lack controllability and heat recovery. Displacement ventilation can also be achieved mechanically with higher energy and maintenance costs. The use of large ducts and openings helps to prevent mixing in closed environments. Recirculation and mixing should be avoided because recirculation prevents dilution of harmful particles and redistributes possibly contaminated air, and mixing increases the concentration and range of infectious particles and keeps larger particles in the air.\\n\\n\\n=== Hand-washing and hygiene ===\\n\\nThorough hand hygiene after any cough or sneeze is required. The WHO also recommends that individuals wash hands often with soap and water for at least twenty seconds, especially after going to the toilet or when hands are visibly dirty, before eating and after blowing one\\'s nose. When soap and water are not available, the CDC recommends using an alcohol-based hand sanitiser with at least 60% alcohol. For areas where commercial hand sanitisers are not readily available, the WHO provides two formulations for local production. In these formulations, the antimicrobial activity arises from ethanol or isopropanol. Hydrogen peroxide is used to help eliminate bacterial spores in the alcohol; it is \"not an active substance for hand antisepsis.\" Glycerol is added as a humectant.\\n\\n\\n=== Social distancing ===\\n\\nSocial distancing (also known as physical distancing) includes infection control actions intended to slow the spread of the disease by minimising close contact between individuals. Methods include quarantines; travel restrictions; and the closing of schools, workplaces, stadiums, theatres, or shopping centres. Individuals may apply social distancing methods by staying at home, limiting travel, avoiding crowded areas, using no-contact greetings, and physically distancing themselves from others. Many governments are now mandating or recommending social distancing in regions affected by the outbreak.Outbreaks have occurred in prisons due to crowding and an inability to enforce adequate social distancing. In the United States, the prisoner population is aging and many of them are at high risk for poor outcomes from COVID‑19 due to high rates of coexisting heart and lung disease, and poor access to high-quality healthcare.\\n\\n\\n=== Surface cleaning ===\\nAfter being expelled from the body, coronaviruses can survive on surfaces for hours to days. If a person touches the dirty surface, they may deposit the virus at the eyes, nose, or mouth where it can enter the body and cause infection. Evidence indicates that contact with infected surfaces is not the main driver of COVID‑19, leading to recommendations for optimised disinfection procedures to avoid issues such as the increase of antimicrobial resistance through the use of inappropriate cleaning products and processes. Deep cleaning and other surface sanitation has been criticized as hygiene theater, giving a false sense of security against something primarily spread through the air.The amount of time that the virus can survive depends significantly on the type of surface, the temperature, and the humidity. Coronaviruses die very quickly when exposed to the UV light in sunlight. Like other enveloped viruses, SARS-CoV-2 survives longest when the temperature is at room temperature or lower, and when the relative humidity is low (<50%).On many surfaces, including glass, some types of plastic, stainless steel, and skin, the virus can remain infective for several days indoors at room temperature, or even about a week under ideal conditions. On some surfaces, including cotton fabric and copper, the virus usually dies after a few hours. The virus dies faster on porous surfaces than on non-porous surfaces due to capillary action within pores and faster aerosol droplet evaporation. However, of the many surfaces tested, two with the longest survival times are N95 respirator masks and surgical masks, both of which are considered porous surfaces.The CDC says that in most situations, cleaning surfaces with soap or detergent, not disinfecting, is enough to reduce risk of transmission. The CDC recommends that if a COVID‑19 case is suspected or confirmed at a facility such as an office or day care, all areas such as offices, bathrooms, common areas, shared electronic equipment like tablets, touch screens, keyboards, remote controls, and ATM machines used by the ill persons should be disinfected. Surfaces may be decontaminated with 62–71 percent ethanol, 50–100 percent isopropanol, 0.1 percent sodium hypochlorite, 0.5 percent hydrogen peroxide, and 0.2–7.5 percent povidone-iodine. Other solutions, such as benzalkonium chloride and chlorhexidine gluconate, are less effective. Ultraviolet germicidal irradiation may also be used. A datasheet comprising the authorised substances to disinfection in the food industry (including suspension or surface tested, kind of surface, use dilution, disinfectant and inocuylum volumes) can be seen in the supplementary material of.\\n\\n\\n=== Self-isolation ===\\nSelf-isolation at home has been recommended for those diagnosed with COVID‑19 and those who suspect they have been infected. Health agencies have issued detailed instructions for proper self-isolation. Many governments have mandated or recommended self-quarantine for entire populations. The strongest self-quarantine instructions have been issued to those in high-risk groups. Those who may have been exposed to someone with COVID‑19 and those who have recently travelled to a country or region with the widespread transmission have been advised to self-quarantine for 14 days from the time of last possible exposure.\\n\\n\\n=== Healthy diet and lifestyle ===\\nThe Harvard T.H. Chan School of Public Health recommends a healthy diet, being physically active, managing psychological stress, and getting enough sleep.Consistently meeting scientific guidelines of 150+ minutes per week of exercise or similar physical activity was shown to be associated with a smaller risk of hospitalisation and death due to COVID‑19, even when considering likely risk factors such as elevated Body mass index (BMI).A meta-analysis, published online in October 2021, concluded that \"Vitamin D supplementation in SARS-CoV-2 positive patients has the potential to positively impact patients with both mild and severe symptoms.\" The largest observational study on the subject using online questionnaires, with over 6,000 participants and a dosage regime near the RDI, is set to conclude in July 2021. One of the collaborators in the study is Synergy Biologics Ltd, a manufacturer of vitamin D3 supplements.\\n\\n\\n=== International travel-related control measures ===\\nA 2021 Cochrane rapid review found that based upon low-certainty evidence, international travel-related control measures such as restricting cross-border travel may help to contain the spread of COVID‑19. Additionally, symptom/exposure-based screening measures at borders may miss many positive cases. While test-based border screening measures may be more effective, it could also miss many positive cases if only conducted upon arrival without follow-up. The review concluded that a minimum 10-day quarantine may be beneficial in preventing the spread of COVID‑19 and may be more effective if combined with an additional control measure like border screening.\\n\\n\\n== Treatment ==\\n\\n\\n== Prognosis ==\\n\\nThe severity of COVID‑19 varies. The disease may take a mild course with few or no symptoms, resembling other common upper respiratory diseases such as the common cold. In 3–4% of cases (7.4% for those over age 65) symptoms are severe enough to cause hospitalization. Mild cases typically recover within two weeks, while those with severe or critical diseases may take three to six weeks to recover. Among those who have died, the time from symptom onset to death has ranged from two to eight weeks. The Italian Istituto Superiore di Sanità reported that the median time between the onset of symptoms and death was twelve days, with seven being hospitalised. However, people transferred to an ICU had a median time of ten days between hospitalisation and death. Prolonged prothrombin time and elevated C-reactive protein levels on admission to the hospital are associated with severe course of COVID‑19 and with a transfer to ICU.Some early studies suggest 10% to 20% of people with COVID‑19 will experience symptoms lasting longer than a month. A majority of those who were admitted to hospital with severe disease report long-term problems including fatigue and shortness of breath. On 30 October 2020, WHO chief Tedros Adhanom warned that \"to a significant number of people, the COVID virus poses a range of serious long-term effects.\" He has described the vast spectrum of COVID‑19 symptoms that fluctuate over time as \"really concerning\". They range from fatigue, a cough and shortness of breath, to inflammation and injury of major organs – including the lungs and heart, and also neurological and psychologic effects. Symptoms often overlap and can affect any system in the body. Infected people have reported cyclical bouts of fatigue, headaches, months of complete exhaustion, mood swings, and other symptoms. Tedros therefore concluded that a strategy of achieving herd immunity by infection, rather than vaccination, is \"morally unconscionable and unfeasible\".In terms of hospital readmissions about 9% of 106,000 individuals had to return for hospital treatment within two months of discharge. The average to readmit was eight days since first hospital visit. There are several risk factors that have been identified as being a cause of multiple admissions to a hospital facility. Among these are advanced age (above 65 years of age) and presence of a chronic condition such as diabetes, COPD, heart failure or chronic kidney disease.According to scientific reviews smokers are more likely to require intensive care or die compared to non-smokers. Acting on the same ACE2 pulmonary receptors affected by smoking, air pollution has been correlated with the disease. Short term and chronic exposure to air pollution seems to enhance morbidity and mortality from COVID‑19. Pre-existing heart and lung diseases and also obesity, especially in conjunction with fatty liver disease, contributes to an increased health risk of COVID‑19.It is also assumed that those that are immunocompromised are at higher risk of getting severely sick from SARS-CoV-2. One research that looked into the COVID‑19 infections in hospitalized kidney transplant recipients found a mortality rate of 11%.Genetics also plays an important role in the ability to fight off the disease. For instance, those that do not produce detectable type I interferons or produce auto-antibodies against these may get much sicker from COVID‑19. Genetic screening is able to detect interferon effector genes.\\n\\n\\n=== Children ===\\n\\nWhile very young children have experienced lower rates of infection, older children have a rate of infection that is similar to the population as a whole. Children are likely to have milder symptoms and are at lower risk of severe disease than adults. The CDC reports that in the US roughly a third of hospitalized children were admitted to the ICU, while a European multinational study of hospitalized children from June 2020 found that about 8% of children admitted to a hospital needed intensive care. Four of the 582 children (0.7%) in the European study died, but the actual mortality rate may be \"substantially lower\" since milder cases that did not seek medical help were not included in the study.\\n\\n\\n=== Complications ===\\n\\nComplications may include pneumonia, acute respiratory distress syndrome (ARDS), multi-organ failure, septic shock, and death. Cardiovascular complications may include heart failure, arrhythmias (including atrial fibrillation), heart inflammation, and thrombosis, particularly venous thromboembolism. Approximately 20–30% of people who present with COVID‑19 have elevated liver enzymes, reflecting liver injury.Neurologic manifestations include seizure, stroke, encephalitis, and Guillain–Barré syndrome (which includes loss of motor functions). Following the infection, children may develop paediatric multisystem inflammatory syndrome, which has symptoms similar to Kawasaki disease, which can be fatal. In very rare cases, acute encephalopathy can occur, and it can be considered in those who have been diagnosed with COVID‑19 and have an altered mental status.In the case of pregnant women, it is important to note that, according to the Centers for Disease Control and Prevention, pregnant women are at increased risk of becoming seriously ill from COVID‑19. This is because pregnant women with COVID‑19 appear to be more likely to develop respiratory and obstetric complications that can lead to miscarriage, premature delivery and intrauterine growth restriction.Fungal infections such as aspergillosis, candidiasis, cryptococcosis and mucormycosis have been recorded in patients recovering from COVID‑19.\\n\\n\\n=== Longer-term effects ===\\n\\nSome early studies suggest that 10–20% of people with COVID‑19 will experience symptoms lasting longer than a month. A majority of those who were admitted to hospital with severe disease report long-term problems, including fatigue and shortness of breath. About 5–10% of patients admitted to hospital progress to severe or critical disease, including pneumonia and acute respiratory failure.By a variety of mechanisms, the lungs are the organs most affected in COVID‑19. In people requiring hospital admission, up to 98% of CT scans performed show lung abnormalities after 28 days of illness even if they had clinically improved.People with advanced age, severe disease, prolonged ICU stays, or who smoke are more likely to have long-lasting effects, including pulmonary fibrosis. Overall, approximately one-third of those investigated after four weeks will have findings of pulmonary fibrosis or reduced lung function as measured by DLCO, even in asymptomatic people, but with the suggestion of continuing improvement with the passing of more time.\\n\\n\\n=== Immunity ===\\n\\nThe immune response by humans to SARS-CoV-2 virus occurs as a combination of the cell-mediated immunity and antibody production, just as with most other infections. B cells interact with T cells and begin dividing before selection into the plasma cell, partly on the basis of their affinity for antigen. Since SARS-CoV-2 has been in the human population only since December 2019, it remains unknown if the immunity is long-lasting in people who recover from the disease. The presence of neutralizing antibodies in blood strongly correlates with protection from infection, but the level of neutralizing antibody declines with time. Those with asymptomatic or mild disease had undetectable levels of neutralizing antibody two months after infection. In another study, the level of neutralizing antibodies fell four-fold one to four months after the onset of symptoms. However, the lack of antibodies in the blood does not mean antibodies will not be rapidly produced upon reexposure to SARS-CoV-2. Memory B cells specific for the spike and nucleocapsid proteins of SARS-CoV-2 last for at least six months after the appearance of symptoms.Reinfection with COVID‑19 is possible but uncommon. The first case of reinfection was documented in August 2020. A systematic review found 17 cases of confirmed reinfection in medical literature as of May 2021.\\n\\n\\n== Mortality ==\\n\\nSeveral measures are commonly used to quantify mortality. These numbers vary by region and over time and are influenced by the volume of testing, healthcare system quality, treatment options, time since the initial outbreak, and population characteristics such as age, sex, and overall health.The mortality rate reflects the number of deaths within a specific demographic group divided by the population of that demographic group. Consequently, the mortality rate reflects the prevalence as well as the severity of the disease within a given population. Mortality rates are highly correlated to age, with relatively low rates for young people and relatively high rates among the elderly. In fact, one relevant factor of mortality rates is the age structure of the countries\\' populations. For example, the case fatality rate for COVID‑19 is lower in India than in the US since India\\'s younger population represents a larger percentage than in the US.\\n\\n\\n=== Case fatality rate ===\\nThe case fatality rate (CFR) reflects the number of deaths divided by the number of diagnosed cases within a given time interval. Based on Johns Hopkins University statistics, the global death-to-case ratio is 1.24% (6,183,197/500,280,791) as of 12 April 2022. The number varies by region.\\n\\n\\t\\t\\n\\t\\t\\n\\t\\t\\n\\n\\n=== Infection fatality rate ===\\nA key metric in gauging the severity of COVID‑19 is the infection fatality rate (IFR), also referred to as the infection fatality ratio or infection fatality risk. This metric is calculated by dividing the total number of deaths from the disease by the total number of infected individuals; hence, in contrast to the CFR, the IFR incorporates asymptomatic and undiagnosed infections as well as reported cases.\\n\\n\\n==== Estimates ====\\n\\nA December 2020 systematic review and meta-analysis estimated that population IFR during the first wave of the pandemic was about 0.5% to 1% in many locations (including France, Netherlands, New Zealand, and Portugal), 1% to 2% in other locations (Australia, England, Lithuania, and Spain), and exceeded 2% in Italy. That study also found that most of these differences in IFR reflected corresponding differences in the age composition of the population and age-specific infection rates; in particular, the metaregression estimate of IFR is very low for children and younger adults (e.g., 0.002% at age 10 and 0.01% at age 25) but increases progressively to 0.4% at age 55, 1.4% at age 65, 4.6% at age 75, and 15% at age 85. These results were also highlighted in a December 2020 report issued by the WHO.\\nAn analysis of those IFR rates indicates that COVID‑19 is hazardous not only for the elderly but also for middle-aged adults, for whom the infection fatality rate of COVID-19 is two orders of magnitude greater than the annualized risk of a fatal automobile accident and far more dangerous than seasonal influenza.\\n\\n\\n==== Earlier estimates of IFR ====\\nAt an early stage of the pandemic, the World Health Organization reported estimates of IFR between 0.3% and 1%. On 2 July, The WHO\\'s chief scientist reported that the average IFR estimate presented at a two-day WHO expert forum was about 0.6%. In August, the WHO found that studies incorporating data from broad serology testing in Europe showed IFR estimates converging at approximately 0.5–1%. Firm lower limits of IFRs have been established in a number of locations such as New York City and Bergamo in Italy since the IFR cannot be less than the population fatality rate. (After sufficient time however, people can get reinfected). As of 10 July, in New York City, with a population of 8.4 million, 23,377 individuals (18,758 confirmed and 4,619 probable) have died with COVID‑19 (0.3% of the population). Antibody testing in New York City suggested an IFR of ~0.9%, and ~1.4%. In Bergamo province, 0.6% of the population has died. In September 2020, the U.S. Centers for Disease Control and Prevention (CDC) reported preliminary estimates of age-specific IFRs for public health planning purposes.\\n\\n\\n=== Sex differences ===\\n\\nCOVID‑19 case fatality rates are higher among men than women in most countries. However, in a few countries like India, Nepal, Vietnam, and Slovenia the fatality cases are higher in women than men. Globally, men are more likely to be admitted to the ICU and more likely to die. One meta-analysis found that globally, men were more likely to get COVID‑19 than women; there were approximately 55 men and 45 women per 100 infections (CI: 51.43–56.58).The Chinese Center for Disease Control and Prevention reported the death rate was 2.8% for men and 1.7% for women. Later reviews in June 2020 indicated that there is no significant difference in susceptibility or in CFR between genders. One review acknowledges the different mortality rates in Chinese men, suggesting that it may be attributable to lifestyle choices such as smoking and drinking alcohol rather than genetic factors. Smoking, which in some countries like China is mainly a male activity, is a habit that contributes to increasing significantly the case fatality rates among men. Sex-based immunological differences, lesser prevalence of smoking in women and men developing co-morbid conditions such as hypertension at a younger age than women could have contributed to the higher mortality in men. In Europe as of February 2020, 57% of the infected people were men and 72% of those died with COVID‑19 were men. As of April 2020, the US government is not tracking sex-related data of COVID‑19 infections. Research has shown that viral illnesses like Ebola, HIV, influenza and SARS affect men and women differently.\\n\\n\\n=== Ethnic differences ===\\nIn the US, a greater proportion of deaths due to COVID‑19 have occurred among African Americans and other minority groups. Structural factors that prevent them from practicing social distancing include their concentration in crowded substandard housing and in \"essential\" occupations such as retail grocery workers, public transit employees, health-care workers and custodial staff. Greater prevalence of lacking health insurance and care of underlying conditions such as diabetes, hypertension, and heart disease also increase their risk of death. Similar issues affect Native American and Latino communities. On the one hand, in the Dominican Republic there is a clear example of both gender and ethnic inequality. In this Latin American territory, there is great inequality and precariousness that especially affects Dominican women, with greater emphasis on those of Haitian descent. According to a US health policy non-profit, 34% of American Indian and Alaska Native People (AIAN) non-elderly adults are at risk of serious illness compared to 21% of white non-elderly adults. The source attributes it to disproportionately high rates of many health conditions that may put them at higher risk as well as living conditions like lack of access to clean water.Leaders have called for efforts to research and address the disparities. In the UK, a greater proportion of deaths due to COVID‑19 have occurred in those of a Black, Asian, and other ethnic minority background. More severe impacts upon victims including the relative incidence of the necessity of hospitalization requirements, and vulnerability to the disease has been associated via DNA analysis to be expressed in genetic variants at chromosomal region 3, features that are associated with European Neanderthal heritage. That structure imposes greater risks that those affected will develop a more severe form of the disease. The findings are from Professor Svante Pääbo and researchers he leads at the Max Planck Institute for Evolutionary Anthropology and the Karolinska Institutet. This admixture of modern human and Neanderthal genes is estimated to have occurred roughly between 50,000 and 60,000 years ago in Southern Europe.\\n\\n\\n=== Comorbidities ===\\nBiological factors (immune response) and the general behaviour (habits) can strongly determine the consequences of COVID‑19. Most of those who die of COVID‑19 have pre-existing (underlying) conditions, including hypertension, diabetes mellitus, and cardiovascular disease. According to March data from the United States, 89% of those hospitalised had preexisting conditions. The Italian Istituto Superiore di Sanità reported that out of 8.8% of deaths where medical charts were available, 96.1% of people had at least one comorbidity with the average person having 3.4 diseases. According to this report the most common comorbidities are hypertension (66% of deaths), type 2 diabetes (29.8% of deaths), Ischemic Heart Disease (27.6% of deaths), atrial fibrillation (23.1% of deaths) and chronic renal failure (20.2% of deaths).\\nMost critical respiratory comorbidities according to the Centers for Disease Control and Prevention (CDC), are: moderate or severe asthma, pre-existing COPD, pulmonary fibrosis, cystic fibrosis. Evidence stemming from meta-analysis of several smaller research papers also suggests that smoking can be associated with worse outcomes. When someone with existing respiratory problems is infected with COVID‑19, they might be at greater risk for severe symptoms. COVID‑19 also poses a greater risk to people who misuse opioids and methamphetamines, insofar as their drug use may have caused lung damage.In August 2020, the CDC issued a caution that tuberculosis (TB) infections could increase the risk of severe illness or death. The WHO recommended that people with respiratory symptoms be screened for both diseases, as testing positive for COVID‑19 could not rule out co-infections. Some projections have estimated that reduced TB detection due to the pandemic could result in 6.3 million additional TB cases and 1.4 million TB-related deaths by 2025.\\n\\n\\n== History ==\\n\\nThe virus is thought to be of natural animal origin, most likely through spillover infection. There are several theories about where the index case originated and investigations into the origin of the pandemic are ongoing. Phylogenetics estimates that SARS-CoV-2 arose in October or November 2019. A phylogenetic algorithm analysis suggested that the virus may have been circulating in Guangdong before Wuhan. Evidence suggests that it descends from a coronavirus that infects wild bats, and spread to humans through an intermediary wildlife host. The possibility that the virus was accidentally released from a laboratory is also under increasingly active consideration. U.S intelligence agencies found that the virus was not developed as a biological weapon and that it is unlikely for it to have been genetically engineered.The first confirmed human infections were in Wuhan, Hubei, China. A study of the first 41 cases of confirmed COVID‑19, published in January 2020 in The Lancet, reported the earliest date of onset of symptoms as 1 December 2019. Official publications from the WHO reported the earliest onset of symptoms as 8 December 2019. Human-to-human transmission was confirmed by the WHO and Chinese authorities by 20 January 2020. According to official Chinese sources, these were mostly linked to the Huanan Seafood Wholesale Market, which also sold live animals. In May 2020, George Gao, the director of the CDC, said animal samples collected from the seafood market had tested negative for the virus, indicating that the market was the site of an early superspreading event, but that it was not the site of the initial outbreak. Traces of the virus have been found in wastewater samples that were collected in Milan and Turin, Italy, on 18 December 2019.By December 2019, the spread of infection was almost entirely driven by human-to-human transmission. The number of COVID-19 cases in Hubei gradually increased, reaching sixty by 20 December, and at least 266 by 31 December. On 24 December, Wuhan Central Hospital sent a bronchoalveolar lavage fluid (BAL) sample from an unresolved clinical case to sequencing company Vision Medicals. On 27 and 28 December, Vision Medicals informed the Wuhan Central Hospital and the Chinese CDC of the results of the test, showing a new coronavirus. A pneumonia cluster of unknown cause was observed on 26 December and treated by the doctor Zhang Jixian in Hubei Provincial Hospital, who informed the Wuhan Jianghan CDC on 27 December. On 30 December, a test report addressed to Wuhan Central Hospital, from company CapitalBio Medlab, stated an erroneous positive result for SARS, causing a group of doctors at Wuhan Central Hospital to alert their colleagues and relevant hospital authorities of the result. The Wuhan Municipal Health Commission issued a notice to various medical institutions on \"the treatment of pneumonia of unknown cause\" that same evening. Eight of these doctors, including Li Wenliang (punished on 3 January), were later admonished by the police for spreading false rumours and another, Ai Fen, was reprimanded by her superiors for raising the alarm.The Wuhan Municipal Health Commission made the first public announcement of a pneumonia outbreak of unknown cause on 31 December, confirming 27 cases – enough to trigger an investigation.During the early stages of the outbreak, the number of cases doubled approximately every seven and a half days. In early and mid-January 2020, the virus spread to other Chinese provinces, helped by the Chinese New Year migration and Wuhan being a transport hub and major rail interchange. On 20 January, China reported nearly 140 new cases in one day, including two people in Beijing and one in Shenzhen. Later official data shows 6,174 people had already developed symptoms by then, and more may have been infected. A report in The Lancet on 24 January indicated human transmission, strongly recommended personal protective equipment for health workers, and said testing for the virus was essential due to its \"pandemic potential\". On 30 January, the WHO declared COVID-19 a Public Health Emergency of International Concern. By this time, the outbreak spread by a factor of 100 to 200 times.Italy had its first confirmed cases on 31 January 2020, two tourists from China. Italy overtook China as the country with the most deaths on 19 March 2020. By 26 March the United States had overtaken China and Italy with the highest number of confirmed cases in the world. Research on coronavirus genomes indicates the majority of COVID-19 cases in New York came from European travellers, rather than directly from China or any other Asian country. Retesting of prior samples found a person in France who had the virus on 27 December 2019, and a person in the United States who died from the disease on 6 February 2020.RT-PCR testing of untreated wastewater samples from Brazil and Italy have suggested detection of SARS-CoV-2 as early as November and December 2019, respectively, but the methods of such sewage studies have not been optimised, many have not been peer-reviewed, details are often missing, and there is a risk of false positives due to contamination or if only one gene target is detected. A September 2020 review journal article said, \"The possibility that the COVID‑19 infection had already spread to Europe at the end of last year is now indicated by abundant, even if partially circumstantial, evidence,\" including pneumonia case numbers and radiology in France and Italy in November and December.As of 1 October 2021, Reuters reported that it had estimated the worldwide total number of deaths due to COVID‑19 to have exceeded five million.\\n\\n\\n== Misinformation ==\\n\\nAfter the initial outbreak of COVID‑19, misinformation and disinformation regarding the origin, scale, prevention, treatment, and other aspects of the disease rapidly spread online.In September 2020, the US Centers for Disease Control and Prevention (CDC) published preliminary estimates of the risk of death by age groups in the United States, but those estimates were widely misreported and misunderstood.\\n\\n\\n== Other species ==\\n\\nHumans appear to be capable of spreading the virus to some other animals, a type of disease transmission referred to as zooanthroponosis.\\nSome pets, especially cats and ferrets, can catch this virus from infected humans. Symptoms in cats include respiratory (such as a cough) and digestive symptoms. Cats can spread the virus to other cats, and may be able to spread the virus to humans, but cat-to-human transmission of SARS-CoV-2 has not been proven. Compared to cats, dogs are less susceptible to this infection. Behaviors which increase the risk of transmission include kissing, licking, and petting the animal.The virus does not appear to be able to infect pigs, ducks, or chickens at all. Mice, rats, and rabbits, if they can be infected at all, are unlikely to be involved in spreading the virus.Tigers and lions in zoos have become infected as a result of contact with infected humans. As expected, monkeys and great ape species such as orangutans can also be infected with the COVID‑19 virus.Minks, which are in the same family as ferrets, have been infected. Minks may be asymptomatic, and can also spread the virus to humans. Multiple countries have identified infected animals in mink farms. Denmark, a major producer of mink pelts, ordered the slaughter of all minks over fears of viral mutations, following an outbreak referred to as Cluster 5. A vaccine for mink and other animals is being researched.\\n\\n\\n== Research ==\\n\\nInternational research on vaccines and medicines in COVID‑19 is underway by government organisations, academic groups, and industry researchers. The CDC has classified it to require a BSL3 grade laboratory. There has been a great deal of COVID‑19 research, involving accelerated research processes and publishing shortcuts to meet the global demand.As of December 2020, hundreds of clinical trials have been undertaken, with research happening on every continent except Antarctica. As of November 2020, more than 200 possible treatments had been studied in humans so far.\\n\\n\\n=== Transmission and prevention research ===\\n\\nModelling research has been conducted with several objectives, including predictions of the dynamics of transmission, diagnosis and prognosis of infection, estimation of the impact of interventions, or allocation of resources. Modelling studies are mostly based on compartmental models in epidemiology, estimating the number of infected people over time under given conditions. Several other types of models have been developed and used during the COVID‑19 including computational fluid dynamics models to study the flow physics of COVID‑19, retrofits of crowd movement models to study occupant exposure, mobility-data based models to investigate transmission, or the use of macroeconomic models to assess the economic impact of the pandemic. Further, conceptual frameworks from crisis management research have been applied to better understand the effects of COVID‑19 on organizations worldwide.\\n\\n\\n=== Treatment-related research ===\\n\\nRepurposed antiviral drugs make up most of the research into COVID‑19 treatments. Other candidates in trials include vasodilators, corticosteroids, immune therapies, lipoic acid, bevacizumab, and recombinant angiotensin-converting enzyme 2.In March 2020, the World Health Organization (WHO) initiated the Solidarity trial to assess the treatment effects of some promising drugs: an experimental drug called remdesivir; anti-malarial drugs chloroquine and hydroxychloroquine; two anti-HIV drugs, lopinavir/ritonavir; and interferon-beta. More than 300 active clinical trials are underway as of April 2020.Research on the antimalarial drugs hydroxychloroquine and chloroquine showed that they were ineffective at best, and that they may reduce the antiviral activity of remdesivir. By May 2020, France, Italy, and Belgium had banned the use of hydroxychloroquine as a COVID‑19 treatment.In June, initial results from the randomised RECOVERY Trial in the United Kingdom showed that dexamethasone reduced mortality by one third for people who are critically ill on ventilators and one fifth for those receiving supplemental oxygen. Because this is a well-tested and widely available treatment, it was welcomed by the WHO, which is in the process of updating treatment guidelines to include dexamethasone and other steroids. Based on those preliminary results, dexamethasone treatment has been recommended by the NIH for patients with COVID‑19 who are mechanically ventilated or who require supplemental oxygen but not in patients with COVID‑19 who do not require supplemental oxygen.In September 2020, the WHO released updated guidance on using corticosteroids for COVID‑19. The WHO recommends systemic corticosteroids rather than no systemic corticosteroids for the treatment of people with severe and critical COVID‑19 (strong recommendation, based on moderate certainty evidence). The WHO suggests not to use corticosteroids in the treatment of people with non-severe COVID‑19 (conditional recommendation, based on low certainty evidence). The updated guidance was based on a meta-analysis of clinical trials of critically ill COVID‑19 patients.In September 2020, the European Medicines Agency (EMA) endorsed the use of dexamethasone in adults and adolescents from twelve years of age and weighing at least 40 kilograms (88 lb) who require supplemental oxygen therapy. Dexamethasone can be taken by mouth or given as an injection or infusion (drip) into a vein.In November 2020, the US Food and Drug Administration (FDA) issued an emergency use authorization for the investigational monoclonal antibody therapy bamlanivimab for the treatment of mild-to-moderate COVID‑19. Bamlanivimab is authorized for people with positive results of direct SARS-CoV-2 viral testing who are twelve years of age and older weighing at least 40 kilograms (88 lb), and who are at high risk for progressing to severe COVID‑19 or hospitalization. This includes those who are 65 years of age or older, or who have chronic medical conditions.In February 2021, the FDA issued an emergency use authorization (EUA) for bamlanivimab and etesevimab administered together for the treatment of mild to moderate COVID‑19 in people twelve years of age or older weighing at least 40 kilograms (88 lb) who test positive for SARS‑CoV‑2 and who are at high risk for progressing to severe COVID‑19. The authorized use includes treatment for those who are 65 years of age or older or who have certain chronic medical conditions.In April 2021, the FDA revoked the emergency use authorization (EUA) that allowed for the investigational monoclonal antibody therapy bamlanivimab, when administered alone, to be used for the treatment of mild-to-moderate COVID‑19 in adults and certain pediatric patients.\\n\\n\\n==== Cytokine storm ====\\n\\nA cytokine storm can be a complication in the later stages of severe COVID‑19. A cytokine storm is a potentially deadly immune reaction where a large amount of pro-inflammatory cytokines and chemokines are released too quickly. A cytokine storm can lead to ARDS and multiple organ failure. Data collected from Jin Yin-tan Hospital in Wuhan, China indicates that patients who had more severe responses to COVID‑19 had greater amounts of pro-inflammatory cytokines and chemokines in their system than patients who had milder responses. These high levels of pro-inflammatory cytokines and chemokines indicate presence of a cytokine storm.Tocilizumab has been included in treatment guidelines by China\\'s National Health Commission after a small study was completed. It is undergoing a Phase II non-randomised trial at the national level in Italy after showing positive results in people with severe disease. Combined with a serum ferritin blood test to identify a cytokine storm (also called cytokine storm syndrome, not to be confused with cytokine release syndrome), it is meant to counter such developments, which are thought to be the cause of death in some affected people. The interleukin-6 receptor (IL-6R) antagonist was approved by the FDA to undergo a Phase III clinical trial assessing its effectiveness on COVID‑19 based on retrospective case studies for the treatment of steroid-refractory cytokine release syndrome induced by a different cause, CAR T cell therapy, in 2017. There is no randomised, controlled evidence that tocilizumab is an efficacious treatment for CRS. Prophylactic tocilizumab has been shown to increase serum IL-6 levels by saturating the IL-6R, driving IL-6 across the blood-brain barrier, and exacerbating neurotoxicity while having no effect on the incidence of CRS.Lenzilumab, an anti-GM-CSF monoclonal antibody, is protective in murine models for CAR T cell-induced CRS and neurotoxicity and is a viable therapeutic option due to the observed increase of pathogenic GM-CSF secreting T cells in hospitalised patients with COVID‑19.\\n\\n\\n==== Passive antibodies ====\\n\\nTransferring purified and concentrated antibodies produced by the immune systems of those who have recovered from COVID‑19 to people who need them is being investigated as a non-vaccine method of passive immunisation. Viral neutralization is the anticipated mechanism of action by which passive antibody therapy can mediate defence against SARS-CoV-2. The spike protein of SARS-CoV-2 is the primary target for neutralizing antibodies. As of 8 August 2020, eight neutralizing antibodies targeting the spike protein of SARS-CoV-2 have entered clinical studies. It has been proposed that selection of broad-neutralizing antibodies against SARS-CoV-2 and SARS-CoV might be useful for treating not only COVID‑19 but also future SARS-related CoV infections. Other mechanisms, however, such as antibody-dependent cellular cytotoxicity or phagocytosis, may be possible. Other forms of passive antibody therapy, for example, using manufactured monoclonal antibodies, are in development.The use of passive antibodies to treat people with active COVID‑19 is also being studied. This involves the production of convalescent serum, which consists of the liquid portion of the blood from people who recovered from the infection and contains antibodies specific to this virus, which is then administered to active patients. This strategy was tried for SARS with inconclusive results. An updated Cochrane review in May 2021 found high certainty evidence that, for the treatment of people with moderate to severe COVID‑19, convalescent plasma did not reduce mortality or bring about symptom improvement. There continues to be uncertainty about the safety of convalescent plasma administration to people with COVID‑19 and differing outcomes measured in different studies limits their use in determining efficacy.\\n\\n\\n=== Bioethics ===\\nSince the outbreak of the COVID‑19 pandemic, scholars have explored the bioethics, normative economics, and political theories of healthcare policies related to the public health crisis. Academics have pointed to the moral distress of healthcare workers, ethics of distributing scarce healthcare resources such as ventilators, and the global justice of vaccine diplomacies. The socio-economic inequalities between genders, races, groups with disabilities, communities, regions, countries, and continents have also drawn attention in academia and the general public.\\n\\n\\n== Effect on other diseases and the pharmacy trade ==\\nThere was a report on 3 March 2021, that social distancing and common wearing of surgical masks and similar as a common precaution against COVID‑19 caused a drop in the spread rate of the common cold and flu.\\n\\n\\n== See also ==\\nCoronavirus diseases, a group of closely related syndromes\\nDisease X, a WHO term\\nLaw of declining virulence – Claim by epidemiologist Theobald Smith\\nTheory of virulence – Theory by biologist Paul W. Ewald\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\n\\n\\n== External links ==\\n\\n\\n=== Health agencies ===\\nCoronavirus disease (COVID‑19) Facts by the World Health Organization (WHO)\\nCoronavirus 2019 (COVID-19) by the US Centers for Disease Control and Prevention (CDC)\\nCoronavirus (COVID‑19) by the UK National Health Service (NHS)\\n\\n\\n=== Directories ===\\nCoronavirus Resource Center at the Center for Inquiry\\nCOVID-19 at Curlie\\nCOVID‑19 Resource Directory on OpenMD\\nCOVID‑19 Information on FireMountain.net\\n\\n\\n=== Medical journals ===\\nCoronavirus Disease 2019 (COVID‑19) by JAMA\\nBMJ\\'s Coronavirus (covid‑19) Hub by the BMJ\\nNovel Coronavirus Information Center by Elsevier\\nCOVID‑19 Resource Centre by The Lancet\\nCoronavirus (COVID‑19) Research Highlights by Springer Nature\\nCoronavirus (Covid‑19) by The New England Journal of Medicine\\nCovid‑19: Novel Coronavirus by Wiley Publishing\\n\\n\\n=== Treatment guidelines ===\\n\"JHMI Clinical Recommendations for Available Pharmacologic Therapies for COVID-19\" (PDF). Johns Hopkins Medicine.\\n\"Bouncing Back From COVID-19: Your Guide to Restoring Movement\" (PDF). Johns Hopkins Medicine.\\n\"Guidelines on the Treatment and Management of Patients with COVID-19\". Infectious Diseases Society of America.\\n\"Coronavirus Disease 2019 (COVID-19) Treatment Guidelines\" (PDF). National Institutes of Health.\\nWorld Health Organization (2022). Therapeutics and COVID-19: living guideline, 14 January 2022 (Report). hdl:10665/351006. WHO/2019-nCoV/therapeutics/2022.1.\\nNHS England and NHS Improvement. National Guidance for post-COVID syndrome assessment clinics (Report).',\n",
              " 'Coronaviruses are a group of related RNA viruses that cause diseases in mammals and birds. In humans and birds, they cause respiratory tract infections that can range from mild to lethal. Mild illnesses in humans include some cases of the common cold (which is also caused by other viruses, predominantly rhinoviruses), while more lethal varieties can cause SARS, MERS and COVID-19, which is causing an ongoing pandemic. In cows and pigs they cause diarrhea, while in mice they cause hepatitis and encephalomyelitis.\\nCoronaviruses constitute the subfamily Orthocoronavirinae, in the family Coronaviridae, order Nidovirales and realm Riboviria. They are enveloped viruses with a positive-sense single-stranded RNA genome and a nucleocapsid of helical symmetry. The genome size of coronaviruses ranges from approximately 26 to 32 kilobases, one of the largest among RNA viruses. They have characteristic club-shaped spikes that project from their surface, which in electron micrographs create an image reminiscent of the solar corona, from which their name derives.\\n\\n\\n== Etymology ==\\nThe name \"coronavirus\" is derived from Latin corona, meaning \"crown\" or \"wreath\", itself a borrowing from Greek κορώνη korṓnē, \"garland, wreath\". The name was coined by June Almeida and David Tyrrell who first observed and studied human coronaviruses. The word was first used in print in 1968 by an informal group of virologists in the journal Nature to designate the new family of viruses. The name refers to the characteristic appearance of virions (the infective form of the virus) by electron microscopy, which have a fringe of large, bulbous surface projections creating an image reminiscent of the solar corona or halo. This morphology is created by the viral spike peplomers, which are proteins on the surface of the virus.The scientific name Coronavirus was accepted as a genus name by the International Committee for the Nomenclature of Viruses (later renamed International Committee on Taxonomy of Viruses) in 1971. As the number of new species increased, the genus was split into four genera, namely Alphacoronavirus, Betacoronavirus, Deltacoronavirus, and Gammacoronavirus in 2009. The common name coronavirus is used to refer to any member of the subfamily Orthocoronavirinae. As of 2020, 45 species are officially recognised.\\n\\n\\n== History ==\\n\\nThe earliest reports of a coronavirus infection in animals occurred in the late 1920s, when an acute respiratory infection of domesticated chickens emerged in North America. Arthur Schalk and M.C. Hawn in 1931 made the first detailed report which described a new respiratory infection of chickens in North Dakota. The infection of new-born chicks was characterized by gasping and listlessness with high mortality rates of 40–90%. Leland David Bushnell and Carl Alfred Brandly isolated the virus that caused the infection in 1933. The virus was then known as infectious bronchitis virus (IBV). Charles D. Hudson and Fred Robert Beaudette cultivated the virus for the first time in 1937. The specimen came to be known as the Beaudette strain. In the late 1940s, two more animal coronaviruses, JHM that causes brain disease (murine encephalitis) and mouse hepatitis virus (MHV) that causes hepatitis in mice were discovered. It was not realized at the time that these three different viruses were related.Human coronaviruses were discovered in the 1960s using two different methods in the United Kingdom and the United States. E.C. Kendall, Malcolm Bynoe, and David Tyrrell working at the Common Cold Unit of the British Medical Research Council collected a unique common cold virus designated B814 in 1961. The virus could not be cultivated using standard techniques which had successfully cultivated rhinoviruses, adenoviruses and other known common cold viruses. In 1965, Tyrrell and Bynoe successfully cultivated the novel virus by serially passing it through organ culture of human embryonic trachea. The new cultivating method was introduced to the lab by Bertil Hoorn. The isolated virus when intranasally inoculated into volunteers caused a cold and was inactivated by ether which indicated it had a lipid envelope. Dorothy Hamre and John Procknow at the University of Chicago isolated a novel cold from medical students in 1962. They isolated and grew the virus in kidney tissue culture, designating it 229E. The novel virus caused a cold in volunteers and, like B814, was inactivated by ether.\\n\\nScottish virologist June Almeida at St Thomas\\' Hospital in London, collaborating with Tyrrell, compared the structures of IBV, B814 and 229E in 1967. Using electron microscopy the three viruses were shown to be morphologically related by their general shape and distinctive club-like spikes. A research group at the National Institute of Health the same year was able to isolate another member of this new group of viruses using organ culture and named one of the samples OC43 (OC for organ culture). Like B814, 229E, and IBV, the novel cold virus OC43 had distinctive club-like spikes when observed with the electron microscope.The IBV-like novel cold viruses were soon shown to be also morphologically related to the mouse hepatitis virus. This new group of viruses were named coronaviruses after their distinctive morphological appearance. Human coronavirus 229E and human coronavirus OC43 continued to be studied in subsequent decades. The coronavirus strain B814 was lost. It is not known which present human coronavirus it was. Other human coronaviruses have since been identified, including SARS-CoV in 2003, HCoV NL63 in 2003, HCoV HKU1 in 2004, MERS-CoV in 2013, and SARS-CoV-2 in 2019. There have also been a large number of animal coronaviruses identified since the 1960s.\\n\\n\\n== Microbiology ==\\n\\n\\n=== Structure ===\\n\\nCoronaviruses are large, roughly spherical particles with unique surface projections. Their size is highly variable with average diameters of 80 to 120 nm. Extreme sizes are known from 50 to 200 nm in diameter. The total molecular mass is on average 40,000 kDa. They are enclosed in an envelope embedded with a number of protein molecules. The lipid bilayer envelope, membrane proteins, and nucleocapsid protect the virus when it is outside the host cell.The viral envelope is made up of a lipid bilayer in which the membrane (M), envelope (E) and spike (S) structural proteins are anchored. The molar ratio of E:S:M in the lipid bilayer is approximately 1:20:300.  The E and M protein are the structural proteins that combined with the lipid bilayer to shape the viral envelope and maintain its size. S proteins are needed for interaction with the host cells. But human coronavirus NL63 is peculiar in that its M protein has the binding site for the host cell, and not its S protein. The diameter of the envelope is 85 nm. The envelope of the virus in electron micrographs appears as a distinct pair of electron-dense shells (shells that are relatively opaque to the electron beam used to scan the virus particle).The M protein is the main structural protein of the envelope that provides the overall shape and is a type III membrane protein. It consists of 218 to 263 Amino acid residues and forms a layer 7.8 nm thick. It has three domains, a short N-terminal ectodomain, a triple-spanning transmembrane domain, and a C-terminal endodomain. The C-terminal domain forms a matrix-like lattice that adds to the extra-thickness of the envelope. Different species can have either N- or O-linked glycans in their protein amino-terminal domain. The M protein is crucial during the assembly, budding, envelope formation, and pathogenesis stages of the virus lifecycle.The E proteins are minor structural proteins and highly variable in different species. There are only about 20 copies of the E protein molecule in a coronavirus particle. They are 8.4 to 12 kDa in size and are composed of 76 to 109 amino acids. They are integral proteins (i.e. embedded in the lipid layer) and have two domains namely a transmembrane domain and an extramembrane C-terminal domain. They are almost fully α-helical, with a single α-helical transmembrane domain, and form pentameric (five-molecular) ion channels in the lipid bilayer. They are responsible for virion assembly, intracellular trafficking and morphogenesis (budding).\\n\\nThe spikes are the most distinguishing feature of coronaviruses and are responsible for the corona- or halo-like surface. On average a coronavirus particle has 74 surface spikes. Each spike is about 20 nm long and is composed of a trimer of the S protein. The S protein is in turn composed of an S1 and S2 subunit. The homotrimeric S protein is a class I fusion protein which mediates the receptor binding and membrane fusion between the virus and host cell. The S1 subunit forms the head of the spike and has the receptor-binding domain (RBD). The S2 subunit forms the stem which anchors the spike in the viral envelope and on protease activation enables fusion. The two subunits remain noncovalently linked as they are exposed on the viral surface until they attach to the host cell membrane. In a functionally active state, three S1 are attached to two S2 subunits. The subunit complex is split into individual subunits when the virus binds and fuses with the host cell under the action of proteases such as cathepsin family and transmembrane protease serine 2 (TMPRSS2) of the host cell.\\n\\nS1 proteins are the most critical components in terms of infection. They are also the most variable components as they are responsible for host cell specificity. They possess two major domains named N-terminal domain (S1-NTD) and C-terminal domain (S1-CTD), both of which serve as the receptor-binding domains. The NTDs recognize and bind sugars on the surface of the host cell. An exception is the MHV NTD that binds to a protein receptor carcinoembryonic antigen-related cell adhesion molecule 1 (CEACAM1). S1-CTDs are responsible for recognizing different protein receptors such as angiotensin-converting enzyme 2 (ACE2), aminopeptidase N (APN), and dipeptidyl peptidase 4 (DPP4).A subset of coronaviruses (specifically the members of betacoronavirus subgroup A) also has a shorter spike-like surface protein called hemagglutinin esterase (HE). The HE proteins occur as homodimers composed of about 400 amino acid residues and are 40 to 50 kDa in size. They appear as tiny surface projections of 5 to 7 nm long embedded in between the spikes. They help in the attachment to and detachment from the host cell.Inside the envelope, there is the nucleocapsid, which is formed from multiple copies of the nucleocapsid (N) protein, which are bound to the positive-sense single-stranded RNA genome in a continuous Beads-on-a-string type conformation. N protein is a phosphoprotein of 43 to 50 kDa in size, and is divided into three conserved domains. The majority of the protein is made up of domains 1 and 2, which are typically rich in arginines and lysines. Domain 3 has a short carboxy terminal end and has a net negative charge due to excess of acidic over basic amino acid residues.\\n\\n\\n=== Genome ===\\n\\nCoronaviruses contain a positive-sense, single-stranded RNA genome. The genome size for coronaviruses ranges from 26.4 to 31.7 kilobases. The genome size is one of the largest among RNA viruses. The genome has a 5′ methylated cap and a 3′ polyadenylated tail.The genome organization for a coronavirus is 5′-leader-UTR-replicase (ORF1ab)-spike (S)-envelope (E)-membrane (M)-nucleocapsid (N)-3′UTR-poly (A) tail. The open reading frames 1a and 1b, which occupy the first two-thirds of the genome, encode the replicase polyprotein (pp1ab). The replicase polyprotein self cleaves to form 16 nonstructural proteins (nsp1–nsp16).The later reading frames encode the four major structural proteins: spike, envelope, membrane, and nucleocapsid. Interspersed between these reading frames are the reading frames for the accessory proteins. The number of accessory proteins and their function is unique depending on the specific coronavirus.\\n\\n\\n=== Replication cycle ===\\n\\n\\n==== Cell entry ====\\n\\nInfection begins when the viral spike protein attaches to its complementary host cell receptor. After attachment, a protease of the host cell cleaves and activates the receptor-attached spike protein. Depending on the host cell protease available, cleavage and activation allows the virus to enter the host cell by endocytosis or direct fusion of the viral envelope with the host membrane.\\n\\n\\n==== Genome translation ====\\nOn entry into the host cell, the virus particle is uncoated, and its genome enters the cell cytoplasm. The coronavirus RNA genome has a 5′ methylated cap and a 3′ polyadenylated tail, which allows it to act like a messenger RNA and be directly translated by the host cell\\'s ribosomes. The host ribosomes translate the initial overlapping open reading frames ORF1a and ORF1b of the virus genome into two large overlapping polyproteins, pp1a and pp1ab.The larger polyprotein pp1ab is a result of a -1 ribosomal frameshift caused by a slippery sequence (UUUAAAC) and a downstream RNA pseudoknot at the end of open reading frame ORF1a. The ribosomal frameshift allows for the continuous translation of ORF1a followed by ORF1b.The polyproteins have their own proteases,  PLpro (nsp3) and 3CLpro (nsp5), which cleave the polyproteins at different specific sites. The cleavage of polyprotein pp1ab yields 16 nonstructural proteins (nsp1 to nsp16). Product proteins include various replication  proteins such as RNA-dependent RNA polymerase (nsp12), RNA helicase (nsp13), and exoribonuclease (nsp14).\\n\\n\\n==== Replicase-transcriptase ====\\n\\nA number of the nonstructural proteins coalesce to form a multi-protein replicase-transcriptase complex (RTC). The main replicase-transcriptase protein is the RNA-dependent RNA polymerase (RdRp). It is directly involved in the replication and transcription of RNA from an RNA strand. The other nonstructural proteins in the complex assist in the replication and transcription process. The exoribonuclease nonstructural protein, for instance, provides extra fidelity to replication by providing a proofreading function which the RNA-dependent RNA polymerase lacks.Replication – One of the main functions of the complex is to replicate the viral genome. RdRp directly mediates the synthesis of negative-sense genomic RNA from the positive-sense genomic RNA. This is followed by the replication of positive-sense genomic RNA from the negative-sense genomic RNA.\\n\\nTranscription – The other important function of the complex is to transcribe the viral genome. RdRp directly mediates the synthesis of negative-sense subgenomic RNA molecules from the positive-sense genomic RNA. This process is followed by the transcription of these negative-sense subgenomic RNA molecules to their corresponding positive-sense mRNAs. The subgenomic mRNAs form a \"nested set\" which have a common 5\\'-head and partially duplicate 3\\'-end.Recombination – The replicase-transcriptase complex is also capable of genetic recombination when at least two viral genomes are present in the same infected cell.  RNA recombination appears to be a major driving force in determining genetic variability within a coronavirus species, the capability of a coronavirus species to jump from one host to another and, infrequently, in determining the emergence of novel coronaviruses.  The exact mechanism of recombination in coronaviruses is unclear, but likely involves template switching during genome replication.\\n\\n\\n==== Assembly and release ====\\nThe replicated positive-sense genomic RNA becomes the genome of the progeny viruses. The mRNAs are gene transcripts of the last third of the virus genome after the initial overlapping reading frame. These mRNAs are translated by the host\\'s ribosomes into the structural proteins and many accessory proteins. RNA translation occurs inside the endoplasmic reticulum. The viral structural proteins S, E, and M move along the secretory pathway into the Golgi intermediate compartment. There, the M proteins direct most protein-protein interactions required for the assembly of viruses following its binding to the nucleocapsid. Progeny viruses are then released from the host cell by exocytosis through secretory vesicles. Once released the viruses can infect other host cells.\\n\\n\\n=== Transmission ===\\nInfected carriers are able to shed viruses into the environment. The interaction of the coronavirus spike protein with its complementary cell receptor is central in determining the tissue tropism, infectivity, and species range of the released virus. Coronaviruses mainly target epithelial cells. They are transmitted from one host to another host, depending on the coronavirus species, by either an aerosol, fomite, or fecal-oral route.Human coronaviruses infect the epithelial cells of the respiratory tract, while animal coronaviruses generally infect the epithelial cells of the digestive tract. SARS coronavirus, for example, infects the human epithelial cells of the lungs via an aerosol route by binding to the angiotensin-converting enzyme 2 (ACE2) receptor. Transmissible gastroenteritis coronavirus (TGEV) infects the pig epithelial cells of the digestive tract via a fecal-oral route by binding to the Alanine aminopeptidase (APN) receptor.\\n\\n\\n== Classification ==\\n\\nCoronaviruses form the subfamily Orthocoronavirinae, which is one of two sub-families in the family Coronaviridae, order Nidovirales, and realm Riboviria. They are divided into the four genera: Alphacoronavirus, Betacoronavirus, Gammacoronavirus and Deltacoronavirus. Alphacoronaviruses and betacoronaviruses infect mammals, while gammacoronaviruses and deltacoronaviruses primarily infect birds.\\n\\nGenus: Alphacoronavirus;Species: Alphacoronavirus 1 (TGEV, Feline coronavirus, Canine coronavirus), Human coronavirus 229E, Human coronavirus NL63, Miniopterus bat coronavirus 1, Miniopterus bat coronavirus HKU8, Porcine epidemic diarrhea virus, Rhinolophus bat coronavirus HKU2, Scotophilus bat coronavirus 512\\nGenus Betacoronavirus;Species: Betacoronavirus 1 (Bovine Coronavirus, Human coronavirus OC43), Hedgehog coronavirus 1, Human coronavirus HKU1, Middle East respiratory syndrome-related coronavirus, Murine coronavirus, Pipistrellus bat coronavirus HKU5, Rousettus bat coronavirus HKU9, Severe acute respiratory syndrome–related coronavirus (SARS-CoV, SARS-CoV-2), Tylonycteris bat coronavirus HKU4\\nGenus Gammacoronavirus;Species: Avian coronavirus, Beluga whale coronavirus SW1\\nGenus Deltacoronavirus\\nSpecies: Bulbul coronavirus HKU11, Porcine coronavirus HKU15\\n\\n\\n== Origin ==\\n\\nThe most recent common ancestor (MRCA) of all coronaviruses is estimated to have existed as recently as 8000 BCE, although some models place the common ancestor as far back as 55 million years or more, implying long term coevolution with bat and avian species. The most recent common ancestor of the alphacoronavirus line has been placed at about 2400 BCE, of the betacoronavirus line at 3300 BCE, of the gammacoronavirus line at 2800 BCE, and the deltacoronavirus line at about 3000 BCE. Bats and birds, as warm-blooded flying vertebrates, are an ideal natural reservoir for the coronavirus gene pool (with bats the reservoir for alphacoronaviruses and betacoronavirus – and birds the reservoir for gammacoronaviruses and deltacoronaviruses). The large number and global range of bat and avian species that host viruses have enabled extensive evolution and dissemination of coronaviruses.Many human coronaviruses have their origin in bats. The human coronavirus NL63 shared a common ancestor with a bat coronavirus (ARCoV.2) between 1190 and 1449 CE. The human coronavirus 229E shared a common ancestor with a bat coronavirus (GhanaGrp1 Bt CoV) between 1686 and 1800 CE. More recently, alpaca coronavirus and human coronavirus 229E diverged sometime before 1960. MERS-CoV emerged in humans from bats through the intermediate host of camels. MERS-CoV, although related to several bat coronavirus species, appears to have diverged from these several centuries ago. The most closely related bat coronavirus and SARS-CoV diverged in 1986. The ancestors of SARS-CoV first infected leaf-nose bats of the genus Hipposideridae; subsequently, they spread to horseshoe bats in the species Rhinolophidae, then to Asian palm civets, and finally to humans.Unlike other betacoronaviruses, bovine coronavirus of the species Betacoronavirus 1 and subgenus Embecovirus is thought to have originated in rodents and not in bats. In the 1790s, equine coronavirus diverged from the bovine coronavirus after a cross-species jump. Later in the 1890s, human coronavirus OC43 diverged from bovine coronavirus after another cross-species spillover event. It is speculated that the flu pandemic of 1890 may have been caused by this spillover event, and not by the influenza virus, because of the related timing, neurological symptoms, and unknown causative agent of the pandemic. Besides causing respiratory infections, human coronavirus OC43 is also suspected of playing a role in neurological diseases. In the 1950s, the human coronavirus OC43 began to diverge into its present genotypes. Phylogenetically, mouse hepatitis virus (Murine coronavirus), which infects the mouse\\'s liver and central nervous system, is related to human coronavirus OC43 and bovine coronavirus. Human coronavirus HKU1, like the aforementioned viruses, also has its origins in rodents.\\n\\n\\n== Infection in humans ==\\n\\nCoronaviruses vary significantly in risk factor. Some can kill more than 30% of those infected, such as MERS-CoV, and some are relatively harmless, such as the common cold. Coronaviruses can cause colds with major symptoms, such as fever, and a sore throat from swollen adenoids. Coronaviruses can cause pneumonia (either direct viral pneumonia or secondary bacterial pneumonia) and bronchitis (either direct viral bronchitis or secondary bacterial bronchitis). The human coronavirus discovered in 2003, SARS-CoV, which causes severe acute respiratory syndrome (SARS), has a unique pathogenesis because it causes both upper and lower respiratory tract infections.Six species of human coronaviruses are known, with one species subdivided into two different strains, making seven strains of human coronaviruses altogether.\\n\\nFour human coronaviruses produce symptoms that are generally mild, even though it is contended they might have been more aggressive in the past:\\nHuman coronavirus OC43 (HCoV-OC43), β-CoV\\nHuman coronavirus HKU1 (HCoV-HKU1), β-CoV\\nHuman coronavirus 229E (HCoV-229E), α-CoV\\nHuman coronavirus NL63 (HCoV-NL63), α-CoV–Three human coronaviruses produce potentially severe symptoms:\\n\\nSevere acute respiratory syndrome coronavirus (SARS-CoV), β-CoV (identified in 2003)\\nMiddle East respiratory syndrome-related coronavirus (MERS-CoV), β-CoV (identified in 2012)\\nSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2), β-CoV (identified in 2019)These cause the diseases commonly called SARS, MERS, and COVID-19 respectively.\\n\\n\\n=== Common cold ===\\n\\nAlthough the common cold is usually caused by rhinoviruses, in about 15% of cases the cause is a coronavirus. The human coronaviruses HCoV-OC43, HCoV-HKU1, HCoV-229E, and HCoV-NL63 continually circulate in the human population in adults and children worldwide and produce the generally mild symptoms of the common cold. The four mild coronaviruses have a seasonal incidence occurring in the winter months in temperate climates.  There is no preponderance in any season in tropical climates.\\n\\n\\n=== Severe acute respiratory syndrome (SARS) ===\\n\\nIn 2003, following the outbreak of severe acute respiratory syndrome (SARS) which had begun the prior year in Asia, and secondary cases elsewhere in the world, the World Health Organization (WHO) issued a press release stating that a novel coronavirus identified by several laboratories was the causative agent for SARS. The virus was officially named the SARS coronavirus (SARS-CoV). More than 8,000 people from 29 countries and territories were infected, and at least 774 died.\\n\\n\\n=== Middle East respiratory syndrome (MERS) ===\\n\\nIn September 2012, a new type of coronavirus was identified, initially called Novel Coronavirus 2012, and now officially named Middle East respiratory syndrome coronavirus (MERS-CoV). The World Health Organization issued a global alert soon after. The WHO update on 28 September 2012 said the virus did not seem to pass easily from person to person. However, on 12 May 2013, a case of human-to-human transmission in France was confirmed by the French Ministry of Social Affairs and Health. In addition, cases of human-to-human transmission were reported by the Ministry of Health in Tunisia. Two confirmed cases involved people who seemed to have caught the disease from their late father, who became ill after a visit to Qatar and Saudi Arabia. Despite this, it appears the virus had trouble spreading from human to human, as most individuals who are infected do not transmit the virus. By 30 October 2013, there were 124 cases and 52 deaths in Saudi Arabia.After the Dutch Erasmus Medical Centre sequenced the virus, the virus was given a new name, Human Coronavirus–Erasmus Medical Centre (HCoV-EMC). The final name for the virus is Middle East respiratory syndrome coronavirus (MERS-CoV). The only U.S. cases (both survived) were recorded in May 2014.In May 2015, an outbreak of MERS-CoV occurred in the Republic of Korea, when a man who had traveled to the Middle East, visited four hospitals in the Seoul area to treat his illness. This caused one of the largest outbreaks of MERS-CoV outside the Middle East. As of December 2019, 2,468 cases of MERS-CoV infection had been confirmed by laboratory tests, 851 of which were fatal, a mortality rate of approximately 34.5%.\\n\\n\\n=== Coronavirus disease 2019 (COVID-19) ===\\n\\nIn December 2019, a pneumonia outbreak was reported in Wuhan, China. On 31 December 2019, the outbreak was traced to a novel strain of coronavirus, which was given the interim name 2019-nCoV by the World Health Organization, later renamed SARS-CoV-2 by the International Committee on Taxonomy of Viruses.\\nAs of 12 April 2022, there have been at least 6,183,197 confirmed deaths and more than 500,280,791 confirmed cases in the COVID-19 pandemic. The Wuhan strain has been identified as a new strain of Betacoronavirus from group 2B with approximately 70% genetic similarity to the SARS-CoV. The virus has a 96% similarity to a bat coronavirus, so it is widely suspected to originate from bats as well.\\n\\n\\n=== Coronavirus HuPn-2018 ===\\n\\nDuring a surveillance study of archived samples of Malaysian viral pneumonia patients, virologists identified a strain of canine coronavirus which has infected humans in 2018.\\n\\n\\n== Infection in animals ==\\nCoronaviruses have been recognized as causing pathological conditions in veterinary medicine since the 1930s.  They infect a range of animals including swine, cattle, horses, camels, cats, dogs, rodents, birds and bats.  The majority of animal related coronaviruses infect the intestinal tract and are transmitted by a fecal-oral route. Significant research efforts have been focused on elucidating the viral pathogenesis of these animal coronaviruses, especially by virologists interested in veterinary and zoonotic diseases.\\n\\n\\n=== Farm animals ===\\nCoronaviruses infect domesticated birds. Infectious bronchitis virus (IBV), a type of coronavirus, causes avian infectious bronchitis. The virus is of concern to the poultry industry because of the high mortality from infection, its rapid spread, and its effect on production. The virus affects both meat production and egg production and causes substantial economic loss. In chickens, infectious bronchitis virus targets not only the respiratory tract but also the urogenital tract. The virus can spread to different organs throughout the chicken. The virus is transmitted by aerosol and food contaminated by feces. Different vaccines against IBV exist and have helped to limit the spread of the virus and its variants. Infectious bronchitis virus is one of a number of strains of the species Avian coronavirus. Another strain of avian coronavirus is turkey coronavirus (TCV) which causes enteritis in turkeys.Coronaviruses also affect other branches of animal husbandry such as pig farming and the Cattle raising. Swine acute diarrhea syndrome coronavirus (SADS-CoV), which is related to bat coronavirus HKU2, causes diarrhea in pigs. Porcine epidemic diarrhea virus (PEDV) is a coronavirus that has recently emerged and similarly causes diarrhea in pigs. Transmissible gastroenteritis virus (TGEV), which is a member of the species Alphacoronavirus 1, is another coronavirus that causes diarrhea in young pigs. In the cattle industry bovine coronavirus (BCV), which is a member of the species Betacoronavirus 1 and related to HCoV-OC43, is responsible for severe profuse enteritis in young calves.\\n\\n\\n=== Domestic pets ===\\nCoronaviruses infect domestic pets such as cats, dogs, and ferrets. There are two forms of feline coronavirus which are both members of the species Alphacoronavirus 1. Feline enteric coronavirus is a pathogen of minor clinical significance, but spontaneous mutation of this virus can result in feline infectious peritonitis (FIP), a disease with high mortality. There are two different coronaviruses that infect dogs. Canine coronavirus (CCoV), which is a member of the species Alphacoronavirus 1, causes mild gastrointestinal disease. Canine respiratory coronavirus (CRCoV), which is a member of the species Betacoronavirus 1 and related to HCoV-OC43, cause respiratory disease. Similarly, there are two types of coronavirus that infect ferrets. Ferret enteric coronavirus causes a gastrointestinal syndrome known as epizootic catarrhal enteritis (ECE), and a more lethal systemic version of the virus (like FIP in cats) known as ferret systemic coronavirus (FSC).\\n\\n\\n=== Laboratory animals ===\\nCoronaviruses infect laboratory animals. Mouse hepatitis virus (MHV), which is a member of the species Murine coronavirus, causes an epidemic murine illness with high mortality, especially among colonies of laboratory mice.  Prior to the discovery of SARS-CoV, MHV was the best-studied coronavirus both in vivo and in vitro as well as at the molecular level.  Some strains of MHV cause a progressive demyelinating encephalitis in mice which has been used as a murine model for multiple sclerosis. Sialodacryoadenitis virus (SDAV), which is a strain of the species Murine coronavirus, is highly infectious coronavirus of laboratory rats, which can be transmitted between individuals by direct contact and indirectly by aerosol.  Rabbit enteric coronavirus causes acute gastrointestinal disease and diarrhea in young European rabbits.  Mortality rates are high.\\n\\n\\n== Prevention and treatment ==\\nA number of vaccines using different methods have been developed against human coronavirus SARS-CoV-2. Antiviral targets against human coronaviruses have also been identified such as viral proteases, polymerases, and entry proteins. Drugs are in development which target these proteins and the different steps of viral replication.Vaccines are available for animal coronaviruses IBV, TGEV, and Canine CoV, although their effectiveness is limited. In the case of outbreaks of highly contagious animal coronaviruses, such as PEDV, measures such as destruction of entire herds of pigs may be used to prevent transmission to other herds.\\n\\n\\n== See also ==\\nCoronavirus diseases\\nZoonosis\\n\\n\\n== References ==\\n\\n\\n== Further reading ==',\n",
              " 'Severe acute respiratory syndrome coronavirus 2 (SARS‑CoV‑2) is a strain of coronavirus that causes COVID-19 (coronavirus disease 2019), the respiratory illness responsible for the ongoing COVID-19 pandemic. The virus previously had a provisional name, 2019 novel coronavirus (2019-nCoV), and has also been called human coronavirus 2019 (HCoV-19 or hCoV-19).  First identified in the city of Wuhan, Hubei, China, the World Health Organization declared the outbreak a Public Health Emergency of International Concern on 30 January 2020, and a pandemic on 11 March 2020. SARS‑CoV‑2 is a positive-sense single-stranded RNA virus that is contagious in humans.SARS‑CoV‑2 is a virus of the species severe acute respiratory syndrome–related coronavirus (SARSr-CoV), related to the SARS-CoV-1 virus that caused the 2002–2004 SARS outbreak. It is of zoonotic origins and has close genetic similarity to bat coronaviruses, suggesting it emerged from a bat-borne virus. Research is ongoing as to whether SARS‑CoV‑2 came directly from bats or indirectly through any intermediate hosts. The virus shows little genetic diversity, indicating that the spillover event introducing SARS‑CoV‑2 to humans is likely to have occurred in late 2019.Epidemiological studies estimate that, in the December 2019 – September 2020 period, each infection resulted in an average of 2.4 to 3.4 new ones when no members of the community are immune and no preventive measures are taken. However, some subsequent variants have become more infectious. The virus primarily spreads between people through close contact and via aerosols and respiratory droplets that are exhaled when talking, breathing, or otherwise exhaling, as well as those produced from coughs or sneezes. It enters human cells by binding to angiotensin-converting enzyme 2 (ACE2), a membrane protein that regulates the renin–angiotensin system.\\n\\n\\n== Terminology ==\\nDuring the initial outbreak in Wuhan, China, various names were used for the virus; some names used by different sources included \"the coronavirus\" or \"Wuhan coronavirus\". In January 2020, the World Health Organization (WHO) recommended \"2019 novel coronavirus\" (2019-nCov) as the provisional name for the virus. This was in accordance with WHO\\'s 2015 guidance against using geographical locations, animal species, or groups of people in disease and virus names.On 11 February 2020, the International Committee on Taxonomy of Viruses adopted the official name \"severe acute respiratory syndrome coronavirus 2\" (SARS‑CoV‑2). To avoid confusion with the disease SARS, the WHO sometimes refers to SARS‑CoV‑2 as \"the COVID-19 virus\" in public health communications and the name HCoV-19 was included in some research articles. Referring to COVID-19 as the \"Wuhan virus\" has been described as dangerous by WHO officials, and as xenophobic by University of California at Berkeley Asian American studies lecturer Harvey Dong.\\n\\n\\n== Infection and transmission ==\\n\\nHuman-to-human transmission of SARS‑CoV‑2 was confirmed on 20 January 2020 during the COVID-19 pandemic. Transmission was initially assumed to occur primarily via respiratory droplets from coughs and sneezes within a range of about 1.8 metres (6 ft). Laser light scattering experiments suggest that speaking is an additional mode of transmission and a far-reaching one, indoors, with little air flow. Other studies have suggested that the virus may be airborne as well, with aerosols potentially being able to transmit the virus. During human-to-human transmission, between 200 and 800 infectious SARS‑CoV‑2 virions are thought to initiate a new infection. If confirmed, aerosol transmission has biosafety implications because a major concern associated with the risk of working with emerging viruses in the laboratory is the generation of aerosols from various laboratory activities which are not immediately recognizable and may affect other scientific personnel. Indirect contact via contaminated surfaces is another possible cause of infection. Preliminary research indicates that the virus may remain viable on plastic (polypropylene) and stainless steel (AISI 304) for up to three days, but it does not survive on cardboard for more than one day or on copper for more than four hours. The virus is inactivated by soap, which destabilizes its lipid bilayer. Viral RNA has also been found in stool samples and semen from infected individuals.The degree to which the virus is infectious during the incubation period is uncertain, but research has indicated that the pharynx reaches peak viral load approximately four days after infection or in the first week of symptoms and declines thereafter. The duration of SARS-CoV-2 RNA shedding is generally between 3 and 46 days after symptom onset.A study by a team of researchers from the University of North Carolina found that the nasal cavity is seemingly the dominant initial site of infection, with subsequent aspiration-mediated virus-seeding into the lungs in SARS‑CoV‑2 pathogenesis. They found that there was an infection gradient from high in proximal towards low in distal pulmonary epithelial cultures, with a focal infection in ciliated cells and type 2 pneumocytes in the airway and alveolar regions respectively.Studies have identified a range of animals—such as cats, ferrets, hamsters, non-human primates, minks, tree shrews, raccoon dogs, fruit bats, and rabbits—that are susceptible and permissive to SARS-CoV-2 infection. Some institutions have advised that those infected with SARS‑CoV‑2 restrict their contact with animals.\\n\\n\\n=== Asymptomatic and presymptomatic transmission ===\\nOn 1 February 2020, the World Health Organization (WHO) indicated that \"transmission from asymptomatic cases is likely not a major driver of transmission\". One meta-analysis found that 17% of infections are asymptomatic, and asymptomatic individuals were 42% less likely to transmit the virus.However, an epidemiological model of the beginning of the outbreak in China suggested that \"pre-symptomatic shedding may be typical among documented infections\" and that subclinical infections may have been the source of a majority of infections. That may explain how out of 217 on board a cruise liner that docked at Montevideo, only 24 of 128 who tested positive for viral RNA showed symptoms. Similarly, a study of ninety-four patients hospitalized in January and February 2020 estimated patients began shedding virus two to three days before symptoms appear and that \"a substantial proportion of transmission probably occurred before first symptoms in the index case\". The authors later published a correction that showed that shedding began earlier than first estimated, four to five days before symptoms appear.\\n\\n\\n=== Reinfection ===\\nThere is uncertainty about reinfection and long-term immunity. It is not known how common reinfection is, but reports have indicated that it is occurring with variable severity.The first reported case of reinfection was a 33-year-old man from Hong Kong who first tested positive on 26 March 2020, was discharged on 15 April 2020 after two negative tests, and tested positive again on 15 August 2020 (142 days later), which was confirmed by whole-genome sequencing showing that the viral genomes between the episodes belong to different clades. The findings had the implications that herd immunity may not eliminate the virus if reinfection is not an uncommon occurrence and that vaccines may not be able to provide lifelong protection against the virus.Another case study described a 25-year-old man from Nevada who tested positive for SARS‑CoV‑2 on 18 April 2020 and on 5 June 2020 (separated by two negative tests). Since genomic analyses showed significant genetic differences between the SARS‑CoV‑2 variant sampled on those two dates, the case study authors determined this was a reinfection. The man\\'s second infection was symptomatically more severe than the first infection, but the mechanisms that could account for this are not known.\\n\\n\\n== Reservoir and origin ==\\n\\nThe first known infections from SARS‑CoV‑2 were discovered in Wuhan, China. The original source of viral transmission to humans remains unclear, as does whether the virus became pathogenic before or after the spillover event. Because many of the early infectees were workers at the Huanan Seafood Market, it has been suggested that the virus might have originated from the market. However, other research indicates that visitors may have introduced the virus to the market, which then facilitated rapid expansion of the infections. A March 2021 WHO-convened report stated that human spillover via an intermediate animal host was the most likely explanation, with direct spillover from bats next most likely. Introduction through the food supply chain and the Huanan Seafood Market was considered another possible, but less likely, explanation. An analysis in November 2021, however, said that the earliest-known case had been misidentified and that the preponderance of early cases linked to the Huanan Market argued for it being the source.For a virus recently acquired through a cross-species transmission, rapid evolution is expected. The mutation rate estimated from early cases of SARS-CoV-2 was of 6.54×10−4 per site per year. Coronaviruses in general have high genetic plasticity, but SARS-CoV-2\\'s viral evolution is slowed by the RNA proofreading capability of its replication machinery. For comparison, the viral mutation rate in vivo of SARS-CoV-2 has been found to be lower than that of influenza.Research into the natural reservoir of the virus that caused the 2002–2004 SARS outbreak has resulted in the discovery of many SARS-like bat coronaviruses, most originating in horseshoe bats. The closest match by far, published on Nature (journal) in February 2022, were viruses BANAL-52 (96.8% resemblance to SARS‑CoV‑2), BANAL-103 and BANAL-236, collected in three different species of bats in Feuang, Laos. An earlier source published in February 2020 identified the virus RaTG13, collected in bats in Mojiang, Yunnan, China to be the closest to SARS‑CoV‑2, with 96.1% resemblance. None of the above are its direct ancestor.\\n\\nBats are considered the most likely natural reservoir of SARS‑CoV‑2. Differences between the bat coronavirus and SARS‑CoV‑2 suggest that humans may have been infected via an intermediate host; although the source of introduction into humans remains unknown.Although the role of pangolins as an intermediate host was initially posited (a study published in July 2020 suggested that pangolins are an intermediate host of SARS‑CoV‑2-like coronaviruses), subsequent studies have not substantiated their contribution to the spillover. Evidence against this hypothesis includes the fact that pangolin virus samples are too distant to SARS-CoV-2: isolates obtained from pangolins seized in Guangdong were only 92% identical in sequence to the SARS‑CoV‑2 genome (matches above 90 percent may sound high, but in genomic terms it is a wide evolutionary gap). In addition, despite similarities in a few critical amino acids, pangolin virus samples exhibit poor binding to the human ACE2 receptor.\\n\\n\\n== Phylogenetics and taxonomy ==\\nSARS‑CoV‑2 belongs to the broad family of viruses known as coronaviruses. It is a positive-sense single-stranded RNA (+ssRNA) virus, with a single linear RNA segment. Coronaviruses infect humans, other mammals, including livestock and companion animals, and avian species. Human coronaviruses are capable of causing illnesses ranging from the common cold to more severe diseases such as Middle East respiratory syndrome (MERS, fatality rate ~34%). SARS-CoV-2 is the seventh known coronavirus to infect people, after 229E, NL63, OC43, HKU1, MERS-CoV, and the original SARS-CoV.Like the SARS-related coronavirus implicated in the 2003 SARS outbreak, SARS‑CoV‑2 is a member of the subgenus Sarbecovirus (beta-CoV lineage B). Coronaviruses undergo frequent recombination. The mechanism of recombination in unsegmented RNA viruses such as SARS-CoV-2 is generally by copy-choice replication, in which gene material switches from one RNA template molecule to another during replication. SARS-CoV-2 RNA sequence is approximately 30,000 bases in length, relatively long for a coronavirus (which in turn carry the largest genomes among all RNA families) Its genome consists nearly entirely of protein-coding sequences, a trait shared with other coronaviruses.\\n\\nA distinguishing feature of SARS‑CoV‑2 is its incorporation of a polybasic site cleaved by furin, which appears to be an important element enhancing its virulence. It was suggested that the acquisition of the furin-cleavage site in the SARS-CoV-2 S protein was essential for zoonotic transfer to humans. The furin protease recognizes the canonical peptide sequence RX[R/K] R↓X where the cleavage site is indicated by a down arrow and X is any amino acid. In SARS-CoV-2 the recognition site is formed by the incorporated 12 codon nucleotide sequence CCT CGG CGG GCA which corresponds to the amino acid sequence P RR A. This sequence is upstream of an arginine and serine which forms the S1/S2 cleavage site (P RR A R↓S) of the spike protein. Although such sites are a common naturally-occurring feature of other viruses within the Subfamily Orthocoronavirinae, it appears in few other viruses from the Beta-CoV genus, and it is unique among members of its subgenus for such a site. The furin cleavage site PRRAR↓ is identical to that of the feline coronavirus, an alphacoronavirus 1 strain.Viral genetic sequence data can provide critical information about whether viruses separated by time and space are likely to be epidemiologically linked. With a sufficient number of sequenced genomes, it is possible to reconstruct a phylogenetic tree of the mutation history of a family of viruses. By 12 January 2020, five genomes of SARS‑CoV‑2 had been isolated from Wuhan and reported by the Chinese Center for Disease Control and Prevention (CCDC) and other institutions; the number of genomes increased to 42 by 30 January 2020. A phylogenetic analysis of those samples showed they were \"highly related with at most seven mutations relative to a common ancestor\", implying that the first human infection occurred in November or December 2019. Examination of the topology of the phylogenetic tree at the start of the pandemic also found high similarities between human isolates.  As of 21 August 2021, 3,422 SARS‑CoV‑2 genomes, belonging to 19 strains, sampled on all continents except Antarctica were publicly available.On 11 February 2020, the International Committee on Taxonomy of Viruses announced that according to existing rules that compute hierarchical relationships among coronaviruses based on five conserved sequences of nucleic acids, the differences between what was then called 2019-nCoV and the virus from the 2003 SARS outbreak were insufficient to make them separate viral species. Therefore, they identified 2019-nCoV as a virus of Severe acute respiratory syndrome–related coronavirus.In July 2020, scientists reported that a more infectious SARS‑CoV‑2 variant with spike protein variant G614 has replaced D614 as the dominant form in the pandemic.Coronavirus genomes and subgenomes encode six open reading frames (ORFs). In October 2020, researchers discovered a possible overlapping gene named ORF3d, in the SARS‑CoV‑2 genome. It is unknown if the protein produced by ORF3d has any function, but it provokes a strong immune response. ORF3d has been identified before, in a variant of coronavirus that infects pangolins.\\n\\n\\n=== Phylogenetic tree ===\\nA phylogenetic tree based on whole-genome sequences of SARS-CoV-2 and related coronaviruses is:\\n\\n\\n==== Variants ====\\n\\nThere are many thousands of variants of SARS-CoV-2, which can be grouped into the much larger clades. Several different clade nomenclatures have been proposed. Nextstrain divides the variants into five clades (19A, 19B, 20A, 20B, and 20C), while GISAID divides them into seven (L, O, V, S, G, GH, and GR).Several notable variants of SARS-CoV-2 emerged in late 2020. The World Health Organization has currently declared five variants of concern, which are as follows:\\nAlpha: Lineage B.1.1.7 emerged in the United Kingdom in September 2020, with evidence of increased transmissibility and virulence. Notable mutations include N501Y and P681H.\\nAn E484K mutation in some lineage B.1.1.7 virions has been noted and is also tracked by various public health agencies.\\nBeta: Lineage B.1.351 emerged in South Africa in May 2020, with evidence of increased transmissibility and changes to antigenicity, with some public health officials raising alarms about its impact on the efficacy of some vaccines. Notable mutations include K417N, E484K and N501Y.\\nGamma: Lineage P.1 emerged in Brazil in November 2020, also with evidence of increased transmissibility and virulence, alongside changes to antigenicity. Similar concerns about vaccine efficacy have been raised. Notable mutations also include K417N, E484K and N501Y.\\nDelta: Lineage B.1.617.2 emerged in India in October 2020. There is also evidence of increased transmissibility and changes to antigenicity.\\nOmicron: Lineage B.1.1.529 emerged in Botswana in November 2021.Other notable variants include 6 other WHO-designated variants under investigation and Cluster 5, which emerged among mink in Denmark and resulted in a mink euthanasia campaign rendering it virtually extinct.\\n\\n\\n== Virology ==\\n\\n\\n=== Structure ===\\n\\nEach SARS-CoV-2 virion is 60–140 nanometres (2.4×10−6–5.5×10−6 in) in diameter; its mass within the global human populace has been estimated as being between 0.1 and 1.0 kg. Like other coronaviruses, SARS-CoV-2 has four structural proteins, known as the S (spike), E (envelope), M (membrane), and N (nucleocapsid) proteins; the N protein holds the RNA genome, and the S, E, and M proteins together create the viral envelope. Coronavirus S proteins are glycoproteins and also type I membrane proteins (membranes containing a single transmembrane domain oriented on the extracellular side). They are divided into two functional parts (S1 and S2). In SARS-CoV-2, the spike protein, which has been imaged at the atomic level using cryogenic electron microscopy, is the protein responsible for allowing the virus to attach to and fuse with the membrane of a host cell; specifically, its S1 subunit catalyzes attachment, the S2 subunit fusion.\\n\\n\\n=== Genome ===\\nSARS-CoV-2 has a linear, positive-sense, single-stranded RNA genome about 30,000 bases long. Its genome has a bias against cytosine (C) and guanine (G) nucleotides, like other coronaviruses. The genome has the highest composition of U (32.2%), followed by A (29.9%), and a similar composition of G (19.6%) and C (18.3%). The nucleotide bias arises from the mutation of guanines and cytosines to adenosines and uracils, respectively. The mutation of CG dinucleotides is thought to arise to avoid the zinc finger antiviral protein related defense mechanism of cells, and to lower the energy to unbind the genome during replication and translation (adenosine and uracil base pair via two hydrogen bonds, cytosine and guanine via three). The depletion of CG dinucleotides in its genome has led the virus to have a noticeable codon usage bias. For instance, arginine\\'s six different codons have a relative synonymous codon usage of AGA (2.67), CGU (1.46), AGG (.81), CGC (.58), CGA (.29), and CGG (.19). A similar codon usage bias trend is seen in other SARS–related coronaviruses.\\n\\n\\n=== Replication cycle ===\\nVirus infections start when viral particles bind to host surface cellular receptors. Protein modeling experiments on the spike protein of the virus soon suggested that SARS‑CoV‑2 has sufficient affinity to the receptor angiotensin converting enzyme 2 (ACE2) on human cells to use them as a mechanism of cell entry. By 22 January 2020, a group in China working with the full virus genome and a group in the United States using reverse genetics methods independently and experimentally demonstrated that ACE2 could act as the receptor for SARS‑CoV‑2. Studies have shown that SARS‑CoV‑2 has a higher affinity to human ACE2 than the original SARS virus. SARS‑CoV‑2 may also use basigin to assist in cell entry.Initial spike protein priming by transmembrane protease, serine 2 (TMPRSS2) is essential for entry of SARS‑CoV‑2. The host protein neuropilin 1 (NRP1) may aid the virus in host cell entry using ACE2. After a SARS‑CoV‑2 virion attaches to a target cell, the cell\\'s TMPRSS2 cuts open the spike protein of the virus, exposing a fusion peptide in the S2 subunit, and the host receptor ACE2. After fusion, an endosome forms around the virion, separating it from the rest of the host cell. The virion escapes when the pH of the endosome drops or when cathepsin, a host cysteine protease, cleaves it. The virion then releases RNA into the cell and forces the cell to produce and disseminate copies of the virus, which infect more cells.SARS‑CoV‑2 produces at least three virulence factors that promote shedding of new virions from host cells and inhibit immune response. Whether they include downregulation of ACE2, as seen in similar coronaviruses, remains under investigation (as of May 2020).\\n\\n\\n== Treatment and drug development ==\\nVery few drugs are known to effectively inhibit SARS‑CoV‑2. Masitinib is a clinically safe drug and was recently found to inhibit its main protease, 3CLpro and showed >200-fold reduction in viral titers in the lungs and nose in mice. However, it is not approved for the treatment of COVID-19 in humans as of August 2021.COVID Moonshot is an international collaborative open-science project started in March 2020 with the goal of developing an un-patented oral antiviral drug for treatment of SARS-CoV-2.\\n\\n\\n== Epidemiology ==\\n\\nRetrospective tests collected within the Chinese surveillance system revealed no clear indication of substantial unrecognized circulation of SARS‑CoV‑2 in Wuhan during the latter part of 2019.A meta-analysis from November 2020 estimated the basic reproduction number (\\n  \\n    \\n      \\n        \\n          R\\n          \\n            0\\n          \\n        \\n      \\n    \\n    {\\\\displaystyle R_{0}}\\n  ) of the virus to be between 2.39 and 3.44. This means each infection from the virus is expected to result in 2.39 to 3.44 new infections when no members of the community are immune and no preventive measures are taken. The reproduction number may be higher in densely populated conditions such as those found on cruise ships. Human behavior affects the R0 value and hence estimates of R0 differ between different countries, cultures, and social norms. For instance, one study found relatively low R0 (~3.5) in Sweden, Belgium and the Netherlands, while Spain and the US had significantly higher R0 values (5.9 to 6.4, respectively).\\nThere have been about 96,000 confirmed cases of infection in mainland China. While the proportion of infections that result in confirmed cases or progress to diagnosable disease remains unclear, one mathematical model estimated that 75,815 people were infected on 25 January 2020 in Wuhan alone, at a time when the number of confirmed cases worldwide was only 2,015. Before 24 February 2020, over 95% of all deaths from COVID-19 worldwide had occurred in Hubei province, where Wuhan is located. As of 12 April 2022, the percentage had decreased to 0.052%.As of 12 April 2022, there have been 500,280,791 total confirmed cases of SARS‑CoV‑2 infection in the ongoing pandemic. The total number of deaths attributed to the virus is 6,183,197.\\n\\n\\n== See also ==\\n3C-like protease (NS5)\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\n\\n\\n== External links ==\\n\"Coronavirus Disease 2019 (COVID-19)\". Centers for Disease Control and Prevention (CDC). 11 February 2020.\\n\"Coronavirus disease (COVID-19) Pandemic\". World Health Organization (WHO).\\n\"SARS-CoV-2 (Severe acute respiratory syndrome coronavirus 2) Sequences\". National Center for Biotechnology Information (NCBI).\\n\"COVID-19 Resource Centre\". The Lancet.\\n\"Coronavirus (Covid-19)\". The New England Journal of Medicine.\\n\"Covid-19: Novel Coronavirus Outbreak\". Wiley.\\n\"SARS-CoV-2\". Virus Pathogen Database and Analysis Resource.\\n\"SARS-CoV-2 related protein structures\". Protein Data Bank.',\n",
              " 'The Omicron variant (B.1.1.529) is a variant of SARS-CoV-2 (the virus that causes COVID-19) that was first reported to the World Health Organization (WHO) from South Africa on 24 November 2021.Omicron multiplies around 70 times faster than the Delta variant in the bronchi (lung airways) but evidence suggests it is less severe than previous strains, especially compared to the Delta variant. Omicron might be less able to penetrate deep lung tissue. Omicron infections are 91 percent less fatal than the delta variant, with 51 percent less risk of hospitalization. However, the estimated difference in the intrinsic risk of hospitalization largely decreases to 0–30 percent when reinfections are excluded. Overall, the extremely high rate of spread, combined with its ability to evade both double vaccination and the body\\'s immune system, means the total number of patients requiring hospital care at any given time is still of great concern.Vaccines continue to provide protection against severe disease and hospitalisation especially after a third dose of an mRNA vaccine is given. Early figures suggest that double vaccination offers 30 to 40 percent protection against infection and around 70 percent protection against hospitalization. A recent third vaccine dose boosts effectiveness against infection to around 75 percent, and 88 percent for severe disease.\\n\\n\\n== Classification ==\\n\\n\\n=== Nomenclature ===\\nOn 26 November, the WHO\\'s Technical Advisory Group on SARS-CoV-2 Virus Evolution declared PANGO lineage B.1.1.529 a variant of concern and designated it with the Greek letter omicron. Greek letters are used to identify variants of SARS-CoV-2. The WHO skipped the preceding letters nu and xi in the Greek alphabet to avoid confusion with the similarities of the English word \"new\" and the Chinese surname Xi. The previous designation was for the \"variant of interest\" Mu.Possibly due to a lack of familiarity with the Greek alphabet among some English speakers and the relative frequency of the Latin prefix \"omni\" in other common speech, the name of the variant has also occasionally been mispronounced and misspelled as \"Omnicron\".The GISAID project has assigned it the clade identifier GR/484A, and the Nextstrain project has assigned it the clade identifiers 21K and 21L, both belonging to a larger Omicron group 21M.\\n\\n\\n=== Mutations ===\\n\\nThe variant has many mutations, some of which have concerned scientists. The Omicron variant has a total of 60 mutations compared to the reference / ancestral variant: 50 nonsynonymous mutations, 8 synonymous mutations, and 2 non-coding mutations. Thirty-two mutations affect the spike protein, the main antigenic target of antibodies generated by infections and of many vaccines widely administered. Many of those mutations had not been observed in other strains.\\nThe variant is characterised by 30 amino acid changes, three small deletions, and one small insertion in the spike protein compared with the original virus, of which 15 are located in the receptor-binding domain (residues 319–541). It also carries a number of changes and deletions in other genomic regions. Additionally, the variant has three mutations at the furin cleavage site. The furin cleavage site increases SARS-CoV-2 infectivity. The mutations by genomic region are the following:\\n\\nSpike protein: A67V, Δ69-70, T95I, G142D, Δ143-145, Δ211, L212I, ins214EPE, G339D, S371L, S373P, S375F, K417N, N440K, G446S, S477N, T478K, E484A, Q493R, G496S, Q498R, N501Y, Y505H, T547K, D614G, H655Y, N679K, P681H, N764K, D796Y, N856K, Q954H, N969K, L981F\\nHalf (15) of these 30 changes are located in the receptor binding domain-RBD (residues 319–541)\\nORF1ab\\nnsp3: K38R, V1069I, Δ1265, L1266I, A1892T\\nnsp4: T492I\\nnsp5: P132H\\nnsp6: Δ105-107, A189V\\nnsp12: P323L\\nnsp14: I42V\\nEnvelope protein: T9I\\nMembrane protein: D3G, Q19E, A63T\\nNucleocapsid protein: P13L, Δ31-33, R203K, G204RA link with HIV infection may explain a large number of mutations in the sequence of the Omicron variant. Indeed, in order to be affected by such a high number of mutations, the virus must have been able to evolve a long time without killing its host, nor being eliminated. One such situation occurs in people with a weakened immune system but receiving enough medical care to survive. This is the case in HIV patients in South Africa, who represent more than 20% of the population. Due to lack of access to clinics, fear of stigmatisation and disrupted healthcare, millions living with HIV in the region are not on effective HIV therapy. HIV prevention could be key to reducing the risk of uncontrolled HIV driving the emergence of Covid variants.In addition, it is believed that one of these many mutations, comprising a 9-nucleotide sequence, may have been acquired from another coronavirus (known as HCoV-229E), responsible for the common cold.  This is not entirely unexpected\\u200a—\\u200aat times, viruses within the body acquire and swap segments of genetic material from each other, and this is one common means of mutation.One hypothesis to explain the novel mutations is that SARS-CoV-2 was transmitted from humans to mice and mutated in a population of mice sometime between mid-2020 and late 2021 before reinfecting humans.\\n\\n\\n==== Sublineages and BA.2 subvariant ====\\nResearchers have established the existence of three sublineages of Omicron. The \\'standard\\' sublineage is now referred to as BA.1 (or B.1.1.529.1), and the two other sublineages are known as BA.2 (or B.1.1.529.2) and BA.3 (or B.1.1.529.3). They share many mutations, but also significantly differ. In general, BA.1 and BA.2 share 32 mutations, but differ by 28. This makes them as different as some other major variants, and it has been suggested that BA.2 should receive its own Greek-letter name. BA.1 has itself been divided in two, the original BA.1 and BA.1.1, where the main difference is that the latter has a R346K mutation.A laboratory study on hamsters and mice in Japan published as a non-peer-reviewed preprint in mid-February 2022 suggested that BA.2, is not only more transmissible than BA.1, but may cause more severe disease. Therapeutic monoclonal antibodies used to treat people infected with COVID did not have much effect on BA.2, which was \"almost completely resistant\" to casirivimab and imdevimab, and 35 times more resistant to sotrovimab than the original B.1.1 virus. The researchers proposed that BA.2 should be recognised as a unique variant of concern, which would be given its own Greek letter.\\n\\n\\n===== Detection =====\\nOrdinary COVID-19 tests, both PCR and rapid, can detect all Omicron subvariants as COVID-19, but further tests are necessary to distinguish the subvariants from each other and from other COVID-19 variants.A notable difference between the \\'standard\\' Omicron subvariant and BA.2 is that the latter lacks the characteristic S-gene target failure (SGTF)-causing deletion (Δ69-70) by which many qPCR tests are able to rapidly detect a case as an Omicron (or Alpha) variant, from the previously dominant  Delta variant. Thus, countries that primarily rely on SGTF for detection may overlook BA.2, and British authorities consider SGTF alone as insufficient for monitoring the spread of Omicron. This has resulted in it having been nicknamed \\'Stealth Omicron\\'. Because BA.2 still can be separated from other variants through normal full sequencing, or checks of certain other mutations, the nickname is however inaccurate. Some countries, such as Denmark and Japan, use a variant qPCR that tests for several mutations, including L452R. It can also distinguish Delta, which has L452R, and all Omicron sublineages, which do not have L452R. As Omicron became dominant and the Delta variant became rare, the SGTF mutation that had made Delta and BA.2 similar in qPCR tests could now be used for easily separating BA.1 and BA.2 from each other. As a consequence, BA.2 could now be regarded as decidedly un-stealthy.The third Omicron sublineage, BA.3, is very rare. It has the same SGTF deletion (Δ69-70) as BA.1.In April 2022 the WHO announced it was tracking BA.4 and BA.5 subvariants with BA.4 having been detected in South Africa, Botswana, Denmark, Scotland and England.Early indications from data collected in South Africa suggested BA.4 and BA.5 have a significant growth advantage over BA.2.\\n\\n\\n===== Affected countries and transmissibility =====\\nAccording to early research, BA.2 is roughly 30% to 60% more transmissible than BA.1. As a consequence, it may prolong a COVID-19 wave when it overtakes BA.1, although it is difficult to assess what part is caused by the higher transmissibility of BA.2 in countries that simultaneously reduce restrictions (allowing easier transmission than in earlier periods with more restrictions). A new BA.1–BA.2 recombinant isolated from the UK in January 2022, dubbed the \"XE\" recombinant, was found by the WHO to be potentially 10% more transmissible than BA.2, making it about 43% to 76% to more transmissible than BA.1, and making the XE recombinant the most contagious variant identified. On 7 April 2022 Brazilian authorities announced the first detected case of a person infected with Omicron XE.The first known sequence of BA.2 was in a sample from 15 November 2021. In mid-December 2021, BA.2 still appeared to be rare with relatively few sequences from half a dozen countries having been uploaded to GISAID, but subsequently numbers rapidly increased. As of 17 January 2022, BA.2 had been detected in at least 40 countries and in all continents except Antarctica. By 31 January, it had been detected in at least 57 countries. In global samples collected from 4 February to 5 March and uploaded to GISAID, BA.2 accounted for c. 34%, compared to 41% for BA.1.1, 25% for BA.1 and less than 1% for BA.3. In a review two weeks later, covering 16 February to 17 March, BA.2 had become the most frequent. However, the data is geographically skewed due to sequencing rate and speed; for example, among the c. 205,000 COVID-19 sequences from March that had been uploaded to GISAID as of 22 March, United Kingdom and Denmark accounted for more than 3⁄4, and most of the remaining were from other European countries, Australia, Canada and the United States (altogether, c. 6,000 were from Africa, Asia and Latin America). Based on GISAID uploads, BA.1 peaked in early January 2022, after which it was overtaken by both BA.1.1 and BA.2. In North America, parts of Europe and parts of Asia, BA.1 was first outcompeted by BA.1.1. For example, in the United States, France and Japan, BA.1.1 became the dominant subvariant in January 2022.By late December 2021/early January 2022, BA.2 appears to have become dominant in at least parts of India (already making up almost 80 percent in Kolkata in late December 2021) and the Philippines, had become frequent in Scandinavia, South Africa and Singapore, and was showing signs of growth in Germany and the United Kingdom. In Japan, which has quarantine and detailed screening of all international travellers, as of 24 January, the vast majority of BA.2 had been detected in people that had arrived from India or the Philippines with cases going back at least to 1 December 2021 (far fewer BA.1 or other variants were detected among arrivals from the two countries in that period), but small numbers had also been detected in people arriving from other countries.In Denmark, the first BA.2 was in a sample collected on 5 December 2021 and extremely few were found in the directly following period. By week fifty (13–19 December) it had started to increase, with BA.2 being at around 2 percent of sequenced cases compared to 46 percent BA.1 (remaining Delta). The frequency of both Omicron subvariants continued to increase throughout the last half of December; in week fifty-two (27 December–2 January), BA.2 had reached 20 percent and BA.1 peaked at 72 percent. In January 2022, BA.1 began decreasing, whereas BA.2 continued its increase. By the second week (10–16 January) of 2022, the frequency of the two was almost equal, both being near 50 percent (around one percent was the rapidly disappearing Delta). In the following week, BA.2 became clearly dominant in Denmark with 65 percent of new cases being the BA.2 subvariant. Trends from the other Scandinavian countries, India, South Africa and the United Kingdom also showed that BA.2 was increasing in proportion to the original BA.1. In early February 2022, it had become the dominant subvariant in South Africa, in late February it had become dominant in Germany and in early March it had become dominant in the United Kingdom. In early March, BA.1.1 was still heavily dominant in the United States (having overtaken BA.1 in January), but BA.2 was increasing in frequency, later becoming dominant in the US by March 29.\\n\\n\\n===== Severity and immunity =====\\nThe risk of hospitalization is the same in BA.1 and BA.2 based on reviews from Denmark, India, South Africa and the United Kingdom. Norwegian studies show that the amount of virus in the upper airways is similar in those infected with BA.1 and BA.2. In general, Omicron (all subvariants) have a higher reinfection rate than other COVID-19 variants. Studies from Denmark and Qatar found that after an infection with BA.1, the vast majority of people were well-protected against a BA.2 infection, although it is unknown how long this protection lasts. Laboratory studies also show that antibodies for BA.1 generally protect against BA.2. In Denmark, preliminary data found breakthrough rates in people that had been vaccinated that were similar to the breakthrough rates seen for BA.1. An initial study by the UK Health Security Agency found that vaccines afford similar levels of protection against symptomatic disease by BA.1 and BA.2, and in both it is considerably higher after two doses and a booster than two doses without booster. Because of the gradually waning effect of vaccines, further, booster vaccination may later be necessary.\\n\\n\\n=== Possible consequences ===\\nThe WHO is concerned that the large number of mutations in Omicron compared to earlier variants may reduce immunity in people who were previously infected and in vaccinated people. It is also possible the Omicron variant might be more infective in this regard than prior variants. The effects of the mutations, if any, are unknown as of late November 2021. The WHO warns that health services could be overwhelmed especially in nations with low vaccination rates where mortality and morbidity rates are likely to be much higher, and urges all nations to increase COVID-19 vaccinations.Professor Paul Morgan, immunologist at Cardiff University, also recommends vaccination. Morgan said, \"I think a blunting rather than a complete loss [of immunity] is the most likely outcome. The virus can\\'t possibly lose every single epitope on its surface, because if it did that spike protein couldn\\'t work any more. So, while some of the antibodies and T cell clones made against earlier versions of the virus, or against the vaccines may not be effective, there will be others, which will remain effective. (...) If half, or two-thirds, or whatever it is, of the immune response is not going to be effective, and you\\'re left with the residual half, then the more boosted that is the better.\"Professor Francois Balloux of the Genetics Institute at University College London said, \"From what we have learned so far, we can be fairly confident that – compared with other variants – Omicron tends to be better able to reinfect people who have been previously infected and received some protection against COVID-19.  That is pretty clear and was anticipated from the mutational changes we have pinpointed in its protein structure. These make it more difficult for antibodies to neutralise the virus.\"On 15 December 2021, the European Centre for Disease Prevention and Control assessed that, even if the variant turns out to be milder than Delta, its spread will very likely increase hospitalizations and fatalities due to the exponential growth in cases caused by increased transmissibility.On 23 December 2021, Nature indicates that, though Omicron likely weakens vaccine protection, reasonable effectiveness against Omicron may be maintained with currently available vaccination and boosting approaches.\\n\\n\\n=== In non-human animals ===\\nIn February 2022, the first confirmed case infecting a wild animal was confirmed by researchers at Pennsylvania State University in white-tailed deer in Staten Island, N.Y.\\n\\n\\n== Signs and symptoms ==\\n\\nA unique reported symptom of the Omicron variant is night sweats. Also, loss of taste and smell seem to be uncommon compared to other strains.A study performed between 1 and 7 December by the Center for Disease Control found that: \"The most commonly reported symptoms [were] cough, fatigue, and congestion or runny nose\" making it difficult to distinguish from a less damaging variant or other virus.Research published in London on 25 December 2021 suggested the most frequent symptoms stated by users of the Zoe Covid app were \"a running nose, headaches, fatigue, sneezing and sore throats.\"\\n\\n\\n== Characteristics ==\\nMany of the mutations to the spike protein are present in other variants of concern and are related to increased infectivity and antibody evasion. Computational modeling suggests that the variant may also escape cell-mediated immunity. On 26 November, the ECDC wrote that an evaluation of the neutralizing capacity of convalescent sera and of vaccines is urgently needed to assess possible immune escape, saying these data are expected within two to three weeks.Although transmission via fomites is rare, preliminary data indicate that the variant lasts for 194 hours on plastic surfaces and 21 hours on skin, compared with just 56 and 7 hours, respectively, for the ancestral Wuhan strain.\\n\\n\\n=== Contagiousness ===\\nIt was not known in November 2021 how the variant would spread in populations with high levels of immunity. It was also not known if the Omicron variant causes a milder or more severe COVID-19 infection. According to pharmaceutical companies, vaccines could be updated to combat the variant \"in around 100 days\" if necessary.Relating to naturally acquired immunity, Anne von Gottberg, an expert at the National Institute for Communicable Diseases, believed at the beginning of December 2021 that immunity granted by previous variants would not protect against Omicron.On 15 December 2021, Jenny Harries, head of the UK Health Security Agency, told a parliamentary committee that the doubling time of COVID-19 in most regions of the UK was now less than two days despite the country\\'s high vaccination rate. She said that the Omicron variant of COVID-19 is \"probably the most significant threat since the start of the pandemic\", and that the number of cases in the next few days would be \"quite staggering compared to the rate of growth that we\\'ve seen in cases for previous variants\".In January 2022, William Schaffner, professor of infectious diseases at Vanderbilt University Medical Center, compared the contagiousness of the Omicron variant to the contagiousness of the measles.\\n\\n\\n==== Immune evasion ====\\n\\nA study suggests that mutations that promote breakthrough infections or antibody-resistance \"like those in Omicron\" could be a new mechanism for viral evolution success of SARS-CoV-2 and that such may become a dominating mechanism of its evolution. A preprint supports such an explanation of Omicron\\'s spread, suggesting that it \"primarily can be ascribed to the immune evasiveness rather than an inherent increase in the basic transmissibility\". Studies showed the variant to escape the majority of existing SARS-CoV-2 neutralizing antibodies, including of sera from vaccinated and convalescent individuals. Nevertheless, current vaccines are expected to protect against severe illness, hospitalizations, and deaths due to Omicron and, on an individual level, the Omicron variant is milder than earlier variants that evolved when the antibody/vaccination share was lower than it was when Omicron emerged.\\n\\n\\n=== Virulence ===\\n\\nAs of 28 November 2021 the World Health Organization\\'s update states \"There is currently no information to suggest that symptoms associated with Omicron are different from ... other variants\". Increased rates of hospitalization in South Africa may be due to a higher number of cases, rather than any specific feature of the Omicron variant.On 4 December 2021, the South African Medical Research Council reported that from 14 to 29 November 2021 at a hospital complex in Tshwane, inpatients were younger than in previous waves and the ICU and oxygen therapy rates were lower than in earlier waves. These observations are not definitive and the clinical profile could change over the following two weeks, allowing for more accurate conclusions about disease severity. Excess deaths nearly doubled in the week of 28 November, suggesting under-reporting, but the level was still much lower than that seen in the second wave in mid-January 2021.\\nOn 12 December, director-general of the World Health Organization Tedros Adhanom asserted that it was wrong for people to consider Omicron as mild. This is because high exposure to previous infections in South Africa likely affects the clinical course of the new infections.On 20 December, a report by the Imperial College COVID-19 Response Team, based on data from England, found that hospitalisation and asymptomatic infection indicators were not significantly associated with Omicron infection, suggesting at most limited changes in severity compared with Delta. On 22 December, the team reported an approximately 41% (95% CI, 37–45%) lower risk of a hospitalization requiring a stay of at least 1 night compared to the Delta variant, and that the data suggest that recipients of 2 doses of the Pfizer–BioNTech, the Moderna or the Oxford–AstraZeneca vaccine remain substantially protected from hospitalization.In January, the CDC confirmed that the variant causes less severe disease than previously dominant variants. The novel Omicron subtype \\'BA.2\\' did not initially show an increase over this lower virulence, albeit one preprint found that \"infection experiments using hamsters show that BA.2 is more pathogenic than BA.1\". In March, a study from Hong Kong indicated that BA.2 may be more severe for children, compared to other variants. Nevertheless, in the U.S., the daily new COVID-19 deaths were higher during Omicron dominance than during Delta\\'s during fall and high volumes of hospitalizations due to the variant\\'s other characteristics can cause indirect harm via local health care system strains, beyond less severe but often non-mild disease effects.\\n\\n\\n== Diagnosis ==\\n\\n\\n=== PCR testing ===\\nThe FDA has published guidelines on how PCR tests will be affected by Omicron. Tests that detect multiple gene targets will continue to identify the testee as positive for COVID-19. S-gene dropout or target failure has been proposed as a shorthand way of differentiating Omicron from Delta. The variant can also be identified by sequencing and genotyping.\\n\\n\\n=== Rapid antigen testing ===\\nIn January 2022 the medicine and therapeutic regulatory agency Therapeutic Goods Administration (TGA) of the Australian Government found that only one of their 23 approved COVID-19 rapid antigen tests (RAT) stated that it detected Omicron.\\n\\n\\n== Prevention ==\\n\\nAs with other variants, the WHO recommended that people continue to keep enclosed spaces well ventilated, avoid crowding and close contact, wear well-fitting masks, clean hands frequently, and get vaccinated.\\nResponse by vaccine producersOn 26 November 2021, BioNTech said it would know in two weeks whether the current vaccine is effective against the variant and that an updated vaccine could be shipped in 100 days if necessary. AstraZeneca, Moderna and Johnson & Johnson were also studying the variant\\'s impact on the effectiveness of their vaccines. On the same day, Novavax stated that it was developing an updated vaccine requiring two doses for the Omicron variant, which the company expected to be ready for testing and manufacturing within a few weeks. On 29 November 2021, The Gamaleya Institute said that Sputnik Light should be effective against the variant, that it would begin adapting Sputnik V, and that a modified version could be ready for mass production in 45 days. Sinovac said it could quickly mass-produce an inactivated vaccine against the variant and that it was monitoring studies and collecting samples of the variant to determine if a new vaccine is needed. On 7 December 2021, at a symposium in Brazil with its partner Instituto Butantan, Sinovac said it would update its vaccine to the new variant and make it available in three months. On December 2, the Finlay Institute was already developing a version of Soberana Plus against the variant. Pfizer hopes to have a vaccine targeted to immunize against Omicron ready by March 2022.\\nWHOOn 29 November 2021, the WHO said cases and infections are expected among those vaccinated, albeit in a small and predictable proportion.\\nVaccine efficacyIn December, studies, some of which using large nationwide datasets from either Israel and Denmark, found that vaccine effectiveness of multiple common two-dosed COVID-19 vaccines is substantially lower against the Omicron variant than for other common variants including the Delta variant, and that a new (often a third) dose – a booster dose – is needed and effective, with it i.a. substantially reducing deaths from the disease compared to cohorts who received no booster but two doses.On 7 December 2021, preliminary results from a laboratory test conducted at the Africa Health Research Institute in Durban with 12 people who received the Pfizer-BioNTech vaccine found a 41-fold reduction in neutralizing antibody activity against the variant in some of the samples. This is a big reduction, but it does not mean that the variant can escape vaccines completely, so vaccination with current vaccines is still recommended. Neutralizing antibody activity against the variant was greater in those fully vaccinated after being infected about a year earlier. Effectiveness estimates will likely change as more data is collected, as antibodies generated by vaccination vary widely between individuals and the sample was small. On 8 December 2021, Pfizer and BioNTech reported that preliminary data indicated that a third dose of the vaccine would provide a similar level of neutralizing antibodies against the variant as seen against other variants after two doses.On 10 December 2021, the UK Health Security Agency reported that early data indicated a 20- to 40-fold reduction in neutralizing activity for Omicron by sera from Pfizer 2-dose vaccinees relative to earlier strains and a 20-fold reduction relative to Delta. The reduction was greater in sera from AstraZeneca 2-dose vaccinees, falling below the detectable threshold. An mRNA booster dose produced a similar increase in neutralising activity regardless of the vaccine used for primary vaccination. After a booster dose (usually with an mRNA vaccine), vaccine effectiveness against symptomatic disease was at 70%–75%, and the effectiveness against severe disease was expected to be higher.In January 2022, results from Israel suggested that a 4th dose of vaccine is only partially effective against the Omicron strain. Many cases of infection broke through, albeit \"a bit less than in the control group\", even though trial participants had higher antibody levels after the 4th dose.\\nWHO recommendations for epidemiologyOn 26 November 2021, the WHO asked nations to do the following:\\n\\nEnhance surveillance and sequencing efforts to better understand circulating SARS-CoV-2 variants.\\nSubmit complete genome sequences and associated metadata to a publicly available database, such as GISAID.\\nReport initial cases/clusters associated with virus-of-concern infection to WHO through the IHR mechanism.\\nWhere capacity exists and in coordination with the international community, perform field investigations and laboratory assessments to improve understanding of the potential impacts of the virus of concern on COVID-19 epidemiology, severity, and the effectiveness of public health and social measures, diagnostic methods, immune responses, antibody neutralization, or other relevant characteristics.\\n\\n\\n== Treatment ==\\n\\nCorticosteroids such as dexamethasone and IL6 receptor blockers such as tocilizumab (Actemra) are known to be effective for managing patients with the earlier strains of severe COVID-19.  The impact on the effectiveness of other treatments was being assessed in 2021.Relating to monoclonal antibodies (mAbs) treatments, similar testing and research is ongoing. Preclinical data on in vitro pseudotyped virus data demonstrate that some mAbs designed to use highly conserved epitopes retain neutralizing activity against key mutations of Omicron substitutions.\\nSimilar results are confirmed by cryo-electron microscopy and X-ray data, also providing the structural approach and molecular basis for the evasion of humoral immunity exhibited by Omicron antigenic shift as well as the importance of targeting conserved epitopes for vaccine and therapeutics design. While 7 clinical mAbs or mAb cocktails experienced loss of neutralizing activity of 1-2 orders of magnitude or greater relative to the prototypic virus, the S309 mAb, the parent mAb of sotrovimab, neutralized Omicron with only 2-3-fold reduced potency.\\nFurther data suggest Omicron would cause significant humoral immune evasion, while neutralizing antibodies targeting the sarbecovirus conserved region remain most effective.\\nIndeed, most receptor-binding motif (RBM)-directed monoclonal antibodies lost in vitro neutralizing activity against Omicron, with only 3 out of 29 mAbs examined in another study retaining unaltered potency. Furthermore, a fraction of broadly neutralizing sarbecovirus mAbs neutralized Omicron through recognition of antigenic sites outside the RBM, including sotrovimab (VIR-7831), S2X259 and S2H97. In particular, sotrovimab is not fully active against the BA.2 Omicron sublineage, and in early 2022 the office of the U.S. ASPR stopped distributing the antibody treatment to states where BA.2 was dominant.\\n\\n\\n== Epidemiology ==\\nOn 26 November 2021, the South African National Institute for Communicable Diseases announced that 30,904 COVID-tests (in one day) detected 2,828 new COVID infections (a 9.2% positivity rate).\\nOne week later, on 3 December 2021, the NICD announced that 65,990 COVID tests had found 16,055 new infections (5.7 times as many as seven days before; positive rate 24.3%) and that 72 percent of them were found in Gauteng.\\nThis province of South Africa is densely populated at about 850 inhabitants per km2. Gauteng\\'s capital Johannesburg is a megacity (about 5.5 million inhabitants in the city itself plus 9.5 million in the urban region).\\nIn November 2021, the transmissibility of the Omicron variant, as compared to the Delta variant or other variants of the COVID-19 virus, was still uncertain. Omicron is frequently able to infect previously Covid-positive people.It has been estimated the Omicron variant diverged in late September or early October 2021, based on Omicron genome comparisons.\\nSequencing data suggests that Omicron had become the dominant variant in South Africa by November 2021, the same month where it had been first identified in the country.Phylogeny suggests a recent emergence. Data from South Africa suggests that Omicron has a pronounced growth advantage there. However, this may be due to transmissibility or immune escape related, or both.\" Also the serial interval plays a role in the growth.\\nDetectable changes in levels of COVID-19 in wastewater samples from South Africa\\'s Gauteng province were seen as early as 17–23 October (week 42). The National Institute for Communicable Diseases reports that children under the age of 2 make up 10% of total hospital admissions in the Omicron point of discovery Tshwane in South Africa. Data on the S gene target failure (SGTF) of sampled cases in South Africa indicates a growth of 21% per day relative to Delta, generating an increased reproduction number by a factor of 2.4. Omicron became the majority strain in South Africa around 10 November. Another analysis showed 32% growth per day in Gauteng, South Africa, having become dominant there around 6 November.In the UK, the logarithmic growth rate of Omicron-associated S gene target failure (SGTF) cases over S gene target positive (SGTP) cases was estimated at 0.37 per day, which is exceptionally high. Furthermore, by 14 December it appears to have become the most dominant strain. Without presuming behavior change in response to the variant, a million infections per day by December 24 are projected for a 2.5 days doubling time. In Denmark, the growth rate has been roughly similar with a doubling time of about 2–3 days, it having become the most prevalent strain on 17 December. Switzerland is not far behind. In Germany Omicron became the most prevalent variant on January 1. In Scotland, Omicron apparently became the most prevalent variant on 17 December. In the Canadian province of Ontario it became the most prevalent strain on 13 December. In the US, the variant appears to have become the most prevalent strain on December 18, growing at 0.24 per day. In Portugal, Omicron had reached 61.5% of cases on 22 December. In Belgium, the strain has become the most prevalent on 25 December, and in the Netherlands on 28 December. In Italy, it had reached 28% of cases on 20 December and was doubling every two days, while it became the dominant variant in Norway on 25 December. In France, it made up about 15% of COVID-19 cases in mid-December, but around 27 December it had increased to more than 60%. Researchers recommend sampling at least 5% of COVID-19 patient samples in order to detect Omicron or other emerging variants.During January 2022, in Denmark the BA.2 variant grew at ~0.10 per day (+11% per day) as a ratio to BA.1 (the legacy Omicron variant), and became the dominant strain in week 2, 2022. In the United Kingdom, the BA.2 variant was growing at ~0.11 per day (+12% per day) as a ratio to BA.1.On 13 January 2022, the BBC reported that the hospitalization rate was higher in the US and Canada than in Europe and South Africa. This was attributed to a combination of a greater number of elderly people than in South Africa, greater prevalence of comorbidities such as hypertension and obesity than in Europe, higher indoor transmission due to the winter, lower vaccination rate in the US than in Europe and Canada, and a possible still high prevalence of the Delta variant, which more often leads to hospitalization.\\n\\n\\n== Statistics ==\\n\\nThe chance of detecting a case particularly depends on a country\\'s sequencing rate. For example, South Africa sequences far more samples than any other country in Africa, but at a considerably lower rate than most Western nations. Furthermore, it can take up to two weeks to return a viral sequence in places with the technical capability, hence solid statistics on confirmed cases lag the actual situation. Denmark and Norway regard cases found by their variant qPCR test, which is relatively fast and checks several genes, as sufficient for counting it as an Omicron, also before full sequencing.\\n\\n\\n== History ==\\nOn 26 November 2021, WHO designated B.1.1.529 as a variant of concern and named it \"Omicron\", after the fifteenth letter in the Greek alphabet. Omicron was first detected on 22 November 2021 in laboratories in Botswana and South Africa based on samples collected on 11–16 November. The first known sample was collected in South Africa on 8 November. The first known case, outside of South Africa, was a person arriving in Hong Kong from South Africa via Qatar on 11 November, and another person who arrived in Belgium from Egypt via Turkey on the same date. As of 7 January 2022, the variant has been confirmed in 135 countries. Omicron has an unusually large number of mutations compared to previous variants. Several of the mutations are novel and involve changes to the spike protein reducing the ability for COVID-19 vaccines to prevent symptomatic disease.A December 2021 article in Science observes Omicron did not evolve from any other variant of note, but instead on a distinct track diverging in perhaps mid-2020. The article expounds on three theories that might explain this surprising genetic lineage:\\n\\nThe virus could have circulated and evolved in a population with little surveillance and sequencing.\\nIt could have gestated in a chronically infected COVID-19 patient.\\nIt might have evolved in a nonhuman species, from which it recently spilled back into people.\\n\\n\\n=== Reported cases ===\\n\\nOn 24 November 2021, the variant was first reported to the WHO from South Africa, based on samples that had been collected from 14 to 16 November. South African scientists were first alerted by samples from the very beginning of November where the PCR tests had S gene target failure (occurs in a few variants, but not in Delta which dominated in the country in October) and by a sudden increase of COVID-19 cases in Gauteng; sequencing revealed that more than 70 percent of samples collected in the province between 14 and 23 November were a new variant. The first confirmed specimens of Omicron were collected on 8 November 2021 in South Africa, and on 9 November in Botswana. Likely Omicron (SGTF) samples had occurred  on 4 November 2021 in Pretoria, South Africa.When WHO was alerted on 24 November, Hong Kong was the only place outside Africa that had confirmed a case of Omicron; one person who traveled from South Africa on 11 November, and another traveler who was cross-infected by this case while staying in the same quarantine hotel.On 25 November, one confirmed case was identified in Israel from a traveler returning from Malawi, along with two who returned from South Africa and one from Madagascar. All four initial cases reported from Botswana occurred among fully vaccinated individuals.On 26 November, Belgium confirmed its first case; an unvaccinated person who had travelled from Egypt via Turkey on 11 November. All three initial confirmed and suspected cases reported from Israel occurred among fully vaccinated individuals, as did a single suspected case in Germany.On 27 November, two cases were detected in the United Kingdom, another two in Munich, Germany and one in Milan, Italy.On 28 November, 13 cases were confirmed in the Netherlands among the 624 airline passengers who arrived from South Africa on 26 November. Confirmation of a further 5 cases among these passengers followed later. Entry into the Netherlands generally required having been vaccinated or PCR-tested, or having recovered. The passengers of these two flights had been tested upon arrival because of the newly imposed restrictions (which were set in place during their flight), after which 61 tested positive for SARS-CoV-2. A further two cases were detected in Australia. Both people landed in Sydney the previous day, and travelled from southern Africa to Sydney Airport via Doha Airport. The two people, who were fully vaccinated, entered isolation; 12 other travellers from southern Africa also entered quarantine for fourteen days, while about 260 other passengers and crew on the flight have been directed to isolate. Two travellers from South Africa who landed in Denmark tested positive for COVID-19; it was confirmed on 28 November that both carried the Omicron variant. On the same day, Austria also confirmed its first Omicron case. A detected Omicron case was reported in the Czech Republic, from a traveler who spent time in Namibia. Canada also reported its first Omicron cases, with two from travelers from Nigeria, therefore becoming the first North American country to report an Omicron case.On 29 November, a positive case was recorded in Darwin, Australia. The person arrived in Darwin on a repatriation flight from Johannesburg, South Africa on 25 November, and was taken to a quarantine facility, where the positive test was recorded. Two more people who travelled to Sydney from southern Africa via Singapore tested positive. Portugal reported 13 Omicron cases, all of them members of a soccer club. Sweden also confirmed their first case on 29 November, as did Spain, when a traveler came from South Africa.On 30 November, the Netherlands reported that Omicron cases had been detected in two samples dating back as early as 19 November. A positive case was recorded in Sydney from a traveller who had visited southern Africa before travel restrictions were imposed, and was subsequently active in the community. Japan also confirmed its first case. Two Israeli doctors have tested positive and have entered isolation. Both of them had received three shots of the Pfizer vaccine prior to testing positive. In Brazil, three cases of the Omicron variant were confirmed in São Paulo. Another five are under suspicion. A person in Leipzig, Germany with no travel history nor contact with travellers tested positive for Omicron.On 1 December, the Omicron variant was detected in three samples in Nigeria that had been collected from travelers from South Africa within the last week. On the same day, public health authorities in the United States announced the country\\'s first confirmed Omicron case. A resident of San Francisco who had been vaccinated returned from South Africa on 22 November, began showing mild symptoms on 25 November and was confirmed to have a mild case of COVID-19 on 29 November. Ireland and South Korea also reported their first cases. South Korea reported its cases from five travelers arriving in South Korea from Nigeria.On 2 December, Dutch health authorities confirmed that all 14 passengers with confirmed Omicron infection on 26 November had been previously vaccinated. The same day, the Norwegian Institute of Public Health confirmed that 50 attendees of a company Christmas party held at a restaurant in Norway\\'s capital, Oslo, were infected with the Omicron variant. France has confirmed only 25 cases of the new Omicron variant but officials say the number could jump significantly in the coming weeks.By 6 December, Malaysia confirmed its first case of the variant. The case was a South African student entering to study at a private university. In Namibia, 18 cases out of 19 positive COVID-19 samples that had been collected between 11 and 26 November were found to be Omicron, indicating a high level of prevalence in the country. Fiji also confirmed two positive cases of the variant. They travelled from Nigeria arriving in Fiji on November 25.On 8 December, WHO announced the variant had been detected in 57 countries.On 9 December, Richard Mihigo, coordinator of the World Health Organisation\\'s Immunisation and Vaccine Development Programme for Africa, announced that Africa accounted for 46% of reported cases of the Omicron variant globally.On 13 December, the first death of a person with Omicron was reported in the UK.On 16 December, New Zealand confirmed its first case of the Omicron variant, an individual who had traveled from Germany via Dubai.The first death of a person with Omicron was reported in Germany on 23 December and in Australia on 27 December.On 3 January 2022, South Korea reported the first two deaths of people who tested positive post mortem for Omicron.On 29 March 2022, Omicron subvariant BA.2 became the dominant strain in the U.S.\\n\\n\\n=== Market reactions ===\\n\\nWorry about the potential economic impact of the Omicron variant led to a drop in global markets on 26 November, including the worst drop of the Dow Jones Industrial Average in 2021, led by travel-related stocks. The price of Brent Crude and West Texas Intermediate oil fell 10% and 11.7%, respectively. Cryptocurrency markets were also routed. The South African rand has also hit an all-time low for 2021, trading at over 16 rand to the dollar, losing 6% of its value in November.In early December 2021, the chairman of the Federal Reserve, Jerome Powell,  testified before the U.S. Senate Committee on Banking that \"The recent rise in COVID-19 cases and the emergence of the Omicron variant pose downside risks to employment and economic activity and increased uncertainty for inflation.\"\\n\\n\\n=== International response ===\\nOn 26 November 2021, WHO advised countries not to impose new restrictions on travel, instead recommending a \"risk-based and scientific\" approach to travel measures. On the same day, the European Centre for Disease Prevention and Control (ECDC) reported modeling indicating that strict travel restrictions would delay the variant\\'s impact on European countries by two weeks, possibly allowing countries to prepare for it.After the WHO announcement, on the same day, several countries announced travel bans from southern Africa in response to the identification of the variant, including the United States, which banned travel from eight African countries, although it notably did not ban travel from any European countries, Israel, Canada, or Australia where cases were also detected at the time the bans were announced. Other countries that also implemented travel bans include Japan, Canada, the European Union, Israel, Australia, the United Kingdom, Singapore, Malaysia, Indonesia, Morocco, and New Zealand.The Brazilian Health Regulatory Agency recommended flight restrictions regarding the new variant. The state of New York declared a state of emergency ahead of a potential Omicron spike, although no cases had yet been detected in the state or the rest of the United States. On 27 November, Switzerland introduced obligatory tests and quarantine for all visitors arriving from countries where the variant was detected, which originally included Belgium and Israel.In response to the various travel bans, South African Minister of Health Joe Phaahla defended his country\\'s handling of the pandemic and said that travel bans went against the \"norms and standards\" of the World Health Organization.Some speculate that travel bans could have a significant impact on South Africa\\'s economy by limiting tourism and could lead to other countries with economies that are reliant on tourism to hide the discovery of new variants of concern. Low vaccine coverage in less-developed nations could create opportunities for the emergence of new variants, and these nations also struggle to gain intellectual property to develop and produce vaccines locally. At the same time, inoculation has slowed in South Africa due to vaccine hesitancy and apathy, with a nationwide vaccination rate of only 35% as of November 2021.On 29 November, the WHO warned countries that the variant poses a very high global risk with severe consequences and that they should prepare by accelerating vaccination of high-priority groups and strengthening health systems. WHO director-general Tedros Adhanom described the global situation as dangerous and precarious and called for a new agreement on the handling of pandemics, as the current system disincentivizes countries from alerting others to threats that will inevitably land on their shores. CEPI CEO Richard Hatchett said that the variant fulfilled predictions that transmission of the virus in low-vaccination areas would accelerate its evolution.In preparation for the Omicron variant arriving in the United States, President Joe Biden has stated that the variant is \"cause for concern, not panic\" and reiterated that the government is prepared for the variant and will have it under control. He also stated that large-scale lockdowns, similar to the ones in 2020 near the beginning of the pandemic, are \"off the table for now.\"In mid-December, multiple Canadian provinces reinstated restrictions on gatherings and events such as sports tournaments, and tightened enforcement of proof of vaccination orders. British Columbia expressly prohibited any non-seated \"organized New Year\\'s Eve event\", while Quebec announced a partial lockdown on 20 December, ordering the closure of all bars, casinos, gyms, schools, and theatres, as well as imposing restrictions on the capacity and operating hours of restaurants, and the prohibition of spectators at professional sporting events.On 18 December, the Netherlands government announced a lockdown intended to prevent spread of the variant during the holiday period.In late December, some countries shortened the typical six-month interval for a booster dose of the vaccine to prepare for a wave of Omicron, as two doses are not enough to stop the infection. UK, South Korea and Thailand reduced to three months; Belgium, four months; France, Singapore, Taiwan, Italy and Australia, five months. Finland reduced it to three months for risk groups. Other countries continued with a six-month booster schedule. While antibody levels begin to drop at four months, a longer interval usually allows time for the immune system\\'s response to mature.\\n\\n\\n== See also ==\\n\\nTimeline of the SARS-CoV-2 Omicron variant\\nCOVID-19 pandemic in Africa\\nCOVID-19 vaccination in Africa\\nCOVID-19 vaccination in Botswana\\nCOVID-19 vaccination in South Africa\\nVariants of SARS-CoV-2\\nOther variants of either interest or concern: Alpha, Beta, Gamma, Delta, Epsilon, Zeta, Eta, Theta, Iota, Kappa, Lambda, Mu\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\nWHO: Update on Omicron\\nLive Omicron Variant Location Map\\nOmicron variant: What have COVID vaccine makers said and are they working on new doses?\\nCallaway E, Ledford H (December 2021). \"How bad is Omicron? What scientists know so far\". Nature. 600 (7888): 197–199. Bibcode:2021Natur.600..197C. doi:10.1038/d41586-021-03614-z. PMID 34857948. S2CID 244840636.\\n\\n\\n== External links ==\\nCorum J, Zimmer C (9 February 2021). \"Coronavirus Variant Tracker\". The New York Times. Archived from the original on 30 November 2021. Retrieved 1 December 2021. Constantly Updated\\nArchived video:\\nMohale F, Phaahla J, Oliveira T, Lessels R, Gottberg A, Moore P, Crisp N (25 November 2021). Health Department briefs media about a new so-called super-variant (Internet video live stream). South Africa: eNCA. Retrieved 27 November 2021. Archived at the Wayback Machine and Ghostarchive.\\nOliveira T, Kamwendo S (26 November 2021). Professor Tulio de Oliviera on the new COVID-19 variant (Internet video live stream). South Africa: SABC News. Archived from the original on 29 November 2021. Retrieved 27 November 2021. Archived at the Wayback Machine and Ghostarchive',\n",
              " 'A virus is a submicroscopic infectious agent that replicates only inside the living cells of an organism. Viruses infect all life forms, from animals and plants to microorganisms, including bacteria and archaea.\\nSince Dmitri Ivanovsky\\'s 1892 article describing a non-bacterial pathogen infecting tobacco plants and the discovery of the tobacco mosaic virus by Martinus Beijerinck in 1898, more than 9,000 virus species have been described in detail of the millions of types of viruses in the environment. Viruses are found in almost every ecosystem on Earth and are the most numerous type of biological entity. The study of viruses is known as virology, a subspeciality of microbiology.  \\nWhen infected, a host cell is often forced to rapidly produce thousands of copies of the original virus. When not inside an infected cell or in the process of infecting a cell, viruses exist in the form of independent particles, or virions, consisting of (i) the genetic material, i.e., long molecules of DNA or RNA that encode the structure of the proteins by which the virus acts; (ii) a protein coat, the capsid, which surrounds and protects the genetic material; and in some cases (iii) an outside envelope of lipids. The shapes of these virus particles range from simple helical and icosahedral forms to more complex structures. Most virus species have virions too small to be seen with an optical microscope and are one-hundredth the size of most bacteria.\\nThe origins of viruses in the evolutionary history of life are unclear: some may have evolved from plasmids—pieces of DNA that can move between cells—while others may have evolved from bacteria. In evolution, viruses are an important means of horizontal gene transfer, which increases genetic diversity in a way analogous to sexual reproduction. Viruses are considered by some biologists to be a life form, because they carry genetic material, reproduce, and evolve through natural selection, although they lack the key characteristics, such as cell structure, that are generally considered necessary criteria for defining life. Because they possess some but not all such qualities, viruses have been described as \"organisms at the edge of life\", and as replicators.Viruses spread in many ways. One transmission pathway is through disease-bearing organisms known as vectors: for example, viruses are often transmitted from plant to plant by insects that feed on plant sap, such as aphids; and viruses in animals can be carried by blood-sucking insects. Many viruses, including influenza viruses, SARS-CoV-2, chickenpox, smallpox, and measles, spread in the air by coughing and sneezing. Norovirus and rotavirus, common causes of viral gastroenteritis, are transmitted by the faecal–oral route, passed by hand-to-mouth contact or in food or water. The infectious dose of norovirus required to produce infection in humans is fewer than 100 particles. HIV is one of several viruses transmitted through sexual contact and by exposure to infected blood. The variety of host cells that a virus can infect is called its \"host range\". This can be narrow, meaning a virus is capable of infecting few species, or broad, meaning it is capable of infecting many.Viral infections in animals provoke an immune response that usually eliminates the infecting virus. Immune responses can also be produced by vaccines, which confer an artificially acquired immunity to the specific viral infection. Some viruses, including those that cause HIV/AIDS, HPV infection, and viral hepatitis, evade these immune responses and result in chronic infections. Several classes of antiviral drugs have been developed.\\n\\n\\n== Etymology ==\\nThe word is from the Latin neuter vīrus referring to poison and other noxious liquids, from the same Indo-European base as Sanskrit viṣa, Avestan vīša, and ancient Greek ἰός (all meaning \\'poison\\'), first attested in English in 1398 in John Trevisa\\'s translation of Bartholomeus Anglicus\\'s De Proprietatibus Rerum. Virulent, from Latin virulentus (\\'poisonous\\'), dates to c. 1400. A meaning of \\'agent that causes infectious disease\\' is first recorded in 1728, long before the discovery of viruses by Dmitri Ivanovsky in 1892. The English plural is viruses (sometimes also  vira),  whereas the Latin word is a mass noun, which has no classically attested plural (vīra is used in Neo-Latin). The adjective viral dates to 1948. The term virion (plural virions), which dates from 1959, is also used to refer to a single viral particle that is released from the cell and is capable of infecting other cells of the same type.\\n\\n\\n== History ==\\n\\nLouis Pasteur was unable to find a causative agent for rabies and speculated about a pathogen too small to be detected by microscopes. In 1884, the French microbiologist Charles Chamberland invented the Chamberland filter (or Pasteur-Chamberland filter) with pores small enough to remove all bacteria from a solution passed through it. In 1892, the Russian biologist Dmitri Ivanovsky used this filter to study what is now known as the tobacco mosaic virus: crushed leaf extracts from infected tobacco plants remained infectious even after filtration to remove bacteria. Ivanovsky suggested the infection might be caused by a toxin produced by bacteria, but he did not pursue the idea. At the time it was thought that all infectious agents could be retained by filters and grown on a nutrient medium—this was part of the germ theory of disease. In 1898, the Dutch microbiologist Martinus Beijerinck repeated the experiments and became convinced that the filtered solution contained a new form of infectious agent. He observed that the agent multiplied only in cells that were dividing, but as his experiments did not show that it was made of particles, he called it a contagium vivum fluidum (soluble living germ) and reintroduced the word virus. Beijerinck maintained that viruses were liquid in nature, a theory later discredited by Wendell Stanley, who proved they were particulate. In the same year, Friedrich Loeffler and Paul Frosch passed the first animal virus, aphthovirus (the agent of foot-and-mouth disease), through a similar filter.In the early 20th century, the English bacteriologist Frederick Twort discovered a group of viruses that infect bacteria, now called bacteriophages (or commonly \\'phages\\'), and the French-Canadian microbiologist Félix d\\'Herelle described viruses that, when added to bacteria on an agar plate, would produce areas of dead bacteria. He accurately diluted a suspension of these viruses and discovered that the highest dilutions (lowest virus concentrations), rather than killing all the bacteria, formed discrete areas of dead organisms. Counting these areas and multiplying by the dilution factor allowed him to calculate the number of viruses in the original suspension. Phages were heralded as a potential treatment for diseases such as typhoid and cholera, but their promise was forgotten with the development of penicillin. The development of bacterial resistance to antibiotics has renewed interest in the therapeutic use of bacteriophages.By the end of the 19th century, viruses were defined in terms of their infectivity, their ability to pass filters, and their requirement for living hosts. Viruses had been grown only in plants and animals. In 1906 Ross Granville Harrison invented a method for growing tissue in lymph, and in 1913 E. Steinhardt, C. Israeli, and R.A. Lambert used this method to grow vaccinia virus in fragments of guinea pig corneal tissue. In 1928, H. B. Maitland and M. C. Maitland grew vaccinia virus in suspensions of minced hens\\' kidneys. Their method was not widely adopted until the 1950s when poliovirus was grown on a large scale for vaccine production.Another breakthrough came in 1931 when the American pathologist Ernest William Goodpasture and Alice Miles Woodruff grew influenza and several other viruses in fertilised chicken eggs. In 1949, John Franklin Enders, Thomas Weller, and Frederick Robbins grew poliovirus in cultured cells from aborted human embryonic tissue, the first virus to be grown without using solid animal tissue or eggs. This work enabled Hilary Koprowski, and then Jonas Salk, to make an effective polio vaccine.The first images of viruses were obtained upon the invention of electron microscopy in 1931 by the German engineers Ernst Ruska and Max Knoll. In 1935, American biochemist and virologist Wendell Meredith Stanley examined the tobacco mosaic virus and found it was mostly made of protein. A short time later, this virus was separated into protein and RNA parts.\\nThe tobacco mosaic virus was the first to be crystallised and its structure could, therefore, be elucidated in detail. The first X-ray diffraction pictures of the crystallised virus were obtained by Bernal and Fankuchen in 1941. Based on her X-ray crystallographic pictures, Rosalind Franklin discovered the full structure of the virus in 1955. In the same year, Heinz Fraenkel-Conrat and Robley Williams showed that purified tobacco mosaic virus RNA and its protein coat can assemble by themselves to form functional viruses, suggesting that this simple mechanism was probably the means through which viruses were created within their host cells.The second half of the 20th century was the golden age of virus discovery, and most of the documented species of animal, plant, and bacterial viruses were discovered during these years. In 1957 equine arterivirus and the cause of Bovine virus diarrhoea (a pestivirus) were discovered. In 1963 the hepatitis B virus was discovered by Baruch Blumberg, and in 1965 Howard Temin described the first retrovirus. Reverse transcriptase, the enzyme that retroviruses use to make DNA copies of their RNA, was first described in 1970 by Temin and David Baltimore independently. In 1983 Luc Montagnier\\'s team at the Pasteur Institute in France, first isolated the retrovirus now called HIV. In 1989 Michael Houghton\\'s team at Chiron Corporation discovered hepatitis C.\\n\\n\\n== Origins ==\\n\\nViruses are found wherever there is life and have probably existed since living cells first evolved. The origin of viruses is unclear because they do not form fossils, so molecular techniques are used to investigate how they arose. In addition, viral genetic material occasionally integrates into the germline of the host organisms, by which they can be passed on vertically to the offspring of the host for many generations. This provides an invaluable source of information for paleovirologists to trace back ancient viruses that have existed up to millions of years ago. There are three main hypotheses that aim to explain the origins of viruses:\\nRegressive hypothesis\\nViruses may have once been small cells that parasitised larger cells. Over time, genes not required by their parasitism were lost. The bacteria rickettsia and chlamydia are living cells that, like viruses, can reproduce only inside host cells. They lend support to this hypothesis, as their dependence on parasitism is likely to have caused the loss of genes that enabled them to survive outside a cell. This is also called the \\'degeneracy hypothesis\\', or \\'reduction hypothesis\\'.\\nCellular origin hypothesis\\nSome viruses may have evolved from bits of DNA or RNA that \"escaped\" from the genes of a larger organism. The escaped DNA could have come from plasmids (pieces of naked DNA that can move between cells) or transposons (molecules of DNA that replicate and move around to different positions within the genes of the cell). Once called \"jumping genes\", transposons are examples of mobile genetic elements and could be the origin of some viruses. They were discovered in maize by Barbara McClintock in 1950. This is sometimes called the \\'vagrancy hypothesis\\', or the \\'escape hypothesis\\'.\\nCo-evolution hypothesis\\nThis is also called the \\'virus-first hypothesis\\' and proposes that viruses may have evolved from complex molecules of protein and nucleic acid at the same time that cells first appeared on Earth and would have been dependent on cellular life for billions of years. Viroids are molecules of RNA that are not classified as viruses because they lack a protein coat. They have characteristics that are common to several viruses and are often called subviral agents. Viroids are important pathogens of plants. They do not code for proteins but interact with the host cell and use the host machinery for their replication. The hepatitis delta virus of humans has an RNA genome similar to viroids but has a protein coat derived from hepatitis B virus and cannot produce one of its own. It is, therefore, a defective virus. Although hepatitis delta virus genome may replicate independently once inside a host cell, it requires the help of hepatitis B virus to provide a protein coat so that it can be transmitted to new cells. In similar manner, the sputnik virophage is dependent on mimivirus, which infects the protozoan Acanthamoeba castellanii. These viruses, which are dependent on the presence of other virus species in the host cell, are called \\'satellites\\' and may represent evolutionary intermediates of viroids and viruses.In the past, there were problems with all of these hypotheses: the regressive hypothesis did not explain why even the smallest of cellular parasites do not resemble viruses in any way. The escape hypothesis did not explain the complex capsids and other structures on virus particles. The virus-first hypothesis contravened the definition of viruses in that they require host cells. Viruses are now recognised as ancient and as having origins that pre-date the divergence of life into the three domains. This discovery has led modern virologists to reconsider and re-evaluate these three classical hypotheses.The evidence for an ancestral world of RNA cells and computer analysis of viral and host DNA sequences are giving a better understanding of the evolutionary relationships between different viruses and may help identify the ancestors of modern viruses. To date, such analyses have not proved which of these hypotheses is correct. It seems unlikely that all currently known viruses have a common ancestor, and viruses have probably arisen numerous times in the past by one or more mechanisms.\\n\\n\\n== Microbiology ==\\n\\n\\n=== Life properties ===\\nScientific opinions differ on whether viruses are a form of life or organic structures that interact with living organisms. They have been described as \"organisms at the edge of life\", since they resemble organisms in that they possess genes, evolve by natural selection, and reproduce by creating multiple copies of themselves through self-assembly. Although they have genes, they do not have a cellular structure, which is often seen as the basic unit of life. Viruses do not have their own metabolism and require a host cell to make new products. They therefore cannot naturally reproduce outside a host cell—although some bacteria such as rickettsia and chlamydia are considered living organisms despite the same limitation. Accepted forms of life use cell division to reproduce, whereas viruses spontaneously assemble within cells. They differ from autonomous growth of crystals as they inherit genetic mutations while being subject to natural selection. Virus self-assembly within host cells has implications for the study of the origin of life, as it lends further credence to the hypothesis that life could have started as self-assembling organic molecules.\\n\\n\\n=== Structure ===\\n\\nViruses display a wide diversity of sizes and shapes, called \\'morphologies\\'. In general, viruses are much smaller than bacteria and more than a thousand bacteriophage viruses would fit inside an Escherichia coli bacterium\\'s cell. Many viruses that have been studied are spherical and have a diameter between 20 and 300 nanometres. Some filoviruses, which are filaments have a total length of up to 1400 nm; their diameters are only about 80 nm. Most viruses cannot be seen with an optical microscope, so scanning and transmission electron microscopes are used to visualise them. To increase the contrast between viruses and the background, electron-dense \"stains\" are used. These are solutions of salts of heavy metals, such as tungsten, that scatter the electrons from regions covered with the stain. When virions are coated with stain (positive staining), fine detail is obscured. Negative staining overcomes this problem by staining the background only.A complete virus particle, known as a virion, consists of nucleic acid surrounded by a protective coat of protein called a capsid. These are formed from  protein subunits called capsomeres. Viruses can have a lipid \"envelope\" derived from the host cell membrane. The capsid is made from proteins encoded by the viral genome and its shape serves as the basis for morphological distinction. Virally-coded protein subunits will self-assemble to form a capsid, in general requiring the presence of the virus genome. Complex viruses code for proteins that assist in the construction of their capsid. Proteins associated with nucleic acid are known as nucleoproteins, and the association of viral capsid proteins with viral nucleic acid is called a nucleocapsid. The capsid and entire virus structure can be mechanically (physically) probed through atomic force microscopy. In general, there are five main morphological virus types:\\n\\nHelical\\nThese viruses are composed of a single type of capsomere stacked around a central axis to form a helical structure, which may have a central cavity, or tube. This arrangement results in virions which can be short and highly rigid rods, or long and very flexible filaments. The genetic material (typically single-stranded RNA, but single-stranded DNA in some cases) is bound into the protein helix by interactions between the negatively charged nucleic acid and positive charges on the protein. Overall, the length of a helical capsid is related to the length of the nucleic acid contained within it, and the diameter is dependent on the size and arrangement of capsomeres. The well-studied tobacco mosaic virus and inovirus  are examples of helical viruses.Icosahedral\\nMost animal viruses are icosahedral or near-spherical with chiral icosahedral symmetry. A regular icosahedron is the optimum way of forming a closed shell from identical subunits. The minimum number of capsomeres required for each triangular face is 3, which gives 60 for the icosahedron. Many viruses, such as rotavirus, have more than 60 capsomers and appear spherical but they retain this symmetry. To achieve this, the capsomeres at the apices are surrounded by five other capsomeres and are called pentons. Capsomeres on the triangular faces are surrounded by six others and are called hexons. Hexons are in essence flat and pentons, which form the 12 vertices, are curved. The same protein may act as the subunit of both the pentamers and hexamers or they may be composed of different proteins.Prolate\\nThis is an icosahedron elongated along the fivefold axis and is a common arrangement of the heads of bacteriophages. This structure is composed of a cylinder with a cap at either end.Enveloped\\nSome species of virus envelop themselves in a modified form of one of the cell membranes, either the outer membrane surrounding an infected host cell or internal membranes such as a nuclear membrane or endoplasmic reticulum, thus gaining an outer lipid bilayer known as a viral envelope. This membrane is studded with proteins coded for by the viral genome and host genome; the lipid membrane itself and any carbohydrates present originate entirely from the host. Influenza virus, HIV (which causes AIDS), and severe acute respiratory syndrome coronavirus 2 (which causes COVID-19) use this strategy. Most enveloped viruses are dependent on the envelope for their infectivity.Complex\\nThese viruses possess a capsid that is neither purely helical nor purely icosahedral, and that may possess extra structures such as protein tails or a complex outer wall. Some bacteriophages, such as Enterobacteria phage T4, have a complex structure consisting of an icosahedral head bound to a helical tail, which may have a hexagonal base plate with protruding protein tail fibres. This tail structure acts like a molecular syringe, attaching to the bacterial host and then injecting the viral genome into the cell.The poxviruses are large, complex viruses that have an unusual morphology. The viral genome is associated with proteins within a central disc structure known as a nucleoid. The nucleoid is surrounded by a membrane and two lateral bodies of unknown function. The virus has an outer envelope with a thick layer of protein studded over its surface. The whole virion is slightly pleomorphic, ranging from ovoid to brick-shaped.\\n\\n\\n==== Giant viruses ====\\n\\nMimivirus is one of the largest characterised viruses, with a capsid diameter of 400 nm. Protein filaments measuring 100 nm project from the surface. The capsid appears hexagonal under an electron microscope, therefore the capsid is probably icosahedral. In 2011, researchers discovered the largest then known virus in samples of water collected from the ocean floor off the coast of Las Cruces, Chile. Provisionally named Megavirus chilensis, it can be seen with a basic optical microscope. In 2013, the Pandoravirus genus was discovered in Chile and Australia, and has genomes about twice as large as Megavirus and Mimivirus. All giant viruses have dsDNA genomes and they are classified into several families: Mimiviridae, Pithoviridae, Pandoraviridae, Phycodnaviridae, and the Mollivirus genus.Some viruses that infect Archaea have complex structures unrelated to any other form of virus, with a wide variety of unusual shapes, ranging from spindle-shaped structures to viruses that resemble hooked rods, teardrops or even bottles. Other archaeal viruses resemble the tailed bacteriophages, and can have multiple tail structures.\\n\\n\\n=== Genome ===\\nAn enormous variety of genomic structures can be seen among viral species; as a group, they contain more structural genomic diversity than plants, animals, archaea, or bacteria. There are millions of different types of viruses, although fewer than 7,000 types have been described in detail. As of January 2021, the NCBI Virus genome database has more than 193,000 complete genome sequences, but there are doubtlessly many more to be discovered.A virus has either a DNA or an RNA genome and is called a DNA virus or an RNA virus, respectively. The vast majority of viruses have RNA genomes. Plant viruses tend to have single-stranded RNA genomes and bacteriophages tend to have double-stranded DNA genomes.Viral genomes are circular, as in the polyomaviruses, or linear, as in the adenoviruses. The type of nucleic acid is irrelevant to the shape of the genome. Among RNA viruses and certain DNA viruses, the genome is often divided up into separate parts, in which case it is called segmented. For RNA viruses, each segment often codes for only one protein and they are usually found together in one capsid. All segments are not required to be in the same virion for the virus to be infectious, as demonstrated by brome mosaic virus and several other plant viruses.A viral genome, irrespective of nucleic acid type, is almost always either single-stranded (ss) or double-stranded (ds). Single-stranded genomes consist of an unpaired nucleic acid, analogous to one-half of a ladder split down the middle. Double-stranded genomes consist of two complementary paired nucleic acids, analogous to a ladder. The virus particles of some virus families, such as those belonging to the Hepadnaviridae, contain a genome that is partially double-stranded and partially single-stranded.For most viruses with RNA genomes and some with single-stranded DNA (ssDNA) genomes, the single strands are said to be either positive-sense (called the \\'plus-strand\\') or negative-sense (called the \\'minus-strand\\'), depending on if they are complementary to the viral messenger RNA (mRNA). Positive-sense viral RNA is in the same sense as viral mRNA and thus at least a part of it can be immediately translated by the host cell. Negative-sense viral RNA is complementary to mRNA and thus must be converted to positive-sense RNA by an RNA-dependent RNA polymerase before translation. DNA nomenclature for viruses with genomic ssDNA is similar to RNA nomenclature, in that positive-strand viral ssDNA is identical in sequence to the viral mRNA and is thus a coding strand, while negative-sense viral ssDNA is complementary to the viral mRNA and is thus a template strand. Several types of ssDNA and ssRNA viruses have genomes that are ambisense in that transcription can occur off both strands in a double-stranded replicative intermediate. Examples include geminiviruses, which are ssDNA plant viruses and arenaviruses, which are ssRNA viruses of animals.\\n\\n\\n==== Genome size ====\\nGenome size varies greatly between species. The smallest—the ssDNA circoviruses, family Circoviridae—code for only two proteins and have a genome size of only two kilobases; the largest—the pandoraviruses—have genome sizes of around two megabases which code for about 2500 proteins. Virus genes rarely have introns and often are arranged in the genome so that they overlap.In general, RNA viruses have smaller genome sizes than DNA viruses because of a higher error-rate when replicating, and have a maximum upper size limit. Beyond this, errors when replicating render the virus useless or uncompetitive. To compensate, RNA viruses often have segmented genomes—the genome is split into smaller molecules—thus reducing the chance that an error in a single-component genome will incapacitate the entire genome. In contrast, DNA viruses generally have larger genomes because of the high fidelity of their replication enzymes. Single-strand DNA viruses are an exception to this rule, as mutation rates for these genomes can approach the extreme of the ssRNA virus case.\\n\\n\\n=== Genetic mutation ===\\n\\nViruses undergo genetic change by several mechanisms. These include a process called antigenic drift where individual bases in the DNA or RNA mutate to other bases. Most of these point mutations are \"silent\"—they do not change the protein that the gene encodes—but others can confer evolutionary advantages such as resistance to antiviral drugs. Antigenic shift occurs when there is a major change in the genome of the virus. This can be a result of recombination or reassortment. When this happens with influenza viruses, pandemics might result. RNA viruses often exist as quasispecies or swarms of viruses of the same species but with slightly different genome nucleoside sequences. Such quasispecies are a prime target for natural selection.Segmented genomes confer evolutionary advantages; different strains of a virus with a segmented genome can shuffle and combine genes and produce progeny viruses (or offspring) that have unique characteristics. This is called reassortment or \\'viral sex\\'.Genetic recombination is the process by which a strand of DNA (or RNA) is broken and then joined to the end of a different DNA (or RNA)  molecule. This can occur when viruses infect cells simultaneously and studies of viral evolution have shown that recombination has been rampant in the species studied. Recombination is common to both RNA and DNA viruses.\\n\\n\\n=== Replication cycle ===\\n\\nViral populations do not grow through cell division, because they are acellular. Instead, they use the machinery and metabolism of a host cell to produce multiple copies of themselves, and they assemble in the cell. When infected, the host cell is forced to rapidly produce thousands of  copies of the original virus.Their life cycle differs greatly between species, but there are six basic stages in their life cycle:Attachment is a specific binding between viral capsid proteins and specific receptors on the host cellular surface. This specificity determines the host range and type of host cell of a virus. For example, HIV infects a limited range of human leucocytes. This is because its surface protein, gp120, specifically interacts with the CD4 molecule—a chemokine receptor—which is most commonly found on the surface of CD4+ T-Cells. This mechanism has evolved to favour those viruses that infect only cells in which they are capable of replication. Attachment to the receptor can induce the viral envelope protein to undergo changes that result in the fusion of viral and cellular membranes, or changes of non-enveloped virus surface proteins that allow the virus to enter.Penetration or viral entry follows attachment: Virions enter the host cell through receptor-mediated endocytosis or membrane fusion. The infection of plant and fungal cells is different from that of animal cells. Plants have a rigid cell wall made of cellulose, and fungi one of chitin, so most viruses can get inside these cells only after trauma to the cell wall. Nearly all plant viruses (such as tobacco mosaic virus) can also move directly from cell to cell, in the form of single-stranded nucleoprotein complexes, through pores called plasmodesmata. Bacteria, like plants, have strong cell walls that a virus must breach to infect the cell. Given that bacterial cell walls are much thinner than plant cell walls due to their much smaller size, some viruses have evolved mechanisms that inject their genome into the bacterial cell across the cell wall, while the viral capsid remains outside.Uncoating is a process in which the viral capsid is removed: This may be by degradation by viral enzymes or host enzymes or by simple dissociation; the end-result is the releasing of the viral genomic nucleic acid.Replication of viruses involves primarily multiplication of the genome. Replication involves the synthesis of viral messenger RNA (mRNA) from \"early\" genes (with exceptions for positive-sense RNA viruses), viral protein synthesis, possible assembly of viral proteins, then viral genome replication mediated by early or regulatory protein expression. This may be followed, for complex viruses with larger genomes, by one or more further rounds of mRNA synthesis: \"late\" gene expression is, in general, of structural or virion proteins.Assembly – Following the structure-mediated self-assembly of the virus particles, some modification of the proteins often occurs. In viruses such as HIV, this modification (sometimes called maturation) occurs after the virus has been released from the host cell.Release – Viruses can be released from the host cell by lysis, a process that kills the cell by bursting its membrane and cell wall if present: this is a feature of many bacterial and some animal viruses. Some viruses undergo a lysogenic cycle where the viral genome is incorporated by genetic recombination into a specific place in the host\\'s chromosome. The viral genome is then known as a \"provirus\" or, in the case of bacteriophages a \"prophage\". Whenever the host divides, the viral genome is also replicated. The viral genome is mostly silent within the host. At some point, the provirus or prophage may give rise to the active virus, which may lyse the host cells. Enveloped viruses (e.g., HIV) typically are released from the host cell by budding. During this process, the virus acquires its envelope, which is a modified piece of the host\\'s plasma or other, internal membrane.\\n\\n\\n=== Genome replication ===\\nThe genetic material within virus particles, and the method by which the material is replicated, varies considerably between different types of viruses.\\n\\nDNA viruses\\nThe genome replication of most DNA viruses takes place in the cell\\'s nucleus. If the cell has the appropriate receptor on its surface, these viruses enter the cell either by direct fusion with the cell membrane (e.g., herpesviruses) or—more usually—by receptor-mediated endocytosis. Most DNA viruses are entirely dependent on the host cell\\'s DNA and RNA synthesising machinery and RNA processing machinery. Viruses with larger genomes may encode much of this machinery themselves. In eukaryotes, the viral genome must cross the cell\\'s nuclear membrane to access this machinery, while in bacteria it need only enter the cell.RNA viruses\\nReplication of RNA viruses usually takes place in the cytoplasm. RNA viruses can be placed into four different groups depending on their modes of replication. The polarity (whether or not it can be used directly by ribosomes to make proteins) of single-stranded RNA viruses largely determines the replicative mechanism; the other major criterion is whether the genetic material is single-stranded or double-stranded. All RNA viruses use their own RNA replicase enzymes to create copies of their genomes.Reverse transcribing viruses\\nReverse transcribing viruses have ssRNA (Retroviridae, Metaviridae, Pseudoviridae) or dsDNA (Caulimoviridae, and Hepadnaviridae) in their particles. Reverse transcribing viruses with RNA genomes (retroviruses) use a DNA intermediate to replicate, whereas those with DNA genomes (pararetroviruses) use an RNA intermediate during genome replication. Both types use a reverse transcriptase, or RNA-dependent DNA polymerase enzyme, to carry out the nucleic acid conversion. Retroviruses integrate the DNA produced by reverse transcription into the host genome as a provirus as a part of the replication process; pararetroviruses do not, although integrated genome copies of especially plant pararetroviruses can give rise to infectious virus. They are susceptible to antiviral drugs that inhibit the reverse transcriptase enzyme, e.g. zidovudine and lamivudine. An example of the first type is HIV, which is a retrovirus. Examples of the second type are the Hepadnaviridae, which includes Hepatitis B virus.\\n\\n\\n=== Cytopathic effects on the host cell ===\\nThe range of structural and biochemical effects that viruses have on the host cell is extensive. These are called \\'cytopathic effects\\'. Most virus infections eventually result in the death of the host cell. The causes of death include cell lysis, alterations to the cell\\'s surface membrane and apoptosis. Often cell death is caused by cessation of its normal activities because of suppression by virus-specific proteins, not all of which are components of the virus particle. The distinction between cytopathic and harmless is gradual. Some viruses, such as Epstein–Barr virus, can cause cells to proliferate without causing malignancy, while others, such as papillomaviruses, are established causes of cancer.\\n\\n\\n=== Dormant and latent infections ===\\nSome viruses cause no apparent changes to the infected cell. Cells in which the virus is latent and inactive show few signs of infection and often function normally. This causes persistent infections and the virus is often dormant for many months or years. This is often the case with herpes viruses.\\n\\n\\n=== Host range ===\\nViruses are by far the most abundant biological entities on Earth and they outnumber all the others put together. They infect all types of cellular life including animals, plants, bacteria and fungi. Different types of viruses can infect only a limited range of hosts and many are species-specific. Some, such as smallpox virus for example, can infect only one species—in this case humans, and are said to have a narrow host range. Other viruses, such as rabies virus, can infect different species of mammals and are said to have a broad range. The viruses that infect plants are harmless to animals, and most viruses that infect other animals are harmless to humans. The host range of some bacteriophages is limited to a single strain of bacteria and they can be used to trace the source of outbreaks of infections by a method called phage typing. The complete set of viruses in an organism or habitat is called the virome; for example, all human viruses constitute the human virome.\\n\\n\\n== Classification ==\\n\\nClassification seeks to describe the diversity of viruses by naming and grouping them on the basis of similarities. In 1962, André Lwoff, Robert Horne, and Paul Tournier were the first to develop a means of virus classification, based on the Linnaean hierarchical system. This system based classification on phylum, class, order, family, genus, and species. Viruses were grouped according to their shared properties (not those of their hosts) and the type of nucleic acid forming their genomes. In 1966, the International Committee on Taxonomy of Viruses (ICTV) was formed. The system proposed by Lwoff, Horne and Tournier was initially not accepted by the ICTV because the small genome size of viruses and their high rate of mutation made it difficult to determine their ancestry beyond order. As such, the Baltimore classification system has come to be used to supplement the more traditional hierarchy. Starting in 2018, the ICTV began to acknowledge deeper evolutionary relationships between viruses that have been discovered over time and adopted a 15-rank classification system ranging from realm to species.\\n\\n\\n=== ICTV classification ===\\nThe ICTV developed the current classification system and wrote guidelines that put a greater weight on certain virus properties to maintain family uniformity. A unified taxonomy (a universal system for classifying viruses) has been established. Only a small part of the total diversity of viruses has been studied. As of 2021, 6 realms, 10 kingdoms, 17 phyla, 2 subphyla, 39 classes, 65 orders, 8 suborders, 233 families, 168 subfamilies, 2,606 genera, 84 subgenera, and 10,434 species of viruses have been defined by the ICTV.The general taxonomic structure of taxon ranges and the suffixes used in taxonomic names are shown hereafter. As of 2020, the ranks of subrealm, subkingdom, and subclass are unused, whereas all other ranks are in use.\\n\\nRealm (-viria)\\nSubrealm (-vira)\\nKingdom (-virae)\\nSubkingdom (-virites)\\nPhylum (-viricota)\\nSubphylum (-viricotina)\\nClass (-viricetes)\\nSubclass (-viricetidae)\\nOrder (-virales)\\nSuborder (-virineae)\\nFamily (-viridae)\\nSubfamily (-virinae)\\nGenus (-virus)\\nSubgenus (-virus)\\nSpecies\\n\\n\\n=== Baltimore classification ===\\n\\nThe Nobel Prize-winning biologist David Baltimore devised the Baltimore classification system. The ICTV classification system is used in conjunction with the Baltimore classification system in modern virus classification.The Baltimore classification of viruses is based on the mechanism of mRNA production. Viruses must generate mRNAs from their genomes to produce proteins and replicate themselves, but different mechanisms are used to achieve this in each virus family. Viral genomes may be single-stranded (ss) or double-stranded (ds), RNA or DNA, and may or may not use reverse transcriptase (RT). In addition, ssRNA viruses may be either sense (+) or antisense (−). This classification places viruses into seven groups:\\n\\nI: dsDNA viruses (e.g. Adenoviruses, Herpesviruses, Poxviruses)\\nII: ssDNA viruses (+ strand or \"sense\") DNA (e.g. Parvoviruses)\\nIII: dsRNA viruses (e.g. Reoviruses)\\nIV: (+)ssRNA viruses (+ strand or sense) RNA (e.g. Coronaviruses, Picornaviruses, Togaviruses)\\nV: (−)ssRNA viruses (− strand or antisense) RNA (e.g. Orthomyxoviruses, Rhabdoviruses)\\nVI: ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses)\\nVII: dsDNA-RT viruses DNA with RNA intermediate in life-cycle (e.g. Hepadnaviruses)\\n\\n\\n== Role in human disease ==\\n\\nExamples of common human diseases caused by viruses include the common cold, influenza, chickenpox, and cold sores. Many serious diseases such as rabies, Ebola virus disease, AIDS (HIV), avian influenza, and SARS are caused by viruses. The relative ability of viruses to cause disease is described in terms of virulence. Other diseases are under investigation to discover if they have a virus as the causative agent, such as the possible connection between human herpesvirus 6 (HHV6) and neurological diseases such as multiple sclerosis and chronic fatigue syndrome. There is controversy over whether the bornavirus, previously thought to cause neurological diseases in horses, could be responsible for psychiatric illnesses in humans.Viruses have different mechanisms by which they produce disease in an organism, which depends largely on the viral species. Mechanisms at the cellular level primarily include cell lysis, the breaking open and subsequent death of the cell. In multicellular organisms, if enough cells die, the whole organism will start to suffer the effects. Although viruses cause disruption of healthy homeostasis, resulting in disease, they may exist relatively harmlessly within an organism. An example would include the ability of the herpes simplex virus, which causes cold sores, to remain in a dormant state within the human body. This is called latency and is a characteristic of the herpes viruses, including Epstein–Barr virus, which causes glandular fever, and varicella zoster virus, which causes chickenpox and shingles. Most people have been infected with at least one of these types of herpes virus. These latent viruses might sometimes be beneficial, as the presence of the virus can increase immunity against bacterial pathogens, such as Yersinia pestis.Some viruses can cause lifelong or chronic infections, where the viruses continue to replicate in the body despite the host\\'s defence mechanisms. This is common in hepatitis B virus and hepatitis C virus infections. People chronically infected are known as carriers, as they serve as reservoirs of infectious virus. In populations with a high proportion of carriers, the disease is said to be endemic.\\n\\n\\n=== Epidemiology ===\\nViral epidemiology is the branch of medical science that deals with the transmission and control of virus infections in humans. Transmission of viruses can be vertical, which means from mother to child, or horizontal, which means from person to person. Examples of vertical transmission include hepatitis B virus and HIV, where the baby is born already infected with the virus. Another, more rare, example is the varicella zoster virus, which, although causing relatively mild infections in children and adults, can be fatal to the foetus and newborn baby.Horizontal transmission is the most common mechanism of spread of viruses in populations. Horizontal transmission can occur when body fluids are exchanged during sexual activity, by exchange of saliva or when contaminated food or water is ingested. It can also occur when aerosols containing viruses are inhaled or by insect vectors such as when infected mosquitoes penetrate the skin of a host. Most types of viruses are restricted to just one or two of these mechanisms and they are referred to as \"respiratory viruses\" or \"enteric viruses\" and so forth. The rate or speed of transmission of viral infections depends on factors that include population density, the number of susceptible individuals, (i.e., those not immune), the quality of healthcare and the weather.Epidemiology is used to break the chain of infection in populations during outbreaks of viral diseases. Control measures are used that are based on knowledge of how the virus is transmitted. It is important to find the source, or sources, of the outbreak and to identify the virus. Once the virus has been identified, the chain of transmission can sometimes be broken by vaccines. When vaccines are not available, sanitation and disinfection can be effective. Often, infected people are isolated from the rest of the community, and those that have been exposed to the virus are placed in quarantine. To control the outbreak of foot-and-mouth disease in cattle in Britain in 2001, thousands of cattle were slaughtered. Most viral infections of humans and other animals have incubation periods during which the infection causes no signs or symptoms. Incubation periods for viral diseases range from a few days to weeks, but are known for most infections. Somewhat overlapping, but mainly following the incubation period, there is a period of communicability—a time when an infected individual or animal is contagious and can infect another person or animal. This, too, is known for many viral infections, and knowledge of the length of both periods is important in the control of outbreaks. When outbreaks cause an unusually high proportion of cases in a population, community, or region, they are called epidemics. If outbreaks spread worldwide, they are called pandemics.\\n\\n\\n=== Epidemics and pandemics ===\\n\\nA pandemic is a worldwide epidemic. The 1918 flu pandemic, which lasted until 1919, was a category 5 influenza pandemic caused by an unusually severe and deadly influenza A virus. The victims were often healthy young adults, in contrast to most influenza outbreaks, which predominantly affect juvenile, elderly, or otherwise-weakened patients. Older estimates say it killed 40–50 million people, while more recent research suggests that it may have killed as many as 100 million people, or 5% of the world\\'s population in 1918.Although viral pandemics are rare events, HIV—which evolved from viruses found in monkeys and chimpanzees—has been pandemic since at least the 1980s. During the 20th century there were four pandemics caused by influenza virus and those that occurred in 1918, 1957 and 1968 were severe. Most researchers believe that HIV originated in sub-Saharan Africa during the 20th century; it is now a pandemic, with an estimated 37.9 million people now living with the disease worldwide. There were about 770,000 deaths from AIDS in 2018. The Joint United Nations Programme on HIV/AIDS (UNAIDS) and the World Health Organization (WHO) estimate that AIDS has killed more than 25 million people since it was first recognised on 5 June 1981, making it one of the most destructive epidemics in recorded history. In 2007 there were 2.7 million new HIV infections and 2 million HIV-related deaths.\\n\\nSeveral highly lethal viral pathogens are members of the Filoviridae. Filoviruses are filament-like viruses that cause viral hemorrhagic fever, and include ebolaviruses and marburgviruses. Marburg virus, first discovered in 1967, attracted widespread press attention in April 2005 for an outbreak in Angola. Ebola virus disease has also caused intermittent outbreaks with high mortality rates since 1976 when it was first identified. The worst and most recent one is the 2013–2016 West Africa epidemic.Except for smallpox, most pandemics are caused by newly evolved viruses. These \"emergent\" viruses are usually mutants of less harmful viruses that have circulated previously either in humans or other animals.Severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS)  are caused by new types of coronaviruses. Other coronaviruses are known to cause mild infections in humans, so the virulence and rapid spread of SARS infections—that by July 2003 had caused around 8,000 cases and 800 deaths—was unexpected and most countries were not prepared.A related coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2), thought to have originated in bats, emerged in Wuhan, China in November 2019 and spread rapidly around the world.  Infections with the virus caused the COVID-19 pandemic that started in 2020. Unprecedented restrictions in peacetime were placed on international travel, and curfews were imposed in several major cities worldwide in response to the pandemic.\\n\\n\\n=== Cancer ===\\n\\nViruses are an established cause of cancer in humans and other species. Viral cancers occur only in a minority of infected persons (or animals). Cancer viruses come from a range of virus families, including both RNA and DNA viruses, and so there is no single type of \"oncovirus\" (an obsolete term originally used for acutely transforming retroviruses). The development of cancer is determined by a variety of factors such as host immunity and mutations in the host. Viruses accepted to cause human cancers include some genotypes of human papillomavirus, hepatitis B virus, hepatitis C virus, Epstein–Barr virus, Kaposi\\'s sarcoma-associated herpesvirus and human T-lymphotropic virus. The most recently discovered human cancer virus is a polyomavirus (Merkel cell polyomavirus) that causes most cases of a rare form of skin cancer called Merkel cell carcinoma.\\nHepatitis viruses can develop into a chronic viral infection that leads to liver cancer. Infection by human T-lymphotropic virus can lead to tropical spastic paraparesis and adult T-cell leukaemia. Human papillomaviruses are an established cause of cancers of cervix, skin, anus, and penis. Within the Herpesviridae, Kaposi\\'s sarcoma-associated herpesvirus causes Kaposi\\'s sarcoma and body-cavity lymphoma, and Epstein–Barr virus causes Burkitt\\'s lymphoma, Hodgkin\\'s lymphoma, B lymphoproliferative disorder, and nasopharyngeal carcinoma. Merkel cell polyomavirus closely related to SV40 and mouse polyomaviruses that have been used as animal models for cancer viruses for over 50 years.\\n\\n\\n=== Host defence mechanisms ===\\n\\nThe body\\'s first line of defence against viruses is the innate immune system. This comprises cells and other mechanisms that defend the host from infection in a non-specific manner. This means that the cells of the innate system recognise, and respond to, pathogens in a generic way, but, unlike the adaptive immune system, it does not confer long-lasting or protective immunity to the host.RNA interference is an important innate defence against viruses. Many viruses have a replication strategy that involves double-stranded RNA (dsRNA). When such a virus infects a cell, it releases its RNA molecule or molecules, which immediately bind to a protein complex called a dicer that cuts the RNA into smaller pieces. A biochemical pathway—the RISC complex—is activated, which ensures cell survival by degrading the viral mRNA. Rotaviruses have evolved to avoid this defence mechanism by not uncoating fully inside the cell, and releasing newly produced mRNA through pores in the particle\\'s inner capsid. Their genomic dsRNA remains protected inside the core of the virion.When the adaptive immune system of a vertebrate encounters a virus, it produces specific antibodies that bind to the virus and often render it non-infectious. This is called humoral immunity. Two types of antibodies are important. The first, called IgM, is highly effective at neutralising viruses but is produced by the cells of the immune system only for a few weeks. The second, called IgG, is produced indefinitely. The presence of IgM in the blood of the host is used to test for acute infection, whereas IgG indicates an infection sometime in the past. IgG antibody is measured when tests for immunity are carried out.Antibodies can continue to be an effective defence mechanism even after viruses have managed to gain entry to the host cell. A protein that is in cells, called TRIM21, can attach to the antibodies on the surface of the virus particle. This primes the subsequent destruction of the virus by the enzymes of the cell\\'s proteosome system.\\n\\nA second defence of vertebrates against viruses is called cell-mediated immunity and involves immune cells known as T cells. The body\\'s cells constantly display short fragments of their proteins on the cell\\'s surface, and, if a T cell recognises a suspicious viral fragment there, the host cell is destroyed by \\'killer T\\' cells and the virus-specific T-cells proliferate. Cells such as the macrophage are specialists at this antigen presentation. The production of interferon is an important host defence mechanism. This is a hormone produced by the body when viruses are present. Its role in immunity is complex; it eventually stops the viruses from reproducing by killing the infected cell and its close neighbours.Not all virus infections produce a protective immune response in this way. HIV evades the immune system by constantly changing the amino acid sequence of the proteins on the surface of the virion. This is known as \"escape mutation\" as the viral epitopes escape recognition by the host immune response. These persistent viruses evade immune control by sequestration, blockade of antigen presentation, cytokine resistance, evasion of natural killer cell activities, escape from apoptosis, and antigenic shift. Other viruses, called \\'neurotropic viruses\\', are disseminated by neural spread where the immune system may be unable to reach them due to immune privilege.\\n\\n\\n=== Prevention and treatment ===\\nBecause viruses use vital metabolic pathways within host cells to replicate, they are difficult to eliminate without using drugs that cause toxic effects to host cells in general. The most effective medical approaches to viral diseases are vaccinations to provide immunity to infection, and antiviral drugs that selectively interfere with viral replication.\\n\\n\\n==== Vaccines ====\\n\\nVaccination is a cheap and effective way of preventing infections by viruses. Vaccines were used to prevent viral infections long before the discovery of the actual viruses. Their use has resulted in a dramatic decline in morbidity (illness) and mortality (death) associated with viral infections such as polio, measles, mumps and rubella. Smallpox infections have been eradicated. Vaccines are available to prevent over thirteen viral infections of humans, and more are used to prevent viral infections of animals. Vaccines can consist of live-attenuated or killed viruses, viral proteins (antigens), or RNA. Live vaccines contain weakened forms of the virus, which do not cause the disease but, nonetheless, confer immunity. Such viruses are called attenuated. Live vaccines can be dangerous when given to people with a weak immunity (who are described as immunocompromised), because in these people, the weakened virus can cause the original disease. Biotechnology and genetic engineering techniques are used to produce subunit vaccines. These vaccines use only the capsid proteins of the virus. Hepatitis B vaccine is an example of this type of vaccine. Subunit vaccines are safe for immunocompromised patients because they cannot cause the disease. The yellow fever virus vaccine, a live-attenuated strain called 17D, is probably the safest and most effective vaccine ever generated.\\n\\n\\n==== Antiviral drugs ====\\n\\nAntiviral drugs are often nucleoside analogues (fake DNA building-blocks), which viruses mistakenly incorporate into their genomes during replication. The life-cycle of the virus is then halted because the newly synthesised DNA is inactive. This is because these analogues lack the hydroxyl groups, which, along with phosphorus atoms, link together to form the strong \"backbone\" of the DNA molecule. This is called DNA chain termination. Examples of nucleoside analogues are aciclovir for Herpes simplex virus infections and lamivudine for HIV and hepatitis B virus infections. Aciclovir is one of the oldest and most frequently prescribed antiviral drugs.\\nOther antiviral drugs in use target different stages of the viral life cycle. HIV is dependent on a proteolytic enzyme called the HIV-1 protease for it to become fully infectious. There is a large class of drugs called protease inhibitors that inactivate this enzyme. There are around thirteen classes of antiviral drugs each targeting different viruses or stages of viral replication.Hepatitis C is caused by an RNA virus. In 80% of people infected, the disease is chronic, and without treatment, they are infected for the remainder of their lives. There are effective treatments that use direct-acting antivirals. The treatment of chronic carriers of the hepatitis B virus has also been developed by using similar strategies that include lamivudine and other anti-viral drugs.\\n\\n\\n== Infection in other species ==\\nViruses infect all cellular life and, although viruses occur universally, each cellular species has its own specific range that often infects only that species. Some viruses, called satellites, can replicate only within cells that have already been infected by another virus.\\n\\n\\n=== Animal viruses ===\\n\\nViruses are important pathogens of livestock. Diseases such as foot-and-mouth disease and bluetongue are caused by viruses. Companion animals such as cats, dogs, and horses, if not vaccinated, are susceptible to serious viral infections. Canine parvovirus is caused by a small DNA virus and infections are often fatal in pups. Like all invertebrates, the honey bee is susceptible to many viral infections. Most viruses co-exist harmlessly in their host and cause no signs or symptoms of disease.\\n\\n\\n=== Plant viruses ===\\n\\nThere are many types of plant viruses, but often they cause only a loss of yield, and it is not economically viable to try to control them. Plant viruses are often spread from plant to plant by organisms, known as vectors. These are usually insects, but some fungi, nematode worms, single-celled organisms, and parasitic plants are vectors. When control of plant virus infections is considered economical, for perennial fruits, for example, efforts are concentrated on killing the vectors and removing alternate hosts such as weeds. Plant viruses cannot infect humans and other animals because they can reproduce only in living plant cells.Originally from Peru, the potato has become a staple crop worldwide. The potato virus Y causes disease in potatoes and related species including tomatoes and peppers. In the 1980s, this virus acquired economical importance when it proved difficult to control in seed potato crops. Transmitted by aphids, this virus can reduce crop yields by up to 80 per cent, causing significant losses to potato yields.Plants have elaborate and effective defence mechanisms against viruses. One of the most effective is the presence of so-called resistance (R) genes. Each R gene confers resistance to a particular virus by triggering localised areas of cell death around the infected cell, which can often be seen with the unaided eye as large spots. This stops the infection from spreading. RNA interference is also an effective defence in plants. When they are infected, plants often produce natural disinfectants that kill viruses, such as salicylic acid, nitric oxide, and reactive oxygen molecules.Plant virus particles or virus-like particles (VLPs) have applications in both biotechnology and nanotechnology. The capsids of most plant viruses are simple and robust structures and can be produced in large quantities either by the infection of plants or by expression in a variety of heterologous systems. Plant virus particles can be modified genetically and chemically to encapsulate foreign material and can be incorporated into supramolecular structures for use in biotechnology.\\n\\n\\n=== Bacterial viruses ===\\n\\nBacteriophages are a common and diverse group of viruses and are the most abundant biological entity in aquatic environments—there are up to ten times more of these viruses in the oceans than there are bacteria, reaching levels of 250,000,000 bacteriophages per millilitre of seawater. These viruses infect specific bacteria by binding to surface receptor molecules and then entering the cell. Within a short amount of time, in some cases, just minutes, bacterial polymerase starts translating viral mRNA into protein. These proteins go on to become either new virions within the cell, helper proteins, which help assembly of new virions, or proteins involved in cell lysis. Viral enzymes aid in the breakdown of the cell membrane, and, in the case of the T4 phage, in just over twenty minutes after injection over three hundred phages could be released.The major way bacteria defend themselves from bacteriophages is by producing enzymes that destroy foreign DNA. These enzymes, called restriction endonucleases, cut up the viral DNA that bacteriophages inject into bacterial cells. Bacteria also contain a system that uses CRISPR sequences to retain fragments of the genomes of viruses that the bacteria have come into contact with in the past, which allows them to block the virus\\'s replication through a form of RNA interference. This genetic system provides bacteria with acquired immunity to infection.\\n\\n\\n=== Archaeal viruses ===\\n\\nSome viruses replicate within archaea: these are DNA viruses with unusual and sometimes unique shapes. These viruses have been studied in most detail in the thermophilic archaea, particularly the orders Sulfolobales and Thermoproteales. Defences against these viruses involve RNA interference from repetitive DNA sequences within archaean genomes that are related to the genes of the viruses. Most archaea have CRISPR–Cas systems as an adaptive defence against viruses. These enable archaea to retain sections of viral DNA, which are then used to target and eliminate subsequent infections by the virus using a process similar to RNA interference.\\n\\n\\n== Role in aquatic ecosystems ==\\n\\nViruses are the most abundant biological entity in aquatic environments. There are about ten million of them in a teaspoon of seawater. Most of these viruses are bacteriophages infecting heterotrophic bacteria and cyanophages infecting cyanobacteria and they are essential to the regulation of saltwater and freshwater ecosystems.\\nBacteriophages are harmless to plants and animals, and are essential to the regulation of marine and freshwater ecosystems  are important mortality agents of phytoplankton, the base of the foodchain in aquatic environments. They infect and destroy bacteria in aquatic microbial communities, and are one of the most important mechanisms of recycling carbon and nutrient cycling in marine environments. The organic molecules released from the dead bacterial cells stimulate fresh bacterial and algal growth, in a process known as the viral shunt. In particular, lysis of bacteria by viruses has been shown to enhance nitrogen cycling and stimulate phytoplankton growth.  Viral activity may also affect the biological pump, the process whereby carbon is sequestered in the deep ocean.Microorganisms constitute more than 90% of the biomass in the sea. It is estimated that viruses kill approximately 20% of this biomass each day and that there are 10 to 15 times as many viruses in the oceans as there are bacteria and archaea. Viruses are also major agents responsible for the destruction of phytoplankton including harmful algal blooms,\\nThe number of viruses in the oceans decreases further offshore and deeper into the water, where there are fewer host organisms.In January 2018, scientists reported that 800 million viruses, mainly of marine origin, are deposited daily from the Earth\\'s atmosphere onto every square meter of the planet\\'s surface, as the result of a global atmospheric stream of viruses, circulating above the weather system but below the altitude of usual airline travel, distributing viruses around the planet.Like any organism, marine mammals are susceptible to viral infections. In 1988 and 2002, thousands of harbour seals were killed in Europe by phocine distemper virus. Many other viruses, including caliciviruses, herpesviruses, adenoviruses and parvoviruses, circulate in marine mammal populations.\\n\\n\\n== Role in evolution ==\\n\\nViruses are an important natural means of transferring genes between different species, which increases genetic diversity and drives evolution. It is thought that viruses played a central role in early evolution, before the diversification of the last universal common ancestor into bacteria, archaea and eukaryotes. Viruses are still one of the largest reservoirs of unexplored genetic diversity on Earth.\\n\\n\\n== Applications ==\\n\\n\\n=== Life sciences and medicine ===\\n\\nViruses are important to the study of molecular and cell biology as they provide simple systems that can be used to manipulate and investigate the functions of cells. The study and use of viruses have provided valuable information about aspects of cell biology. For example, viruses have been useful in the study of genetics and helped our understanding of the basic mechanisms of molecular genetics, such as DNA replication, transcription, RNA processing, translation, protein transport, and immunology.\\nGeneticists often use viruses as vectors to introduce genes into cells that they are studying. This is useful for making the cell produce a foreign substance, or to study the effect of introducing a new gene into the genome. Similarly, virotherapy uses viruses as vectors to treat various diseases, as they can specifically target cells and DNA. It shows promising use in the treatment of cancer and in gene therapy. Eastern European scientists have used phage therapy as an alternative to antibiotics for some time, and interest in this approach is increasing, because of the high level of antibiotic resistance now found in some pathogenic bacteria.\\nThe expression of heterologous proteins by viruses is the basis of several manufacturing processes that are currently being used for the production of various proteins such as vaccine antigens and antibodies. Industrial processes have been recently developed using viral vectors and several pharmaceutical proteins are currently in pre-clinical and clinical trials.\\n\\n\\n==== Virotherapy ====\\n\\nVirotherapy involves the use of genetically modified viruses to treat diseases. Viruses have been modified by scientists to reproduce in cancer cells and destroy them but not infect healthy cells. Talimogene laherparepvec (T-VEC), for example, is a modified herpes simplex virus that has had a gene, which is required for viruses to replicate in healthy cells, deleted and replaced with a human gene (GM-CSF) that stimulates immunity. When this virus infects cancer cells, it destroys them and in doing so the presence the GM-CSF gene attracts dendritic cells from the surrounding tissues of the body. The dendritic cells process the dead cancer cells and present components of them to other cells of the immune system. Having completed successful clinical trials, the virus gained approval for the treatment of melanoma in late 2015. Viruses that have been reprogrammed to kill cancer cells are called oncolytic viruses.\\n\\n\\n=== Materials science and nanotechnology ===\\nCurrent trends in nanotechnology promise to make much more versatile use of viruses. From the viewpoint of a materials scientist, viruses can be regarded as organic nanoparticles.\\nTheir surface carries specific tools that enable them to cross the barriers of their host cells. The size and shape of viruses and the number and nature of the functional groups on their surface are precisely defined. As such, viruses are commonly used in materials science as scaffolds for covalently linked surface modifications. A particular quality of viruses is that they can be tailored by directed evolution. The powerful techniques developed by life sciences are becoming the basis of engineering approaches towards nanomaterials, opening a wide range of applications far beyond biology and medicine.Because of their size, shape, and well-defined chemical structures, viruses have been used as templates for organising materials on the nanoscale. Recent examples include work at the Naval Research Laboratory in Washington, D.C., using Cowpea mosaic virus (CPMV) particles to amplify signals in DNA microarray based sensors. In this application, the virus particles separate the fluorescent dyes used for signalling to prevent the formation of non-fluorescent dimers that act as quenchers. Another example is the use of CPMV as a nanoscale breadboard for molecular electronics.\\n\\n\\n=== Synthetic viruses ===\\nMany viruses can be synthesised de novo (\"from scratch\").  The first synthetic virus was created in 2002. Although somewhat of a misconception, it is not the actual virus that is synthesised, but rather its DNA genome (in case of a DNA virus), or a cDNA copy of its genome (in case of RNA viruses). For many virus families the naked synthetic DNA or RNA (once enzymatically converted back from the synthetic cDNA) is infectious when introduced into a cell. That is, they contain all the necessary information to produce new viruses. This technology is now being used to investigate novel vaccine strategies. The ability to synthesise viruses has far-reaching consequences, since viruses can no longer be regarded as extinct, as long as the information of their genome sequence is known and permissive cells are available. As of June 2021, the full-length genome sequences of 11,464 different viruses, including smallpox, are publicly available in an online database maintained by the National Institutes of Health.\\n\\n\\n=== Weapons ===\\n\\nThe ability of viruses to cause devastating epidemics in human societies has led to the concern that viruses could be weaponised for biological warfare. Further concern was raised by the successful recreation of the infamous 1918 influenza virus in a laboratory.\\nThe smallpox virus devastated numerous societies throughout history before its eradication. There are only two centres in the world authorised by the WHO to keep stocks of smallpox virus: the State Research Center of Virology and Biotechnology VECTOR in Russia and the Centers for Disease Control and Prevention in the United States. It may be used as a weapon, as the vaccine for smallpox sometimes had severe side-effects, it is no longer used routinely in any country. Thus, much of the modern human population has almost no established resistance to smallpox and would be vulnerable to the virus.\\n\\n\\n== See also ==\\n\\n\\n== References ==\\n\\n\\n=== Notes ===\\n\\n\\n=== Bibliography ===\\n\\n\\n== External links ==\\n Media related to Viruses at Wikimedia Commons\\n Data related to Virus at Wikispecies\\nViralZone A Swiss Institute of Bioinformatics resource for all viral families, providing general molecular and epidemiological information',\n",
              " 'There are many variants of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus that causes coronavirus disease 2019 (COVID-19). Some are believed, or have been stated, to be of particular importance due to their potential for increased transmissibility, increased virulence, or reduced effectiveness of vaccines against them. These variants contribute to the continuation of the COVID-19 pandemic.\\nFive SARS-CoV-2 variants have been designated as variants of concern by the World Health Organization: the Alpha, Beta, Gamma, Delta, and Omicron variants.\\n\\n\\n== Overview ==\\n\\nThe emergence of SARS-CoV-2 may have resulted from recombination events between a bat SARS-like coronavirus and a pangolin coronavirus through cross-species transmission. The earliest available SARS-CoV-2 viral genomes were collected from patients in December 2019, and Chinese researchers compared these early genomes with bat and pangolin coronavirus strains to estimate the ancestral human coronavirus type; the identified ancestral genome type was labeled \"S\", and its dominant derived type was labeled \"L\" to reflect the mutant amino acid changes. Independently, Western researchers carried out similar analyses but labeled the ancestral type \"A\" and the derived type \"B\". The B-type mutated into further types including B.1, which is the ancestor of the major global variants of concern, labeled in 2021 by the WHO as alpha, beta, gamma, delta and omicron variants.Early in the pandemic, the relatively low number of infections (compared with later stages of the pandemic) resulted in fewer opportunities for mutation of the viral genome and, therefore, fewer opportunities for the occurrence of differentiated variants. Since the occurrence of variants was rarer, the observation of S-protein mutations in the receptor-binding domain (RBD) region interacting with ACE2 was also not frequent.As time went on, the evolution of SARS-CoV-2\\'s genome (by means of random mutations) led to mutant specimens of the virus (i.e., genetic variants), observed to be more transmissible, to be naturally selected. Notably, both the Alpha and the Delta variants were observed to be more transmissible than previously identified viral strains.Some SARS-CoV-2 variants are considered to be of concern as they maintain (or even increase) their replication fitness in the face of rising population immunity, either by infection recovery or via vaccination. Some of the variants of concern show mutations in the RBD of the S-protein.The following table presents information and relative risk level for currently and formerly circulating variants of concern (VOC). The intervals assume a 95% confidence or credibility level, unless otherwise stated. Currently, all estimates are approximations due to the limited availability of data for studies. For Alpha, Beta, Gamma and Delta, there is no change in test accuracy, and neutralising antibody activity is retained by some monoclonal antibodies. PCR tests continue to detect the Omicron variant.\\n  Very high risk   High risk   Medium risk   Low risk   Unknown risk\\n\\n\\n== Nomenclature ==\\n\\nSARS-CoV-2 variants are grouped according to their lineage and component mutations. As of July 2021, no consistent nomenclature was established for it. Many organisations, including governments and news outlets, referred colloquially to concerning variants by the country in which they were first identified. After months of discussions, the World Health Organization announced Greek-letter names for important strains on 31 May 2021, so they could be easily referred to in a simple, easy to say, and non-stigmatising fashion. This decision may have partially been taken because of criticism from governments on using country names to refer to variants of the virus; the WHO mentioned the potential for mentioning country names to cause stigma. After using all the letters from Alpha to Mu (see below), in November 2021 the WHO skipped the next two letters of the Greek alphabet, Nu and Xi, and used Omicron, prompting speculation that Xi was skipped to avoid offending Chinese President Xi Jinping. The WHO gave as the explanation that Nu is too easily confounded with \"new\" and Xi is a common last name. In the event that the WHO uses the entirety of the Greek alphabet, the agency considered naming future variants after constellations.\\n\\n\\n=== Lineages and clades ===\\nWhile there are many thousands of variants of SARS-CoV-2, subtypes of the virus can be put into larger groupings such as lineages or clades. Three main, generally used nomenclatures have been proposed:\\n\\nAs of January 2021, GISAID—referring to SARS-CoV-2 as hCoV-19—had identified eight global clades (S, O, L, V, G, GH, GR, and GV).\\nIn 2017, Hadfield et al. announced Nextstrain, intended \"for real-time tracking of pathogen evolution\". Nextstrain has later been used for tracking SARS-CoV-2, identifying 13 major clades (19A–B, 20A–20J and 21A) as of June 2021.\\nIn 2020, Rambaut et al. of the Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) software team proposed in an article \"a dynamic nomenclature for SARS-CoV-2 lineages that focuses on actively circulating virus lineages and those that spread to new locations\"; as of August 2021, 1340 lineages had been designated.Each national public health institute may also institute its own nomenclature system for the purposes of tracking specific variants. For example, Public Health England designated each tracked variant by year, month and number in the format [YYYY] [MM]/[NN], prefixing \\'VUI\\' or \\'VOC\\' for a variant under investigation or a variant of concern respectively. This system has now been modified and now uses the format [YY] [MMM]-[NN], where the month is written out using a three-letter code.\\n\\n\\n=== Classification of variants ===\\nVariants that appear to meet one or more specific criteria considered during the COVID-19 pandemic may be labeled \"variants of interest\" or \"variants under investigation\" (\\'VUI\\') pending verification and validation of these properties. Once validated, variants of interest /VUI  may be renamed \"variants of concern\" by monitoring organizations, such as the CDC in the US. A related category is \"variant of high consequence\", used by the CDC if there is clear evidence that the effectiveness of prevention or intervention measures for a particular variant is substantially reduced.\\n\\n\\n== Reference sequence ==\\nAs it is currently not known when the index case or \"patient zero\" occurred, the choice of reference sequence for a given study is relatively arbitrary, with different notable research studies\\' choices varying as follows:\\n\\nThe earliest sequence, Wuhan-1, was collected on 24 December 2019.\\nOne group (Sudhir Kumar et al.) refers extensively to an NCBI reference genome (GenBankID:NC_045512; GISAID ID: EPI_ISL_402125), this sample was collected on 26 December 2019, although they also used the WIV04 GISAID reference genome (ID: EPI_ISL_402124), in their analyses.\\nAccording to another source (Zhukova et al.), the sequence WIV04/2019, belonging to the GISAID S clade / PANGO A lineage / Nextstrain 19B clade, is thought to most closely reflect the sequence of the original virus infecting humans—known as \"sequence zero\". WIV04/2019 was sampled from a symptomatic patient on 30 December 2019 and is widely used (especially by those collaborating with GISAID) as a reference sequence.The variant first sampled and identified in Wuhan, China is considered by researchers to differ from the progenitor genome by three mutations. Subsequently, many distinct lineages of SARS-CoV-2 have evolved.\\n\\n\\n== Notability criteria ==\\nViruses generally acquire mutations over time, giving rise to new variants. When a new variant appears to be growing in a population, it can be labelled as an \"emerging variant\". In the case of SARS-CoV-2, new lineages often differ from one another by just a few nucleotides.Some of the potential consequences of emerging variants are the following:\\nIncreased transmissibility\\nIncreased morbidity\\nIncreased mortality\\nAbility to evade detection by diagnostic tests\\nDecreased susceptibility to antiviral drugs (if and when such drugs are available)\\nDecreased susceptibility to neutralising antibodies, either therapeutic (e.g., convalescent plasma or monoclonal antibodies) or in laboratory experiments\\nAbility to evade natural immunity (e.g., causing reinfections)\\nAbility to infect vaccinated individuals\\nIncreased risk of particular conditions such as multisystem inflammatory syndrome or long COVID.\\nIncreased affinity for particular demographic or clinical groups, such as children or immunocompromised individuals.Variants that appear to meet one or more of these criteria may be labelled \"variants under investigation\" or \"variants of interest\" pending verification and validation of these properties. The primary characteristic of a variant of interest is that it shows evidence that demonstrates it is the cause of an increased proportion of cases or unique outbreak clusters; however, it must also have limited prevalence or expansion at national levels, or the classification would be elevated to a \"variant of concern\". If there is clear evidence that the effectiveness of prevention or intervention measures for a particular variant is substantially reduced, that variant is termed a \"variant of high consequence\".\\n\\n\\n== Variants of concern (WHO) ==\\nListed below are the Variants of Concern (VOC) recognised by the World Health Organization as of June 2021. Other organisations such as the CDC in the United States have at times used a slightly different list. As of July 2021, their list matched that of the WHO.\\n\\n\\n=== Alpha (lineage B.1.1.7) ===\\n\\nFirst detected in October 2020 during the COVID-19 pandemic in the United Kingdom from a sample taken the previous month in Kent, lineage B.1.1.7, labelled Alpha variant by the WHO, was previously known as the first Variant Under Investigation in December 2020 (VUI – 202012/01) and later notated as VOC-202012/01. It is also known as 20I (V1), 20I/501Y.V1 (formerly 20B/501Y.V1), or 501Y.V1. From October to December 2020, its prevalence doubled every 6.5 days, the presumed generational interval. It is correlated with a significant increase in the rate of COVID-19 infection in United Kingdom, associated partly with the N501Y mutation. There was some evidence that this variant had 40–80% increased transmissibility (with most estimates lying around the middle to higher end of this range), and early analyses suggested an increase in lethality, though later work found no evidence of increased virulence. As of May 2021, the Alpha variant had been detected in some 120 countries.\\n\\n\\n==== B.1.1.7 with E484K ====\\nVariant of Concern 21FEB-02 (previously written as VOC-202102/02), described by Public Health England (PHE) as \"B.1.1.7 with E484K\" is of the same lineage in the Pango nomenclature system, but has an additional E484K mutation. As of 17 March 2021, there were 39 confirmed cases of VOC-21FEB-02 in the UK. On 4 March 2021, scientists reported B.1.1.7 with E484K mutations in the state of Oregon. In 13 test samples analysed, one had this combination, which appeared to have arisen spontaneously and locally, rather than being imported. Other names for this variant include B.1.1.7+E484K and B.1.1.7 Lineage with S:E484K.\\n\\n\\n=== Beta (lineage B.1.351) ===\\n\\nOn 18 December 2020, the 501.V2 variant, also known as 501.V2, 20H (V2), 20H/501Y.V2 (formerly 20C/501Y.V2), 501Y.V2, VOC-20DEC-02 (formerly VOC-202012/02), or lineage B.1.351, was first detected in South Africa and reported by the country\\'s health department. It has been labelled as Beta variant by WHO. Researchers and officials reported that the prevalence of the variant was higher among young people with no underlying health conditions, and by comparison with other variants it is more frequently resulting in serious illness in those cases. The South African health department also indicated that the variant may be driving the second wave of the COVID-19 epidemic in the country due to the variant spreading at a more rapid pace than other earlier variants of the virus.Scientists noted that the variant contains several mutations that allow it to attach more easily to human cells because of the following three mutations in the receptor-binding domain (RBD) in the spike glycoprotein of the virus: N501Y, K417N, and E484K. The N501Y mutation has also been detected in the United Kingdom.\\n\\n\\n=== Gamma (lineage P.1) ===\\n\\nThe Gamma variant or lineage P.1, termed Variant of Concern 21JAN-02 (formerly VOC-202101/02) by Public Health England, 20J (V3) or 20J/501Y.V3 by Nextstrain, or just 501Y.V3, was detected in Tokyo on 6 January 2021 by the National Institute of Infectious Diseases (NIID). It has been labelled as Gamma variant by WHO. The new variant was first identified in four people who arrived in Tokyo having travelled from the Brazilian Amazonas state on 2 January 2021. On 12 January 2021, the Brazil-UK CADDE Centre confirmed 13 local cases of the new Gamma variant in the Amazon rainforest. This variant of SARS-CoV-2 has been named lineage P.1 (although it is a descendant of B.1.1.28, the name B.1.1.28.1 is not permitted and thus the resultant name is P.1), and has 17 unique amino acid changes, 10 of which in its spike protein, including the three concerning mutations: N501Y, E484K and K417T.:\\u200aFigure 5\\u200aThe N501Y and E484K mutations favour the formation of a stable RBD-hACE2 complex, thus, enhancing the binding affinity of RBD to hACE2. However, the K417T mutation disfavours complex formation between RBD and hACE2, which has been demonstrated to reduce the binding affinity.The new variant was absent in samples collected from March to November 2020 in Manaus, Amazonas state, but it was detected for the same city in 42% of the samples from 15 to 23 December 2020, followed by 52.2% during 15–31 December and 85.4% during 1–9 January 2021. A study found that infections by Gamma can produce nearly ten times more viral load compared to persons infected by one of the other lineages identified in Brazil (B.1.1.28 or B.1.195). Gamma also showed 2.2 times higher transmissibility with the same ability to infect both adults and older persons, suggesting P.1 and P.1-like lineages are more successful at infecting younger humans irrespective of sex.A study of samples collected in Manaus between November 2020 and January 2021, indicated that the Gamma variant is 1.4–2.2 times more transmissible and was shown to be capable of evading 25–61% of inherited immunity from previous coronavirus diseases, leading to the possibility of reinfection after recovery from an earlier COVID-19 infection. As for the fatality ratio, infections by Gamma were also found to be 10–80% more lethal.A study found that people fully vaccinated with Pfizer or Moderna have significantly decreased neutralisation effect against Gamma, although the actual impact on the course of the disease is uncertain.\\nA pre-print study by the Oswaldo Cruz Foundation published in early April found that the real-world performance of people with the initial dose of the Sinovac\\'s Coronavac Vaccine had approximately 50% efficacy rate. They expected the efficacy to be higher after the 2nd dose. As of July 2021, the study is ongoing.Preliminary data from two studies indicate that the Oxford–AstraZeneca vaccine is effective against the Gamma variant, although the exact level of efficacy has not yet been released. Preliminary data from a study conducted by Instituto Butantan suggest that CoronaVac is effective against the Gamma variant as well, and as of July 2021 has yet to be expanded to obtain definitive data.\\n\\n\\n=== Delta (lineage B.1.617.2) ===\\n\\nThe Delta variant, also known as B.1.617.2, G/452R.V3, 21A or 21A/S:478K, is a globally dominant variant that spread to at least 185 countries. It was first discovered in India. Descendant of lineage B.1.617, which also includes the Kappa variant under investigation, it was first discovered in October 2020 and has since spread internationally. On 6 May 2021, British scientists declared B.1.617.2 (which notably lacks mutation at E484Q) as a \"variant of concern\", labelling it VOC-21APR-02, after they flagged evidence that it spreads more quickly than the original version of the virus and could spread quicker or as quickly as Alpha. It carries L452R and P681R mutations in Spike; unlike Kappa it carries T478K but not E484Q.\\nOn 3 June 2021, Public Health England reported that twelve of the 42 deaths from the Delta variant in England were among the fully vaccinated, and that it was spreading almost twice as fast as the Alpha variant. Also on 11 June, Foothills Medical Centre in Calgary, Canada reported that half of their 22 cases of the Delta variant occurred among the fully vaccinated.In June 2021, reports began to appear of a variant of Delta with the K417N mutation. The mutation, also present in the Beta and Gamma variants, raised concerns about the possibility of reduced effectiveness of vaccines and antibody treatments and increased risk of reinfection. The variant, called \"Delta with K417N\" by Public Health England, includes two clades corresponding to the Pango lineages AY.1 and AY.2. It has been nicknamed \"Delta plus\" from \"Delta plus K417N\". The name of the mutation, K417N, refers to an exchange whereby lysine (K) is replaced by asparagine (N) at position 417. On 22 June, India\\'s Ministry of Health and Family Welfare declared the \"Delta plus\" variant of COVID-19 a Variant of Concern after 22 cases of the variant were reported in India. After the announcement, leading virologists said there was insufficient data to support labelling the variant as a distinct variant of concern, pointing to the small number of patients studied. In the UK in July 2021, AY.4.2 was identified. Alongside those previously mentioned it also gained the nickname \\'Delta Plus\\', on the strength of its extra mutations, Y145H and A222V. These are not unique to it, but distinguish it from the original Delta variant.\\n\\n\\n=== Omicron ===\\n\\n\\n==== Lineage B.1.1.529 ====\\nThe Omicron variant, known as lineage B.1.1.529, was declared a variant of concern by the World Health Organization on 26 November 2021.The variant has a large number of mutations, of which some are concerning. The number of cases in the B.1.1.529 lineage is increasing in all areas of South Africa. Some evidence shows that this variant has an increased risk of reinfection. Studies are underway to evaluate the exact impact on transmissibility, mortality, and other factors.Named Omicron by the WHO, it was identified in November 2021 in Botswana and South Africa; one case had travelled to Hong Kong, one confirmed case was identified in Israel in a traveler returning from Malawi, along with two who returned from South Africa and one from Madagascar. Belgium confirmed the first detected case in Europe on 26 November 2021 in an individual who had returned from Egypt on 11 November. Indian SARS-CoV-2 Genomics Consortium (INSACOG) in its January 2022 bulletin noted that Omicron is in community transmission in India where new cases have been rising exponentially.\\n\\n\\n===== Sublineages (including BA.2) =====\\nAccording to the WHO, BA.1, BA.1.1, and BA.2 are the most common sublineages of Omicron globally as of February 2022. BA.2 contains 28 unique genetic changes, including four in its spike protein, compared to BA.1, which had already acquired 60 mutations since the ancestral Wuhan strain, including 32 in the spike protein. BA.2 is more transmissible than BA.1. It was causing most cases in England by mid-March 2022, and by the end of March, BA.2 became dominant in the US.\\n\\n\\n== Variants of interest (WHO) ==\\nListed below are the Variants of Interest (VOI) which are, as of August 2021, recognised by the World Health Organization. Other organisations such as the CDC in the United States may at times use a slightly different list.\\n\\n\\n=== Lambda (lineage C.37) ===\\n\\nThe Lambda variant, also known as lineage C.37, was first detected in Peru in August 2020 and was designated by the WHO as a variant of interest on 14 June 2021. It spread to at least 30 countries around the world and, as of July 2021, it is unknown whether it is more infectious and resistant to vaccines than other strains.\\n\\n\\n=== Mu (lineage B.1.621) ===\\n\\nThe Mu variant, also known as lineage B.1.621, was first detected in Colombia in January 2021 and was designated by the WHO as a variant of interest on 30 August 2021. There have been outbreaks in South America and Europe.\\n\\n\\n== Former variants of interest ==\\n\\n\\n=== Epsilon (lineages B.1.429, B.1.427, CAL.20C) ===\\n\\nThe Epsilon variant or lineage B.1.429, also known as CAL.20C or CA VUI1, 21C or 20C/S:452R, is defined by five distinct mutations (I4205V and D1183Y in the ORF1ab gene, and S13I, W152C, L452R in the spike protein\\'s S-gene), of which the L452R (previously also detected in other unrelated lineages) was of particular concern. From 17 March to 29 June 2021, the CDC listed B.1.429 and the related B.1.427 as \"variants of concern\". As of July 2021, Epsilon is no longer considered a variant of interest by the WHO, as it was overtaken by Alpha.From September 2020 to January 2021, it was 19% to 24% more transmissible than earlier variants in California. Neutralisation against it by antibodies from natural infections and vaccinations was moderately reduced, but it remained detectable in most diagnostic tests.Epsilon (CAL.20C) was first observed in July 2020 by researchers at the Cedars-Sinai Medical Center, California, in one of 1,230 virus samples collected in Los Angeles County since the start of the COVID-19 epidemic. It was not detected again until September when it reappeared among samples in California, but numbers remained very low until November. In November 2020, the Epsilon variant accounted for 36 per cent of samples collected at Cedars-Sinai Medical Center, and by January 2021, the Epsilon variant accounted for 50 per cent of samples. In a joint press release by University of California, San Francisco, California Department of Public Health, and Santa Clara County Public Health Department, the variant was also detected in multiple counties in Northern California. From November to December 2020, the frequency of the variant in sequenced cases from Northern California rose from 3% to 25%. In a preprint, CAL.20C is described as belonging to clade 20C and contributing approximately 36% of samples, while an emerging variant from the 20G clade accounts for some 24% of the samples in a study focused on Southern California. Note, however, that in the US as a whole, the 20G clade predominates, as of January 2021. Following the increasing numbers of Epsilon in California, the variant has been detected at varying frequencies in most US states. Small numbers have been detected in other countries in North America, and in Europe, Asia and Australia. After an initial increase, its frequency rapidly dropped from February 2021 as it was being outcompeted by the more transmissible Alpha. In April, Epsilon remained relatively frequent in parts of northern California, but it had virtually disappeared from the south of the state and had never been able to establish a foothold elsewhere; only 3.2% of all cases in the United States were Epsilon, whereas more than two-thirds were Alpha.\\n\\n\\n=== Zeta (lineage P.2) ===\\n\\nZeta variant or lineage P.2, a sub-lineage of B.1.1.28 like Gamma (P.1), was first detected in circulation in the state of Rio de Janeiro; it harbours the E484K mutation, but not the N501Y and K417T mutations. It evolved independently in Rio de Janeiro without being directly related to the Gamma variant from Manaus. Though previously Zeta was labeled a variant of interest, as of July 2021, it is no longer considered as such by the WHO.\\n\\n\\n=== Eta (lineage B.1.525) ===\\n\\nThe Eta variant or lineage B.1.525, also called VUI-21FEB-03 (previously VUI-202102/03) by Public Health England (PHE) and formerly known as UK1188, 21D or 20A/S:484K, does not carry the same N501Y mutation found in Alpha, Beta and Gamma, but carries the same E484K-mutation as found in the Gamma, Zeta, and Beta variants, and also carries the same ΔH69/ΔV70 deletion (a deletion of the amino acids histidine and valine in positions 69 and 70) as found in Alpha, N439K variant (B.1.141 and B.1.258) and Y453F variant (Cluster 5). Eta differs from all other variants by having both the E484K-mutation and a new F888L mutation (a substitution of phenylalanine (F) with leucine (L) in the S2 domain of the spike protein). As of 5 March 2021, it had been detected in 23 countries. It has also been reported in Mayotte, the overseas department/region of France. The first cases were detected in December 2020 in the UK and Nigeria, and as of 15 February 2021, it had occurred in the highest frequency among samples in the latter country. As of 24 February 56 cases were found in the UK. Denmark, which sequences all its COVID-19 cases, found 113 cases of this variant from 14 January to 21 February 2021, of which seven were directly related to foreign travel to Nigeria.As of July 2021, UK experts are studying it to ascertain how much of a risk it could be. It is currently regarded as a \"variant under investigation\", but pending further study, it may become a \"variant of concern\". Ravi Gupta, from the University of Cambridge said in a BBC interview that lineage B.1.525 appeared to have \"significant mutations\" already seen in some of the other newer variants, which means their likely effect is to some extent more predictable.\\n\\n\\n=== Theta (lineage P.3) ===\\n\\nOn 18 February 2021, the Department of Health of the Philippines confirmed the detection of two mutations of COVID-19 in Central Visayas after samples from patients were sent to undergo genome sequencing. The mutations were later named as E484K and N501Y, which were detected in 37 out of 50 samples, with both mutations co-occurrent in 29 out of these.On 13 March, the Department of Health confirmed the mutations constitutes a variant which was designated as lineage P.3. On the same day, it also confirmed the first COVID-19 case caused by the Gamma variant in the country. The Philippines had 98 cases of the Theta variant on 13 March. On 12 March it was announced that Theta had also been detected in Japan. On 17 March, the United Kingdom confirmed its first two cases, where PHE termed it VUI-21MAR-02.\\nOn 30 April 2021, Malaysia detected 8 cases of the Theta variant in Sarawak.As of July 2021, Theta is no longer considered a variant of interest by the WHO.\\n\\n\\n=== Iota (lineage B.1.526) ===\\n\\nIn November 2020, a mutant variant was discovered in New York City, which was named lineage B.1.526. As of 11 April 2021, the variant has been detected in at least 48 U.S. states and 18 countries. In a pattern mirroring Epsilon, Iota was initially able to reach relatively high levels in some states, but by May 2021 it was outcompeted by the more transmissible Delta and Alpha.\\n\\n\\n=== Kappa (lineage B.1.617.1) ===\\n\\nThe Kappa variant is one of the three sublineages of lineage B.1.617. It is also known as lineage B.1.617.1, 21B or 21A/S:154K, and was first detected in India in December 2020. By the end of March 2021, Kappa accounted for more than half of the sequences being submitted from India. On 1 April 2021, it was designated a variant under investigation (VUI-21APR-01) by Public Health England. It has the notable mutations L452R, E484Q, P681R.\\n\\n\\n== Variants under monitoring (WHO) ==\\nDefined as variants with genetic changes suspected to affect virus characteristics and some indication of posing a future risk, but with unclear evidence of phenotypic or epidemiological impact, requiring enhanced monitoring and repeat assessment after new evidence. Some former variants of interest are monitored as well.\\n\\n\\n== Formerly monitored variants (WHO) ==\\n\\n\\n== Other notable variants ==\\nLineage B.1.1.207 was first sequenced in August 2020 in Nigeria; the implications for transmission and virulence are unclear but it has been listed as an emerging variant by the US Centers for Disease Control. Sequenced by the African Centre of Excellence for Genomics of Infectious Diseases in Nigeria, this variant has a P681H mutation, shared in common with the Alpha variant. It shares no other mutations with the Alpha variant and as of late December 2020 this variant accounts for around 1% of viral genomes sequenced in Nigeria, though this may rise. As of May 2021, lineage B.1.1.207 has been detected in 10 countries.Lineage B.1.1.317, while not considered a variant of concern, is noteworthy in that Queensland Health forced 2 people undertaking hotel quarantine in Brisbane, Australia to undergo an additional 5 days\\' quarantine on top of the mandatory 14 days after it was confirmed they were infected with this variant.Lineage B.1.616, being identified in Brittany, Western France in early January 2021 and designated by WHO as \"Variant under investigation\" in March 2021, was reported to be difficult to detect from nasopharyngeal swab sampling method of coronavirus detection, and detection of the virus needs to rely on samples from lower respiratory tract.Lineage B.1.618 was first isolated in October 2020. It has the E484K mutation in common with several other variants, and showed significant spread in April 2021 in West Bengal, India. As of 23 April 2021, the PANGOLIN database showed 135 sequences detected in India, with single-figure numbers in each of eight other countries worldwide.In July 2021, scientists reported in a preprint which was published in a journal in February 2022, the detection of anomalous unnamed unknown-host SARS-CoV-2 lineages via wastewater surveillance in New York City. They hypothesized that \"these lineages are derived from unsampled human COVID-19 infections or that they indicate the presence of a non-human animal reservoir\". Lineage B.1.640.2 (also known as the IHU variant) was detected in October 2021 by researchers at the Institut Hospitalo-Universitaire (IHU) in Marseille. They found the variant in a traveler who returned to France from Cameroon and reportedly infected 12 people. The B.1.640 lineage, which includes B.1.640.2, was designated a variant under monitoring (VUM) by the World Health Organization (WHO) on 22 November 2021. However, the WHO has reported that lineage B.1.640.2 has spread much slower than the Omicron variant, and so is of relatively little concern.  According to a preprint study, lineage B.1.640.2 has two already known spike protein mutations – E484K and N501Y – among a total of 46 nucleotide substitutions and 37 deletions.\\n\\n\\n== Notable missense mutations ==\\nThere have been a number of missense mutations observed of SARS-CoV-2.\\n\\n\\n=== del 69-70 ===\\nThe name of the mutation, del 69-70, or 69-70 del, or other similar notations, refers to the deletion of amino acid at position 69 to 70. The mutation is found in the Alpha variant, and could lead to \"spike gene target failure\" and result in false negative result in PCR virus test.\\n\\n\\n=== RSYLTPGD246-253N ===\\nOtherwise referred to as del 246-252, or other various similar expression, refer to the deletion of amino acid from the position of 246 to 252, in the N-terminal domain of spike protein, accompanied with a replacement of the aspartic acid (D) at the position 253 for asparagine (N).The 7 amino acid deletion mutation is currently described as unique in the Lambda variant, and have been attributed to as one of the cause of the strain\\'s increased capability to escape from neutralizing antibodies according to preprint paper.\\n\\n\\n=== N440K ===\\nThe name of the mutation, N440K, refers to an exchange whereby the asparagine (N) is replaced by lysine (K) at position 440.This mutation has been observed in cell cultures to be 10 times more infective compared to the previously widespread A2a strain (A97V substitution in RdRP sequence) and 1000 times more in the lesser widespread A3i strain (D614G substitution in Spike and a and P323L substitution in RdRP). It was involved in rapid surges of COVID-19 cases in India in May 2021. India has the largest proportion of N440K mutated variants followed by the US and Germany.\\n\\n\\n=== G446V ===\\nThe name of the mutation, G446V, refers to an exchange whereby the glycine (G) is replaced by valine (V) at position 446.The mutation, identified in Japan among inbound travelers starting from May, and among 33 samples from individuals related to 2020 Tokyo Olympic Games and 2020 Tokyo Paralympic Games, are said to be possible to impact affinity of multiple monoclonal antibody, although its clinical impact against the use of antibody medicine is still yet to be known.\\n\\n\\n=== L452R ===\\nThe name of the mutation, L452R, refers to an exchange whereby the leucine (L) is replaced by arginine (R) at position 452.L452R is found in both the Delta and Kappa variants which first circulated in India, but have since spread around the world. L452R is a relevant mutation in this strain that enhances ACE2 receptor binding ability and can reduce vaccine-stimulated antibodies from attaching to this altered spike protein.\\nL452R, some studies show, could even make the coronavirus resistant to T cells, that are class of cells necessary to target and destroy virus-infected cells. They are different from antibodies that are useful in blocking coronavirus particles and preventing it from proliferating.\\n\\n\\n=== Y453F ===\\nThe name of the mutation, Y453F, refers to an exchange whereby the tyrosine (Y) is replaced by phenylalanine (F) at position 453. The mutation have been found potentially linked to the spread of SARS-CoV-2 among minks in the Netherlands in 2020.\\n\\n\\n=== S477G/N ===\\nA highly flexible region in the receptor binding domain (RBD) of SARS-CoV-2, starting from residue 475 and continuing up to residue 485, was identified using bioinformatics and statistical methods in several studies. The University of Graz and the Biotech Company Innophore have shown in a recent publication that structurally, the position S477 shows the highest flexibility among them.At the same time, S477 is hitherto the most frequently exchanged amino acid residue in the RBDs of SARS-CoV-2 mutants. By using molecular dynamics simulations of RBD during the binding process to hACE2, it has been shown that both S477G and S477N strengthen the binding of the SARS-COV-2 spike with the hACE2 receptor. The vaccine developer BioNTech referenced this amino acid exchange as relevant regarding future vaccine design in a preprint published in February 2021.\\n\\n\\n=== E484Q ===\\nThe name of the mutation, E484Q, refers to an exchange whereby the glutamic acid (E) is replaced by glutamine (Q) at position 484.The Kappa variant circulating in India has E484Q. These variants were initially (but misleadingly) referred to as a \"double mutant\". E484Q may enhance ACE2 receptor binding ability, and may reduce vaccine-stimulated antibodies\\' ability to attach to this altered spike protein.\\n\\n\\n=== E484K ===\\nThe name of the mutation, E484K, refers to an exchange whereby the glutamic acid (E) is replaced by lysine (K) at position 484. It is nicknamed \"Eeek\".E484K has been reported to be an escape mutation (i.e., a mutation that improves a virus\\'s ability to evade the host\\'s immune system) from at least one form of monoclonal antibody against SARS-CoV-2, indicating there may be a \"possible change in antigenicity\". The Gamma variant (lineage P.1), the Zeta variant (lineage P.2, also known as lineage B.1.1.28.2) and the Beta variant (501.V2) exhibit this mutation. A limited number of lineage B.1.1.7 genomes with E484K mutation have also been detected. Monoclonal and serum-derived antibodies are reported to be from 10 to 60 times less effective in neutralising virus bearing the E484K mutation. On 2 February 2021, medical scientists in the United Kingdom reported the detection of E484K in 11 samples (out of 214,000 samples), a mutation that may compromise current vaccine effectiveness.\\n\\n\\n=== F490S ===\\nF490S denotes a change from phenylalanine (F) to serine (S) in amino-acid position 490.It is one of the mutation found in Lambda, and have been associated with reduced susceptibility to antibody generated by those who were infected with other strains, meaning antibody treatment against people infected with strains carrying such mutation would be less effective.\\n\\n\\n=== N501Y ===\\nN501Y denotes a change from asparagine (N) to tyrosine (Y) in amino-acid position 501. N501Y has been nicknamed \"Nelly\".This change is believed by PHE to increase binding affinity because of its position inside the spike glycoprotein\\'s receptor-binding domain, which binds ACE2 in human cells; data also support the hypothesis of increased binding affinity from this change. Molecular interaction modelling and the free energy of binding calculations has demonstrated that the mutation N501Y has the highest binding affinity in variants of concern RBD to hACE2. Variants with N501Y include Gamma, Alpha (VOC 20DEC-01), Beta, and COH.20G/501Y (identified in Columbus, Ohio). This last became the dominant form of the virus in Columbus in late December 2020 and January and appears to have evolved independently of other variants.\\n\\n\\n=== N501S ===\\nN501S denotes a change from asparagine (N) to serine (S) in amino-acid position 501.As of September 2021, there are 8 cases of patients around the world infected with Delta variant which feature this N501S mutation. As it is considered a mutation similar to N501Y, it is suspected to have similar characteristics as N501Y mutation, which is believed to increase the infectivity of the virus, however the exact effect is unknown yet.\\n\\n\\n=== D614G ===\\n\\nD614G is a missense mutation that affects the spike protein of SARS-CoV-2. From early appearances in Eastern China early in 2020, the frequency of this mutation in the global viral population increased early on during the pandemic. G (glycine) quickly replaced D (aspartic acid) at position 614 in Europe, though more slowly in China and the rest of East Asia, supporting the hypothesis that G increased the transmission rate, which is consistent with higher viral titres and infectivity in vitro. Researchers with the PANGOLIN tool nicknamed this mutation \"Doug\".In July 2020, it was reported that the more infectious D614G SARS-CoV-2 variant had become the dominant form in the pandemic. PHE confirmed that the D614G mutation had a \"moderate effect on transmissibility\" and was being tracked internationally.The global prevalence of D614G correlates with the prevalence of loss of smell (anosmia) as a symptom of COVID-19, possibly mediated by higher binding of the RBD to the ACE2 receptor or higher protein stability and hence higher infectivity of the olfactory epithelium.Variants containing the D614G mutation are found in the G clade by GISAID and the B.1 clade by the PANGOLIN tool.\\n\\n\\n=== Q677P/H ===\\nThe name of the mutation, Q677P/H, refers to an exchange whereby the glutamine (Q) is replaced by proline (P) or histidine (H) at position 677. There are several sub-lineages containing the Q677P mutation; six of these, which also contain various different combinations of other mutations, are referred to by names of birds. One of the earlier ones noticed for example is known as \"Pelican,\" while the most common of these as of early 2021 was provisionally named \"Robin 1.\"The mutation has been reported in multiple lineages circulating inside the United States as of late 2020 and also some lineages outside the country. \\'Pelican\\' was first detected in Oregon, and as of early 2021 \\'Robin 1\\' was found often in the Midwestern United States, while another Q667H sub-lineage, \\'Robin 2\\', was found mostly in the southeastern United States. The frequency of such mutation being recorded has increased from late 2020 to early 2021.\\n\\n\\n=== P681H ===\\n\\nThe name of the mutation, P681H, refers to an exchange whereby the proline (P) is replaced by histidine (H) at position 681.In January 2021, scientists reported in a preprint that the mutation P681H, a characteristic feature of the Alpha variant and lineage B.1.1.207 (identified in Nigeria), is showing a significant exponential increase in worldwide frequency, thus following a trend to be expected in the lower limb of the logistics curve. This may be compared with the trend of the now globally prevalent D614G.\\n\\n\\n=== P681R ===\\nThe name of the mutation, P681R, refers to an exchange whereby the proline (P) is replaced by arginine (R) at position 681.Indian SARS-CoV-2 Genomics Consortium (INSACOG) found that other than the two mutations E484Q and L452R, there is also a third significant mutation, P681R in lineage B.1.617. All three concerning mutations are on the spike protein, the operative part of the coronavirus that binds to receptor cells of the body.\\n\\n\\n=== A701V ===\\nAccording to initial media reports, the Malaysian Ministry of Health announced on 23 December 2020 that it had discovered a mutation in the SARS-CoV-2 genome which they designated as A701B(sic), among 60 samples collected from the Benteng Lahad Datu cluster in Sabah. The mutation was characterised as being similar to the one found recently at that time in South Africa, Australia, and the Netherlands, although it was uncertain if this mutation was more infectious or aggressive than before. The provincial government of Sulu in neighbouring Philippines temporarily suspended travel to Sabah in response to the discovery of \\'A701B\\' due to uncertainty over the nature of the mutation.On 25 December 2020, the Malaysian Ministry of Health described a mutation A701V as circulating and present in 85% of cases (D614G was present in 100% of cases) in Malaysia. These reports also referred to samples collected from the Benteng Lahad Datu cluster. The text of the announcement was mirrored verbatim on the Facebook page of Noor Hisham Abdullah, Malay Director-General of Health, who was quoted in some of the news articles.The A701V mutation has the amino acid alanine (A) substituted by valine (V) at position 701 in the spike protein. Globally, South Africa, Australia, Netherlands and England also reported A701V at about the same time as Malaysia. In GISAID, the prevalence of this mutation is found to be about 0.18%. of cases.On 14 April 2021, the Malaysian Ministry of Health reported that the third wave, which had started in Sabah, has involved the introduction of variants with D614G and A701V mutations.\\n\\n\\n== Recombinant variants ==\\nThe British government has reported a number of recombinant variants of SARS-CoV-2. These recombinant lineages have been given the Pango lineage identifiers XD, XE, and XF.XE is a recombinant lineage of Pango lineages BA.1 and BA.2. As of March 2022 XE was believed to have a growth rate 9.8% greater than BA.2.\\n\\n\\n== Differential vaccine effectiveness ==\\n\\nThe interplay between the SARS-CoV-2 virus and its human hosts was initially natural but then started being altered by the rising availability of vaccines seen in 2021. The potential emergence of a SARS-CoV-2 variant that is moderately or fully resistant to the antibody response elicited by the COVID-19 vaccines may necessitate modification of the vaccines. The emergence of vaccine-resistant variants is more likely in a highly vaccinated population with uncontrolled transmission.As of February 2021, the US Food and Drug Administration believed that all FDA authorized vaccines remained effective in protecting against circulating strains of SARS-CoV-2.\\n\\n\\n== Data and methods ==\\nModern DNA sequencing, where available, may permit rapid detection (sometimes known as \\'real-time detection\\') of genetic variants that appear in pathogens during disease outbreaks. Through use of phylogenetic tree visualisation software, records of genome sequences can be clustered into groups of identical genomes all containing the same set of mutations. Each group represents a \\'variant\\', \\'clade\\', or \\'lineage\\', and comparison of the sequences allows the evolutionary path of a virus to be deduced. For SARS-CoV-2, until  March 2021, over 330,000 viral genomic sequences had been generated by molecular epidemiology studies across the world.\\n\\n\\n=== New variant detection and assessment ===\\nOn 26 January 2021, the British government said it would share its genomic sequencing capabilities with other countries in order to increase the genomic sequencing rate and trace new variants, and announced a \"New Variant Assessment Platform\". As of January 2021, more than half of all genomic sequencing of COVID-19 was carried out in the UK.Wastewater surveillance was demonstrated to be one technique to detect SARS-CoV-2 variants and to track their rise for studying related ongoing infection dynamics.\\n\\n\\n=== Testing ===\\nWhether one or more mutations visible in RT-PCR tests can be used reliably to identify a variant depends on the prevalence of other variants currently circulating in the same population.\\n\\n\\n== Incubation theory for multiple mutated variants ==\\n\\nResearchers have suggested that multiple mutations can arise in the course of the persistent infection of an immunocompromised patient, particularly when the virus develops escape mutations under the selection pressure of antibody or convalescent plasma treatment, with the same deletions in surface antigens repeatedly recurring in different patients.\\n\\n\\n== Cross-species transmission ==\\n\\nThere is a risk that COVID-19 could transfer from humans to other animal populations and could combine with other animal viruses to create yet more variants that are dangerous to humans. Reverse zoonosis spillovers may cause reservoirs for mutating variants that spill back to humans – another possible source for variants of concern, in addition to immunocompromised people.\\n\\n\\n=== Cluster 5 ===\\n\\nIn early November 2020, Cluster 5, also referred to as ΔFVI-spike by the Danish State Serum Institute (SSI), was discovered in Northern Jutland, Denmark. It is believed to have been spread from minks to humans via mink farms. On 4 November 2020, it was announced that the mink population in Denmark would be culled to prevent the possible spread of this mutation and reduce the risk of new mutations happening. A lockdown and travel restrictions were introduced in seven municipalities of Northern Jutland to prevent the mutation from spreading, which could compromise national or international responses to the COVID-19 pandemic. By 5 November 2020, some 214 mink-related human cases had been detected.The WHO stated that cluster 5 had a \"moderately decreased sensitivity to neutralising antibodies\". SSI warned that the mutation could reduce the effect of COVID-19 vaccines under development, although it was unlikely to render them useless. Following the lockdown and mass-testing, SSI announced on 19 November 2020 that cluster 5 in all probability had become extinct. As of 1 February 2021, authors to a peer-reviewed paper, all of whom were from the SSI, assessed that cluster 5 was not in circulation in the human population.\\n\\n\\n== See also ==\\n\\nRaTG13, the second closest known relative to SARS-CoV-2\\nPandemic prevention § Surveillance and mapping\\nCOVID-19 vaccine#Effectiveness\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\n\\n\\n== External links ==\\nCoVariants\\nCov-Lineages\\nGISAID – hCov19 Variants',\n",
              " 'The Gamma variant (P.1) is one of the variants of SARS-CoV-2, the virus that causes COVID-19. This variant of SARS-CoV-2 has been named lineage P.1 and has 17 amino acid substitutions, ten of which in its spike protein, including these three designated to be of particular concern: N501Y, E484K and K417T. This variant of SARS-CoV-2 was first detected by the National Institute of Infectious Diseases (NIID) of Japan, on 6 January 2021 in four people who had arrived in Tokyo having visited Amazonas, Brazil, four days earlier. It was subsequently declared to be in circulation in Brazil. Under the simplified naming scheme proposed by the World Health Organization, P.1 has been labeled Gamma variant, and is currently considered a variant of concern.Gamma caused widespread infection in early 2021 in the city of Manaus, the capital of Amazonas, although the city had already experienced widespread infection in May 2020, with a study indicating high seroprevalence of antibodies for SARS-CoV-2. A research article published in Science Journal indicate that P.1 infected people have a greater chance of transmissibility and death than B.1.1.28 infected ones.The Gamma variant comprises the two distinct subvariants 28-AM-1 and 28-AM-2, which both carry the K417T, E484K, N501Y mutations, and which both developed independently of each other within the same Brazilian Amazonas region.Gamma is notably different from the Zeta variant (lineage P.2) which also circulated strongly in Brazil. In particular, Zeta only carries the E484K mutation and has neither of the other two mutations of concern, N501Y and K417T.\\n\\n\\n== Classification ==\\nInitial reports claimed that both P.1 and P.2 were two separate and different descendants of the Brazilian lineage B.1.1.248. However, B.1.1.248 later lost its status as a distinct lineage and was reclassified to B.1.1.28. P.1 has also been called B.1.1.28.1, while P.2 has been B.1.1.28.2 or VUI-202101/01. Since only three sublevels are permitted in the PANGO Lineage system of nomenclature, hence the designation of B.1.1.28.1 to P.1 and B.1.1.28.2 to P.2.Following its detection, genome data for four samples of the new variant were shared to GISAID having been assigned the ID range: EPI_ISL_792680 to EPI_ISL_792683.\\n\\n\\n=== Mutations ===\\n\\nVariants of SARS-CoV-2\\nAs well as having eight mutations (four of these synonymous genetic mutations) in its open reading frames (ORF1a and ORF1b) – one of which is a set of deletions – Gamma has 10 defining mutations in its spike protein, including N501Y and E484K. It also has two mutations – one an insertion – in its ORF8 gene and one in its N gene.\\n\\n\\n=== Descendant and sublineages ===\\nCoronavirus lineage B.1.1.28 has originated four known lineages classified as variant of interest (VOI) or variant of concern (VOC): lineages P.1, P.2, P.3 and P.4.\\nLineage P.2 (B.1.1.28.2, Zeta variant), first detected in October 2020 in the state of Rio de Janeiro, Brazil, only shares one mutation of concern with P.1, which is the E484K. The other P.2 mutations are without concern and rarely found for other variants. The five P.2-specific mutations are: E484K in S-gene, A119S in N-gene, 5’UTR C100U, plus L3468V and synC11824U in ORF1ab-gene. Other mutations commonly found in P.2 are: 3’UTR C29754U,  F120F (synC28253U) in ORF8, M234I in the N-gene, plus L3930F and synA12964G in ORF1ab.Lineage P.3 (Theta variant) was first identified in the Philippines on 18 February 2021 when two mutations of concern were detected in Central Visayas.The remaining B.1.1.28 derivative virus is lineage P.4. Although researchers have not identified its precise origin, it was first sequenced in Itirapina, Brazil, and was already circulating in various municipalities in the state of São Paulo of the same country. It carries a mutation of concern in the spike protein called L452R which is also present in lineage B.1.617 (Delta and Kappa variants) detected in India, Epsilon variant (lineages B.1.427 and B.1.429) from California, United States. The branch of this lineage is P.4.1 (VUI-NP13L)—suspected to have arisen in Goiás, Brazil, around June–July 2020— also rapidly spread to the southeast of the country, where for example Taquara had its first genome sequence, and to the northeast of the nation. It was detected internationally, with reported cases in Japan, Netherlands and England. The P.4.1 has V1176F and D614G mutations in spike protein.\\n\\n\\n== Prevention ==\\n\\n\\n== Statistics ==\\n\\n\\n== History ==\\nOn 12 January 2021, the Brazil–United Kingdom CADDE Centre confirmed 13 local cases of lineage P.1 in Manaus, Amazonas state, the largest city of the Amazon rain forest. The new lineage was absent in 27 samples collected from March to November 2020 from Manaus, but it was identified for the same city in 42% (n=13/31) of the samples collected 15–23 December 2020, followed by 52.2% (n=35/67) during 15–31 December 2020 and 85.4% (n=41/48) during 1–9 January 2021. Most notably, the P.2 was rapidly outcompeted by P1 going from the second half of December to 1–9 January, where the lineage P.2 share for Manaus decreased from 25.4% to 6.3%.A study of 180 sequenced Brazilian samples collected in the state of Rio de Janeiro during 2020, identified emergence of the novel lineage P.2 of SARS-CoV-2 (originating from B.1.1.28). P.2 was first detected by genome sequencing in October 2020, but it was estimated to have emerged in early July 2020. As of December 2020, although having significantly increased in frequency throughout the state, it was still largely confined to the state capital Rio de Janeiro. In May 2020 the main lineages behind the COVID-19 positives were B.1.1.33 (70%) and B.1.1.28 (20%), whereas by September the main lineages were B.1.1.33 (50%) and B.1.1.28 (40%), with no detected presence of P.2, while during October and November P.2 was the most common lineage with a share close to 50% (according to the Pangolin tool). The study also found the E484K mutation as \"widely spread\" across all analysed P.2 samples (36 out of 38).Researchers at the Oswaldo Cruz Foundation published a preprint genomic epidemiology study of 250 collected genomes from different places in Amazonas and found that P.1 infections can produce nearly 10 times more viral load than in other COVID-19-infected persons involving lineages B.1.1.28 and B.1.195. The lineage also showed 2.2 times higher transmissibility with the same ability to infect both adults (18–59 years old) and older persons (60 years old and higher), suggesting P.1 and its sublineages are more successful at infecting younger humans with no gender differential.The Centre for Arbovirus Discovery, Diagnosis, Genomics and Epidemiology (CADDE) produced another journal article of samples collected in Manaus between November 2020 and January 2021. The study indicated lineage P.1 to be about 2.0 times (50% CrI, 1.7–2.4 times) more transmissible and was shown to be capable of evading about 32% (50% CrI, 21–46%) of inherited immunity from previous coronavirus diseases, leading to the possibility of reinfection. These increased statistics also had the same reflection in fatality, in that P.1 infections can be about 50% (50% CrI, 20–90%) more lethal. As part of ongoing research, the variant\\'s capacity to neutralise antibodies has been evaluated by scientists in a published preprint work demonstrating that 8 CoronaVac-immunised persons had a poor blood plasma response against lineage P.1. Since the study only had a small number of participants, it was not possible to establish any statistical conclusion as a larger number of vaccinated people would need to be studied. Scientists at MIT, Harvard and Cambridge, and hospitals physicians in Boston, corroborated that people fully vaccinated  with Pfizer and Moderna vaccines have significantly decreased neutralisation with P.1—in a preprint work.\\n\\n\\n== See also ==\\n\\nVariants of SARS-CoV-2: Alpha, Beta, Delta, Epsilon, Zeta, Eta, Theta, Iota, Kappa, Lambda, Mu, Omicron\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== External links ==\\nPANGO lineages: New variant report -Report on global distribution of 3 variants including P.1, a descendent of B.1.1.28\\nCOG-UK Report on SARS-CoV-2 Spike mutations of interest in the UK',\n",
              " 'Angiotensin-converting enzyme 2 (ACE2) is an enzyme that can be found either attached to the membrane of cells (mACE2) in the intestines, kidney, testis, gallbladder, and heart or in a soluble form (sACE2). Both membrane bound and soluble ACE2 are integral parts of the renin–angiotensin–aldosterone system (RAAS) that exists to keep the body\\'s blood pressure in check. While mACE2 does not appear to factor into the harmful phase of RAAS (the increase of blood pressure), its existence is vital in order for the enzyme ADAM17 to cleave its extracellular domain to create soluble ACE2 (sACE2). Soluble ACE2 lowers blood pressure by catalyzing the hydrolysis of angiotensin II (a vasoconstrictor peptide) into angiotensin (1–7)  (a vasodilator) which in turns binds to MasR receptors creating localized vasodilation and hence decreasing blood pressure. This decrease in blood pressure makes the entire process a promising drug target for treating cardiovascular diseases.mACE2 also serves as the entry point into cells for some coronaviruses, including  HCoV-NL63, SARS-CoV, and SARS-CoV-2. The SARS-CoV-2 spike protein itself is known to damage the endothelium via downregulation of ACE2. The human version of the enzyme can be referred to as hACE2.\\n\\n\\n== Structure ==\\nMembrane bound Angiotensin-converting enzyme 2 (mACE2) is a zinc-containing metalloenzyme located on the surface of intestinal enterocytes, renal tubular cells and other cells. mACE2 protein contains an N-terminal peptidase M2 domain and a C-terminal collectrin renal amino acid transporter domain.\\nmACE2 is a single-pass type I membrane protein, with its enzymatically active domain exposed on the surface of cells in the intestines and other tissues. The extracellular domain of mACE2 can be cleaved from the transmembrane domain by another enzyme known as ADAM17 a member of the sheddase enzyme family, during the protective phase of RAAS, the Renin Angiotension Aldosterone System which regulates our body\\'s blood pressure. The resulting cleaved protein is known as soluble ACE2 or sACE2. It is released into the bloodstream where one of sACE2\\'s functions is to turn excess angiotensin II into angiotensin 1-7 which binds to MasR receptors creating localized vasodilation and hence decreasing blood pressure. Excess sACE2 may ultimately be excreted in the urine.\\n\\n\\n== Location within the human body ==\\nmACE2 is attached to the cell membrane of mainly enterocytes of the small intestine and duodenum, proximal tubular cells of the kidneys, glandular cells of the gallbladder, as well as Sertoli cells and Leydig cells of the testis. The expression profile of mACE2 in the human body was recently thoroughly evaluated by the Human Protein Atlas team using a large-scale multiomics approach combining several different methods for analysis of gene expression, including a stringent immunohistochemical analysis using two independent antibodies. mACE2 expression was in addition to the above-mentioned tissues, also seen in endothelial cells and pericytes of blood vessels within certain tissues, cardiomyocytes in heart tissue, and a smaller subset of cells within the thyroid gland, epididymis, seminal vesicle, pancreas, liver and placenta. Despite the fact that the respiratory system is the primary route of SARS-CoV-2 infection, very limited expression is seen, both at protein and mRNA level. The expression within the respiratory system is mainly restricted to the upper bronchial and nasal epithelia, especially in the ciliated cells.\\n\\n\\n== Function ==\\nAs part of the renin–angiotensin–aldosterone system (RAAS) protective phase, soluble ACE2\\'s (sACE2) important function is to act as a counterbalance to the angiotensin-converting enzyme (ACE). ACE cleaves angiotensin I hormone into the vasoconstricting  angiotensin II which causes a cascade of hormonal reactions which is part of the body\\'s harmful phase of RAAS, which ultimately leads to an increase in the body\\'s blood pressure.  ACE2 has an opposing effect to ACE, degrading angiotensin II into angiotensin (1-7), thereby lowering blood pressure.sACE2, as part of RAAS\\'s protective phase, cleaves the carboxyl-terminal amino acid phenylalanine from angiotensin II (Asp-Arg-Val-Tyr-Ile-His-Pro-Phe) and hydrolyses it into the vasodilator angiotensin (1-7) (H-Asp-Arg-Val-Tyr-Ile-His-Pro-OH), which binds to Mas Receptors and ultimately leads to a decrease in blood pressure.  sACE2 can also cleave numerous peptides, including [des-Arg9]-bradykinin, apelin, neurotensin, dynorphin A, and ghrelin.mACE2 also regulates the membrane trafficking of the neutral amino acid transporter SLC6A19 and has been implicated in Hartnup\\'s disease.Research in mice has shown that ACE2 (whether it is the membrane bound version or soluble is inconclusive) is involved in regulation of the blood glucose level but its mechanism is yet to be confirmed.\\n\\n\\n== Coronavirus entry point ==\\nAs a transmembrane protein, mACE2 serves as the main entry point into cells for some coronaviruses, including HCoV-NL63, SARS-CoV (the virus that causes SARS), and SARS-CoV-2 (the virus that causes COVID-19).  More specifically, the binding of the spike S1 protein of SARS-CoV and SARS-CoV-2 to the enzymatic domain of mACE2 on the surface of cells results in endocytosis and translocation of both the virus and the enzyme into endosomes located within cells. The binding of the SARS-CoV-2 virus through mACE2 receptors present in heart tissue may be responsible for direct viral injury leading to myocarditis. In a study done during the SARS outbreak, SARS virus RNA was ascertained in the autopsy of heart specimens in 35% of the patients who died due to SARS. It was also observed that an already diseased heart has increased expression of mACE2 receptors contrasted to healthy individuals. This entry process also requires priming of the S protein by the host serine protease TMPRSS2, the inhibition of which is under current investigation as a potential therapeutic. It has also been shown that disruption of S-protein glycosylation significantly impairs viral entry, indicating the importance of glycan-protein interactions in the process. Amino acid variation in the ACE2 protein is able to affect the affinity of binding with the S protein of SARS-CoV-2.This has led some to hypothesize that decreasing the levels of mACE2, in cells, might help in fighting the infection.  Furthermore, according to studies conducted on mice, the interaction of the spike protein of the coronavirus with mACE2 induces a drop in the levels of mACE2 in cells through internalization and degradation of the protein and hence may contribute to lung damage.On the other hand, sACE2 has been shown to have a protective effect against virus-induced lung injury by increasing the production of the vasodilator angiotensin 1–7. Furthermore, some researchers have hypothesized that sACE2 (which is created during the Protective Phase of RAAS) is not only involved in binding to Angiotensin II to create Angiotensin I-7 which lowers blood pressure by vasodilation, but that free and soluble ACE2 may also be binding to coronavirus spike proteins, hence making those coronavirus spikes unavailable for binding to mACE-2 sites. But even with only tiny amounts of mACE2, SARS-CoV-2 virus can gain entry into cells if TMPRSS2 is present.Both ACE inhibitors and angiotensin II receptor blockers (ARBs) that are used to treat high blood pressure have been shown in rodent studies to upregulate mACE2 expression, possibly affecting the severity of coronavirus infections.However, a systematic review and meta-analysis published on July 11, 2012, found that \"use of ACE inhibitors was associated with a significant 34% reduction in risk of pneumonia compared with controls.\" Moreover, \"the risk of pneumonia was also reduced in patients treated with ACE inhibitors who were at higher risk of pneumonia, in particular those with stroke and heart failure. Use of ACE inhibitors was also associated with a reduction in pneumonia related mortality, although the results were less robust than for overall risk of pneumonia.\"  An April 2020 study of patients hospitalized in Hubei Province in China found a death rate of 3.7% for patients suffering from hypertension who were taking ACE inhibitors or ARBs. The death rate was compared with 9.8% of hospitalized patients with hypertension not taking such drugs, suggesting that ACE inhibitors and ARBs are not harmful and may help against the coronavirus.Despite lack of conclusive evidence, some have advocated for and against the cessation of ACE inhibitor or ARB treatment in COVID-19 patients with hypertension. However, multiple professional societies and regulatory bodies have recommended continuing standard ACE inhibitor and ARB therapy.Plasma ACE2 levels predict outcome of COVID-19 in hospitalized patients, with higher plasma levels being correlated with worse disease outcomes. Patients with high blood pressure or heart disease show elevated ACE2 plasma levels.\\n\\n\\n== Recombinant human ACE2 ==\\nRecombinant human ACE2 (rhACE2) is surmised to be a novel therapy for acute lung injury, and appeared to improve pulmonary blood flow and oxygen saturation in piglets with a lipopolysaccharide-induced acute respiratory distress syndrome. The half-life of rhACE2 in human beings is about 10 hours, and the onset of action is 30 minutes in addition to the course of effect (duration) of 24 hours.  Several findings suggest that rhACE2 may be a promising drug for those with intolerance to classic renin-angiotensin system inhibitors (RAS inhibitors) or in diseases where circulating angiotensin II is elevated.An in vitro study focused on the early stages of infection found that clinical-grade human recombinant soluble ACE2 (hrsACE2) reduced SARS-CoV-2 recovery from vero cells by a factor of 1,000-5,000. The equivalent mouse rsACE2 did not have such an effect. This study suggests that rhsACE2 not only restores the renin-angiotensin system to balance as in the earlier ARDS studies, but also directly slows down infection by this virus – possibly as a decoy.Infused rhACE2 has been evaluated in clinical trials for the treatment of acute respiratory distress syndrome (ARDS). rhACE2 is in phase II trial for severe COVID-19.\\n\\n\\n== See also ==\\nRenin–angiotensin system\\nAngiotensin-converting enzyme\\nSARS-CoV-2 § Structural biology\\n\\n\\n== References ==\\n\\n\\n== External links ==\\nHuman ACE2 genome location and ACE2 gene details page  in the UCSC Genome Browser.\\nAngiotensin-converting enzyme 2 in Membranome database\\n3D structure of complex of a neurotransmitter sodium symporter B(0)AT1, ACE2, and SARS-CoV-2  receptor-binding domain in OPM database\\nOverview of all the structural information available in the PDB for UniProt: Q9BYF1 (Angiotensin-converting enzyme 2) at the PDBe-KB.',\n",
              " 'The Delta variant (B.1.617.2) is a variant of SARS-CoV-2, the virus that causes COVID-19. It was first detected in India in late 2020. The Delta variant was named on 31 May 2021 and had spread to over 179 countries by 22 November 2021. The World Health Organization (WHO) indicated in June 2021 that the Delta variant was becoming the dominant strain globally.It has mutations in the gene encoding the SARS-CoV-2 spike protein causing the substitutions T478K, P681R and L452R, which are known to affect transmissibility of the virus as well as whether it can be neutralised by antibodies for previously circulating variants of the COVID-19 virus. It is thought to be one of the most transmissible respiratory viruses known. In August 2021, Public Health England (PHE) reported secondary attack rate in household contacts of non-travel or unknown cases for Delta to be 10.8% vis-à-vis 10.2% for the Alpha variant; the case fatality rate for those 386,835 people with Delta is 0.3%, where 46% of the cases and 6% of the deaths are unvaccinated and below 50 years old. Immunity from previous recovery or COVID-19 vaccines are effective in preventing severe disease or hospitalisation from infection with the variant.On 7 May 2021, PHE changed their classification of lineage B.1.617.2 from a variant under investigation (VUI) to a variant of concern (VOC) based on an assessment of transmissibility being at least equivalent to B.1.1.7 (Alpha variant); the UK\\'s SAGE using May data estimated a \"realistic\" possibility of being 50% more transmissible. On 11 May 2021, the WHO also classified this lineage VOC, and said that it showed evidence of higher transmissibility and reduced neutralisation. On 15 June 2021, the Centers for Disease Control and Prevention (CDC) declared Delta a variant of concern.The variant is thought to be partly responsible for India\\'s deadly second wave of the pandemic beginning in February 2021. It later contributed to a third wave in Fiji, the United Kingdom and South Africa, and the WHO warned in July 2021 that it could have a similar effect elsewhere in Europe and Africa. By late July, it had also driven an increase in daily infections in parts of Asia, the United States, Australia, and New Zealand. Yet another COVID variant, Omicron, followed.\\n\\n\\n== Classification ==\\nThe Delta variant has mutations in the gene encoding the SARS-CoV-2 spike protein causing the substitutions D614G, T478K, P681R and L452R. It is identified as the 21A, 21I, and 21J clades under the Nextstrain phylogenetic classification system.\\n\\n\\n=== Names ===\\nThe virus has also been referred to by the term \"Indian Variant\" as it was originally detected in India. However, the Delta variant is only one of three variants of the lineage B.1.617, all of which were first detected in India. At the end of May 2021, the WHO assigned the label Delta to lineage B.1.617.2 after introducing a new policy of using Greek letters for variants of concern and variants of interest.\\n\\n\\n=== Other sublineages of B.1.617 ===\\nThere are three sublineages of lineage B.1.617 categorised so far.\\nB.1.617.1 (Kappa variant) was designated a Variant Under Investigation in April 2021 by Public Health England. Later in April 2021, two other variants B.1.617.2 and B.1.617.3 were designated as Variants Under Investigation. While B.1.617.3 shares the L452R and E484Q mutations found in B.1.617.1, B.1.617.2 lacks the E484Q mutation. B.1.617.2 has the T478K mutation, not found in B.1.617.1 and B.1.617.3. Simultaneously, the ECDC released a brief maintaining all three sublineages of B.1.617 as VOI, estimating that a \"greater understanding of the risks related to these B.1.617 lineages is needed before any modification of current measures can be considered\".\\n\\n\\n=== Mutations ===\\n\\nThe Delta/ B.1.617.2 genome has 13 mutations (15 or 17 according to some sources, depending on whether more common mutations are included) which produce alterations in the amino-acid sequences of the proteins it encodes.The list of spike protein mutations is: 19R, (G142D), Δ156-157, R158G, L452R, T478K, D614G, P681R, D950N  according to GVN, or T19R, G142D, del 156-157, R158G, L452R, T478K, D614G, P681R according to Genscript  Four of them, all of which are in the virus\\'s spike protein code, are of particular concern:\\n\\nD614G. The substitution at position 614, an aspartic acid-to-glycine substitution, is shared with other highly transmissible variants like Alpha, Beta and Gamma.\\nT478K. The exchange at position 478 is a threonine-to-lysine substitution.\\nL452R. The substitution at position 452, a leucine-to-arginine substitution, confers stronger affinity of the spike protein for the ACE2 receptor and decreased recognition capability of the immune system.\\nP681R. The substitution at position 681, a proline-to-arginine substitution, which, according to William A. Haseltine, may boost cell-level infectivity of the variant \"by facilitating cleavage of the S precursor protein to the active S1/S2 configuration\".The E484Q mutation is not present in the B.1.617.2 genome.\\n\\n\\n=== Lineages ===\\nAs of August 2021, Delta variants have been subdivided in the Pango lineage designation system into variants from AY.1 to AY.28. However, there is no information on whether such classification correlates with biological characteristic changes of the virus. It is said that, as of August 2021, AY.4 to AY.11 are predominant in the UK, AY.12 in Israel, AY.2, AY.3, AY.13, AY.14, AY.25 in the US, AY.20 in the US and Mexico, AY.15 in Canada, AY.16 in Kenya, AY.17 in Ireland and Northern Ireland, AY.19 in South Africa, AY.21 in Italy and Switzerland, AY.22 in Portugal, AY.24 in Indonesia, and AY.23 in Indonesia, Singapore, Japan, and South Korea.\\n\\n\\n==== \"Delta plus\" variant ====\\n\\nDelta with K417N originally corresponded to lineages AY.1 and AY.2, subsequently also lineage AY.3, and has been nicknamed \"Delta plus\" or \"Nepal variant\". It has the K417N mutation, which is also present in the Beta variant. The exchange at position 417 is a lysine-to-asparagine substitution.As of mid-October 2021, the AY.3 variant accounted for a cumulative prevalence of approximately 5%  in the United States, and 2% worldwide. In mid-October the AY.4.2 Delta sublineage was expanding in England, and being monitored and assessed. It contains mutations A222V and Y145H in its spike protein, not considered of particular concern. It has been suggested that AY.4.2 might be 10-15% more transmissible than the original Delta variant. Mid-October 2021, AY.4.2 accounted for an estimated 10% of cases, and has led to an additional growth rate rising to about 1% (10% of 10%) per generational time of five days or so. This additional growth rate would grow with increasing prevalence. Without AY.4.2 and no other changes, the number of cases in the UK would have been about 10% lower. AY.4.2 grows about 15% faster per week. In the UK it was reclassified as a \"variant under investigation\" (but not \"of concern\") in late October 2021. In Denmark, after a drop in AY.4.2 cases, a new fast surge was detected and monitored, but was not yet considered a cause of concern.\\n\\n\\n== Symptoms ==\\n\\nThe most common symptoms may have changed from the most common symptoms previously associated with standard COVID-19. Infected people may mistake the symptoms for a bad cold and not realize they need to isolate. Common symptoms reported have been headaches, sore throat, a runny nose or a fever. In the United Kingdom, when the Delta variant accounted for 91 percent of new cases, one study found that the most reported symptoms were headache, sore throat, and runny nose.\\n\\n\\n== Prevention ==\\n\\nWHO has not issued preventative measures against Delta specifically; non-pharmaceutical measures recommended to prevent wild type COVID-19 should still be effective.  These would include washing hands, wearing a mask, maintaining distance from others, avoiding touching the mouth, nose or eyes, avoiding crowded indoor spaces with poor ventilation especially where people are talking, going to get tested if one develops symptoms and isolating if one becomes sick.  Public Health authorities should continue to find infected individuals using testing, trace their contacts, and isolate those who have tested positive or been exposed.  Event organizers should assess the potential risks of any mass gathering and develop a plan to mitigate these risks.  See also Non-pharmaceutical intervention (epidemiology).\\nICMR found that convalescent sera of the COVID-19 cases and recipients of Bharat Biotech\\'s BBV152 (Covaxin) were able to neutralise VUI B.1.617 although with a lower efficacy.Anurag Agrawal, the director of the Institute of Genomics and Integrative Biology (IGIB), said the study on the effectiveness of the available vaccines on lineage B.1.617 suggests that post-vaccination, the infections are milder.\\nAnthony Fauci, the Chief Medical Advisor to the President of the United States, has also expressed his confidence regarding the preliminary results. In an interview on 28 April, he said: This is something where we\\'re still gaining data daily. But the most recent data was looking at convalescent sera of COVID-19 cases and people who received the vaccine used in India, the Covaxin. It was found to neutralise the 617 variants.\\nAnother study by the Centre for Cellular and Molecular Biology (CCMB) in Hyderabad found Covishield (Oxford–AstraZeneca) vaccinated sera offers protection against lineage B.1.617.A study conducted by the Public Health England, has found that compared to those who were unvaccinated those who were vaccinated with either the Pfizer-BioNTech or AstraZeneca-Oxford had 33% less instances of symptomatic disease caused by the variant after the first dose. Among those who were two weeks after the receiving their second dose of the Pfizer-BioNTech vaccine 88% less subjects had symptomatic disease from the Delta variant versus those that were unvaccinated. Among those who were two weeks after the receiving their second dose of the AstraZeneca-Oxford vaccine 60% less subjects had symptomatic disease from the Delta variant versus those that were unvaccinated.A study by a group of researchers from the Francis Crick Institute, published in The Lancet, shows that humans fully vaccinated with the Pfizer-BioNTech vaccine are likely to have more than five times lower levels of neutralizing antibodies against the Delta variant compared to the original COVID-19 strain.In June 2021, Public Health England announced it had conducted a study which found that after two shots, the Pfizer-BioNTech vaccine and the AstraZeneca vaccine are respectively 96% and 92% effective at preventing hospitalisation from the Delta variant.On July 3, researchers from the universities of Toronto and Ottawa in Ontario, Canada, released a preprint study suggesting that the Moderna vaccine may be effective against death or hospitalization from the Delta variant.In a study of the University of Sri Jayewardenepura in July 2021 found the Sinopharm BIBP vaccine vaccine caused seroconversion in 95% of individuals studied that had received both doses of the vaccine. The rate was higher in 20-39 age group (98.9%) but slightly lower in the over 60 age group (93.3%). Neutralising antibodies were present among 81.25% of the vaccinated individuals studied.On 29 June 2021, the director of the Gamaleya Institute, Denis Logunov, said that Sputnik V is about 90% effective against the Delta variant.On July 21, researchers from Public Health England published a study finding that the Pfizer vaccine was 93.7% effective against symptomatic disease from Delta after 2 doses, while the Astrazeneca vaccine was 67% effective.On August 2, several experts expressed concern that achieving herd immunity may not currently be possible because the Delta variant is transmitted among those immunized with current vaccines.On August 10, a study showed that the full vaccination coverage rate is correlated inversely to the SARS-CoV-2 delta variant mutation frequency in 16 countries (R-squared=0.878). Data strongly indicates that full vaccination against COVID-19 may slow down virus evolution.\\n\\n\\n== Treatment ==\\n\\nIn vitro experiments suggest that bamlanivimab may not be effective against Delta on its own.  At high enough concentrations, casirivimab, etesevimab and imdevimab appear to still be effective.  A preprint study suggests that sotrovimab may also be effective against Delta.  Doctors in Singapore have been using supplemental oxygen, remdesivir and corticosteroids on more Delta patients than they did on previous variants.\\n\\n\\n== Epidemiology ==\\n\\n\\n=== Transmissibility ===\\nUK scientists have said that the Delta variant is between 40% and 60% more transmissible than the previously dominant Alpha variant, which was first identified in the UK (as the Kent variant). Given that Alpha is already 150% as transmissible as the original SARS-CoV-2 strain that emerged in late 2019 in Wuhan, and if Delta is 150% as transmissible as Alpha, then Delta may be 225% as transmissible as the original strain. BBC reported that \\n  \\n    \\n      \\n        \\n          R\\n          \\n            0\\n          \\n        \\n      \\n    \\n    {\\\\displaystyle R_{0}}\\n   – basic reproduction number, or the expected number of cases directly generated by one case in a population where all individuals are susceptible to infection – for the first detected SARS-CoV-2 virus is 2.4-2.6, whereas Alpha\\'s reproduction number is 4-5 and Delta\\'s is 5–9. These basic reproduction numbers can be compared to MERS (0.29-0.80), seasonal influenza (1.2-1.4), Ebola (1.4-1.8), common cold (2-3), SARS (2–4), smallpox (3.5-6), and chickenpox (10-12). Due to Delta\\'s high transmissibility even those that are vaccinated are vulnerable, albeit to a lesser extent.A study published online (not peer-reviewed) by Guangdong Provincial Center for Disease Control and Prevention may partly explain the increased transmissibility: people with infection caused by Delta had 1,000 times more copies of the virus in the respiratory tracts than those with infection caused by variants first identified in the beginning of the pandemic; and it took on average 4 days for people infected with Delta for the virus to be detectable compared to 6 days with initially identified variants.Surveillance data from the U.S., Germany and the Netherlands indicates the Delta variant is growing by about a factor of 4 every two weeks with respect to the Alpha variant.In India, the United Kingdom, Portugal, Russia, Mexico, Australia, Indonesia, South Africa, Germany, Luxembourg, the United States, the Netherlands, Denmark, France and probably many other countries, the Delta variant had become the dominant strain by July 2021. Depending on country, there is typically a lag from a few days to several weeks between cases and variant reporting.  As of July 20, this variant had spread to 124 countries, and WHO had indicated that it was becoming the dominant strain, if not one already.In the Netherlands, the virus was still able to propagate significantly in the population with over 93.4% of blood donors being tested positive for SARS-CoV-2 antibodies after week 28, 2021. Many people there are not fully vaccinated, so those antibodies would have been developed from exposure to the wild virus or from a vaccine. Similar high seroimmunity levels occur in the United Kingdom in blood donors and general surveillance.A preprint found that the viral load in the first positive test of infections with the variant was on average ∼1000 times higher than with compared infections during 2020. Preliminary data from a study with 100,000 volunteers in the UK from May to July 2021, when Delta was spreading rapidly, indicates that vaccinated people who test positive for COVID-19, including asymptomatic cases, have a lower viral load in average. Data from the US, UK, and Singapore indicate that vaccinated people infected by Delta may have viral loads as high as unvaccinated infected people, but might remain infectious for a shorter period.\\n\\n\\n=== Infection age groups ===\\nSurveillance data from the Indian government\\'s Integrated Disease Surveillance Programme (IDSP) shows that around 32% of patients, both hospitalised and outside hospitals, were aged below 30 in the second wave compared to 31% during the first wave, among people aged 30–40 the infection rate stayed at 21%. Hospitalisation in the 20-39 bracket increased to 25.5% from 23.7% while the 0-19 range increased to 5.8% from 4.2%. The data also showed a higher proportion of asymptomatic patients were admitted during the second wave, with more complaints of breathlessness.\\n\\n\\n=== Virulence ===\\n\\nA few early studies suggest the Delta variant causes more severe illness than other strains. On 7 June 2021, researchers at the National Centre for Infectious Diseases in Singapore posted a paper suggesting that patients testing positive for Delta are more likely to develop pneumonia and/or require oxygen than patients with wild type or Alpha. On June 11, Public Health England released a report finding that there was \"significantly increased risk of hospitalization\" from Delta as compared with Alpha; the risk was approximately twice as high for those infected with the Delta variant. On June 14, researchers from Public Health Scotland found that the risk of hospitalization from Delta was roughly double that of from Alpha. On July 7, a preprint study from epidemiologists at the University of Toronto found that Delta had a 120% greater – or more than twice as large – risk of hospitalization, 287% greater risk of ICU admission and 137% greater risk of death compared to non-variant of concern strains of SARS-COV-2. However, on July 9, Public Health England reported that the Delta variant in England had a case fatality rate (CFR) of 0.2%, while the Alpha variant\\'s case fatality rate was 1.9%, although the report warns that \"case fatality rates are not comparable across variants as they have peaked at different points in the pandemic, and so vary in background hospital pressure, vaccination availability and rates and case profiles, treatment options, and impact of reporting delay, among other factors.\" James McCreadie, a spokesperson for Public Health England, clarified \"It is too early to assess the case fatality ratio compared to other variants.\"A Canadian study released on 5 October 2021 revealed that the Delta variant caused a 108 percent rise in hospitalization, 235 percent increase in ICU admission, and a 133 percent surge in death compared to other variants. is more serious and resulted in an increased risk of death compared to previous variants, odds that are significantly decreased with immunization.\\n\\n\\n== Statistics ==\\nThe chance of detecting  a Delta case varies significantly, especially depending on a country\\'s sequencing rate (less than 0.05% of all COVID-19 cases have been sequenced in the lowest-sequencing countries to around 50 percent in the highest).By 22 June 2021, more than 4,500 sequences of the variant had been detected in about 78 countries. Reported numbers of sequences in countries with detections are:\\n\\n\\n== History ==\\nThe first cases of the variant outside India were detected in late February 2021, including the United Kingdom on 22 February, the United States on 23 February and Singapore on 26 February.British scientists at Public Health England redesignated the B.1.617.2 variant on 7 May 2021 as \"variant of concern\" (VOC-21APR-02), after they flagged evidence in May 2021 that it spreads more quickly than the original version of the virus. Another reason was that they identified 48 clusters of B.1.617.2, some of which revealed a degree of community transmission. With cases from the Delta variant having risen quickly, British scientists considered the Delta variant having overtaken the Alpha variant as the dominant variant of SARS-CoV-2 in the UK in early June 2021. Researchers at Public Health England later found that over 90% of new cases in the UK in the early part of June 2021 were the Delta variant; they also cited evidence that the Delta variant was associated with an approximately 60% increased risk of household transmission compared to the Alpha variant.Canada\\'s first confirmed case of the variant was identified in Quebec on 21 April 2021, and later the same day 39 cases of the variant were identified in British Columbia. Alberta reported a single case of the variant on 22 April 2021. Nova Scotia reported two Delta variant cases in June 2021.Fiji also confirmed its first case of the variant on 19 April 2021 in Lautoka, and has since then climbed up to 47,000 cases and counting. The variant has been identified as a super-spreader and has led to the lockdowns of five cities (Lautoka, Nadi, Suva, Lami and Nausori), an area which accounts for almost two-thirds of the country\\'s population.\\nOn 29 April 2021, health officials from Finland\\'s Ministry of Social Affairs and Health (STM) and the Finnish Institute for Health and Welfare (THL) reported that the variant had been detected in three samples dating back to March 2021.The Philippines confirmed its first two cases of the variant on 11 May 2021, despite the imposed travel ban of the country from the nations in the Indian subcontinent (except for Bhutan and Maldives). Both patients have no travel history from India for the past 14 days, but instead from Oman and UAE.North Macedonia confirmed its first case of the variant on 7 June 2021 after a person who was recovering from the virus in Iraq was transported to North Macedonia. In a laboratory test, the variant was detected in the person. On 22 June 2021, the country reported its second case of the Delta variant in a colleague of the first case who had also been in Iraq and who subsequently developed symptoms.The detection of B.1.617 was hampered in some countries by a lack of specialised kits for the variant and laboratories that can perform the genetic test. For example, as of 18 May, Pakistan had not reported any cases, but authorities noted that 15% of COVID-19 samples in the country were of an \"unknown variant\"; they could not say if it was B.1.617 because they were unable to test for it. Other countries had reported travellers arriving from Pakistan that were infected with B.1.617.In June 2021, scientist Vinod Scaria of India\\'s Institute of Genomics and Integrative Biology highlighted the existence of the B.1.617.2.1 variant, also known as AY.1 or Delta plus, which has an additional K417N mutation compared to the Delta variant. B.1.617.2.1 was detected in Europe in March 2021, and has since been detected in Asia and America.On 9 July 2021, Public Health England issued Technical Briefing 18 on SARS-CoV-2 variants, documenting 112 deaths among 45,136 UK cases of SARS-CoV-2 Delta variant with 28 days follow-up with a fatality rate of 0.2%. Briefing 16 notes that \"[M]ortality is a lagged indicator, which means that the number of cases who have completed 28 days of follow up is very low – therefore, it is too early to provide a formal assessment of the case fatality of Delta, stratified by age, compared to other variants.\"  Briefing 18 warns that \"Case fatality is not comparable across variants as they have peaked at different points in the pandemic, and so vary in background hospital pressure, vaccination availability and rates and case profiles, treatment options, and impact of reporting delay, among other factors.\"  The most concerning issue is the logistic growth rate of 0.93/week relative to Alpha. This means that per week, the number of Delta samples/cases is growing by a factor of exp (0.93)=2.5 with respect to the Alpha variant. This results, under the same infection prevention measures, in a much greater case load over time until a large fraction of people have been infected by it.\\n\\n\\n=== Government responses ===\\nAfter the rise in cases from the second wave, at least 20 countries imposed travel bans and restrictions on passengers from India in April and May. UK prime minister Boris Johnson cancelled his visit to India twice, while Japanese Prime Minister Yoshihide Suga postponed his April trip.In May 2021, residents of two tower blocks in Velbert, Germany, were quarantined after a woman in the building tested positive for the Delta variant.In May, Delhi Chief Minister Arvind Kejriwal said that a new coronavirus variant from Singapore was extremely dangerous for children and could result in a third wave in India.\\nFrom 16 May to 13 June 2021, as well as 22 July 2021 to 10 August 2021; Singapore entered lockdowns, known as \"Phase 2 Heightened Alert\", similar to 2020.\\nOn 14 June, the British prime minister Boris Johnson announced that the proposed end of all restrictions on 21 June in the United Kingdom was delayed for up to four weeks and vaccination roll-out was accelerated following concerns over the Delta variant, which accounted for the vast majority (90%) of new infections. UK scientists have said that the Delta variant is between 40% and 60% more transmissible than the previously dominant Alpha variant, which was first identified in the UK (as the Kent variant).On 23 June, the province of Ontario in Canada accelerated 2nd dose vaccine appointments for people living in Delta hot spots such as Toronto, Peel and Hamilton.On 25 June, Israel restored their mask mandate citing the threat of Delta.On 28 June, Sydney and Darwin went back into lockdown because of Delta outbreaks. South Africa banned indoor and outdoor gatherings apart from funerals, imposed a curfew, and banned the sale of alcohol.On 3 July, the islands of Bali and Java in Indonesia went into emergency lockdown.On 8 July, Japanese Prime Minister Yoshihide Suga announced that Tokyo would once again enter a state of emergency, and that most spectators would be barred from attending the Olympics set to start there on 23 July.On 9 July, Seoul, South Korea ramped up restrictions urging people to wear masks outdoors, and limiting the size of gatherings.On 12 July, French President Emmanuel Macron announced that all health care workers will need to be vaccinated by 15 September and that France will start using health passports to enter bars, cafés, restaurants and shopping centres from August.Los Angeles announced it will require masks indoors starting 17 July 2021.The United Kingdom lifted most COVID-19 restrictions on 19 July, despite a surge in cases as the Delta variant became dominant. The government cited the protection and wide coverage of the COVID-19 vaccination programme, although health experts expressed concern at the move.On 23 July, Vietnam extended its lockdown of Ho Chi Minh City to 1 August, and announced lockdown restrictions would be put in place in Hanoi, affecting a third of the country\\'s population. The Delta variant had brought upon the country\\'s largest outbreak to date, after mostly successful containment measures throughout 2020.On 17 August, New Zealand went into an alert level 4 lockdown, following a positive case being reported in Auckland. More cases soon followed in the Coromandel Peninsula. This was the first reported community transmission case in the country in 170 days (since February 2021).\\n\\n\\n== See also ==\\n\\nVariants of SARS-CoV-2: Alpha, Beta, Gamma, Epsilon, Zeta, Eta, Theta, Iota, Kappa, Lambda, Mu, Omicron, Lineage B.1.617\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== External links ==\\nPublic Health England: Variants of concern or under investigation, B.1.617',\n",
              " 'There are several ongoing efforts by scientists, governments, international organisations, and others to determine the origin of SARS-CoV-2, the virus responsible for the COVID-19 pandemic. Most scientists say that as with other pandemics in human history, the virus is likely of zoonotic origin in a natural setting, and ultimately originated from a bat-borne virus. Several other explanations, including many conspiracy theories, have been proposed about the origins of the virus.SARS-CoV-2 has close genetic similarity to multiple previously identified bat coronaviruses, suggesting it crossed over into humans from bats. Research is ongoing as to whether SARS-CoV-2 came directly from bats or indirectly through any intermediate hosts. Initial genome sequences of the virus showed little genetic diversity, although subsequently a number of stable variants emerged (some spreading more vigorously), indicating that the spillover event introducing SARS-CoV-2 to humans is likely to have occurred in late 2019. Health authorities and scientists internationally state that as with the 2002–2004 outbreak of SARS-1, efforts to trace the specific geographic and taxonomic origins of SARS-CoV-2 could take years, and the results could be inconclusive.In January 2021, the World Health Assembly (decision-making body of the World Health Organization, WHO) commissioned a study on the origins of the virus, to be conducted jointly between WHO experts and Chinese scientists. In March 2021, the findings of this study were published online in a report to the WHO Commissioner-General. Echoing the assessment of most virologists, the report determined that the virus most likely had a zoonotic origin in bats, possibly transmitted through an intermediate host. It also stated that a laboratory origin for the virus was \"extremely unlikely.\"Scientists found the conclusions of the WHO report to be helpful but noted that more work was needed. In the United States, the European Union, and other countries, some criticised the lack of transparency and data access in the report\\'s formulation. The WHO issued its 30 March report alongside a statement of the WHO director-general Tedros Adhanom Ghebreyesus saying that the matter \"requires further investigation\". The U.S. government and 13 other countries and the EU issued statements on the same day, echoing Tedros\\'s critique of the report for lacking transparency and data access in its formulation. In a later press conference, the WHO director-general said it was \"premature\" for the WHO\\'s report to rule out a potential link between a laboratory leak and called on China to provide \\'raw data\\' and lab audits in a second phase of investigations. On 12 October 2021, WHO announced a new team to study the origins of the COVID-19 pandemic.Earlier, on 22 July 2021, the Chinese government had held a press conference in which Zeng Yixin, Vice Health Minister of the National Health Commission (NHC), said that China would not participate in a second phase of the WHO\\'s investigation, denouncing it as \"shocking\" and \"arrogant\".\\n\\n\\n== Scientific background ==\\n\\nCOVID-19 is caused by infection with a virus called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). SARS-CoV-2 appears to have originated in bats and to have spread to humans by zoonotic transfer. Its exact evolutionary history, the identity and provenance of its most recent ancestors, and the place, time, and mechanism of transmission of the first human infection, remain unknown. The biology and regional distribution of other coronaviruses in southeast Asia, including SARS-CoV, help scientists understand more about the origins of SARS-CoV-2.Taxonomically, SARS-CoV-2 is a virus of the species severe acute respiratory syndrome–related coronavirus (SARSr-CoV). It is believed to have zoonotic origins and has close genetic similarity to bat coronaviruses, suggesting it emerged from a bat-borne virus. Research is ongoing as to whether SARS-CoV-2 came directly from bats or indirectly through any intermediate hosts. The virus shows little genetic diversity, indicating that the spillover event introducing SARS-CoV-2 to humans is likely to have occurred in late 2019. Ultimately, the specific evolutionary history of SARS-CoV-2 in relation to other coronaviruses will be critical to understanding how, where and when the virus spilled over into a human population.\\n\\n\\n=== Reservoir and origin ===\\n\\nThe first known infections from SARS‑CoV‑2 were discovered in Wuhan, China. The original source of viral transmission to humans remains unclear, as does whether the virus became pathogenic before or after the spillover event. Because many of the early infectees were workers at the Huanan Seafood Market, it has been suggested that the virus might have originated from the market. However, other research indicates that visitors may have introduced the virus to the market, which then facilitated rapid expansion of the infections. A March 2021 WHO-convened report stated that human spillover via an intermediate animal host was the most likely explanation, with direct spillover from bats next most likely. Introduction through the food supply chain and the Huanan Seafood Market was considered another possible, but less likely, explanation. An analysis in November 2021, however, said that the earliest-known case had been misidentified and that the preponderance of early cases linked to the Huanan Market argued for it being the source.For a virus recently acquired through a cross-species transmission, rapid evolution is expected. The mutation rate estimated from early cases of SARS-CoV-2 was of 6.54×10−4 per site per year. Coronaviruses in general have high genetic plasticity, but SARS-CoV-2\\'s viral evolution is slowed by the RNA proofreading capability of its replication machinery. For comparison, the viral mutation rate in vivo of SARS-CoV-2 has been found to be lower than that of influenza.Research into the natural reservoir of the virus that caused the 2002–2004 SARS outbreak has resulted in the discovery of many SARS-like bat coronaviruses, most originating in horseshoe bats. The closest match by far, published on Nature (journal) in February 2022, were viruses BANAL-52 (96.8% resemblance to SARS‑CoV‑2), BANAL-103 and BANAL-236, collected in three different species of bats in Feuang, Laos. An earlier source published in February 2020 identified the virus RaTG13, collected in bats in Mojiang, Yunnan, China to be the closest to SARS‑CoV‑2, with 96.1% resemblance. None of the above are its direct ancestor.\\n\\nBats are considered the most likely natural reservoir of SARS‑CoV‑2. Differences between the bat coronavirus and SARS‑CoV‑2 suggest that humans may have been infected via an intermediate host; although the source of introduction into humans remains unknown.Although the role of pangolins as an intermediate host was initially posited (a study published in July 2020 suggested that pangolins are an intermediate host of SARS‑CoV‑2-like coronaviruses), subsequent studies have not substantiated their contribution to the spillover. Evidence against this hypothesis includes the fact that pangolin virus samples are too distant to SARS-CoV-2: isolates obtained from pangolins seized in Guangdong were only 92% identical in sequence to the SARS‑CoV‑2 genome (matches above 90 percent may sound high, but in genomic terms it is a wide evolutionary gap). In addition, despite similarities in a few critical amino acids, pangolin virus samples exhibit poor binding to the human ACE2 receptor.Available scientific evidence suggests that SARS-CoV-2 has a natural zoonotic origin. Yet its origin, which remains unknown, has become debated within the context of global geopolitical tensions. Early in the pandemic, conspiracy theories spread on social media claiming that the virus was a biological weapon developed by China, amplified by echo chambers in the American far-right. Other conspiracy theories promoted misinformation that the virus is not communicable or was created to profit from new vaccines.Some politicians and scientists have speculated that the virus may have accidentally leaked from the Wuhan Institute of Virology. This has led to calls in the media for further investigations into the matter. Many virologists who have studied coronaviruses consider the possibility very remote. The WHO-convened Global Study of the Origins of SARS-CoV-2 from March 2021 stated that such an explanation is extremely unlikely. In an interview on 12 August 2021, Peter Ben Embarek, the chief investigator of the WHO team, told a Danish TV documentary that the WHO team felt pressured from Chinese scientists to put \"extremely unlikely\" as their assessment.A 2021 study found possible links between climate change and transmission of COVID-19 by bats. The authors found that climate-driven changes in the distribution and richness of bat species harboring coronaviruses may have occurred leading to a concentration of such species in eastern Asian hotspots (southern China, Myanmar and Laos). Tell conditions favors transmission of viruses. In this area live pangolins that are sold among other in the Wuhan wet market and are suspected of being the intermediate host in the transmission of SARS-COV-2 who cause the pandemic. The study was cited at 22 other studies in the National Library of Medicine of the USA.  Before the word health day 2022, the World Health Organization mentioned on its site that: \"There is growing evidence of a direct connection between environmental change and the emergence or transmission of COVID-19\".\\n\\n\\n=== Phylogenetics and taxonomy ===\\nSARS‑CoV‑2 belongs to the broad family of viruses known as coronaviruses. It is a positive-sense single-stranded RNA (+ssRNA) virus, with a single linear RNA segment. Coronaviruses infect humans, other mammals, including livestock and companion animals, and avian species. Human coronaviruses are capable of causing illnesses ranging from the common cold to more severe diseases such as Middle East respiratory syndrome (MERS, fatality rate ~34%). SARS-CoV-2 is the seventh known coronavirus to infect people, after 229E, NL63, OC43, HKU1, MERS-CoV, and the original SARS-CoV.Like the SARS-related coronavirus implicated in the 2003 SARS outbreak, SARS‑CoV‑2 is a member of the subgenus Sarbecovirus (beta-CoV lineage B). Coronaviruses undergo frequent recombination. The mechanism of recombination in unsegmented RNA viruses such as SARS-CoV-2 is generally by copy-choice replication, in which gene material switches from one RNA template molecule to another during replication. SARS-CoV-2 RNA sequence is approximately 30,000 bases in length, relatively long for a coronavirus (which in turn carry the largest genomes among all RNA families) Its genome consists nearly entirely of protein-coding sequences, a trait shared with other coronaviruses.\\n\\nA distinguishing feature of SARS‑CoV‑2 is its incorporation of a polybasic site cleaved by furin, which appears to be an important element enhancing its virulence. In SARS-CoV-2 the recognition site is formed by the incorporated 12 codon nucleotide sequence CCT CGG CGG GCA which corresponds to the amino acid sequence P RR A. This sequence is upstream of an arginine and serine which forms the S1/S2 cleavage site (P RR A R↓S) of the spike protein. Although such sites are a common naturally-occurring feature of other viruses within the Subfamily Orthocoronavirinae, it appears in few other viruses from the Beta-CoV genus, and it is unique among members of its subgenus for such a site. The furin cleavage site PRRAR↓ is identical to that of the feline coronavirus, an alphacoronavirus 1 strain.Viral genetic sequence data can provide critical information about whether viruses separated by time and space are likely to be epidemiologically linked. With a sufficient number of sequenced genomes, it is possible to reconstruct a phylogenetic tree of the mutation history of a family of viruses. By 12 January 2020, five genomes of SARS‑CoV‑2 had been isolated from Wuhan and reported by the Chinese Center for Disease Control and Prevention (CCDC) and other institutions; the number of genomes increased to 42 by 30 January 2020. A phylogenetic analysis of those samples showed they were \"highly related with at most seven mutations relative to a common ancestor\", implying that the first human infection occurred in November or December 2019. Examination of the topology of the phylogenetic tree at the start of the pandemic also found high similarities between human isolates.  As of 21 August 2021, 3,422 SARS‑CoV‑2 genomes, belonging to 19 strains, sampled on all continents except Antarctica were publicly available.On 11 February 2020, the International Committee on Taxonomy of Viruses announced that according to existing rules that compute hierarchical relationships among coronaviruses based on five conserved sequences of nucleic acids, the differences between what was then called 2019-nCoV and the virus from the 2003 SARS outbreak were insufficient to make them separate viral species. Therefore, they identified 2019-nCoV as a virus of Severe acute respiratory syndrome–related coronavirus.\\n\\n\\n== Origin scenarios ==\\nThe origin of SARS-CoV-2 has been the subject of debate. There are multiple proposed explanations for how SARS-CoV-2 was introduced into, and evolved adaptations suited to, the human population. There is significant evidence and agreement that the most likely original viral reservoir for SARS-CoV-2 is horseshoe bats, with the closest known viral relative being RaTG13. The evolutionary distance between SARS-CoV-2 and RaTG13 is estimated to be about 50 years (between 38 and 72 years). The earliest human cases of SARS-CoV-2 were identified in Wuhan, but the index case remains unknown. RaTG13 was sampled from bats in Yunnan, located roughly 1,300 km (810 mi) away from Wuhan, and there are relatively few bat coronaviruses from Hubei province. Each origin hypothesis attempts to explain this gap in virus evolution and location a different way. These scenarios continue to be investigated to identify the definitive origin of the virus.\\n\\n\\n=== Direct zoonotic transmission in a natural setting ===\\nThe most direct pathway of introduction is direct zoonotic transmission (also known as spillover) from the reservoir species to humans. Scientists consider this to be a highly likely origin of the SARS-CoV-2 virus in humans. Human contact with bats has increased as human population centers encroach on bat habitats, leading to increased opportunities for spillover. Bats are a significant reservoir species for a diverse range of coronaviruses, and humans have been found with antibodies for them suggesting that this form of direct infection by bats is common. However, in this scenario, the direct ancestor of SARS-CoV-2 remains undiscovered in bats.\\n\\n\\n=== Intermediate host ===\\nIn addition to direct spillover, another pathway, considered highly likely by scientists, is that of transmission through an intermediate host. Specifically, this implies that a cross species transmission occurred prior to the human outbreak and that it had pathogenic results on the animal. This pathway has the potential to allow for greater adaptation to human transmission via animals with more similar protein shapes to humans, though this is not required for the scenario to occur. The evolutionary separation from bat viruses is explained in this case by the virus\\' presence in an unknown species with less viral surveillance than bats. The virus\\' ability to easily infect and adapt to additional species (including mink) provides evidence that such a route of transmission is possible.\\n\\n\\n=== Cold/food chain ===\\nAnother proposed introduction to humans is through fresh or frozen food products, referred to as the cold/food chain. Scientists do not consider this to be a likely origin of SARS-CoV-2 in humans. This scenario\\'s source animal could be either a direct or intermediary species as described above. Many investigations centered around the Huanan Seafood Wholesale Market in Wuhan, which had an early cluster of cases. While there have been food-borne outbreaks of human viruses in the past, and evidence of re-introduction of SARS-CoV-2 into China through imported frozen foods, investigations found no conclusive evidence of viral contamination in products at the Huanan Market.\\n\\n\\n=== Laboratory incident ===\\n\\nA final scenario, assessed to be unlikely by most experts, is known as the lab leak hypothesis. Considered \"extremely unlikely\" by a WHO-convened study, the theory considers the possibility that the virus was introduced to humans due to an intentional or unintentional laboratory escape. This scenario would have involved laboratory staff becoming infected from contact with live bats (wild or captive), from contact with biological samples, or from contact with viruses being grown in vitro. The Wuhan Institute of Virology (WIV) has performed research into bat coronaviruses since 2005, and identified the RaTG13 virus in 2013, which is the closest known relative of SARS-CoV-2. WIV research interests have included investigations into the source of the 2002–2004 SARS outbreak and 2012 MERS outbreak, in collaboration with US researchers. The proximity of the laboratory to the Huanan seafood market has led some to speculate there may be a link between the two, and politicians, media personalities, and some scientists have called for further investigations into the matter. Experts have ruled out intentional manipulation of the virus (i.e. biological engineering) as a plausible origin, due to a lack of supporting evidence, and growing evidence in favor of a natural origin.New York-based, National Institutes of Health–funded EcoHealth Alliance has been the subject of controversy and increased scrutiny due to its ties to the Wuhan Institute of Virology. Under political pressure, the National Institutes of Health (NIH) withdrew funding to EcoHealth Alliance in July 2020.Subsequent investigations (such as the WHO-convened study) considered the possibility of a collected natural virus inadvertently infecting laboratory staff. On 15 July 2021 WHO director-general Tedros Adhanom stated in a speech that there had been a \"premature push\" to discredit the idea of a lab leak. Later on in the same speech, he expanded, \"I was a lab technician, an immunologist, and worked in the lab. And lab accidents happen.\" The WHO has planned a second phase of investigation that will include \"audits of relevant laboratories and research institutions\". However, China has rejected this plan stating \"it is impossible for us to accept such an origin-tracing plan\".An even less likely lab-leak theory has been proposed by some members of the Chinese government who have claimed that the virus came from a U.S. military laboratory. Chinese demands to investigate the U.S. lab at Fort Detrick have been said to be intended to deflect attention from Wuhan.\\n\\n\\n== Investigations ==\\n\\n\\n=== Chinese government ===\\n\\nThe first investigation conducted in China was by the Wuhan Municipal Health Commission, responding to hospitals reporting cases of pneumonia of unknown etiology, resulting in the closure of the Huanan Seafood Wholesale Market on 1 January 2020 for sanitation and disinfection. The market was originally suspected of being the source of the virus; however, the Chinese government and the WHO determined later that it was not.In April 2020, China imposed restrictions on publishing academic research on the novel coronavirus. Investigations into the origin of the virus would receive extra scrutiny and must be approved by Central Government officials. The restrictions do not ban research or publication, including with non-Chinese researchers; Ian Lipkin, a US scientist, has been working with a team of Chinese researchers under the auspices of the Chinese Center for Disease Control and Prevention, a Chinese government agency, to investigate the origin of the virus. Lipkin has long-standing relationships with Chinese officials, including premier Li Keqiang, because of his contributions to rapid testing for SARS in 2003.\\n\\n\\n=== United States government ===\\n\\n\\n==== Trump administration ====\\nOn 6 February 2020, the director of the White House\\'s Office of Science and Technology Policy requested the National Academies of Sciences, Engineering, and Medicine to convene a meeting of \"experts, world class geneticists, coronavirus experts, and evolutionary biologists\", to \"assess what data, information and samples are needed to address the unknowns, in order to understand the evolutionary origins of COVID-19 and more effectively respond to both the outbreak and any resulting information\".In April 2020, it was reported that the US intelligence community was investigating whether the virus came from an accidental leak from a Chinese lab. The hypothesis was one of several possibilities being pursued by the investigators. US Secretary of Defense Mark Esper said the results of the investigation were \"inconclusive\". By the end of April 2020, the Office of the Director of National Intelligence said the US intelligence community believed the coronavirus was not man-made or genetically modified.US officials criticised the \"terms of reference\" allowing Chinese scientists to do the first phase of preliminary research. On 15 January 2021, US Secretary of State Mike Pompeo said that to assist the WHO investigative team\\'s work and ensure a transparent, thorough investigation of COVID-19\\'s origin, the US was sharing new information and urging the WHO to press the Chinese government to address three specific issues, including the illnesses of several researchers inside the WIV in autumn 2019 \"with symptoms consistent with both COVID-19 and common seasonal illnesses\", the WIV\\'s research on \"RaTG13\" and \"gain of function\", and the WIV\\'s links to the People\\'s Liberation Army. On 18 January, the US called on China to allow the WHO\\'s expert team to interview \"care givers, former patients and lab workers\" in the city of Wuhan, drawing a rebuke from the Chinese government. Australia also called for the WHO team to have access to \"relevant data, information and key locations\".A classified report from May 2020 by the Lawrence Livermore National Laboratory, a US government national laboratory, concluded that the hypothesis that the virus leaked from the WIV \"is plausible and deserves further investigation\", although the report also notes that the virus could have developed naturally, echoing the consensus of the American intelligence community, and provides no \"smoking gun\" towards either hypothesis.\\n\\n\\n==== Biden administration ====\\nOn 13 February 2021, the White House said it had \"deep concerns\" about both the way the WHO\\'s findings were communicated and the process used to reach them. Mirroring concerns raised by the Trump administration, National Security Adviser Jake Sullivan stated that it was essential that the WHO-convened report be independent and \"free from alteration by the Chinese government\". On 14 April 2021, the Director of National Intelligence Avril Haines, along with other Biden administration officials, said that they had not ruled out the possibility of a laboratory accident as the origin of the COVID-19 virus.On 26 May 2021, President Joe Biden directed the U.S. intelligence community to produce a report within 90 days on whether the COVID-19 virus originated from a human contact with an infected animal or from an accidental lab leak, stating his national security staff said there is insufficient evidence to determine either hypothesis to be more likely. On 26 August 2021, the Office of the Director of National Intelligence released an unclassified summary of their findings, with the main point being that the report remained inconclusive as to the origin of the virus, with intelligence agencies divided on the question. The report also concluded that the virus was most likely not genetically engineered, and that China had no foreknowledge of the virus prior to the outbreak. The report concluded that a final determination of the origin was unlikely without cooperation from the Chinese government, saying their prior lack of transparency \"reflect[ed] in part China\\'s government\\'s own uncertainty about where an investigation could lead, as well as its frustration that the international community is using the issue to exert political pressure on China.\" Chinese foreign ministry spokesman Wang Wenbin said that the US intelligence report was \"unscientific and has no credibility\".On 23 May 2021, The Wall Street Journal reported that a previously undisclosed US intelligence report stated that three researchers from the Wuhan Institute of Virology became ill enough in November 2019 to seek hospital care. The report did not specify what the illness was. Officials familiar with the intelligence differed as to the strength to which it corroborates the hypothesis that the virus responsible for COVID-19 was leaked from the WIV. The WSJ report notes that it is not unusual for people in China to go to the hospital with uncomplicated influenza or common cold symptoms.Yuan Zhiming, director of the WIV\\'s Wuhan National Biosafety Laboratory, responded in the Global Times, a Chinese state media outlet, that the \"claims are groundless\". Marion Koopmans, a member of the WHO study team, described the number of flu-like illnesses at the WIV in 2019 as \"completely normal\". Workers at the WIV must provide yearly serum samples. WIV virologist Shi Zhengli said in 2020 that, based on an evaluation of those serum samples, all staff tested negative for COVID-19 antibodies.The resurgence of the theory of a laboratory accident was fueled in part by the publication, in May 2021, of early emails between Anthony Fauci and scientists discussing the issue, before deliberate manipulation was ruled out as of March 2020.On 14 July 2021, the House Committee on Science, Space and Technology held the first congressional hearing on the origins of the virus. Bill Foster, an Illinois Democrat who chaired the hearing, said the Chinese government\\'s lack of transparency is not in itself evidence of a lab leak and cautioned that answers may not be known even after the administration produces its intelligence report. Expert witnesses Stanley Perlman and David Relman presented to the congressman different proposed explanations for the origins of the virus and how to conduct further investigations.On 16 July 2021, CNN reported that Biden administration officials considered the lab leak theory \"as credible\" as the natural origins theory.\\n\\n\\n=== World Health Organization ===\\nThe World Health Organization has declared that finding where SARS-CoV-2 came from is a priority and that it is \"essential for understanding how the pandemic started.\" In May 2020, the World Health Assembly, which governs the World Health Organization (WHO), passed a motion calling for a \"comprehensive, independent and impartial\" study into the COVID-19 pandemic. A record 137 countries, including China, co-sponsored the motion, giving overwhelming international endorsement to the study. In mid 2020, the World Health Organization (WHO) began negotiations with the government of China on conducting an official study into the origins of COVID-19.\\nIn November 2020, the WHO published a two-phase study plan. The purpose of the first phase was to better understand how the virus \"might have started circulating in Wuhan\", and a second phase involves longer-term studies based on the findings of the first phase. WHO director-general Tedros Adhanom said \"We need to know the origin of this virus because it can help us to prevent future outbreaks,\" adding, \"There is nothing to hide. We want to know the origin, and that\\'s it.\" He also urged countries not to politicise the origin tracing process, saying that would only create barriers to learning the truth.\\n\\n\\n==== Phase 1 ====\\nFor the first phase, the WHO formed a team of ten researchers with expertise in virology, public health and animals to conduct a thorough study. One of the team\\'s tasks was to retrospectively ascertain what wildlife was being sold in local wet markets in Wuhan. The WHO\\'s phase one team arrived and quarantined in Wuhan, Hubei, China in January 2021.Members of the team included Thea Fisher, John Watson, Marion Koopmans, Dominic Dwyer, Vladimir Dedkov, Hung Nguyen-Viet, Fabian Leendertz, Peter Daszak, Farag El Moubasher, and Ken Maeda. The team also included five WHO experts led by Peter Ben Embarek, two Food and Agriculture Organization representatives, and two representatives from the World Organisation for Animal Health.The inclusion of Peter Daszak in the team stirred controversy. Daszak is the head of EcoHealth Alliance, a nonprofit that studies spillover events, and has been a longtime collaborator of over 15 years with Shi Zhengli, Wuhan Institute of Virology\\'s director of the Center for Emerging Infectious Diseases. While Daszak is highly knowledgeable about Chinese laboratories and the emergence of diseases in the area, his close connection with the WIV was seen by some as a conflict of interest in the WHO\\'s study. When a BBC News journalist asked about his relationship with the WIV, Daszak said, \"We file our papers, it\\'s all there for everyone to see.\"The team was denied access to raw data, including the list of early patients, swabs, and blood samples. It was allowed only a few hours of supervised access to the Wuhan Institute of Virology.\\n\\n\\n===== Findings =====\\nIn February 2021, after conducting part of their study, the WHO stated that the likely origin of COVID-19 was a zoonotic event from a virus circulating in bats, likely through another animal carrier, and that the time of transmission to humans was likely towards the end of 2019.The Chinese and the international experts who jointly carried out the WHO-convened study consider it \"extremely unlikely\" that COVID-19 leaked from a lab. No evidence of a lab leak from the Wuhan Institute of Virology was found by the WHO team, with team leader Peter Ben Embarek stating that it was \"very unlikely\" due to the safety protocols in place. During a 60 Minutes interview with Lesley Stahl, Peter Daszak, another member of the WHO team, described the investigation process to be a series of questions and answers between the WHO team and the Wuhan lab staff. Stahl made the comment that the team was \"just taking their word for it\", to which Daszak replied, \"Well, what else can we do? There\\'s a limit to what you can do and we went right up to that limit. We asked them tough questions. They weren\\'t vetted in advance. And the answers they gave, we found to be believable—correct and convincing.\"The investigation also stated that transfer from animals to humans was unlikely to have occurred at the Huanan Seafood Market, since infections without a known epidemiological link were confirmed before the outbreak around the market. In an announcement that surprised some foreign experts, the joint investigation concluded that early transmission via the cold chain of frozen products was \"possible.\"In March 2021, the WHO published a written report with the results of the study. The joint team stated that there are four scenarios for introduction:\\ndirect zoonotic transmission to humans (spillover), assessed as \"possible to likely\"\\nintroduction through an intermediate host followed by a spillover, assessed as \"likely to very likely\"\\nintroduction through the (cold) food chain, assessed as \"possible\"\\nintroduction through a laboratory incident, assessed as \"extremely unlikely\"The report mentions that direct zoonotic transmission to humans has a precedent, as most current human coronaviruses originated in animals. Zoonotic transmission is also supported by the fact that RaTG13 binds to hACE2, although the fit is not optimal.The investigative team noted the requirement for further studies, noting that these would \"potentially increase knowledge and understanding globally.\":\\u200a9\\u200a\\n\\n\\n===== Reactions =====\\nWHO director-general Tedros Adhanom, who was not directly involved with the investigation, said he was ready to dispatch additional missions involving specialist experts and that further research was required. He said in a statement, \"Some explanations may be more probable than others, but for now all possibilities remain on the table.\" He also said, \"We have not yet found the source of the virus, and we must continue to follow the science and leave no stone unturned as we do.\" Tedros called on China to provide \"more timely and comprehensive data sharing\" as part of future investigations.News outlets noted that, though it was unrealistic to expect quick and huge results from the report, it \"offered few clear-cut conclusions regarding the start of the pandemic\", \"failed to audit the Chinese official position at some parts of the report\", and was \"biased according to critics\". Other scientists praised how the report details the pathways that can shed light on the origin, if explored later.After the publication of the report, politicians, talk show hosts, journalists, and some scientists advanced unsupported claims that SARS-CoV-2 may have come from the WIV. In the United States, calls to investigate a laboratory leak reached \"fever pitch\", fueling aggressive rhetoric resulting in antipathy towards people of Asian ancestry, and the bullying of scientists. The European Union, United States, and 13 other countries criticised the WHO-convened study, calling for transparency from China and access to the raw data and original samples. Chinese officials described these criticisms as an attempt to politicise the study. Scientists involved in the WHO report, including Liang Wannian, John Watson, and Peter Daszak, objected to the criticism, and said that the report was an example of the collaboration and dialogue required to successfully continue investigations into the matter.In a letter published in Science, a number of scientists, including Ralph Baric, argued that the accidental laboratory leak hypothesis had not been sufficiently investigated and remained possible, calling for greater clarity and additional data. Their letter was criticized by some virologists and public health experts, who said that a \"hostile\" and \"divisive\" focus on the WIV was unsupported by evidence, and would cause Chinese scientists and authorities to share less, rather than more data.\\n\\n\\n==== Phase 2 ====\\n\\nOn 27 May 2021, Danish epidemiologist Tina Fischer spoke on the This Week in Virology podcast, advocating for a second phase of the study to audit blood samples for COVID-19 antibodies in China. WHO-convened study team member Marion Koopmans, on that same broadcast, advocated for WHO member states to make a decision on the second phase of the study, though she also cautioned that an investigatory audit of the laboratory itself may be inconclusive. In early July 2021, WHO emergency chief Michael Ryan said the final details of phase 2 were being worked out in negotiations between WHO and its member states, as the WHO works \"by persuasion\" and cannot compel any member state (including China) to cooperate.In July 2021 China rejected WHO requests for greater transparency, cooperation, and access to data as part of Phase 2. On 16 July 2021, Foreign Ministry spokesperson Zhao Lijian declared that China\\'s position was that future investigations should be conducted elsewhere and should focus on cold chain transmission and the US military\\'s labs. On 22 July 2021, the Chinese government held a press conference in which Zeng Yixin, Vice Health Minister of the National Health Commission (NHC), said that China would not participate in a second phase of the WHO\\'s investigation, denouncing it as \"shocking\" and \"arrogant\". He elaborated \"In some aspects, the WHO\\'s plan for next phase of investigation of the coronavirus origin doesn\\'t respect common sense, and it\\'s against science. It\\'s impossible for us to accept such a plan.\"\\n\\n\\n=== The Lancet COVID-19 Commission task force ===\\nIn November 2020, an international task force led by Peter Daszak, president of EcoHealth Alliance, was formed as part of The Lancet COVID-19 Commission, backed by the medical journal The Lancet. Daszak stated that the task force was formed to \"conduct a thorough and rigorous investigation into the origins and early spread of SARS-CoV-2\". The task force has twelve members with backgrounds in One Health, outbreak investigation, virology, lab biosecurity and disease ecology.  The task force plans to analyse scientific findings and does not plan to visit China. In June 2021, The Lancet announced that Daszak had recused himself from the commission. On 25 September 2021, the task force work was folded after procedural concerns and a need to broaden its scope to examine transparency and government regulation of risky laboratory research.\\n\\n\\n=== Independent investigations ===\\nIn June 2021, the NIH announced that a set of sequence data had been removed from the Sequence Read Archive (SRA) in June 2020. The removal was performed according to standard practice at the request of the investigators who owned the rights to the sequences, with the investigators\\' rationale that the sequences would be submitted to another database. The investigators subsequently published a paper in an academic journal the same month they were removed from the NIH database which described the sequences in detail and discussed their evolutionary relationship to other sequences, but did not include the raw data. Virologist David Robertson said that it was difficult to conclude it was a cover-up rather than the more likely explanation: a mundane deletion of data without malfeasance. The missing genetic sequence data was restored in a correction published 29 July 2021 after it was stated to be a copy-edit error.\\n\\n\\n=== International calls for investigations ===\\nIn April 2020, Australian foreign minister Marise Payne and Australian prime minister Scott Morrison called for an independent international inquiry into the origins of the coronavirus pandemic. A few days later, German chancellor Angela Merkel also pressed China for transparency about the origin of the coronavirus, following similar concerns raised by the French president Emmanuel Macron. The UK also expressed support for an investigation, although both France and UK said the priority at the time was to first fight the virus. Some public health experts have also called for an independent examination of COVID-19\\'s origins, \"arguing WHO does not have the political clout to conduct such a forensic analysis\". In May 2021, Prime Minister Justin Trudeau told reporters Canada would \"support the call by the United States and others to better understand the origins of COVID-19.\" In June 2021, at the G7 summit in Cornwall, the attending leaders issued a joint statement calling for a new investigation, citing China\\'s refusal to cooperate with certain aspects of the original WHO-convened study. This resistance to international pressure was one of the key findings of a Wall Street Journal investigation into the pandemic origin.The divisive nature of the debate has led scientists to call for less political pressure on the topic. Public health analysts have remarked that the debate over the origins of SARS-CoV-2 is fueling unnecessary confrontation, resulting in bullying and harassment of scientists, and is deepening existing geopolitical tensions and hindering collaboration at a time where such mutual cooperation is required, both to deal with the current pandemic and in preparation for future such outbreaks. This comes in the face of scientists having predicted such events for decades: according to Katie Woolaston, researcher at the Queensland University of Technology, \"The environmental drivers of pandemics are not being widely discussed\". The debate comes at a moment of difficult global relations with Chinese authorities. Researchers have noted that the politicisation of the debate is making the process more difficult, and that words are often twisted to become \"fodder for conspiracy theories.\" A letter published in The Lancet in July 2021 remarked that the atmosphere of speculation surrounding the issue was of no help in making an objective assessment of the situation. In response to this letter, in a communication published in the same journal, a small group of scientists opposed the idea that scientists should promote unity and called for openness to alternative hypotheses. Despite the unlikelihood of the event, and although definitive answers are likely to take years of research, biosecurity experts have called for a review of global biosecurity policies, citing known gaps in international standards for biosafety. The situation has also reignited a debate over gain-of-function research, although the intense political rhetoric surrounding the issue has threatened to sideline serious inquiry over policy in this domain.\\n\\n\\n== See also ==\\nScientific Advisory Group for Origins of Novel Pathogens\\nWorld Health Organization\\'s response to the COVID-19 pandemic\\n\\n\\n== References ==']"
            ]
          },
          "execution_count": 46,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Returning full documents\n",
        "terms = ['genome', 'etiology']                   # find all documents with the word \"genome\" or \"etiology\"\n",
        "result_docs = []\n",
        "\n",
        "for term in terms:\n",
        "  doc_ids = term_occurence_dict[term]\n",
        "  for doc_id in doc_ids:\n",
        "    result_docs.append(documents[doc_id])\n",
        "\n",
        "result_docs = list(set(result_docs))\n",
        "result_docs                                      # return full documents"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "MqoQYhpWCuen"
      },
      "source": [
        "Term-document pair dictionary created successfully."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "fa0_qBLKgvwN",
        "outputId": "9e36f4a3-3760-45a1-df95-f47ec1dba15e"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "03        1\n",
              "030       1\n",
              "035       1\n",
              "03614     1\n",
              "037       1\n",
              "         ..\n",
              "zoos      2\n",
              "zoster    1\n",
              "zulia     1\n",
              "zycov     1\n",
              "zürich    1\n",
              "Length: 13249, dtype: int64"
            ]
          },
          "execution_count": 47,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Calculating document frequency (i.e. how many documents each phrase appears in)\n",
        "dfs = (dataframe > 0).sum(axis=0)\n",
        "dfs"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "GcOEZ2iSg4kq",
        "outputId": "f4058286-f3d0-48ac-f426-1a3382445270"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "03        1.477121\n",
              "030       1.477121\n",
              "035       1.477121\n",
              "03614     1.477121\n",
              "037       1.477121\n",
              "            ...   \n",
              "zoos      1.176091\n",
              "zoster    1.477121\n",
              "zulia     1.477121\n",
              "zycov     1.477121\n",
              "zürich    1.477121\n",
              "Length: 13249, dtype: float64"
            ]
          },
          "execution_count": 48,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Creating idf for every term in the dataset\n",
        "N = dataframe.shape[0]\n",
        "idfs = np.log10(N/dfs)\n",
        "idfs"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "iifmtGOgu40Q"
      },
      "source": [
        "An index of inverse document frequencies _(idf)_ has now been created for every vocabulary term in the dataset.<br> \n",
        "\n",
        "### **Section 3: Creating a BM25 Model**\n",
        "The seminal Best Match 25 (BM25) probabilistic Information Retrieval (IR) model (Robertson _et al_, 1994) is, to date, the standard IR model against which other models are measured. It is calculated by the function"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "XUk5Sbm3phWd"
      },
      "source": [
        "**BM25 Function**\n",
        "\n",
        "<center>\n",
        "<div>\n",
        "<img src= \"https://drive.google.com/uc?id=1cvkpSt2PtX2f9eqcqCQNPG9P7CKHbRM7\" alt= \"BM25_formula\" width=500/>\n",
        "</div>\n",
        "</center>  "
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7zPBQI_HGhPt"
      },
      "source": [
        "#### **Section 3.1: Creating retrieval function**<br> \n",
        "In this section, a simplified version of the above function is used to build an instance algorithm of the Okapi BM25 in the next few series of steps, according to the equation:\n",
        "\n",
        "<center>\n",
        "<div>\n",
        "<img src= \"https://drive.google.com/uc?id=1OrKL8Wncj_iG6-7rv3eEfk_NNoGep04y\" alt= \"BM25_formula\" width=400/>\n",
        "</div>\n",
        "</center>\n",
        "\n",
        "where RSV = Retrieval Status Value. "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "OTe48g-IosXp"
      },
      "outputs": [],
      "source": [
        "# Defining variables\n",
        "k_1 = 1.2                                       # single value\n",
        "b = 0.8                                         # single value\n",
        "\n",
        "# Considering all words in doc\n",
        "dls = [len(d.split(' ')) for d in documents]    # vector\n",
        "\n",
        "# Considering words minus stop words in doc. \n",
        "dls = dataframe.sum(axis=1).tolist()\n",
        "avgdl = np.mean(dls)                            # single value"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "rBLETSc0remH"
      },
      "outputs": [],
      "source": [
        "# Calculating BM25 term frequency quantification\n",
        "numerator = np.array((k_1 + 1) * dataframe)\n",
        "denominator = np.array(k_1 *((1 - b) + b * (dls / avgdl))).reshape(N,1) + np.array(dataframe)\n",
        "\n",
        "BM25_tf = numerator / denominator\n",
        "\n",
        "idfs = np.array(idfs)\n",
        "\n",
        "BM25_score = BM25_tf * idfs"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "RN-FT4Mcr-Xk",
        "outputId": "12b97aeb-b6cf-437b-8367-d57baaa0c77c"
      },
      "outputs": [
        {
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-6eb30e2c-88fe-445c-8be9-151e9f0b119b\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>03</th>\n",
              "      <th>030</th>\n",
              "      <th>035</th>\n",
              "      <th>03614</th>\n",
              "      <th>037</th>\n",
              "      <th>04</th>\n",
              "      <th>041</th>\n",
              "      <th>042</th>\n",
              "      <th>043</th>\n",
              "      <th>045</th>\n",
              "      <th>...</th>\n",
              "      <th>zones</th>\n",
              "      <th>zooanthroponosis</th>\n",
              "      <th>zoom</th>\n",
              "      <th>zoonosis</th>\n",
              "      <th>zoonotic</th>\n",
              "      <th>zoos</th>\n",
              "      <th>zoster</th>\n",
              "      <th>zulia</th>\n",
              "      <th>zycov</th>\n",
              "      <th>zürich</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.055068</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.625033</td>\n",
              "      <td>0.840050</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.449506</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>1.868284</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.043824</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>1.757947</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.286587</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.59437</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.59437</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>10</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.295175</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.95913</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.763664</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>12</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>13</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>14</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.342624</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>15</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>1.207561</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>2.535139</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>16</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>17</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>18</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>1.268154</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>19</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>1.334699</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.676327</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>20</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.277397</td>\n",
              "      <td>0.950437</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>21</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.738906</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>22</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>23</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.612612</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>24</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>25</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.278536</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>26</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>27</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.347095</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>28</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>1.721081</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>29</th>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>...</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>30 rows × 13249 columns</p>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-6eb30e2c-88fe-445c-8be9-151e9f0b119b')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-6eb30e2c-88fe-445c-8be9-151e9f0b119b button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-6eb30e2c-88fe-445c-8be9-151e9f0b119b');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "          03       030      035     03614      037        04       041  \\\n",
              "0   0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "1   0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  1.449506   \n",
              "2   0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "3   0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "4   0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "5   0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "6   1.868284  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "7   0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "8   0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "9   0.000000  0.000000  0.00000  0.000000  1.59437  0.000000  0.000000   \n",
              "10  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "11  0.000000  0.000000  0.95913  0.000000  0.00000  0.000000  0.000000   \n",
              "12  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "13  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "14  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "15  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "16  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "17  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "18  0.000000  0.000000  0.00000  1.268154  0.00000  0.000000  0.000000   \n",
              "19  0.000000  0.000000  0.00000  0.000000  0.00000  1.334699  0.000000   \n",
              "20  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "21  0.000000  0.000000  0.00000  0.000000  0.00000  0.738906  0.000000   \n",
              "22  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "23  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "24  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "25  0.000000  1.278536  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "26  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "27  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "28  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "29  0.000000  0.000000  0.00000  0.000000  0.00000  0.000000  0.000000   \n",
              "\n",
              "        042       043       045  ...     zones  zooanthroponosis      zoom  \\\n",
              "0   0.00000  0.000000  0.000000  ...  0.000000          1.055068  0.000000   \n",
              "1   0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "2   0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "3   0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "4   0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "5   0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "6   0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "7   0.00000  1.757947  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "8   0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "9   1.59437  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "10  0.00000  0.000000  1.295175  ...  0.000000          0.000000  0.000000   \n",
              "11  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "12  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "13  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "14  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "15  0.00000  0.000000  0.000000  ...  1.207561          0.000000  2.535139   \n",
              "16  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "17  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "18  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "19  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "20  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "21  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "22  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "23  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "24  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "25  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "26  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "27  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "28  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "29  0.00000  0.000000  0.000000  ...  0.000000          0.000000  0.000000   \n",
              "\n",
              "    zoonosis  zoonotic      zoos    zoster     zulia     zycov    zürich  \n",
              "0   0.000000  0.625033  0.840050  0.000000  0.000000  0.000000  0.000000  \n",
              "1   0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "2   0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "3   0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "4   0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "5   0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "6   1.043824  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "7   0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "8   0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  1.286587  \n",
              "9   0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "10  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "11  0.000000  0.000000  0.763664  0.000000  0.000000  0.000000  0.000000  \n",
              "12  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "13  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "14  0.000000  0.000000  0.000000  0.000000  2.342624  0.000000  0.000000  \n",
              "15  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "16  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "17  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "18  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "19  0.000000  0.000000  0.000000  0.000000  0.000000  1.676327  0.000000  \n",
              "20  1.277397  0.950437  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "21  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "22  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "23  0.000000  0.000000  0.000000  1.612612  0.000000  0.000000  0.000000  \n",
              "24  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "25  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "26  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "27  0.000000  1.347095  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "28  0.000000  1.721081  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "29  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  0.000000  \n",
              "\n",
              "[30 rows x 13249 columns]"
            ]
          },
          "execution_count": 51,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Outputting BM25 scores\n",
        "bm25_idf = pd.DataFrame(BM25_score, columns=vocabulary)\n",
        "bm25_idf"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "pVSGKbKq2i0C"
      },
      "source": [
        "The above dataframe is equivalent to a table of BM25 scores for each document."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "va4hAYt5cfaG"
      },
      "source": [
        "#### **Section 3.2: Searching and ranking documents for queries**<br> \n",
        "The next few steps demonstrate the search engine receiving a query input, searching the documents index, and outputting a ranked list of suitable documents in answer to the search: "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "uqpeiAeAsb_L"
      },
      "outputs": [],
      "source": [
        "# Querying search engine for the terms 'vaccine' or 'symptom'\n",
        "q_terms = ['vaccine', 'symptom']"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "b9V9XNaUfqpl"
      },
      "source": [
        "To find documents that fit the query well based on the BM25 scores:"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "LK6YSn_rtU1b"
      },
      "outputs": [],
      "source": [
        "# Cutting up BM25 dataframe in terms of the query terms\n",
        "q_terms_only_df = bm25_idf[q_terms]\n",
        "score_q_d = q_terms_only_df.sum(axis=1)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "RYADTNSItfqL",
        "outputId": "662561a8-41bf-4828-fccd-be57b2deff25"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "[('Long COVID is a condition characterized by long-term consequences persisting or appearing after the typical convalescence period of COVID-19. It is also known as post-COVID-19 syndrome, post-COVID-19 condition, post-acute sequelae of COVID-19 (PASC), or chronic COVID syndrome (CCS). Long COVID can affect nearly every organ system, with sequelae including respiratory system disorders, nervous system and neurocognitive disorders, mental health disorders, metabolic disorders, cardiovascular disorders, gastrointestinal disorders, malaise, fatigue, musculoskeletal pain, and anemia. A wide range of symptoms are commonly reported, including fatigue, headaches, shortness of breath, anosmia (loss of smell), parosmia (distorted smell), muscle weakness, low fever and cognitive dysfunction.The exact nature of symptoms and the number of people who experience long-term symptoms are unknown; these vary according to the definition used, the population being studied, and the time period used in the study. A survey by the UK Office for National Statistics estimated that about 14% of people who tested positive for SARS-CoV-2 experienced one or more symptoms for longer than three months. A study from University of Oxford of 273,618 survivors of COVID-19, mainly from the United States, showed that about 37% experienced one or more symptoms between three and six months after diagnosis.While studies into various aspects of long COVID are under way, as of November 2021, the definition of the illness is still unclear, as is its mechanism. Health systems in some countries and jurisdictions have been mobilized to deal with this group of patients by creating specialized clinics and providing advice. Overall, however, it is considered by default to be a diagnosis of exclusion.\\n\\n\\n== Terminology and definitions ==\\nLong COVID is a patient-created term which was reportedly first used in May 2020 as a hashtag on Twitter by Elisa Perego, an archaeologist at University College London.Long COVID has no single, strict definition. It is normal and expected that people who experience severe symptoms or complications such as post-intensive care syndrome or secondary infections will take longer to recover than people who did not require hospitalization (called mild COVID-19) and had no such complications. It can be difficult to determine whether an individual\\'s set of ongoing symptoms represents a normal, prolonged convalescence, or extended \\'long COVID\\'. One rule of thumb is that long COVID represents symptoms that have been present for longer than two months, though there is no reason to believe that this choice of cutoff is specific to infection with the SARS-CoV-2 virus.\\n\\n\\n=== World Health Organization clinical case definition ===\\nThe World Health Organization (WHO) released a clinical case definition in October 2021, which subsequently was published in the journal The Lancet Infectious Diseases:\\npost-COVID-19 condition occurs in individuals with a history of probable or confirmed SARS-CoV-2 infection, usually 3 months from the onset, with symptoms that last for at least 2 months and cannot be explained by an alternative diagnosis. Common symptoms include, but are not limited to, fatigue, shortness of breath, and cognitive dysfunction, and generally have an impact on everyday functioning. Symptoms might be new onset following initial recovery from an acute COVID-19 episode or persist from the initial illness. Symptoms might also fluctuate or relapse over time.\\n\\n\\n=== British definition ===\\nThe British National Institute for Health and Care Excellence (NICE) divides COVID-19 into three clinical case definitions:\\n\\nacute COVID-19 for signs and symptoms during the first four weeks after infection with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the first, and\\nlong Covid for new or ongoing symptoms four weeks or more after the start of acute COVID-19, which is divided into the other two:\\nongoing symptomatic COVID-19 for effects from four to twelve weeks after onset, and\\npost-COVID-19 syndrome for effects that persist 12 or more weeks after onset.NICE describes the term long COVID, which it uses \"in addition to the clinical case definitions\", as \"commonly used to describe signs and symptoms that continue or develop after acute COVID-19. It includes both ongoing symptomatic COVID-19 (from four to twelve weeks) and post-COVID-19 syndrome (12 weeks or more)\".NICE defines post-COVID-19 syndrome as \"Signs and symptoms that develop during or after an infection consistent with COVID‑19, continue for more than 12 weeks and are not explained by an alternative diagnosis. It usually presents with clusters of symptoms, often overlapping, which can fluctuate and change over time and can affect any system in the body. Post‑COVID‑19 syndrome may be considered before 12 weeks while the possibility of an alternative underlying disease is also being assessed\".\\n\\n\\n=== United States definition ===\\nIn February 2021, the U.S. National Institutes of Health (NIH) Director Francis Collins, indicated long Covid symptoms for individuals who \"don\\'t recover fully over a period of a few weeks\",  be collectively referred to as Post-Acute Sequelae of SARS-CoV-2 infection (PASC). The NIH listed long COVID symptoms of fatigue, shortness of breath, \"brain fog\", sleep disorders, intermittent fevers, gastrointestinal symptoms, anxiety, and depression. Symptoms can persist for months and can range from mild to incapacitating, with new symptoms arising well after the time of infection. The Centers for Disease Control and Prevention (CDC) term Post-Covid Conditions qualifies long Covid as symptoms four or more weeks after first infection.\\n\\n\\n== Symptoms ==\\n\\nA multinational online survey with 3,762 participants with illness lasting more than 28 days found that recovery takes longer than 35 weeks for 91% of them. On average, participants experienced 56 symptoms (standard deviation ± 25.5) in nine organ systems. Symptoms varied over time, and the most common symptoms after six months were fatigue, post-exertional malaise and cognitive dysfunction.Symptom relapse occurred in 86% of participants triggered by physical or mental effort or by stress. Three groups of symptoms were identified: initial symptoms that peak in the first two to three weeks and then subside; stable symptoms; and symptoms that increase markedly in the first two months and then stabilize.Symptoms reported by people with long COVID include:\\n\\n\\n== Epidemiology ==\\nSome reports of long-term illness after infection appeared early during the COVID-19 pandemic, including in people who had a mild (not requiring hospitalization) or moderate (requiring oxygen supplementation) initial infection as well as those who were admitted to hospital with more severe infection.  In a 4-year survey following SARS in Hong Kong, 42.5% of survivors reported at least one diagnosable psychiatric disorder ranging from post-traumatic stress disorders, depression, and chronic fatigue.As of January 2021, the precise incidence was unknown.  The incidence declines over time, as many people slowly recover.  Some early studies suggested that between 20% and 33% of people with COVID-19 experienced symptoms lasting longer than a month.  A telephone survey in the U.S. in the first half of 2020 showed that about 35% of people who had tested positive for SARS-CoV-2 experienced a range of symptoms that lasted longer than three weeks.  As of December 2020, the Office of National Statistics in the UK estimated that, of all people with a positive test for SARS-CoV-2, about 21% experienced symptoms for longer than five weeks, and about 10% experienced symptoms for longer than 12 weeks.Some studies have suggested that some children experience lingering symptoms of SARS-CoV-2 infection.Although anyone who gets infected can develop long COVID, people who become so sick that they require hospitalization take longer to recover. A majority (up to 80%) of those who were admitted to hospital with severe disease experience long-term problems including fatigue and shortness of breath (dyspnoea). Patients with severe initial infection, particularly those who required mechanical ventilation to help breathing, are also likely to suffer from post-intensive care syndrome following recovery.A study of patients who had been hospitalised in Wuhan found that the majority still had at least one symptom after six months. Patients who had been more severely ill still showed severe incapacity in lung function. Among the 1733 patients who had been discharged from hospital and followed up about six months later, the most common symptoms were fatigue or muscle weakness (63%), sleep difficulties (26%), and anxiety or depression (23%).Some people suffer long-term neurologic symptoms despite never having been hospitalized for COVID-19; the first study of this population was published in March 2021. Most frequently, these non-hospitalized patients experienced \"prominent and persistent \\'brain fog\\' and fatigue that affect their cognition and quality of life.\"In January 2021, a study in the UK reported that 30% of recovered patients were readmitted to hospital within 140 days, and 12% of the total died. Many patients had developed diabetes for the first time, as well as problems with heart, liver and kidney problems. The mode of insulin failure was at that point unknown.In March 2021, the Indonesian Doctors Association, in a survey of 463 people, suggested that 63.5% of respondents self-reported lingering symptoms after SARS-CoV-2 infection. The exact set of symptoms was not specified, however, according to the article, fatigue and cough were the most commonly reported symptoms, followed by muscle pain and headache.In May 2021, a global systematic review led by researchers at Stanford University reported that a wide variety of symptoms persisted in more than 70% of COVID-19 patients months after recovering from the initial phase of the disease. Most patients in this study were previously hospitalized. The most common lingering symptoms included shortness of breath, fatigue, and sleep disorders. In total, the study reported 84 clinical signs or symptoms, including loss of taste and smell, cognitive disorders such as memory loss and difficulty concentrating, depression, and anxiety.In June 2021, a long-term study in Bergen, Norway, found 52% (31/61) of a cohort of home-isolated young adults cases, ages 16–30, continued to experience symptoms at six months. The study captured 82% (312) of the pandemic first wave cases: 247 home-isolated and 65 hospitalized.There is little data on long COVID following breakthrough infections (cases in fully vaccinated people). In July 2021, a study with 1,497 fully vaccinated healthcare professionals in Israel reported that 19% of those testing positive (7 out of 36 infected people) also had long COVID-19 symptoms at 6 weeks or more, with the Alpha variant identified in 85% of positive cases.In August 2021, a study of 1,276 hospital survivors of the early 2020 outbreak in Wuhan reported that, while many symptoms disappeared over time and 88% of those previously employed returned to their original work, 49% of them had at least one sequela 12 months after discharge from the hospital, and the general health status of the group was lower than that of the control population. Women had a greater chance of fatigue, muscle weakness, anxiety, depression, or impaired lung diffusion compared to men. The study also found that survivors are at greater risk for psychiatric outcomes such as anxiety and depression.A September 2021 study published in The Lancet found that having two doses of a COVID-19 vaccine halved the odds of long COVID.\\n\\n\\n== Causes ==\\nIt is currently unknown why most people recover fully within two to three weeks and others experience symptoms for weeks or months longer.  Although the exact processes that cause long COVID remain unknown, a number of mechanisms have been suggested.\\nA March 2021 review article cited the following pathophysiological processes as the predominant causes of long COVID:\\ndirect toxicity in virus infected tissue, especially the lungs\\nongoing inflammation due to post-infection immune system dysregulation\\nvascular injury and ischemia caused by virus induced hypercoagulability (tendency to form internal clots) and thromboses (internal blood clots)\\nimpaired regulation of the renin-angiotensin system related to the effect of SARS-CoV-2 on ACE2 containing tissueIn October 2020, a review by the United Kingdom\\'s National Institute for Health Research hypothesized that ongoing long COVID symptoms may be due to four syndromes:\\npermanent damage to the lungs and heart,\\npost-intensive care syndrome,\\nPost-viral fatigue, sometimes regarded as the same as myalgic encephalomyelitis / chronic fatigue syndrome (ME/CFS), and\\ncontinuing COVID-19 symptoms.Other situations that might cause new and ongoing symptoms include:\\n\\nthe virus being present for a longer time than usual, due to an ineffective immune response;\\nreinfection (e.g., with another strain of the virus);\\ndamage caused by inflammation and a strong immune response to the infection;\\nphysical deconditioning due to a lack of exercise while ill;\\npost-traumatic stress or other mental sequelae, especially in people who had previously experienced anxiety, depression, insomnia, or other mental health difficulties;\\ninhibited oxygen exchange as a result of persistent circulating blood plasma microclots; and\\ndevelopment of various autoantibodies after infection.\\n\\n\\n=== Similarities to other syndromes ===\\nLong COVID is similar to post-Ebola syndrome and the post-infection syndromes seen in chikungunya and myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS) which is often triggered by infection and immune activation and was previously also known as “post-viral fatigue.” The pathophysiology of long COVID may be similar to these other conditions. Some long COVID patients in Canada have been diagnosed with ME/CFS, a \"debilitating, multi-system neurological disease that is believed to be triggered by an infectious illness in the majority of cases\". Lucinda Bateman, a specialist in ME/CFS in Salt Lake City in the US believes that the two syndromes are identical. There is need for more research into ME/CFS; Anthony Fauci, chief medical adviser to the US government, said that COVID-19 is a \"well-identified etiologic agent that should be very helpful now in getting us to be able to understand [ME/CFS]\".\\n\\n\\n=== Risk factors ===\\nAccording to a King\\'s College London study initially posted on 21 October 2020 risk factors for long COVID may include:\\nAge – particularly those aged over 50\\nObesity\\nAsthma\\nReporting more than five symptoms (e.g. more than cough, fatigue, headache, diarrhoea, loss of sense of smell) in the first week of COVID-19 infection; five is the median number reportedWomen are less likely to develop severe acute COVID but more likely to develop long COVID than men.  Some research suggests this is due primarily to hormonal differences, while other research points to other factors, including chromosomal genetics, sex-dependent differences in immune system behavior, and non-biological factors may be relevant.\\n\\n\\n== Diagnosis ==\\nXenon MRI is being used to study long COVID, because it provides patients and physicians with explanations for previously unexplained observations. Xenon MRI can measure gas exchange and provide information on how much air is taken up by a patient\\'s bloodstream, which is being researched in long-haul COVID patients.Xenon MRI can quantify three components of lung function: ventilation, barrier tissue uptake and gas exchange. Xenon-129 is soluble in pulmonary tissue, which allows the evaluation of lung functions such as perfusion and gas exchange (an advantage over helium). Ventilation measures how the air is distributed in the lung and can provide the locations of potentially compromised lung areas if no xenon reaches those areas. Barrier tissue uptake and gas exchange measure how much air diffuses across the alveolar-capillary membrane. Xenon MRI helps determine how well air is taken in by the lungs, absorbed into lung tissue, and taken up by the blood.\\n\\n\\n== Health system responses ==\\n\\n\\n=== Australia ===\\nIn October 2020, a guide published by the Royal Australian College of General Practitioners (RACGP) says that ongoing post-COVID-19 infection symptoms such as fatigue, shortness of breath and chest pain will require management by GPs, in addition to the more severe conditions already documented.In December 2021, research by a health economics expert at Deakin University suggests that even without fully understanding the Omicron variant\\'s effects yet, a further 10,000 to 133,000 long COVID cases are likely to emerge on top of the current approximately 9450 in New South Wales and 19,800 in  Victoria, after border and other restrictions had been recently lifted. The RACGP released new guidelines for general practitioners to manage a large number of new long COVID patients.\\n\\n\\n=== South Africa ===\\nIn October 2020, the DATCOV Hospital Surveillance Department of the National Institute for Communicable Diseases (NICD) looked into a partnership with the International Severe Acute Respiratory and emerging Infection Consortium (ISARIC) (an open-access research and data resource) in order to conduct clinical research into the impact PASC may have within the South African context. As of 30 January 2021, the project has yet to receive ethical approval for the commencement of data collection. Ethics approval was granted on 3 February 2021 and formal data collection began on 8 February 2021.\\n\\n\\n=== United Kingdom ===\\nIn Britain, the National Health Service set up specialist clinics for the treatment of long COVID. The four Chief Medical Officers of the UK were warned of academic concern over long COVID on 21 September 2020 in a letter written by Trisha Greenhalgh published in The BMJ signed by academics including David Hunter, Martin McKee, Susan Michie, Melinda Mills, Christina Pagel, Stephen Reicher, Gabriel Scally, Devi Sridhar, Charles Tannock, Yee Whye Teh, and Harry Burns, former CMO for Scotland.In October 2020, NHS England\\'s head Simon Stevens announced the NHS had committed £10 million to be spent that year on setting up long COVID clinics to assess patients\\' physical, cognitive, and psychological conditions and to provide specialist treatment. Future clinical guidelines were announced, with further research on 10,000 patients planned and a designated task-force to be set up, along with an online rehabilitation service – \"Your Covid Recovery\". The clinics include a variety of medical professionals and therapists, with the aim of providing \"joined-up care for physical and mental health\".The National Institute for Health Research has allocated funding for research into the mechanisms behind symptoms of Long COVID.In December 2020, University College London Hospitals (UCLH) opened a second Long Covid clinic at the National Hospital for Neurology and Neurosurgery for patients with post-Covid neurological issues. The first clinic had opened in May, primarily focused on respiratory problems, but both clinics refer patients to other specialists where needed, including cardiologists, physiotherapists and psychiatrists.  By March 2021 there were 69 long covid clinics in the English NHS. mostly focussing on assessing patients, with more planned to open. There were fears that community rehabilitation services did not have capacity to manage large numbers of referrals.On 18 December 2020, the National Institute for Health and Care Excellence (NICE), the Royal College of General Practitioners (RCGP) and the Scottish Intercollegiate Guidelines Network (SIGN) published a guide to the management of Long COVID. The guideline was reviewed by representatives of the UK doctors #longcovid group, an online support group for long covid haulers, who said that it could be improved by introducing a more comprehensive description of the clinical features and physical nature of long COVID, among other changes.In November 2021 complaints were reported from NHS staff that neither their employers nor their trades unions were supportive, though the British Medical Association was pushing for long-covid to be classed as an occupational disease.\\n\\n\\n=== United States ===\\nDr. Anthony S. Fauci has described long-term COVID-19 as \"...a phenomenon that is really quite real and quite extensive,\" but also said that the number of cases is unknown.On 23 February 2021, National Institutes of Health Director Francis Collins announced a major initiative to identify the causes and ultimately the means of prevention and treatment of people who are suffering from long COVID. Part of this initiative includes the creation of the COVID-19 Project, which will gather data on neurological symptoms associated with PASC.\\nOn 28 April 2021, the Subcommittee on Health of the U.S. House of Representatives\\' Committee on Energy and Commerce held a hearing about long COVID. In February 2022 it was announced that at least sixty-six hospitals and health systems had launched COVID recovery programs to aid patients who experience long term or lingering sympotoms.\\n\\n\\n== Public response ==\\nSome people experiencing long COVID have formed groups on social media sites. There is an active international long COVID patient advocacy movement which includes research led by patients themselves. In many of these groups, individuals express frustration and their sense that their problems have been dismissed by medical professionals.\\n\\n\\n== Special populations ==\\n\\n\\n=== Children ===\\nA study from the Office for National Statistics with 20,000 participants, including children and adults, found that, in children who tested positive, at least one symptom persisted after five weeks in 9.8% of children aged two to eleven years and in 13% of children aged 12 to 16 years. A 2022 University College London study in the UK, found that children ages 11-17 who had a positive PCR test for COVID were more likely to have three or more symptoms three months after their diagnosis compared to those with a negative test. A study in Italy, which analyzed 129 children under the age of 18, examined health data obtained via a questionnaire between September 2020 and 1 January 2021. 53% of the group experienced COVID-19 symptoms more than 120 days after their diagnosis, and 43% were still impaired by the symptoms. Symptoms included insomnia, fatigue, muscle pain, chest tightness and pain, nasal congestion, tiredness, and difficulty concentrating. A case report of five children in Sweden also reported symptoms (fatigue, heart palpitations, dyspnoea, headaches, muscle weakness and difficulty concentrating) persisting for 6–8 months after diagnosis.\\n\\n\\n== See also ==\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\n\\n\\n=== News ===\\n\\n\\n== External links ==\\n Long Covid on YouTube (21 October 2020) – UK Government film about long COVID.\\nLiving with Covid19. A review of what is known about long Covid by the NIHR.\\n\"PHOSP\". Home. University of Leicester. The Post-hospitalisation COVID-19 study (PHOSP-COVID) is a consortium of leading researchers and clinicians from across the UK working together to understand and improve long-term health outcomes for patients who have been in hospital with confirmed or suspected COVID-19.',\n",
              "  1.197186317095343),\n",
              " ('Symptoms of COVID-19 are variable, ranging from mild symptoms to critical and possibly fatal illness. Common symptoms include coughing, fever, loss of smell (anosmia) and taste (ageusia), with less common ones including headaches, nasal congestion and runny nose, muscle pain, sore throat, diarrhea, eye irritation, and toes swelling or turning purple, and in moderate to severe cases breathing difficulties. People with the COVID-19 infection may have different symptoms, and their symptoms may change over time. Three common clusters of symptoms have been identified: one respiratory symptom cluster with cough, sputum, shortness of breath, and fever; a musculoskeletal symptom cluster with muscle and joint pain, headache, and fatigue; a cluster of digestive symptoms with abdominal pain, vomiting, and diarrhea. In people without prior ear, nose, and throat disorders, loss of taste combined with loss of smell is associated with COVID-19 and is reported in as many as 88% of symptomatic cases.Of people who show symptoms, 81% develop only mild to moderate symptoms (up to mild pneumonia), while 14% develop severe symptoms (dyspnea, hypoxia, or more than 50% lung involvement on imaging) which requiring hospitalization and 5% of patients suffer critical symptoms (respiratory failure, septic shock, or multiorgan dysfunction) requiring ICU admission. At least a third of the people who are infected with the virus do not develop noticeable symptoms at any point in time. These asymptomatic carriers tend not to get tested and can still spread the disease. Other infected people will develop symptoms later, called \"pre-symptomatic\", or have very mild symptoms and can also spread the virus.As is common with infections, there is a delay between the moment a person first becomes infected and the appearance of the first symptoms. The median delay for COVID-19 is four to five days possibly being infectious on 1-4 of those days. Most symptomatic people experience symptoms within two to seven days after exposure, and almost all will experience at least one symptom within 12 days.Most people recover from the acute phase of the disease. However, some people – over half of a cohort of home-isolated young adults – continue to experience a range of effects, such as fatigue, for months even after recovery, a condition called long COVID; long-term damage to organs has been observed. Multi-year studies are underway to further investigate the potential long-term effects of the disease.\\n\\n\\n== Overview ==\\n\\nSome less common symptoms of COVID-19 can be relatively non-specific; however the most common symptoms are fever, dry cough, and loss of taste and smell.  Among those who develop symptoms, approximately one in five may become more seriously ill and have difficulty in breathing. Emergency symptoms include difficulty in breathing, persistent chest pain or pressure, sudden confusion, loss of mobility and speech, and bluish face or lips; immediate medical attention is advised if these symptoms are present. Further development of the disease can lead to complications including pneumonia, acute respiratory distress syndrome, sepsis, septic shock, and kidney failure.\\nSome symptoms usually appear sooner than others. In August 2020, scientists at the University of Southern California reported the \"likely\" order of initial symptoms of the COVID-19 disease as a fever followed by a cough and muscle pain, and that nausea and vomiting usually appear before diarrhea. This contrasts with the most common path for influenza where it is common to develop a cough first and fever later. While health agency guidelines tend to recommend isolating for 14 days while watching for symptoms to develop, there is limited evidence that symptoms may develop for some patients more than 14 days after initial exposure.\\n\\n\\n== Different Covid variants ==\\nThe frequency of symptoms predominating for people suffering from different variants may differ from what was observed in the earlier phases of the pandemic.\\n\\n\\n=== Delta ===\\nPeople infected with the Delta variant may mistake the symptoms for a bad cold and not realize they need to isolate. Common symptoms reported as of June 2021 have been headaches, sore throat, runny nose, and fever. In the United Kingdom, when the Delta variant accounted for 91 percent of new cases at that time, one study found that the most reported symptoms were headache, sore throat, and runny nose.\\n\\n\\n=== Omicron ===\\n\\nBritish epidemiologist Tim Spector said in mid-December 2021 that the majority of symptoms of the Omicron variant were the same as a common cold, including headaches, sore throat, runny nose, fatigue and sneezing, so that people with cold symptoms should take a test. \"Things like fever, cough and loss of smell are now in the minority of symptoms we are seeing. Most people don’t have classic symptoms.\" People with cold symptoms in London (where Covid was spreading rapidly) are \"far more likely\" to have Covid than a cold.A unique reported symptom of the Omicron variant is night sweats. Also, loss of taste and smell seem to be uncommon compared to other strains.\\n\\n\\n== Systemic ==\\nTypical systemic symptoms include fatigue, and muscle and joint pains. Some people have a sore throat.\\n\\n\\n=== Fever ===\\nFever is one of the most common symptoms in COVID-19 patients.  However, the absence of the symptom itself at an initial screening does not rule out COVID-19.  Fever in the first week of a COVID-19 infection is part of the body\\'s natural immune response; however in severe cases, if the infections develop into a cytokine storm the fever is counterproductive.  As of September 2020, little research had focused on relating fever intensity to outcomes.A June 2020 systematic review reported a 75–81% prevalence of fever.  As of July 2020, the European Centre for Disease Prevention and Control (ECDC) reported a prevalence rate of ~45% for fever.\\n\\n\\n=== Pain ===\\nA June 2020 systematic review reported a 27–35% prevalence of fatigue, 14–19% for muscle pain, 10–14% for sore throat.  As of July 2020, the ECDC reported a prevalence rate of ~63% for muscle weakness (asthenia), ~63% for muscle pain (myalgia), and ~53% for sore throat.\\n\\n\\n== Respiratory ==\\nCough is another typical symptom of COVID-19, which could be either dry or a productive cough.Some symptoms, such as difficulty breathing, are more common in patients who need hospital care.  Shortness of breath tends to develop later in the illness. Persistent anosmia or hyposmia or ageusia or dysgeusia has been documented in 20% of cases for longer than 30 days.Respiratory complications may include pneumonia and acute respiratory distress syndrome (ARDS).As of July 2020, the ECDC reported a prevalence rate of ~68% for nasal obstruction, ~63% for cough, ~60% for rhinorrhoea or runny nose.  A June 2020 systematic review reported a 54–61% prevalence of dry cough and 22–28% for productive cough.\\n\\n\\n== Cardiovascular ==\\nCardiovascular complications may include heart failure, arrhythmias, heart inflammation, and blood clots. They are common traits in severe COVID-19 patients due to the relation with the respiratory system.Hypertension seems to be the most prevalent risk factor for myocardial injury in COVID-19 disease. It was reported in 58% of individuals with cardiac injury in a recent meta-analysis.Several cases of acute myocarditis associated with COVID-19 have been described around the globe and are diagnosed in multiple ways. Taking into consideration serology, leukocytosis with neutrophilia and lymphopenia was found in many patients. Cardiac biomarkers troponin and N-terminal (NT)-prohormone BNP (NT-proBNP) were seen elevated. Similarly, the level of inflammation-related markers such as C-reactive protein (CRP), D-dimer, IL-6, procalcitonin was significantly increased, indicating an inflammatory process in the body. Electrocardiogram findings were variable and ranged from sinus tachycardia, ST-segment elevation, T-wave inversion and ST-depression. In one case, viral particles were seen in the interstitial cell, and another case reported SARS-CoV-2 RT–PCR positivity in the cardiac tissue suggestive of direct viral injury to the myocardium. Endomyocardial biopsy [EMB] remains the gold standard invasive technique in diagnosing myocarditis; however, due to the increased risk of infection, it is not done in COVID-19 patients.\\nThe binding of the SARS-CoV-2 virus through ACE2 receptors present in heart tissue may be responsible for direct viral injury leading to myocarditis. In a study done during the SARS outbreak, SARS virus RNA was ascertained in the autopsy of heart specimens in 35% of the patients who died due to SARS. It was also observed that an already diseased heart has increased expression of ACE2 receptor contrasted to healthy individuals. Hyperactive immune responses in COVID-19 Patients may lead to the initiation of the cytokine storm. This excess release of cytokines may lead to myocardial injury.\\n\\n\\n== Neurological ==\\nPatients with COVID-19 can present with neurological symptoms that can be broadly divided into central nervous system involvement, such as headache, dizziness, altered mental state, and disorientation, and peripheral nervous system involvement, such as anosmia and dysgeusia. Some patients experience cognitive dysfunction called \"COVID fog\", or \"COVID brain fog\", involving memory loss, inattention, poor concentration or disorientation.  Other neurologic manifestations include seizures, strokes, encephalitis, and Guillain–Barré syndrome (which includes loss of motor functions).As of July 2020, the ECDC reported a prevalence rate of ~70% for headache.  A June 2020 systematic review reported a 10–16% prevalence of headache.\\n\\n\\n=== Loss of smell ===\\n\\nIn some people, COVID-19 causes people to temporarily lose their sense of smell either partially or fully.This symptom, if it is present at all, often appears early in the illness. Its onset is often reported to be sudden.  Smell usually returns to normal within a month.  However, for some patients it improves very slowly and is associated with odors being perceived as unpleasant or different than they originally did (parosmia), and for some people smell does not return for at least many months.  It is an unusual symptom for other respiratory diseases, so it is used for symptom-based screening.Loss of smell has several consequences.  Loss of smell increases foodborne illness due to inability to detect spoiled food, and may increase fire hazards due to inability to detect smoke.  It has also been linked to depression.  If smell does not return, smell training is a potential option.It is sometimes the only symptom to be reported, implying that it has a neurological basis separate from nasal congestion.  As of January 2021, it is believed that these symptoms are caused by infection of sustentacular cells that support and provide nutrients to sensory neurons in the nose, rather than infection of the neurons themselves.  Sustentacular cells have many Angiotensin-converting enzyme 2 (ACE2) receptors on their surfaces, while olfactory sensory neurons do not.  Loss of smell may also be the result of inflammation in the olfactory bulb.A June 2020 systematic review found a 29–54% prevalence of olfactory dysfunction for people with COVID-19, while an August 2020 study using a smell-identification test reported that 96% of people with COVID-19 had some olfactory dysfunction, and 18% had total smell loss.  Another June 2020 systematic review reported a 4–55% prevalence of hyposmia.  As of July 2020, the ECDC reported a prevalence rate of ~70% for loss of smell.A disturbance in smell or taste is more commonly found in younger people, and perhaps because of this, it is correlated with a lower risk of medical complications.\\n\\n\\n=== Loss of taste and chemesthesis ===\\nIn some people, COVID-19 causes people to temporarily experience changes in how food tastes (dysgeusia or ageusia).  Changes to chemesthesis, which includes chemically triggered sensations such as spiciness, are also reported.  As of January 2021, the mechanism for taste and chemesthesis symptoms were not well understood.A June 2020 systematic review found a 24–54% prevalence of gustatory dysfunction for people with COVID-19.  Another June 2020 systematic review reported a 1–8% prevalence of hypogeusia.  As of July 2020, the ECDC reported a prevalence rate of ~54% for gustatory dysfunction.\\n\\n\\n=== Other neurological and psychiatric symptoms ===\\nOther neurological symptoms appear to be rare, but may affect half of patients who are hospitalized with severe COVID-19.  Some reported symptoms include delirium, stroke, brain hemorrhage, memory loss, psychosis, peripheral nerve damage, anxiety, and post-traumatic stress disorder.  Neurological symptoms in many cases are correlated with damage to the brain\\'s blood supply or encephalitis, which can progress in some cases to acute disseminated encephalomyelitis.  Strokes have been reported in younger people without conventional risk factors.As of September 2020, it was unclear whether these symptoms were due to direct infection of brain cells, or of overstimulation of the immune system.A June 2020 systematic review reported a 6–16% prevalence of vertigo or dizziness, 7–15% for confusion, and 0–2% for ataxia.\\n\\n\\n== Blood clots and bleeding ==\\nPatients are at increased risk of a range of different blood clots, some potentially fatal, for months following COVID infection.  The Guardian wrote, \"Overall, they [a Swedish medical team] identified a 33-fold increase in the risk of pulmonary embolism, a fivefold increase in the risk of DVT [(Deep vein thrombosis)] and an almost twofold increase in the risk of bleeding in the 30 days after infection. People remained at increased risk of pulmonary embolism for six months after becoming infected, and for two and three months for bleeding and DVT.  Although the risks were highest in patients with more severe illness, even those with mild Covid had a threefold increased risk of DVT and a sevenfold increased risk of pulmonary embolism. No increased risk of bleeding was found in those who experienced mild infections.\"  Anne-Marie Fors Connolly at Umeå University said, “If you suddenly find yourself short of breath, and it doesn’t pass, [and] you’ve been infected with the coronavirus, then it might be an idea to seek help, because we find this increased risk for up to six months.”\\n\\n\\n== Other ==\\nOther symptoms are less common among people with COVID-19. Some people experience gastrointestinal symptoms such as loss of appetite, diarrhea, nausea or vomiting.  A June 2020 systematic review reported a 8–12% prevalence of diarrhea, and 3–10% for nausea.Less common symptoms include chills, coughing out blood, diarrhea, and rash.  The so-called \"COVID toes\" are pink to violaceous papules arising on the hands and feet. These chilblain-like lesions often occur only in younger patients and do not appear until late in the disease or during convalescence. Certain genetic polymorphisms (in the TREX1 gene) have been linked to susceptibility towards developing COVID-toe.  A June 2020 systematic review reported a 0–1% prevalence of rash in COVID-19 patients.Approximately 20–30% of people who present with COVID-19 have elevated liver enzymes, reflecting liver injury.Complications include multi-organ failure, septic shock, and death.\\n\\n\\n== Stages of COVID-19 infection ==\\nThere are three stages, according to the way COVID-19 infection can be tackled by pharmacological agents, in which the disease can be classified. Stage I is the early infection phase during which the domination of upper respiratory tract symptoms is present. Stage II is the pulmonary phase in which the patient develops pneumonia with all its associated symptoms; this stage is split with Stage IIa is without hypoxia and Stage IIb having hypoxia. Stage III is the hyperinflammation phase, the most severe phase, in which the patient develops acute respiratory distress syndrome (ARDS), sepsis and multi-organ failure.\\nA similar stereotyped course was postulated to be: the first phase of an incubation period, a second phase corresponding to the viral phase, a third phase corresponding to the state of inflammatory pneumonia, a fourth phase corresponding to the brutal clinical aggravation reflected by acute respiratory distress syndrome (ARDS), and finally, in survivors, a fifth phase potentially including lung fibrosis, and persisting in the form of “post-covid” symptoms.\\n\\n\\n== Longer-term effects ==\\n\\n\\n=== Multisystem inflammatory syndrome in children ===\\n\\nFollowing the infection, children may develop multisystem inflammatory syndrome, also called paediatric multisystem inflammatory syndrome.  This has symptoms similar to Kawasaki disease, which can possibly be fatal.\\n\\n\\n=== Long COVID ===\\n\\n\\n== References ==',\n",
              "  1.183299417489003),\n",
              " ('COVID-19 testing involves analyzing samples to assess the current or past presence of SARS-CoV-2. The two main branches detect either the presence of the virus or of antibodies produced in response to infection. Molecular tests for viral presence through its molecular components are used to diagnose individual cases and to allow public health authorities to trace and contain outbreaks. Antibody tests (serology immunoassays) instead show whether someone once had the disease. They are less useful for diagnosing current infections because antibodies may not develop for weeks after infection. It is used to assess disease prevalence, which aids the estimation of the infection fatality rate.Individual jurisdictions have adopted varied testing protocols, including whom to test, how often to test, analysis protocols, sample collection and the uses of test results. This variation has likely significantly impacted reported statistics, including case and test numbers, case fatality rates and case demographics. Because SARS-CoV-2 transmission occurs days after exposure (and before onset of symptoms), there is an urgent need for frequent surveillance and rapid availability of results.Test analysis is often performed in automated, high-throughput, medical laboratories by medical laboratory scientists. Alternatively, point-of-care testing can be done in physician\\'s offices and parking lots, workplaces, institutional settings or transit hubs.\\n\\n\\n== Methods ==\\n\\nPositive viral tests indicate a current infection, while positive antibody tests indicate a prior infection. Other techniques include a CT scan, checking for elevated body temperature, checking for low blood oxygen level, and detection by trained dogs.\\n\\n\\n=== Detection of the virus ===\\nDetection of the virus is usually done either by looking for the virus\\'s inner RNA, or pieces of protein on the outside of the virus. Tests that look for the viral antigens (parts of the virus) are called antigen tests.\\nThere are multiple types of tests that look for the virus by detecting the presence of the virus\\'s RNA.  These are called nucleic acid or molecular tests, after molecular biology.  As of 2021, the most common form of molecular test is the reverse transcription polymerase chain reaction (RT-PCR) test.  Other methods used in molecular tests include CRISPR, isothermal nucleic acid amplification, digital polymerase chain reaction, microarray analysis, and next-generation sequencing.\\n\\n\\n==== Reverse transcription polymerase chain reaction (RT-PCR) test ====\\nPolymerase chain reaction (PCR) is a process that amplifies (replicates) a small, well-defined segment of DNA many hundreds of thousands of times, creating enough of it for analysis. Test samples are treated with certain chemicals that allow DNA to be extracted. Reverse transcription converts RNA into DNA.\\nReverse transcription polymerase chain reaction (RT-PCR) first uses reverse transcription to obtain DNA, followed by  PCR to amplify that DNA, creating enough to be analyzed. RT-PCR can thereby detect SARS-CoV-2, which contains only RNA. The RT-PCR process generally requires a few hours. These tests are also referred to as molecular or genetic assays.Real-time PCR (qPCR) provides advantages including automation, higher-throughput and more reliable instrumentation. It has become the preferred method.The combined technique has been described as real-time RT-PCR or quantitative RT-PCR and is sometimes abbreviated qRT-PCR, rRT-PCR or RT-qPCR, although sometimes RT-PCR or PCR are used. The Minimum Information for Publication of Quantitative Real-Time PCR Experiments (MIQE) guidelines propose the term RT-qPCR, but not all authors adhere to this.\\nAverage sensitivity for rapid molecular tests depend on the brand. For ID NOW, the average sensitivity was 73.0% with an average specificity of 99.7%; for Xpert Xpress the average sensitivity was 100% with an average specificity of 97.2%.\\nIn a diagnostic test, sensitivity is a measure of how well a test can identify true positives and specificity is a measure of how well a test can identify true negatives. For all testing, both diagnostic and screening, there is usually a trade-off between sensitivity and specificity, such that higher sensitivities will mean lower specificities and vice versa.   A 90% specific test will correctly identify 90% of those who are uninfected, leaving 10% with a false positive result.\\nSamples can be obtained by various methods, including a nasopharyngeal swab, sputum (coughed up material), throat swabs, deep airway material collected via suction catheter or saliva. Drosten et al. remarked that for 2003 SARS, \"from a diagnostic point of view, it is important to note that nasal and throat swabs seem less suitable for diagnosis, since these materials contain considerably less viral RNA than sputum, and the virus may escape detection if only these materials are tested.\"Sensitivity of clinical samples by RT-PCR is 63% for nasal swab, 32% for pharyngeal swab, 48% for feces, 72–75% for sputum, and 93–95% for bronchoalveolar lavage.The likelihood of detecting the virus depends on collection method and how much time has passed since infection. According to Drosten tests performed with throat swabs are reliable only in the first week. Thereafter the virus may abandon the throat and multiply in the lungs. In the second week, sputum or deep airways collection is preferred.Collecting saliva may be as effective as nasal and throat swabs, although this is not certain. Sampling saliva may reduce the risk for health care professionals by eliminating close physical interaction. It is also more comfortable for the patient. Quarantined people can collect their own samples. A saliva test\\'s diagnostic value depends on sample site (deep throat, oral cavity, or salivary glands). Some studies have  found that saliva yielded greater sensitivity and consistency when compared with swab samples.On 15 August 2020, the US FDA granted an emergency use authorization for a saliva test developed at Yale University that gives results in hours.On 4 January 2021, the US FDA issued an alert about the risk of false results, particularly false negative results, with the Curative SARS-Cov-2 Assay real-time RT-PCR test.Viral burden measured in upper respiratory specimens declines after symptom onset. Following recovery, many patients no longer have detectable viral RNA in upper respiratory specimens. Among those who do, RNA concentrations three days following recovery are generally below the range in which replication-competent virus has been reliably isolated. No clear correlation has been described between length of illness and duration of post-recovery shedding of viral RNA in upper respiratory specimens.\\n\\n\\t\\t\\n\\t\\t\\n\\t\\t\\n\\n\\n==== Other molecular tests ====\\nIsothermal nucleic acid amplification tests also amplify the virus\\'s genome. They are faster than PCR because they don\\'t involve repeated heating and cooling cycles. These tests typically detect DNA using fluorescent tags, which are read out with specialized machines.CRISPR gene editing technology was modified to perform the detection: if the CRISPR enzyme attaches to the sequence, it colors a paper strip. The researchers expect the resulting test to be cheap and easy to use in point-of-care settings. The test amplifies RNA directly, without the RNA-to-DNA conversion step of RT-PCR.\\n\\n\\n==== Antigen tests ====\\n\\nAn antigen is the part of a pathogen that elicits an immune response. Antigen tests look for antigen proteins from the viral surface. In the case of a coronavirus, these are usually proteins from the surface spikes. SARS-CoV-2 antigens can be detected before onset of COVID-19 symptoms (as soon as SARS-CoV-2 virus particles) with more rapid test results, but with less sensitivity than PCR tests for the virus.Typical antigen tests are COVID-19 rapid antigen tests, often known as Lateral Flow Tests (LFT).\\nAntigen tests may be one way to scale up testing to much greater levels. Isothermal nucleic acid amplification tests can process only one sample at a time per machine. RT-PCR tests are accurate but require too much time, energy and trained personnel to run the tests. \"There will never be the ability on a [PCR] test to do 300 million tests a day or to test everybody before they go to work or to school,\" Deborah Birx, head of the White House Coronavirus Task Force, said on 17 April 2020. \"But there might be with the antigen test.\"Samples may be collected via nasopharyngeal swab, a swab of the anterior nares, or from saliva (obtained by various methods including lollipop tests for children). The sample is then exposed to paper strips containing artificial antibodies designed to bind to coronavirus antigens. Antigens bind to the strips and give a visual readout. The process takes less than 30 minutes, can deliver results at point of care, and does not require expensive equipment or extensive training.Swabs of respiratory viruses often lack enough antigen material to be detectable. This is especially true for asymptomatic patients who have little if any nasal discharge. Viral proteins are not amplified in an antigen test. A Cochrane review based on 64 studies investigating the efficacy of 16 different antigen tests determined that they correctly identified COVID-19 infection in an average of 72% of people with symptoms, compared to 58% of people without symptoms. Tests were most accurate (78%) when used in the first week after symptoms first developed, likely because people have the most virus in their system in the first days after they are infected.  While some scientists doubt whether an antigen test can be useful against COVID-19, others have argued that antigen tests are highly sensitive when viral load is high and people are contagious, making them suitable for public health screening. Routine antigen tests can quickly identify when asymptomatic people are contagious, while follow-up PCR can be used if confirmatory diagnosis is needed.\\n\\n\\n=== Antibody tests ===\\n\\nThe body responds to a viral infection by producing antibodies that help neutralize the virus. Blood tests (also called serology tests or serology immunoassays) can detect the presence of such antibodies. Antibody tests can be used to assess what fraction of a population has once been infected, which can then be used to calculate the disease\\'s mortality rate. They can also be used to determine how much antibody is contained in a unit of convalescent plasma, for COVID-19 treatment, or to verify if a given vaccine generates an adequate immune response.SARS-CoV-2 antibodies\\' potency and protective period have not been established.  Therefore, a positive antibody test may not imply immunity to a future infection. Further, whether mild or asymptomatic infections produce sufficient antibodies for a test to detect has not been established. Antibodies for some diseases persist in the bloodstream for many years, while others fade away.The most notable antibodies are IgM and IgG. IgM antibodies are generally detectable several days after initial infection, although levels over the course of infection and beyond are not well characterized. IgG antibodies generally become detectable 10–14 days after infection and normally peak around 28 days after infection. This pattern of antibody development seen with other infections, often does not apply to SARS-CoV-2, however, with IgM sometimes occurring after IgG, together with IgG or not occurring at all.  Generally, however, median IgM detection occurs 5 days after symptom onset, whereas IgG is detected a median 14 days after symptom onset. IgG levels significantly decline after two or three months.Genetic tests verify infection earlier than antibody tests. Only 30% of those with a positive genetic test produced a positive antibody test on day 7 of their infection.\\n\\n\\n==== Antibody Test Types ====\\n\\n\\n===== Rapid diagnostic test (RDT) =====\\nRDTs typically use a small, portable, positive/negative lateral flow assay that can be executed at point of care. RDTs may process blood samples, saliva samples, or nasal swab fluids. RDTs produce colored lines to indicate positive or negative results.\\n\\n\\n===== Enzyme-linked immunosorbent assay (ELISA) =====\\nELISAs can be qualitative or quantitative and generally require a lab. These tests usually use whole blood, plasma, or serum samples. A plate is coated with a viral protein, such as a SARS-CoV-2 spike protein. Samples are incubated with the protein, allowing any antibodies to bind to it. The antibody-protein complex can then be detected with another wash of antibodies that produce a color/fluorescent readout.\\n\\n\\n===== Neutralization assay =====\\nNeutralization assays assess whether sample antibodies prevent viral infection in test cells. These tests sample blood, plasma or serum. The test cultures cells that allow viral reproduction (e.g., Vero E6 cells). By varying antibody concentrations, researchers can visualize and quantify how many test antibodies block virus replication.\\n\\n\\n===== Chemiluminescent immunoassay =====\\nChemiluminescent immunoassays are quantitative lab tests. They sample blood, plasma, or serum. Samples are mixed with a known viral protein, buffer reagents and specific, enzyme-labeled antibodies. The result is luminescent. A chemiluminescent microparticle immunoassay uses magnetic, protein-coated microparticles. Antibodies react to the viral protein, forming a complex. Secondary enzyme-labeled antibodies are added and bind to these complexes. The resulting chemical reaction produces light. The radiance is used to calculate the number of antibodies. This test can  identify multiple types of antibodies, including IgG, IgM, and IgA.\\n\\n\\n==== Neutralizing vis-à-vis binding antibodies ====\\nMost if not all large scale COVID-19 antibody testing looks for binding antibodies only and does not measure the more important neutralizing antibodies (NAb). A NAb is an antibody that neutralizes the infectivity of a virus particle by blocking its attachment to or entry into a susceptible cell; enveloped viruses, like e.g. SARS-CoV-2, are neutralized by the blocking of steps in the replicative cycle up to and including membrane fusion. A non-neutralizing antibody either does not bind to the crucial structures on the virus surface or binds but leaves the virus particle infectious; the antibody may still contribute to the destruction of virus particles or infected cells by the immune system. It may even enhance infectivity by interacting with receptors on macrophages. Since most COVID-19 antibody tests return a positive result if they find only binding antibodies, these tests cannot indicate that the subject has generated protective NAbs that protect against re-infection.It is expected that binding antibodies imply the presence of NAbs and for many viral diseases total antibody responses correlate somewhat with NAb responses but this is not established for COVID-19. A study of 175 recovered patients in China who experienced mild symptoms reported that 10 individuals had no detectable NAbs at discharge, or thereafter. How these patients recovered without the help of NAbs and whether they were at risk of re-infection was not addressed. An additional source of uncertainty is that even if NAbs are present, viruses such as HIV can evade NAb responses.Studies have indicated that NAbs to the original SARS virus (the predecessor to the current SARS-CoV-2) can remain active for two years and are gone after six years. Nevertheless, memory cells including Memory B cells and Memory T cells can last much longer and may have the ability to reduce reinfection severity.\\n\\n\\t\\t\\n\\t\\t\\n\\n\\n=== Other tests ===\\n\\n\\n==== Sniff tests ====\\nSudden loss of smell can be used to screen people on a daily basis for COVID-19. A study by the National Institutes of Health showed that those infected with SARS-CoV-2 could not smell a 25% mixture of ethanol and water.  Because various conditions can lead to the loss of the sense of smell, a sniff test would not be definitive but indicate the need for a PCR test. Because the loss of the sense of smell shows up before other symptoms, there has been a call for widespread sniff testing. Health care bureaucracies have generally ignored sniff tests even though they are quick, easy and capable of being self-administered daily. This has led some medical journals to write editorials supporting the adoption of sniff testing.\\n\\n\\n==== Imaging ====\\nTypical visible features on CT initially include bilateral multilobar ground-glass opacities with a peripheral or posterior distribution. COVID-19 can be identified with higher precision using CT than with RT-PCR.Subpleural dominance, crazy paving, and consolidation may develop as the disease evolves. Chest CT scans and chest x-rays are not recommended for diagnosing COVID-19. Radiologic findings in COVID-19 lack specificity.\\n\\n\\n==== Serology (CoLab score) tests ====\\nThe standard blood test (quick scan) taken at the emergency room measures different values. By use of the blood quick scan the CoLab score is calculated with a developed algorithm based on how the coronavirus causes changes in the blood. The software is intended for use in emergency rooms to quickly rule out the presence of the disease in incoming patients. A not negative result is followed by a PCR (polymerase chain reaction) or LAMP (loop-mediated isothermal amplification) test.\\n\\n\\n==== Breath tests ====\\nThe breath test by a Coronavirus breathalyzer is a pre-screening test for people who have no or mild symptoms of COVID-19. A not negative result is followed by a PCR or LAMP test.\\n\\n\\n==== Animals ====\\nIn May 2021, Reuters reported that Dutch researchers at Wageningen University had shown that trained bees could detect the virus in infected samples in seconds and this could benefit countries where test facilities are in short supply.  A two-month study by the Necker-Cochin hospital Paris in conjunction with the French national veterinary school reported in May 2021 that dogs were more reliable than current lateral flow tests.Researchers in Paris in March 2022 reported in a preprint not yet peer-reviewed that trained dogs were very effective for rapidly detecting the presence of SARS-Cov2 in people, whether displaying symptoms or not. The dogs were presented with sweat samples to smell from 335 people, of whom 78 with symptoms and 31 without tested positive by PCR. The dogs detected 97% of the symptomatic and 100% of the asymptomatic infections. They were 91% accurate at identifying volunteers who were not infected, and 94% accurate at ruling out the infection in people without symptoms. The authors said \"Canine testing is non-invasive and provides immediate and reliable results.Further studies will be focused on direct sniffing by dogs to evaluate sniffer dogs for mass pre-test in airports, harbors, railways stations, cultural activities or sporting events.\"\\n\\n\\n==== Functional assays ====\\nTollotest is a molecular test that detects the activity of a SARS-CoV2 protease, which is a biomarker for active infection.\\n\\n\\n== History ==\\n\\nIn January 2020, scientists from China published the first genetic sequences of SARS-CoV-2 via GISAID, a program that normally handled genetic sequence data.  Researchers around the world used that data to build molecular tests for the virus.  Antigen- and antibody-based tests were developed later.Even once the first tests were created, the supply was limited.  As a result, no countries had reliable data on the prevalence of the virus early in the pandemic. The WHO and other experts called for ramping up testing as the best way to slow the spread of the virus.  Shortages of reagent and other testing supplies became a bottleneck for mass testing in the EU, the UK and the US.  Early tests also encountered problems with reliability.\\n\\n\\n== Testing protocols ==\\n\\n\\n=== Drive-through testing ===\\nIn drive-through testing, the person undergoing testing remains in a vehicle while a healthcare professional approaches the vehicle and obtains a sample, all while taking appropriate precautions such as wearing personal protective equipment (PPE). Drive-through centers helped South Korea accelerate its testing program.\\n\\n\\n=== Home collection ===\\n\\nIn Hong Kong test subjects can stay home and receive a specimen tube. They spit into it, return it and later get the result.\\n\\n\\n=== Pooled testing ===\\n\\nPooled testing can improve turnaround time, by combining a number of samples to be tested together. If the pool result is negative, all samples are negative. If the test result is positive, samples will need to be individually tested.In Israel, researchers at Technion and Rambam Hospital developed a method for testing samples from 64 patients simultaneously, by pooling the samples and only testing further if the combined sample was positive. Pool testing was then adopted in Israel, Germany, Ghana South Korea, Nebraska, China and the Indian states of Uttar Pradesh, West Bengal, Punjab, Chhattisgarh and Maharashtra.Open source, multiplexed designs released by Origami Assays can test as many as 1122 patient samples using only 93 assays. These balanced designs can be run in small laboratories without robotic liquid handlers.\\n\\n\\n=== Multi-tiered testing ===\\nOne study proposed a rapid immune response assay as a screening test, with a confirmatory nucleic acid test for diagnosis, followed by a rapid antibody test to determine course of action and assess population exposure/herd immunity.\\n\\n\\n=== Required volume ===\\nRequired testing levels are a function of disease spread. The more the cases, the more tests are needed to manage the outbreak. COVID-19 tends to grow exponentially at the beginning of an outbreak, meaning that the number of required tests initially also grows exponentially. If properly targeted testing grows more rapidly than cases, it can be contained.WHO recommends increasing testing until fewer than 10% are positive in any given jurisdiction.\\n\\n\\n==== United States ====\\n\\nEconomist Paul Romer reported that the US has the technical capacity to scale up to 20 million tests per day, which is his estimate of the scale needed to fully remobilize the economy. The Edmond J. Safra Center for Ethics estimated on 4 April that this capacity could be available by late July. Romer pointed to single-molecule real-time sequencing equipment from Pacific Biosciences and to the Ion Torrent Next-Generation Sequencing equipment from ThermoFisher Scientific. According to Romer, \"Recent research papers suggest that any one of these has the potential to scale up to millions of tests per day.\" This plan requires removing regulatory hurdles. Romer estimated that $100 billion would cover the costs.Romer also claimed that high test accuracy is not required if tests are administered frequently enough. He ran model simulations in which 7% of the population is tested every day using a test with a 20% false negative rate and a 1% false positive rate. The average person would be tested roughly every two weeks. Those who tested positive would go into quarantine. Romer\\'s simulation indicated that the fraction of the population that is infected at any given time (known as the attack rate) peaks reaches roughly 8% in about thirty days before gradually declining, in most runs reaching zero at 500 days, with cumulative prevalence remaining below 20%.\\n\\n\\n=== Snapshot mass-testing ===\\nA study found that, despite possibly suboptimal implementation, the snapshot mass-testing approach conducted by Slovakia by which ~80% of its population was tested for COVID-19 within a weekend at the end of October 2020 was highly efficacious, decreasing observed prevalence by 58% within one week and by 70% compared to a hypothetical scenario of no snapshot mass-testing. The significant reduction resulted from a set of complementary lockdown and quarantine measures whereby citizens who tested positive were quarantined synchronously the weeks afterwards.\\n\\n\\n=== Surveillance and screening of populations ===\\n\\nAs of August 2020, the WHO recognizes wastewater surveillance of SARS-CoV-2 as a potentially useful source of information on the prevalence and temporal trends of COVID-19 in communities, while highlighting that gaps in research such as viral shedding characteristics should be addressed. Such aggregative testing may have detected early cases. Studies show that wastewater-based epidemiology has the potential for an early warning system and monitoring for COVID-19 infections. This may prove particularly useful once large shares of regional populations are vaccinated or recovered and don\\'t need to conduct rapid tests while in some cases being infectious nevertheless.\\n\\n\\n== Available tests ==\\n\\nCountries around the world developed tests independently and in partnership with others.\\n\\n\\n=== Nucleic acid tests ===\\nTests developed in China, France, Germany, Hong Kong, Japan, the United Kingdom, and the US targeted different parts of the viral genome. WHO adopted the German system for manufacturing kits sent to low-income countries without the resources to develop their own.PowerChek Coronavirus looks for the \"E\" gene shared by all beta coronaviruses, and the RdRp gene specific to SARS-CoV-2.\\n\\nAbbott Laboratories\\' ID Now nucleic acid test uses isothermal amplification technology. The assay amplifies a unique region of the virus\\'s RdRp gene; the resulting copies are then detected with \"fluorescently-labeled molecular beacons\". The test kit uses the company\\'s \"toaster-size\" ID Now device, which is widely deployed in the US. The device can be used in laboratories or in point of care settings, and provides results in 13 minutes or less.Primerdesign offers its Genesig Real-Time PCR Coronavirus (COVID‑19). Cobas SARS-CoV-2 Qualitative assay runs on the Cobas® 6800/8800 Systems by Roche Molecular Systems. They are offered by the United Nations and other procurement agencies.\\n\\n\\n=== Antigen tests ===\\n\\nQuidel\\'s \"Sofia 2 SARS Antigen FIA\" is a lateral flow test that uses monoclonal antibodies to detect the virus\\'s nucleocapsid (N) protein. The result is read out by the company\\'s Sofia 2 device using immunofluorescence. The test is simpler and cheaper but less accurate than nucleic acid tests. It can be deployed in laboratories or at point of care and gives results in 15 minutes. A false negative result occurs if the sample\\'s antigen level is positive but below the test\\'s detection limit, requiring confirmation with a nucleic acid test.The Innova SARS-CoV-2 Antigen Rapid Qualitative Test was never approved for use in the United States, but was being sold by the company anyway. The FDA inspected Innova facilities in California in March and April 2021, and found inadequate quality assurance of tests manufactured in China. On 23 April 2021, the company issued a recall. The FDA warned consumers to return or destroy the devices because the rate of false positives and false negatives found in clinical trials were higher than the rate claimed by the packaging.  Over 1 billion tests from the company have been distributed in the UK, with £3 billion in funding as part of Operation Moonshot, and the MHRK has authorized exceptional use until at least 28 August 2021. Concerned experts pointed out that accuracy dropped significantly when screening was conducted by the public instead of by a medical professional, and that the test was not designed to screen asymptomatic people. A 2020 study found 79% of positive cases were found when used by laboratory scientists, but only 58% when used by the general public and 40% when used for city-wide screening in Liverpool.\\n\\n\\n=== Serology (antibody) tests ===\\nAntibodies are usually detectable 14 days after the onset of the infection. Multiple jurisdictions survey their populations using these tests. The test requires a blood sample.\\nPrivate US labs including Quest Diagnostics and LabCorp offer antibody testing upon request.Certain antibody tests are available in several European countries and also in the US. Quotient Limited developed a CE marked antibody test that also received an US FDA Emergency Use Authorization.Roche offers a selective ELISA serology test.A summary review in BMJ has noted that while some \"serological tests … might be cheaper and easier to implement at the point of care [than RT-PCR]\", and such testing can identify previously infected individuals, \"caution is warranted … using serological tests for … epidemiological surveillance\". The review called for higher quality studies assessing accuracy with reference to a standard of \"RT-PCR performed on at least two consecutive specimens, and, when feasible, includ[ing] viral cultures.\" CEBM researchers have called for in-hospital \\'case definition\\' to record \"CT lung findings and associated blood tests\" and for the WHO to produce a \"protocol to standardise the use and interpretation of PCR\" with continuous re-calibration.\\n\\n\\n== Accuracy ==\\nAccuracy is measured in terms of specificity and selectivity. Test errors can be false positives (the test is positive, but the virus is not present) or false negatives, (the test is negative, but the virus is present).\\n\\n\\n=== Sensitivity and specificity ===\\n\\nSensitivity indicates whether the test accurately identifies whether the virus is present. Each test requires a minimum level of viral load in order to produce a positive result.  A 90% sensitive test will correctly identify 90% of infections, missing the other 10% (a false negative). Even relatively high sensitivity rates can produce high rates of false negatives in populations with low incidence rates.\\nIn a diagnostic test, sensitivity is a measure of how well a test can identify true positives and specificity is a measure of how well a test can identify true negatives. For all testing, both diagnostic and screening, there is usually a trade-off between sensitivity and specificity, such that higher sensitivities will mean lower specificities and vice versa.   A 90% specific test will correctly identify 90% of those who are uninfected, leaving 10% with a false positive result.Low-specificity tests have a low positive predictive value (PPV) when prevalence is low. For example, suppose incidence is 5%. Testing 100 people at random using a test that has a specificity of 95% would yield on average 5 people who are actually negative who would incorrectly test positive. Since 5% of the subjects actually are positive, another five would also test positive correctly, totaling 10 positive results. Thus, the PPV is 50%, an outcome no different from a coin toss. In this situation retesting those with a positive result increases the PPV to 94.5%, meaning that only 4.5% of the second tests would return the incorrect result, on average less than 1 incorrect result.\\n\\n\\n=== Causes of test error ===\\nThe time course of infection affects the accuracy of some tests. Samples may be collected before the virus has had a chance to establish itself or after the body has begun to eliminate it. A May 2020 review of PCR-RT testing found that the median probability of a false-negative result decreased from 100% on day 1, to 67% on day 4. On the day of symptom onset, the probability was 38%, which decreased to 20% 3 days later.\\n\\n\\n=== PCR-based test ===\\n\\nRT-PCR is the most commonly-used diagnostic test. PCR tests by nasopharyngeal swab have a sensitivity of 73%, but systematic analysis of specificity has not been determined due to the lack of PCR studies with a control group.In one study sensitivity was highest at week one (100%), followed by 89.3%, 66.1%, 32.1%, 5.4% and zero by week six since symptom onset.Sensitivity is also a function of the number of PCR cycles, as well as time and temperature between sample collection and analysis. A cycle threshold of 20 cycles would be adequate to detect SARS-Cov-2 in a highly infective person. Cycle thresholds above 34 are increasingly likely to give false positives outside of high biosafety level facilities.On July 16, 2020, Dr. Anthony Fauci of the US CDC indicated that positive results obtained from RT-PCR tests run at more than 35 cycles were almost always \"just dead nucleotides\". On August 29, 2020, the New York Times reported that, \"In three sets of testing data that include cycle thresholds, compiled by officials in Massachusetts, New York and Nevada … most tests set the limit at 40 [cycles], a few at 37” and that the CDC was examining the use of cycle threshold measures “for policy decisions,” On July 21, 2021, the CDC, in their \"Real-Time RT-PCR Diagnostic Pan: Instructions for Use\", indicated tests results should be determined at 40 cycles.A Dutch CDC-led laboratory investigation compared 7 PCR kits. Test kits made by BGI, R-Biopharm AG, BGI, KH Medical and Seegene showed high sensitivity.High sensitivity kits are recommended to assess people without symptoms, while lower sensitivity tests are adequate when diagnosing symptomatic patients.The University of Oxford\\'s Centre for Evidence-Based Medicine (CEBM) has pointed to mounting evidence that \"a good proportion of \\'new\\' mild cases and people re-testing positives via RT-PCR after quarantine or discharge from hospital are not infectious, but are simply clearing harmless virus particles which their immune system has efficiently dealt with\", and have called for \"an international effort to standardize and periodically calibrate testing\". On 7 September, the UK government issued \"guidance for procedures to be implemented in laboratories to provide assurance of positive SARS-CoV-2 RNA results during periods of low prevalence, when there is a reduction in the predictive value of positive test results\".On 4 January 2021, the US FDA issued an alert about the risk of false results, particularly false negative results, with the Curative SARS-Cov-2 Assay real-time RT-PCR test.\\n\\n\\n=== Isothermal nucleic amplification test ===\\nOne study reported that the ID Now COVID-19 test showed sensitivity of 85.2%. Abbott responded that the issue could have been caused by analysis delays. Another study rejected the test in their clinical setting because of this low sensitivity.\\n\\n\\n== Confirmatory testing ==\\nThe WHO recommends countries that do not have testing capacity and national laboratories with limited experience on COVID‑19 send their first five positives and the first ten negative COVID‑19 samples to one of the 16 WHO reference laboratories for confirmatory testing. Out of the sixteen reference laboratories, seven are in Asia, five in Europe, two in Africa, one in North America and one in Australia.\\n\\n\\n== National responses ==\\n\\n\\n=== Iceland ===\\nIceland managed the pandemic with aggressive contact tracing, inbound travel restrictions, testing, and quarantining, but with less aggressive lock-downs.\\n\\n\\n=== India ===\\n\\n\\n=== Italy ===\\nResearchers tested the entire population of Vò, the site of Italy\\'s first COVID‑19 death. They tested about 3,400 people twice, at an interval of ten days. About half the people testing positive had no symptoms. All discovered cases were quarantined. Along with restricting travel to the commune, new infections were eliminated.\\n\\n\\n=== Japan ===\\nUnlike other Asian countries, Japan did not experience a pandemic of SARS or MERS, so the country\\'s PCR testing system was not well developed. Japan preferentially tested patients with severe illness and their close contacts at the beginning. Japan\\'s Novel Coronavirus Expert Meeting chose cluster measures to identify infections clusters. The Expert Meeting analyzed the outbreak from Wuhan and identified conditions leading to clusters (closed spaces, crowded spaces and close-contact), and asked people to avoid them.In January, contact tracers took action shortly after the first infection was found. Only administrative tests were carried out at first, until insurance began covering PCR tests on 6 March. Private companies began to test, and the test system gradually expanded.On 3 April, those with positive tests were legally permitted to recuperate at home or in a hotel if they had asymptomatic or mild illness, ending the hospital bed shortage. The first wave (from China) was contained, but a second wave (caused by returnees from Europe and the US) in mid-March led to spreading infection in April. On 7 April, Japan declared a state of emergency,  (less strict than a lockdown, because it did not block cities or restrict outings). On 13 May, antigen test kits became covered by insurance, and were combined with a PCR test for  diagnosis.Japan\\'s PCR test count per capita remained far smaller than in some other countries even though its positive test rate was lower. Excess mortality was observed in March. The Expert Meeting stated, \"The Japanese health care system originally carries out pneumonia surveillance, allowing it to detect most of the severely ill patients who develop pneumonia. There are a large number of CT scanners in Japan and they have spread to small hospitals all over the country, so pneumonia patients are rarely missed. In that sense, it meets the same standards as other countries that mainly carry out PCR tests.\" The group recommended using CT scans data and doctor\\'s findings for diagnosis. On the Diamond Princess cruise ship, many people who initially tested negative later tested positive. Half of coronavirus-positives there who remained mild or asymptomatic had pneumonia findings on CT scans and their CT image showed a frosted glass shadow that is characteristic of infection.As of 18 July, Japan\\'s daily PCR testing capacity was about 32,000, more than three times the 10,000 cases as of April. When the antigen test is added to it, the number is about 58,000. The number of tests per 1,000 people in the United States is about 27 times that of Japan, the UK is 20 times, Italy is 8 times, and South Korea is twice (as of 26 July).\\nThe number of those infected with coronavirus and inpatients has increased in July, but the number of serious cases has not increased. This is thought to be due to the proper testing of those infected in July compared to those in April. In April, the number of tests could not catch up with the increase in the number of infected people, and the test standards were strict, so the test positive rate exceeded 30% at the peak. It means that there were quite a few cases where the those infected was not PCR tested. It is thought that the severe case was preferentially tested though there were a lot of mild cases and asymptomatic carriers mainly in the young during the first wave. In other words, it became possible to grasp the actual situation of infection much better than before by strengthening the testing system. At the end of July, accommodation facilities for mild and asymptomatic carriers became full, and the authorities requested hospitals to prepare beds for the mild. However, it became difficult to treat patients with other illnesses and to maintain the ICU system including the staff due to the occupation of hospital beds by patients with mild symptoms.\\n\\n\\n=== Russia ===\\nOn 27 April 2020, Russia tested 3 million people and had 183,000 positive results. On 28 April Anna Popova, head of Federal Service for Surveillance in Healthcare (Roszdravnadzor) stated that 506 laboratories were testing; that 45% of those who tested positive had no symptoms; that 5% of patients had a severe form; and 40% of infections were from family members. Illness improved from six days to one day after symptoms appeared. Antibody testing was carried out on 3,200 Moscow doctors, finding 20% immunity.\\n\\n\\n=== Singapore ===\\nWith contact tracing, inbound travel restrictions, testing, and quarantining, Singapore arrested the initial spread without complete lockdown.\\n\\n\\n=== Slovakia ===\\nIn late October 2020 Slovakia tested 3.62 million people in a weekend, from a population of 5.4m, representing 67% of the total (or 82% of the adult population), 38,359 tested positive, representing 1.06% of those tested. The government considered the mass test would significantly assist in controlling the virus and avoid a lockdown and may repeat the exercise at a later date.\\n\\n\\n=== South Korea ===\\nSouth Korea\\'s broad testing approach helped reduce spread. Testing capacity, largely in private sector labs, was built up over several years by the South Korean government in the early 2000s.The government exploited the resident registration number (RRN) system. Authorities mobilized young men who were eligible for military service as social service agents, security and public health doctors. Public health doctors were mainly dispatched to public health centers and life treatment centers where mildly ill patients were accommodated. They performed PCR tests and managed mild patients. Social service agents worked in pharmacies to fill staff shortages. Korea\\'s 10k PCR tests per million residents was the world\\'s highest as of 13 April rising to 20k by mid-June. Twenty-seven Korean companies exported test kits worth $48.6 million in March, and were asked to provide test kits or humanitarian assistance by more than 120 countries. Korean authorities set up a treatment center to isolate and manage patients with asymptomatic and minor illnesses in one facility in order to vacate hospital beds for the more severely ill.\\nCenters were sited mainly at national facilities and corporate training centers. The failure of Korea\\'s MERS quarantine in May 2015 left Korea more prepared for COVID-19 than countries that did not face that pandemic. Then President Park Geun-hye allowed Korean CDC-approved private sector testing for infectious diseases in 2016. Korea already had a system for isolating, testing and treating infectious disease patients separately from others. Patients with respiratory illness but no epidemiological relevance were treated at the National Hospital, and those with epidemiological relevance were treated at selected clinics.Korea established a large scale drive-through/walk-through\" test testing program. However, the most common method was \"mobile examination\". In Daegu City, 54% of samples were collected by 23 March in home or hospital. Collecting samples door-to-door of avoided the risk of travel by possibly infected patients, but required additional staff. Korea solved the problem by drafting more than 2,700 public insurance doctors.The government disclosed personal information to the public via KCDC without patient consent. The authorities used digital surveillance to trace possible spread.\\n\\n\\n=== Taiwan ===\\nHealth insurance IDs and national identification card numbers were used to trace contacts.\\n\\n\\n=== United Arab Emirates ===\\nIn January 2021, the COVID-19 testing results of the UAE came under scrutiny, as Denmark suspended the Emirati flights for five days. The European nation said that it barred the flights from the UAE due to growing suspicion of irregularities in the testing process being followed in the Gulf nation. Denmark\\'s Minister of Transport, Benny Engelbrecht said that they were taking time to ensure that the negative tests of travelers from the Emirates were a real screening carried out appropriately.\\n\\n\\n=== United States ===\\n\\n\\n==== New York State ====\\nNew York State\\'s control measures consisted of PCR tests, stay-at-home measures and strengthening the healthcare system. On 29 February before its first case, the state allowed testing at the Wordsworth Center. They managed to convince the CDC to approve tests at state laboratories and the FDA to approve a test kit. As of 13 March the state was conducting more than 1,000 daily tests, growing to 10,000/day on 19 March. In April, the number exceeded 20,000. Many people queued at hospitals to get tested. On 21 March New York City health officials directed medical providers to test only those entering the hospital, for lack of PPE.\\n\\n\\n==== USS Theodore Roosevelt ====\\n\\nFollowing an outbreak, 94% of the 4,800 aircraft carrier crew were tested. Roughly 60 percent of the 600-plus sailors who tested positive were asymptomatic. Five infected sailors who completed quarantine subsequently developed flu-like symptoms and again tested positive.\\n\\n\\n==== Nevada ====\\nIn 2020, Nevada received a donation of 250,000 Covid testing kits, which were a product of China\\'s leading genetics company, BGI Group. A UAE-based firm owned by Tahnoun bin Zayed Al Nahyan, Group 42 partnered with the BGI Group to supply the testing kits to Nevada. However, the US Department of Homeland Security and the State Department raised a warning for Nevada hospitals to not use the Chinese-made testing kits, as there were concerns around the involvement of the Chinese government, test accuracy and privacy of the patients.\\n\\n\\n==== Delayed testing ====\\nA shortage of trained medical laboratory scientists, assay reagents, analyzers, transport medium, and PPE coupled with high demand had limited initially limited the availability of testing and led to significantly increased turnaround times.\\n\\n\\n== Testing statistics by country ==\\nTesting strategies vary by country and over time, with some countries testing very widely, while others have at times focused narrowly on only testing the seriously ill. The country that tests only people showing symptoms will have a higher figure for \"Confirmed / tested\" than the country that also tests others. If two countries are alike in every respect, including which people they test, the one that tests more people will have a higher \"Confirmed / population\". Studies have also found that countries that test more, relative to the number of deaths, have lower estimated case fatality rates and younger age distributions of cases.\\n\\n\\n== See also ==\\n2002–2004 SARS outbreak\\nCoronavirus breathalyzer\\nCoronavirus disease 2019\\nCOVID-19 misinformation § PCR testing\\nCOVID-19 pandemic\\nPhilippine government response to the COVID-19 pandemic § COVID-19 testing controversy\\n\\n\\n== References ==\\n This article incorporates public domain material from the Centers for Disease Control and Prevention document: \"Symptom-Based Strategy to Discontinue Isolation for Persons with COVID-19\". Retrieved 5 May 2020.\\n\\n\\n== Further reading ==\\nCorman VM, Landt O, Kaiser M, Molenkamp R, Meijer A, Chu DK,  et al. (January 2020). \"Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR\". Euro Surveill. 25 (3). doi:10.2807/1560-7917.ES.2020.25.3.2000045. PMC 6988269. PMID 31992387.\\nGuglielmi G (July 2020). \"The explosion of new coronavirus tests that could help to end the pandemic\". Nature. 583 (7817): 506–9. Bibcode:2020Natur.583..506G. doi:10.1038/d41586-020-02140-8. PMID 32681157.\\nKevadiya, Bhavesh D.; Machhi, Jatin; Herskovitz, Jonathan; Oleynikov, Maxim D.; Blomberg, Wilson R.; Bajwa, Neha; Soni, Dhruvkumar; Das, Srijanee; Hasan, Mahmudul; Patel, Milankumar; Senan, Ahmed M. (15 February 2021). \"Diagnostics for SARS-CoV-2 infections\". Nature Materials. 20 (5): 593–605. Bibcode:2021NatMa..20..593K. doi:10.1038/s41563-020-00906-z. ISSN 1476-4660. PMC 8264308. PMID 33589798. S2CID 231930978.\\n\\n\\n== External links ==\\nRitchie, Hannah; Ortiz-Ospina, Esteban; Beltekian, Diana; Mathieu, Edouard; Hasell, Joe; MacDonald, Bobbie; Giattino, Charlie; Appel, Cameron; Rodés-Guirao, Lucas; Roser, Max (13 July 2020). \"Coronavirus (COVID-19) Testing\". Our World in Data. – International testing statistics updated twice a week.\\n\"COVID-19 diagnostic tech tableau\". BioCentury. Retrieved 22 June 2020.\\nCOVID-19 Testing (at least) – now Free for all? (CDC; US Congress; CSPAN video/6:00; 12 March 2020)\\n\"EUA Authorized Serology Test Performance\". U.S. Food and Drug Administration (FDA). 25 May 2021.\\n\"Global Progress on COVID-19 Serology-Based Testing\". Johns Hopkins Center for Health Security.\\n\"Testing FAQ\". Johns Hopkins Coronavirus Resource Center.\\n\"New Rapid Testing Device for COVID-19 Immunity\". Mantracourt Electronics & University of Exeter.',\n",
              "  1.1714024037145976),\n",
              " ('Coronavirus disease 2019 (COVID-19) is a contagious disease caused by a virus, the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The first known case was identified in Wuhan, China, in December 2019. The disease has since spread worldwide, leading to the ongoing COVID-19 pandemic.Symptoms of COVID‑19 are variable, but often include fever, cough, headache, fatigue, breathing difficulties, loss of smell, and loss of taste. Symptoms may begin one to fourteen days after exposure to the virus. At least a third of people who are infected do not develop noticeable symptoms. Of those people who develop symptoms noticeable enough to be classed as patients, most (81%) develop mild to moderate symptoms (up to mild pneumonia), while 14% develop severe symptoms (dyspnea, hypoxia, or more than 50% lung involvement on imaging), and 5% suffer critical symptoms (respiratory failure, shock, or multiorgan dysfunction). Older people are at a higher risk of developing severe symptoms. Some people continue to experience a range of effects (long COVID) for months after recovery, and damage to organs has been observed. Multi-year studies are underway to further investigate the long-term effects of the disease.COVID‑19 transmits when people breathe in air contaminated by droplets and small airborne particles containing the virus. The risk of breathing these in is highest when people are in close proximity, but they can be inhaled over longer distances, particularly indoors. Transmission can also occur if splashed or sprayed with contaminated fluids in the eyes, nose or mouth, and, rarely, via contaminated surfaces. People remain contagious for up to 20 days, and can spread the virus even if they do not develop symptoms.Several COVID-19 testing methods have been developed to diagnose the disease. The standard diagnostic method is by detection of the virus\\'s nucleic acid by real-time reverse transcription polymerase chain reaction (rRT-PCR), transcription-mediated amplification (TMA), or by reverse transcription loop-mediated isothermal amplification (RT-LAMP) from a nasopharyngeal swab.\\nSeveral COVID-19 vaccines have been approved and distributed in various countries, which have initiated mass vaccination campaigns. Other preventive measures include physical or social distancing, quarantining, ventilation of indoor spaces, covering coughs and sneezes, hand washing, and keeping unwashed hands away from the face. The use of face masks or coverings has been recommended in public settings to minimize the risk of transmission. While work is underway to develop drugs that inhibit the virus, the primary treatment is symptomatic. Management involves the treatment of symptoms, supportive care, isolation, and experimental measures.\\n\\n\\n== Nomenclature ==\\n\\nDuring the initial outbreak in Wuhan, the virus and disease were commonly referred to as \"coronavirus\" and \"Wuhan coronavirus\", with the disease sometimes called \"Wuhan pneumonia\". In the past, many diseases have been named after geographical locations, such as the Spanish flu, Middle East respiratory syndrome, and Zika virus. In January 2020, the World Health Organization (WHO) recommended 2019-nCoV and 2019-nCoV acute respiratory disease as interim names for the virus and disease per 2015 guidance and international guidelines against using geographical locations or groups of people in disease and virus names to prevent social stigma. The official names COVID‑19 and SARS-CoV-2 were issued by the WHO on 11 February 2020. The Director-General, Tedros Adhanom explained that CO stands for corona, VI for virus, D for disease, and 19 for 2019, the year in which the outbreak was first identified. The WHO additionally uses \"the COVID‑19 virus\" and \"the virus responsible for COVID‑19\" in public communications.\\n\\n\\n== Signs and symptoms ==\\n\\n\\n== Cause ==\\nCOVID‑19 is caused by infection with a strain of coronavirus known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).\\n\\n\\n=== Transmission ===\\n\\n\\n=== Virology ===\\n\\nSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a novel severe acute respiratory syndrome coronavirus. It was first isolated from three people with pneumonia connected to the cluster of acute respiratory illness cases in Wuhan. All structural features of the novel SARS-CoV-2 virus particle occur in related coronaviruses in nature.Outside the human body, the virus is destroyed by household soap, which bursts its protective bubble.SARS-CoV-2 is closely related to the original SARS-CoV. It is thought to have an animal (zoonotic) origin. Genetic analysis has revealed that the coronavirus genetically clusters with the genus Betacoronavirus, in subgenus Sarbecovirus (lineage B) together with two bat-derived strains. It is 96% identical at the whole genome level to other bat coronavirus samples (BatCov RaTG13). The structural proteins of SARS-CoV-2 include membrane glycoprotein (M), envelope protein (E), nucleocapsid protein (N), and the spike protein (S). The M protein of SARS-CoV-2 is about 98% similar to the M protein of bat SARS-CoV, maintains around 98% homology with pangolin SARS-CoV, and has 90% homology with the M protein of SARS-CoV; whereas, the similarity is only around 38% with the M protein of MERS-CoV.\\n\\n\\n=== SARS-CoV-2 variants ===\\n\\nThe many thousands of SARS-CoV-2 variants are grouped into either clades or lineages. The WHO, in collaboration with partners, expert networks, national authorities, institutions and researchers, have established nomenclature systems for naming and tracking SARS-CoV-2 genetic lineages by GISAID, Nextstrain and Pango. At the present time, the expert group convened by WHO has recommended the labeling of variants using letters of the Greek Alphabet, for example, Alpha, Beta, Delta, and Gamma, giving the justification that they \"will be easier and more practical to discussed by non-scientific audiences.\" Nextstrain divides the variants into five clades (19A, 19B, 20A, 20B, and 20C), while GISAID divides them into seven (L, O, V, S, G, GH, and GR). The Pango tool groups variants into lineages, with many circulating lineages being classed under the B.1 lineage.Several notable variants of SARS-CoV-2 emerged throughout 2020. Cluster 5 emerged among minks and mink farmers in Denmark. After strict quarantines and a mink euthanasia campaign, the cluster was assessed to no longer be circulating among humans in Denmark as of 1 February 2021.As of December 2021, there are five dominant variants of SARS-CoV-2 spreading among global populations: the Alpha variant (B.1.1.7, formerly called the UK variant), first found in London and Kent, the Beta variant (B.1.351, formerly called the South Africa variant), the Gamma variant (P.1, formerly called the Brazil variant), the Delta variant (B.1.617.2, formerly called the India variant), and the Omicron variant (B.1.1.529), which had spread to 57 countries as of 7 December.\\n\\n\\n== Pathophysiology ==\\n\\nThe SARS-CoV-2 virus can infect a wide range of cells and systems of the body. COVID‑19 is most known for affecting the upper respiratory tract (sinuses, nose, and throat) and the lower respiratory tract (windpipe and lungs). The lungs are the organs most affected by COVID‑19 because the virus accesses host cells via the receptor for the enzyme angiotensin-converting enzyme 2 (ACE2), which is most abundant on the surface of type II alveolar cells of the lungs. The virus uses a special surface glycoprotein called a \"spike\" to connect to the ACE2 receptor and enter the host cell.\\n\\n\\n=== Respiratory tract ===\\nFollowing viral entry, COVID‑19 infects the ciliated epithelium of the nasopharynx and upper airways.\\n\\n\\n=== Nervous system ===\\nOne common symptom, loss of smell, results from infection of the support cells of the olfactory epithelium, with subsequent damage to the olfactory neurons. The involvement of both the central and peripheral nervous system in COVID‑19 has been reported in many medical publications. It is clear that many people with COVID-19 exhibit neurological or mental health issues. The virus is not detected in the CNS of the majority of COVID-19 patients with neurological issues. However, SARS-CoV-2 has been detected at low levels in the brains of those who have died from COVID‑19, but these results need to be confirmed. While virus has been detected in cerebrospinal fluid of autopsies, the exact mechanism by which it invades the CNS remains unclear and may first involve invasion of peripheral nerves given the low levels of ACE2 in the brain. The virus may also enter the bloodstream from the lungs and cross the blood-brain barrier to gain access to the CNS, possibly within an infected white blood cell.\\n  Research conducted when Alpha was the dominant variant has suggested COVID-19 May cause brain damage. It is currently unknown if such damage is temporary or permanent, and whether Omicron has similar effects.  Observed individuals infected with COVID-19 (most with mild cases) experienced an additional 0.2% to 2% of brain tissue lost in regions of the brain connected to the sense of smell compared with uninfected individuals, and the overall effect on the brain was equivalent on average to at least one extra year of normal aging; infected individuals also scored lower on several cognitive tests.  All effects were more pronounced among older ages.\\n\\n\\n=== Gastrointestinal tract ===\\nThe virus also affects gastrointestinal organs as ACE2 is abundantly expressed in the glandular cells of gastric, duodenal and rectal epithelium as well as endothelial cells and enterocytes of the small intestine.\\n\\n\\n=== Cardiovascular system ===\\nThe virus can cause acute myocardial injury and chronic damage to the cardiovascular system. An acute cardiac injury was found in 12% of infected people admitted to the hospital in Wuhan, China, and is more frequent in severe disease. Rates of cardiovascular symptoms are high, owing to the systemic inflammatory response and immune system disorders during disease progression, but acute myocardial injuries may also be related to ACE2 receptors in the heart. ACE2 receptors are highly expressed in the heart and are involved in heart function. A high incidence of thrombosis and venous thromboembolism have been found in people transferred to Intensive care units (ICU) with COVID‑19 infections, and may be related to poor prognosis. Blood vessel dysfunction and clot formation (as suggested by high D-dimer levels caused by blood clots) are thought to play a significant role in mortality, incidences of clots leading to pulmonary embolisms, and ischaemic events within the brain have been noted as complications leading to death in people infected with SARS-CoV-2. Infection appears to set off a chain of vasoconstrictive responses within the body, constriction of blood vessels within the pulmonary circulation has also been posited as a mechanism in which oxygenation decreases alongside the presentation of viral pneumonia. Furthermore, microvascular (arterioles and capillaries) blood vessel damage has been reported in a small number of tissue samples of the brains – without detected SARS-CoV-2 – and the olfactory bulbs from those who have died from COVID‑19. COVID‑19 was also found to cause substantial – including morphological and mechanical – changes to blood cells – such as increased sizes – sometimes persisting for months after hospital discharge.\\n\\n\\n=== Other organs ===\\nAnother common cause of death is complications related to the kidneys. Early reports show that up to 30% of hospitalized patients both in China and in New York have experienced some injury to their kidneys, including some persons with no previous kidney problems.Autopsies of people who died of COVID‑19 have found diffuse alveolar damage, and lymphocyte-containing inflammatory infiltrates within the lung.\\n\\n\\n=== Immunopathology ===\\n\\nAlthough SARS-CoV-2 has a tropism for ACE2-expressing epithelial cells of the respiratory tract, people with severe COVID‑19 have symptoms of systemic hyperinflammation. Clinical laboratory findings of elevated IL‑2, IL‑7, IL‑6, granulocyte-macrophage colony-stimulating factor (GM‑CSF), interferon gamma-induced protein 10 (IP‑10), monocyte chemoattractant protein 1 (MCP1), macrophage inflammatory protein 1‑alpha (MIP‑1‑alpha), and tumour necrosis factor (TNF‑α) indicative of cytokine release syndrome (CRS) suggest an underlying immunopathology.Additionally, people with COVID‑19 and acute respiratory distress syndrome (ARDS) have classical serum biomarkers of CRS, including elevated C-reactive protein (CRP), lactate dehydrogenase (LDH), D-dimer, and ferritin.Systemic inflammation results in vasodilation, allowing inflammatory lymphocytic and monocytic infiltration of the lung and the heart. In particular, pathogenic GM-CSF-secreting T cells were shown to correlate with the recruitment of inflammatory IL-6-secreting monocytes and severe lung pathology in people with COVID‑19. Lymphocytic infiltrates have also been reported at autopsy.\\n\\n\\n=== Viral and host factors ===\\n\\n\\n==== Virus proteins ====\\n\\nMultiple viral and host factors affect the pathogenesis of the virus. The S-protein, otherwise known as the spike protein, is the viral component that attaches to the host receptor via the ACE2 receptors. It includes two subunits: S1 and S2. S1 determines the virus-host range and cellular tropism via the receptor-binding domain. S2 mediates the membrane fusion of the virus to its potential cell host via the H1 and HR2, which are heptad repeat regions. Studies have shown that S1 domain induced IgG and IgA antibody levels at a much higher capacity. It is the focus spike proteins expression that are involved in many effective COVID‑19 vaccines.The M protein is the viral protein responsible for the transmembrane transport of nutrients. It is the cause of the bud release and the formation of the viral envelope. The N and E protein are accessory proteins that interfere with the host\\'s immune response.\\n\\n\\n==== Host factors ====\\nHuman angiotensin converting enzyme 2 (hACE2) is the host factor that SARS-COV2 virus targets causing COVID‑19. Theoretically, the usage of angiotensin receptor blockers (ARB) and ACE inhibitors upregulating ACE2 expression might increase morbidity with COVID‑19, though animal data suggest some potential protective effect of ARB; however no clinical studies have proven susceptibility or outcomes. Until further data is available, guidelines and recommendations for hypertensive patients remain.The effect of the virus on ACE2 cell surfaces leads to leukocytic infiltration, increased blood vessel permeability, alveolar wall permeability, as well as decreased secretion of lung surfactants. These effects cause the majority of the respiratory symptoms. However, the aggravation of local inflammation causes a cytokine storm eventually leading to a systemic inflammatory response syndrome.Among healthy adults not exposed to SARS-CoV-2, about 35% have CD4+ T cells that recognize the SARS-CoV-2 S protein (particularly the S2 subunit) and about 50% react to other proteins of the virus, suggesting cross-reactivity from previous common colds caused by other coronaviruses.It is unknown whether different persons use similar antibody genes in response to COVID‑19.\\n\\n\\n=== Host cytokine response ===\\n\\nThe severity of the inflammation can be attributed to the severity of what is known as the cytokine storm. Levels of interleukin 1B, interferon-gamma, interferon-inducible protein 10, and monocyte chemoattractant protein 1 were all associated with COVID‑19 disease severity. Treatment has been proposed to combat the cytokine storm as it remains to be one of the leading causes of morbidity and mortality in COVID‑19 disease.A cytokine storm is due to an acute hyperinflammatory response that is responsible for clinical illness in an array of diseases but in COVID‑19, it is related to worse prognosis and increased fatality. The storm causes acute respiratory distress syndrome, blood clotting events such as strokes, myocardial infarction, encephalitis, acute kidney injury, and vasculitis. The production of IL-1, IL-2, IL-6, TNF-alpha, and interferon-gamma, all crucial components of normal immune responses, inadvertently become the causes of a cytokine storm. The cells of the central nervous system, the microglia, neurons, and astrocytes, are also involved in the release of pro-inflammatory cytokines affecting the nervous system, and effects of cytokine storms toward the CNS are not uncommon.\\n\\n\\n=== Pregnancy response ===\\nThere are many unknowns for pregnant women during the COVID‑19 pandemic. Given that they are prone to suffering from complications and severe disease infection with other types of coronaviruses, they have been identified as a vulnerable group and advised to take supplementary preventive measures.Physiological responses to pregnancy can include:\\n\\nImmunological: The immunological response to COVID-19, like other viruses, depends on a working immune system. It adapts during pregnancy to allow the development of the fetus whose genetic load is only partially shared with their mother, leading to a different immunological reaction to infections during the course of pregnancy.\\nRespiratory: Many factors can make pregnant women more vulnerable to hard respiratory infections. One of them is the total reduction of the lungs\\' capacity and inability to clear secretions.\\nCoagulation: During pregnancy, there are higher levels of circulating coagulation factors, and the pathogenesis of SARS-CoV-2 infection can be implicated. The thromboembolic events with associated mortality are a risk for pregnant women.However, from the evidence base, it is difficult to conclude whether pregnant women are at increased risk of grave consequences of this virus.In addition to the above, other clinical studies have proved that SARS-CoV-2 can affect the period of pregnancy in different ways. On the one hand, there is little evidence of its impact up to 12 weeks gestation. On the other hand, COVID-19 infection may cause increased rates of unfavorable outcomes in the course of the pregnancy. Some examples of these could be fetal growth restriction, preterm birth, and perinatal mortality, which refers to the fetal death past 22 or 28 completed weeks of pregnancy as well as the death among live-born children up to seven completed days of life.Unvaccinated women in later stages of pregnancy with COVID-19 are more likely than other patients to need very intensive care. Babies born to mothers with COVID-19 are more likely to have breathing problems. Pregnant women are strongly encouraged to get vaccinated.\\n\\n\\n== Diagnosis ==\\n\\nCOVID‑19 can provisionally be diagnosed on the basis of symptoms and confirmed using reverse transcription polymerase chain reaction (RT-PCR) or other nucleic acid testing of infected secretions. Along with laboratory testing, chest CT scans may be helpful to diagnose COVID‑19 in individuals with a high clinical suspicion of infection. Detection of a past infection is possible with serological tests, which detect antibodies produced by the body in response to the infection.\\n\\n\\n=== Viral testing ===\\n\\nThe standard methods of testing for presence of SARS-CoV-2 are nucleic acid tests, which detects the presence of viral RNA fragments. As these tests detect RNA but not infectious virus, its \"ability to determine duration of infectivity of patients is limited.\" The test is typically done on respiratory samples obtained by a nasopharyngeal swab; however, a nasal swab or sputum sample may also be used. Results are generally available within hours. The WHO has published several testing protocols for the disease.Several laboratories and companies have developed serological tests, which detect antibodies produced by the body in response to infection. Several have been evaluated by Public Health England and approved for use in the UK.The University of Oxford\\'s CEBM has pointed to mounting evidence that \"a good proportion of \\'new\\' mild cases and people re-testing positives after quarantine or discharge from hospital are not infectious, but are simply clearing harmless virus particles which their immune system has efficiently dealt with\" and have called for \"an international effort to standardize and periodically calibrate testing\" On 7 September, the UK government issued \"guidance for procedures to be implemented in laboratories to provide assurance of positive SARS-CoV-2 RNA results during periods of low prevalence, when there is a reduction in the predictive value of positive test results\".\\n\\n\\n=== Imaging ===\\n\\nChest CT scans may be helpful to diagnose COVID‑19 in individuals with a high clinical suspicion of infection but are not recommended for routine screening. Bilateral multilobar ground-glass opacities with a peripheral, asymmetric, and posterior distribution are common in early infection. Subpleural dominance, crazy paving (lobular septal thickening with variable alveolar filling), and consolidation may appear as the disease progresses. Characteristic imaging features on chest radiographs and computed tomography (CT) of people who are symptomatic include asymmetric peripheral ground-glass opacities without pleural effusions.Many groups have created COVID‑19 datasets that include imagery such as the Italian Radiological Society which has compiled an international online database of imaging findings for confirmed cases. Due to overlap with other infections such as adenovirus, imaging without confirmation by rRT-PCR is of limited specificity in identifying COVID‑19. A large study in China compared chest CT results to PCR and demonstrated that though imaging is less specific for the infection, it is faster and more sensitive.\\n\\n\\n=== Coding ===\\nIn late 2019, the WHO assigned emergency ICD-10 disease codes U07.1 for deaths from lab-confirmed SARS-CoV-2 infection and U07.2 for deaths from clinically or epidemiologically diagnosed COVID‑19 without lab-confirmed SARS-CoV-2 infection.\\n\\n\\n=== Pathology ===\\nThe main pathological findings at autopsy are:\\n\\nMacroscopy: pericarditis, lung consolidation and pulmonary oedema\\nLung findings:\\nminor serous exudation, minor fibrin exudation\\npulmonary oedema, pneumocyte hyperplasia, large atypical pneumocytes, interstitial inflammation with lymphocytic infiltration and multinucleated giant cell formation\\ndiffuse alveolar damage (DAD) with diffuse alveolar exudates. DAD is the cause of acute respiratory distress syndrome (ARDS) and severe hypoxemia.\\norganisation of exudates in alveolar cavities and pulmonary interstitial fibrosis\\nplasmocytosis in BAL\\nBlood and vessels: disseminated intravascular coagulation (DIC); leukoerythroblastic reaction, endotheliitis, hemophagocytosis\\nHeart: cardiac muscle cell necrosis\\nLiver: microvesicular steatosis\\nNose: shedding of olfactory epithelium\\nBrain: infarction\\nKidneys: acute tubular damage.\\nSpleen: white pulp depletion.\\n\\n\\n== Prevention ==\\n\\nPreventive measures to reduce the chances of infection include getting vaccinated, staying at home, wearing a mask in public, avoiding crowded places, keeping distance from others, ventilating indoor spaces, managing potential exposure durations, washing hands with soap and water often and for at least twenty seconds, practising good respiratory hygiene, and avoiding touching the eyes, nose, or mouth with unwashed hands.Those diagnosed with COVID‑19 or who believe they may be infected are advised by the CDC to stay home except to get medical care, call ahead before visiting a healthcare provider, wear a face mask before entering the healthcare provider\\'s office and when in any room or vehicle with another person, cover coughs and sneezes with a tissue, regularly wash hands with soap and water and avoid sharing personal household items.The first COVID‑19 vaccine was granted regulatory approval on 2 December 2020 by the UK medicines regulator MHRA. It was evaluated for emergency use authorization (EUA) status by the US FDA, and in several other countries. Initially, the US National Institutes of Health guidelines do not recommend any medication for prevention of COVID‑19, before or after exposure to the SARS-CoV-2 virus, outside the setting of a clinical trial. Without a vaccine, other prophylactic measures, or effective treatments, a key part of managing COVID‑19 is trying to decrease and delay the epidemic peak, known as \"flattening the curve\". This is done by slowing the infection rate to decrease the risk of health services being overwhelmed, allowing for better treatment of active cases, and delaying additional cases until effective treatments or a vaccine become available.\\n\\n\\n=== Vaccine ===\\n\\n\\n=== Face masks and respiratory hygiene ===\\n\\nThe WHO and the US CDC recommend individuals wear non-medical face coverings in public settings where there is an increased risk of transmission and where social distancing measures are difficult to maintain. This recommendation is meant to reduce the spread of the disease by asymptomatic and pre-symptomatic individuals and is complementary to established preventive measures such as social distancing. Face coverings limit the volume and travel distance of expiratory droplets dispersed when talking, breathing, and coughing. A face covering without vents or holes will also filter out particles containing the virus from inhaled and exhaled air, reducing the chances of infection. But, if the mask include an exhalation valve, a wearer that is infected (maybe without having noticed that, and asymptomatic) would transmit the virus outwards through it, despite any certification they can have. So the masks with exhalation valve are not for the infected wearers, and are not reliable to stop the pandemic in a large scale. Many countries and local jurisdictions encourage or mandate the use of face masks or cloth face coverings by members of the public to limit the spread of the virus.Masks are also strongly recommended for those who may have been infected and those taking care of someone who may have the disease. When not wearing a mask, the CDC recommends covering the mouth and nose with a tissue when coughing or sneezing and recommends using the inside of the elbow if no tissue is available. Proper hand hygiene after any cough or sneeze is encouraged. Healthcare professionals interacting directly with people who have COVID‑19 are advised to use respirators at least as protective as NIOSH-certified N95 or equivalent, in addition to other personal protective equipment.\\n\\n\\n=== Indoor ventilation and avoiding crowded indoor spaces ===\\nThe CDC recommends that crowded indoor spaces should be avoided. When indoors, increasing the rate of air change, decreasing recirculation of air and increasing the use of outdoor air can reduce transmission. The WHO recommends ventilation and air filtration in public spaces to help clear out infectious aerosols.Exhaled respiratory particles can build-up within enclosed spaces with inadequate ventilation. The risk of COVID‑19 infection increases especially in spaces where people engage in physical exertion or raise their voice (e.g., exercising, shouting, singing) as this increases exhalation of respiratory droplets. Prolonged exposure to these conditions, typically more than 15 minutes, leads to higher risk of infection.Displacement ventilation with large natural inlets can move stale air directly to the exhaust in laminar flow while significantly reducing the concentration of droplets and particles. Passive ventilation reduces energy consumption and maintenance costs but may lack controllability and heat recovery. Displacement ventilation can also be achieved mechanically with higher energy and maintenance costs. The use of large ducts and openings helps to prevent mixing in closed environments. Recirculation and mixing should be avoided because recirculation prevents dilution of harmful particles and redistributes possibly contaminated air, and mixing increases the concentration and range of infectious particles and keeps larger particles in the air.\\n\\n\\n=== Hand-washing and hygiene ===\\n\\nThorough hand hygiene after any cough or sneeze is required. The WHO also recommends that individuals wash hands often with soap and water for at least twenty seconds, especially after going to the toilet or when hands are visibly dirty, before eating and after blowing one\\'s nose. When soap and water are not available, the CDC recommends using an alcohol-based hand sanitiser with at least 60% alcohol. For areas where commercial hand sanitisers are not readily available, the WHO provides two formulations for local production. In these formulations, the antimicrobial activity arises from ethanol or isopropanol. Hydrogen peroxide is used to help eliminate bacterial spores in the alcohol; it is \"not an active substance for hand antisepsis.\" Glycerol is added as a humectant.\\n\\n\\n=== Social distancing ===\\n\\nSocial distancing (also known as physical distancing) includes infection control actions intended to slow the spread of the disease by minimising close contact between individuals. Methods include quarantines; travel restrictions; and the closing of schools, workplaces, stadiums, theatres, or shopping centres. Individuals may apply social distancing methods by staying at home, limiting travel, avoiding crowded areas, using no-contact greetings, and physically distancing themselves from others. Many governments are now mandating or recommending social distancing in regions affected by the outbreak.Outbreaks have occurred in prisons due to crowding and an inability to enforce adequate social distancing. In the United States, the prisoner population is aging and many of them are at high risk for poor outcomes from COVID‑19 due to high rates of coexisting heart and lung disease, and poor access to high-quality healthcare.\\n\\n\\n=== Surface cleaning ===\\nAfter being expelled from the body, coronaviruses can survive on surfaces for hours to days. If a person touches the dirty surface, they may deposit the virus at the eyes, nose, or mouth where it can enter the body and cause infection. Evidence indicates that contact with infected surfaces is not the main driver of COVID‑19, leading to recommendations for optimised disinfection procedures to avoid issues such as the increase of antimicrobial resistance through the use of inappropriate cleaning products and processes. Deep cleaning and other surface sanitation has been criticized as hygiene theater, giving a false sense of security against something primarily spread through the air.The amount of time that the virus can survive depends significantly on the type of surface, the temperature, and the humidity. Coronaviruses die very quickly when exposed to the UV light in sunlight. Like other enveloped viruses, SARS-CoV-2 survives longest when the temperature is at room temperature or lower, and when the relative humidity is low (<50%).On many surfaces, including glass, some types of plastic, stainless steel, and skin, the virus can remain infective for several days indoors at room temperature, or even about a week under ideal conditions. On some surfaces, including cotton fabric and copper, the virus usually dies after a few hours. The virus dies faster on porous surfaces than on non-porous surfaces due to capillary action within pores and faster aerosol droplet evaporation. However, of the many surfaces tested, two with the longest survival times are N95 respirator masks and surgical masks, both of which are considered porous surfaces.The CDC says that in most situations, cleaning surfaces with soap or detergent, not disinfecting, is enough to reduce risk of transmission. The CDC recommends that if a COVID‑19 case is suspected or confirmed at a facility such as an office or day care, all areas such as offices, bathrooms, common areas, shared electronic equipment like tablets, touch screens, keyboards, remote controls, and ATM machines used by the ill persons should be disinfected. Surfaces may be decontaminated with 62–71 percent ethanol, 50–100 percent isopropanol, 0.1 percent sodium hypochlorite, 0.5 percent hydrogen peroxide, and 0.2–7.5 percent povidone-iodine. Other solutions, such as benzalkonium chloride and chlorhexidine gluconate, are less effective. Ultraviolet germicidal irradiation may also be used. A datasheet comprising the authorised substances to disinfection in the food industry (including suspension or surface tested, kind of surface, use dilution, disinfectant and inocuylum volumes) can be seen in the supplementary material of.\\n\\n\\n=== Self-isolation ===\\nSelf-isolation at home has been recommended for those diagnosed with COVID‑19 and those who suspect they have been infected. Health agencies have issued detailed instructions for proper self-isolation. Many governments have mandated or recommended self-quarantine for entire populations. The strongest self-quarantine instructions have been issued to those in high-risk groups. Those who may have been exposed to someone with COVID‑19 and those who have recently travelled to a country or region with the widespread transmission have been advised to self-quarantine for 14 days from the time of last possible exposure.\\n\\n\\n=== Healthy diet and lifestyle ===\\nThe Harvard T.H. Chan School of Public Health recommends a healthy diet, being physically active, managing psychological stress, and getting enough sleep.Consistently meeting scientific guidelines of 150+ minutes per week of exercise or similar physical activity was shown to be associated with a smaller risk of hospitalisation and death due to COVID‑19, even when considering likely risk factors such as elevated Body mass index (BMI).A meta-analysis, published online in October 2021, concluded that \"Vitamin D supplementation in SARS-CoV-2 positive patients has the potential to positively impact patients with both mild and severe symptoms.\" The largest observational study on the subject using online questionnaires, with over 6,000 participants and a dosage regime near the RDI, is set to conclude in July 2021. One of the collaborators in the study is Synergy Biologics Ltd, a manufacturer of vitamin D3 supplements.\\n\\n\\n=== International travel-related control measures ===\\nA 2021 Cochrane rapid review found that based upon low-certainty evidence, international travel-related control measures such as restricting cross-border travel may help to contain the spread of COVID‑19. Additionally, symptom/exposure-based screening measures at borders may miss many positive cases. While test-based border screening measures may be more effective, it could also miss many positive cases if only conducted upon arrival without follow-up. The review concluded that a minimum 10-day quarantine may be beneficial in preventing the spread of COVID‑19 and may be more effective if combined with an additional control measure like border screening.\\n\\n\\n== Treatment ==\\n\\n\\n== Prognosis ==\\n\\nThe severity of COVID‑19 varies. The disease may take a mild course with few or no symptoms, resembling other common upper respiratory diseases such as the common cold. In 3–4% of cases (7.4% for those over age 65) symptoms are severe enough to cause hospitalization. Mild cases typically recover within two weeks, while those with severe or critical diseases may take three to six weeks to recover. Among those who have died, the time from symptom onset to death has ranged from two to eight weeks. The Italian Istituto Superiore di Sanità reported that the median time between the onset of symptoms and death was twelve days, with seven being hospitalised. However, people transferred to an ICU had a median time of ten days between hospitalisation and death. Prolonged prothrombin time and elevated C-reactive protein levels on admission to the hospital are associated with severe course of COVID‑19 and with a transfer to ICU.Some early studies suggest 10% to 20% of people with COVID‑19 will experience symptoms lasting longer than a month. A majority of those who were admitted to hospital with severe disease report long-term problems including fatigue and shortness of breath. On 30 October 2020, WHO chief Tedros Adhanom warned that \"to a significant number of people, the COVID virus poses a range of serious long-term effects.\" He has described the vast spectrum of COVID‑19 symptoms that fluctuate over time as \"really concerning\". They range from fatigue, a cough and shortness of breath, to inflammation and injury of major organs – including the lungs and heart, and also neurological and psychologic effects. Symptoms often overlap and can affect any system in the body. Infected people have reported cyclical bouts of fatigue, headaches, months of complete exhaustion, mood swings, and other symptoms. Tedros therefore concluded that a strategy of achieving herd immunity by infection, rather than vaccination, is \"morally unconscionable and unfeasible\".In terms of hospital readmissions about 9% of 106,000 individuals had to return for hospital treatment within two months of discharge. The average to readmit was eight days since first hospital visit. There are several risk factors that have been identified as being a cause of multiple admissions to a hospital facility. Among these are advanced age (above 65 years of age) and presence of a chronic condition such as diabetes, COPD, heart failure or chronic kidney disease.According to scientific reviews smokers are more likely to require intensive care or die compared to non-smokers. Acting on the same ACE2 pulmonary receptors affected by smoking, air pollution has been correlated with the disease. Short term and chronic exposure to air pollution seems to enhance morbidity and mortality from COVID‑19. Pre-existing heart and lung diseases and also obesity, especially in conjunction with fatty liver disease, contributes to an increased health risk of COVID‑19.It is also assumed that those that are immunocompromised are at higher risk of getting severely sick from SARS-CoV-2. One research that looked into the COVID‑19 infections in hospitalized kidney transplant recipients found a mortality rate of 11%.Genetics also plays an important role in the ability to fight off the disease. For instance, those that do not produce detectable type I interferons or produce auto-antibodies against these may get much sicker from COVID‑19. Genetic screening is able to detect interferon effector genes.\\n\\n\\n=== Children ===\\n\\nWhile very young children have experienced lower rates of infection, older children have a rate of infection that is similar to the population as a whole. Children are likely to have milder symptoms and are at lower risk of severe disease than adults. The CDC reports that in the US roughly a third of hospitalized children were admitted to the ICU, while a European multinational study of hospitalized children from June 2020 found that about 8% of children admitted to a hospital needed intensive care. Four of the 582 children (0.7%) in the European study died, but the actual mortality rate may be \"substantially lower\" since milder cases that did not seek medical help were not included in the study.\\n\\n\\n=== Complications ===\\n\\nComplications may include pneumonia, acute respiratory distress syndrome (ARDS), multi-organ failure, septic shock, and death. Cardiovascular complications may include heart failure, arrhythmias (including atrial fibrillation), heart inflammation, and thrombosis, particularly venous thromboembolism. Approximately 20–30% of people who present with COVID‑19 have elevated liver enzymes, reflecting liver injury.Neurologic manifestations include seizure, stroke, encephalitis, and Guillain–Barré syndrome (which includes loss of motor functions). Following the infection, children may develop paediatric multisystem inflammatory syndrome, which has symptoms similar to Kawasaki disease, which can be fatal. In very rare cases, acute encephalopathy can occur, and it can be considered in those who have been diagnosed with COVID‑19 and have an altered mental status.In the case of pregnant women, it is important to note that, according to the Centers for Disease Control and Prevention, pregnant women are at increased risk of becoming seriously ill from COVID‑19. This is because pregnant women with COVID‑19 appear to be more likely to develop respiratory and obstetric complications that can lead to miscarriage, premature delivery and intrauterine growth restriction.Fungal infections such as aspergillosis, candidiasis, cryptococcosis and mucormycosis have been recorded in patients recovering from COVID‑19.\\n\\n\\n=== Longer-term effects ===\\n\\nSome early studies suggest that 10–20% of people with COVID‑19 will experience symptoms lasting longer than a month. A majority of those who were admitted to hospital with severe disease report long-term problems, including fatigue and shortness of breath. About 5–10% of patients admitted to hospital progress to severe or critical disease, including pneumonia and acute respiratory failure.By a variety of mechanisms, the lungs are the organs most affected in COVID‑19. In people requiring hospital admission, up to 98% of CT scans performed show lung abnormalities after 28 days of illness even if they had clinically improved.People with advanced age, severe disease, prolonged ICU stays, or who smoke are more likely to have long-lasting effects, including pulmonary fibrosis. Overall, approximately one-third of those investigated after four weeks will have findings of pulmonary fibrosis or reduced lung function as measured by DLCO, even in asymptomatic people, but with the suggestion of continuing improvement with the passing of more time.\\n\\n\\n=== Immunity ===\\n\\nThe immune response by humans to SARS-CoV-2 virus occurs as a combination of the cell-mediated immunity and antibody production, just as with most other infections. B cells interact with T cells and begin dividing before selection into the plasma cell, partly on the basis of their affinity for antigen. Since SARS-CoV-2 has been in the human population only since December 2019, it remains unknown if the immunity is long-lasting in people who recover from the disease. The presence of neutralizing antibodies in blood strongly correlates with protection from infection, but the level of neutralizing antibody declines with time. Those with asymptomatic or mild disease had undetectable levels of neutralizing antibody two months after infection. In another study, the level of neutralizing antibodies fell four-fold one to four months after the onset of symptoms. However, the lack of antibodies in the blood does not mean antibodies will not be rapidly produced upon reexposure to SARS-CoV-2. Memory B cells specific for the spike and nucleocapsid proteins of SARS-CoV-2 last for at least six months after the appearance of symptoms.Reinfection with COVID‑19 is possible but uncommon. The first case of reinfection was documented in August 2020. A systematic review found 17 cases of confirmed reinfection in medical literature as of May 2021.\\n\\n\\n== Mortality ==\\n\\nSeveral measures are commonly used to quantify mortality. These numbers vary by region and over time and are influenced by the volume of testing, healthcare system quality, treatment options, time since the initial outbreak, and population characteristics such as age, sex, and overall health.The mortality rate reflects the number of deaths within a specific demographic group divided by the population of that demographic group. Consequently, the mortality rate reflects the prevalence as well as the severity of the disease within a given population. Mortality rates are highly correlated to age, with relatively low rates for young people and relatively high rates among the elderly. In fact, one relevant factor of mortality rates is the age structure of the countries\\' populations. For example, the case fatality rate for COVID‑19 is lower in India than in the US since India\\'s younger population represents a larger percentage than in the US.\\n\\n\\n=== Case fatality rate ===\\nThe case fatality rate (CFR) reflects the number of deaths divided by the number of diagnosed cases within a given time interval. Based on Johns Hopkins University statistics, the global death-to-case ratio is 1.24% (6,183,197/500,280,791) as of 12 April 2022. The number varies by region.\\n\\n\\t\\t\\n\\t\\t\\n\\t\\t\\n\\n\\n=== Infection fatality rate ===\\nA key metric in gauging the severity of COVID‑19 is the infection fatality rate (IFR), also referred to as the infection fatality ratio or infection fatality risk. This metric is calculated by dividing the total number of deaths from the disease by the total number of infected individuals; hence, in contrast to the CFR, the IFR incorporates asymptomatic and undiagnosed infections as well as reported cases.\\n\\n\\n==== Estimates ====\\n\\nA December 2020 systematic review and meta-analysis estimated that population IFR during the first wave of the pandemic was about 0.5% to 1% in many locations (including France, Netherlands, New Zealand, and Portugal), 1% to 2% in other locations (Australia, England, Lithuania, and Spain), and exceeded 2% in Italy. That study also found that most of these differences in IFR reflected corresponding differences in the age composition of the population and age-specific infection rates; in particular, the metaregression estimate of IFR is very low for children and younger adults (e.g., 0.002% at age 10 and 0.01% at age 25) but increases progressively to 0.4% at age 55, 1.4% at age 65, 4.6% at age 75, and 15% at age 85. These results were also highlighted in a December 2020 report issued by the WHO.\\nAn analysis of those IFR rates indicates that COVID‑19 is hazardous not only for the elderly but also for middle-aged adults, for whom the infection fatality rate of COVID-19 is two orders of magnitude greater than the annualized risk of a fatal automobile accident and far more dangerous than seasonal influenza.\\n\\n\\n==== Earlier estimates of IFR ====\\nAt an early stage of the pandemic, the World Health Organization reported estimates of IFR between 0.3% and 1%. On 2 July, The WHO\\'s chief scientist reported that the average IFR estimate presented at a two-day WHO expert forum was about 0.6%. In August, the WHO found that studies incorporating data from broad serology testing in Europe showed IFR estimates converging at approximately 0.5–1%. Firm lower limits of IFRs have been established in a number of locations such as New York City and Bergamo in Italy since the IFR cannot be less than the population fatality rate. (After sufficient time however, people can get reinfected). As of 10 July, in New York City, with a population of 8.4 million, 23,377 individuals (18,758 confirmed and 4,619 probable) have died with COVID‑19 (0.3% of the population). Antibody testing in New York City suggested an IFR of ~0.9%, and ~1.4%. In Bergamo province, 0.6% of the population has died. In September 2020, the U.S. Centers for Disease Control and Prevention (CDC) reported preliminary estimates of age-specific IFRs for public health planning purposes.\\n\\n\\n=== Sex differences ===\\n\\nCOVID‑19 case fatality rates are higher among men than women in most countries. However, in a few countries like India, Nepal, Vietnam, and Slovenia the fatality cases are higher in women than men. Globally, men are more likely to be admitted to the ICU and more likely to die. One meta-analysis found that globally, men were more likely to get COVID‑19 than women; there were approximately 55 men and 45 women per 100 infections (CI: 51.43–56.58).The Chinese Center for Disease Control and Prevention reported the death rate was 2.8% for men and 1.7% for women. Later reviews in June 2020 indicated that there is no significant difference in susceptibility or in CFR between genders. One review acknowledges the different mortality rates in Chinese men, suggesting that it may be attributable to lifestyle choices such as smoking and drinking alcohol rather than genetic factors. Smoking, which in some countries like China is mainly a male activity, is a habit that contributes to increasing significantly the case fatality rates among men. Sex-based immunological differences, lesser prevalence of smoking in women and men developing co-morbid conditions such as hypertension at a younger age than women could have contributed to the higher mortality in men. In Europe as of February 2020, 57% of the infected people were men and 72% of those died with COVID‑19 were men. As of April 2020, the US government is not tracking sex-related data of COVID‑19 infections. Research has shown that viral illnesses like Ebola, HIV, influenza and SARS affect men and women differently.\\n\\n\\n=== Ethnic differences ===\\nIn the US, a greater proportion of deaths due to COVID‑19 have occurred among African Americans and other minority groups. Structural factors that prevent them from practicing social distancing include their concentration in crowded substandard housing and in \"essential\" occupations such as retail grocery workers, public transit employees, health-care workers and custodial staff. Greater prevalence of lacking health insurance and care of underlying conditions such as diabetes, hypertension, and heart disease also increase their risk of death. Similar issues affect Native American and Latino communities. On the one hand, in the Dominican Republic there is a clear example of both gender and ethnic inequality. In this Latin American territory, there is great inequality and precariousness that especially affects Dominican women, with greater emphasis on those of Haitian descent. According to a US health policy non-profit, 34% of American Indian and Alaska Native People (AIAN) non-elderly adults are at risk of serious illness compared to 21% of white non-elderly adults. The source attributes it to disproportionately high rates of many health conditions that may put them at higher risk as well as living conditions like lack of access to clean water.Leaders have called for efforts to research and address the disparities. In the UK, a greater proportion of deaths due to COVID‑19 have occurred in those of a Black, Asian, and other ethnic minority background. More severe impacts upon victims including the relative incidence of the necessity of hospitalization requirements, and vulnerability to the disease has been associated via DNA analysis to be expressed in genetic variants at chromosomal region 3, features that are associated with European Neanderthal heritage. That structure imposes greater risks that those affected will develop a more severe form of the disease. The findings are from Professor Svante Pääbo and researchers he leads at the Max Planck Institute for Evolutionary Anthropology and the Karolinska Institutet. This admixture of modern human and Neanderthal genes is estimated to have occurred roughly between 50,000 and 60,000 years ago in Southern Europe.\\n\\n\\n=== Comorbidities ===\\nBiological factors (immune response) and the general behaviour (habits) can strongly determine the consequences of COVID‑19. Most of those who die of COVID‑19 have pre-existing (underlying) conditions, including hypertension, diabetes mellitus, and cardiovascular disease. According to March data from the United States, 89% of those hospitalised had preexisting conditions. The Italian Istituto Superiore di Sanità reported that out of 8.8% of deaths where medical charts were available, 96.1% of people had at least one comorbidity with the average person having 3.4 diseases. According to this report the most common comorbidities are hypertension (66% of deaths), type 2 diabetes (29.8% of deaths), Ischemic Heart Disease (27.6% of deaths), atrial fibrillation (23.1% of deaths) and chronic renal failure (20.2% of deaths).\\nMost critical respiratory comorbidities according to the Centers for Disease Control and Prevention (CDC), are: moderate or severe asthma, pre-existing COPD, pulmonary fibrosis, cystic fibrosis. Evidence stemming from meta-analysis of several smaller research papers also suggests that smoking can be associated with worse outcomes. When someone with existing respiratory problems is infected with COVID‑19, they might be at greater risk for severe symptoms. COVID‑19 also poses a greater risk to people who misuse opioids and methamphetamines, insofar as their drug use may have caused lung damage.In August 2020, the CDC issued a caution that tuberculosis (TB) infections could increase the risk of severe illness or death. The WHO recommended that people with respiratory symptoms be screened for both diseases, as testing positive for COVID‑19 could not rule out co-infections. Some projections have estimated that reduced TB detection due to the pandemic could result in 6.3 million additional TB cases and 1.4 million TB-related deaths by 2025.\\n\\n\\n== History ==\\n\\nThe virus is thought to be of natural animal origin, most likely through spillover infection. There are several theories about where the index case originated and investigations into the origin of the pandemic are ongoing. Phylogenetics estimates that SARS-CoV-2 arose in October or November 2019. A phylogenetic algorithm analysis suggested that the virus may have been circulating in Guangdong before Wuhan. Evidence suggests that it descends from a coronavirus that infects wild bats, and spread to humans through an intermediary wildlife host. The possibility that the virus was accidentally released from a laboratory is also under increasingly active consideration. U.S intelligence agencies found that the virus was not developed as a biological weapon and that it is unlikely for it to have been genetically engineered.The first confirmed human infections were in Wuhan, Hubei, China. A study of the first 41 cases of confirmed COVID‑19, published in January 2020 in The Lancet, reported the earliest date of onset of symptoms as 1 December 2019. Official publications from the WHO reported the earliest onset of symptoms as 8 December 2019. Human-to-human transmission was confirmed by the WHO and Chinese authorities by 20 January 2020. According to official Chinese sources, these were mostly linked to the Huanan Seafood Wholesale Market, which also sold live animals. In May 2020, George Gao, the director of the CDC, said animal samples collected from the seafood market had tested negative for the virus, indicating that the market was the site of an early superspreading event, but that it was not the site of the initial outbreak. Traces of the virus have been found in wastewater samples that were collected in Milan and Turin, Italy, on 18 December 2019.By December 2019, the spread of infection was almost entirely driven by human-to-human transmission. The number of COVID-19 cases in Hubei gradually increased, reaching sixty by 20 December, and at least 266 by 31 December. On 24 December, Wuhan Central Hospital sent a bronchoalveolar lavage fluid (BAL) sample from an unresolved clinical case to sequencing company Vision Medicals. On 27 and 28 December, Vision Medicals informed the Wuhan Central Hospital and the Chinese CDC of the results of the test, showing a new coronavirus. A pneumonia cluster of unknown cause was observed on 26 December and treated by the doctor Zhang Jixian in Hubei Provincial Hospital, who informed the Wuhan Jianghan CDC on 27 December. On 30 December, a test report addressed to Wuhan Central Hospital, from company CapitalBio Medlab, stated an erroneous positive result for SARS, causing a group of doctors at Wuhan Central Hospital to alert their colleagues and relevant hospital authorities of the result. The Wuhan Municipal Health Commission issued a notice to various medical institutions on \"the treatment of pneumonia of unknown cause\" that same evening. Eight of these doctors, including Li Wenliang (punished on 3 January), were later admonished by the police for spreading false rumours and another, Ai Fen, was reprimanded by her superiors for raising the alarm.The Wuhan Municipal Health Commission made the first public announcement of a pneumonia outbreak of unknown cause on 31 December, confirming 27 cases – enough to trigger an investigation.During the early stages of the outbreak, the number of cases doubled approximately every seven and a half days. In early and mid-January 2020, the virus spread to other Chinese provinces, helped by the Chinese New Year migration and Wuhan being a transport hub and major rail interchange. On 20 January, China reported nearly 140 new cases in one day, including two people in Beijing and one in Shenzhen. Later official data shows 6,174 people had already developed symptoms by then, and more may have been infected. A report in The Lancet on 24 January indicated human transmission, strongly recommended personal protective equipment for health workers, and said testing for the virus was essential due to its \"pandemic potential\". On 30 January, the WHO declared COVID-19 a Public Health Emergency of International Concern. By this time, the outbreak spread by a factor of 100 to 200 times.Italy had its first confirmed cases on 31 January 2020, two tourists from China. Italy overtook China as the country with the most deaths on 19 March 2020. By 26 March the United States had overtaken China and Italy with the highest number of confirmed cases in the world. Research on coronavirus genomes indicates the majority of COVID-19 cases in New York came from European travellers, rather than directly from China or any other Asian country. Retesting of prior samples found a person in France who had the virus on 27 December 2019, and a person in the United States who died from the disease on 6 February 2020.RT-PCR testing of untreated wastewater samples from Brazil and Italy have suggested detection of SARS-CoV-2 as early as November and December 2019, respectively, but the methods of such sewage studies have not been optimised, many have not been peer-reviewed, details are often missing, and there is a risk of false positives due to contamination or if only one gene target is detected. A September 2020 review journal article said, \"The possibility that the COVID‑19 infection had already spread to Europe at the end of last year is now indicated by abundant, even if partially circumstantial, evidence,\" including pneumonia case numbers and radiology in France and Italy in November and December.As of 1 October 2021, Reuters reported that it had estimated the worldwide total number of deaths due to COVID‑19 to have exceeded five million.\\n\\n\\n== Misinformation ==\\n\\nAfter the initial outbreak of COVID‑19, misinformation and disinformation regarding the origin, scale, prevention, treatment, and other aspects of the disease rapidly spread online.In September 2020, the US Centers for Disease Control and Prevention (CDC) published preliminary estimates of the risk of death by age groups in the United States, but those estimates were widely misreported and misunderstood.\\n\\n\\n== Other species ==\\n\\nHumans appear to be capable of spreading the virus to some other animals, a type of disease transmission referred to as zooanthroponosis.\\nSome pets, especially cats and ferrets, can catch this virus from infected humans. Symptoms in cats include respiratory (such as a cough) and digestive symptoms. Cats can spread the virus to other cats, and may be able to spread the virus to humans, but cat-to-human transmission of SARS-CoV-2 has not been proven. Compared to cats, dogs are less susceptible to this infection. Behaviors which increase the risk of transmission include kissing, licking, and petting the animal.The virus does not appear to be able to infect pigs, ducks, or chickens at all. Mice, rats, and rabbits, if they can be infected at all, are unlikely to be involved in spreading the virus.Tigers and lions in zoos have become infected as a result of contact with infected humans. As expected, monkeys and great ape species such as orangutans can also be infected with the COVID‑19 virus.Minks, which are in the same family as ferrets, have been infected. Minks may be asymptomatic, and can also spread the virus to humans. Multiple countries have identified infected animals in mink farms. Denmark, a major producer of mink pelts, ordered the slaughter of all minks over fears of viral mutations, following an outbreak referred to as Cluster 5. A vaccine for mink and other animals is being researched.\\n\\n\\n== Research ==\\n\\nInternational research on vaccines and medicines in COVID‑19 is underway by government organisations, academic groups, and industry researchers. The CDC has classified it to require a BSL3 grade laboratory. There has been a great deal of COVID‑19 research, involving accelerated research processes and publishing shortcuts to meet the global demand.As of December 2020, hundreds of clinical trials have been undertaken, with research happening on every continent except Antarctica. As of November 2020, more than 200 possible treatments had been studied in humans so far.\\n\\n\\n=== Transmission and prevention research ===\\n\\nModelling research has been conducted with several objectives, including predictions of the dynamics of transmission, diagnosis and prognosis of infection, estimation of the impact of interventions, or allocation of resources. Modelling studies are mostly based on compartmental models in epidemiology, estimating the number of infected people over time under given conditions. Several other types of models have been developed and used during the COVID‑19 including computational fluid dynamics models to study the flow physics of COVID‑19, retrofits of crowd movement models to study occupant exposure, mobility-data based models to investigate transmission, or the use of macroeconomic models to assess the economic impact of the pandemic. Further, conceptual frameworks from crisis management research have been applied to better understand the effects of COVID‑19 on organizations worldwide.\\n\\n\\n=== Treatment-related research ===\\n\\nRepurposed antiviral drugs make up most of the research into COVID‑19 treatments. Other candidates in trials include vasodilators, corticosteroids, immune therapies, lipoic acid, bevacizumab, and recombinant angiotensin-converting enzyme 2.In March 2020, the World Health Organization (WHO) initiated the Solidarity trial to assess the treatment effects of some promising drugs: an experimental drug called remdesivir; anti-malarial drugs chloroquine and hydroxychloroquine; two anti-HIV drugs, lopinavir/ritonavir; and interferon-beta. More than 300 active clinical trials are underway as of April 2020.Research on the antimalarial drugs hydroxychloroquine and chloroquine showed that they were ineffective at best, and that they may reduce the antiviral activity of remdesivir. By May 2020, France, Italy, and Belgium had banned the use of hydroxychloroquine as a COVID‑19 treatment.In June, initial results from the randomised RECOVERY Trial in the United Kingdom showed that dexamethasone reduced mortality by one third for people who are critically ill on ventilators and one fifth for those receiving supplemental oxygen. Because this is a well-tested and widely available treatment, it was welcomed by the WHO, which is in the process of updating treatment guidelines to include dexamethasone and other steroids. Based on those preliminary results, dexamethasone treatment has been recommended by the NIH for patients with COVID‑19 who are mechanically ventilated or who require supplemental oxygen but not in patients with COVID‑19 who do not require supplemental oxygen.In September 2020, the WHO released updated guidance on using corticosteroids for COVID‑19. The WHO recommends systemic corticosteroids rather than no systemic corticosteroids for the treatment of people with severe and critical COVID‑19 (strong recommendation, based on moderate certainty evidence). The WHO suggests not to use corticosteroids in the treatment of people with non-severe COVID‑19 (conditional recommendation, based on low certainty evidence). The updated guidance was based on a meta-analysis of clinical trials of critically ill COVID‑19 patients.In September 2020, the European Medicines Agency (EMA) endorsed the use of dexamethasone in adults and adolescents from twelve years of age and weighing at least 40 kilograms (88 lb) who require supplemental oxygen therapy. Dexamethasone can be taken by mouth or given as an injection or infusion (drip) into a vein.In November 2020, the US Food and Drug Administration (FDA) issued an emergency use authorization for the investigational monoclonal antibody therapy bamlanivimab for the treatment of mild-to-moderate COVID‑19. Bamlanivimab is authorized for people with positive results of direct SARS-CoV-2 viral testing who are twelve years of age and older weighing at least 40 kilograms (88 lb), and who are at high risk for progressing to severe COVID‑19 or hospitalization. This includes those who are 65 years of age or older, or who have chronic medical conditions.In February 2021, the FDA issued an emergency use authorization (EUA) for bamlanivimab and etesevimab administered together for the treatment of mild to moderate COVID‑19 in people twelve years of age or older weighing at least 40 kilograms (88 lb) who test positive for SARS‑CoV‑2 and who are at high risk for progressing to severe COVID‑19. The authorized use includes treatment for those who are 65 years of age or older or who have certain chronic medical conditions.In April 2021, the FDA revoked the emergency use authorization (EUA) that allowed for the investigational monoclonal antibody therapy bamlanivimab, when administered alone, to be used for the treatment of mild-to-moderate COVID‑19 in adults and certain pediatric patients.\\n\\n\\n==== Cytokine storm ====\\n\\nA cytokine storm can be a complication in the later stages of severe COVID‑19. A cytokine storm is a potentially deadly immune reaction where a large amount of pro-inflammatory cytokines and chemokines are released too quickly. A cytokine storm can lead to ARDS and multiple organ failure. Data collected from Jin Yin-tan Hospital in Wuhan, China indicates that patients who had more severe responses to COVID‑19 had greater amounts of pro-inflammatory cytokines and chemokines in their system than patients who had milder responses. These high levels of pro-inflammatory cytokines and chemokines indicate presence of a cytokine storm.Tocilizumab has been included in treatment guidelines by China\\'s National Health Commission after a small study was completed. It is undergoing a Phase II non-randomised trial at the national level in Italy after showing positive results in people with severe disease. Combined with a serum ferritin blood test to identify a cytokine storm (also called cytokine storm syndrome, not to be confused with cytokine release syndrome), it is meant to counter such developments, which are thought to be the cause of death in some affected people. The interleukin-6 receptor (IL-6R) antagonist was approved by the FDA to undergo a Phase III clinical trial assessing its effectiveness on COVID‑19 based on retrospective case studies for the treatment of steroid-refractory cytokine release syndrome induced by a different cause, CAR T cell therapy, in 2017. There is no randomised, controlled evidence that tocilizumab is an efficacious treatment for CRS. Prophylactic tocilizumab has been shown to increase serum IL-6 levels by saturating the IL-6R, driving IL-6 across the blood-brain barrier, and exacerbating neurotoxicity while having no effect on the incidence of CRS.Lenzilumab, an anti-GM-CSF monoclonal antibody, is protective in murine models for CAR T cell-induced CRS and neurotoxicity and is a viable therapeutic option due to the observed increase of pathogenic GM-CSF secreting T cells in hospitalised patients with COVID‑19.\\n\\n\\n==== Passive antibodies ====\\n\\nTransferring purified and concentrated antibodies produced by the immune systems of those who have recovered from COVID‑19 to people who need them is being investigated as a non-vaccine method of passive immunisation. Viral neutralization is the anticipated mechanism of action by which passive antibody therapy can mediate defence against SARS-CoV-2. The spike protein of SARS-CoV-2 is the primary target for neutralizing antibodies. As of 8 August 2020, eight neutralizing antibodies targeting the spike protein of SARS-CoV-2 have entered clinical studies. It has been proposed that selection of broad-neutralizing antibodies against SARS-CoV-2 and SARS-CoV might be useful for treating not only COVID‑19 but also future SARS-related CoV infections. Other mechanisms, however, such as antibody-dependent cellular cytotoxicity or phagocytosis, may be possible. Other forms of passive antibody therapy, for example, using manufactured monoclonal antibodies, are in development.The use of passive antibodies to treat people with active COVID‑19 is also being studied. This involves the production of convalescent serum, which consists of the liquid portion of the blood from people who recovered from the infection and contains antibodies specific to this virus, which is then administered to active patients. This strategy was tried for SARS with inconclusive results. An updated Cochrane review in May 2021 found high certainty evidence that, for the treatment of people with moderate to severe COVID‑19, convalescent plasma did not reduce mortality or bring about symptom improvement. There continues to be uncertainty about the safety of convalescent plasma administration to people with COVID‑19 and differing outcomes measured in different studies limits their use in determining efficacy.\\n\\n\\n=== Bioethics ===\\nSince the outbreak of the COVID‑19 pandemic, scholars have explored the bioethics, normative economics, and political theories of healthcare policies related to the public health crisis. Academics have pointed to the moral distress of healthcare workers, ethics of distributing scarce healthcare resources such as ventilators, and the global justice of vaccine diplomacies. The socio-economic inequalities between genders, races, groups with disabilities, communities, regions, countries, and continents have also drawn attention in academia and the general public.\\n\\n\\n== Effect on other diseases and the pharmacy trade ==\\nThere was a report on 3 March 2021, that social distancing and common wearing of surgical masks and similar as a common precaution against COVID‑19 caused a drop in the spread rate of the common cold and flu.\\n\\n\\n== See also ==\\nCoronavirus diseases, a group of closely related syndromes\\nDisease X, a WHO term\\nLaw of declining virulence – Claim by epidemiologist Theobald Smith\\nTheory of virulence – Theory by biologist Paul W. Ewald\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\n\\n\\n== External links ==\\n\\n\\n=== Health agencies ===\\nCoronavirus disease (COVID‑19) Facts by the World Health Organization (WHO)\\nCoronavirus 2019 (COVID-19) by the US Centers for Disease Control and Prevention (CDC)\\nCoronavirus (COVID‑19) by the UK National Health Service (NHS)\\n\\n\\n=== Directories ===\\nCoronavirus Resource Center at the Center for Inquiry\\nCOVID-19 at Curlie\\nCOVID‑19 Resource Directory on OpenMD\\nCOVID‑19 Information on FireMountain.net\\n\\n\\n=== Medical journals ===\\nCoronavirus Disease 2019 (COVID‑19) by JAMA\\nBMJ\\'s Coronavirus (covid‑19) Hub by the BMJ\\nNovel Coronavirus Information Center by Elsevier\\nCOVID‑19 Resource Centre by The Lancet\\nCoronavirus (COVID‑19) Research Highlights by Springer Nature\\nCoronavirus (Covid‑19) by The New England Journal of Medicine\\nCovid‑19: Novel Coronavirus by Wiley Publishing\\n\\n\\n=== Treatment guidelines ===\\n\"JHMI Clinical Recommendations for Available Pharmacologic Therapies for COVID-19\" (PDF). Johns Hopkins Medicine.\\n\"Bouncing Back From COVID-19: Your Guide to Restoring Movement\" (PDF). Johns Hopkins Medicine.\\n\"Guidelines on the Treatment and Management of Patients with COVID-19\". Infectious Diseases Society of America.\\n\"Coronavirus Disease 2019 (COVID-19) Treatment Guidelines\" (PDF). National Institutes of Health.\\nWorld Health Organization (2022). Therapeutics and COVID-19: living guideline, 14 January 2022 (Report). hdl:10665/351006. WHO/2019-nCoV/therapeutics/2022.1.\\nNHS England and NHS Improvement. National Guidance for post-COVID syndrome assessment clinics (Report).',\n",
              "  1.1294824682884341),\n",
              " ('There are many variants of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus that causes coronavirus disease 2019 (COVID-19). Some are believed, or have been stated, to be of particular importance due to their potential for increased transmissibility, increased virulence, or reduced effectiveness of vaccines against them. These variants contribute to the continuation of the COVID-19 pandemic.\\nFive SARS-CoV-2 variants have been designated as variants of concern by the World Health Organization: the Alpha, Beta, Gamma, Delta, and Omicron variants.\\n\\n\\n== Overview ==\\n\\nThe emergence of SARS-CoV-2 may have resulted from recombination events between a bat SARS-like coronavirus and a pangolin coronavirus through cross-species transmission. The earliest available SARS-CoV-2 viral genomes were collected from patients in December 2019, and Chinese researchers compared these early genomes with bat and pangolin coronavirus strains to estimate the ancestral human coronavirus type; the identified ancestral genome type was labeled \"S\", and its dominant derived type was labeled \"L\" to reflect the mutant amino acid changes. Independently, Western researchers carried out similar analyses but labeled the ancestral type \"A\" and the derived type \"B\". The B-type mutated into further types including B.1, which is the ancestor of the major global variants of concern, labeled in 2021 by the WHO as alpha, beta, gamma, delta and omicron variants.Early in the pandemic, the relatively low number of infections (compared with later stages of the pandemic) resulted in fewer opportunities for mutation of the viral genome and, therefore, fewer opportunities for the occurrence of differentiated variants. Since the occurrence of variants was rarer, the observation of S-protein mutations in the receptor-binding domain (RBD) region interacting with ACE2 was also not frequent.As time went on, the evolution of SARS-CoV-2\\'s genome (by means of random mutations) led to mutant specimens of the virus (i.e., genetic variants), observed to be more transmissible, to be naturally selected. Notably, both the Alpha and the Delta variants were observed to be more transmissible than previously identified viral strains.Some SARS-CoV-2 variants are considered to be of concern as they maintain (or even increase) their replication fitness in the face of rising population immunity, either by infection recovery or via vaccination. Some of the variants of concern show mutations in the RBD of the S-protein.The following table presents information and relative risk level for currently and formerly circulating variants of concern (VOC). The intervals assume a 95% confidence or credibility level, unless otherwise stated. Currently, all estimates are approximations due to the limited availability of data for studies. For Alpha, Beta, Gamma and Delta, there is no change in test accuracy, and neutralising antibody activity is retained by some monoclonal antibodies. PCR tests continue to detect the Omicron variant.\\n  Very high risk   High risk   Medium risk   Low risk   Unknown risk\\n\\n\\n== Nomenclature ==\\n\\nSARS-CoV-2 variants are grouped according to their lineage and component mutations. As of July 2021, no consistent nomenclature was established for it. Many organisations, including governments and news outlets, referred colloquially to concerning variants by the country in which they were first identified. After months of discussions, the World Health Organization announced Greek-letter names for important strains on 31 May 2021, so they could be easily referred to in a simple, easy to say, and non-stigmatising fashion. This decision may have partially been taken because of criticism from governments on using country names to refer to variants of the virus; the WHO mentioned the potential for mentioning country names to cause stigma. After using all the letters from Alpha to Mu (see below), in November 2021 the WHO skipped the next two letters of the Greek alphabet, Nu and Xi, and used Omicron, prompting speculation that Xi was skipped to avoid offending Chinese President Xi Jinping. The WHO gave as the explanation that Nu is too easily confounded with \"new\" and Xi is a common last name. In the event that the WHO uses the entirety of the Greek alphabet, the agency considered naming future variants after constellations.\\n\\n\\n=== Lineages and clades ===\\nWhile there are many thousands of variants of SARS-CoV-2, subtypes of the virus can be put into larger groupings such as lineages or clades. Three main, generally used nomenclatures have been proposed:\\n\\nAs of January 2021, GISAID—referring to SARS-CoV-2 as hCoV-19—had identified eight global clades (S, O, L, V, G, GH, GR, and GV).\\nIn 2017, Hadfield et al. announced Nextstrain, intended \"for real-time tracking of pathogen evolution\". Nextstrain has later been used for tracking SARS-CoV-2, identifying 13 major clades (19A–B, 20A–20J and 21A) as of June 2021.\\nIn 2020, Rambaut et al. of the Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) software team proposed in an article \"a dynamic nomenclature for SARS-CoV-2 lineages that focuses on actively circulating virus lineages and those that spread to new locations\"; as of August 2021, 1340 lineages had been designated.Each national public health institute may also institute its own nomenclature system for the purposes of tracking specific variants. For example, Public Health England designated each tracked variant by year, month and number in the format [YYYY] [MM]/[NN], prefixing \\'VUI\\' or \\'VOC\\' for a variant under investigation or a variant of concern respectively. This system has now been modified and now uses the format [YY] [MMM]-[NN], where the month is written out using a three-letter code.\\n\\n\\n=== Classification of variants ===\\nVariants that appear to meet one or more specific criteria considered during the COVID-19 pandemic may be labeled \"variants of interest\" or \"variants under investigation\" (\\'VUI\\') pending verification and validation of these properties. Once validated, variants of interest /VUI  may be renamed \"variants of concern\" by monitoring organizations, such as the CDC in the US. A related category is \"variant of high consequence\", used by the CDC if there is clear evidence that the effectiveness of prevention or intervention measures for a particular variant is substantially reduced.\\n\\n\\n== Reference sequence ==\\nAs it is currently not known when the index case or \"patient zero\" occurred, the choice of reference sequence for a given study is relatively arbitrary, with different notable research studies\\' choices varying as follows:\\n\\nThe earliest sequence, Wuhan-1, was collected on 24 December 2019.\\nOne group (Sudhir Kumar et al.) refers extensively to an NCBI reference genome (GenBankID:NC_045512; GISAID ID: EPI_ISL_402125), this sample was collected on 26 December 2019, although they also used the WIV04 GISAID reference genome (ID: EPI_ISL_402124), in their analyses.\\nAccording to another source (Zhukova et al.), the sequence WIV04/2019, belonging to the GISAID S clade / PANGO A lineage / Nextstrain 19B clade, is thought to most closely reflect the sequence of the original virus infecting humans—known as \"sequence zero\". WIV04/2019 was sampled from a symptomatic patient on 30 December 2019 and is widely used (especially by those collaborating with GISAID) as a reference sequence.The variant first sampled and identified in Wuhan, China is considered by researchers to differ from the progenitor genome by three mutations. Subsequently, many distinct lineages of SARS-CoV-2 have evolved.\\n\\n\\n== Notability criteria ==\\nViruses generally acquire mutations over time, giving rise to new variants. When a new variant appears to be growing in a population, it can be labelled as an \"emerging variant\". In the case of SARS-CoV-2, new lineages often differ from one another by just a few nucleotides.Some of the potential consequences of emerging variants are the following:\\nIncreased transmissibility\\nIncreased morbidity\\nIncreased mortality\\nAbility to evade detection by diagnostic tests\\nDecreased susceptibility to antiviral drugs (if and when such drugs are available)\\nDecreased susceptibility to neutralising antibodies, either therapeutic (e.g., convalescent plasma or monoclonal antibodies) or in laboratory experiments\\nAbility to evade natural immunity (e.g., causing reinfections)\\nAbility to infect vaccinated individuals\\nIncreased risk of particular conditions such as multisystem inflammatory syndrome or long COVID.\\nIncreased affinity for particular demographic or clinical groups, such as children or immunocompromised individuals.Variants that appear to meet one or more of these criteria may be labelled \"variants under investigation\" or \"variants of interest\" pending verification and validation of these properties. The primary characteristic of a variant of interest is that it shows evidence that demonstrates it is the cause of an increased proportion of cases or unique outbreak clusters; however, it must also have limited prevalence or expansion at national levels, or the classification would be elevated to a \"variant of concern\". If there is clear evidence that the effectiveness of prevention or intervention measures for a particular variant is substantially reduced, that variant is termed a \"variant of high consequence\".\\n\\n\\n== Variants of concern (WHO) ==\\nListed below are the Variants of Concern (VOC) recognised by the World Health Organization as of June 2021. Other organisations such as the CDC in the United States have at times used a slightly different list. As of July 2021, their list matched that of the WHO.\\n\\n\\n=== Alpha (lineage B.1.1.7) ===\\n\\nFirst detected in October 2020 during the COVID-19 pandemic in the United Kingdom from a sample taken the previous month in Kent, lineage B.1.1.7, labelled Alpha variant by the WHO, was previously known as the first Variant Under Investigation in December 2020 (VUI – 202012/01) and later notated as VOC-202012/01. It is also known as 20I (V1), 20I/501Y.V1 (formerly 20B/501Y.V1), or 501Y.V1. From October to December 2020, its prevalence doubled every 6.5 days, the presumed generational interval. It is correlated with a significant increase in the rate of COVID-19 infection in United Kingdom, associated partly with the N501Y mutation. There was some evidence that this variant had 40–80% increased transmissibility (with most estimates lying around the middle to higher end of this range), and early analyses suggested an increase in lethality, though later work found no evidence of increased virulence. As of May 2021, the Alpha variant had been detected in some 120 countries.\\n\\n\\n==== B.1.1.7 with E484K ====\\nVariant of Concern 21FEB-02 (previously written as VOC-202102/02), described by Public Health England (PHE) as \"B.1.1.7 with E484K\" is of the same lineage in the Pango nomenclature system, but has an additional E484K mutation. As of 17 March 2021, there were 39 confirmed cases of VOC-21FEB-02 in the UK. On 4 March 2021, scientists reported B.1.1.7 with E484K mutations in the state of Oregon. In 13 test samples analysed, one had this combination, which appeared to have arisen spontaneously and locally, rather than being imported. Other names for this variant include B.1.1.7+E484K and B.1.1.7 Lineage with S:E484K.\\n\\n\\n=== Beta (lineage B.1.351) ===\\n\\nOn 18 December 2020, the 501.V2 variant, also known as 501.V2, 20H (V2), 20H/501Y.V2 (formerly 20C/501Y.V2), 501Y.V2, VOC-20DEC-02 (formerly VOC-202012/02), or lineage B.1.351, was first detected in South Africa and reported by the country\\'s health department. It has been labelled as Beta variant by WHO. Researchers and officials reported that the prevalence of the variant was higher among young people with no underlying health conditions, and by comparison with other variants it is more frequently resulting in serious illness in those cases. The South African health department also indicated that the variant may be driving the second wave of the COVID-19 epidemic in the country due to the variant spreading at a more rapid pace than other earlier variants of the virus.Scientists noted that the variant contains several mutations that allow it to attach more easily to human cells because of the following three mutations in the receptor-binding domain (RBD) in the spike glycoprotein of the virus: N501Y, K417N, and E484K. The N501Y mutation has also been detected in the United Kingdom.\\n\\n\\n=== Gamma (lineage P.1) ===\\n\\nThe Gamma variant or lineage P.1, termed Variant of Concern 21JAN-02 (formerly VOC-202101/02) by Public Health England, 20J (V3) or 20J/501Y.V3 by Nextstrain, or just 501Y.V3, was detected in Tokyo on 6 January 2021 by the National Institute of Infectious Diseases (NIID). It has been labelled as Gamma variant by WHO. The new variant was first identified in four people who arrived in Tokyo having travelled from the Brazilian Amazonas state on 2 January 2021. On 12 January 2021, the Brazil-UK CADDE Centre confirmed 13 local cases of the new Gamma variant in the Amazon rainforest. This variant of SARS-CoV-2 has been named lineage P.1 (although it is a descendant of B.1.1.28, the name B.1.1.28.1 is not permitted and thus the resultant name is P.1), and has 17 unique amino acid changes, 10 of which in its spike protein, including the three concerning mutations: N501Y, E484K and K417T.:\\u200aFigure 5\\u200aThe N501Y and E484K mutations favour the formation of a stable RBD-hACE2 complex, thus, enhancing the binding affinity of RBD to hACE2. However, the K417T mutation disfavours complex formation between RBD and hACE2, which has been demonstrated to reduce the binding affinity.The new variant was absent in samples collected from March to November 2020 in Manaus, Amazonas state, but it was detected for the same city in 42% of the samples from 15 to 23 December 2020, followed by 52.2% during 15–31 December and 85.4% during 1–9 January 2021. A study found that infections by Gamma can produce nearly ten times more viral load compared to persons infected by one of the other lineages identified in Brazil (B.1.1.28 or B.1.195). Gamma also showed 2.2 times higher transmissibility with the same ability to infect both adults and older persons, suggesting P.1 and P.1-like lineages are more successful at infecting younger humans irrespective of sex.A study of samples collected in Manaus between November 2020 and January 2021, indicated that the Gamma variant is 1.4–2.2 times more transmissible and was shown to be capable of evading 25–61% of inherited immunity from previous coronavirus diseases, leading to the possibility of reinfection after recovery from an earlier COVID-19 infection. As for the fatality ratio, infections by Gamma were also found to be 10–80% more lethal.A study found that people fully vaccinated with Pfizer or Moderna have significantly decreased neutralisation effect against Gamma, although the actual impact on the course of the disease is uncertain.\\nA pre-print study by the Oswaldo Cruz Foundation published in early April found that the real-world performance of people with the initial dose of the Sinovac\\'s Coronavac Vaccine had approximately 50% efficacy rate. They expected the efficacy to be higher after the 2nd dose. As of July 2021, the study is ongoing.Preliminary data from two studies indicate that the Oxford–AstraZeneca vaccine is effective against the Gamma variant, although the exact level of efficacy has not yet been released. Preliminary data from a study conducted by Instituto Butantan suggest that CoronaVac is effective against the Gamma variant as well, and as of July 2021 has yet to be expanded to obtain definitive data.\\n\\n\\n=== Delta (lineage B.1.617.2) ===\\n\\nThe Delta variant, also known as B.1.617.2, G/452R.V3, 21A or 21A/S:478K, is a globally dominant variant that spread to at least 185 countries. It was first discovered in India. Descendant of lineage B.1.617, which also includes the Kappa variant under investigation, it was first discovered in October 2020 and has since spread internationally. On 6 May 2021, British scientists declared B.1.617.2 (which notably lacks mutation at E484Q) as a \"variant of concern\", labelling it VOC-21APR-02, after they flagged evidence that it spreads more quickly than the original version of the virus and could spread quicker or as quickly as Alpha. It carries L452R and P681R mutations in Spike; unlike Kappa it carries T478K but not E484Q.\\nOn 3 June 2021, Public Health England reported that twelve of the 42 deaths from the Delta variant in England were among the fully vaccinated, and that it was spreading almost twice as fast as the Alpha variant. Also on 11 June, Foothills Medical Centre in Calgary, Canada reported that half of their 22 cases of the Delta variant occurred among the fully vaccinated.In June 2021, reports began to appear of a variant of Delta with the K417N mutation. The mutation, also present in the Beta and Gamma variants, raised concerns about the possibility of reduced effectiveness of vaccines and antibody treatments and increased risk of reinfection. The variant, called \"Delta with K417N\" by Public Health England, includes two clades corresponding to the Pango lineages AY.1 and AY.2. It has been nicknamed \"Delta plus\" from \"Delta plus K417N\". The name of the mutation, K417N, refers to an exchange whereby lysine (K) is replaced by asparagine (N) at position 417. On 22 June, India\\'s Ministry of Health and Family Welfare declared the \"Delta plus\" variant of COVID-19 a Variant of Concern after 22 cases of the variant were reported in India. After the announcement, leading virologists said there was insufficient data to support labelling the variant as a distinct variant of concern, pointing to the small number of patients studied. In the UK in July 2021, AY.4.2 was identified. Alongside those previously mentioned it also gained the nickname \\'Delta Plus\\', on the strength of its extra mutations, Y145H and A222V. These are not unique to it, but distinguish it from the original Delta variant.\\n\\n\\n=== Omicron ===\\n\\n\\n==== Lineage B.1.1.529 ====\\nThe Omicron variant, known as lineage B.1.1.529, was declared a variant of concern by the World Health Organization on 26 November 2021.The variant has a large number of mutations, of which some are concerning. The number of cases in the B.1.1.529 lineage is increasing in all areas of South Africa. Some evidence shows that this variant has an increased risk of reinfection. Studies are underway to evaluate the exact impact on transmissibility, mortality, and other factors.Named Omicron by the WHO, it was identified in November 2021 in Botswana and South Africa; one case had travelled to Hong Kong, one confirmed case was identified in Israel in a traveler returning from Malawi, along with two who returned from South Africa and one from Madagascar. Belgium confirmed the first detected case in Europe on 26 November 2021 in an individual who had returned from Egypt on 11 November. Indian SARS-CoV-2 Genomics Consortium (INSACOG) in its January 2022 bulletin noted that Omicron is in community transmission in India where new cases have been rising exponentially.\\n\\n\\n===== Sublineages (including BA.2) =====\\nAccording to the WHO, BA.1, BA.1.1, and BA.2 are the most common sublineages of Omicron globally as of February 2022. BA.2 contains 28 unique genetic changes, including four in its spike protein, compared to BA.1, which had already acquired 60 mutations since the ancestral Wuhan strain, including 32 in the spike protein. BA.2 is more transmissible than BA.1. It was causing most cases in England by mid-March 2022, and by the end of March, BA.2 became dominant in the US.\\n\\n\\n== Variants of interest (WHO) ==\\nListed below are the Variants of Interest (VOI) which are, as of August 2021, recognised by the World Health Organization. Other organisations such as the CDC in the United States may at times use a slightly different list.\\n\\n\\n=== Lambda (lineage C.37) ===\\n\\nThe Lambda variant, also known as lineage C.37, was first detected in Peru in August 2020 and was designated by the WHO as a variant of interest on 14 June 2021. It spread to at least 30 countries around the world and, as of July 2021, it is unknown whether it is more infectious and resistant to vaccines than other strains.\\n\\n\\n=== Mu (lineage B.1.621) ===\\n\\nThe Mu variant, also known as lineage B.1.621, was first detected in Colombia in January 2021 and was designated by the WHO as a variant of interest on 30 August 2021. There have been outbreaks in South America and Europe.\\n\\n\\n== Former variants of interest ==\\n\\n\\n=== Epsilon (lineages B.1.429, B.1.427, CAL.20C) ===\\n\\nThe Epsilon variant or lineage B.1.429, also known as CAL.20C or CA VUI1, 21C or 20C/S:452R, is defined by five distinct mutations (I4205V and D1183Y in the ORF1ab gene, and S13I, W152C, L452R in the spike protein\\'s S-gene), of which the L452R (previously also detected in other unrelated lineages) was of particular concern. From 17 March to 29 June 2021, the CDC listed B.1.429 and the related B.1.427 as \"variants of concern\". As of July 2021, Epsilon is no longer considered a variant of interest by the WHO, as it was overtaken by Alpha.From September 2020 to January 2021, it was 19% to 24% more transmissible than earlier variants in California. Neutralisation against it by antibodies from natural infections and vaccinations was moderately reduced, but it remained detectable in most diagnostic tests.Epsilon (CAL.20C) was first observed in July 2020 by researchers at the Cedars-Sinai Medical Center, California, in one of 1,230 virus samples collected in Los Angeles County since the start of the COVID-19 epidemic. It was not detected again until September when it reappeared among samples in California, but numbers remained very low until November. In November 2020, the Epsilon variant accounted for 36 per cent of samples collected at Cedars-Sinai Medical Center, and by January 2021, the Epsilon variant accounted for 50 per cent of samples. In a joint press release by University of California, San Francisco, California Department of Public Health, and Santa Clara County Public Health Department, the variant was also detected in multiple counties in Northern California. From November to December 2020, the frequency of the variant in sequenced cases from Northern California rose from 3% to 25%. In a preprint, CAL.20C is described as belonging to clade 20C and contributing approximately 36% of samples, while an emerging variant from the 20G clade accounts for some 24% of the samples in a study focused on Southern California. Note, however, that in the US as a whole, the 20G clade predominates, as of January 2021. Following the increasing numbers of Epsilon in California, the variant has been detected at varying frequencies in most US states. Small numbers have been detected in other countries in North America, and in Europe, Asia and Australia. After an initial increase, its frequency rapidly dropped from February 2021 as it was being outcompeted by the more transmissible Alpha. In April, Epsilon remained relatively frequent in parts of northern California, but it had virtually disappeared from the south of the state and had never been able to establish a foothold elsewhere; only 3.2% of all cases in the United States were Epsilon, whereas more than two-thirds were Alpha.\\n\\n\\n=== Zeta (lineage P.2) ===\\n\\nZeta variant or lineage P.2, a sub-lineage of B.1.1.28 like Gamma (P.1), was first detected in circulation in the state of Rio de Janeiro; it harbours the E484K mutation, but not the N501Y and K417T mutations. It evolved independently in Rio de Janeiro without being directly related to the Gamma variant from Manaus. Though previously Zeta was labeled a variant of interest, as of July 2021, it is no longer considered as such by the WHO.\\n\\n\\n=== Eta (lineage B.1.525) ===\\n\\nThe Eta variant or lineage B.1.525, also called VUI-21FEB-03 (previously VUI-202102/03) by Public Health England (PHE) and formerly known as UK1188, 21D or 20A/S:484K, does not carry the same N501Y mutation found in Alpha, Beta and Gamma, but carries the same E484K-mutation as found in the Gamma, Zeta, and Beta variants, and also carries the same ΔH69/ΔV70 deletion (a deletion of the amino acids histidine and valine in positions 69 and 70) as found in Alpha, N439K variant (B.1.141 and B.1.258) and Y453F variant (Cluster 5). Eta differs from all other variants by having both the E484K-mutation and a new F888L mutation (a substitution of phenylalanine (F) with leucine (L) in the S2 domain of the spike protein). As of 5 March 2021, it had been detected in 23 countries. It has also been reported in Mayotte, the overseas department/region of France. The first cases were detected in December 2020 in the UK and Nigeria, and as of 15 February 2021, it had occurred in the highest frequency among samples in the latter country. As of 24 February 56 cases were found in the UK. Denmark, which sequences all its COVID-19 cases, found 113 cases of this variant from 14 January to 21 February 2021, of which seven were directly related to foreign travel to Nigeria.As of July 2021, UK experts are studying it to ascertain how much of a risk it could be. It is currently regarded as a \"variant under investigation\", but pending further study, it may become a \"variant of concern\". Ravi Gupta, from the University of Cambridge said in a BBC interview that lineage B.1.525 appeared to have \"significant mutations\" already seen in some of the other newer variants, which means their likely effect is to some extent more predictable.\\n\\n\\n=== Theta (lineage P.3) ===\\n\\nOn 18 February 2021, the Department of Health of the Philippines confirmed the detection of two mutations of COVID-19 in Central Visayas after samples from patients were sent to undergo genome sequencing. The mutations were later named as E484K and N501Y, which were detected in 37 out of 50 samples, with both mutations co-occurrent in 29 out of these.On 13 March, the Department of Health confirmed the mutations constitutes a variant which was designated as lineage P.3. On the same day, it also confirmed the first COVID-19 case caused by the Gamma variant in the country. The Philippines had 98 cases of the Theta variant on 13 March. On 12 March it was announced that Theta had also been detected in Japan. On 17 March, the United Kingdom confirmed its first two cases, where PHE termed it VUI-21MAR-02.\\nOn 30 April 2021, Malaysia detected 8 cases of the Theta variant in Sarawak.As of July 2021, Theta is no longer considered a variant of interest by the WHO.\\n\\n\\n=== Iota (lineage B.1.526) ===\\n\\nIn November 2020, a mutant variant was discovered in New York City, which was named lineage B.1.526. As of 11 April 2021, the variant has been detected in at least 48 U.S. states and 18 countries. In a pattern mirroring Epsilon, Iota was initially able to reach relatively high levels in some states, but by May 2021 it was outcompeted by the more transmissible Delta and Alpha.\\n\\n\\n=== Kappa (lineage B.1.617.1) ===\\n\\nThe Kappa variant is one of the three sublineages of lineage B.1.617. It is also known as lineage B.1.617.1, 21B or 21A/S:154K, and was first detected in India in December 2020. By the end of March 2021, Kappa accounted for more than half of the sequences being submitted from India. On 1 April 2021, it was designated a variant under investigation (VUI-21APR-01) by Public Health England. It has the notable mutations L452R, E484Q, P681R.\\n\\n\\n== Variants under monitoring (WHO) ==\\nDefined as variants with genetic changes suspected to affect virus characteristics and some indication of posing a future risk, but with unclear evidence of phenotypic or epidemiological impact, requiring enhanced monitoring and repeat assessment after new evidence. Some former variants of interest are monitored as well.\\n\\n\\n== Formerly monitored variants (WHO) ==\\n\\n\\n== Other notable variants ==\\nLineage B.1.1.207 was first sequenced in August 2020 in Nigeria; the implications for transmission and virulence are unclear but it has been listed as an emerging variant by the US Centers for Disease Control. Sequenced by the African Centre of Excellence for Genomics of Infectious Diseases in Nigeria, this variant has a P681H mutation, shared in common with the Alpha variant. It shares no other mutations with the Alpha variant and as of late December 2020 this variant accounts for around 1% of viral genomes sequenced in Nigeria, though this may rise. As of May 2021, lineage B.1.1.207 has been detected in 10 countries.Lineage B.1.1.317, while not considered a variant of concern, is noteworthy in that Queensland Health forced 2 people undertaking hotel quarantine in Brisbane, Australia to undergo an additional 5 days\\' quarantine on top of the mandatory 14 days after it was confirmed they were infected with this variant.Lineage B.1.616, being identified in Brittany, Western France in early January 2021 and designated by WHO as \"Variant under investigation\" in March 2021, was reported to be difficult to detect from nasopharyngeal swab sampling method of coronavirus detection, and detection of the virus needs to rely on samples from lower respiratory tract.Lineage B.1.618 was first isolated in October 2020. It has the E484K mutation in common with several other variants, and showed significant spread in April 2021 in West Bengal, India. As of 23 April 2021, the PANGOLIN database showed 135 sequences detected in India, with single-figure numbers in each of eight other countries worldwide.In July 2021, scientists reported in a preprint which was published in a journal in February 2022, the detection of anomalous unnamed unknown-host SARS-CoV-2 lineages via wastewater surveillance in New York City. They hypothesized that \"these lineages are derived from unsampled human COVID-19 infections or that they indicate the presence of a non-human animal reservoir\". Lineage B.1.640.2 (also known as the IHU variant) was detected in October 2021 by researchers at the Institut Hospitalo-Universitaire (IHU) in Marseille. They found the variant in a traveler who returned to France from Cameroon and reportedly infected 12 people. The B.1.640 lineage, which includes B.1.640.2, was designated a variant under monitoring (VUM) by the World Health Organization (WHO) on 22 November 2021. However, the WHO has reported that lineage B.1.640.2 has spread much slower than the Omicron variant, and so is of relatively little concern.  According to a preprint study, lineage B.1.640.2 has two already known spike protein mutations – E484K and N501Y – among a total of 46 nucleotide substitutions and 37 deletions.\\n\\n\\n== Notable missense mutations ==\\nThere have been a number of missense mutations observed of SARS-CoV-2.\\n\\n\\n=== del 69-70 ===\\nThe name of the mutation, del 69-70, or 69-70 del, or other similar notations, refers to the deletion of amino acid at position 69 to 70. The mutation is found in the Alpha variant, and could lead to \"spike gene target failure\" and result in false negative result in PCR virus test.\\n\\n\\n=== RSYLTPGD246-253N ===\\nOtherwise referred to as del 246-252, or other various similar expression, refer to the deletion of amino acid from the position of 246 to 252, in the N-terminal domain of spike protein, accompanied with a replacement of the aspartic acid (D) at the position 253 for asparagine (N).The 7 amino acid deletion mutation is currently described as unique in the Lambda variant, and have been attributed to as one of the cause of the strain\\'s increased capability to escape from neutralizing antibodies according to preprint paper.\\n\\n\\n=== N440K ===\\nThe name of the mutation, N440K, refers to an exchange whereby the asparagine (N) is replaced by lysine (K) at position 440.This mutation has been observed in cell cultures to be 10 times more infective compared to the previously widespread A2a strain (A97V substitution in RdRP sequence) and 1000 times more in the lesser widespread A3i strain (D614G substitution in Spike and a and P323L substitution in RdRP). It was involved in rapid surges of COVID-19 cases in India in May 2021. India has the largest proportion of N440K mutated variants followed by the US and Germany.\\n\\n\\n=== G446V ===\\nThe name of the mutation, G446V, refers to an exchange whereby the glycine (G) is replaced by valine (V) at position 446.The mutation, identified in Japan among inbound travelers starting from May, and among 33 samples from individuals related to 2020 Tokyo Olympic Games and 2020 Tokyo Paralympic Games, are said to be possible to impact affinity of multiple monoclonal antibody, although its clinical impact against the use of antibody medicine is still yet to be known.\\n\\n\\n=== L452R ===\\nThe name of the mutation, L452R, refers to an exchange whereby the leucine (L) is replaced by arginine (R) at position 452.L452R is found in both the Delta and Kappa variants which first circulated in India, but have since spread around the world. L452R is a relevant mutation in this strain that enhances ACE2 receptor binding ability and can reduce vaccine-stimulated antibodies from attaching to this altered spike protein.\\nL452R, some studies show, could even make the coronavirus resistant to T cells, that are class of cells necessary to target and destroy virus-infected cells. They are different from antibodies that are useful in blocking coronavirus particles and preventing it from proliferating.\\n\\n\\n=== Y453F ===\\nThe name of the mutation, Y453F, refers to an exchange whereby the tyrosine (Y) is replaced by phenylalanine (F) at position 453. The mutation have been found potentially linked to the spread of SARS-CoV-2 among minks in the Netherlands in 2020.\\n\\n\\n=== S477G/N ===\\nA highly flexible region in the receptor binding domain (RBD) of SARS-CoV-2, starting from residue 475 and continuing up to residue 485, was identified using bioinformatics and statistical methods in several studies. The University of Graz and the Biotech Company Innophore have shown in a recent publication that structurally, the position S477 shows the highest flexibility among them.At the same time, S477 is hitherto the most frequently exchanged amino acid residue in the RBDs of SARS-CoV-2 mutants. By using molecular dynamics simulations of RBD during the binding process to hACE2, it has been shown that both S477G and S477N strengthen the binding of the SARS-COV-2 spike with the hACE2 receptor. The vaccine developer BioNTech referenced this amino acid exchange as relevant regarding future vaccine design in a preprint published in February 2021.\\n\\n\\n=== E484Q ===\\nThe name of the mutation, E484Q, refers to an exchange whereby the glutamic acid (E) is replaced by glutamine (Q) at position 484.The Kappa variant circulating in India has E484Q. These variants were initially (but misleadingly) referred to as a \"double mutant\". E484Q may enhance ACE2 receptor binding ability, and may reduce vaccine-stimulated antibodies\\' ability to attach to this altered spike protein.\\n\\n\\n=== E484K ===\\nThe name of the mutation, E484K, refers to an exchange whereby the glutamic acid (E) is replaced by lysine (K) at position 484. It is nicknamed \"Eeek\".E484K has been reported to be an escape mutation (i.e., a mutation that improves a virus\\'s ability to evade the host\\'s immune system) from at least one form of monoclonal antibody against SARS-CoV-2, indicating there may be a \"possible change in antigenicity\". The Gamma variant (lineage P.1), the Zeta variant (lineage P.2, also known as lineage B.1.1.28.2) and the Beta variant (501.V2) exhibit this mutation. A limited number of lineage B.1.1.7 genomes with E484K mutation have also been detected. Monoclonal and serum-derived antibodies are reported to be from 10 to 60 times less effective in neutralising virus bearing the E484K mutation. On 2 February 2021, medical scientists in the United Kingdom reported the detection of E484K in 11 samples (out of 214,000 samples), a mutation that may compromise current vaccine effectiveness.\\n\\n\\n=== F490S ===\\nF490S denotes a change from phenylalanine (F) to serine (S) in amino-acid position 490.It is one of the mutation found in Lambda, and have been associated with reduced susceptibility to antibody generated by those who were infected with other strains, meaning antibody treatment against people infected with strains carrying such mutation would be less effective.\\n\\n\\n=== N501Y ===\\nN501Y denotes a change from asparagine (N) to tyrosine (Y) in amino-acid position 501. N501Y has been nicknamed \"Nelly\".This change is believed by PHE to increase binding affinity because of its position inside the spike glycoprotein\\'s receptor-binding domain, which binds ACE2 in human cells; data also support the hypothesis of increased binding affinity from this change. Molecular interaction modelling and the free energy of binding calculations has demonstrated that the mutation N501Y has the highest binding affinity in variants of concern RBD to hACE2. Variants with N501Y include Gamma, Alpha (VOC 20DEC-01), Beta, and COH.20G/501Y (identified in Columbus, Ohio). This last became the dominant form of the virus in Columbus in late December 2020 and January and appears to have evolved independently of other variants.\\n\\n\\n=== N501S ===\\nN501S denotes a change from asparagine (N) to serine (S) in amino-acid position 501.As of September 2021, there are 8 cases of patients around the world infected with Delta variant which feature this N501S mutation. As it is considered a mutation similar to N501Y, it is suspected to have similar characteristics as N501Y mutation, which is believed to increase the infectivity of the virus, however the exact effect is unknown yet.\\n\\n\\n=== D614G ===\\n\\nD614G is a missense mutation that affects the spike protein of SARS-CoV-2. From early appearances in Eastern China early in 2020, the frequency of this mutation in the global viral population increased early on during the pandemic. G (glycine) quickly replaced D (aspartic acid) at position 614 in Europe, though more slowly in China and the rest of East Asia, supporting the hypothesis that G increased the transmission rate, which is consistent with higher viral titres and infectivity in vitro. Researchers with the PANGOLIN tool nicknamed this mutation \"Doug\".In July 2020, it was reported that the more infectious D614G SARS-CoV-2 variant had become the dominant form in the pandemic. PHE confirmed that the D614G mutation had a \"moderate effect on transmissibility\" and was being tracked internationally.The global prevalence of D614G correlates with the prevalence of loss of smell (anosmia) as a symptom of COVID-19, possibly mediated by higher binding of the RBD to the ACE2 receptor or higher protein stability and hence higher infectivity of the olfactory epithelium.Variants containing the D614G mutation are found in the G clade by GISAID and the B.1 clade by the PANGOLIN tool.\\n\\n\\n=== Q677P/H ===\\nThe name of the mutation, Q677P/H, refers to an exchange whereby the glutamine (Q) is replaced by proline (P) or histidine (H) at position 677. There are several sub-lineages containing the Q677P mutation; six of these, which also contain various different combinations of other mutations, are referred to by names of birds. One of the earlier ones noticed for example is known as \"Pelican,\" while the most common of these as of early 2021 was provisionally named \"Robin 1.\"The mutation has been reported in multiple lineages circulating inside the United States as of late 2020 and also some lineages outside the country. \\'Pelican\\' was first detected in Oregon, and as of early 2021 \\'Robin 1\\' was found often in the Midwestern United States, while another Q667H sub-lineage, \\'Robin 2\\', was found mostly in the southeastern United States. The frequency of such mutation being recorded has increased from late 2020 to early 2021.\\n\\n\\n=== P681H ===\\n\\nThe name of the mutation, P681H, refers to an exchange whereby the proline (P) is replaced by histidine (H) at position 681.In January 2021, scientists reported in a preprint that the mutation P681H, a characteristic feature of the Alpha variant and lineage B.1.1.207 (identified in Nigeria), is showing a significant exponential increase in worldwide frequency, thus following a trend to be expected in the lower limb of the logistics curve. This may be compared with the trend of the now globally prevalent D614G.\\n\\n\\n=== P681R ===\\nThe name of the mutation, P681R, refers to an exchange whereby the proline (P) is replaced by arginine (R) at position 681.Indian SARS-CoV-2 Genomics Consortium (INSACOG) found that other than the two mutations E484Q and L452R, there is also a third significant mutation, P681R in lineage B.1.617. All three concerning mutations are on the spike protein, the operative part of the coronavirus that binds to receptor cells of the body.\\n\\n\\n=== A701V ===\\nAccording to initial media reports, the Malaysian Ministry of Health announced on 23 December 2020 that it had discovered a mutation in the SARS-CoV-2 genome which they designated as A701B(sic), among 60 samples collected from the Benteng Lahad Datu cluster in Sabah. The mutation was characterised as being similar to the one found recently at that time in South Africa, Australia, and the Netherlands, although it was uncertain if this mutation was more infectious or aggressive than before. The provincial government of Sulu in neighbouring Philippines temporarily suspended travel to Sabah in response to the discovery of \\'A701B\\' due to uncertainty over the nature of the mutation.On 25 December 2020, the Malaysian Ministry of Health described a mutation A701V as circulating and present in 85% of cases (D614G was present in 100% of cases) in Malaysia. These reports also referred to samples collected from the Benteng Lahad Datu cluster. The text of the announcement was mirrored verbatim on the Facebook page of Noor Hisham Abdullah, Malay Director-General of Health, who was quoted in some of the news articles.The A701V mutation has the amino acid alanine (A) substituted by valine (V) at position 701 in the spike protein. Globally, South Africa, Australia, Netherlands and England also reported A701V at about the same time as Malaysia. In GISAID, the prevalence of this mutation is found to be about 0.18%. of cases.On 14 April 2021, the Malaysian Ministry of Health reported that the third wave, which had started in Sabah, has involved the introduction of variants with D614G and A701V mutations.\\n\\n\\n== Recombinant variants ==\\nThe British government has reported a number of recombinant variants of SARS-CoV-2. These recombinant lineages have been given the Pango lineage identifiers XD, XE, and XF.XE is a recombinant lineage of Pango lineages BA.1 and BA.2. As of March 2022 XE was believed to have a growth rate 9.8% greater than BA.2.\\n\\n\\n== Differential vaccine effectiveness ==\\n\\nThe interplay between the SARS-CoV-2 virus and its human hosts was initially natural but then started being altered by the rising availability of vaccines seen in 2021. The potential emergence of a SARS-CoV-2 variant that is moderately or fully resistant to the antibody response elicited by the COVID-19 vaccines may necessitate modification of the vaccines. The emergence of vaccine-resistant variants is more likely in a highly vaccinated population with uncontrolled transmission.As of February 2021, the US Food and Drug Administration believed that all FDA authorized vaccines remained effective in protecting against circulating strains of SARS-CoV-2.\\n\\n\\n== Data and methods ==\\nModern DNA sequencing, where available, may permit rapid detection (sometimes known as \\'real-time detection\\') of genetic variants that appear in pathogens during disease outbreaks. Through use of phylogenetic tree visualisation software, records of genome sequences can be clustered into groups of identical genomes all containing the same set of mutations. Each group represents a \\'variant\\', \\'clade\\', or \\'lineage\\', and comparison of the sequences allows the evolutionary path of a virus to be deduced. For SARS-CoV-2, until  March 2021, over 330,000 viral genomic sequences had been generated by molecular epidemiology studies across the world.\\n\\n\\n=== New variant detection and assessment ===\\nOn 26 January 2021, the British government said it would share its genomic sequencing capabilities with other countries in order to increase the genomic sequencing rate and trace new variants, and announced a \"New Variant Assessment Platform\". As of January 2021, more than half of all genomic sequencing of COVID-19 was carried out in the UK.Wastewater surveillance was demonstrated to be one technique to detect SARS-CoV-2 variants and to track their rise for studying related ongoing infection dynamics.\\n\\n\\n=== Testing ===\\nWhether one or more mutations visible in RT-PCR tests can be used reliably to identify a variant depends on the prevalence of other variants currently circulating in the same population.\\n\\n\\n== Incubation theory for multiple mutated variants ==\\n\\nResearchers have suggested that multiple mutations can arise in the course of the persistent infection of an immunocompromised patient, particularly when the virus develops escape mutations under the selection pressure of antibody or convalescent plasma treatment, with the same deletions in surface antigens repeatedly recurring in different patients.\\n\\n\\n== Cross-species transmission ==\\n\\nThere is a risk that COVID-19 could transfer from humans to other animal populations and could combine with other animal viruses to create yet more variants that are dangerous to humans. Reverse zoonosis spillovers may cause reservoirs for mutating variants that spill back to humans – another possible source for variants of concern, in addition to immunocompromised people.\\n\\n\\n=== Cluster 5 ===\\n\\nIn early November 2020, Cluster 5, also referred to as ΔFVI-spike by the Danish State Serum Institute (SSI), was discovered in Northern Jutland, Denmark. It is believed to have been spread from minks to humans via mink farms. On 4 November 2020, it was announced that the mink population in Denmark would be culled to prevent the possible spread of this mutation and reduce the risk of new mutations happening. A lockdown and travel restrictions were introduced in seven municipalities of Northern Jutland to prevent the mutation from spreading, which could compromise national or international responses to the COVID-19 pandemic. By 5 November 2020, some 214 mink-related human cases had been detected.The WHO stated that cluster 5 had a \"moderately decreased sensitivity to neutralising antibodies\". SSI warned that the mutation could reduce the effect of COVID-19 vaccines under development, although it was unlikely to render them useless. Following the lockdown and mass-testing, SSI announced on 19 November 2020 that cluster 5 in all probability had become extinct. As of 1 February 2021, authors to a peer-reviewed paper, all of whom were from the SSI, assessed that cluster 5 was not in circulation in the human population.\\n\\n\\n== See also ==\\n\\nRaTG13, the second closest known relative to SARS-CoV-2\\nPandemic prevention § Surveillance and mapping\\nCOVID-19 vaccine#Effectiveness\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\n\\n\\n== External links ==\\nCoVariants\\nCov-Lineages\\nGISAID – hCov19 Variants',\n",
              "  0.8469668857519411),\n",
              " ('A COVID‑19 vaccine is a vaccine intended to provide acquired immunity against severe acute respiratory syndrome coronavirus 2 (SARS‑CoV‑2), the virus that causes coronavirus disease 2019 (COVID‑19).\\nPrior to the COVID‑19 pandemic, an established body of knowledge existed about the structure and function of coronaviruses causing diseases like severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS). This knowledge accelerated the development of various vaccine platforms during early 2020. The initial focus of SARS-CoV-2 vaccines was on preventing symptomatic, often severe illness. On 10 January 2020, the SARS-CoV-2 genetic sequence data was shared through GISAID, and by 19 March, the global pharmaceutical industry announced a major commitment to address COVID‑19.The COVID‑19 vaccines are widely credited for their role in reducing the severity and death caused by COVID‑19. Many countries have implemented phased distribution plans that prioritize those at highest risk of complications, such as the elderly, and those at high risk of exposure and transmission, such as healthcare workers.As of 31 March 2022, 11.29 billion doses of COVID‑19 vaccines have been administered worldwide based on official reports from national public health agencies. By December 2020, more than 10 billion vaccine doses had been preordered by countries, with about half of the doses purchased by high-income countries comprising 14% of the world\\'s population.\\nDespite the extremely rapid development of effective mRNA and viral vector vaccines, worldwide vaccine equity has not been achieved. The development and use of whole inactivated virus (WIV)  and protein-based vaccines has also been recommended, especially for use in developing countries.\\n\\n\\n== Background ==\\n\\n\\t\\t\\n\\nPrior to COVID‑19, a vaccine for an infectious disease had never been produced in less than several years – and no vaccine existed for preventing a coronavirus infection in humans. However, vaccines have been produced against several animal diseases caused by coronaviruses, including (as of 2003) infectious bronchitis virus in birds, canine coronavirus, and feline coronavirus. Previous projects to develop vaccines for viruses in the family Coronaviridae that affect humans have been aimed at severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS). Vaccines against SARS and MERS have been tested in non-human animals.\\nAccording to studies published in 2005 and 2006, the identification and development of novel vaccines and medicines to treat SARS was a priority for governments and public health agencies around the world at that time. There is no cure or protective vaccine proven to be safe and effective against SARS in humans. There is also no proven vaccine against MERS. When MERS became prevalent, it was believed that existing SARS research might provide a useful template for developing vaccines and therapeutics against a MERS-CoV infection. As of March 2020, there was one (DNA-based) MERS vaccine which completed Phase I clinical trials in humans, and three others in progress, all being viral-vectored vaccines: two adenoviral-vectored (ChAdOx1-MERS, BVRS-GamVac) and one MVA-vectored (MVA-MERS-S).Vaccines that use an inactive or weakened virus that has been grown in eggs typically take more than a decade to develop. In contrast, mRNA is a molecule that can be made quickly, and research on mRNA to fight diseases was begun decades before the COVID‑19 pandemic by scientists such as Drew Weissman and Katalin Karikó, who tested on mice. Moderna began human testing of an mRNA vaccine in 2015. Viral vector vaccines were also developed for the COVID‑19 pandemic after the technology was previously cleared for Ebola.As multiple COVID‑19 vaccines have been authorized or licensed for use, real-world vaccine effectiveness (RWE) is being assessed using case control and observational studies. A study is investigating the long-lasting protection against SARS-CoV-2 provided by the mRNA vaccines.\\n\\n\\n== Formulation ==\\nAs of September 2020, eleven of the vaccine candidates in clinical development use adjuvants to enhance immunogenicity. An immunological adjuvant is a substance formulated with a vaccine to elevate the immune response to an antigen, such as the COVID‑19 virus or influenza virus. Specifically, an adjuvant may be used in formulating a COVID‑19 vaccine candidate to boost its immunogenicity and efficacy to reduce or prevent COVID‑19 infection in vaccinated individuals. Adjuvants used in COVID‑19 vaccine formulation may be particularly effective for technologies using the inactivated COVID‑19 virus and recombinant protein-based or vector-based vaccines. Aluminum salts, known as \"alum\", were the first adjuvant used for licensed vaccines, and are the adjuvant of choice in some 80% of adjuvanted vaccines. The alum adjuvant initiates diverse molecular and cellular mechanisms to enhance immunogenicity, including release of proinflammatory cytokines.\\n\\n\\n== Sequencing ==\\nIn November 2021, the full nucleotide sequences of the AstraZeneca and Pfizer/BioNTech vaccines were released by the UK Medicines and Healthcare products Regulatory Agency, in response to a freedom of information request.\\n\\n\\n== Clinical research ==\\n\\n\\n=== Post-vaccination complications ===\\n\\n\\n== Vaccine types ==\\n\\nAt least nine different technology platforms are under research and development to create an effective vaccine against COVID‑19. Most of the platforms of vaccine candidates in clinical trials are focused on the coronavirus spike protein (S protein) and its variants as the primary antigen of COVID‑19 infection, since the S protein triggers strong B-cell and T-cell immune responses. However, other coronavirus proteins are also being investigated for vaccine development, like the nucleocapsid, because they also induce a robust T-cell response and their genes are more conserved and recombine less frequently (compared to Spike). Future generations of COVID-19 vaccines that may target more and conserved genomic regions will also act as an insurance against the manifestation of catastrophic scenarios concerning the future evolutionary path of SARS-CoV-2, or any similar Coronavirus epidemic/pandemic.Platforms developed in 2020 involved nucleic acid technologies (nucleoside-modified messenger RNA and DNA), non-replicating viral vectors, peptides, recombinant proteins, live attenuated viruses, and inactivated viruses.Many vaccine technologies being developed for COVID‑19 are not like vaccines already in use to prevent influenza, but rather are using \"next-generation\" strategies for precise targeting of COVID‑19 infection mechanisms. Several of the synthetic vaccines use a 2P mutation to lock the spike protein into its prefusion configuration, stimulating an adaptive immune response to the virus before it attaches to a human cell. Vaccine platforms in development may improve flexibility for antigen manipulation, and effectiveness for targeting mechanisms of COVID‑19 infection in susceptible population subgroups, such as healthcare workers, the elderly, children, pregnant women, and people with weakened immune systems.\\n\\n\\n=== mRNA vaccines ===\\n\\nSeveral COVID‑19 vaccines, including the Pfizer–BioNTech and Moderna vaccines, have been developed to use RNA to stimulate an immune response. When introduced into human tissue, the vaccine contains either self-replicating RNA or messenger RNA (mRNA), which both cause cells to express the SARS-CoV-2 spike protein. This teaches the body how to identify and destroy the corresponding pathogen. RNA vaccines often, but not always, use nucleoside-modified messenger RNA. The delivery of mRNA is achieved by a coformulation of the molecule into lipid nanoparticles which protect the RNA strands and help their absorption into the cells.RNA vaccines were the first COVID‑19 vaccines to be authorized in the United Kingdom, the United States and the European Union. Authorized vaccines of this type are the Pfizer–BioNTech and Moderna vaccines. The CVnCoV RNA vaccine from CureVac failed in clinical trials.Severe allergic reactions are rare. In December 2020, 1,893,360 first doses of Pfizer–BioNTech COVID‑19 vaccine administration resulted in 175 cases of severe allergic reaction, of which 21 were anaphylaxis. For 4,041,396 Moderna COVID‑19 vaccine dose administrations in December 2020 and January 2021, only ten cases of anaphylaxis were reported. Lipid nanoparticles (LNPs) were most likely responsible for the allergic reactions.\\n\\n\\n=== Adenovirus vector vaccines ===\\nThese vaccines are examples of non-replicating viral vector vaccines, using an adenovirus shell containing DNA that encodes a SARS‑CoV‑2 protein. The viral vector-based vaccines against COVID‑19 are non-replicating, meaning that they do not make new virus particles, but rather produce only the antigen which elicits a systemic immune response.Authorized vaccines of this type are the Oxford–AstraZeneca COVID‑19 vaccine, the Sputnik V COVID‑19 vaccine, Convidecia, and the Janssen COVID‑19 vaccine.Convidecia and the Janssen COVID‑19 vaccine are both one-shot vaccines which offer less complicated logistics and can be stored under ordinary refrigeration for several months.Sputnik V uses Ad26 for its first dose, which is the same as Janssen\\'s only dose, and Ad5 for the second dose, which is the same as Convidecia\\'s only dose.\\nOn 11 August 2021, the developers of Sputnik V proposed, in view of the Delta case surge, that Pfizer test the Ad26 component (termed its \\'Light\\' version) as a booster shot:Delta cases surge in US & Israel shows mRNA vaccines need a heterogeneous booster to strengthen & prolong immune response. #SputnikV pioneered mix&match approach, combo trials & showed 83.1% efficacy vs Delta. Today RDIF offers Pfizer to start trial with Sputnik Light as booster. \\n\\n\\n=== Inactivated virus vaccines ===\\nInactivated vaccines consist of virus particles that are grown in culture and then killed using a method such as heat or formaldehyde to lose disease producing capacity, while still stimulating an immune response.Authorized vaccines of this type are the Chinese CoronaVac and the Sinopharm BIBP and WIBP vaccines; the Indian Covaxin; later this year the Russian CoviVac; the Kazakhstani vaccine QazVac; and the Iranian COVIran Barekat. Vaccines in clinical trials include the Valneva COVID‑19 vaccine.\\n\\n\\n=== Subunit vaccines ===\\nSubunit vaccines present one or more antigens without introducing whole pathogen particles. The antigens involved are often protein subunits, but can be any molecule that is a fragment of the pathogen.The authorized vaccines of this type are the peptide vaccine EpiVacCorona, ZF2001, MVC-COV1901, and Corbevax. Vaccines with pending authorizations or include the Novavax COVID‑19 vaccine, Soberana 02 (a conjugate vaccine), and the Sanofi–GSK vaccine.\\nThe V451 vaccine was previously in clinical trials, which were terminated because it was found that the vaccine may potentially cause incorrect results for subsequent HIV testing.\\n\\n\\n=== Intranasal ===\\nIntranasal vaccines target mucosal immunity in the nasal mucosa which is a portal for viral entrance to the body. These vaccines are designed to stimulate nasal immune factors, such as IgA. In addition to inhibiting the virus, nasal vaccines provide ease of administration because no needles (and the accompanying needle phobia) are involved. Nasal vaccines have been approved for influenza, but not for COVID-19.\\n\\n\\n=== Other types ===\\nAdditional types of vaccines that are in clinical trials include virus-like particle vaccines, multiple DNA plasmid vaccines, at least two lentivirus vector vaccines, a conjugate vaccine, and a vesicular stomatitis virus displaying the SARS‑CoV‑2 spike protein.Scientists investigated whether existing vaccines for unrelated conditions could prime the immune system and lessen the severity of COVID‑19 infection. There is experimental evidence that the BCG vaccine for tuberculosis has non-specific effects on the immune system, but no evidence that this vaccine is effective against COVID‑19.\\n\\n\\n== Planning and development ==\\n\\nSince January 2020, vaccine development has been expedited via unprecedented collaboration in the multinational pharmaceutical industry and between governments.Multiple steps along the entire development path are evaluated, including:\\nthe level of acceptable toxicity of the vaccine (its safety),\\ntargeting vulnerable populations,\\nthe need for vaccine efficacy breakthroughs,\\nthe duration of vaccination protection,\\nspecial delivery systems (such as oral or nasal, rather than by injection),\\ndose regimen,\\nstability and storage characteristics,\\nemergency use authorization before formal licensing,\\noptimal manufacturing for scaling to billions of doses, and\\ndissemination of the licensed vaccine.\\n\\n\\n=== Challenges ===\\nThere have been several unique challenges with COVID‑19 vaccine development.\\nThe urgency to create a vaccine for COVID‑19 led to compressed schedules that shortened the standard vaccine development timeline, in some cases combining clinical trial steps over months, a process typically conducted sequentially over several years. Public health programs have been described as in \"[a] race to vaccinate individuals\" with the early wave vaccines.Timelines for conducting clinical research – normally a sequential process requiring years – are being compressed into safety, efficacy, and dosing trials running simultaneously over months, potentially compromising safety assurance. As an example, Chinese vaccine developers and the government Chinese Center for Disease Control and Prevention began their efforts in January 2020, and by March were pursuing numerous candidates on short timelines, with the goal to showcase Chinese technology strengths over those of the United States, and to reassure the Chinese people about the quality of vaccines produced in China.The rapid development and urgency of producing a vaccine for the COVID‑19 pandemic was expected to increase the risks and failure rate of delivering a safe, effective vaccine. Additionally, research at universities is obstructed by physical distancing and closing of laboratories.Vaccines must progress through several phases of clinical trials to test for safety, immunogenicity, effectiveness, dose levels and adverse effects of the candidate vaccine. Vaccine developers have to invest resources internationally to find enough participants for Phase II–III clinical trials when the virus has proved to be a \"moving target\" of changing transmission rates across and within countries, forcing companies to compete for trial participants. Clinical trial organizers also may encounter people unwilling to be vaccinated due to vaccine hesitancy or disbelief in the science of the vaccine technology and its ability to prevent infection. As new vaccines are developed during the COVID‑19 pandemic, licensure of COVID‑19 vaccine candidates requires submission of a full dossier of information on development and manufacturing quality.\\n\\n\\n=== Organizations ===\\nInternationally, the Access to COVID‑19 Tools Accelerator is a G20 and World Health Organization (WHO) initiative announced in April 2020. It is a cross-discipline support structure to enable partners to share resources and knowledge. It comprises four pillars, each managed by two to three collaborating partners: Vaccines (also called \"COVAX\"), Diagnostics, Therapeutics, and Health Systems Connector. The WHO\\'s April 2020 \"R&D Blueprint (for the) novel Coronavirus\" documented a \"large, international, multi-site, individually randomized controlled clinical trial\" to allow \"the concurrent evaluation of the benefits and risks of each promising candidate vaccine within 3–6 months of it being made available for the trial.\" The WHO vaccine coalition will prioritize which vaccines should go into Phase II and III clinical trials, and determine harmonized Phase III protocols for all vaccines achieving the pivotal trial stage.National governments have also been involved in vaccine development. Canada announced funding of 96 projects for development and production of vaccines at Canadian companies and universities with plans to establish a \"vaccine bank\" that could be used if another coronavirus outbreak occurs, and to support clinical trials and develop manufacturing and supply chains for vaccines.China provided low-rate loans to one vaccine developer through its central bank, and \"quickly made land available for the company\" to build production plants. Three Chinese vaccine companies and research institutes are supported by the government for financing research, conducting clinical trials, and manufacturing.The United Kingdom government formed a COVID‑19 vaccine task force in April 2020 to stimulate local efforts for accelerated development of a vaccine through collaborations of industry, universities, and government agencies. The UK\\'s Vaccine Taskforce contributed to every phase of development from research to manufacturing.In the United States, the Biomedical Advanced Research and Development Authority (BARDA), a federal agency funding disease-fighting technology, announced investments to support American COVID‑19 vaccine development, and manufacture of the most promising candidates. In May 2020, the government announced funding for a fast-track program called Operation Warp Speed. By March 2021, BARDA had funded an estimated $19.3 billion in COVID‑19 vaccine development.Large pharmaceutical companies with experience in making vaccines at scale, including Johnson & Johnson, AstraZeneca, and GlaxoSmithKline (GSK), formed alliances with biotechnology companies, governments, and universities to accelerate progression toward effective vaccines.\\n\\n\\n=== History ===\\n\\n\\n== Effectiveness ==\\n\\n\\n=== Waning  effectiveness ===\\nIn older English care home residents protection against severe illness, hospitalisation and death was high immediately after vaccination,  However protection reduced significantly in the months following vaccination.  Protection among younger care home staff remained higher.  Regular boosters are recommended for older people and boosters every six months for care home residents appear reasonable.\\n\\n\\n=== Effectiveness against transmission ===\\nFully vaccinated individuals with breakthrough infections have peak viral load similar to unvaccinated cases and can efficiently transmit infection in household settings. However, the magnitude of transmission by individuals with Delta breakthrough infections appears to be 50% increased in unvaccinated individuals vs that in vaccinated individuals.\\n\\n\\n== Adverse events ==\\nSerious adverse events (AEs)associated with receipt of new vaccines targeting COVID‑19 are of high interest to the public. The official databases of reported adverse events include the World Health Organization’s Vigibase, the United States Vaccine Adverse Events Reporting System (VAERS) and the United Kingdom’s Yellow Card system. WHO’s Vigibase registered over 3.3 Million reported AEs as of March 1, 2022. This represents approximately 70% of all AEs reported for all vaccines since reporting began in 1964.All vaccines that are administered via intramuscular injection, including COVID‑19 vaccines, have side effects related to the mild trauma associated with the procedure and introduction of a foreign substance into the body. These include soreness, redness, rash, and inflammation at the injection site. Other common side effects include fatigue, headache, myalgia (muscle pain), and arthralgia (joint pain) which generally resolve within a few days.Increased menstrual disturbances in women age 18-30 have been reported. From a US control group of 2403 vaccinated participants, 10.6% had a change in cycle length of more than 8 days, which is deemed as clinically significant. Cycle lengths returned to normal after 2 full cycles post vaccination. A Norwegian study asking 5688 participants had 37.8% noting at least one change, with heavier than normal bleeding being the change most related to vaccination.One less-frequent side effect (that generally occurs in less than 1 in 1,000 people) is hypersensitivity (allergy) to one or more of the vaccine\\'s ingredients, which in some rare cases may cause anaphylaxis.\\nAnaphylaxis has occurred in approximately 2 to 5 people per million vaccinated in the United States. An increased risk of rare and potentially fatal thrombosis events have been associated following the administration of the Janssen (Johnson and Johnson) and Oxford-AstraZeneca COVID‑19 vaccines, with the highest reported rate among females in their 30s and 40s. The rate of thrombosis events following vaccination with the Johnson and Johnson and AstraZeneca vaccines has been estimated at 1 case per 100,000 vaccinations compared to between 0.22 and 1.57 cases per 100,000 per year in the general population. There is no increased risk for thrombotic events after vaccination with mRNA COVID‑19 vaccines like Pfizer and Moderna.\\n\\n\\n== Mix and match ==\\nAccording to studies, the combination of two different COVID-19 vaccines, also called cross vaccination or mix-and-match method, provides protection equivalent to that of mRNA vaccines – including protection against the Delta variant. Individuals who receive the combination of two different vaccines produce strong immune responses, with side effects no worse than those caused by standard regimens.\\n\\n\\n== Duration of immunity ==\\nAvailable data show that fully vaccinated individuals and those previously infected with SARS-CoV-2 have a low risk of subsequent infection for at least 6 months. Data are currently insufficient to determine an antibody titer threshold that indicates when an individual is protected from infection. Multiple studies show that antibody titers are associated with protection at the population level, but individual protection titers remain unknown. For some populations, such as the elderly and the immunocompromised, protection levels may be reduced after both vaccination and infection. Finally, current data suggest that the level of protection may not be the same for all variants of the virus.As new data continue to emerge, recommendations will need to be updated periodically. It is important to note that at this time, there is no authorized or approved test that providers or the public can use to reliably determine if a person is protected from infection.\\n\\n\\n== Society and culture ==\\n\\n\\n=== Distribution ===\\n\\nNote about table to the right: Number and percentage of people who have received at least one dose of a COVID‑19 vaccine (unless noted otherwise). May include vaccination of non-citizens, which can push totals beyond 100% of the local population. Table is updated daily by a bot.\\n\\n\\n=== Access ===\\n\\nCountries have extremely unequal access to the COVID-19 vaccine. Vaccine equity has not been achieved, or even approximated. The inequity has harmed both countries with poor access and countries with good access.Nations pledged to buy doses of the COVID‑19 vaccine before the doses were available. Though high-income nations represent only 14% of the global population, as of 15 November 2020, they had contracted to buy 51% of all pre-sold doses. Some high-income nations bought more doses than would be necessary to vaccinate their entire populations.\\n\\nOn 18 January 2021, WHO Director-General Tedros Adhanom Ghebreyesus warned of problems with equitable distribution: \"More than 39 million doses of vaccine have now been administered in at least 49 higher-income countries. Just 25 doses have been given in one lowest-income country. Not 25 million; not 25 thousand; just 25.\"In March, it was revealed the US attempted to convince Brazil not to purchase the Sputnik V COVID‑19 vaccine, fearing \"Russian influence\" in Latin America. Some nations involved in long-standing territorial disputes have reportedly had their access to vaccines blocked by competing nations; Palestine has accused Israel of blocking vaccine delivery to Gaza, while Taiwan has suggested that China has hampered its efforts to procure vaccine doses.A single dose of the COVID‑19 vaccine by AstraZeneca would cost 47 Egyptian pounds (EGP), and the authorities are selling it between 100 and 200 EGP. A report by Carnegie Endowment for International Peace cited the poverty rate in Egypt as around 29.7 percent, which constitutes approximately 30.5 million people, and claimed that about 15 million of the Egyptians would be unable to gain access to the luxury of vaccination. A human rights lawyer, Khaled Ali, launched a lawsuit against the government, forcing them to provide vaccination free of cost to all members of the public.According to immunologist Dr. Anthony Fauci, mutant strains of the virus and limited vaccine distribution pose continuing risks and he said: \"we have to get the entire world vaccinated, not just our own country.\" Edward Bergmark and Arick Wierson are calling for a global vaccination effort and wrote that the wealthier nations\\' \"me-first\" mentality could ultimately backfire because the spread of the virus in poorer countries would lead to more variants, against which the vaccines could be less effective.On 10 March 2021, the United States, Britain, European Union member states and some other members of the World Trade Organization (WTO) blocked a push by more than eighty developing countries to waive COVID‑19 vaccine patent rights in an effort to boost production of vaccines for poor nations. On 5 May 2021, the US government under President Joe Biden announced that it supports waiving intellectual property protections for COVID‑19 vaccines. The Members of the European Parliament have backed a motion demanding the temporary lifting of intellectual properties rights for COVID‑19 vaccines.\\n\\nIn a meeting in April 2021, the World Health Organization\\'s emergency committee addressed concerns of persistent inequity in the global vaccine distribution. Although 9 percent of the world\\'s population lives in the 29 poorest countries, these countries had received only 0.3% of all vaccines administered as of May 2021. On 15 March, Brazilian journalism agency Agência Pública reported that the country vaccinated about twice as many people who declare themselves white than black and noted that mortality from COVID‑19 is higher in the black population.In May 2021, UNICEF made an urgent appeal to industrialised nations to pool their excess COVID‑19 vaccine capacity to make up for a 125-million-dose gap in the COVAX program. The program mostly relied on the Oxford–AstraZeneca COVID‑19 vaccine produced by Serum Institute of India, which faced serious supply problems due to increased domestic vaccine needs in India from March to June 2021. Only a limited amount of vaccines can be distributed efficiently, and the shortfall of vaccines in South America and parts of Asia are due to a lack of expedient donations by richer nations. International aid organisations have pointed at Nepal, Sri Lanka, and Maldives as well as Argentina and Brazil, and some parts of the Caribbean as problem areas, where vaccines are in short supply. In mid-May 2021, UNICEF was also critical of the fact that most proposed donations of Moderna and Pfizer vaccines were not slated for delivery until the second half of 2021, or early in 2022.On 1 July 2021, the heads of the World Bank Group, the International Monetary Fund, the World Health Organization, and the World Trade Organization said in a joint statement: \"As many countries are struggling with new variants and a third wave of COVID‑19 infections, accelerating access to vaccines becomes even more critical to ending the pandemic everywhere and achieving broad-based growth. We are deeply concerned about the limited vaccines, therapeutics, diagnostics, and support for deliveries available to developing countries.\" In July 2021, The BMJ reported that countries have thrown out over 250,000 vaccine doses as supply exceeded demand and strict laws prevented the sharing of vaccines. A survey by The New York Times found that over a million doses of vaccine had been thrown away in ten U.S. states because federal regulations prohibit recalling them, preventing their redistribution abroad. Furthermore, doses donated close to expiration often cannot be administered quickly enough by recipient countries and end up having to be discarded. To help overcome this problem, the Prime minister of India announced that they would make their digital vaccination management platform CoWIN open to the global community. He also announced that India would also release the source code for contact tracing app Aarogya Setu for developers around the world. Around 142 countries including Afghanistan, Bangladesh, Bhutan, Maldives, Guyana, Antigua & Barbuda, St. Kitts & Nevis and Zambia expressed their interest in the application for COVID managment. Amnesty International and Oxfam International have criticized the support of vaccine monopolies by the governments of producing countries, noting that this is dramatically increasing the dose price by five times and often much more, creating an economic barrier to access for poor countries. Médecins Sans Frontières (Doctors without Borders) has also criticized vaccine monopolies and repeatedly called from their suspension, supporting the TRIPS Waiver. The waiver was first proposed in October 2020, and has support from most countries, but delayed by opposition from EU (especially Germany - major EU countries such as France, Italy and Spain support the exemption), UK, Norway, and Switzerland, among others. MSF called for a Day of Action in September 2021 to put pressure on the WTO Minister\\'s meeting in November, which was expected to discuss the TRIPS IP waiver.\\n\\nOn 4 August 2021, to reduce unequal distribution between rich and poor countries, the WHO called for a moratorium on a booster dose at least until the end of September. However, on 18 August, the United States government announced plans to offer booster doses 8 months after the initial course to the general population, starting with priority groups. Before the announcement, the WHO harshly criticized this type of decision, citing the lack of evidence for the need for boosters, except for patients with specific conditions. At this time, vaccine coverage of at least one dose was 58% in high-income countries and only 1.3% in low-income countries, and 1.14 million Americans already received an unauthorized booster dose. US officials argued that waning efficacy against mild and moderate disease might indicate reduced protection against severe disease in the coming months. Israel, France, Germany, and the United Kingdom have also started planning boosters for specific groups. On 14 September 2021, more than 140 former world leaders, and Nobel laureates, including former President of France François Hollande, former Prime Minister of the United Kingdom Gordon Brown, former Prime Minister of New Zealand Helen Clark, and Professor Joseph Stiglitz, called on the candidates to be the next German chancellor to declare themselves in favour of waiving intellectual property rules for COVID‑19 vaccines and transferring vaccine technologies. In November 2021, nursing unions in 28 countries have filed a formal appeal with the United Nations over the refusal of the UK, EU, Norway, Switzerland, and Singapore to temporarily waive patents for Covid vaccines.During his first international trip, President of Peru Pedro Castillo spoke at the seventy-sixth session of the United Nations General Assembly on 21 September 2021, proposing the creation of an international treaty signed by world leaders and pharmaceutical companies to guarantee universal vaccine access, arguing \"The battle against the pandemic has shown us the failure of the international community to cooperate under the principle of solidarity\".Optimizing the societal benefit of vaccination may benefit from a strategy that is tailored to the state of the pandemic, the demographics of a country, the age of the recipients, the availability of vaccines, and the individual risk for severe disease: In the UK, the interval between prime and boost dose was extended to vaccinate as many persons as early as possible, many countries are starting to give an additional booster shot to the immunosuppressed and the elderly, and research predicts an additional benefit of personalizing vaccine dose in the setting of limited vaccine availability when a wave of virus Variants of Concern hits a country.Despite the extremely rapid development of effective mRNA and viral vector vaccines, vaccine equity has not been achieved. The World Health Organization called for 70 per cent of the global population to be vaccinated by mid-2022, but as of March 2022 it was estimated that only one per cent of the 10 billion doses given worldwide had been administered in low-income countries. An additional 6 billion vaccinations may be needed to fill vaccine access gaps, particularly in developing countries. Given the projected availability of the newer vaccines, the development and use of whole inactivated virus (WIV)  and protein-based vaccines are also recommended. Organizations such as the Developing Country Vaccine Manufacturer Network could help to support the production of such vaccines in developing countries, with  lower production costs and greater ease of deployment.While vaccines substantially reduce the probability of infection, it is still possible for fully vaccinated people to contract and spread COVID‑19. Public health agencies have recommended that vaccinated people continue using preventive measures (wear face masks, social distance, wash hands) to avoid infecting others, especially vulnerable people, particularly in areas with high community spread. Governments have indicated that such recommendations will be reduced as vaccination rates increase and community spread declines.\\n\\n\\n==== Economics ====\\n\\nVaccine inequity damages the global economy, disrupting the global supply chain. Most vaccines were being reserved for wealthy countries, as of September 2021, some of which have more vaccine than is needed to fully vaccinate their populations. When people, undervaccinated, needlessly die, suffer disability, and live under lockdown restrictions, they cannot supply the same goods and services. This harms the economies of undervaccinated and overvaccinated countries alike. Since rich countries have larger economies, rich countries may lose more money to vaccine inequity than poor ones, though the poor ones will lose a higher percentage of GDP and suffer longer-term effects. High-income countries would profit an estimated US$4.80 for every $1 spent on giving vaccines to lower-income countries.The International Monetary Fund sees the vaccine divide between rich and poor nations as a serious obstacle to a global economic recovery. Vaccine inequity disproportionately affects refuge-providing states, as they tend to be poorer, and refugees and displaced people are economically more vulnerable even within those low-income states, so they have suffered more economically from vaccine inequity.\\n\\n\\n=== Liability ===\\nSeveral governments agreed to shield pharmaceutical companies like Pfizer and Moderna from negligence claims related to COVID‑19 vaccines (and treatments), as in previous pandemics, when governments also took on liability for such claims.\\nIn the US, these liability shields took effect on 4 February 2020, when the US Secretary of Health and Human Services Alex Azar published a notice of declaration under the Public Readiness and Emergency Preparedness Act (PREP Act) for medical countermeasures against COVID‑19, covering \"any vaccine, used to treat, diagnose, cure, prevent, or mitigate COVID‑19, or the transmission of SARS-CoV-2 or a virus mutating therefrom\". The declaration precludes \"liability claims alleging negligence by a manufacturer in creating a vaccine, or negligence by a health care provider in prescribing the wrong dose, absent willful misconduct.\" In other words, absent \"willful misconduct\", these companies can not be sued for money damages for any injuries that occur between 2020 and 2024 from the administration of vaccines and treatments related to COVID‑19. The declaration is effective in the United States through 1 October 2024.In December 2020, the UK government granted Pfizer legal indemnity for its COVID‑19 vaccine.In the European Union, the COVID‑19 vaccines are licensed under a Conditional Marketing Authorisation which does not exempt manufacturers from civil and administrative liability claims. While the purchasing contracts with vaccine manufacturers remain secret, they do not contain liability exemptions even for side-effects not known at the time of licensure.The Bureau of Investigative Journalism, a nonprofit news organization, reported in an investigation that unnamed officials in some countries, such as Argentina and Brazil, said that Pfizer demanded guarantees against costs of legal cases due to adverse effects in the form of liability waivers and sovereign assets such as federal bank reserves, embassy buildings or military bases, going beyond the expected from other countries such as the US. During the pandemic parliamentary inquiry in Brazil, Pfizer\\'s representative said that its terms for Brazil are the same as for all other countries with which it has signed deals.\\n\\n\\n=== Controversy ===\\nIn June 2021, a report revealed that the UB-612 vaccine, developed by the US-based COVAXX, was a for-profit venture initiated  by the Blackwater founder Erik Prince. In a series of text messages to Paul Behrends, the close associate recruited for the COVAXX project, Prince described the profit-making possibilities in selling the COVID‑19 vaccines. COVAXX provided no data from the clinical trials on safety or efficacy it conducted in Taiwan. The responsibility of creating distribution networks was assigned to an Abu Dhabi-based entity, which was mentioned as \"Windward Capital\" on the COVAXX letterhead but was actually Windward Holdings. The firm\\'s sole shareholder, which handled \"professional, scientific and technical activities\", was Erik Prince. In March 2021, COVAXX raised $1.35 billion in a private placement.\\n\\n\\n=== Misinformation and hesitancy ===\\n\\n\\n== See also ==\\n2009 swine flu pandemic vaccine\\nCOVID‑19 drug development\\nCOVID-19 vaccine clinical research: Effectiveness\\nCOVID‑19 drug repurposing research\\nCOVID‑19 vaccine card\\nUniversal coronavirus vaccine\\nVaccine passports during the COVID‑19 pandemic\\nList of COVID-19 vaccine authorizations\\nCOVID-19 vaccine side effects\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\n\\n\\n=== Vaccine protocols ===\\n\\n\\n== External links ==\\n\"The different types of COVID-19 vaccines\". World Health Organization (WHO).\\n\"COVID-19 vaccine tracker and landscape\". World Health Organization (WHO).\\nCOVID‑19 Vaccine Tracker. Regulatory Affairs Professionals Society (RAPS)\\n\"STAT\\'s Covid-19 Drugs and Vaccines Tracker\". Stat. 27 April 2020.\\n\"COVID-19 vaccines: development, evaluation, approval and monitoring\". European Medicines Agency (EMA).\\n\"Vaccine Development – 101\". U.S. Food and Drug Administration (FDA). 27 August 2021.\\n\"Tracking Omicron and Other Coronavirus Variants\". The New York Times.\\nM.I.T. Lecture 10: Kizzmekia Corbett, Vaccines on YouTube\\nM.I.T. Lecture 12: Dan Barouch, Covid-19 Vaccine Development on YouTube\\n\"COVID-19 vaccines: research and development\". European Medicines Agency (EMA).\\n\"The 5 Stages of COVID-19 Vaccine Development: What You Need to Know About How a Clinical Trial Works\". Johnson & Johnson. 23 September 2020.\\n\"How patent laws get in the way of the global coronavirus vaccine rollout\". The Conversation.\\n\"A New Entry in the Race for a Coronavirus Vaccine: Hope\". The New York Times. Archived from the original on 20 May 2020.\\n\"Halting Progress and Happy Accidents: How mRNA Vaccines Were Made\". The New York Times. Archived from the original on 15 January 2022.\\n\"Coronavirus vaccine – weekly summary of Yellow Card reporting\". Medicines and Healthcare products Regulatory Agency (MHRA).\\n\"Joint Statement from HHS Public Health and Medical Experts on COVID-19 Booster Shots\". U.S. Centers for Disease Control and Prevention (CDC). 18 August 2021.',\n",
              "  0.3835283412327698),\n",
              " ('The global COVID-19 pandemic arrived in Europe with its first confirmed case in Bordeaux, France, on 24 January 2020, and subsequently spread widely across the continent. By 17 March 2020, every country in Europe had confirmed a case, and all have reported at least one death, with the exception of Vatican City.\\nItaly was the first European country to experience a major outbreak in early 2020, becoming the first country worldwide to introduce a national lockdown. By 13 March 2020, the World Health Organization (WHO) declared Europe the epicentre of the pandemic and it remained so until the WHO announced it has been overtaken by South America on 22 May. By 18 March 2020, lockdowns introduced in Europe affected more than 250 million people. Despite deployment of COVID-19 vaccines, Europe became the pandemic\\'s epicentre once again in late 2021. On 11 January 2022, Dr. Hans Kluge, the WHO Regional Director for Europe said, \"more than 50 percent of the population in the region will be infected with Omicron in the next six to eight weeks\".As the outbreak became a major crisis across Europe, national and European Union responses have led to debate over restrictions of civil liberties and the extent of European Union solidarity.\\n\\n\\n== Statistics by country and territory ==\\n\\nIn late August, 88% of COVID-19 deaths in Europe were among people over age 65, according to a 30 August report from the WHO.\\n\\n\\n== Pandemic by country and territory ==\\n\\n\\n=== Albania ===\\n\\n\\n=== Andorra ===\\n\\n\\n=== Austria ===\\n\\n\\n=== Belarus ===\\n\\n\\n=== Belgium ===\\n\\n\\n=== Bosnia and Herzegovina ===\\n\\n\\n=== Bulgaria ===\\n\\n\\n=== Croatia ===\\n\\nOn 25 February, Croatia confirmed its first case, a 26-year-old man who had been in Milan, Italy. On 26 February, two new cases were confirmed, one being the twin brother of the first. In March 2020, a cluster of cases were reported in numerous Croatian cities. On 12 March, the first recovery was reported, and on 18 March the first death from the virus was confirmed. On 19 March, the number of recorded cases surpassed 100. On 21 March, it surpassed 200. On 25 March, it surpassed 400. On 31 March, it surpassed 800. The pandemic in Croatia occurred during the Croatian Presidency of the Council of the European Union.On 22 March, an intense earthquake hit Zagreb, the capital of Croatia, causing problems in enforcement of social distancing measures set out by the Government. The earthquake could also be felt across much of Croatia, Bosnia and Herzegovina, Hungary, Slovenia, and Austria.According to Oxford University, as of 24 March, Croatia was the country with the world\\'s strictest restrictions and measures for infection reduction in relation to the number of infected. The government set up a website for all information about the virus and a new phone line 113 that has volunteers answering questions.\\n\\n\\n=== Cyprus ===\\n\\nOn 9 March, Cyprus confirmed its first two cases, one in Nicosia and one in Limassol.Cases reached 500 in April, 1,000 in July, 4,000 in October and 10,000 in November.\\n\\n\\n=== Czech Republic ===\\n\\nThe first case was reported in the country on 1 March. As of 22 March there had been 1120 confirmed cases, with one lethal outcome.On 12 March, the Czech Republic declared a 30-day state of emergency and barred entry to non-residents from China, South Korea, Iran, Italy, Spain, Austria, Germany, Switzerland, Sweden, UK, Norway, Denmark and France.The Czech Republic banned people from going out in public without wearing a face mask or covering their nose and mouth.\\n\\n\\n=== Denmark ===\\n\\nOn 27 February, Denmark confirmed its first case.As of 16 March, there have been 898 confirmed cases in Denmark, including 11 in the Faroe Islands (see below).Numerous preventive measures gradually were implemented. Starting on 13 March, schools, universities and similar places were closed, while most people in non-essential functions have been sent home to work. On 14 March, the borders were closed for all entries, except Danish citizens, people with a residence permit, people with an important reason for visiting, and transport of goods.\\n\\n\\n=== Estonia ===\\n\\nOn 27 February, Estonia confirmed its first case, an Iranian citizen, travelling from Iran via Turkey.As of 11 March, there were 17 confirmed cases in Estonia. 12 of them had returned with the infection from Northern Italy, one from France, one from Iran and one from undisclosed risk area. First two cases of virus transmitting locally were in Saaremaa after international volleyball competition involving a team from Milan.From 12 March, the virus was spreading locally. The number of infected grew fast reaching 109 on 14 March.\\n\\n\\n=== Finland ===\\n\\nThe first case was reported in the country on 28 January. As of 21 July there were over 7,000 confirmed cases. As of 9 December (2020) the confirmed cases in Finland are 28,732 over the entire period. A total of 433 deaths associated with the disease have been reported until now.\\n\\n\\n=== France ===\\n\\nOn 24 January, the first case in Europe was confirmed in Bordeaux. Two more cases were confirmed in Paris by the end of the day, all of them originated from China. A cluster of infections was discovered in Haute-Savoie which originated from a British national who had visited Singapore.From 31 January to 9 February, nearly 550 people were repatriated from Wuhan on a series of evacuation flights arriving at Creil Air Base in Oise and Istres-Le Tubé Air Base in Istres.On 14 February, an 80-year-old Chinese tourist died in Bichat–Claude Bernard Hospital, Paris, marking the first death from COVID-19 in Europe and France.According to regional council president Jean Rottner, the starting point for the first intense wave in Alsace was the Fasting Meeting of the Protestant Free Church of La Porte Ouverte in Mulhouse, with more than 2500 visitors, in mid-February.\\nOn 12 March, French president Emmanuel Macron announced on public television that all schools and all universities would close from Monday 16 March until further notice. The next day, the Prime Minister Édouard Philippe banned gatherings of more than 100 people, not including public transportation. The following day, the prime minister ordered the closure of all nonessential public places, including restaurants, cafés, cinemas, and discothèques, effective at midnight.As of 14 March, there had been 4,499 confirmed cases (a near-four-fold increase over the number 5 days previously), and 91 deaths in France.As of 20 March, the number of confirmed cases had risen to 12,612, while the number of deaths reached 450.\\nAs of 30 March, more than six hundred doctors and other medical workers are suing the former Minister of Health and the Prime Minister for \"culpable negligence\" in failing to prepare for the epidemic.Until 1 April, the daily update of the number of deaths in France included only deaths in hospitals, with deaths in retirement homes or at home not included in the update. Deaths in retirement homes were included in the reported figures from 2 April, causing totals to increase substantially.On 4 May, retroactive testing of samples in one French hospital showed a patient \"who had COVID-19 as early as Dec. 27, nearly a month before the French government confirmed its first cases.\" According to the researchers who reported the discovery, this indicates that the virus was present in the population well before the first confirmed cases, a finding also echoed by US researchers. In July 2020, Prime Minister of France Jean Castex, announced that health care pay workers will see $9 billion in pay raises, as a result of their efforts during the corornavirus pandemic.\\n\\n\\n=== Germany ===\\n\\nIn Germany, the first case of COVID-19 was recorded in the state of Bavaria on 27 January 2020. Daily case numbers began to decrease after 8 April, but rose sharply again in October.\\nAs of 30 January 2021, 2,207,393 cases have been reported with 56,286 deaths and approximately 1,930,592 recoveries. The case fatality rate is 2.5%. The low preliminary fatality rate in Germany, compared to Italy and Spain, has resulted in a discussion and explanations that cite among others the country\\'s higher number of tests performed, absence of COVID-19 analyses in autopsies and higher proportion of positive cases among younger people. Hessian state finance minister Thomas Schäfer committed suicide on 28 March 2020. State Premier Volker Bouffier said that Schäfer had had considerable worries in particular over the economic burden that the COVID-19 pandemic would place on the population.\\n\\n\\n=== Gibraltar ===\\n\\nFirst case was identified on 4 March 2020, only 187 cases to the end of July then increases with 100 added in August and the same in September. A contact App  \"BEAT Covid Gibraltar\" was released on 18 June.\\n\\n\\n=== Greece ===\\n\\nOn 26 February, the first case in Greece was confirmed, a 38-year-old woman from Thessaloniki who had recently visited Northern Italy. The next day, the first patient\\'s 9-year-old child and another 40-year-old woman, who had travelled to Italy, also tested positive. Subsequent cases is Greece were mainly related to people who had travelled to Italy and a group of pilgrims who had travelled to Israel and Egypt, as well as their contacts. Health and state authorities issued precautionary guidelines and recommendations, while measures up to that point were taken locally and included the closure of schools and the suspension of cultural events in the affected areas (particularly Ilia, Achaea and Zakynthos). The first confirmed death from COVID-19 in Greece was a 66-year-old man, who died on 12 March.By 27 July there were 4,227 confirmed cases in Greece, some of which detected among tourists arriving to the country from mid-June onwards, 202 deaths and 3,562 recoveries. The Greek National Public Health Organization (NPHO), in collaboration with local authorities and doctors, is tracking and testing everyone who came in close contact with the patients. By 10 March, with 89 confirmed cases and no deaths in the country, the government decided to suspend the operation of educational institutions of all levels nationwide and then, on 13 March, to close down all cafes, bars, museums, shopping centres, sports facilities and restaurants in the country. On 16 March, all retail shops were also closed and all services in all areas of religious worship of any religion or dogma were suspended. On 18 and 19 March, the government announced a series of measures of more than 10 billion euros to support the economy, businesses and employees.On 22 March the Greek authorities announced significant restrictions on all nonessential transport and movement across the country, starting from 6 a.m. on 23 March. Starting from 4 May, Greece has begun easing its lockdown restrictions after a 42-day lockdown, with the gradual lifting of movement restrictions and restart of business activity.\\n\\n\\n=== Guernsey ===\\n\\nOn 9 March, the first case in the Crown dependency was confirmed. On 27 May it was announced that there were no known active cases in the Bailiwick. There were no active cases for 129 days before an arrival from the UK tested positive on 6 September.\\n\\n\\n=== Hungary ===\\n\\nOn 4 March Prime Minister Viktor Orbán announced that two Iranian students had been infected with the virus. The students are asymptomatic and have been transported to Saint Ladislaus Hospital in Budapest.\\n\\n\\n=== Iceland ===\\n\\nThe first case was confirmed in Iceland on 28 February, an Icelandic male in his forties who had been on a ski trip to Andalo in Northern Italy and returned home on 22 February.Iceland\\'s Department of Civil Protection and Emergency Management declared a state of emergency on 6 March after two cases of community transmission in Iceland were confirmed, bringing the total number of confirmed cases to 43. On 13 March, it was announced at an official press conference that a four-week ban on public gatherings of more than 100 persons would be put into effect as of Monday 16 March. Universities and secondary schools will be closed for four weeks. International airports and harbours will remain exempt from these measures.\\n\\n\\n=== Ireland ===\\n\\nThe National Public Health Emergency Team of Ireland announced the first case in the Republic of Ireland on 29 February, involving a male student from the east of the country, who had arrived there from Northern Italy. On 3 March, a second case was confirmed of a female in the east of the country who had been to Northern Italy. Response to the outbreak has included cancellation of St Patrick\\'s Day parades and all festivals. On 11 March, an elderly patient in Naas General Hospital in County Kildare became Ireland\\'s first fatality from the virus. On 12 March, Taoiseach Leo Varadkar announced the closure of all schools, colleges and childcare facilities. On 20 March, an emergency legislation was signed into law by President Michael D. Higgins giving the state permission to detain people, restrict travel and keep people in their homes to help combat the spread of the pandemic.\\n\\n\\n==== Lockdown ====\\n\\nOn 12 March, Taoiseach Leo Varadkar announced the closure of all schools, colleges and childcare facilities in Ireland until the end of August. On 27 March, Varadkar announced a national stay-at-home order for at least two weeks; the public were ordered to stay at home in all circumstances. All non-essential shops and services, including all pubs, bars, hotels and nightclubs closed and all public and private gatherings of any number of people was banned. The Garda Síochána (Irish police) were given power to enforce the measures, which were repeatedly extended until 18 May.A roadmap to easing restrictions in Ireland that included five stages was adopted by the government on 1 May 2020 and subsequently published online. The fourth and final phase of easing COVID-19 restrictions in Ireland was initially scheduled to take place on 20 July, but was repeatedly postponed until 31 August at the earliest.On 15 September, the Government announced a medium-term plan for living with COVID-19 that included five levels of restrictions.All non-essential businesses and services closed and all public and private gatherings of any number of people was banned again on 21 October following the Government\\'s announcement to move the entire country to Level 5 lockdown restrictions for six weeks until 1 December. On 27 November, the Government agreed to ease restrictions from 1 December.A third wave of COVID-19 arrived in Ireland on 21 December. The Government acted swiftly and on 22 December, Level 5 lockdown restrictions with a number of adjustments were announced, which came into effect from Christmas Eve until 12 January 2021 at the earliest.All non-essential businesses and services closed and all public and private gatherings of any number of people was banned again on 31 December (New Year\\'s Eve) following the Government\\'s announcement to move the entire country to full Level 5 lockdown restrictions from 30 December until 31 January 2021 at the earliest, in an attempt to get a third surge in cases of COVID-19 under control. On 26 January, the Government extended the Level 5 lockdown restrictions until 5 March. On 23 February, the Government extended the Level 5 lockdown restrictions for another six weeks until 5 April (Easter Monday) at the earliest, while its new revised Living with COVID-19 plan was published.On 30 March, the Government announced a phased easing of restrictions from Monday 12 April. On 29 April, the Government announced a reopening plan for the country throughout May and June from 10 May, with a further reopening planned announced on 31 August that would see all remaining COVID-19 restrictions in Ireland eased by 22 October.After a fourth wave of COVID-19 arrived in Ireland in October, the Government published on 19 October a revised plan for the easing of restrictions, with nightclubs allowed to reopen, however the continued use of masks, vaccine certificates and social distancing measures would remain in place until at least February 2022. On 3 December, the Government reintroduced a series of measures that would commence from 7 December amid concerns of the Omicron variant, with nightclubs to close, indoor cultural and sporting events to operate at 50% capacity and a maximum of four households allowed to meet indoors.The Omicron variant caused a fifth wave of COVID-19 to arrive in late December and early January 2022, with record levels of cases reported over the Christmas and New Year period. As cases began to fall sharply, Taoiseach Micheál Martin announced on 21 January the easing of almost all COVID-19 restrictions, with the requirements of vaccine certificates and social distancing to end, restrictions on household visits and capacity limits for indoor and outdoor events to end, nightclubs to reopen and pubs and restaurants to resume normal trading times, while rules on isolation and the wearing of masks would remain. Remaining COVID-19 restrictions were agreed to be removed from 28 February, with mask wearing in schools, indoor retail settings and on public transport to be voluntary, restrictions in schools to end and testing to be scaled back.As of 21 February 2022, the Department of Health have confirmed 1,276,778 confirmed cases and 6,443 deaths.\\n\\n\\n=== Italy ===\\n\\nOn 31 January, the first two cases were confirmed in Rome. Two Chinese tourists, who arrived in Milan on 23 January via Milan Malpensa Airport and travelled to Rome on a tourist bus, tested positive for and were hospitalised in Lazzaro Spallanzani National Institute for Infectious Diseases.On 6 February, one of the Italians repatriated from Wuhan, China, tested positive, bringing the total number of cases in Italy to three. On 22 February, the repatriated Italian recovered and was discharged from the hospital. On 22 and 26 February, the two Chinese tourists hospitalized in Rome tested negative.On 21 February, a cluster of cases was detected starting with 16 confirmed cases in Lombardy, with additional 60 cases on 22 February, and Italy\\'s first deaths reported on the same day. As of late February, Italy was hit harder than anywhere else in the EU by the COVID-19 pandemic.By 17 March 2020, there were 2,503 deaths and 31,506 confirmed cases. By 15 May 2020, there had been 223,885 cases and 31,610 deaths, according to Protezione Civile bulletins.On 3 June, Italy reopened its borders to EU residents and ended travel restrictions.On 13 July, the Head of the ICU at a Bergamo hospital said that former patients treated for COVID-19 had developed serious long-term health issues.Cases reached 100,000 in March, 200,000 in April, 300,000 in September, 600,000 in October and 1,600,000 in November.\\n\\n\\n==== Lockdown ====\\n\\nOn 8 March, Italian prime minister Giuseppe Conte signed a decree enacting a quarantine for the entire region of Lombardy – home to more than 10 million people and the financial capital, Milan – and multiple other provinces, totalling around 16 million residents. The lockdown decree included the power to impose fines on anyone caught entering or leaving Lombardy, the worst-affected region, until 3 April.On 9 March, the lockdown orders were extended to the whole of Italy, effectively quarantining more than 60 million people.On 20 November 2020, according to high-frequency data, the lockdowns in France and Italy weighed down on public mobility more than in any other European country. France\\'s official statistic agency INSEE found that the data collected by Google on the amount of time people spend at home is particularly closely correlated with the percentage of slow down experienced by an economy during the lockdown.\\n\\n\\n=== Jersey ===\\n\\nThe first case was recognised on 10 March which resulted within days in flights and ferries being cancelled except for essential travel and freight, thus isolating the island by the end of March. By 30 June there were no active cases although there have since been small numbers of cases brought into the island when lockdown was relaxed and travel permitted. On 14 October a contact App was launched. Community contracted cases caused numbers to rise from 600 to 1,000 in November resulting in another lockdown.\\n\\n\\n=== Kosovo ===\\n\\nThe first two cases in Kosovo were reported on 13 March. In the ten-day period 13–23 March, the number of positive cases with coronavirus rose to 35. In midnight of 23 March 26 new cases were confirmed, bringing the total number of cases to 61. On 25 March, the government collapsed with a vote of no confidence in Prime Minister Albin Kurti.\\n\\n\\n=== Latvia ===\\n\\n\\n=== Liechtenstein ===\\n\\nThe first case was reported in the country on 3 March.\\n\\n\\n=== Lithuania ===\\n\\nThe first case was reported in the country on 28 February. By 17 March there were 21 cases, mostly in Vilnius, Kaunas and Klaipėda. The first infected Lithuanian recovered on 14 March. Cases reached 2,000 in July, 10,000 in October and 50,000 in November.\\n\\n\\n=== Luxembourg ===\\n\\nThe first case was reported in the country on 29 February. Positive cases exceeded 6,600 on 30 June and 8,500 on 30 September.\\n\\n\\n=== Malta ===\\n\\nOn 7 March, Malta reported its first 3 cases of coronavirus: an Italian family consisting of a 12-year-old girl and her parents, who arrived in Malta on 3 March from Rome after a holiday in Trentino. Until around September, there were barely any cases; with that maximum cases per day at around 30. But then a big wave hit Malta and cases were being of over 100 everyday, and the most cases found in a single day was 510. Then in May 2021, the cases started calming down with Malta as of 20 June 2021 having just under 50 active cases, with over 30 thousand total.\\n\\n\\n=== Moldova ===\\n\\n\\n=== Monaco ===\\n\\nThe first case was reported in the country on 29 February.\\n\\n\\n=== Montenegro ===\\n\\n\\n=== Netherlands ===\\n\\nOn 27 February, the Netherlands confirmed its first case, a man who had been in the Lombardy region of Italy.On 6 March, RIVM announced the first death.On 9 March, RIVM announced 56 more confirmed cases in addition to the total of the 77 infected patients declared in the previous day, bringing the total to 321 infected and 3 dead.On 15 March, the Dutch government announced in a press conference that all schools, day care facilities, colleges, universities, and universities of applied sciences would close until 6 April, though children of parents in vital industries could still go to school or the day care if they could not be taken care of otherwise.As of 16 March, the total number of confirmed cases of COVID-19 had risen to 1,413 and the total number of confirmed deaths had risen to 24. This was an increase of 278 infections and 4 deaths in comparison to 15 March.On the same date, the Prime Minister of the Netherlands, Mark Rutte, had announced that the country would not go into complete lockdown. Instead the situation would be controlled as much as possible by delaying the spread of the virus, relying on measures taken earlier by the government, such as social distancing and prohibiting gatherings of 100 people and over. It is expected the coronavirus will keep spreading and a large part of the population will become infected. Instead of opting to lock down the Netherlands for the time required, which would have many (negative) consequences while the benefits remain uncertain and would have to stay in place, it would be attempted to build herd immunity in a controlled manner. Depending on how the virus behaves, the government would decide if additional measures are required.Further measures were introduced on 23 March. All events would be banned until 1 September. Gatherings of more than three people, except for families, were to be prohibited. City mayors received greater authorization to enforce the rules. Fines woul be issued to those not complying with the new rules.Cases reached 10,000 in March 2020, 50,000 in June 2020, 100,000 in September 2020, 500,000 in November 2020 and 1,000,000 in February 2021.\\n\\n\\n=== North Macedonia ===\\n\\nOn 26 February, North Macedonia confirmed its first case, a woman that had returned from Italy. On 10 March every school, kindergarten and university was closed. As of 5 April 2021, the country has reported 134,867 cases, 3,940 deaths and a total of 109,297 recovered.To stop the spreading of the virus, the country had a strict 61-hour curfew every weekend. For holidays, like Easter, the country had an 85-hour curfew. Furthermore, the state of emergency which had expired was extended for 14 more days until 30 May, and then again for 14 more days until 13 June.\\n\\n\\n=== Norway ===\\n\\nOn 26 February, Norway confirmed its first case, in a person who had returned from China the previous week.On 7 March, there were 147 confirmed cases in Norway. Most of the cases could either be linked to outbreaks abroad or close encounters with these. Altogether 89 have been infected on travel in Italy.On 12 March 2020, all kindergartens, schools, colleges and universities were closed until at least 26 March 2020.\\n\\n\\n=== Poland ===\\n\\n\\n=== Portugal ===\\n\\nOn 2 March, the first two cases were confirmed in Portugal, both in the city of Porto. One was a doctor who had returned from holiday in northern Italy, and the other a worker from Spain.On 18 March, the President of the Republic, Marcelo Rebelo de Sousa, declared the entirety of the Portuguese territory in a state of emergency for the following fifteen days, with the possibility of renewal, the first since the Carnation Revolution in 1974.On 24 March, the Portuguese government admitted that they could not contain the COVID-19 pandemic anymore, as it is wide spreading and will enter the \\'Mitigation Phase\\' on 26 March.Cases reached 10,000 in April, 100,000 in October and 300,000 in December.\\n\\n\\n=== Romania ===\\n\\n\\n=== Russia ===\\n\\nRussia implemented early preventive measures to curb the spread of COVID-19 in the country by imposing quarantines, carrying raids on potential virus carriers and using modern facial recognition technologies to impose quarantine measures.On 2 March, Western Russia confirmed its first case in Moscow Oblast. Previously on 31 January Russia confirmed first two cases in its Asian part, one in Tyumen Oblast and another in Zabaykalsky Krai. Both were Chinese nationals, who have since recovered.On 7 March, four new cases were confirmed, three was in Lipetsk and one in Saint Petersburg. All people visited Italy in the previous two weeks.On 8 March, three news cases were confirmed, in Moscow, Belgorod and Kaliningrad Oblasts. All people returned from Italy.On 10 March, Moscow Mayor Sergey Sobyanin signed a decree for banning mass events in Moscow with more than 5000 participants from 10 March to 10 April.By 15 April, cases were confirmed in all of Western Russia\\'s federal subjects with the confirmation of the first case in Nenets Autonomous Okrug.\\n\\n\\n=== San Marino ===\\n\\nOn 27 February, San Marino confirmed its first case, an 88-year-old man with pre-existing medical conditions, who came from Italy. He was hospitalised at a hospital in Rimini, Italy. As of 25 March 2020: With 700 confirmed cases out of a population of 33,344 (as of 2018), it is the country with the highest percentage of confirmed cases per capita at 2.1% – 1 confirmed case per 48 inhabitants. Also, with 42 confirmed deaths, the country has the highest rate of confirmed deaths per capita at 0.126% of the total population – 1 death per 794 inhabitants.\\n\\n\\n=== Serbia ===\\n\\nOn 29 February, a massive infection occurred in the town of Valjevo after a private party involving a guest from Austria who had previously stayed in Italy. On 6 March, the first case was confirmed in Serbia by Minister of Health Zlatibor Lončar, of a man who had traveled to Budapest. President Aleksandar Vucic and the Government of the Republic of Serbia introduced a state of emergency on 15 March. Two days later, curfew and quarantine were introduced as safeguards against massive infection.\\n\\n\\n=== Slovakia ===\\n\\nOn 6 March, Slovakia confirmed its first case, a 52-year-old man from a small village near Bratislava. He had not travelled anywhere in recent weeks but his son, who didn\\'t show any symptoms, had visited Venice. On 7 March, the virus was also confirmed in his wife and son.In October 2020, Slovakia started implementation of a short-period mass-testing programme to test two-thirds of its citizens for COVID-19.\\n\\n\\n=== Slovenia ===\\n\\nOn 4 March Slovenia confirmed its first case. A patient of about 60 years of age had returned from Morocco few days earlier (via Italy) and was admitted to a hospital in Ljubljana.On 6 March, a total of eight cases were reported, three of them medical professionals, who contracted the virus on holiday in Italy. By 9 March 16 people were confirmed to be infected.\\n\\n\\n=== Spain ===\\n\\nOn 31 January, Spain confirmed its first case, in the Canary Island of La Gomera. A tourist from Germany tested positive and was admitted to University Hospital of the Nuestra Señora de Candelaria. On 19 February, 2,500 football fans from Valencia attended a Champions League game in Bergamo, the hot spot of the outbreak in Italy.On 24 February, a medical doctor from Lombardy, Italy who was vacationing in Tenerife, tested positive at the University Hospital of the Nuestra Señora de Candelaria in Spain. Afterwards, multiple cases were detected in Tenerife involving people who had come in contact with the doctor. Other cases involving individuals who visited Italy were also discovered on Spanish mainland.On Saturday 14 March, the Spanish government imposed a nationwide quarantine, banned all trips that are not necessary and announced that companies may be intervened to guarantee supplies. However, with universities and schools closed earlier that week, bars and parks were full, and due to slow enactment \"part of the population of Madrid and other cities had dispersed across the country\". As of 17 March 2020, there had been 11,826 confirmed cases with 1,028 recoveries and 533 deaths in Spain.On 28 March, the Spanish government tightened up its national lockdown, ordering all non-essential workers to stay at home for the next two weeks. Nearly 900,000 workers lost their jobs in Spain since it went into lockdown in mid-March 2020. Public transport has also been greatly affected by the lockdowns and the severe restrictions established by the government. The relationship between the users who have continued using the urban bus and the characteristics of the stop\\'s surroundings have been analysed.On 10 January 2022, Spanish Prime Minister Pedro Sánchez initiated a debate in the European Union to treat COVID-19 as an “endemic illness” rather than a pandemic. He suggested a move towards a flu-like monitoring system amid falling death rates and hospitalisations on the European continent.\\n\\n\\n=== Sweden ===\\n\\nOn 31 January, Sweden confirmed its first case. A woman in her 20s, who had visited Wuhan, tested positive and was admitted at Ryhov County Hospital in Jönköping.On 26 February, following the outbreak in Italy and in Iran, infection clusters originating from these two countries appeared in Sweden. A number of individuals in Västra Götaland, Jönköping, Stockholm and Uppsala tested positive and were admitted to the infectious disease units in the respective counties.The country\\'s first fatality came on 11 March, that of a man in Stockholm over 60 who had other illnesses prior to infection.\\nAs of 12 March, national testing strategy shifted to only the elderly, the severely ill, and healthcare personnel. The official recommendation for symptoms that were not serious, as of 13 March, was to stay at home and not visit healthcare. This has led to statistics becoming less useful.As of 14 March 924 people were reported as having become infected. The Ministry for Foreign Affairs of Sweden made the recommendation to refrain from unnecessary travel to all countries.On 22 January 2021 the Ministry of Justice of Sweden implemented travel bans from Denmark and the United Kingdom due to the mutation of SARS-CoV-2.\\n\\n\\n=== Switzerland ===\\n\\nOn 25 February, following the outbreak in Italy, Switzerland confirmed its first case, a 70-year-old man in the Italian-speaking canton of Ticino, who had previously visited Milan. Afterwards, multiple cases related to the Italy clusters were discovered in multiple cantons including Basel-City, Zürich and Graubünden. Multiple isolated cases not related to the Italian clusters were also confirmed.On 28 February, the Federal Council announced an immediate ban on all events with more than 1,000 participants.As of 10 March, there were 500 confirmed cases in Switzerland.\\nOn 16 March 2020, the Federal Council announced further measures, and a revised ordinance. Measures include the closure of bars, shops and other gathering places until 19 April, but leaves open certain essentials, such as grocery stores, pharmacies, (a reduced) public transport and the postal service. Those measures were prolonged until 26 April 2020.On 23 December, following an unexpectedly quick authorization by Swissmedic of the Pfizer–BioNTech COVID-19 vaccine, the vaccination campaign was launched in several cantons. A 90-year-old woman from the Lucerne region became the first vaccinated patient in continental Europe outside Russia. All cantons are expected to start vaccinating by 11 January 2021.\\n\\n\\n=== Turkey ===\\n\\n\\n=== Ukraine ===\\n\\n\\n=== United Kingdom ===\\n\\nOn 31 January, England confirmed the first two cases, both members of a family of Chinese nationals staying in a hotel in York who were taken to specialist facilities in Newcastle upon Tyne. Afterwards, several confirmed cases were detected across the UK.\\nThe UK government implemented preventive measures to curb the spread of infections which included contact tracing, isolation and testing, some of which were related to the Italy clusters. The NHS set up drive-through screening centres at several hospitals to test members of the public showing symptoms. This effort was later replaced with screening focused on diagnosing patients in secondary care.On 2 March, Ministers approved the Department of Health and Social Care COVID-19 action plan, which sets out actions to date, future measures, cooperation between devolved political and health authorities, and the level of preparedness of the country\\'s four National Health Services. It outlined the government\\'s objectives to deploy phased actions to Contain, Delay, and Mitigate any outbreak, using Research to inform policy development.On 23 March, it was announced that the UK would be entering a nationwide lockdown. The public were advised to stay at home except for essential shopping and one hour of exercise a day. These measures came into effect on 26 March and lasted until 1 June when measures were eased allowing early years pupils and exams students to return to school in a limited capacity. These measures are further eased on 15 June when retail and attractions were able to open for the first time since March.On 25 March, the UK Parliament legislated to provide the government and authorities with emergency powers to handle the COVID-19 pandemic, such as the power to restrict public gatherings, order businesses to close, and the ability to detain those suspected of having the virus. The Coronavirus Act 2020 received Royal Assent on 25 March and came into force on the same day.On 5 April, the Prime Minister of the United Kingdom Boris Johnson was admitted to hospital after testing positive to COVID-19 10 days earlier. He was admitted into intensive care the following night, when his symptoms further worsened. He left hospital on 12 April.The NHS was given access to emergency supplies of PPE and patients that did not need to remain in hospital were moved to residential care homes if needed. However, there was no need for a negative COVID-19 test before patients were transferred to these settings, this was only required from 15 April. On 15 May, the government approved the first social care specific legislation that helped to ensure that staff had access to adequate PPE and protect both those working and living in care homes.As of 6 May, the total of confirmed cases was 201,101; the total of recorded deaths in all settings was 30,076, the highest in Europe and second highest in all the countries after the United States. It is estimated that care home residents accounted for 30–40% of these deaths, with social care workers being twice as likely to die from COVID-19 as the general population. However, the death toll did not continue to increase throughout the summer, and Brazil and Mexico now have more deaths as of 1 August 2020.During August and September, the number of infections per day began to increase significantly although the death rate remained fairly low. Boris Johnson announced changes to restrictions on 22 September which included forcing pubs, bars and restaurants to close by 10 pm in England and the so-called rule of six saying no more than six people should meet. In October, Scotland\\'s first minister declared new restrictions it meant that some areas of the country those like Edinburgh had to close bars, pubs and go back to doing self-service. Northern Ireland\\'s executive that also have control over health policy tightened restrictions to make them close to a lockdown as did Wales announced by Welsh first minister. It also emerged that Boris Johnson had been advised by SAGE committee of scientists to have a short lockdown back on 21 September. Boris subsequently introduced a three-tier system of restrictions Liverpool was put under the top tier meaning tougher restrictions but falling short of a full lockdown. The UK has since recorded over 40,000 deaths.\\nOn 5 November, the UK had to enter a second national lockdown due to the rising number of cases and hospitalisations. Following the end of this lockdown on 2 December case numbers began to rise again and over 70,000 deaths have been recorded in relation to COVID-19 as of 11 December 2020. On 4 January 2021, Prime Minister Boris Johnson addressed the nation announcing a third lockdown. The UK went into Lockdown that day due to rapidly increasing numbers of cases and deaths caused by a new more infectious variant of COVID-19 spreading around the UK.The UK has begun to vaccinate its population against COVID-19. The first vaccine, the Pfizer/BioNTech vaccine for COVID-19, was approved for use in the UK on 2 December 2020, the first of the 800,000 immediately available doses was administered on 8 December. The second vaccine approved for use in the UK was the Oxford University/AstraZeneca COVID-19 vaccine, which was approved for use on 30 December 2020, with the first dose of this vaccine given on 4 January 2021. A third vaccine, the Moderna vaccine, was approved for use in the UK on 8 January 2021, and the first dose was administered in Wales on 6 April.\\n\\n\\n=== Vatican City ===\\n\\n\\n== Pandemic development graphs ==\\n\\n\\n=== Confirmed cases and deaths by date ===\\nConfirmed cases by date\\n\\nDeaths by date\\n\\n\\n=== Total confirmed cases since first day of the outbreak ===\\n\\n\\n=== Growth rates comparison since outbreak ===\\nSyncing the start date to the 10th case, comparing the daily growth in cases\\n\\nSyncing the start date to the 10th case, comparing the daily deaths\\n\\nDaily new cases; syncing the start date to the 100th case:\\n\\nDaily new cases; syncing the start date to the 500th case:\\n\\n\\n== Response and criticism ==\\n\\n\\n=== Immigrants and refugees ===\\nThe European Union closed borders to non-nationals on 17 March. The next day, Greece imposed restrictions on refugees’ movement within camps. Thousands of asylum seekers are living in crowded camps, and there are fears that pandemic could not be controlled under such conditions. The Greek prime minister K. Mitsotakis said that Europe should do more to help because Greece \"cannot resolve this crisis instantly and alone\". Unnamed Greek officials have stated concerns that Turkey may send infected refugees and migrants towards the islands. Early in April Malta and Italy closed their ports to vessels carrying asylum seekers from North Africa.\\n\\n\\n=== Vaccination ===\\n\\nIn early 2021, EU commission chief Ursula von der Leyen was criticised for the EU\\'s slow rollout, and she noted that the EU was “late to authorise” and said that ordering vaccines on behalf of member states was “the right thing to do”.\\nSome countries withheld recommending the AstraZeneca vaccine due to concerns of blood clots.  By 20 June 2021, nearly half the population of the European Union had received at least one dose of COVID-19 vaccination.In November 2021, a study by the ECDC estimated that 470,000 lives over the age of 60 had been saved since the start of vaccination roll-out in the European region.\\n\\n\\n=== Herd immunity through infection in Iceland ===\\n\\nOn 23 February 2022, the Ministry of Health lifted all remaining COVID-19 restrictions, including gathering limits, restricted opening hours for bars, and border restrictions.  Adopting a herd immunity approach, the ministry stated that “widespread societal resistance to COVID-19 is the main route out of the epidemic,” and “to achieve this, as many people as possible need to be infected with the virus as the vaccines are not enough, even though they provide good protection against serious illness”.\\n\\n\\n=== Criticism ===\\n\\n\\n==== Travel bans and border closures ====\\n\\nAlthough by 7 March some European politicians such as France\\'s Marine Le Pen had called for Europe\\'s internal borders to be temporarily closed, the European Union by 13 March continued to reject the idea of suspending the Schengen free travel area and introducing border controls with Italy. The deputy leader of the Swiss Ticino League, Lorenzo Quadri, by 29 February had criticised the decision, saying, \"It is alarming that the dogma of wide-open borders is considered a priority.\" United States President Donald Trump said by 12 March the European Union had \"failed to take the same precautions and restrict travel from China and other hot spots\" as the US had implemented. Trump also said that \"As a result a large number of new clusters in the United States were seeded by travellers from Europe.\" Research on coronavirus genomes indicates the majority of COVID-19 cases in New York came from European travelers, rather than directly from China or any other Asian country.By 9 March, Czech Prime Minister Andrej Babiš stated that \"European countries cannot ban the entry of Italian citizens within the Schengen area. The only possible way is to have the Italian prime minister call on his fellow citizens to refrain from traveling to other countries of the European Union.\"After Slovakia, Denmark, the Czech Republic and Poland announced complete closure of their national borders, the European Commission President Ursula von der Leyen said by 12 March that \"Certain controls may be justified, but general travel bans are not seen as being the most effective by the World Health Organization. Moreover, they have a strong social and economic impact, they disrupt people’s lives and business across the borders.\" European Union leaders condemned the US decision to restrict travel from Europe to the United States. European Council President Charles Michel and Ursula von der Leyen said in a joint statement: \"The European Union disapproves of the fact that the US decision to impose a travel ban was taken unilaterally and without consultation.\" Ursula von der Leyen admitted by 17 March that \"all of us who are not experts initially underestimated the coronavirus.\"As of 22 February 2021, the UK has banned direct flights from 33 countries, including Portugal, South Africa, Peru and the United Arab Emirates. All travelers entering the UK via indirect flights have been ordered to quarantine in a designated hotel for 10 days. Since passengers coming from high-risk countries with South African virus’ mutant (like the UAE and South Africa) are mixing with other travellers before reaching their accommodation, this travel ban is being criticised extensively.\\n\\n\\n==== EU solidarity ====\\n\\nThe Italian government has criticised EU\\'s lack of solidarity with Italy. Politico reported on 7 March that \"EU countries have so far refused Italy\\'s plea for help fighting coronavirus, as national capitals worry that they may need to stockpile face masks and other medical gear to help their own citizens, officials and diplomats said.\" Maurizio Massari, Italy\\'s ambassador to the EU, said that \"Only China responded bilaterally. Certainly, this is not a good sign of European solidarity.\" Serbian President Aleksandar Vučić said that \"European solidarity does not exist. That was a fairy tale.\"Eventually, in July 2020 the European Council agreed to a massive recovery fund of 750 billion € branded Next Generation EU (NGEU) to support member states hit by the COVID-19 pandemic. The NGEU fund goes over the years 2021 – 2023 and will be tied to the 2021–2027 budget of the EU (MFF). The comprehensive packages of NGEU and MFF will reach the size of 1824.3 Billion €.In response to the pandemic, the European Investment Bank Group is establishing a €25 billion Pan-European Guarantee fund. It is projected that the guarantee fund would raise up to €200 billion.\\n\\n\\n==== Reaction time of Spain ====\\n\\nAccording to The Guardian, Spain\\'s initially slow response to COVID-19 caused the epidemic to become severe even though it did not share a land border with Italy or other severely affected countries. An analysis in Vox hypothesised that the minority government did not want to risk its hold on power by banning large gatherings early; Prime Minister Pedro Sánchez initially defended his decision to allow large gatherings to continue.\\n\\n\\n==== Military exercises during pandemic ====\\nThe planned NATO \"Defender 2020\" military exercise in Germany, Poland, and the Baltic states, the largest NATO war manoeuvres since the end of the Cold War, was to be held on a reduced scale because of the COVID-19 pandemic. The Campaign for Nuclear Disarmament\\'s general secretary Kate Hudson criticised the Defender 2020 exercise: \"In the current public-health crisis, it jeopardises the lives not only of the troops from the US and the many European countries participating but the inhabitants of the countries in which they are operating.\"\\n\\n\\n==== Restrictions on civil liberties ====\\nThere was concern that measures taken by some national governments on occasion of the COVID-19 pandemic would have the aim or effect of restricting democracy and civil liberties and rights. In Hungary, prime minister Viktor Orban acquired near absolute powers through such legislation on 3 April.\\n\\n\\n==== Use of scientific advice ====\\nThe European Union\\'s Chief Scientific Advisors issued a statement on 24 June 2020, providing guidance for how scientific advice should be given and interpreted during the pandemic. One key point made by the Advisors was that scientists must be clearer about the degree of uncertainty that characterises the evolving evidence on which their advice is based, for instance around the use of face-masks. They also emphasised that scientific advice must be separated from decision-making, and this separation must be made clear by politicians.In April 2021, the leaders of the Society for Aerosol Research warned the debate on COVID-19 measures does not reflect current scientific knowledge. They said protection against infection must take place above all where people spend time indoors, because \"the transmission of the SARS-CoV-2 viruses takes place almost without exception indoors.\"\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== External links ==\\nWHO Coronavirus Disease (COVID-19)\\nWorldwide Coronavirus Map, confirmed Cases – Map the route paths of coronavirus confirmed cases.\\nncov – CSSE – Map of the expansion of the virus in Europe and in the world',\n",
              "  0.3402103494512287),\n",
              " ('The first cases of the COVID-19 pandemic of coronavirus disease 2019 in North America were reported in the United States on 23 January 2020. Cases were reported in all North American countries after Saint Kitts and Nevis confirmed a case on 25 March, and in all North American territories after Bonaire confirmed a case on 16 April.On 26 March 2020, the United States became the country in North America with the highest number of confirmed COVID-19 infections, at over 82,000 cases. On 11 April 2020, the United States became the country in North America with the highest official death toll for COVID-19, at over 20,000 deaths. As of 11 April 2022, there are nearly 97.1 million cases and about 1.4 million deaths in North America; nearly 93.2 million have recovered from COVID-19, meaning that about 24 out of 25 cases have recovered or that the recovery rate is about 96%.As of 11 April 2022, the United States has had the highest number of infections in North America, at nearly 82.1 million cases, as well as the highest death toll, at over a million deaths. There have been about 79.9 million recoveries in the United States as of 11 April 2022, meaning that nearly 39 out of 40 cases in the country have recovered or that the recovery rate is nearly 97.4%. On 20 March 2022, the number of COVID-19 deaths in the United States exceeded a million.\\nAs of 11 April 2022, Canada has reported nearly 3.6 million cases and about 38,000 deaths, while Mexico, which was overtaken in terms of the number of cases on 11 March 2022, the second anniversary of the day when the COVID-19 outbreak became a pandemic, by Japan, the second most affected country in East Asia, has reported about 5.7 million cases and about 320,000 deaths. The state in the United States with the highest number of cases and the highest death toll is California, at about 9.1 million cases and nearly 90,000 deaths as of 11 April 2022.\\n\\n\\n== Statistics by country and territory ==\\n \\n\\n\\n== Timeline by country and territory ==\\nCounting of cases are subject to the number of tested people.\\n\\n\\n=== Antigua and Barbuda ===\\n\\nThe COVID-19 pandemic was confirmed to have reached Antigua and Barbuda on 13 March 2020.As of 24 May 2021, Antigua and Barbuda reported 1,258 total cases, 16 active cases, and 42 deaths.\\n\\n\\n=== The Bahamas ===\\n\\nThe COVID-19 pandemic was confirmed to have reached the Bahamas on 15 March 2020 with the announcement of the first case.As of 24 May 2021, the Bahamas reported 11,499 total cases, 1,002 active cases, and 225 deaths. This comes out to 493 deaths per one million population.\\n\\n\\n=== Barbados ===\\n\\nBarbados announced its first two confirmed cases on 17 March and declared a public health emergency on 26 March.As of 26 January 2021, Barbados reported 1,401 total cases, 649 active cases, and ten deaths. Between January and March the number of infections went up eight times.\\n\\n\\n=== Belize ===\\n\\nThe COVID-19 pandemic was confirmed to have reached Belize on 23 March 2020.As of 13 January 2021, Belize reported 11,366 total cases, 587 active cases, and 277 deaths. This comes out to 690 deaths per one million population.\\n\\n\\n=== British Overseas Territories ===\\n\\n\\n==== Anguilla ====\\n\\nThe first two cases of the virus were confirmed on 26 March. On 26 April 2020, all patients recovered.As of 13 January 2021, Anguilla reported 15 total cases, 13 recoveries, and two deaths.\\n\\n\\n==== Bermuda ====\\n\\nThe COVID-19 pandemic was confirmed to have reached the British Overseas Territory of Bermuda on 18 March 2020.\\n\\n\\n==== Cayman Islands ====\\n\\nThe COVID-19 pandemic was confirmed to have reached the British Overseas Territory of the Cayman Islands on 12 March 2020.On 17 December 2020, Mercer University pre-med student Skylar Mack, 18, from United States, and her boyfriend Vanjae Ramgeet, 24, of the Cayman Islands, were sentenced to four months in prison for her violating the island\\'s mandatory two-week quarantine two days after arriving from the United States, and for his aiding and abetting her. The sentence was later reduced to two months.As of 13 January 2021, the Cayman Islands reported 362 total cases, 35 active cases, and two deaths.\\n\\n\\n==== Montserrat ====\\n\\nThe territory\\'s first case was confirmed on 17 March. Schools have been closed and public gatherings banned as a precautionary measure. By 15 May, all cases had fully recovered. On 10 July, a 12th case was discovered.As of 13 January 2021, Montserrat reported 13 total cases, 12 recoveries, and one death.\\n\\n\\n==== Turks and Caicos Islands ====\\n\\nThe COVID-19 pandemic was confirmed to have reached the British Overseas Territory of the Turks and Caicos Islands on 23 March 2020. On 12 May, the last two cases recovered. There are currently no active cases, because one person left the country. On 20 June, a new case was discovered.As of 13 January 2021, Turks and Caicos reported 1,011 total cases, 186 active cases, and six deaths.\\n\\n\\n==== British Virgin Islands ====\\n\\nOn 25 March, the first two cases in the country were confirmed.As of 13 January 2021, the British Virgin Islands reported 114 total cases, 18 active cases, and one death.\\n\\n\\n=== Canada ===\\n\\n\\n=== Colombia ===\\n\\n\\n==== San Andrés and Providencia ====\\n\\n\\n=== Costa Rica ===\\n\\nOn 6 March, the first case in Costa Rica was confirmed, which was also the first such case in Central America.As of 13 January 2021, Costa Rica reported 182,156 total cases, 39,805 active cases, and 2,384 deaths. This comes out to 466 deaths per one million population.\\n\\n\\n=== Cuba ===\\n\\nOn 11 March, the first cases in Cuba were confirmed. As of 12 May, new cases had fallen to less than 20 per day, and a program of mass testing was beginning.Infections went up fourfold in January and February 2021. Cuba\\'s death toll of 324 is well under the world average per capita, but health authorities worry that it is increasing. The government says two of its four vaccines should begin final trials in late March.\\n\\n\\n=== Greenland (Kingdom of Denmark) ===\\n\\nThe COVID-19 pandemic was confirmed to have spread to Greenland – an autonomous territory of the Kingdom of Denmark – in March 2020.\\n\\n\\n=== Dominica ===\\n\\nOn 22 March, the first case of COVID-19 was confirmed on the island of Dominica. It was a woman who recently came back from the UK.As of 13 January 2021, Dominica reported 109 total cases, eight active cases, but no deaths.\\n\\n\\n=== Dominican Republic ===\\n\\nOn 1 March, the first case in the Dominican Republic was confirmed, which was also the first case in the Caribbean.As of 13 January 2021, the Dominican Republic reported 186,383 total cases, 43,738 active cases, and 2,428 deaths. This comes out to 223 deaths per one million population.\\n\\n\\n=== Dutch Caribbean ===\\n\\n\\n==== Aruba ====\\n\\nOn 13 March 2020, Prime Minister Evelyn Wever-Croes announced the first two confirmed cases of coronavirus on the island.As a result, the country restricted entry of all individuals coming from Europe via air and seaports – starting on 15 March and in effect until 31 March – with the exception of those who are Aruban citizens. They also suspended public and private school classes for the week of 16 March, as well as all large-scale public gatherings. On 29 May, all cases recovered. On 29 June, two more cases had been discovered.As of 13 January 2021, Aruba reported 6,228 total cases, 624 active cases, and 52 deaths. This comes out to 486 deaths per one million population.\\n\\n\\n==== Curaçao ====\\n\\nThe 2020 coronavirus pandemic was documented for the first time in Curaçao on 13 March 2020. The case was a 68-year-old man who was on vacation from the Netherlands. By 9 July, all cases recovered. On 15 July, a new case was discovered. On 6 August, all cases resolved.As of 13 January 2021, Curaçao reported 4,488 total cases, 373 active cases, and 19 deaths. This comes out to 116\\t deaths per one million population.\\n\\n\\n==== Sint Maarten ====\\n\\nAs of 18 March there has been one confirmed case in Sint Maarten. Schools have been shut for a period of two weeks. By 15 June, all cases recovered. On 1 July, a new case was discovered, which resolved 3 July. On 15 July, a 79th case was discovered.As of 13 January 2021, Sint Martin reported 1,589 total cases, 102 active cases, and 27 deaths.\\n\\n\\n==== Caribbean Netherlands ====\\nAs of 13 January 2021, the Caribbean Netherlands reported 249 total cases, 65 active cases, and three deaths.\\n\\n\\n===== Bonaire =====\\n\\nOn 16 April 2020, Edison Rijna, Island Governor of Bonaire announced the first case of COVID-19 on the island.\\nThe island was already closed to international travel. On 28 April 2020, all cases had recovered. On 14 July, two new cases had been discovered.\\n\\n\\n===== Saba =====\\n\\nOn 12 April, the first case was confirmed in Saba. Schools, bars and \\'non-essential services\\' are all currently shut. On 12 May, all cases on Saba recovered. On 1 August, two new cases were discovered.\\n\\n\\n===== Sint Eustatius =====\\n\\nOn 31 March 2020, the first two cases were confirmed, they were two young men from the Netherlands who arrived on 15 March and self isolated after arrival. On 5 May all cases recovered.\\n\\n\\n=== El Salvador ===\\n\\nThe COVID-19 pandemic was confirmed to have reached El Salvador on 18 March 2020.As of 13 January 2021, El Salvador reported 49,539 total cases, 4,282 active cases, and 1,447 deaths. This comes out to 222 deaths per one million population.\\n\\n\\n=== Overseas France ===\\nTwo cases of the coronavirus were confirmed on 1 March in the French collectivity of Saint Martin, having travelled from France through Dutch Sint Maarten and French collectivity of Saint Barthélemy, where they infected their son who is a resident. They then returned to Sint Maarten and were detected at the airport and transferred to French Saint Martin hospital for isolation.\\nAccording to the French Ministry of Health situation reports, as of 6 March, there have been two confirmed cases in Martinique, two in Saint Martin and one in Saint Barthélemy. As of 5 April, St. Pierre and Miquelon recorded its first verified case.\\n\\n\\n==== Region of Guadeloupe ====\\n\\nThe COVID-19 pandemic was confirmed to have spread to the French overseas department and region of Guadeloupe on 12 March 2020.As of 13 January 2021, Guadeloupe reported 8,776 total cases, 6,380 active cases, and 154 deaths. This comes out to 385 deaths per one million population.\\n\\n\\n==== Region of Martinique ====\\n\\nThe COVID-19 pandemic was confirmed to have reached the French overseas department and region of Martinique on 5 March 2020.As of 13 January 2021, Martinique reported 6,184 total cases, 6,043 active cases, and 43 deaths. This comes out to 115 deaths per one million population.\\n\\n\\n==== Collectivity of Saint Barthélemy ====\\n\\nThe COVID-19 pandemic was confirmed to have reached the French overseas collectivity of Saint Barthélemy on 1 March 2020. The last positive case was on 31 March. On 21 April, the last case recovered. Between 18 and 24 July, a new case was imported.As of 13 January 2021, Saint Barthélemy reported 206 total cases, 33 active cases, and one death.\\n\\n\\n==== Collectivity of Saint Martin ====\\n\\nThe COVID-19 pandemic was confirmed to have reached the French overseas collectivity of Saint Martin on 1 March 2020.As of 13 January 2021, Saint Martin reported 1,025 total cases, 158 active cases, and 12 deaths.\\n\\n\\n==== Saint Pierre and Miquelon ====\\n\\nThe COVID-19 pandemic was confirmed to have reached the French overseas collectivity of Saint Pierre and Miquelon on 5 April 2020.As of 13 January 2021, Saint Pierre and Miquelon reported 16 total cases, all of whom had recovered.\\n\\n\\n=== Grenada ===\\n\\nOn 22 March, the first case in the country was confirmed.As of 13 January 2021, the British Virgin Islands reported 132 total cases, eight active cases, and one death.\\n\\n\\n=== Guatemala ===\\n\\nThe COVID-19 pandemic was confirmed to have reached Guatemala on 13 March 2020.As of 13 January 2021, Guatemala reported 145,986 total cases, 8,362 active cases, and 5,117 deaths. This comes out to 283 deaths per one million population.\\n\\n\\n=== Haiti ===\\n\\nOn 19 March, the first two cases in the country were confirmed.As of 13 January 2021, Haiti reported 10,569 total cases, 1,443 active cases, and 238 deaths. This comes out to 21 deaths per one million population.\\n\\n\\n=== Honduras ===\\n\\nOn 10 March, the first two cases in Honduras were confirmed.President Juan Orlando Hernandez and his wife tested positive for COVID-19 in June.As of 13 January 2021, Honduras reported 129,805 total cases, 67,122 active cases, and 3,294 deaths. This comes out to 330 deaths per one million population.78-year-old Cardinal Óscar Rodríguez Maradiaga was diagnosed with COVID-19 on 4 February 2021.28-year-old nursing student Keyla Patricia Martínez was arrested in La Esperanza, Intibucá Department and was murdered by choking while in police custody. Her death was originally called a suicide, but following protests it was reclassified as homicide.\\n\\n\\n=== Jamaica ===\\n\\nThe government announced a travel ban between China and Jamaica. All people entering Jamaica from China will be subject to immediate quarantine for at least 14 days, and anyone who was allowed to land and shows symptoms of the virus will be put in immediate isolation. In keeping with the new policy, 19 Chinese nationals who arrived at the Norman Manley International Airport on the evening of 31 January were denied entry, quarantined and put on a flight back to China on 1 February.On 10 March, the Ministry of Health and Wellness (MoHW) confirmed the first case in Jamaica, a female patient who arrived from the United Kingdom on 4 March. The health minister reported that she has been in isolation since 9 March after showing respiratory symptoms. Following the update, the travel ban imposed was expanded to include France, Germany, and Spain.On 11 March, the country\\'s health minister confirmed the second \"imported corona virus  [sic]\" case.As of 13 January 2021, Jamaica reported 13,852 total cases, 1,967 active cases, and 317 deaths. This comes out to 107 deaths per one million population.Confirmed cases doubled in the first two months of 2021, and the death toll stood at 422 on 1 March. All beds dedicated to COVID-19 isolation were full as of 26 February. Jamaica will receive a vaccine donation from India of 50,000 vaccine doses on 4 March, 124,800 doses via COVAX later in March, and 1.8 million from the African Medical Supply Platform in April.\\n\\n\\n=== Mexico ===\\n\\nOn 28 February 2020, Mexico confirmed its first three cases. The country\\'s first coronavirus-related death was reported on 18 March. Almost every state reported at least one case of infection. Mexico entered Phase 2 of 3, indicating community transmission, on 24 March. Mexico had 292 imported cases of infection, 70 cases linked to importation, and five cases that were unlinked to foreign contact.Mexico began vaccinating health workers in Mexico City and Coahuila on 24 December. Vaccinations were expanded to 879 hospitals in all 32 federal entities on 13 January 2021. As of this date Mexico reported 1,556,028 confirmed cases (13th highest in the world), 251,992 active cases, and 135,682 deaths. This is 1,046 deaths per one million inhabitants.\\n\\n\\n=== Nicaragua ===\\n\\nAn ongoing pandemic of coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) was shown to have spread to Nicaragua when the first case, a Nicaraguan citizen who had returned to the country from Panama, was confirmed on 18 March 2020.As of 13 January 2021, Nicaragua reported 6,152 total cases, 1,760 active cases, and 167 deaths. This comes out to 25 deaths per one million population.\\n\\n\\n=== Panama ===\\n\\nThe Panamanian government has enhanced its sanitary control and screening measures at all ports of entry, to prevent the spread of the virus, isolating and testing potential cases.On 9 March, the health ministry (MINSA) announced Panama\\'s first coronavirus case, a Panamanian woman in her 40s who had returned from Spain.On the following day, the MINSA announced seven more COVID-19 cases and one coronavirus-related death.As of 13 January 2021, Panama reported 288,408 total cases, 56,673 active cases, and 4,594 deaths. This comes out to 1,056 deaths per one million population.\\n\\n\\n=== Puerto Rico (United States) ===\\n\\nAs of 19 March the territory has had five confirmed cases. On 17 March governor Wanda Vázquez Garced announced a 24/7 lockdown, with people only allowed to leave their homes for food, gas or medicines.\\n\\n\\n=== Saint Kitts and Nevis ===\\n\\nOn 25 March, the first two cases in the country were confirmed. By 19 May, all cases recovered.As of 13 January 2021, Saint Pierre and Miquelon reported 34 total cases, three active cases, but no deaths.\\n\\n\\n=== Saint Lucia ===\\n\\nThe COVID-19 pandemic was confirmed to have reached Saint Lucia on 13 March 2020. On 22 April 2020, it was announced that all confirmed cases had recovered. On 28 April 2020, two new cases were discovered.As of 11 October 2021, Saint Lucia reported 12,017 total cases, 837 active cases, and 224 deaths. Infections increased ten times between January and March.\\n\\n\\n=== Saint Vincent and the Grenadines ===\\n\\nOn 11 March, Saint Vincent and the Grenadines confirmed its first case.St. Vincent and Grenadines registered its first COVID-19 death in 2021 and by 1 March had eight fatal victims.\\n\\n\\n=== Trinidad and Tobago ===\\n\\nOn 12 March, Trinidad and Tobago confirmed its first case of COVID-19. It was a 52-year-old man who had recently been to Switzerland. He was self-isolated before he began experiencing symptoms of COVID-19.As of 13 January 2021, Trinidad and Tobago reported 7,305 total cases, 285 active cases, and 129 deaths. This comes out to 92 deaths per one million population.\\n\\n\\n=== United States ===\\n\\nOn 20 January, the United States confirmed its first case, of a 35-year-old man who had returned on 15 January to the state of Washington after visiting family in Wuhan, China. The man sought clinical assistance on 19 January.On 27 February, the CDC reported a case in California which may be the first instance of community transmission in the US.On 29 February, officials of Washington State confirmed the first reported death from COVID-19 in the US.By 11 March, the U.S. had tested fewer than 10,000 people. By the end of the month, over 1,000,000 people had been tested. However, health experts stated that this level of testing was still inadequate.On 26 March, the United States surpassed China and Italy as the country with the most confirmed COVID-19 cases, with a figure above 82,000. The U.S. federal government\\'s health inspectors surveyed 323 hospitals in late March; reporting \"severe shortages\" of test supplies, \"widespread shortages\" of personal protective equipment (PPE), and other strained resources due to extended patient stays while awaiting test results.On 11 April, the United States surpassed Italy as the country with the most confirmed COVID-19 deaths, with a total of over 20,000. The U.S. also became the first to record 2,000 deaths in a single day. Wyoming became the 50th state to be issued a disaster declaration.By 20 April, the federal government stated it was conducting 150,000 tests per day, and claimed that this number would be enough to allow for schools and businesses to reopen. Health experts estimate that 500,000 to 1,000,000 tests per day would be needed to properly track the spread of the COVID-19, to avoid a new wave of infections.On 1 October it was announced first that Hope Hicks had tested positive, and a few hours later it was announced that both President Donald Trump and First Lady Melania had tested positive for COVID-19, who were all part of the broader White House COVID-19 outbreak. Both Trumps recovered from the disease.\\nOn 9 November, the U.S. surpassed 10 million COVID-19 cases, according to data from Johns Hopkins.On 11 December, the United States Food and Drug Administration (FDA) authorized for the emergency use of the Pfizer-BioNTech COVID-19 vaccine. Following the issuing, it was estimated that 20,000,000 could be vaccinated over the course of a few weeks.As of 17 February 2021, the U.S. has recorded more than 27 million cases of COVID-19; 483,000 have died.\\n\\n\\n==== Alaska ====\\n\\nThe COVID-19 pandemic was confirmed to have reached the U.S. state of Alaska on 12 March 2020.\\n\\n\\n==== California ====\\n\\n\\n==== Florida ====\\n\\n\\n==== New Jersey ====\\n\\n\\n==== New York ====\\n\\n\\n==== Ohio ====\\n\\nThe COVID-19 pandemic was confirmed to have reached the U.S. state of Ohio on 9 March 2020, when the state\\'s first cases were reported. The first death from COVID-19 in Ohio was reported on 19 March. Subsequently, records supported by further testing showed that undetected cases had existed in Ohio since early January, with the first confirmed death on 17 March. By 23 April, Ohio had 656 confirmed deaths; by 1 May, there were 1002 confirmed deaths. Accurate data was difficult to obtain due to limited test availability. By 12 December, a total of 553,461 cases had been reported leading to 31,803 hospitalizations and 7,477 deaths.\\n\\n\\n==== Texas ====\\n\\n\\n==== Washington State ====\\n\\nThe first confirmed case relating to the COVID-19 pandemic in the United States was announced by the state of Washington on 21 January 2020. Washington made the first announcement of a death from the disease in the U.S. on 29 February and later announced that two deaths there on 26 February were also due to COVID-19. Until mid-March, Washington had the highest absolute number of confirmed cases and the highest number per capita of any state in the country, until it was surpassed by New York state on 10 April 2020. Many of the deceased were residents of a nursing home in Kirkland, an Eastside suburb of Seattle in King County.\\n\\n\\n== See also ==\\n\\n\\n== References ==\\n\\n\\n== External links ==\\nWorldwide Coronavirus Map, confirmed Cases – Map the route paths of coronavirus confirmed cases.\\nncov – CSSE – Map of the expansion of the virus in Europe and in the world\\nCoronavirus Updates in North America Archived 28 November 2020 at the Wayback Machine – Daily updated totals of the virus in all countries on North America',\n",
              "  0.3019838416567531),\n",
              " ('The COVID-19 pandemic began in Asia in Wuhan, Hubei, China, and has spread widely through the continent. As of 14 April 2022, at least one case of COVID-19 had been reported in every country in Asia except North Korea and Turkmenistan.\\nThe Asian countries with the highest numbers of confirmed coronavirus cases are India, Turkey, Iran, South Korea, and Vietnam. Despite being the first area of the world hit by the outbreak, the early wide-scale response of some Asian states, particularly Bhutan, Singapore, Taiwan, and Vietnam has allowed them to fare comparatively well. China was criticised for initially minimising the severity of the outbreak, but its wide-scale response has largely contained the disease since March 2020.As of July 2021, the highest numbers of deaths are recorded in India, Indonesia, Iran, and Turkey, each with more than 90,000 deaths and more than 900,000 deaths combined.  However, the death toll in Iran and Indonesia are claimed to be much higher than the official figures. Per capita, the highest deaths have been disproportionally in several Western Asian states, with Georgia having the highest figure closely followed by Armenia, and Iran in third, whereas China had the lowest.\\n\\n\\n== Statistics by country and territory ==\\n\\n\\n== Timeline by country and territory ==\\n\\n\\n=== Afghanistan ===\\n\\nOn 23 February 2020, at least three citizens of Herat who had recently returned from Qom were suspected of COVID-19 infection. Blood samples were sent to Kabul for further testing. Afghanistan later closed its border with Iran.On 24 February, Afghanistan confirmed the first COVID-19 case involving one of the three people from Herat, a 35-year-old man who tested positive for SARS-CoV-2. On 7 March, three new cases were confirmed in Herat Province. On 10 March, the first case reported outside of Herat province, was in Samangan Province, bringing to the total to five cases.\\n\\n\\n=== Armenia ===\\n\\nArmenia confirmed the first case of coronavirus during the late night of 29 February/early morning of 1 March, announcing a 29-year-old Armenian citizen had returned from Iran and was confirmed positive for the virus. His wife was tested and results came in negative. Prime Minister Nikol Pashinyan declared that he is \"now in good condition.\" About 30 people who came in contact with him are being tested and will be quarantined. Armenia had earlier closed its border with Iran. As of 15 March there are 23 confirmed cases with over 300 being in quarantine.\\n\\nOn 23 March, it confirmed 23 cases.\\n\\n\\n=== Azerbaijan ===\\n\\nOn 28 February, Azerbaijan confirmed the first case from a Russian national, who was travelling from Iran. On 12 March, a woman died from multiorgan failure who had been diagnosed with COVID-19 a day earlier. This marked the first death of coronavirus in Azerbaijan. On 22 March, the first domestic human to human transmission was confirmed. On 31 March, Azerbaijan declared nationwide quarantine. People are required to stay in private houses and apartments, permanent or temporary places of residence until 20 April.\\n\\n\\n=== Bahrain ===\\n\\nThe first case in the country was confirmed on 21 February. The index case was a school bus driver who had travelled to Iran. Bahrain has recorded a total of 2,009 COVID-19 cases including 7 deaths and 1,026 recoveries. The Bahraini government has unveiled a stimulus packages of 4.3 billion Bahraini Dinars that include exempting consumers from bills of electricity and water for three months.\\n\\n\\n=== Bangladesh ===\\nThe first three COVID-19 cases of the country were found on 7 March 2020 that was officially confirmed on 8 March 2020 by Professor Dr Meerjady Sabrina Flora, Former Director, Institute of Epidemiology, Disease Control and Research (IEDCR), then. Two of those affected returned to Bangladesh from Italy and one was a family member of one of those two. On 18 March, the first known coronavirus death in the country was reported.On 22 March, Bangladesh declared a 10-day shut down effective from 26 March to 4 April to fight the spread of coronavirus.\\nBangladesh on Wednesday reported the fifth death from the coronavirus though no new case of the infection came out in the last 24 hours as the country suspended all domestic flights, trains and public transport to fight the pandemic.The Institute of Epidemiology, Disease Control and Research (IEDCR) has confirmed that one more person has died of coronavirus (COVID-19) infection in Bangladesh, taking the number of deaths from the disease in the country to five, the Dhaka Tribune reported.Bangladesh on Wednesday confirmed another death taking the death toll in the country to six while number of positive cases rose to 54.The nationwide lockdown has been extended till 9 April to curb the spread, however Prime Minister Sheikh Hasina on Tuesday said that offices and industries could resume work. Health minister Zahid Malik said \"another 300 ventilators are being imported. There are about 700 ventilators across private hospitals\".\\n\\n\\n=== Bhutan ===\\n\\nOn 6 March, the first case in the country was confirmed, a 76-year-old US male who travelled to the country.\\n\\n\\n=== Brunei ===\\n\\nOn 9 March, the Ministry of Health confirmed that a preliminary coronavirus test had returned positive for a 53-year-old male who had returned from Kuala Lumpur, Malaysia on 3 March. The patient was moved to the National Isolation Centre in Tutong for treatment.\\n\\n\\n=== Cambodia ===\\n\\nOn 27 January, Cambodia confirmed the first COVID-19 case in Sihanoukville, a 60-year-old Chinese man, travelling to the coastal city from Wuhan with his family on 23 January. Three other members of his family were placed under quarantine as they did not appear to have symptoms, while he was placed in a separate room at the Preah Sihanouk Referral Hospital.\\nBy 10 February, after two weeks of being treated and kept under observation, he had fully recovered, Health Ministry stated on account of testing negative for the third time by Pasteur Institute of Cambodia. The family were finally discharged and flew back to their home country on the next day as of the 80 Chinese nationals who arrived in Sihanoukville on the same flight as the patient, most had since returned to China, although the city of Wuhan remained under quarantine at that time.\\n\\n\\n=== China ===\\n\\nThe COVID-19 pandemic first originated in Wuhan, Hubei in which it manifested itself as cluster of mysterious suspected pneumonia cases.Sophisticated modelling of the outbreak suggests that the number of cases in mainland China would have been many times higher without interventions such as early detection, and isolation of the infected.\\n\\n\\n=== Christmas Island ===\\nOn 8 March, the Australian external territory of Christmas Island reported its first case of COVID-19.\\n\\n\\n=== Cyprus ===\\n\\nOn 9 March, Cyprus confirmed its first 2 cases, one in Nicosia and one in Limassol.\\n\\n\\n=== East Timor ===\\n\\nOn 20 March, East Timor confirmed its first COVID-19 case.\\n\\n\\n=== Georgia ===\\n\\nAll flights from China and Wuhan to Tbilisi International Airport were cancelled until 27 January. The Health Ministry announced that all arriving passengers from China would be screened. Georgia also temporarily shut down all flights to Iran.On 26 February, Georgia confirmed its first COVID-19 case. A 50-year-old man, who returned to Georgia from Iran, was admitted to Infectious Diseases Hospital in Tbilisi. He came back to the Georgian border via Azerbaijan by taxi.On 28 February, Georgia confirmed that a 31-year-old Georgia woman who had travelled to Italy tested positive and was admitted to Infectious Diseases Hospital in Tbilisi.29 more are being kept in isolation in a Tbilisi hospital, with Head of the Georgian National Centre for Disease Control, Amiran Gamkrelidze stating there was a \"high probability\" that some of them have the virus.On 5 March, five people have tested positive for the new coronavirus COVID-19 in Georgia increasing the total number of people infected in the country to nine. Head of the Georgian National Centre for Disease Control Amiran Gamkrelidze made the announcement at the recent news briefing following today. He said, all of the five people belong to the same cluster who travelled together to Italy and returned to Georgia on Sunday.On 7 March, three people have tested positive for the new coronavirus in Georgia increasing the total number of people infected individuals in the country to twelve. Head of the Georgian National Centre for Disease Control Amiran Gamkrelidze said at a news briefing the following day that there is still no reason to panic. One of the infected individuals is Gamkrelidze\\'s son Nikoloz. Gamkrelidze wrote on his Facebook page that he contracted the illness from a coworker, who has been tested positive for COVID-19 on Wednesday. Georgia has suspended direct flights with Italy to prevent the spread of coronavirus in the country. Coronavirus in Georgia has mostly been detected in passengers who have travelled in Italy recently.\\n\\n\\n=== Hong Kong ===\\n\\nAs of 1 March, Hong Kong\\'s Centre for Health Protection had identified 100 cases (Including 2 Suspected Recovered Cases) in Hong Kong, with 36 patients since recovered and 2 died. By 2 April, the number of confirmed or probable cases in Hong Kong has risen to 767 after an influx of returning overseas students. 467, or 60.89% of cases were imported cases.\\n\\n\\n=== India ===\\n\\n\\n=== Indonesia ===\\n\\nCOVID-19 was confirmed to have spread to Indonesia on 2 March 2020, after a dance instructor and her mother tested positive for the virus. Both were infected from a Japanese national.By 9 April 2020, the pandemic had spread to all 34 provinces in the country. Jakarta, West Java, and Central Java are the worst-hit provinces, together accounting almost half of the national total cases. On 13 July 2020, the recoveries exceeded active cases for the first time.As of 14 April 2022, Indonesia has reported 6,037,742 cases, the second highest in Southeast Asia, behind Vietnam. With 155,794 deaths, Indonesia ranks second in Asia and ninth in the world. Review of data, however, indicated that the number of deaths may be much higher than what has been reported as those who died with acute COVID-19 symptoms but had not been confirmed or tested were not counted in the official death figure.Indonesia has tested 61,653,691 people against its 270 million population so far, or around 228,175 people per million. The World Health Organization has urged the nation to perform more tests, especially on suspected patients.Instead of implementing a nationwide lockdown, the government applied \"large-scale social restrictions\" (Indonesian: Pembatasan Sosial Berskala Besar, abbreviated as PSBB), which was later modified into the \"community activities restrictions enforcement\" (Indonesian: Pemberlakuan Pembatasan Kegiatan Masyarakat, abbreviated as PPKM).On 13 January 2021, President Joko Widodo was vaccinated at the presidential palace, officially kicking off Indonesia\\'s vaccination program. As of 14 April 2022 at 18:00 WIB (UTC+7), 197,949,702 people had received the first dose of the vaccine and 162,269,263 people had been fully vaccinated; 29,410,363 of them had been inoculated with the booster or the third dose.\\n\\n\\n=== Iran ===\\n\\nIran reported its first confirmed cases of SARS-CoV-2 infections on 19 February 2020 in Qom. Later that day, the Ministry of Health and Medical Education stated that both had died.By 21 February, a total of 18 people had been confirmed to have SARS-CoV-2 infections and four COVID-19 deaths had occurred. On 24 February, according to the Ministry of Health and Medical Education, twelve COVID-19 deaths had occurred in Iran, out of a total of 64 SARS-CoV-2 confirmed infections.On 25 February, Iran\\'s Deputy Health Minister, Iraj Harirchi tested positive for COVID-19, having shown some signs of infection during the press conference. On 3 March, the official number of deaths in Iran rose to 77, the second highest deaths recorded outside China after Italy which has surpassed Iran, although the number of deaths is believed to be higher, up to 1,200 deaths due to Iranian Government\\'s censorship and its eventual mishandling of virus outbreak. Iran currently has the most cases in Western Asia as well as the fourth most cases worldwide, with China, South Korea, and Italy surpassing Iran.Iran\\'s death toll goes to 2,234 on 26 March as 29,000 cases are reported. Public gatherings are banned as is transportation between cities; public parks are closed.\\n\\n\\n=== Iraq ===\\n\\nThe first case in the country was confirmed on 22 February.\\n\\n\\n=== Israel ===\\n\\nOn 21 February, Israel confirmed the first case of COVID-19.On 20 March, the first confirmed death in Israel was reported.As of 9 August, there are a total of 82,324 confirmed cases, with 57,071 recovered and 593 deaths.\\n\\n\\n=== Japan ===\\n\\nJanuary 16, 2020, the first case was confirmed in a 30-year-old Chinese national who had previously travelled to Wuhan, developed a fever on 3 January, and returned to Japan on 6 January. The first mass infection was confirmed on a cruise ship returning to Japan, with 713 cases and 13 deaths. The cruise ship left Yokohama on January 20, 2020 and called at Kagoshima, Hong Kong, Vietnam, Taiwan and Okinawa before returning to Yokohama on February 3.Since then, there have been 6 peaks of infection and death in Japan by February 2022, with the fifth being caused by the Delta variant and the sixth by the Omicron variant. As of February 2022, the total number of infected persons was about 4.16 million and the total number of deaths was about 21,000.The 2020 Tokyo Olympics was postponed from July 2020 to July 2021 due to COVID-19. Around the same time as the Olympics began, the delta variant began to spread in Japan, marking the fifth peak in COVID-19 infections since the Games ended.\\n\\n\\n=== Jordan ===\\n\\nOn 2 March, the first case in the country was confirmed. Jordan has 212 confirmed infections on 26 March. Anyone who disobeys nightly curfew will be fined up to 500 dinars (around $700). The government placed Irbid under quarantine after it recorded 26 cases in the area.\\n\\n\\n=== Kazakhstan ===\\n\\nOn 13 March, the first two cases in the country were confirmed.As of 10 June, there are 13,319 confirmed cases with 62 deaths.\\n\\n\\n=== Kuwait ===\\n\\nThe first case in the country was confirmed on 24 February.\\nThe Kuwaiti prime minister stressed that the State of Kuwait greatly values the contribution of the large Indian community there and would continue to ensure their safety and welfare in the present situation, a statement issued by the Prime Minister\\'s Office said.Modi expressed his thanks and appreciation for the reassurance.Both leaders discussed the domestic and international aspects of the ongoing COVID-19 pandemic, the statement said.\\n\\n\\n=== Kyrgyzstan ===\\n\\nOn 18 March, the first three cases in the country were confirmed.\\nKyrgyzstan had confirmed its first three coronavirus cases, Healthcare Minister Kosmosbek Cholponbayev said on Wednesday.Three Kyrgyz nationals tested positive after arriving from Saudi Arabia.\\n\\n\\n=== Laos ===\\n\\nAs of 23 April, there are 19 confirmed cases in Laos.\\n\\n\\n=== Lebanon ===\\n\\nOn 21 February 2020, Lebanon confirmed the first case of COVID-19, a 45-year-old woman travelling from Qom, Iran tested positive for SARS-CoV-2 and was transferred to a hospital in Beirut. Lebanon had 386 cases and nine deaths through 25 March, when it instituted a lockdown through 12 April. Essential services, such as drugstores and supermarkets, must close at nightfall.The number of COVID-19 infections remained unchanged at 333, NNA said.Meanwhile, the cabinet decided to extend the curfew to 13 April, citing the increasing number of coronavirus infections.\\n\\n\\n=== Macau ===\\n\\nThe first case in Macau was confirmed on 22 January. As of 9 August, Macau has confirmed 46 cases, with all cases discharged.\\n\\n\\n=== Malaysia ===\\n\\nEight Chinese nationals were quarantined at a hotel in Johor Bahru on 24 January after coming into contact with an infected person in neighbouring Singapore. Despite early reports of them testing negative for the virus, three of them were confirmed to be infected on 25 January.On 16 February, the 15th infected patient involving a Chinese female national had fully recovered, becoming the 8th patient cured from the virus in Malaysia. The following day, the first infected Malaysian also reportedly recovered, becoming the 9th cured.In March 2020, several Southeast Asian countries experienced a significant rise in cases following an event held by Tablighi Jamaat at Jamek Mosque in Sri Petaling, Kuala Lumpur, where many people are believed to have been infected. By 17 March, almost two-thirds of the 673 cases confirmed in Malaysia were related to the event. More than 620 people, including those from other countries, who attended the event have tested positive, making it the largest-known centre of transmission in South East Asia. In response to the rapid spread of cases, the Government introduced Movement Control Order lockdown restrictions on 18 March 2020, which helped to lower the infection and death rates.The number of active cases peaked in April and slowly declined, leading to a relaxation of Movement Control Order lockdown restrictions over the next several months. Since mid-September, an outbreak of cases in Sabah, Selangor, Negeri Sembilan, Johor, Penang and Kedah led to a resurgence of COVID-19 cases throughout the country. By 18 November 2020, the total number of cases in Malaysia had exceeded the 50,000 mark. A Sin Chew Daily editorial has attributed the rapid surge of cases to the failure of the public, businesses and their employees including migrant workers to practise health and social distancing procedures during the relaxation of Movement Control Order restrictions throughout 2020.By 24 December 2020, the total number of cases in Malaysia had exceeded the 100,000 mark. By 6 January 2021, the number of recovered had exceeded 100,000. On the same day, the Director General reported there were 252 active clusters in Malaysia. In late February 2021, the Malaysian Government government launched a twelve-month immunization program, with Prime Minister Muhyiddin Yassin being the first individual to receive the COVID-19 vaccine.By 22 May, the total number of cases had reached the 500,000 mark, reaching 505,115. By 2 June 2021, the total number of recoveries had exceeded the 500,000 mark, reaching 501,898. By 4 July, eight percent of the Malaysian population (roughly 2,618,316 people) had completed two doses of COVID-19 vaccination.On 26 July, the total number of cases exceeded the one million mark, reaching 1,013,438. By 5 August, the death toll had reached the 10,000 mark, reaching 10,019. By 7 August, the total number of recoveries had reached the 1 million mark, reaching 1,009,343.\\n\\n\\n=== Maldives ===\\n\\nOn 7 March, the first two cases in the country were confirmed. By 21 July there are 3,252 confirmed cases in Maldives.\\n\\n\\n=== Mongolia ===\\n\\nOn 10 March, the first case have been confirmed, a 57-year-old French citizen came from Moscow-Ulaanbaatar flight on 2 March and symptoms were shown on 7 March.\\n\\n\\n=== Myanmar ===\\n\\nOn 23 March, Myanmar confirmed its first and second COVID-19 cases.\\nMyanmar reported its first coronavirus death on 31 March, a 69-year-old man who also had cancer and died in a hospital in the commercial capital of Yangon, a government spokeswoman said.He had sought medical treatment in Australia and stopped in Singapore on his way home, according to the health ministry.After the coup d\\'état on February 1, 2021, testing collapsed and the medical response to COVID-19 in the country became severely hampered.\\n\\n\\n=== Nepal ===\\n\\nA Nepali student who had returned to Kathmandu from Wuhan became the first case of the country and South Asia on 23 January 2020. The first case of local transmission inside the country was confirmed on 4 April, while the first COVID-19 death came on 14 May. The country observed an almost four-month-long nationwide lockdown between 24 March and 21 July. As of 8 March 2022, the country has a total of 977,641 cases, 959,817 recoveries, and 11,949 deaths.\\n\\n\\n=== Oman ===\\n\\nOn 24 February, the first two cases in the country were confirmed. As of 22 July, there are a total of 69,887 confirmed cases, with 46,608 discharged and 337 deaths.\\n\\n\\n=== Pakistan ===\\n\\nPakistan reported its first two cases of COVID-19 on 26 February 2020. By early September 2020, Pakistan had the 10th-highest number of confirmed cases in Asia.\\n\\n\\n=== Palestine ===\\n\\nThe first seven cases were confirmed in the State of Palestine on 5 March.\\n\\n\\n=== Philippines ===\\n\\n\\n=== Qatar ===\\n\\nQatar confirmed its first case on 29 February, in a person who had returned from Iran. The first death in Qatar was recorded in on 28 March 2020, a 57-year-old Bangladeshi national who was already suffering from chronic disease.\\n\\n\\n=== Russia ===\\n\\nRussia implemented preventive measures to curb the spread of COVID-19 in the country by imposing quarantines, carrying raids on potential virus carriers and using facial recognition to impose quarantine measures.On 31 January, two cases were confirmed, one in Tyumen Oblast, another in Zabaykalsky Krai. Both were Chinese nationals, who have since recovered. By 17 April, first case was confirmed in the Altai Republic, thus all 27 federal subjects of Asian Russia had confirmed cases.\\n\\n\\n=== Saudi Arabia ===\\n\\nOn 27 February, Saudi Arabia announced temporary suspension of entry for individuals wanting to perform Umrah pilgrimage in Mecca or to visit the Prophet\\'s Mosque in Medina, as well as tourists. The rule was also extended to visitors traveling from countries where SARS-CoV-2 posed a risk.On 28 February, the Foreign Minister of Saudi Arabia announced a temporary suspension of entry for Gulf Cooperation Council (GCC) citizens to Mecca and Medina. Citizens of the GCC who had been in Saudi Arabia for more than 14 continuous days and didn\\'t show any symptoms of COVID-19 would be excluded from this rule.Saudi Arabia confirmed the first case on 2 March, a Saudi national returning from Iran via Bahrain.On Thursday, 19 March Saudi Arabia suspended the holding of daily prayers and the weekly Friday prayers inside and outside the walls of the two mosques in Mecca and Medina  to limit the spread of coronavirus. As of Thursday, 334 confirmed cases have been reported in Saudi Arabia with eight cases been recovered. No deaths have been reported.\\nOn Friday, 20 March, Saudi Arabia announced it is going to suspend all domestic flights, buses, taxis and trains for 14 days amid the COVID-19 pandemic.At the virtual G20 meeting, chaired by King Salman on 25 March, collective pledges were made to inject $4.8 trillion into the global economy to counteract the social and financial impacts of the pandemic.On 26 March, authorities announced a total lockdown of Riyadh, Mecca and Medina, plus a nationwide curfew. 1,012 cases and four deaths are reported.\\n\\n\\n=== Singapore ===\\n\\n\\n=== South Korea ===\\n\\nThe first confirmed case in South Korea was announced 20 January 2020. The number of confirmed cases increased on 19 February by 20, and on 20 February by 58, giving a total of 346 confirmed cases on 21 February 2020, according to the Centers for Disease Control and Prevention Korea (KCDC), with the sudden jump mostly attributed to \"Patient No. 31\" who attended a gathering at a Shincheonji Church of Jesus the Temple of the Tabernacle of the Testimony church in Daegu. As of 20 February 2020, the number of confirmed cases in South Korea was the third largest after China and the infections on the Diamond Princess. By 24 February, the number of confirmed cases in South Korea was the second largest; as of 14 March 2020, the number was the fourth largest. A reason for the high number of confirmed cases is the high number of tests conducted. In South Korea more than 66,650 people were tested within a week of its first case of community transmission, and South Korea quickly became able to test 10,000 people a day.\\n\\n\\n=== Sri Lanka ===\\n\\nThe first case in the country was confirmed on 27 January 2020. The country has 135,796 cases with 892 deaths as of 13 May 2021. As of 10 October 2020, Sri Lankan authorities have tracked down over 1,430,864 people who had contacted the identified patients and had ordered self quarantine for such people. Near tested 3 million PCR tested on 5 May 2021.\\n\\n\\n=== Syria ===\\n\\nDue to Syria already coping with the rampant civil war, fearing that Syria will be the most affected country is raising concerns, following a number of cases found in neighboring Iraq, Lebanon and Jordan, and collapsed healthcare system as the result of the civil war. The Government of Iraqi Kurdistan, in a rare collaboration with its Syrian counterpart on 2 March, ordered complete closure of Syrian–Iraqi border to halt the spread.The first case in Syria was confirmed on 22 March.\\n\\n\\n=== Taiwan ===\\n\\nThe pandemic has had a smaller impact in Taiwan than in most other industrialized countries, with a total of eleven deaths out of a population of 23 million as of 11 April 2021, a rate of 0.042 deaths per 100,000 people. The number of active cases peaked on 6 April 2020 at 307 cases. Out of approximately 1,000 cases total, only 77 (along with one additional case of unknown origin) were infected within Taiwan due to strict border control and quarantine measures of incoming travelers and thorough contact tracing of all confirmed cases, allowing for minimal disruption of society, education, and industry. No lockdowns have been imposed in Taiwan. All the other cases have been imported from abroad.\\n\\n\\n=== Tajikistan ===\\n\\nOn 30 April 2020, the first 15 cases of COVID-19 were reported in Tajikistan.\\n\\n\\n=== Thailand ===\\n\\nOn 13 January, Thailand had its first case, also the first outside China.On 1 March, the first confirmed death in Thailand was reported.As of 19 July, there were a total of 3,246 confirmed cases with 58 deaths and 3,096 recoveries.\\n\\n\\n=== Turkey ===\\n\\n\\n=== United Arab Emirates ===\\n\\nThe first case in the United Arab Emirates was confirmed on 29 January 2020. It was the first country in the Middle East to report a confirmed case.The first death due to COVID-19 was reported on 20 March 2020.\\n\\n\\n=== Uzbekistan ===\\n\\nOn 15 March, the first case in the country was confirmed.\\n\\n\\n=== Vietnam ===\\n\\nThe first two confirmed cases in Vietnam were admitted to Cho Ray Hospital, Ho Chi Minh City on 23 January 2020: a 66-year-old Chinese man traveling from Wuhan to Hanoi to visit his son, and his son who believed to have contracted the virus from his father when they met in Nha Trang.On 21 March, Vietnam suspended entry for all foreigners from midnight of 22 March, and concentrated isolation for 14 days in all cases of entry for Vietnamese citizens. From 1 April, Vietnam implemented nationwide isolation for 15 days.\\n\\n\\n=== Yemen ===\\n\\nThe pandemic was confirmed to have spread to Yemen when its first confirmed case in Hadhramaut was reported on 10 April.The country is seen to be extremely vulnerable to the outbreak, given the dire humanitarian situation due to the civil war, exacerbated by the famine, cholera outbreaks, and military blockade by Saudi Arabia and allies.\\n\\n\\n== Prevention in other countries and territories ==\\n\\n\\n=== Turkmenistan ===\\n\\nAs of May 2020, there are no confirmed COVID-19 cases in Turkmenistan. The government has censored use of the word \"coronavirus\" to control information about the virus, and experts suspect that it may be spreading in the country unreported.Turkmenistan is a notoriously opaque state, purposefully isolated by an authoritarian regime led by former president Gurbanguly Berdimuhamedow, who has also been described as totalitarian. Independent media in the country is virtually nonexistent so reporting about the current situation is difficult due to the inability to access and confirm reliable information from the country.\\n\\n\\n=== North Korea ===\\n\\nThere are no confirmed cases in North Korea. North Korea was one of the first countries to close borders due to COVID-19. In February, wearing face masks was obligatory and visiting public places such as restaurants was forbidden. Ski resorts and spas were closed and military parades, marathons, and other public events were cancelled.On 31 March 2020, the Asia Times reported that North Korea\\'s measures against the pandemic seemed largely successful. Edwin Salvador, WHO\\'s representative in North Korea, reported that as of 2 April 709 people had been tested, with no confirmed cases, and 509 people were in quarantine. On 23 April, US analyst website 38 North reported that North Korea\\'s early and extensive response appeared to be successful in containing the virus.Some anonymous reports claimed that North Korea was ineffective in curbing the disease, and that an order was enacted to \\'spot and shoot\\', so as to hide the cases being reported nationwide. Rumours also spread that first cases were reported when three soldiers were found infected, and that they were shot to death.\\n\\n\\n== See also ==\\nCOVID-19 pandemic by country and territory\\n\\n\\n== Notes ==\\n\\nMap Notes\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\nAgarwal, Sumit, et al. eds. Impact of COVID-19 on Asian Economies and Policy Responses (2021) excerpt\\nM. Mofijur et al. \"Impact of COVID-19 on the social, economic, environmental and energy domains: Lessons learnt from a global pandemic\" Sustainable Production and Consumption (2020), doi: https://doi.org/10.1016/j.spc.2020.10.016\\n\\n\\n== External links ==\\nCoronavirus COVID-19 Global Cases and historical data by Johns Hopkins University',\n",
              "  0.2550187376809749),\n",
              " ('The article contains the number of cases of coronavirus disease 2019 (COVID-19) reported by each country, territory, and subnational area to the World Health Organization (WHO) and published in WHO reports, tables, and spreadsheets. For more international statistics in table and map form, see COVID-19 pandemic by country and territory. As of 14 April 2022, 501,918,823 cases have been stated by government agencies from around the world to be confirmed.\\n109 countries and territories have more confirmed cases than the People\\'s Republic of China, the country where the outbreak began. Thailand was the first country to report at least one case outside China. The United States and Italy were first two countries to overtake China in terms of the number of confirmed cases. The country that overtook China in terms of the number of confirmed cases several days later was the United Kingdom. Japan was the first country in East Asia to overtake China in terms of the number of confirmed cases. The second country in East Asia that overtook China in terms of the number of confirmed cases was South Korea, while the third and most recent one was Mongolia. The most recent country that overtook China in terms of the number of confirmed cases was New Zealand, while the most recent territory was Hong Kong. Today, 22 most affected countries have at least five million cases, incl. the United States, India, Brazil, France, Germany, the United Kingdom, South Korea, Russia, Italy, Spain, Argentina, Japan, Poland and Australia. At the moment, 41 most affected countries, incl. Greece, Thailand, Romania, Czechia, South Africa, Portugal, Canada and Chile, have at least two million cases.\\nThe first person infected with the disease, known as COVID-19, was discovered at the beginning of December 2019. The disease has spread very easily to the United States, India, Brazil, France, Germany, the United Kingdom, South Korea, Russia, Italy, Spain, Argentina, Japan, Poland, Australia, Thailand, Czechia, South Africa, Portugal, Canada, Chile, Greece, Romania, Hungary and Egypt and many other countries. The COVID-19 outbreak has been a pandemic since 11 March 2020. A total of about 6.2 million deaths worldwide pertaining to COVID-19 was reported as of April 2022. At the beginning of December 2021, the second anniversary of the beginning of the COVID-19 outbreak was commemorated. As of 14 April 2022, 109 countries and territories have at least 200,000 confirmed COVID-19 cases, and of them, 86 (nearly 78.9%) have at least half a million confirmed COVID-19 cases, incl. Egypt and Hungary. On 11 March 2022, the second anniversary of the day when the COVID-19 outbreak became a pandemic was commemorated.\\nOn 11 February 2022, Japan, formerly the most affected country in East Asia, joined the list of 20 most affected countries, incl. the United States, India, Brazil, France, Germany, the United Kingdom, Russia, Italy, Spain, Argentina and Poland. More than a week later, on 20 February 2022, South Korea, formerly the second most affected country in East Asia, joined the list of 40 most affected countries, incl. Australia, Thailand, Czechia, South Africa, Portugal, Canada, Chile, Greece and Romania. At the beginning of March 2022, South Korea joined the list of 30 most affected countries on the first anniversary of the day when it overtook China in terms of the number of cases. As of 14 April 2022, France is the most affected country in Europe, while Germany is the second most affected country and the United Kingdom the third most affected country. On 4 March 2022, South Korea joined the list of 20 most affected countries, incl. eight in Europe, such as France, Germany and the United Kingdom. On 26 March 2022, South Korea joined the list of ten most affected countries, like the United Kingdom.\\nThe first section contains summary information: the total number of countries and territories with at least 100, 1,000, 10,000, 100,000, a million and ten million cases; the number of cases reported to WHO; the countries and territories that have reported no cases yet to WHO; and two charts showing the 20 countries and territories with the highest numbers of cases and deaths per capita. In the second section, the table has a timeline of confirmed cases of COVID-19. The number of countries affected is shown, along with the number of days taken for the number of cases to double. The table can be sorted by country or date of first confirmed case.\\n\\n\\n== Current situation ==\\n\\n\\n=== Current situation (cases) ===\\nWorldwideAs of 14 April 2022: \\n\\n109 countries and territories with more cases than mainland China. New Zealand was the most recent country to overtake China in terms of the number of cases while Hong Kong was the most recent territory.\\n218 countries and territories with at least 100 cases. In some of those countries, it took 20 days to reach 100.\\n215 countries and territories with at least 1,000 cases. From 100 to 1,000, it took nine days in some of those countries.\\n191 countries and territories with at least 10,000 cases. From 1,000 to 10,000, it took ten days in some of those countries.\\n125 countries and territories with at least 100,000 cases. From 10,000 to 100,000, it took an average of 15 days in some of those countries.\\n61 countries and territories with at least a million cases, such as Japan. From 100,000 to a million, it took an average of 39 days in some of those countries.\\n12 countries with at least ten million cases, such as the United States, the United Kingdom and South Korea. From a million to ten million, it took an average of less than six months in three of those countries.\\n\\n\\n== By country ==\\nThe table shows COVID-19 pandemic cases in the following countries:\\n\\nArgentina\\nAustralia\\nBrazil\\nCanada\\nChile\\nCzechia\\nEgypt\\nFrance\\nGermany\\nGreece\\nHungary\\nIndia\\nItaly\\nJapan\\nPoland\\nPortugal\\nRomania\\nRussia\\nSouth Africa\\nSouth Korea\\nSpain\\nThailand\\nUnited Kingdom\\nUnited States\\n\\n\\n== Countries and territories without confirmed cases ==\\nBelow is a list of countries and dependent territories that have confirmed no cases of COVID-19, in the order of greatest population.\\n\\n\\n=== UN member states ===\\nAs of 14 April 2022, three sovereign states (one in Oceania and two in Asia) have reported no cases of COVID-19 to WHO. Below is a list of the countries, ordered by population.\\n\\n\\n=== Dependent territories ===\\nAs of 14 April 2022, two dependencies have reported no cases of the disease to WHO.\\n\\n\\n== Monthly links ==\\nThe pages below provide tables of daily figures:\\n\\nCOVID-19 pandemic cases in January 2020\\nCOVID-19 pandemic cases in February 2020\\nCOVID-19 pandemic cases in March 2020\\nCOVID-19 pandemic cases in April 2020\\nCOVID-19 pandemic cases in May 2020\\nCOVID-19 pandemic cases in June 2020\\nCOVID-19 pandemic cases in July 2020\\nCOVID-19 pandemic cases in August 2020\\nCOVID-19 pandemic cases in September 2020\\nCOVID-19 pandemic cases in October 2020\\nCOVID-19 pandemic cases in November 2020\\nCOVID-19 pandemic cases in December 2020\\nCOVID-19 pandemic cases in January 2021\\nCOVID-19 pandemic cases in February 2021\\nCOVID-19 pandemic cases in March 2021\\nCOVID-19 pandemic cases in April 2021\\nCOVID-19 pandemic cases in May 2021\\nCOVID-19 pandemic cases in June 2021\\nCOVID-19 pandemic cases in July 2021\\nCOVID-19 pandemic cases in August 2021\\nCOVID-19 pandemic cases in September 2021\\nCOVID-19 pandemic cases in October 2021\\nCOVID-19 pandemic cases in November 2021\\nCOVID-19 pandemic cases in December 2021\\nCOVID-19 pandemic cases in January 2022\\nCOVID-19 pandemic cases in February 2022\\nCOVID-19 pandemic cases in March 2022\\nCOVID-19 pandemic cases in April 2022\\nCOVID-19 pandemic cases in May 2022\\nCOVID-19 pandemic cases in June 2022\\nCOVID-19 pandemic cases in July 2022\\nCOVID-19 pandemic cases in August 2022\\n\\n\\n== Cumulative monthly case totals by country ==\\n\\n\\n=== 2020 ===\\n\\n\\n=== 2021 ===\\n\\n\\n=== 2022 ===\\n\\n\\n== Reliability of the confirmed counts ==\\nDoctors and media expressed scepticism of official infection counts in some countries, and statistical analysis suggested anomalies. Sceptics of the Russian official counts included medical doctor Anastasia Vasilyeva, doctors Ivan Konovalov and Samuel Greene, and medical news editor Aleksey Torgashev. The Nexta media agency stated that it had obtained leaked administrative documents showing higher infection counts than those publicly stated by Belarusian authorities. Statistical analysis of official counts from around the world found signs of \"suspiciously low statistical noise\" in the officially confirmed counts of some countries, with the Algerian counts including a 28-day sequence with a strongly sub-Poissonian noise level, and 28-day sequences best modelled as having sub-Poissonian noise for the daily confirmed counts in Tajikistan, Turkey, Russia, Belarus, Albania, the United Arab Emirates and Nicaragua.Lower statistical noise in the official infection counts is statistically associated with less press freedom, as measured by the Reporters Without Borders Press Freedom Index. Benford\\'s law analysis of the rising phase of the pandemic per country found that the official counts are more statistically credible for countries that are more democratic, have higher GDP per capita, or have more health funding or better universal health insurance systems.\\n\\n\\n== See also ==\\n\\nCOVID-19 pandemic\\nCOVID-19 pandemic death rates by country\\nCOVID-19 pandemic by country and territory\\n\\n\\n== References ==\\n\\n\\n== External links ==\\nCOVID-19 Risk Factors',\n",
              "  0.0)]"
            ]
          },
          "execution_count": 54,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Outputting top 10 retrieved documents by rank and scores\n",
        "sorted(zip(documents[:10],score_q_d.values), key = lambda tup:tup[1], reverse=True)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "jHPeaUbbiM56"
      },
      "source": [
        "Retrieval function performs successfully and as expected. Because the BM25 function normalises by document length, documents with equal number of query terms but with more words are scored lower, while those with fewer words are scored higher. Also, documents with rare terms are ranked higher by virtue of Luhn's Analysis (Kocabaş _et al_, 2011)<br> \n",
        "\n",
        "### **Section 4: Evaluating BM25 model**<br> \n",
        "The performance of the created model is evaluated in the next series of steps:"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "rEeFgugS22pM"
      },
      "outputs": [],
      "source": [
        "df = dataframe"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Fl04zqKE29dm",
        "outputId": "ff302146-17d4-4983-b8f8-5c9f1e467fd1"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "pandas.core.frame.DataFrame"
            ]
          },
          "execution_count": 56,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "type(df)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 235
        },
        "id": "cePqoGU83yAm",
        "outputId": "c15453b5-bf43-4b1d-cb5c-12143c199da1"
      },
      "outputs": [
        {
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-a9768dbc-91aa-4c61-948a-4cd5707fd4a3\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>03</th>\n",
              "      <th>030</th>\n",
              "      <th>035</th>\n",
              "      <th>03614</th>\n",
              "      <th>037</th>\n",
              "      <th>04</th>\n",
              "      <th>041</th>\n",
              "      <th>042</th>\n",
              "      <th>043</th>\n",
              "      <th>045</th>\n",
              "      <th>...</th>\n",
              "      <th>zones</th>\n",
              "      <th>zooanthroponosis</th>\n",
              "      <th>zoom</th>\n",
              "      <th>zoonosis</th>\n",
              "      <th>zoonotic</th>\n",
              "      <th>zoos</th>\n",
              "      <th>zoster</th>\n",
              "      <th>zulia</th>\n",
              "      <th>zycov</th>\n",
              "      <th>zürich</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>5 rows × 13249 columns</p>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-a9768dbc-91aa-4c61-948a-4cd5707fd4a3')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-a9768dbc-91aa-4c61-948a-4cd5707fd4a3 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-a9768dbc-91aa-4c61-948a-4cd5707fd4a3');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "   03  030  035  03614  037  04  041  042  043  045  ...  zones  \\\n",
              "0   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "1   0    0    0      0    0   0    1    0    0    0  ...      0   \n",
              "2   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "3   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "4   0    0    0      0    0   0    0    0    0    0  ...      0   \n",
              "\n",
              "   zooanthroponosis  zoom  zoonosis  zoonotic  zoos  zoster  zulia  zycov  \\\n",
              "0                 1     0         0         1     1       0      0      0   \n",
              "1                 0     0         0         0     0       0      0      0   \n",
              "2                 0     0         0         0     0       0      0      0   \n",
              "3                 0     0         0         0     0       0      0      0   \n",
              "4                 0     0         0         0     0       0      0      0   \n",
              "\n",
              "   zürich  \n",
              "0       0  \n",
              "1       0  \n",
              "2       0  \n",
              "3       0  \n",
              "4       0  \n",
              "\n",
              "[5 rows x 13249 columns]"
            ]
          },
          "execution_count": 57,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "df[:5]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "C4a9Ndom4Yrl"
      },
      "outputs": [],
      "source": [
        "# Turning term frequencies into BM25 term Frequencies\n",
        "\n",
        "def BM25_IDF_df(df):\n",
        "  \"\"\"\n",
        "  This definition calculates BM25-IDF weights before hand\n",
        "  \"\"\"\n",
        "\n",
        "  dfs = (df > 0).sum(axis=0)\n",
        "  N = df.shape[0]\n",
        "  idfs = -np.log(dfs / N)\n",
        "\n",
        "# Defining variables\n",
        "  k_1 = 1.4                                             # single value\n",
        "  b = 0.75                                              # single value\n",
        "\n",
        "# Considering all words in doc\n",
        "  dls = [len(d.split(' ')) for d in documents]          # vector\n",
        "\n",
        "# Considering words minus stop words in doc. \n",
        "  dls = dataframe.sum(axis=1).tolist()\n",
        "  avgdl = np.mean(dls)                                  # single value\n",
        "  return pd.DataFrame(BM25_score, columns=vocabulary)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 279
        },
        "id": "KeYvT2TC4-_F",
        "outputId": "a1229d65-65ff-4ad3-c4d1-9359f2ae0482"
      },
      "outputs": [
        {
          "data": {
            "text/html": [
              "\n",
              "  <div id=\"df-d5eed9e3-784e-4590-b58e-7a128470754d\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>03</th>\n",
              "      <th>030</th>\n",
              "      <th>035</th>\n",
              "      <th>03614</th>\n",
              "      <th>037</th>\n",
              "      <th>04</th>\n",
              "      <th>041</th>\n",
              "      <th>042</th>\n",
              "      <th>043</th>\n",
              "      <th>045</th>\n",
              "      <th>...</th>\n",
              "      <th>zones</th>\n",
              "      <th>zooanthroponosis</th>\n",
              "      <th>zoom</th>\n",
              "      <th>zoonosis</th>\n",
              "      <th>zoonotic</th>\n",
              "      <th>zoos</th>\n",
              "      <th>zoster</th>\n",
              "      <th>zulia</th>\n",
              "      <th>zycov</th>\n",
              "      <th>zürich</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>...</td>\n",
              "      <td>0.0</td>\n",
              "      <td>1.055068</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.625033</td>\n",
              "      <td>0.84005</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>1.449506</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>...</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>...</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>...</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>...</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>5 rows × 13249 columns</p>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-d5eed9e3-784e-4590-b58e-7a128470754d')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-d5eed9e3-784e-4590-b58e-7a128470754d button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-d5eed9e3-784e-4590-b58e-7a128470754d');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ],
            "text/plain": [
              "    03  030  035  03614  037   04       041  042  043  045  ...  zones  \\\n",
              "0  0.0  0.0  0.0    0.0  0.0  0.0  0.000000  0.0  0.0  0.0  ...    0.0   \n",
              "1  0.0  0.0  0.0    0.0  0.0  0.0  1.449506  0.0  0.0  0.0  ...    0.0   \n",
              "2  0.0  0.0  0.0    0.0  0.0  0.0  0.000000  0.0  0.0  0.0  ...    0.0   \n",
              "3  0.0  0.0  0.0    0.0  0.0  0.0  0.000000  0.0  0.0  0.0  ...    0.0   \n",
              "4  0.0  0.0  0.0    0.0  0.0  0.0  0.000000  0.0  0.0  0.0  ...    0.0   \n",
              "\n",
              "   zooanthroponosis  zoom  zoonosis  zoonotic     zoos  zoster  zulia  zycov  \\\n",
              "0          1.055068   0.0       0.0  0.625033  0.84005     0.0    0.0    0.0   \n",
              "1          0.000000   0.0       0.0  0.000000  0.00000     0.0    0.0    0.0   \n",
              "2          0.000000   0.0       0.0  0.000000  0.00000     0.0    0.0    0.0   \n",
              "3          0.000000   0.0       0.0  0.000000  0.00000     0.0    0.0    0.0   \n",
              "4          0.000000   0.0       0.0  0.000000  0.00000     0.0    0.0    0.0   \n",
              "\n",
              "   zürich  \n",
              "0     0.0  \n",
              "1     0.0  \n",
              "2     0.0  \n",
              "3     0.0  \n",
              "4     0.0  \n",
              "\n",
              "[5 rows x 13249 columns]"
            ]
          },
          "execution_count": 59,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "bm25_df = BM25_IDF_df(df) # a dataframe with BM25-idf weights\n",
        "bm25_df[:5]"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "pqBkVSqnFtVH"
      },
      "source": [
        "The queries \"vaccine effectiveness\" and \"cell membrane\" are input to observe the relevance of documents returned in response to the queries:"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "h1rwQ2sVKTjW",
        "outputId": "ea480606-4d5d-459c-d93e-bdb6c45cc94f"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "{0: 'vaccine effectiveness', 1: 'cell membrane'}"
            ]
          },
          "execution_count": 60,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Inputting query \n",
        "queries = dict(enumerate(['vaccine effectiveness','cell membrane']))      # create dictionary = {query_id: query}\n",
        "queries"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "8tyeC3deY6OR",
        "outputId": "d25089f2-1f9b-40b6-f0d6-07109adc81b7"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "['SARS-CoV-2 Gamma variant',\n",
              " 'Variants of SARS-CoV-2',\n",
              " 'SARS-CoV-2 Delta variant',\n",
              " 'Coronavirus',\n",
              " 'Investigations into the origin of COVID-19',\n",
              " 'Severe acute respiratory syndrome coronavirus 2',\n",
              " 'COVID-19',\n",
              " 'Angiotensin-converting enzyme 2',\n",
              " 'SARS-CoV-2 Omicron variant',\n",
              " 'Virus']"
            ]
          },
          "execution_count": 61,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "terms = ['vaccine effectiveness','cell membrane']\n",
        "result_atcls"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "0kkvTqO-Zs_u",
        "outputId": "3e434d3f-8760-42e0-bc63-5825e341ee5b"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "['Coronavirus disease 2019 (COVID-19) is a contagious disease caused by a virus, the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The first known case was identified in Wuhan, China, in December 2019. The disease has since spread worldwide, leading to the ongoing COVID-19 pandemic.Symptoms of COVID‑19 are variable, but often include fever, cough, headache, fatigue, breathing difficulties, loss of smell, and loss of taste. Symptoms may begin one to fourteen days after exposure to the virus. At least a third of people who are infected do not develop noticeable symptoms. Of those people who develop symptoms noticeable enough to be classed as patients, most (81%) develop mild to moderate symptoms (up to mild pneumonia), while 14% develop severe symptoms (dyspnea, hypoxia, or more than 50% lung involvement on imaging), and 5% suffer critical symptoms (respiratory failure, shock, or multiorgan dysfunction). Older people are at a higher risk of developing severe symptoms. Some people continue to experience a range of effects (long COVID) for months after recovery, and damage to organs has been observed. Multi-year studies are underway to further investigate the long-term effects of the disease.COVID‑19 transmits when people breathe in air contaminated by droplets and small airborne particles containing the virus. The risk of breathing these in is highest when people are in close proximity, but they can be inhaled over longer distances, particularly indoors. Transmission can also occur if splashed or sprayed with contaminated fluids in the eyes, nose or mouth, and, rarely, via contaminated surfaces. People remain contagious for up to 20 days, and can spread the virus even if they do not develop symptoms.Several COVID-19 testing methods have been developed to diagnose the disease. The standard diagnostic method is by detection of the virus\\'s nucleic acid by real-time reverse transcription polymerase chain reaction (rRT-PCR), transcription-mediated amplification (TMA), or by reverse transcription loop-mediated isothermal amplification (RT-LAMP) from a nasopharyngeal swab.\\nSeveral COVID-19 vaccines have been approved and distributed in various countries, which have initiated mass vaccination campaigns. Other preventive measures include physical or social distancing, quarantining, ventilation of indoor spaces, covering coughs and sneezes, hand washing, and keeping unwashed hands away from the face. The use of face masks or coverings has been recommended in public settings to minimize the risk of transmission. While work is underway to develop drugs that inhibit the virus, the primary treatment is symptomatic. Management involves the treatment of symptoms, supportive care, isolation, and experimental measures.\\n\\n\\n== Nomenclature ==\\n\\nDuring the initial outbreak in Wuhan, the virus and disease were commonly referred to as \"coronavirus\" and \"Wuhan coronavirus\", with the disease sometimes called \"Wuhan pneumonia\". In the past, many diseases have been named after geographical locations, such as the Spanish flu, Middle East respiratory syndrome, and Zika virus. In January 2020, the World Health Organization (WHO) recommended 2019-nCoV and 2019-nCoV acute respiratory disease as interim names for the virus and disease per 2015 guidance and international guidelines against using geographical locations or groups of people in disease and virus names to prevent social stigma. The official names COVID‑19 and SARS-CoV-2 were issued by the WHO on 11 February 2020. The Director-General, Tedros Adhanom explained that CO stands for corona, VI for virus, D for disease, and 19 for 2019, the year in which the outbreak was first identified. The WHO additionally uses \"the COVID‑19 virus\" and \"the virus responsible for COVID‑19\" in public communications.\\n\\n\\n== Signs and symptoms ==\\n\\n\\n== Cause ==\\nCOVID‑19 is caused by infection with a strain of coronavirus known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).\\n\\n\\n=== Transmission ===\\n\\n\\n=== Virology ===\\n\\nSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a novel severe acute respiratory syndrome coronavirus. It was first isolated from three people with pneumonia connected to the cluster of acute respiratory illness cases in Wuhan. All structural features of the novel SARS-CoV-2 virus particle occur in related coronaviruses in nature.Outside the human body, the virus is destroyed by household soap, which bursts its protective bubble.SARS-CoV-2 is closely related to the original SARS-CoV. It is thought to have an animal (zoonotic) origin. Genetic analysis has revealed that the coronavirus genetically clusters with the genus Betacoronavirus, in subgenus Sarbecovirus (lineage B) together with two bat-derived strains. It is 96% identical at the whole genome level to other bat coronavirus samples (BatCov RaTG13). The structural proteins of SARS-CoV-2 include membrane glycoprotein (M), envelope protein (E), nucleocapsid protein (N), and the spike protein (S). The M protein of SARS-CoV-2 is about 98% similar to the M protein of bat SARS-CoV, maintains around 98% homology with pangolin SARS-CoV, and has 90% homology with the M protein of SARS-CoV; whereas, the similarity is only around 38% with the M protein of MERS-CoV.\\n\\n\\n=== SARS-CoV-2 variants ===\\n\\nThe many thousands of SARS-CoV-2 variants are grouped into either clades or lineages. The WHO, in collaboration with partners, expert networks, national authorities, institutions and researchers, have established nomenclature systems for naming and tracking SARS-CoV-2 genetic lineages by GISAID, Nextstrain and Pango. At the present time, the expert group convened by WHO has recommended the labeling of variants using letters of the Greek Alphabet, for example, Alpha, Beta, Delta, and Gamma, giving the justification that they \"will be easier and more practical to discussed by non-scientific audiences.\" Nextstrain divides the variants into five clades (19A, 19B, 20A, 20B, and 20C), while GISAID divides them into seven (L, O, V, S, G, GH, and GR). The Pango tool groups variants into lineages, with many circulating lineages being classed under the B.1 lineage.Several notable variants of SARS-CoV-2 emerged throughout 2020. Cluster 5 emerged among minks and mink farmers in Denmark. After strict quarantines and a mink euthanasia campaign, the cluster was assessed to no longer be circulating among humans in Denmark as of 1 February 2021.As of December 2021, there are five dominant variants of SARS-CoV-2 spreading among global populations: the Alpha variant (B.1.1.7, formerly called the UK variant), first found in London and Kent, the Beta variant (B.1.351, formerly called the South Africa variant), the Gamma variant (P.1, formerly called the Brazil variant), the Delta variant (B.1.617.2, formerly called the India variant), and the Omicron variant (B.1.1.529), which had spread to 57 countries as of 7 December.\\n\\n\\n== Pathophysiology ==\\n\\nThe SARS-CoV-2 virus can infect a wide range of cells and systems of the body. COVID‑19 is most known for affecting the upper respiratory tract (sinuses, nose, and throat) and the lower respiratory tract (windpipe and lungs). The lungs are the organs most affected by COVID‑19 because the virus accesses host cells via the receptor for the enzyme angiotensin-converting enzyme 2 (ACE2), which is most abundant on the surface of type II alveolar cells of the lungs. The virus uses a special surface glycoprotein called a \"spike\" to connect to the ACE2 receptor and enter the host cell.\\n\\n\\n=== Respiratory tract ===\\nFollowing viral entry, COVID‑19 infects the ciliated epithelium of the nasopharynx and upper airways.\\n\\n\\n=== Nervous system ===\\nOne common symptom, loss of smell, results from infection of the support cells of the olfactory epithelium, with subsequent damage to the olfactory neurons. The involvement of both the central and peripheral nervous system in COVID‑19 has been reported in many medical publications. It is clear that many people with COVID-19 exhibit neurological or mental health issues. The virus is not detected in the CNS of the majority of COVID-19 patients with neurological issues. However, SARS-CoV-2 has been detected at low levels in the brains of those who have died from COVID‑19, but these results need to be confirmed. While virus has been detected in cerebrospinal fluid of autopsies, the exact mechanism by which it invades the CNS remains unclear and may first involve invasion of peripheral nerves given the low levels of ACE2 in the brain. The virus may also enter the bloodstream from the lungs and cross the blood-brain barrier to gain access to the CNS, possibly within an infected white blood cell.\\n  Research conducted when Alpha was the dominant variant has suggested COVID-19 May cause brain damage. It is currently unknown if such damage is temporary or permanent, and whether Omicron has similar effects.  Observed individuals infected with COVID-19 (most with mild cases) experienced an additional 0.2% to 2% of brain tissue lost in regions of the brain connected to the sense of smell compared with uninfected individuals, and the overall effect on the brain was equivalent on average to at least one extra year of normal aging; infected individuals also scored lower on several cognitive tests.  All effects were more pronounced among older ages.\\n\\n\\n=== Gastrointestinal tract ===\\nThe virus also affects gastrointestinal organs as ACE2 is abundantly expressed in the glandular cells of gastric, duodenal and rectal epithelium as well as endothelial cells and enterocytes of the small intestine.\\n\\n\\n=== Cardiovascular system ===\\nThe virus can cause acute myocardial injury and chronic damage to the cardiovascular system. An acute cardiac injury was found in 12% of infected people admitted to the hospital in Wuhan, China, and is more frequent in severe disease. Rates of cardiovascular symptoms are high, owing to the systemic inflammatory response and immune system disorders during disease progression, but acute myocardial injuries may also be related to ACE2 receptors in the heart. ACE2 receptors are highly expressed in the heart and are involved in heart function. A high incidence of thrombosis and venous thromboembolism have been found in people transferred to Intensive care units (ICU) with COVID‑19 infections, and may be related to poor prognosis. Blood vessel dysfunction and clot formation (as suggested by high D-dimer levels caused by blood clots) are thought to play a significant role in mortality, incidences of clots leading to pulmonary embolisms, and ischaemic events within the brain have been noted as complications leading to death in people infected with SARS-CoV-2. Infection appears to set off a chain of vasoconstrictive responses within the body, constriction of blood vessels within the pulmonary circulation has also been posited as a mechanism in which oxygenation decreases alongside the presentation of viral pneumonia. Furthermore, microvascular (arterioles and capillaries) blood vessel damage has been reported in a small number of tissue samples of the brains – without detected SARS-CoV-2 – and the olfactory bulbs from those who have died from COVID‑19. COVID‑19 was also found to cause substantial – including morphological and mechanical – changes to blood cells – such as increased sizes – sometimes persisting for months after hospital discharge.\\n\\n\\n=== Other organs ===\\nAnother common cause of death is complications related to the kidneys. Early reports show that up to 30% of hospitalized patients both in China and in New York have experienced some injury to their kidneys, including some persons with no previous kidney problems.Autopsies of people who died of COVID‑19 have found diffuse alveolar damage, and lymphocyte-containing inflammatory infiltrates within the lung.\\n\\n\\n=== Immunopathology ===\\n\\nAlthough SARS-CoV-2 has a tropism for ACE2-expressing epithelial cells of the respiratory tract, people with severe COVID‑19 have symptoms of systemic hyperinflammation. Clinical laboratory findings of elevated IL‑2, IL‑7, IL‑6, granulocyte-macrophage colony-stimulating factor (GM‑CSF), interferon gamma-induced protein 10 (IP‑10), monocyte chemoattractant protein 1 (MCP1), macrophage inflammatory protein 1‑alpha (MIP‑1‑alpha), and tumour necrosis factor (TNF‑α) indicative of cytokine release syndrome (CRS) suggest an underlying immunopathology.Additionally, people with COVID‑19 and acute respiratory distress syndrome (ARDS) have classical serum biomarkers of CRS, including elevated C-reactive protein (CRP), lactate dehydrogenase (LDH), D-dimer, and ferritin.Systemic inflammation results in vasodilation, allowing inflammatory lymphocytic and monocytic infiltration of the lung and the heart. In particular, pathogenic GM-CSF-secreting T cells were shown to correlate with the recruitment of inflammatory IL-6-secreting monocytes and severe lung pathology in people with COVID‑19. Lymphocytic infiltrates have also been reported at autopsy.\\n\\n\\n=== Viral and host factors ===\\n\\n\\n==== Virus proteins ====\\n\\nMultiple viral and host factors affect the pathogenesis of the virus. The S-protein, otherwise known as the spike protein, is the viral component that attaches to the host receptor via the ACE2 receptors. It includes two subunits: S1 and S2. S1 determines the virus-host range and cellular tropism via the receptor-binding domain. S2 mediates the membrane fusion of the virus to its potential cell host via the H1 and HR2, which are heptad repeat regions. Studies have shown that S1 domain induced IgG and IgA antibody levels at a much higher capacity. It is the focus spike proteins expression that are involved in many effective COVID‑19 vaccines.The M protein is the viral protein responsible for the transmembrane transport of nutrients. It is the cause of the bud release and the formation of the viral envelope. The N and E protein are accessory proteins that interfere with the host\\'s immune response.\\n\\n\\n==== Host factors ====\\nHuman angiotensin converting enzyme 2 (hACE2) is the host factor that SARS-COV2 virus targets causing COVID‑19. Theoretically, the usage of angiotensin receptor blockers (ARB) and ACE inhibitors upregulating ACE2 expression might increase morbidity with COVID‑19, though animal data suggest some potential protective effect of ARB; however no clinical studies have proven susceptibility or outcomes. Until further data is available, guidelines and recommendations for hypertensive patients remain.The effect of the virus on ACE2 cell surfaces leads to leukocytic infiltration, increased blood vessel permeability, alveolar wall permeability, as well as decreased secretion of lung surfactants. These effects cause the majority of the respiratory symptoms. However, the aggravation of local inflammation causes a cytokine storm eventually leading to a systemic inflammatory response syndrome.Among healthy adults not exposed to SARS-CoV-2, about 35% have CD4+ T cells that recognize the SARS-CoV-2 S protein (particularly the S2 subunit) and about 50% react to other proteins of the virus, suggesting cross-reactivity from previous common colds caused by other coronaviruses.It is unknown whether different persons use similar antibody genes in response to COVID‑19.\\n\\n\\n=== Host cytokine response ===\\n\\nThe severity of the inflammation can be attributed to the severity of what is known as the cytokine storm. Levels of interleukin 1B, interferon-gamma, interferon-inducible protein 10, and monocyte chemoattractant protein 1 were all associated with COVID‑19 disease severity. Treatment has been proposed to combat the cytokine storm as it remains to be one of the leading causes of morbidity and mortality in COVID‑19 disease.A cytokine storm is due to an acute hyperinflammatory response that is responsible for clinical illness in an array of diseases but in COVID‑19, it is related to worse prognosis and increased fatality. The storm causes acute respiratory distress syndrome, blood clotting events such as strokes, myocardial infarction, encephalitis, acute kidney injury, and vasculitis. The production of IL-1, IL-2, IL-6, TNF-alpha, and interferon-gamma, all crucial components of normal immune responses, inadvertently become the causes of a cytokine storm. The cells of the central nervous system, the microglia, neurons, and astrocytes, are also involved in the release of pro-inflammatory cytokines affecting the nervous system, and effects of cytokine storms toward the CNS are not uncommon.\\n\\n\\n=== Pregnancy response ===\\nThere are many unknowns for pregnant women during the COVID‑19 pandemic. Given that they are prone to suffering from complications and severe disease infection with other types of coronaviruses, they have been identified as a vulnerable group and advised to take supplementary preventive measures.Physiological responses to pregnancy can include:\\n\\nImmunological: The immunological response to COVID-19, like other viruses, depends on a working immune system. It adapts during pregnancy to allow the development of the fetus whose genetic load is only partially shared with their mother, leading to a different immunological reaction to infections during the course of pregnancy.\\nRespiratory: Many factors can make pregnant women more vulnerable to hard respiratory infections. One of them is the total reduction of the lungs\\' capacity and inability to clear secretions.\\nCoagulation: During pregnancy, there are higher levels of circulating coagulation factors, and the pathogenesis of SARS-CoV-2 infection can be implicated. The thromboembolic events with associated mortality are a risk for pregnant women.However, from the evidence base, it is difficult to conclude whether pregnant women are at increased risk of grave consequences of this virus.In addition to the above, other clinical studies have proved that SARS-CoV-2 can affect the period of pregnancy in different ways. On the one hand, there is little evidence of its impact up to 12 weeks gestation. On the other hand, COVID-19 infection may cause increased rates of unfavorable outcomes in the course of the pregnancy. Some examples of these could be fetal growth restriction, preterm birth, and perinatal mortality, which refers to the fetal death past 22 or 28 completed weeks of pregnancy as well as the death among live-born children up to seven completed days of life.Unvaccinated women in later stages of pregnancy with COVID-19 are more likely than other patients to need very intensive care. Babies born to mothers with COVID-19 are more likely to have breathing problems. Pregnant women are strongly encouraged to get vaccinated.\\n\\n\\n== Diagnosis ==\\n\\nCOVID‑19 can provisionally be diagnosed on the basis of symptoms and confirmed using reverse transcription polymerase chain reaction (RT-PCR) or other nucleic acid testing of infected secretions. Along with laboratory testing, chest CT scans may be helpful to diagnose COVID‑19 in individuals with a high clinical suspicion of infection. Detection of a past infection is possible with serological tests, which detect antibodies produced by the body in response to the infection.\\n\\n\\n=== Viral testing ===\\n\\nThe standard methods of testing for presence of SARS-CoV-2 are nucleic acid tests, which detects the presence of viral RNA fragments. As these tests detect RNA but not infectious virus, its \"ability to determine duration of infectivity of patients is limited.\" The test is typically done on respiratory samples obtained by a nasopharyngeal swab; however, a nasal swab or sputum sample may also be used. Results are generally available within hours. The WHO has published several testing protocols for the disease.Several laboratories and companies have developed serological tests, which detect antibodies produced by the body in response to infection. Several have been evaluated by Public Health England and approved for use in the UK.The University of Oxford\\'s CEBM has pointed to mounting evidence that \"a good proportion of \\'new\\' mild cases and people re-testing positives after quarantine or discharge from hospital are not infectious, but are simply clearing harmless virus particles which their immune system has efficiently dealt with\" and have called for \"an international effort to standardize and periodically calibrate testing\" On 7 September, the UK government issued \"guidance for procedures to be implemented in laboratories to provide assurance of positive SARS-CoV-2 RNA results during periods of low prevalence, when there is a reduction in the predictive value of positive test results\".\\n\\n\\n=== Imaging ===\\n\\nChest CT scans may be helpful to diagnose COVID‑19 in individuals with a high clinical suspicion of infection but are not recommended for routine screening. Bilateral multilobar ground-glass opacities with a peripheral, asymmetric, and posterior distribution are common in early infection. Subpleural dominance, crazy paving (lobular septal thickening with variable alveolar filling), and consolidation may appear as the disease progresses. Characteristic imaging features on chest radiographs and computed tomography (CT) of people who are symptomatic include asymmetric peripheral ground-glass opacities without pleural effusions.Many groups have created COVID‑19 datasets that include imagery such as the Italian Radiological Society which has compiled an international online database of imaging findings for confirmed cases. Due to overlap with other infections such as adenovirus, imaging without confirmation by rRT-PCR is of limited specificity in identifying COVID‑19. A large study in China compared chest CT results to PCR and demonstrated that though imaging is less specific for the infection, it is faster and more sensitive.\\n\\n\\n=== Coding ===\\nIn late 2019, the WHO assigned emergency ICD-10 disease codes U07.1 for deaths from lab-confirmed SARS-CoV-2 infection and U07.2 for deaths from clinically or epidemiologically diagnosed COVID‑19 without lab-confirmed SARS-CoV-2 infection.\\n\\n\\n=== Pathology ===\\nThe main pathological findings at autopsy are:\\n\\nMacroscopy: pericarditis, lung consolidation and pulmonary oedema\\nLung findings:\\nminor serous exudation, minor fibrin exudation\\npulmonary oedema, pneumocyte hyperplasia, large atypical pneumocytes, interstitial inflammation with lymphocytic infiltration and multinucleated giant cell formation\\ndiffuse alveolar damage (DAD) with diffuse alveolar exudates. DAD is the cause of acute respiratory distress syndrome (ARDS) and severe hypoxemia.\\norganisation of exudates in alveolar cavities and pulmonary interstitial fibrosis\\nplasmocytosis in BAL\\nBlood and vessels: disseminated intravascular coagulation (DIC); leukoerythroblastic reaction, endotheliitis, hemophagocytosis\\nHeart: cardiac muscle cell necrosis\\nLiver: microvesicular steatosis\\nNose: shedding of olfactory epithelium\\nBrain: infarction\\nKidneys: acute tubular damage.\\nSpleen: white pulp depletion.\\n\\n\\n== Prevention ==\\n\\nPreventive measures to reduce the chances of infection include getting vaccinated, staying at home, wearing a mask in public, avoiding crowded places, keeping distance from others, ventilating indoor spaces, managing potential exposure durations, washing hands with soap and water often and for at least twenty seconds, practising good respiratory hygiene, and avoiding touching the eyes, nose, or mouth with unwashed hands.Those diagnosed with COVID‑19 or who believe they may be infected are advised by the CDC to stay home except to get medical care, call ahead before visiting a healthcare provider, wear a face mask before entering the healthcare provider\\'s office and when in any room or vehicle with another person, cover coughs and sneezes with a tissue, regularly wash hands with soap and water and avoid sharing personal household items.The first COVID‑19 vaccine was granted regulatory approval on 2 December 2020 by the UK medicines regulator MHRA. It was evaluated for emergency use authorization (EUA) status by the US FDA, and in several other countries. Initially, the US National Institutes of Health guidelines do not recommend any medication for prevention of COVID‑19, before or after exposure to the SARS-CoV-2 virus, outside the setting of a clinical trial. Without a vaccine, other prophylactic measures, or effective treatments, a key part of managing COVID‑19 is trying to decrease and delay the epidemic peak, known as \"flattening the curve\". This is done by slowing the infection rate to decrease the risk of health services being overwhelmed, allowing for better treatment of active cases, and delaying additional cases until effective treatments or a vaccine become available.\\n\\n\\n=== Vaccine ===\\n\\n\\n=== Face masks and respiratory hygiene ===\\n\\nThe WHO and the US CDC recommend individuals wear non-medical face coverings in public settings where there is an increased risk of transmission and where social distancing measures are difficult to maintain. This recommendation is meant to reduce the spread of the disease by asymptomatic and pre-symptomatic individuals and is complementary to established preventive measures such as social distancing. Face coverings limit the volume and travel distance of expiratory droplets dispersed when talking, breathing, and coughing. A face covering without vents or holes will also filter out particles containing the virus from inhaled and exhaled air, reducing the chances of infection. But, if the mask include an exhalation valve, a wearer that is infected (maybe without having noticed that, and asymptomatic) would transmit the virus outwards through it, despite any certification they can have. So the masks with exhalation valve are not for the infected wearers, and are not reliable to stop the pandemic in a large scale. Many countries and local jurisdictions encourage or mandate the use of face masks or cloth face coverings by members of the public to limit the spread of the virus.Masks are also strongly recommended for those who may have been infected and those taking care of someone who may have the disease. When not wearing a mask, the CDC recommends covering the mouth and nose with a tissue when coughing or sneezing and recommends using the inside of the elbow if no tissue is available. Proper hand hygiene after any cough or sneeze is encouraged. Healthcare professionals interacting directly with people who have COVID‑19 are advised to use respirators at least as protective as NIOSH-certified N95 or equivalent, in addition to other personal protective equipment.\\n\\n\\n=== Indoor ventilation and avoiding crowded indoor spaces ===\\nThe CDC recommends that crowded indoor spaces should be avoided. When indoors, increasing the rate of air change, decreasing recirculation of air and increasing the use of outdoor air can reduce transmission. The WHO recommends ventilation and air filtration in public spaces to help clear out infectious aerosols.Exhaled respiratory particles can build-up within enclosed spaces with inadequate ventilation. The risk of COVID‑19 infection increases especially in spaces where people engage in physical exertion or raise their voice (e.g., exercising, shouting, singing) as this increases exhalation of respiratory droplets. Prolonged exposure to these conditions, typically more than 15 minutes, leads to higher risk of infection.Displacement ventilation with large natural inlets can move stale air directly to the exhaust in laminar flow while significantly reducing the concentration of droplets and particles. Passive ventilation reduces energy consumption and maintenance costs but may lack controllability and heat recovery. Displacement ventilation can also be achieved mechanically with higher energy and maintenance costs. The use of large ducts and openings helps to prevent mixing in closed environments. Recirculation and mixing should be avoided because recirculation prevents dilution of harmful particles and redistributes possibly contaminated air, and mixing increases the concentration and range of infectious particles and keeps larger particles in the air.\\n\\n\\n=== Hand-washing and hygiene ===\\n\\nThorough hand hygiene after any cough or sneeze is required. The WHO also recommends that individuals wash hands often with soap and water for at least twenty seconds, especially after going to the toilet or when hands are visibly dirty, before eating and after blowing one\\'s nose. When soap and water are not available, the CDC recommends using an alcohol-based hand sanitiser with at least 60% alcohol. For areas where commercial hand sanitisers are not readily available, the WHO provides two formulations for local production. In these formulations, the antimicrobial activity arises from ethanol or isopropanol. Hydrogen peroxide is used to help eliminate bacterial spores in the alcohol; it is \"not an active substance for hand antisepsis.\" Glycerol is added as a humectant.\\n\\n\\n=== Social distancing ===\\n\\nSocial distancing (also known as physical distancing) includes infection control actions intended to slow the spread of the disease by minimising close contact between individuals. Methods include quarantines; travel restrictions; and the closing of schools, workplaces, stadiums, theatres, or shopping centres. Individuals may apply social distancing methods by staying at home, limiting travel, avoiding crowded areas, using no-contact greetings, and physically distancing themselves from others. Many governments are now mandating or recommending social distancing in regions affected by the outbreak.Outbreaks have occurred in prisons due to crowding and an inability to enforce adequate social distancing. In the United States, the prisoner population is aging and many of them are at high risk for poor outcomes from COVID‑19 due to high rates of coexisting heart and lung disease, and poor access to high-quality healthcare.\\n\\n\\n=== Surface cleaning ===\\nAfter being expelled from the body, coronaviruses can survive on surfaces for hours to days. If a person touches the dirty surface, they may deposit the virus at the eyes, nose, or mouth where it can enter the body and cause infection. Evidence indicates that contact with infected surfaces is not the main driver of COVID‑19, leading to recommendations for optimised disinfection procedures to avoid issues such as the increase of antimicrobial resistance through the use of inappropriate cleaning products and processes. Deep cleaning and other surface sanitation has been criticized as hygiene theater, giving a false sense of security against something primarily spread through the air.The amount of time that the virus can survive depends significantly on the type of surface, the temperature, and the humidity. Coronaviruses die very quickly when exposed to the UV light in sunlight. Like other enveloped viruses, SARS-CoV-2 survives longest when the temperature is at room temperature or lower, and when the relative humidity is low (<50%).On many surfaces, including glass, some types of plastic, stainless steel, and skin, the virus can remain infective for several days indoors at room temperature, or even about a week under ideal conditions. On some surfaces, including cotton fabric and copper, the virus usually dies after a few hours. The virus dies faster on porous surfaces than on non-porous surfaces due to capillary action within pores and faster aerosol droplet evaporation. However, of the many surfaces tested, two with the longest survival times are N95 respirator masks and surgical masks, both of which are considered porous surfaces.The CDC says that in most situations, cleaning surfaces with soap or detergent, not disinfecting, is enough to reduce risk of transmission. The CDC recommends that if a COVID‑19 case is suspected or confirmed at a facility such as an office or day care, all areas such as offices, bathrooms, common areas, shared electronic equipment like tablets, touch screens, keyboards, remote controls, and ATM machines used by the ill persons should be disinfected. Surfaces may be decontaminated with 62–71 percent ethanol, 50–100 percent isopropanol, 0.1 percent sodium hypochlorite, 0.5 percent hydrogen peroxide, and 0.2–7.5 percent povidone-iodine. Other solutions, such as benzalkonium chloride and chlorhexidine gluconate, are less effective. Ultraviolet germicidal irradiation may also be used. A datasheet comprising the authorised substances to disinfection in the food industry (including suspension or surface tested, kind of surface, use dilution, disinfectant and inocuylum volumes) can be seen in the supplementary material of.\\n\\n\\n=== Self-isolation ===\\nSelf-isolation at home has been recommended for those diagnosed with COVID‑19 and those who suspect they have been infected. Health agencies have issued detailed instructions for proper self-isolation. Many governments have mandated or recommended self-quarantine for entire populations. The strongest self-quarantine instructions have been issued to those in high-risk groups. Those who may have been exposed to someone with COVID‑19 and those who have recently travelled to a country or region with the widespread transmission have been advised to self-quarantine for 14 days from the time of last possible exposure.\\n\\n\\n=== Healthy diet and lifestyle ===\\nThe Harvard T.H. Chan School of Public Health recommends a healthy diet, being physically active, managing psychological stress, and getting enough sleep.Consistently meeting scientific guidelines of 150+ minutes per week of exercise or similar physical activity was shown to be associated with a smaller risk of hospitalisation and death due to COVID‑19, even when considering likely risk factors such as elevated Body mass index (BMI).A meta-analysis, published online in October 2021, concluded that \"Vitamin D supplementation in SARS-CoV-2 positive patients has the potential to positively impact patients with both mild and severe symptoms.\" The largest observational study on the subject using online questionnaires, with over 6,000 participants and a dosage regime near the RDI, is set to conclude in July 2021. One of the collaborators in the study is Synergy Biologics Ltd, a manufacturer of vitamin D3 supplements.\\n\\n\\n=== International travel-related control measures ===\\nA 2021 Cochrane rapid review found that based upon low-certainty evidence, international travel-related control measures such as restricting cross-border travel may help to contain the spread of COVID‑19. Additionally, symptom/exposure-based screening measures at borders may miss many positive cases. While test-based border screening measures may be more effective, it could also miss many positive cases if only conducted upon arrival without follow-up. The review concluded that a minimum 10-day quarantine may be beneficial in preventing the spread of COVID‑19 and may be more effective if combined with an additional control measure like border screening.\\n\\n\\n== Treatment ==\\n\\n\\n== Prognosis ==\\n\\nThe severity of COVID‑19 varies. The disease may take a mild course with few or no symptoms, resembling other common upper respiratory diseases such as the common cold. In 3–4% of cases (7.4% for those over age 65) symptoms are severe enough to cause hospitalization. Mild cases typically recover within two weeks, while those with severe or critical diseases may take three to six weeks to recover. Among those who have died, the time from symptom onset to death has ranged from two to eight weeks. The Italian Istituto Superiore di Sanità reported that the median time between the onset of symptoms and death was twelve days, with seven being hospitalised. However, people transferred to an ICU had a median time of ten days between hospitalisation and death. Prolonged prothrombin time and elevated C-reactive protein levels on admission to the hospital are associated with severe course of COVID‑19 and with a transfer to ICU.Some early studies suggest 10% to 20% of people with COVID‑19 will experience symptoms lasting longer than a month. A majority of those who were admitted to hospital with severe disease report long-term problems including fatigue and shortness of breath. On 30 October 2020, WHO chief Tedros Adhanom warned that \"to a significant number of people, the COVID virus poses a range of serious long-term effects.\" He has described the vast spectrum of COVID‑19 symptoms that fluctuate over time as \"really concerning\". They range from fatigue, a cough and shortness of breath, to inflammation and injury of major organs – including the lungs and heart, and also neurological and psychologic effects. Symptoms often overlap and can affect any system in the body. Infected people have reported cyclical bouts of fatigue, headaches, months of complete exhaustion, mood swings, and other symptoms. Tedros therefore concluded that a strategy of achieving herd immunity by infection, rather than vaccination, is \"morally unconscionable and unfeasible\".In terms of hospital readmissions about 9% of 106,000 individuals had to return for hospital treatment within two months of discharge. The average to readmit was eight days since first hospital visit. There are several risk factors that have been identified as being a cause of multiple admissions to a hospital facility. Among these are advanced age (above 65 years of age) and presence of a chronic condition such as diabetes, COPD, heart failure or chronic kidney disease.According to scientific reviews smokers are more likely to require intensive care or die compared to non-smokers. Acting on the same ACE2 pulmonary receptors affected by smoking, air pollution has been correlated with the disease. Short term and chronic exposure to air pollution seems to enhance morbidity and mortality from COVID‑19. Pre-existing heart and lung diseases and also obesity, especially in conjunction with fatty liver disease, contributes to an increased health risk of COVID‑19.It is also assumed that those that are immunocompromised are at higher risk of getting severely sick from SARS-CoV-2. One research that looked into the COVID‑19 infections in hospitalized kidney transplant recipients found a mortality rate of 11%.Genetics also plays an important role in the ability to fight off the disease. For instance, those that do not produce detectable type I interferons or produce auto-antibodies against these may get much sicker from COVID‑19. Genetic screening is able to detect interferon effector genes.\\n\\n\\n=== Children ===\\n\\nWhile very young children have experienced lower rates of infection, older children have a rate of infection that is similar to the population as a whole. Children are likely to have milder symptoms and are at lower risk of severe disease than adults. The CDC reports that in the US roughly a third of hospitalized children were admitted to the ICU, while a European multinational study of hospitalized children from June 2020 found that about 8% of children admitted to a hospital needed intensive care. Four of the 582 children (0.7%) in the European study died, but the actual mortality rate may be \"substantially lower\" since milder cases that did not seek medical help were not included in the study.\\n\\n\\n=== Complications ===\\n\\nComplications may include pneumonia, acute respiratory distress syndrome (ARDS), multi-organ failure, septic shock, and death. Cardiovascular complications may include heart failure, arrhythmias (including atrial fibrillation), heart inflammation, and thrombosis, particularly venous thromboembolism. Approximately 20–30% of people who present with COVID‑19 have elevated liver enzymes, reflecting liver injury.Neurologic manifestations include seizure, stroke, encephalitis, and Guillain–Barré syndrome (which includes loss of motor functions). Following the infection, children may develop paediatric multisystem inflammatory syndrome, which has symptoms similar to Kawasaki disease, which can be fatal. In very rare cases, acute encephalopathy can occur, and it can be considered in those who have been diagnosed with COVID‑19 and have an altered mental status.In the case of pregnant women, it is important to note that, according to the Centers for Disease Control and Prevention, pregnant women are at increased risk of becoming seriously ill from COVID‑19. This is because pregnant women with COVID‑19 appear to be more likely to develop respiratory and obstetric complications that can lead to miscarriage, premature delivery and intrauterine growth restriction.Fungal infections such as aspergillosis, candidiasis, cryptococcosis and mucormycosis have been recorded in patients recovering from COVID‑19.\\n\\n\\n=== Longer-term effects ===\\n\\nSome early studies suggest that 10–20% of people with COVID‑19 will experience symptoms lasting longer than a month. A majority of those who were admitted to hospital with severe disease report long-term problems, including fatigue and shortness of breath. About 5–10% of patients admitted to hospital progress to severe or critical disease, including pneumonia and acute respiratory failure.By a variety of mechanisms, the lungs are the organs most affected in COVID‑19. In people requiring hospital admission, up to 98% of CT scans performed show lung abnormalities after 28 days of illness even if they had clinically improved.People with advanced age, severe disease, prolonged ICU stays, or who smoke are more likely to have long-lasting effects, including pulmonary fibrosis. Overall, approximately one-third of those investigated after four weeks will have findings of pulmonary fibrosis or reduced lung function as measured by DLCO, even in asymptomatic people, but with the suggestion of continuing improvement with the passing of more time.\\n\\n\\n=== Immunity ===\\n\\nThe immune response by humans to SARS-CoV-2 virus occurs as a combination of the cell-mediated immunity and antibody production, just as with most other infections. B cells interact with T cells and begin dividing before selection into the plasma cell, partly on the basis of their affinity for antigen. Since SARS-CoV-2 has been in the human population only since December 2019, it remains unknown if the immunity is long-lasting in people who recover from the disease. The presence of neutralizing antibodies in blood strongly correlates with protection from infection, but the level of neutralizing antibody declines with time. Those with asymptomatic or mild disease had undetectable levels of neutralizing antibody two months after infection. In another study, the level of neutralizing antibodies fell four-fold one to four months after the onset of symptoms. However, the lack of antibodies in the blood does not mean antibodies will not be rapidly produced upon reexposure to SARS-CoV-2. Memory B cells specific for the spike and nucleocapsid proteins of SARS-CoV-2 last for at least six months after the appearance of symptoms.Reinfection with COVID‑19 is possible but uncommon. The first case of reinfection was documented in August 2020. A systematic review found 17 cases of confirmed reinfection in medical literature as of May 2021.\\n\\n\\n== Mortality ==\\n\\nSeveral measures are commonly used to quantify mortality. These numbers vary by region and over time and are influenced by the volume of testing, healthcare system quality, treatment options, time since the initial outbreak, and population characteristics such as age, sex, and overall health.The mortality rate reflects the number of deaths within a specific demographic group divided by the population of that demographic group. Consequently, the mortality rate reflects the prevalence as well as the severity of the disease within a given population. Mortality rates are highly correlated to age, with relatively low rates for young people and relatively high rates among the elderly. In fact, one relevant factor of mortality rates is the age structure of the countries\\' populations. For example, the case fatality rate for COVID‑19 is lower in India than in the US since India\\'s younger population represents a larger percentage than in the US.\\n\\n\\n=== Case fatality rate ===\\nThe case fatality rate (CFR) reflects the number of deaths divided by the number of diagnosed cases within a given time interval. Based on Johns Hopkins University statistics, the global death-to-case ratio is 1.24% (6,183,197/500,280,791) as of 12 April 2022. The number varies by region.\\n\\n\\t\\t\\n\\t\\t\\n\\t\\t\\n\\n\\n=== Infection fatality rate ===\\nA key metric in gauging the severity of COVID‑19 is the infection fatality rate (IFR), also referred to as the infection fatality ratio or infection fatality risk. This metric is calculated by dividing the total number of deaths from the disease by the total number of infected individuals; hence, in contrast to the CFR, the IFR incorporates asymptomatic and undiagnosed infections as well as reported cases.\\n\\n\\n==== Estimates ====\\n\\nA December 2020 systematic review and meta-analysis estimated that population IFR during the first wave of the pandemic was about 0.5% to 1% in many locations (including France, Netherlands, New Zealand, and Portugal), 1% to 2% in other locations (Australia, England, Lithuania, and Spain), and exceeded 2% in Italy. That study also found that most of these differences in IFR reflected corresponding differences in the age composition of the population and age-specific infection rates; in particular, the metaregression estimate of IFR is very low for children and younger adults (e.g., 0.002% at age 10 and 0.01% at age 25) but increases progressively to 0.4% at age 55, 1.4% at age 65, 4.6% at age 75, and 15% at age 85. These results were also highlighted in a December 2020 report issued by the WHO.\\nAn analysis of those IFR rates indicates that COVID‑19 is hazardous not only for the elderly but also for middle-aged adults, for whom the infection fatality rate of COVID-19 is two orders of magnitude greater than the annualized risk of a fatal automobile accident and far more dangerous than seasonal influenza.\\n\\n\\n==== Earlier estimates of IFR ====\\nAt an early stage of the pandemic, the World Health Organization reported estimates of IFR between 0.3% and 1%. On 2 July, The WHO\\'s chief scientist reported that the average IFR estimate presented at a two-day WHO expert forum was about 0.6%. In August, the WHO found that studies incorporating data from broad serology testing in Europe showed IFR estimates converging at approximately 0.5–1%. Firm lower limits of IFRs have been established in a number of locations such as New York City and Bergamo in Italy since the IFR cannot be less than the population fatality rate. (After sufficient time however, people can get reinfected). As of 10 July, in New York City, with a population of 8.4 million, 23,377 individuals (18,758 confirmed and 4,619 probable) have died with COVID‑19 (0.3% of the population). Antibody testing in New York City suggested an IFR of ~0.9%, and ~1.4%. In Bergamo province, 0.6% of the population has died. In September 2020, the U.S. Centers for Disease Control and Prevention (CDC) reported preliminary estimates of age-specific IFRs for public health planning purposes.\\n\\n\\n=== Sex differences ===\\n\\nCOVID‑19 case fatality rates are higher among men than women in most countries. However, in a few countries like India, Nepal, Vietnam, and Slovenia the fatality cases are higher in women than men. Globally, men are more likely to be admitted to the ICU and more likely to die. One meta-analysis found that globally, men were more likely to get COVID‑19 than women; there were approximately 55 men and 45 women per 100 infections (CI: 51.43–56.58).The Chinese Center for Disease Control and Prevention reported the death rate was 2.8% for men and 1.7% for women. Later reviews in June 2020 indicated that there is no significant difference in susceptibility or in CFR between genders. One review acknowledges the different mortality rates in Chinese men, suggesting that it may be attributable to lifestyle choices such as smoking and drinking alcohol rather than genetic factors. Smoking, which in some countries like China is mainly a male activity, is a habit that contributes to increasing significantly the case fatality rates among men. Sex-based immunological differences, lesser prevalence of smoking in women and men developing co-morbid conditions such as hypertension at a younger age than women could have contributed to the higher mortality in men. In Europe as of February 2020, 57% of the infected people were men and 72% of those died with COVID‑19 were men. As of April 2020, the US government is not tracking sex-related data of COVID‑19 infections. Research has shown that viral illnesses like Ebola, HIV, influenza and SARS affect men and women differently.\\n\\n\\n=== Ethnic differences ===\\nIn the US, a greater proportion of deaths due to COVID‑19 have occurred among African Americans and other minority groups. Structural factors that prevent them from practicing social distancing include their concentration in crowded substandard housing and in \"essential\" occupations such as retail grocery workers, public transit employees, health-care workers and custodial staff. Greater prevalence of lacking health insurance and care of underlying conditions such as diabetes, hypertension, and heart disease also increase their risk of death. Similar issues affect Native American and Latino communities. On the one hand, in the Dominican Republic there is a clear example of both gender and ethnic inequality. In this Latin American territory, there is great inequality and precariousness that especially affects Dominican women, with greater emphasis on those of Haitian descent. According to a US health policy non-profit, 34% of American Indian and Alaska Native People (AIAN) non-elderly adults are at risk of serious illness compared to 21% of white non-elderly adults. The source attributes it to disproportionately high rates of many health conditions that may put them at higher risk as well as living conditions like lack of access to clean water.Leaders have called for efforts to research and address the disparities. In the UK, a greater proportion of deaths due to COVID‑19 have occurred in those of a Black, Asian, and other ethnic minority background. More severe impacts upon victims including the relative incidence of the necessity of hospitalization requirements, and vulnerability to the disease has been associated via DNA analysis to be expressed in genetic variants at chromosomal region 3, features that are associated with European Neanderthal heritage. That structure imposes greater risks that those affected will develop a more severe form of the disease. The findings are from Professor Svante Pääbo and researchers he leads at the Max Planck Institute for Evolutionary Anthropology and the Karolinska Institutet. This admixture of modern human and Neanderthal genes is estimated to have occurred roughly between 50,000 and 60,000 years ago in Southern Europe.\\n\\n\\n=== Comorbidities ===\\nBiological factors (immune response) and the general behaviour (habits) can strongly determine the consequences of COVID‑19. Most of those who die of COVID‑19 have pre-existing (underlying) conditions, including hypertension, diabetes mellitus, and cardiovascular disease. According to March data from the United States, 89% of those hospitalised had preexisting conditions. The Italian Istituto Superiore di Sanità reported that out of 8.8% of deaths where medical charts were available, 96.1% of people had at least one comorbidity with the average person having 3.4 diseases. According to this report the most common comorbidities are hypertension (66% of deaths), type 2 diabetes (29.8% of deaths), Ischemic Heart Disease (27.6% of deaths), atrial fibrillation (23.1% of deaths) and chronic renal failure (20.2% of deaths).\\nMost critical respiratory comorbidities according to the Centers for Disease Control and Prevention (CDC), are: moderate or severe asthma, pre-existing COPD, pulmonary fibrosis, cystic fibrosis. Evidence stemming from meta-analysis of several smaller research papers also suggests that smoking can be associated with worse outcomes. When someone with existing respiratory problems is infected with COVID‑19, they might be at greater risk for severe symptoms. COVID‑19 also poses a greater risk to people who misuse opioids and methamphetamines, insofar as their drug use may have caused lung damage.In August 2020, the CDC issued a caution that tuberculosis (TB) infections could increase the risk of severe illness or death. The WHO recommended that people with respiratory symptoms be screened for both diseases, as testing positive for COVID‑19 could not rule out co-infections. Some projections have estimated that reduced TB detection due to the pandemic could result in 6.3 million additional TB cases and 1.4 million TB-related deaths by 2025.\\n\\n\\n== History ==\\n\\nThe virus is thought to be of natural animal origin, most likely through spillover infection. There are several theories about where the index case originated and investigations into the origin of the pandemic are ongoing. Phylogenetics estimates that SARS-CoV-2 arose in October or November 2019. A phylogenetic algorithm analysis suggested that the virus may have been circulating in Guangdong before Wuhan. Evidence suggests that it descends from a coronavirus that infects wild bats, and spread to humans through an intermediary wildlife host. The possibility that the virus was accidentally released from a laboratory is also under increasingly active consideration. U.S intelligence agencies found that the virus was not developed as a biological weapon and that it is unlikely for it to have been genetically engineered.The first confirmed human infections were in Wuhan, Hubei, China. A study of the first 41 cases of confirmed COVID‑19, published in January 2020 in The Lancet, reported the earliest date of onset of symptoms as 1 December 2019. Official publications from the WHO reported the earliest onset of symptoms as 8 December 2019. Human-to-human transmission was confirmed by the WHO and Chinese authorities by 20 January 2020. According to official Chinese sources, these were mostly linked to the Huanan Seafood Wholesale Market, which also sold live animals. In May 2020, George Gao, the director of the CDC, said animal samples collected from the seafood market had tested negative for the virus, indicating that the market was the site of an early superspreading event, but that it was not the site of the initial outbreak. Traces of the virus have been found in wastewater samples that were collected in Milan and Turin, Italy, on 18 December 2019.By December 2019, the spread of infection was almost entirely driven by human-to-human transmission. The number of COVID-19 cases in Hubei gradually increased, reaching sixty by 20 December, and at least 266 by 31 December. On 24 December, Wuhan Central Hospital sent a bronchoalveolar lavage fluid (BAL) sample from an unresolved clinical case to sequencing company Vision Medicals. On 27 and 28 December, Vision Medicals informed the Wuhan Central Hospital and the Chinese CDC of the results of the test, showing a new coronavirus. A pneumonia cluster of unknown cause was observed on 26 December and treated by the doctor Zhang Jixian in Hubei Provincial Hospital, who informed the Wuhan Jianghan CDC on 27 December. On 30 December, a test report addressed to Wuhan Central Hospital, from company CapitalBio Medlab, stated an erroneous positive result for SARS, causing a group of doctors at Wuhan Central Hospital to alert their colleagues and relevant hospital authorities of the result. The Wuhan Municipal Health Commission issued a notice to various medical institutions on \"the treatment of pneumonia of unknown cause\" that same evening. Eight of these doctors, including Li Wenliang (punished on 3 January), were later admonished by the police for spreading false rumours and another, Ai Fen, was reprimanded by her superiors for raising the alarm.The Wuhan Municipal Health Commission made the first public announcement of a pneumonia outbreak of unknown cause on 31 December, confirming 27 cases – enough to trigger an investigation.During the early stages of the outbreak, the number of cases doubled approximately every seven and a half days. In early and mid-January 2020, the virus spread to other Chinese provinces, helped by the Chinese New Year migration and Wuhan being a transport hub and major rail interchange. On 20 January, China reported nearly 140 new cases in one day, including two people in Beijing and one in Shenzhen. Later official data shows 6,174 people had already developed symptoms by then, and more may have been infected. A report in The Lancet on 24 January indicated human transmission, strongly recommended personal protective equipment for health workers, and said testing for the virus was essential due to its \"pandemic potential\". On 30 January, the WHO declared COVID-19 a Public Health Emergency of International Concern. By this time, the outbreak spread by a factor of 100 to 200 times.Italy had its first confirmed cases on 31 January 2020, two tourists from China. Italy overtook China as the country with the most deaths on 19 March 2020. By 26 March the United States had overtaken China and Italy with the highest number of confirmed cases in the world. Research on coronavirus genomes indicates the majority of COVID-19 cases in New York came from European travellers, rather than directly from China or any other Asian country. Retesting of prior samples found a person in France who had the virus on 27 December 2019, and a person in the United States who died from the disease on 6 February 2020.RT-PCR testing of untreated wastewater samples from Brazil and Italy have suggested detection of SARS-CoV-2 as early as November and December 2019, respectively, but the methods of such sewage studies have not been optimised, many have not been peer-reviewed, details are often missing, and there is a risk of false positives due to contamination or if only one gene target is detected. A September 2020 review journal article said, \"The possibility that the COVID‑19 infection had already spread to Europe at the end of last year is now indicated by abundant, even if partially circumstantial, evidence,\" including pneumonia case numbers and radiology in France and Italy in November and December.As of 1 October 2021, Reuters reported that it had estimated the worldwide total number of deaths due to COVID‑19 to have exceeded five million.\\n\\n\\n== Misinformation ==\\n\\nAfter the initial outbreak of COVID‑19, misinformation and disinformation regarding the origin, scale, prevention, treatment, and other aspects of the disease rapidly spread online.In September 2020, the US Centers for Disease Control and Prevention (CDC) published preliminary estimates of the risk of death by age groups in the United States, but those estimates were widely misreported and misunderstood.\\n\\n\\n== Other species ==\\n\\nHumans appear to be capable of spreading the virus to some other animals, a type of disease transmission referred to as zooanthroponosis.\\nSome pets, especially cats and ferrets, can catch this virus from infected humans. Symptoms in cats include respiratory (such as a cough) and digestive symptoms. Cats can spread the virus to other cats, and may be able to spread the virus to humans, but cat-to-human transmission of SARS-CoV-2 has not been proven. Compared to cats, dogs are less susceptible to this infection. Behaviors which increase the risk of transmission include kissing, licking, and petting the animal.The virus does not appear to be able to infect pigs, ducks, or chickens at all. Mice, rats, and rabbits, if they can be infected at all, are unlikely to be involved in spreading the virus.Tigers and lions in zoos have become infected as a result of contact with infected humans. As expected, monkeys and great ape species such as orangutans can also be infected with the COVID‑19 virus.Minks, which are in the same family as ferrets, have been infected. Minks may be asymptomatic, and can also spread the virus to humans. Multiple countries have identified infected animals in mink farms. Denmark, a major producer of mink pelts, ordered the slaughter of all minks over fears of viral mutations, following an outbreak referred to as Cluster 5. A vaccine for mink and other animals is being researched.\\n\\n\\n== Research ==\\n\\nInternational research on vaccines and medicines in COVID‑19 is underway by government organisations, academic groups, and industry researchers. The CDC has classified it to require a BSL3 grade laboratory. There has been a great deal of COVID‑19 research, involving accelerated research processes and publishing shortcuts to meet the global demand.As of December 2020, hundreds of clinical trials have been undertaken, with research happening on every continent except Antarctica. As of November 2020, more than 200 possible treatments had been studied in humans so far.\\n\\n\\n=== Transmission and prevention research ===\\n\\nModelling research has been conducted with several objectives, including predictions of the dynamics of transmission, diagnosis and prognosis of infection, estimation of the impact of interventions, or allocation of resources. Modelling studies are mostly based on compartmental models in epidemiology, estimating the number of infected people over time under given conditions. Several other types of models have been developed and used during the COVID‑19 including computational fluid dynamics models to study the flow physics of COVID‑19, retrofits of crowd movement models to study occupant exposure, mobility-data based models to investigate transmission, or the use of macroeconomic models to assess the economic impact of the pandemic. Further, conceptual frameworks from crisis management research have been applied to better understand the effects of COVID‑19 on organizations worldwide.\\n\\n\\n=== Treatment-related research ===\\n\\nRepurposed antiviral drugs make up most of the research into COVID‑19 treatments. Other candidates in trials include vasodilators, corticosteroids, immune therapies, lipoic acid, bevacizumab, and recombinant angiotensin-converting enzyme 2.In March 2020, the World Health Organization (WHO) initiated the Solidarity trial to assess the treatment effects of some promising drugs: an experimental drug called remdesivir; anti-malarial drugs chloroquine and hydroxychloroquine; two anti-HIV drugs, lopinavir/ritonavir; and interferon-beta. More than 300 active clinical trials are underway as of April 2020.Research on the antimalarial drugs hydroxychloroquine and chloroquine showed that they were ineffective at best, and that they may reduce the antiviral activity of remdesivir. By May 2020, France, Italy, and Belgium had banned the use of hydroxychloroquine as a COVID‑19 treatment.In June, initial results from the randomised RECOVERY Trial in the United Kingdom showed that dexamethasone reduced mortality by one third for people who are critically ill on ventilators and one fifth for those receiving supplemental oxygen. Because this is a well-tested and widely available treatment, it was welcomed by the WHO, which is in the process of updating treatment guidelines to include dexamethasone and other steroids. Based on those preliminary results, dexamethasone treatment has been recommended by the NIH for patients with COVID‑19 who are mechanically ventilated or who require supplemental oxygen but not in patients with COVID‑19 who do not require supplemental oxygen.In September 2020, the WHO released updated guidance on using corticosteroids for COVID‑19. The WHO recommends systemic corticosteroids rather than no systemic corticosteroids for the treatment of people with severe and critical COVID‑19 (strong recommendation, based on moderate certainty evidence). The WHO suggests not to use corticosteroids in the treatment of people with non-severe COVID‑19 (conditional recommendation, based on low certainty evidence). The updated guidance was based on a meta-analysis of clinical trials of critically ill COVID‑19 patients.In September 2020, the European Medicines Agency (EMA) endorsed the use of dexamethasone in adults and adolescents from twelve years of age and weighing at least 40 kilograms (88 lb) who require supplemental oxygen therapy. Dexamethasone can be taken by mouth or given as an injection or infusion (drip) into a vein.In November 2020, the US Food and Drug Administration (FDA) issued an emergency use authorization for the investigational monoclonal antibody therapy bamlanivimab for the treatment of mild-to-moderate COVID‑19. Bamlanivimab is authorized for people with positive results of direct SARS-CoV-2 viral testing who are twelve years of age and older weighing at least 40 kilograms (88 lb), and who are at high risk for progressing to severe COVID‑19 or hospitalization. This includes those who are 65 years of age or older, or who have chronic medical conditions.In February 2021, the FDA issued an emergency use authorization (EUA) for bamlanivimab and etesevimab administered together for the treatment of mild to moderate COVID‑19 in people twelve years of age or older weighing at least 40 kilograms (88 lb) who test positive for SARS‑CoV‑2 and who are at high risk for progressing to severe COVID‑19. The authorized use includes treatment for those who are 65 years of age or older or who have certain chronic medical conditions.In April 2021, the FDA revoked the emergency use authorization (EUA) that allowed for the investigational monoclonal antibody therapy bamlanivimab, when administered alone, to be used for the treatment of mild-to-moderate COVID‑19 in adults and certain pediatric patients.\\n\\n\\n==== Cytokine storm ====\\n\\nA cytokine storm can be a complication in the later stages of severe COVID‑19. A cytokine storm is a potentially deadly immune reaction where a large amount of pro-inflammatory cytokines and chemokines are released too quickly. A cytokine storm can lead to ARDS and multiple organ failure. Data collected from Jin Yin-tan Hospital in Wuhan, China indicates that patients who had more severe responses to COVID‑19 had greater amounts of pro-inflammatory cytokines and chemokines in their system than patients who had milder responses. These high levels of pro-inflammatory cytokines and chemokines indicate presence of a cytokine storm.Tocilizumab has been included in treatment guidelines by China\\'s National Health Commission after a small study was completed. It is undergoing a Phase II non-randomised trial at the national level in Italy after showing positive results in people with severe disease. Combined with a serum ferritin blood test to identify a cytokine storm (also called cytokine storm syndrome, not to be confused with cytokine release syndrome), it is meant to counter such developments, which are thought to be the cause of death in some affected people. The interleukin-6 receptor (IL-6R) antagonist was approved by the FDA to undergo a Phase III clinical trial assessing its effectiveness on COVID‑19 based on retrospective case studies for the treatment of steroid-refractory cytokine release syndrome induced by a different cause, CAR T cell therapy, in 2017. There is no randomised, controlled evidence that tocilizumab is an efficacious treatment for CRS. Prophylactic tocilizumab has been shown to increase serum IL-6 levels by saturating the IL-6R, driving IL-6 across the blood-brain barrier, and exacerbating neurotoxicity while having no effect on the incidence of CRS.Lenzilumab, an anti-GM-CSF monoclonal antibody, is protective in murine models for CAR T cell-induced CRS and neurotoxicity and is a viable therapeutic option due to the observed increase of pathogenic GM-CSF secreting T cells in hospitalised patients with COVID‑19.\\n\\n\\n==== Passive antibodies ====\\n\\nTransferring purified and concentrated antibodies produced by the immune systems of those who have recovered from COVID‑19 to people who need them is being investigated as a non-vaccine method of passive immunisation. Viral neutralization is the anticipated mechanism of action by which passive antibody therapy can mediate defence against SARS-CoV-2. The spike protein of SARS-CoV-2 is the primary target for neutralizing antibodies. As of 8 August 2020, eight neutralizing antibodies targeting the spike protein of SARS-CoV-2 have entered clinical studies. It has been proposed that selection of broad-neutralizing antibodies against SARS-CoV-2 and SARS-CoV might be useful for treating not only COVID‑19 but also future SARS-related CoV infections. Other mechanisms, however, such as antibody-dependent cellular cytotoxicity or phagocytosis, may be possible. Other forms of passive antibody therapy, for example, using manufactured monoclonal antibodies, are in development.The use of passive antibodies to treat people with active COVID‑19 is also being studied. This involves the production of convalescent serum, which consists of the liquid portion of the blood from people who recovered from the infection and contains antibodies specific to this virus, which is then administered to active patients. This strategy was tried for SARS with inconclusive results. An updated Cochrane review in May 2021 found high certainty evidence that, for the treatment of people with moderate to severe COVID‑19, convalescent plasma did not reduce mortality or bring about symptom improvement. There continues to be uncertainty about the safety of convalescent plasma administration to people with COVID‑19 and differing outcomes measured in different studies limits their use in determining efficacy.\\n\\n\\n=== Bioethics ===\\nSince the outbreak of the COVID‑19 pandemic, scholars have explored the bioethics, normative economics, and political theories of healthcare policies related to the public health crisis. Academics have pointed to the moral distress of healthcare workers, ethics of distributing scarce healthcare resources such as ventilators, and the global justice of vaccine diplomacies. The socio-economic inequalities between genders, races, groups with disabilities, communities, regions, countries, and continents have also drawn attention in academia and the general public.\\n\\n\\n== Effect on other diseases and the pharmacy trade ==\\nThere was a report on 3 March 2021, that social distancing and common wearing of surgical masks and similar as a common precaution against COVID‑19 caused a drop in the spread rate of the common cold and flu.\\n\\n\\n== See also ==\\nCoronavirus diseases, a group of closely related syndromes\\nDisease X, a WHO term\\nLaw of declining virulence – Claim by epidemiologist Theobald Smith\\nTheory of virulence – Theory by biologist Paul W. Ewald\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\n\\n\\n== External links ==\\n\\n\\n=== Health agencies ===\\nCoronavirus disease (COVID‑19) Facts by the World Health Organization (WHO)\\nCoronavirus 2019 (COVID-19) by the US Centers for Disease Control and Prevention (CDC)\\nCoronavirus (COVID‑19) by the UK National Health Service (NHS)\\n\\n\\n=== Directories ===\\nCoronavirus Resource Center at the Center for Inquiry\\nCOVID-19 at Curlie\\nCOVID‑19 Resource Directory on OpenMD\\nCOVID‑19 Information on FireMountain.net\\n\\n\\n=== Medical journals ===\\nCoronavirus Disease 2019 (COVID‑19) by JAMA\\nBMJ\\'s Coronavirus (covid‑19) Hub by the BMJ\\nNovel Coronavirus Information Center by Elsevier\\nCOVID‑19 Resource Centre by The Lancet\\nCoronavirus (COVID‑19) Research Highlights by Springer Nature\\nCoronavirus (Covid‑19) by The New England Journal of Medicine\\nCovid‑19: Novel Coronavirus by Wiley Publishing\\n\\n\\n=== Treatment guidelines ===\\n\"JHMI Clinical Recommendations for Available Pharmacologic Therapies for COVID-19\" (PDF). Johns Hopkins Medicine.\\n\"Bouncing Back From COVID-19: Your Guide to Restoring Movement\" (PDF). Johns Hopkins Medicine.\\n\"Guidelines on the Treatment and Management of Patients with COVID-19\". Infectious Diseases Society of America.\\n\"Coronavirus Disease 2019 (COVID-19) Treatment Guidelines\" (PDF). National Institutes of Health.\\nWorld Health Organization (2022). Therapeutics and COVID-19: living guideline, 14 January 2022 (Report). hdl:10665/351006. WHO/2019-nCoV/therapeutics/2022.1.\\nNHS England and NHS Improvement. National Guidance for post-COVID syndrome assessment clinics (Report).',\n",
              " 'Coronaviruses are a group of related RNA viruses that cause diseases in mammals and birds. In humans and birds, they cause respiratory tract infections that can range from mild to lethal. Mild illnesses in humans include some cases of the common cold (which is also caused by other viruses, predominantly rhinoviruses), while more lethal varieties can cause SARS, MERS and COVID-19, which is causing an ongoing pandemic. In cows and pigs they cause diarrhea, while in mice they cause hepatitis and encephalomyelitis.\\nCoronaviruses constitute the subfamily Orthocoronavirinae, in the family Coronaviridae, order Nidovirales and realm Riboviria. They are enveloped viruses with a positive-sense single-stranded RNA genome and a nucleocapsid of helical symmetry. The genome size of coronaviruses ranges from approximately 26 to 32 kilobases, one of the largest among RNA viruses. They have characteristic club-shaped spikes that project from their surface, which in electron micrographs create an image reminiscent of the solar corona, from which their name derives.\\n\\n\\n== Etymology ==\\nThe name \"coronavirus\" is derived from Latin corona, meaning \"crown\" or \"wreath\", itself a borrowing from Greek κορώνη korṓnē, \"garland, wreath\". The name was coined by June Almeida and David Tyrrell who first observed and studied human coronaviruses. The word was first used in print in 1968 by an informal group of virologists in the journal Nature to designate the new family of viruses. The name refers to the characteristic appearance of virions (the infective form of the virus) by electron microscopy, which have a fringe of large, bulbous surface projections creating an image reminiscent of the solar corona or halo. This morphology is created by the viral spike peplomers, which are proteins on the surface of the virus.The scientific name Coronavirus was accepted as a genus name by the International Committee for the Nomenclature of Viruses (later renamed International Committee on Taxonomy of Viruses) in 1971. As the number of new species increased, the genus was split into four genera, namely Alphacoronavirus, Betacoronavirus, Deltacoronavirus, and Gammacoronavirus in 2009. The common name coronavirus is used to refer to any member of the subfamily Orthocoronavirinae. As of 2020, 45 species are officially recognised.\\n\\n\\n== History ==\\n\\nThe earliest reports of a coronavirus infection in animals occurred in the late 1920s, when an acute respiratory infection of domesticated chickens emerged in North America. Arthur Schalk and M.C. Hawn in 1931 made the first detailed report which described a new respiratory infection of chickens in North Dakota. The infection of new-born chicks was characterized by gasping and listlessness with high mortality rates of 40–90%. Leland David Bushnell and Carl Alfred Brandly isolated the virus that caused the infection in 1933. The virus was then known as infectious bronchitis virus (IBV). Charles D. Hudson and Fred Robert Beaudette cultivated the virus for the first time in 1937. The specimen came to be known as the Beaudette strain. In the late 1940s, two more animal coronaviruses, JHM that causes brain disease (murine encephalitis) and mouse hepatitis virus (MHV) that causes hepatitis in mice were discovered. It was not realized at the time that these three different viruses were related.Human coronaviruses were discovered in the 1960s using two different methods in the United Kingdom and the United States. E.C. Kendall, Malcolm Bynoe, and David Tyrrell working at the Common Cold Unit of the British Medical Research Council collected a unique common cold virus designated B814 in 1961. The virus could not be cultivated using standard techniques which had successfully cultivated rhinoviruses, adenoviruses and other known common cold viruses. In 1965, Tyrrell and Bynoe successfully cultivated the novel virus by serially passing it through organ culture of human embryonic trachea. The new cultivating method was introduced to the lab by Bertil Hoorn. The isolated virus when intranasally inoculated into volunteers caused a cold and was inactivated by ether which indicated it had a lipid envelope. Dorothy Hamre and John Procknow at the University of Chicago isolated a novel cold from medical students in 1962. They isolated and grew the virus in kidney tissue culture, designating it 229E. The novel virus caused a cold in volunteers and, like B814, was inactivated by ether.\\n\\nScottish virologist June Almeida at St Thomas\\' Hospital in London, collaborating with Tyrrell, compared the structures of IBV, B814 and 229E in 1967. Using electron microscopy the three viruses were shown to be morphologically related by their general shape and distinctive club-like spikes. A research group at the National Institute of Health the same year was able to isolate another member of this new group of viruses using organ culture and named one of the samples OC43 (OC for organ culture). Like B814, 229E, and IBV, the novel cold virus OC43 had distinctive club-like spikes when observed with the electron microscope.The IBV-like novel cold viruses were soon shown to be also morphologically related to the mouse hepatitis virus. This new group of viruses were named coronaviruses after their distinctive morphological appearance. Human coronavirus 229E and human coronavirus OC43 continued to be studied in subsequent decades. The coronavirus strain B814 was lost. It is not known which present human coronavirus it was. Other human coronaviruses have since been identified, including SARS-CoV in 2003, HCoV NL63 in 2003, HCoV HKU1 in 2004, MERS-CoV in 2013, and SARS-CoV-2 in 2019. There have also been a large number of animal coronaviruses identified since the 1960s.\\n\\n\\n== Microbiology ==\\n\\n\\n=== Structure ===\\n\\nCoronaviruses are large, roughly spherical particles with unique surface projections. Their size is highly variable with average diameters of 80 to 120 nm. Extreme sizes are known from 50 to 200 nm in diameter. The total molecular mass is on average 40,000 kDa. They are enclosed in an envelope embedded with a number of protein molecules. The lipid bilayer envelope, membrane proteins, and nucleocapsid protect the virus when it is outside the host cell.The viral envelope is made up of a lipid bilayer in which the membrane (M), envelope (E) and spike (S) structural proteins are anchored. The molar ratio of E:S:M in the lipid bilayer is approximately 1:20:300.  The E and M protein are the structural proteins that combined with the lipid bilayer to shape the viral envelope and maintain its size. S proteins are needed for interaction with the host cells. But human coronavirus NL63 is peculiar in that its M protein has the binding site for the host cell, and not its S protein. The diameter of the envelope is 85 nm. The envelope of the virus in electron micrographs appears as a distinct pair of electron-dense shells (shells that are relatively opaque to the electron beam used to scan the virus particle).The M protein is the main structural protein of the envelope that provides the overall shape and is a type III membrane protein. It consists of 218 to 263 Amino acid residues and forms a layer 7.8 nm thick. It has three domains, a short N-terminal ectodomain, a triple-spanning transmembrane domain, and a C-terminal endodomain. The C-terminal domain forms a matrix-like lattice that adds to the extra-thickness of the envelope. Different species can have either N- or O-linked glycans in their protein amino-terminal domain. The M protein is crucial during the assembly, budding, envelope formation, and pathogenesis stages of the virus lifecycle.The E proteins are minor structural proteins and highly variable in different species. There are only about 20 copies of the E protein molecule in a coronavirus particle. They are 8.4 to 12 kDa in size and are composed of 76 to 109 amino acids. They are integral proteins (i.e. embedded in the lipid layer) and have two domains namely a transmembrane domain and an extramembrane C-terminal domain. They are almost fully α-helical, with a single α-helical transmembrane domain, and form pentameric (five-molecular) ion channels in the lipid bilayer. They are responsible for virion assembly, intracellular trafficking and morphogenesis (budding).\\n\\nThe spikes are the most distinguishing feature of coronaviruses and are responsible for the corona- or halo-like surface. On average a coronavirus particle has 74 surface spikes. Each spike is about 20 nm long and is composed of a trimer of the S protein. The S protein is in turn composed of an S1 and S2 subunit. The homotrimeric S protein is a class I fusion protein which mediates the receptor binding and membrane fusion between the virus and host cell. The S1 subunit forms the head of the spike and has the receptor-binding domain (RBD). The S2 subunit forms the stem which anchors the spike in the viral envelope and on protease activation enables fusion. The two subunits remain noncovalently linked as they are exposed on the viral surface until they attach to the host cell membrane. In a functionally active state, three S1 are attached to two S2 subunits. The subunit complex is split into individual subunits when the virus binds and fuses with the host cell under the action of proteases such as cathepsin family and transmembrane protease serine 2 (TMPRSS2) of the host cell.\\n\\nS1 proteins are the most critical components in terms of infection. They are also the most variable components as they are responsible for host cell specificity. They possess two major domains named N-terminal domain (S1-NTD) and C-terminal domain (S1-CTD), both of which serve as the receptor-binding domains. The NTDs recognize and bind sugars on the surface of the host cell. An exception is the MHV NTD that binds to a protein receptor carcinoembryonic antigen-related cell adhesion molecule 1 (CEACAM1). S1-CTDs are responsible for recognizing different protein receptors such as angiotensin-converting enzyme 2 (ACE2), aminopeptidase N (APN), and dipeptidyl peptidase 4 (DPP4).A subset of coronaviruses (specifically the members of betacoronavirus subgroup A) also has a shorter spike-like surface protein called hemagglutinin esterase (HE). The HE proteins occur as homodimers composed of about 400 amino acid residues and are 40 to 50 kDa in size. They appear as tiny surface projections of 5 to 7 nm long embedded in between the spikes. They help in the attachment to and detachment from the host cell.Inside the envelope, there is the nucleocapsid, which is formed from multiple copies of the nucleocapsid (N) protein, which are bound to the positive-sense single-stranded RNA genome in a continuous Beads-on-a-string type conformation. N protein is a phosphoprotein of 43 to 50 kDa in size, and is divided into three conserved domains. The majority of the protein is made up of domains 1 and 2, which are typically rich in arginines and lysines. Domain 3 has a short carboxy terminal end and has a net negative charge due to excess of acidic over basic amino acid residues.\\n\\n\\n=== Genome ===\\n\\nCoronaviruses contain a positive-sense, single-stranded RNA genome. The genome size for coronaviruses ranges from 26.4 to 31.7 kilobases. The genome size is one of the largest among RNA viruses. The genome has a 5′ methylated cap and a 3′ polyadenylated tail.The genome organization for a coronavirus is 5′-leader-UTR-replicase (ORF1ab)-spike (S)-envelope (E)-membrane (M)-nucleocapsid (N)-3′UTR-poly (A) tail. The open reading frames 1a and 1b, which occupy the first two-thirds of the genome, encode the replicase polyprotein (pp1ab). The replicase polyprotein self cleaves to form 16 nonstructural proteins (nsp1–nsp16).The later reading frames encode the four major structural proteins: spike, envelope, membrane, and nucleocapsid. Interspersed between these reading frames are the reading frames for the accessory proteins. The number of accessory proteins and their function is unique depending on the specific coronavirus.\\n\\n\\n=== Replication cycle ===\\n\\n\\n==== Cell entry ====\\n\\nInfection begins when the viral spike protein attaches to its complementary host cell receptor. After attachment, a protease of the host cell cleaves and activates the receptor-attached spike protein. Depending on the host cell protease available, cleavage and activation allows the virus to enter the host cell by endocytosis or direct fusion of the viral envelope with the host membrane.\\n\\n\\n==== Genome translation ====\\nOn entry into the host cell, the virus particle is uncoated, and its genome enters the cell cytoplasm. The coronavirus RNA genome has a 5′ methylated cap and a 3′ polyadenylated tail, which allows it to act like a messenger RNA and be directly translated by the host cell\\'s ribosomes. The host ribosomes translate the initial overlapping open reading frames ORF1a and ORF1b of the virus genome into two large overlapping polyproteins, pp1a and pp1ab.The larger polyprotein pp1ab is a result of a -1 ribosomal frameshift caused by a slippery sequence (UUUAAAC) and a downstream RNA pseudoknot at the end of open reading frame ORF1a. The ribosomal frameshift allows for the continuous translation of ORF1a followed by ORF1b.The polyproteins have their own proteases,  PLpro (nsp3) and 3CLpro (nsp5), which cleave the polyproteins at different specific sites. The cleavage of polyprotein pp1ab yields 16 nonstructural proteins (nsp1 to nsp16). Product proteins include various replication  proteins such as RNA-dependent RNA polymerase (nsp12), RNA helicase (nsp13), and exoribonuclease (nsp14).\\n\\n\\n==== Replicase-transcriptase ====\\n\\nA number of the nonstructural proteins coalesce to form a multi-protein replicase-transcriptase complex (RTC). The main replicase-transcriptase protein is the RNA-dependent RNA polymerase (RdRp). It is directly involved in the replication and transcription of RNA from an RNA strand. The other nonstructural proteins in the complex assist in the replication and transcription process. The exoribonuclease nonstructural protein, for instance, provides extra fidelity to replication by providing a proofreading function which the RNA-dependent RNA polymerase lacks.Replication – One of the main functions of the complex is to replicate the viral genome. RdRp directly mediates the synthesis of negative-sense genomic RNA from the positive-sense genomic RNA. This is followed by the replication of positive-sense genomic RNA from the negative-sense genomic RNA.\\n\\nTranscription – The other important function of the complex is to transcribe the viral genome. RdRp directly mediates the synthesis of negative-sense subgenomic RNA molecules from the positive-sense genomic RNA. This process is followed by the transcription of these negative-sense subgenomic RNA molecules to their corresponding positive-sense mRNAs. The subgenomic mRNAs form a \"nested set\" which have a common 5\\'-head and partially duplicate 3\\'-end.Recombination – The replicase-transcriptase complex is also capable of genetic recombination when at least two viral genomes are present in the same infected cell.  RNA recombination appears to be a major driving force in determining genetic variability within a coronavirus species, the capability of a coronavirus species to jump from one host to another and, infrequently, in determining the emergence of novel coronaviruses.  The exact mechanism of recombination in coronaviruses is unclear, but likely involves template switching during genome replication.\\n\\n\\n==== Assembly and release ====\\nThe replicated positive-sense genomic RNA becomes the genome of the progeny viruses. The mRNAs are gene transcripts of the last third of the virus genome after the initial overlapping reading frame. These mRNAs are translated by the host\\'s ribosomes into the structural proteins and many accessory proteins. RNA translation occurs inside the endoplasmic reticulum. The viral structural proteins S, E, and M move along the secretory pathway into the Golgi intermediate compartment. There, the M proteins direct most protein-protein interactions required for the assembly of viruses following its binding to the nucleocapsid. Progeny viruses are then released from the host cell by exocytosis through secretory vesicles. Once released the viruses can infect other host cells.\\n\\n\\n=== Transmission ===\\nInfected carriers are able to shed viruses into the environment. The interaction of the coronavirus spike protein with its complementary cell receptor is central in determining the tissue tropism, infectivity, and species range of the released virus. Coronaviruses mainly target epithelial cells. They are transmitted from one host to another host, depending on the coronavirus species, by either an aerosol, fomite, or fecal-oral route.Human coronaviruses infect the epithelial cells of the respiratory tract, while animal coronaviruses generally infect the epithelial cells of the digestive tract. SARS coronavirus, for example, infects the human epithelial cells of the lungs via an aerosol route by binding to the angiotensin-converting enzyme 2 (ACE2) receptor. Transmissible gastroenteritis coronavirus (TGEV) infects the pig epithelial cells of the digestive tract via a fecal-oral route by binding to the Alanine aminopeptidase (APN) receptor.\\n\\n\\n== Classification ==\\n\\nCoronaviruses form the subfamily Orthocoronavirinae, which is one of two sub-families in the family Coronaviridae, order Nidovirales, and realm Riboviria. They are divided into the four genera: Alphacoronavirus, Betacoronavirus, Gammacoronavirus and Deltacoronavirus. Alphacoronaviruses and betacoronaviruses infect mammals, while gammacoronaviruses and deltacoronaviruses primarily infect birds.\\n\\nGenus: Alphacoronavirus;Species: Alphacoronavirus 1 (TGEV, Feline coronavirus, Canine coronavirus), Human coronavirus 229E, Human coronavirus NL63, Miniopterus bat coronavirus 1, Miniopterus bat coronavirus HKU8, Porcine epidemic diarrhea virus, Rhinolophus bat coronavirus HKU2, Scotophilus bat coronavirus 512\\nGenus Betacoronavirus;Species: Betacoronavirus 1 (Bovine Coronavirus, Human coronavirus OC43), Hedgehog coronavirus 1, Human coronavirus HKU1, Middle East respiratory syndrome-related coronavirus, Murine coronavirus, Pipistrellus bat coronavirus HKU5, Rousettus bat coronavirus HKU9, Severe acute respiratory syndrome–related coronavirus (SARS-CoV, SARS-CoV-2), Tylonycteris bat coronavirus HKU4\\nGenus Gammacoronavirus;Species: Avian coronavirus, Beluga whale coronavirus SW1\\nGenus Deltacoronavirus\\nSpecies: Bulbul coronavirus HKU11, Porcine coronavirus HKU15\\n\\n\\n== Origin ==\\n\\nThe most recent common ancestor (MRCA) of all coronaviruses is estimated to have existed as recently as 8000 BCE, although some models place the common ancestor as far back as 55 million years or more, implying long term coevolution with bat and avian species. The most recent common ancestor of the alphacoronavirus line has been placed at about 2400 BCE, of the betacoronavirus line at 3300 BCE, of the gammacoronavirus line at 2800 BCE, and the deltacoronavirus line at about 3000 BCE. Bats and birds, as warm-blooded flying vertebrates, are an ideal natural reservoir for the coronavirus gene pool (with bats the reservoir for alphacoronaviruses and betacoronavirus – and birds the reservoir for gammacoronaviruses and deltacoronaviruses). The large number and global range of bat and avian species that host viruses have enabled extensive evolution and dissemination of coronaviruses.Many human coronaviruses have their origin in bats. The human coronavirus NL63 shared a common ancestor with a bat coronavirus (ARCoV.2) between 1190 and 1449 CE. The human coronavirus 229E shared a common ancestor with a bat coronavirus (GhanaGrp1 Bt CoV) between 1686 and 1800 CE. More recently, alpaca coronavirus and human coronavirus 229E diverged sometime before 1960. MERS-CoV emerged in humans from bats through the intermediate host of camels. MERS-CoV, although related to several bat coronavirus species, appears to have diverged from these several centuries ago. The most closely related bat coronavirus and SARS-CoV diverged in 1986. The ancestors of SARS-CoV first infected leaf-nose bats of the genus Hipposideridae; subsequently, they spread to horseshoe bats in the species Rhinolophidae, then to Asian palm civets, and finally to humans.Unlike other betacoronaviruses, bovine coronavirus of the species Betacoronavirus 1 and subgenus Embecovirus is thought to have originated in rodents and not in bats. In the 1790s, equine coronavirus diverged from the bovine coronavirus after a cross-species jump. Later in the 1890s, human coronavirus OC43 diverged from bovine coronavirus after another cross-species spillover event. It is speculated that the flu pandemic of 1890 may have been caused by this spillover event, and not by the influenza virus, because of the related timing, neurological symptoms, and unknown causative agent of the pandemic. Besides causing respiratory infections, human coronavirus OC43 is also suspected of playing a role in neurological diseases. In the 1950s, the human coronavirus OC43 began to diverge into its present genotypes. Phylogenetically, mouse hepatitis virus (Murine coronavirus), which infects the mouse\\'s liver and central nervous system, is related to human coronavirus OC43 and bovine coronavirus. Human coronavirus HKU1, like the aforementioned viruses, also has its origins in rodents.\\n\\n\\n== Infection in humans ==\\n\\nCoronaviruses vary significantly in risk factor. Some can kill more than 30% of those infected, such as MERS-CoV, and some are relatively harmless, such as the common cold. Coronaviruses can cause colds with major symptoms, such as fever, and a sore throat from swollen adenoids. Coronaviruses can cause pneumonia (either direct viral pneumonia or secondary bacterial pneumonia) and bronchitis (either direct viral bronchitis or secondary bacterial bronchitis). The human coronavirus discovered in 2003, SARS-CoV, which causes severe acute respiratory syndrome (SARS), has a unique pathogenesis because it causes both upper and lower respiratory tract infections.Six species of human coronaviruses are known, with one species subdivided into two different strains, making seven strains of human coronaviruses altogether.\\n\\nFour human coronaviruses produce symptoms that are generally mild, even though it is contended they might have been more aggressive in the past:\\nHuman coronavirus OC43 (HCoV-OC43), β-CoV\\nHuman coronavirus HKU1 (HCoV-HKU1), β-CoV\\nHuman coronavirus 229E (HCoV-229E), α-CoV\\nHuman coronavirus NL63 (HCoV-NL63), α-CoV–Three human coronaviruses produce potentially severe symptoms:\\n\\nSevere acute respiratory syndrome coronavirus (SARS-CoV), β-CoV (identified in 2003)\\nMiddle East respiratory syndrome-related coronavirus (MERS-CoV), β-CoV (identified in 2012)\\nSevere acute respiratory syndrome coronavirus 2 (SARS-CoV-2), β-CoV (identified in 2019)These cause the diseases commonly called SARS, MERS, and COVID-19 respectively.\\n\\n\\n=== Common cold ===\\n\\nAlthough the common cold is usually caused by rhinoviruses, in about 15% of cases the cause is a coronavirus. The human coronaviruses HCoV-OC43, HCoV-HKU1, HCoV-229E, and HCoV-NL63 continually circulate in the human population in adults and children worldwide and produce the generally mild symptoms of the common cold. The four mild coronaviruses have a seasonal incidence occurring in the winter months in temperate climates.  There is no preponderance in any season in tropical climates.\\n\\n\\n=== Severe acute respiratory syndrome (SARS) ===\\n\\nIn 2003, following the outbreak of severe acute respiratory syndrome (SARS) which had begun the prior year in Asia, and secondary cases elsewhere in the world, the World Health Organization (WHO) issued a press release stating that a novel coronavirus identified by several laboratories was the causative agent for SARS. The virus was officially named the SARS coronavirus (SARS-CoV). More than 8,000 people from 29 countries and territories were infected, and at least 774 died.\\n\\n\\n=== Middle East respiratory syndrome (MERS) ===\\n\\nIn September 2012, a new type of coronavirus was identified, initially called Novel Coronavirus 2012, and now officially named Middle East respiratory syndrome coronavirus (MERS-CoV). The World Health Organization issued a global alert soon after. The WHO update on 28 September 2012 said the virus did not seem to pass easily from person to person. However, on 12 May 2013, a case of human-to-human transmission in France was confirmed by the French Ministry of Social Affairs and Health. In addition, cases of human-to-human transmission were reported by the Ministry of Health in Tunisia. Two confirmed cases involved people who seemed to have caught the disease from their late father, who became ill after a visit to Qatar and Saudi Arabia. Despite this, it appears the virus had trouble spreading from human to human, as most individuals who are infected do not transmit the virus. By 30 October 2013, there were 124 cases and 52 deaths in Saudi Arabia.After the Dutch Erasmus Medical Centre sequenced the virus, the virus was given a new name, Human Coronavirus–Erasmus Medical Centre (HCoV-EMC). The final name for the virus is Middle East respiratory syndrome coronavirus (MERS-CoV). The only U.S. cases (both survived) were recorded in May 2014.In May 2015, an outbreak of MERS-CoV occurred in the Republic of Korea, when a man who had traveled to the Middle East, visited four hospitals in the Seoul area to treat his illness. This caused one of the largest outbreaks of MERS-CoV outside the Middle East. As of December 2019, 2,468 cases of MERS-CoV infection had been confirmed by laboratory tests, 851 of which were fatal, a mortality rate of approximately 34.5%.\\n\\n\\n=== Coronavirus disease 2019 (COVID-19) ===\\n\\nIn December 2019, a pneumonia outbreak was reported in Wuhan, China. On 31 December 2019, the outbreak was traced to a novel strain of coronavirus, which was given the interim name 2019-nCoV by the World Health Organization, later renamed SARS-CoV-2 by the International Committee on Taxonomy of Viruses.\\nAs of 12 April 2022, there have been at least 6,183,197 confirmed deaths and more than 500,280,791 confirmed cases in the COVID-19 pandemic. The Wuhan strain has been identified as a new strain of Betacoronavirus from group 2B with approximately 70% genetic similarity to the SARS-CoV. The virus has a 96% similarity to a bat coronavirus, so it is widely suspected to originate from bats as well.\\n\\n\\n=== Coronavirus HuPn-2018 ===\\n\\nDuring a surveillance study of archived samples of Malaysian viral pneumonia patients, virologists identified a strain of canine coronavirus which has infected humans in 2018.\\n\\n\\n== Infection in animals ==\\nCoronaviruses have been recognized as causing pathological conditions in veterinary medicine since the 1930s.  They infect a range of animals including swine, cattle, horses, camels, cats, dogs, rodents, birds and bats.  The majority of animal related coronaviruses infect the intestinal tract and are transmitted by a fecal-oral route. Significant research efforts have been focused on elucidating the viral pathogenesis of these animal coronaviruses, especially by virologists interested in veterinary and zoonotic diseases.\\n\\n\\n=== Farm animals ===\\nCoronaviruses infect domesticated birds. Infectious bronchitis virus (IBV), a type of coronavirus, causes avian infectious bronchitis. The virus is of concern to the poultry industry because of the high mortality from infection, its rapid spread, and its effect on production. The virus affects both meat production and egg production and causes substantial economic loss. In chickens, infectious bronchitis virus targets not only the respiratory tract but also the urogenital tract. The virus can spread to different organs throughout the chicken. The virus is transmitted by aerosol and food contaminated by feces. Different vaccines against IBV exist and have helped to limit the spread of the virus and its variants. Infectious bronchitis virus is one of a number of strains of the species Avian coronavirus. Another strain of avian coronavirus is turkey coronavirus (TCV) which causes enteritis in turkeys.Coronaviruses also affect other branches of animal husbandry such as pig farming and the Cattle raising. Swine acute diarrhea syndrome coronavirus (SADS-CoV), which is related to bat coronavirus HKU2, causes diarrhea in pigs. Porcine epidemic diarrhea virus (PEDV) is a coronavirus that has recently emerged and similarly causes diarrhea in pigs. Transmissible gastroenteritis virus (TGEV), which is a member of the species Alphacoronavirus 1, is another coronavirus that causes diarrhea in young pigs. In the cattle industry bovine coronavirus (BCV), which is a member of the species Betacoronavirus 1 and related to HCoV-OC43, is responsible for severe profuse enteritis in young calves.\\n\\n\\n=== Domestic pets ===\\nCoronaviruses infect domestic pets such as cats, dogs, and ferrets. There are two forms of feline coronavirus which are both members of the species Alphacoronavirus 1. Feline enteric coronavirus is a pathogen of minor clinical significance, but spontaneous mutation of this virus can result in feline infectious peritonitis (FIP), a disease with high mortality. There are two different coronaviruses that infect dogs. Canine coronavirus (CCoV), which is a member of the species Alphacoronavirus 1, causes mild gastrointestinal disease. Canine respiratory coronavirus (CRCoV), which is a member of the species Betacoronavirus 1 and related to HCoV-OC43, cause respiratory disease. Similarly, there are two types of coronavirus that infect ferrets. Ferret enteric coronavirus causes a gastrointestinal syndrome known as epizootic catarrhal enteritis (ECE), and a more lethal systemic version of the virus (like FIP in cats) known as ferret systemic coronavirus (FSC).\\n\\n\\n=== Laboratory animals ===\\nCoronaviruses infect laboratory animals. Mouse hepatitis virus (MHV), which is a member of the species Murine coronavirus, causes an epidemic murine illness with high mortality, especially among colonies of laboratory mice.  Prior to the discovery of SARS-CoV, MHV was the best-studied coronavirus both in vivo and in vitro as well as at the molecular level.  Some strains of MHV cause a progressive demyelinating encephalitis in mice which has been used as a murine model for multiple sclerosis. Sialodacryoadenitis virus (SDAV), which is a strain of the species Murine coronavirus, is highly infectious coronavirus of laboratory rats, which can be transmitted between individuals by direct contact and indirectly by aerosol.  Rabbit enteric coronavirus causes acute gastrointestinal disease and diarrhea in young European rabbits.  Mortality rates are high.\\n\\n\\n== Prevention and treatment ==\\nA number of vaccines using different methods have been developed against human coronavirus SARS-CoV-2. Antiviral targets against human coronaviruses have also been identified such as viral proteases, polymerases, and entry proteins. Drugs are in development which target these proteins and the different steps of viral replication.Vaccines are available for animal coronaviruses IBV, TGEV, and Canine CoV, although their effectiveness is limited. In the case of outbreaks of highly contagious animal coronaviruses, such as PEDV, measures such as destruction of entire herds of pigs may be used to prevent transmission to other herds.\\n\\n\\n== See also ==\\nCoronavirus diseases\\nZoonosis\\n\\n\\n== References ==\\n\\n\\n== Further reading ==',\n",
              " 'Severe acute respiratory syndrome coronavirus 2 (SARS‑CoV‑2) is a strain of coronavirus that causes COVID-19 (coronavirus disease 2019), the respiratory illness responsible for the ongoing COVID-19 pandemic. The virus previously had a provisional name, 2019 novel coronavirus (2019-nCoV), and has also been called human coronavirus 2019 (HCoV-19 or hCoV-19).  First identified in the city of Wuhan, Hubei, China, the World Health Organization declared the outbreak a Public Health Emergency of International Concern on 30 January 2020, and a pandemic on 11 March 2020. SARS‑CoV‑2 is a positive-sense single-stranded RNA virus that is contagious in humans.SARS‑CoV‑2 is a virus of the species severe acute respiratory syndrome–related coronavirus (SARSr-CoV), related to the SARS-CoV-1 virus that caused the 2002–2004 SARS outbreak. It is of zoonotic origins and has close genetic similarity to bat coronaviruses, suggesting it emerged from a bat-borne virus. Research is ongoing as to whether SARS‑CoV‑2 came directly from bats or indirectly through any intermediate hosts. The virus shows little genetic diversity, indicating that the spillover event introducing SARS‑CoV‑2 to humans is likely to have occurred in late 2019.Epidemiological studies estimate that, in the December 2019 – September 2020 period, each infection resulted in an average of 2.4 to 3.4 new ones when no members of the community are immune and no preventive measures are taken. However, some subsequent variants have become more infectious. The virus primarily spreads between people through close contact and via aerosols and respiratory droplets that are exhaled when talking, breathing, or otherwise exhaling, as well as those produced from coughs or sneezes. It enters human cells by binding to angiotensin-converting enzyme 2 (ACE2), a membrane protein that regulates the renin–angiotensin system.\\n\\n\\n== Terminology ==\\nDuring the initial outbreak in Wuhan, China, various names were used for the virus; some names used by different sources included \"the coronavirus\" or \"Wuhan coronavirus\". In January 2020, the World Health Organization (WHO) recommended \"2019 novel coronavirus\" (2019-nCov) as the provisional name for the virus. This was in accordance with WHO\\'s 2015 guidance against using geographical locations, animal species, or groups of people in disease and virus names.On 11 February 2020, the International Committee on Taxonomy of Viruses adopted the official name \"severe acute respiratory syndrome coronavirus 2\" (SARS‑CoV‑2). To avoid confusion with the disease SARS, the WHO sometimes refers to SARS‑CoV‑2 as \"the COVID-19 virus\" in public health communications and the name HCoV-19 was included in some research articles. Referring to COVID-19 as the \"Wuhan virus\" has been described as dangerous by WHO officials, and as xenophobic by University of California at Berkeley Asian American studies lecturer Harvey Dong.\\n\\n\\n== Infection and transmission ==\\n\\nHuman-to-human transmission of SARS‑CoV‑2 was confirmed on 20 January 2020 during the COVID-19 pandemic. Transmission was initially assumed to occur primarily via respiratory droplets from coughs and sneezes within a range of about 1.8 metres (6 ft). Laser light scattering experiments suggest that speaking is an additional mode of transmission and a far-reaching one, indoors, with little air flow. Other studies have suggested that the virus may be airborne as well, with aerosols potentially being able to transmit the virus. During human-to-human transmission, between 200 and 800 infectious SARS‑CoV‑2 virions are thought to initiate a new infection. If confirmed, aerosol transmission has biosafety implications because a major concern associated with the risk of working with emerging viruses in the laboratory is the generation of aerosols from various laboratory activities which are not immediately recognizable and may affect other scientific personnel. Indirect contact via contaminated surfaces is another possible cause of infection. Preliminary research indicates that the virus may remain viable on plastic (polypropylene) and stainless steel (AISI 304) for up to three days, but it does not survive on cardboard for more than one day or on copper for more than four hours. The virus is inactivated by soap, which destabilizes its lipid bilayer. Viral RNA has also been found in stool samples and semen from infected individuals.The degree to which the virus is infectious during the incubation period is uncertain, but research has indicated that the pharynx reaches peak viral load approximately four days after infection or in the first week of symptoms and declines thereafter. The duration of SARS-CoV-2 RNA shedding is generally between 3 and 46 days after symptom onset.A study by a team of researchers from the University of North Carolina found that the nasal cavity is seemingly the dominant initial site of infection, with subsequent aspiration-mediated virus-seeding into the lungs in SARS‑CoV‑2 pathogenesis. They found that there was an infection gradient from high in proximal towards low in distal pulmonary epithelial cultures, with a focal infection in ciliated cells and type 2 pneumocytes in the airway and alveolar regions respectively.Studies have identified a range of animals—such as cats, ferrets, hamsters, non-human primates, minks, tree shrews, raccoon dogs, fruit bats, and rabbits—that are susceptible and permissive to SARS-CoV-2 infection. Some institutions have advised that those infected with SARS‑CoV‑2 restrict their contact with animals.\\n\\n\\n=== Asymptomatic and presymptomatic transmission ===\\nOn 1 February 2020, the World Health Organization (WHO) indicated that \"transmission from asymptomatic cases is likely not a major driver of transmission\". One meta-analysis found that 17% of infections are asymptomatic, and asymptomatic individuals were 42% less likely to transmit the virus.However, an epidemiological model of the beginning of the outbreak in China suggested that \"pre-symptomatic shedding may be typical among documented infections\" and that subclinical infections may have been the source of a majority of infections. That may explain how out of 217 on board a cruise liner that docked at Montevideo, only 24 of 128 who tested positive for viral RNA showed symptoms. Similarly, a study of ninety-four patients hospitalized in January and February 2020 estimated patients began shedding virus two to three days before symptoms appear and that \"a substantial proportion of transmission probably occurred before first symptoms in the index case\". The authors later published a correction that showed that shedding began earlier than first estimated, four to five days before symptoms appear.\\n\\n\\n=== Reinfection ===\\nThere is uncertainty about reinfection and long-term immunity. It is not known how common reinfection is, but reports have indicated that it is occurring with variable severity.The first reported case of reinfection was a 33-year-old man from Hong Kong who first tested positive on 26 March 2020, was discharged on 15 April 2020 after two negative tests, and tested positive again on 15 August 2020 (142 days later), which was confirmed by whole-genome sequencing showing that the viral genomes between the episodes belong to different clades. The findings had the implications that herd immunity may not eliminate the virus if reinfection is not an uncommon occurrence and that vaccines may not be able to provide lifelong protection against the virus.Another case study described a 25-year-old man from Nevada who tested positive for SARS‑CoV‑2 on 18 April 2020 and on 5 June 2020 (separated by two negative tests). Since genomic analyses showed significant genetic differences between the SARS‑CoV‑2 variant sampled on those two dates, the case study authors determined this was a reinfection. The man\\'s second infection was symptomatically more severe than the first infection, but the mechanisms that could account for this are not known.\\n\\n\\n== Reservoir and origin ==\\n\\nThe first known infections from SARS‑CoV‑2 were discovered in Wuhan, China. The original source of viral transmission to humans remains unclear, as does whether the virus became pathogenic before or after the spillover event. Because many of the early infectees were workers at the Huanan Seafood Market, it has been suggested that the virus might have originated from the market. However, other research indicates that visitors may have introduced the virus to the market, which then facilitated rapid expansion of the infections. A March 2021 WHO-convened report stated that human spillover via an intermediate animal host was the most likely explanation, with direct spillover from bats next most likely. Introduction through the food supply chain and the Huanan Seafood Market was considered another possible, but less likely, explanation. An analysis in November 2021, however, said that the earliest-known case had been misidentified and that the preponderance of early cases linked to the Huanan Market argued for it being the source.For a virus recently acquired through a cross-species transmission, rapid evolution is expected. The mutation rate estimated from early cases of SARS-CoV-2 was of 6.54×10−4 per site per year. Coronaviruses in general have high genetic plasticity, but SARS-CoV-2\\'s viral evolution is slowed by the RNA proofreading capability of its replication machinery. For comparison, the viral mutation rate in vivo of SARS-CoV-2 has been found to be lower than that of influenza.Research into the natural reservoir of the virus that caused the 2002–2004 SARS outbreak has resulted in the discovery of many SARS-like bat coronaviruses, most originating in horseshoe bats. The closest match by far, published on Nature (journal) in February 2022, were viruses BANAL-52 (96.8% resemblance to SARS‑CoV‑2), BANAL-103 and BANAL-236, collected in three different species of bats in Feuang, Laos. An earlier source published in February 2020 identified the virus RaTG13, collected in bats in Mojiang, Yunnan, China to be the closest to SARS‑CoV‑2, with 96.1% resemblance. None of the above are its direct ancestor.\\n\\nBats are considered the most likely natural reservoir of SARS‑CoV‑2. Differences between the bat coronavirus and SARS‑CoV‑2 suggest that humans may have been infected via an intermediate host; although the source of introduction into humans remains unknown.Although the role of pangolins as an intermediate host was initially posited (a study published in July 2020 suggested that pangolins are an intermediate host of SARS‑CoV‑2-like coronaviruses), subsequent studies have not substantiated their contribution to the spillover. Evidence against this hypothesis includes the fact that pangolin virus samples are too distant to SARS-CoV-2: isolates obtained from pangolins seized in Guangdong were only 92% identical in sequence to the SARS‑CoV‑2 genome (matches above 90 percent may sound high, but in genomic terms it is a wide evolutionary gap). In addition, despite similarities in a few critical amino acids, pangolin virus samples exhibit poor binding to the human ACE2 receptor.\\n\\n\\n== Phylogenetics and taxonomy ==\\nSARS‑CoV‑2 belongs to the broad family of viruses known as coronaviruses. It is a positive-sense single-stranded RNA (+ssRNA) virus, with a single linear RNA segment. Coronaviruses infect humans, other mammals, including livestock and companion animals, and avian species. Human coronaviruses are capable of causing illnesses ranging from the common cold to more severe diseases such as Middle East respiratory syndrome (MERS, fatality rate ~34%). SARS-CoV-2 is the seventh known coronavirus to infect people, after 229E, NL63, OC43, HKU1, MERS-CoV, and the original SARS-CoV.Like the SARS-related coronavirus implicated in the 2003 SARS outbreak, SARS‑CoV‑2 is a member of the subgenus Sarbecovirus (beta-CoV lineage B). Coronaviruses undergo frequent recombination. The mechanism of recombination in unsegmented RNA viruses such as SARS-CoV-2 is generally by copy-choice replication, in which gene material switches from one RNA template molecule to another during replication. SARS-CoV-2 RNA sequence is approximately 30,000 bases in length, relatively long for a coronavirus (which in turn carry the largest genomes among all RNA families) Its genome consists nearly entirely of protein-coding sequences, a trait shared with other coronaviruses.\\n\\nA distinguishing feature of SARS‑CoV‑2 is its incorporation of a polybasic site cleaved by furin, which appears to be an important element enhancing its virulence. It was suggested that the acquisition of the furin-cleavage site in the SARS-CoV-2 S protein was essential for zoonotic transfer to humans. The furin protease recognizes the canonical peptide sequence RX[R/K] R↓X where the cleavage site is indicated by a down arrow and X is any amino acid. In SARS-CoV-2 the recognition site is formed by the incorporated 12 codon nucleotide sequence CCT CGG CGG GCA which corresponds to the amino acid sequence P RR A. This sequence is upstream of an arginine and serine which forms the S1/S2 cleavage site (P RR A R↓S) of the spike protein. Although such sites are a common naturally-occurring feature of other viruses within the Subfamily Orthocoronavirinae, it appears in few other viruses from the Beta-CoV genus, and it is unique among members of its subgenus for such a site. The furin cleavage site PRRAR↓ is identical to that of the feline coronavirus, an alphacoronavirus 1 strain.Viral genetic sequence data can provide critical information about whether viruses separated by time and space are likely to be epidemiologically linked. With a sufficient number of sequenced genomes, it is possible to reconstruct a phylogenetic tree of the mutation history of a family of viruses. By 12 January 2020, five genomes of SARS‑CoV‑2 had been isolated from Wuhan and reported by the Chinese Center for Disease Control and Prevention (CCDC) and other institutions; the number of genomes increased to 42 by 30 January 2020. A phylogenetic analysis of those samples showed they were \"highly related with at most seven mutations relative to a common ancestor\", implying that the first human infection occurred in November or December 2019. Examination of the topology of the phylogenetic tree at the start of the pandemic also found high similarities between human isolates.  As of 21 August 2021, 3,422 SARS‑CoV‑2 genomes, belonging to 19 strains, sampled on all continents except Antarctica were publicly available.On 11 February 2020, the International Committee on Taxonomy of Viruses announced that according to existing rules that compute hierarchical relationships among coronaviruses based on five conserved sequences of nucleic acids, the differences between what was then called 2019-nCoV and the virus from the 2003 SARS outbreak were insufficient to make them separate viral species. Therefore, they identified 2019-nCoV as a virus of Severe acute respiratory syndrome–related coronavirus.In July 2020, scientists reported that a more infectious SARS‑CoV‑2 variant with spike protein variant G614 has replaced D614 as the dominant form in the pandemic.Coronavirus genomes and subgenomes encode six open reading frames (ORFs). In October 2020, researchers discovered a possible overlapping gene named ORF3d, in the SARS‑CoV‑2 genome. It is unknown if the protein produced by ORF3d has any function, but it provokes a strong immune response. ORF3d has been identified before, in a variant of coronavirus that infects pangolins.\\n\\n\\n=== Phylogenetic tree ===\\nA phylogenetic tree based on whole-genome sequences of SARS-CoV-2 and related coronaviruses is:\\n\\n\\n==== Variants ====\\n\\nThere are many thousands of variants of SARS-CoV-2, which can be grouped into the much larger clades. Several different clade nomenclatures have been proposed. Nextstrain divides the variants into five clades (19A, 19B, 20A, 20B, and 20C), while GISAID divides them into seven (L, O, V, S, G, GH, and GR).Several notable variants of SARS-CoV-2 emerged in late 2020. The World Health Organization has currently declared five variants of concern, which are as follows:\\nAlpha: Lineage B.1.1.7 emerged in the United Kingdom in September 2020, with evidence of increased transmissibility and virulence. Notable mutations include N501Y and P681H.\\nAn E484K mutation in some lineage B.1.1.7 virions has been noted and is also tracked by various public health agencies.\\nBeta: Lineage B.1.351 emerged in South Africa in May 2020, with evidence of increased transmissibility and changes to antigenicity, with some public health officials raising alarms about its impact on the efficacy of some vaccines. Notable mutations include K417N, E484K and N501Y.\\nGamma: Lineage P.1 emerged in Brazil in November 2020, also with evidence of increased transmissibility and virulence, alongside changes to antigenicity. Similar concerns about vaccine efficacy have been raised. Notable mutations also include K417N, E484K and N501Y.\\nDelta: Lineage B.1.617.2 emerged in India in October 2020. There is also evidence of increased transmissibility and changes to antigenicity.\\nOmicron: Lineage B.1.1.529 emerged in Botswana in November 2021.Other notable variants include 6 other WHO-designated variants under investigation and Cluster 5, which emerged among mink in Denmark and resulted in a mink euthanasia campaign rendering it virtually extinct.\\n\\n\\n== Virology ==\\n\\n\\n=== Structure ===\\n\\nEach SARS-CoV-2 virion is 60–140 nanometres (2.4×10−6–5.5×10−6 in) in diameter; its mass within the global human populace has been estimated as being between 0.1 and 1.0 kg. Like other coronaviruses, SARS-CoV-2 has four structural proteins, known as the S (spike), E (envelope), M (membrane), and N (nucleocapsid) proteins; the N protein holds the RNA genome, and the S, E, and M proteins together create the viral envelope. Coronavirus S proteins are glycoproteins and also type I membrane proteins (membranes containing a single transmembrane domain oriented on the extracellular side). They are divided into two functional parts (S1 and S2). In SARS-CoV-2, the spike protein, which has been imaged at the atomic level using cryogenic electron microscopy, is the protein responsible for allowing the virus to attach to and fuse with the membrane of a host cell; specifically, its S1 subunit catalyzes attachment, the S2 subunit fusion.\\n\\n\\n=== Genome ===\\nSARS-CoV-2 has a linear, positive-sense, single-stranded RNA genome about 30,000 bases long. Its genome has a bias against cytosine (C) and guanine (G) nucleotides, like other coronaviruses. The genome has the highest composition of U (32.2%), followed by A (29.9%), and a similar composition of G (19.6%) and C (18.3%). The nucleotide bias arises from the mutation of guanines and cytosines to adenosines and uracils, respectively. The mutation of CG dinucleotides is thought to arise to avoid the zinc finger antiviral protein related defense mechanism of cells, and to lower the energy to unbind the genome during replication and translation (adenosine and uracil base pair via two hydrogen bonds, cytosine and guanine via three). The depletion of CG dinucleotides in its genome has led the virus to have a noticeable codon usage bias. For instance, arginine\\'s six different codons have a relative synonymous codon usage of AGA (2.67), CGU (1.46), AGG (.81), CGC (.58), CGA (.29), and CGG (.19). A similar codon usage bias trend is seen in other SARS–related coronaviruses.\\n\\n\\n=== Replication cycle ===\\nVirus infections start when viral particles bind to host surface cellular receptors. Protein modeling experiments on the spike protein of the virus soon suggested that SARS‑CoV‑2 has sufficient affinity to the receptor angiotensin converting enzyme 2 (ACE2) on human cells to use them as a mechanism of cell entry. By 22 January 2020, a group in China working with the full virus genome and a group in the United States using reverse genetics methods independently and experimentally demonstrated that ACE2 could act as the receptor for SARS‑CoV‑2. Studies have shown that SARS‑CoV‑2 has a higher affinity to human ACE2 than the original SARS virus. SARS‑CoV‑2 may also use basigin to assist in cell entry.Initial spike protein priming by transmembrane protease, serine 2 (TMPRSS2) is essential for entry of SARS‑CoV‑2. The host protein neuropilin 1 (NRP1) may aid the virus in host cell entry using ACE2. After a SARS‑CoV‑2 virion attaches to a target cell, the cell\\'s TMPRSS2 cuts open the spike protein of the virus, exposing a fusion peptide in the S2 subunit, and the host receptor ACE2. After fusion, an endosome forms around the virion, separating it from the rest of the host cell. The virion escapes when the pH of the endosome drops or when cathepsin, a host cysteine protease, cleaves it. The virion then releases RNA into the cell and forces the cell to produce and disseminate copies of the virus, which infect more cells.SARS‑CoV‑2 produces at least three virulence factors that promote shedding of new virions from host cells and inhibit immune response. Whether they include downregulation of ACE2, as seen in similar coronaviruses, remains under investigation (as of May 2020).\\n\\n\\n== Treatment and drug development ==\\nVery few drugs are known to effectively inhibit SARS‑CoV‑2. Masitinib is a clinically safe drug and was recently found to inhibit its main protease, 3CLpro and showed >200-fold reduction in viral titers in the lungs and nose in mice. However, it is not approved for the treatment of COVID-19 in humans as of August 2021.COVID Moonshot is an international collaborative open-science project started in March 2020 with the goal of developing an un-patented oral antiviral drug for treatment of SARS-CoV-2.\\n\\n\\n== Epidemiology ==\\n\\nRetrospective tests collected within the Chinese surveillance system revealed no clear indication of substantial unrecognized circulation of SARS‑CoV‑2 in Wuhan during the latter part of 2019.A meta-analysis from November 2020 estimated the basic reproduction number (\\n  \\n    \\n      \\n        \\n          R\\n          \\n            0\\n          \\n        \\n      \\n    \\n    {\\\\displaystyle R_{0}}\\n  ) of the virus to be between 2.39 and 3.44. This means each infection from the virus is expected to result in 2.39 to 3.44 new infections when no members of the community are immune and no preventive measures are taken. The reproduction number may be higher in densely populated conditions such as those found on cruise ships. Human behavior affects the R0 value and hence estimates of R0 differ between different countries, cultures, and social norms. For instance, one study found relatively low R0 (~3.5) in Sweden, Belgium and the Netherlands, while Spain and the US had significantly higher R0 values (5.9 to 6.4, respectively).\\nThere have been about 96,000 confirmed cases of infection in mainland China. While the proportion of infections that result in confirmed cases or progress to diagnosable disease remains unclear, one mathematical model estimated that 75,815 people were infected on 25 January 2020 in Wuhan alone, at a time when the number of confirmed cases worldwide was only 2,015. Before 24 February 2020, over 95% of all deaths from COVID-19 worldwide had occurred in Hubei province, where Wuhan is located. As of 12 April 2022, the percentage had decreased to 0.052%.As of 12 April 2022, there have been 500,280,791 total confirmed cases of SARS‑CoV‑2 infection in the ongoing pandemic. The total number of deaths attributed to the virus is 6,183,197.\\n\\n\\n== See also ==\\n3C-like protease (NS5)\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\n\\n\\n== External links ==\\n\"Coronavirus Disease 2019 (COVID-19)\". Centers for Disease Control and Prevention (CDC). 11 February 2020.\\n\"Coronavirus disease (COVID-19) Pandemic\". World Health Organization (WHO).\\n\"SARS-CoV-2 (Severe acute respiratory syndrome coronavirus 2) Sequences\". National Center for Biotechnology Information (NCBI).\\n\"COVID-19 Resource Centre\". The Lancet.\\n\"Coronavirus (Covid-19)\". The New England Journal of Medicine.\\n\"Covid-19: Novel Coronavirus Outbreak\". Wiley.\\n\"SARS-CoV-2\". Virus Pathogen Database and Analysis Resource.\\n\"SARS-CoV-2 related protein structures\". Protein Data Bank.',\n",
              " 'The Omicron variant (B.1.1.529) is a variant of SARS-CoV-2 (the virus that causes COVID-19) that was first reported to the World Health Organization (WHO) from South Africa on 24 November 2021.Omicron multiplies around 70 times faster than the Delta variant in the bronchi (lung airways) but evidence suggests it is less severe than previous strains, especially compared to the Delta variant. Omicron might be less able to penetrate deep lung tissue. Omicron infections are 91 percent less fatal than the delta variant, with 51 percent less risk of hospitalization. However, the estimated difference in the intrinsic risk of hospitalization largely decreases to 0–30 percent when reinfections are excluded. Overall, the extremely high rate of spread, combined with its ability to evade both double vaccination and the body\\'s immune system, means the total number of patients requiring hospital care at any given time is still of great concern.Vaccines continue to provide protection against severe disease and hospitalisation especially after a third dose of an mRNA vaccine is given. Early figures suggest that double vaccination offers 30 to 40 percent protection against infection and around 70 percent protection against hospitalization. A recent third vaccine dose boosts effectiveness against infection to around 75 percent, and 88 percent for severe disease.\\n\\n\\n== Classification ==\\n\\n\\n=== Nomenclature ===\\nOn 26 November, the WHO\\'s Technical Advisory Group on SARS-CoV-2 Virus Evolution declared PANGO lineage B.1.1.529 a variant of concern and designated it with the Greek letter omicron. Greek letters are used to identify variants of SARS-CoV-2. The WHO skipped the preceding letters nu and xi in the Greek alphabet to avoid confusion with the similarities of the English word \"new\" and the Chinese surname Xi. The previous designation was for the \"variant of interest\" Mu.Possibly due to a lack of familiarity with the Greek alphabet among some English speakers and the relative frequency of the Latin prefix \"omni\" in other common speech, the name of the variant has also occasionally been mispronounced and misspelled as \"Omnicron\".The GISAID project has assigned it the clade identifier GR/484A, and the Nextstrain project has assigned it the clade identifiers 21K and 21L, both belonging to a larger Omicron group 21M.\\n\\n\\n=== Mutations ===\\n\\nThe variant has many mutations, some of which have concerned scientists. The Omicron variant has a total of 60 mutations compared to the reference / ancestral variant: 50 nonsynonymous mutations, 8 synonymous mutations, and 2 non-coding mutations. Thirty-two mutations affect the spike protein, the main antigenic target of antibodies generated by infections and of many vaccines widely administered. Many of those mutations had not been observed in other strains.\\nThe variant is characterised by 30 amino acid changes, three small deletions, and one small insertion in the spike protein compared with the original virus, of which 15 are located in the receptor-binding domain (residues 319–541). It also carries a number of changes and deletions in other genomic regions. Additionally, the variant has three mutations at the furin cleavage site. The furin cleavage site increases SARS-CoV-2 infectivity. The mutations by genomic region are the following:\\n\\nSpike protein: A67V, Δ69-70, T95I, G142D, Δ143-145, Δ211, L212I, ins214EPE, G339D, S371L, S373P, S375F, K417N, N440K, G446S, S477N, T478K, E484A, Q493R, G496S, Q498R, N501Y, Y505H, T547K, D614G, H655Y, N679K, P681H, N764K, D796Y, N856K, Q954H, N969K, L981F\\nHalf (15) of these 30 changes are located in the receptor binding domain-RBD (residues 319–541)\\nORF1ab\\nnsp3: K38R, V1069I, Δ1265, L1266I, A1892T\\nnsp4: T492I\\nnsp5: P132H\\nnsp6: Δ105-107, A189V\\nnsp12: P323L\\nnsp14: I42V\\nEnvelope protein: T9I\\nMembrane protein: D3G, Q19E, A63T\\nNucleocapsid protein: P13L, Δ31-33, R203K, G204RA link with HIV infection may explain a large number of mutations in the sequence of the Omicron variant. Indeed, in order to be affected by such a high number of mutations, the virus must have been able to evolve a long time without killing its host, nor being eliminated. One such situation occurs in people with a weakened immune system but receiving enough medical care to survive. This is the case in HIV patients in South Africa, who represent more than 20% of the population. Due to lack of access to clinics, fear of stigmatisation and disrupted healthcare, millions living with HIV in the region are not on effective HIV therapy. HIV prevention could be key to reducing the risk of uncontrolled HIV driving the emergence of Covid variants.In addition, it is believed that one of these many mutations, comprising a 9-nucleotide sequence, may have been acquired from another coronavirus (known as HCoV-229E), responsible for the common cold.  This is not entirely unexpected\\u200a—\\u200aat times, viruses within the body acquire and swap segments of genetic material from each other, and this is one common means of mutation.One hypothesis to explain the novel mutations is that SARS-CoV-2 was transmitted from humans to mice and mutated in a population of mice sometime between mid-2020 and late 2021 before reinfecting humans.\\n\\n\\n==== Sublineages and BA.2 subvariant ====\\nResearchers have established the existence of three sublineages of Omicron. The \\'standard\\' sublineage is now referred to as BA.1 (or B.1.1.529.1), and the two other sublineages are known as BA.2 (or B.1.1.529.2) and BA.3 (or B.1.1.529.3). They share many mutations, but also significantly differ. In general, BA.1 and BA.2 share 32 mutations, but differ by 28. This makes them as different as some other major variants, and it has been suggested that BA.2 should receive its own Greek-letter name. BA.1 has itself been divided in two, the original BA.1 and BA.1.1, where the main difference is that the latter has a R346K mutation.A laboratory study on hamsters and mice in Japan published as a non-peer-reviewed preprint in mid-February 2022 suggested that BA.2, is not only more transmissible than BA.1, but may cause more severe disease. Therapeutic monoclonal antibodies used to treat people infected with COVID did not have much effect on BA.2, which was \"almost completely resistant\" to casirivimab and imdevimab, and 35 times more resistant to sotrovimab than the original B.1.1 virus. The researchers proposed that BA.2 should be recognised as a unique variant of concern, which would be given its own Greek letter.\\n\\n\\n===== Detection =====\\nOrdinary COVID-19 tests, both PCR and rapid, can detect all Omicron subvariants as COVID-19, but further tests are necessary to distinguish the subvariants from each other and from other COVID-19 variants.A notable difference between the \\'standard\\' Omicron subvariant and BA.2 is that the latter lacks the characteristic S-gene target failure (SGTF)-causing deletion (Δ69-70) by which many qPCR tests are able to rapidly detect a case as an Omicron (or Alpha) variant, from the previously dominant  Delta variant. Thus, countries that primarily rely on SGTF for detection may overlook BA.2, and British authorities consider SGTF alone as insufficient for monitoring the spread of Omicron. This has resulted in it having been nicknamed \\'Stealth Omicron\\'. Because BA.2 still can be separated from other variants through normal full sequencing, or checks of certain other mutations, the nickname is however inaccurate. Some countries, such as Denmark and Japan, use a variant qPCR that tests for several mutations, including L452R. It can also distinguish Delta, which has L452R, and all Omicron sublineages, which do not have L452R. As Omicron became dominant and the Delta variant became rare, the SGTF mutation that had made Delta and BA.2 similar in qPCR tests could now be used for easily separating BA.1 and BA.2 from each other. As a consequence, BA.2 could now be regarded as decidedly un-stealthy.The third Omicron sublineage, BA.3, is very rare. It has the same SGTF deletion (Δ69-70) as BA.1.In April 2022 the WHO announced it was tracking BA.4 and BA.5 subvariants with BA.4 having been detected in South Africa, Botswana, Denmark, Scotland and England.Early indications from data collected in South Africa suggested BA.4 and BA.5 have a significant growth advantage over BA.2.\\n\\n\\n===== Affected countries and transmissibility =====\\nAccording to early research, BA.2 is roughly 30% to 60% more transmissible than BA.1. As a consequence, it may prolong a COVID-19 wave when it overtakes BA.1, although it is difficult to assess what part is caused by the higher transmissibility of BA.2 in countries that simultaneously reduce restrictions (allowing easier transmission than in earlier periods with more restrictions). A new BA.1–BA.2 recombinant isolated from the UK in January 2022, dubbed the \"XE\" recombinant, was found by the WHO to be potentially 10% more transmissible than BA.2, making it about 43% to 76% to more transmissible than BA.1, and making the XE recombinant the most contagious variant identified. On 7 April 2022 Brazilian authorities announced the first detected case of a person infected with Omicron XE.The first known sequence of BA.2 was in a sample from 15 November 2021. In mid-December 2021, BA.2 still appeared to be rare with relatively few sequences from half a dozen countries having been uploaded to GISAID, but subsequently numbers rapidly increased. As of 17 January 2022, BA.2 had been detected in at least 40 countries and in all continents except Antarctica. By 31 January, it had been detected in at least 57 countries. In global samples collected from 4 February to 5 March and uploaded to GISAID, BA.2 accounted for c. 34%, compared to 41% for BA.1.1, 25% for BA.1 and less than 1% for BA.3. In a review two weeks later, covering 16 February to 17 March, BA.2 had become the most frequent. However, the data is geographically skewed due to sequencing rate and speed; for example, among the c. 205,000 COVID-19 sequences from March that had been uploaded to GISAID as of 22 March, United Kingdom and Denmark accounted for more than 3⁄4, and most of the remaining were from other European countries, Australia, Canada and the United States (altogether, c. 6,000 were from Africa, Asia and Latin America). Based on GISAID uploads, BA.1 peaked in early January 2022, after which it was overtaken by both BA.1.1 and BA.2. In North America, parts of Europe and parts of Asia, BA.1 was first outcompeted by BA.1.1. For example, in the United States, France and Japan, BA.1.1 became the dominant subvariant in January 2022.By late December 2021/early January 2022, BA.2 appears to have become dominant in at least parts of India (already making up almost 80 percent in Kolkata in late December 2021) and the Philippines, had become frequent in Scandinavia, South Africa and Singapore, and was showing signs of growth in Germany and the United Kingdom. In Japan, which has quarantine and detailed screening of all international travellers, as of 24 January, the vast majority of BA.2 had been detected in people that had arrived from India or the Philippines with cases going back at least to 1 December 2021 (far fewer BA.1 or other variants were detected among arrivals from the two countries in that period), but small numbers had also been detected in people arriving from other countries.In Denmark, the first BA.2 was in a sample collected on 5 December 2021 and extremely few were found in the directly following period. By week fifty (13–19 December) it had started to increase, with BA.2 being at around 2 percent of sequenced cases compared to 46 percent BA.1 (remaining Delta). The frequency of both Omicron subvariants continued to increase throughout the last half of December; in week fifty-two (27 December–2 January), BA.2 had reached 20 percent and BA.1 peaked at 72 percent. In January 2022, BA.1 began decreasing, whereas BA.2 continued its increase. By the second week (10–16 January) of 2022, the frequency of the two was almost equal, both being near 50 percent (around one percent was the rapidly disappearing Delta). In the following week, BA.2 became clearly dominant in Denmark with 65 percent of new cases being the BA.2 subvariant. Trends from the other Scandinavian countries, India, South Africa and the United Kingdom also showed that BA.2 was increasing in proportion to the original BA.1. In early February 2022, it had become the dominant subvariant in South Africa, in late February it had become dominant in Germany and in early March it had become dominant in the United Kingdom. In early March, BA.1.1 was still heavily dominant in the United States (having overtaken BA.1 in January), but BA.2 was increasing in frequency, later becoming dominant in the US by March 29.\\n\\n\\n===== Severity and immunity =====\\nThe risk of hospitalization is the same in BA.1 and BA.2 based on reviews from Denmark, India, South Africa and the United Kingdom. Norwegian studies show that the amount of virus in the upper airways is similar in those infected with BA.1 and BA.2. In general, Omicron (all subvariants) have a higher reinfection rate than other COVID-19 variants. Studies from Denmark and Qatar found that after an infection with BA.1, the vast majority of people were well-protected against a BA.2 infection, although it is unknown how long this protection lasts. Laboratory studies also show that antibodies for BA.1 generally protect against BA.2. In Denmark, preliminary data found breakthrough rates in people that had been vaccinated that were similar to the breakthrough rates seen for BA.1. An initial study by the UK Health Security Agency found that vaccines afford similar levels of protection against symptomatic disease by BA.1 and BA.2, and in both it is considerably higher after two doses and a booster than two doses without booster. Because of the gradually waning effect of vaccines, further, booster vaccination may later be necessary.\\n\\n\\n=== Possible consequences ===\\nThe WHO is concerned that the large number of mutations in Omicron compared to earlier variants may reduce immunity in people who were previously infected and in vaccinated people. It is also possible the Omicron variant might be more infective in this regard than prior variants. The effects of the mutations, if any, are unknown as of late November 2021. The WHO warns that health services could be overwhelmed especially in nations with low vaccination rates where mortality and morbidity rates are likely to be much higher, and urges all nations to increase COVID-19 vaccinations.Professor Paul Morgan, immunologist at Cardiff University, also recommends vaccination. Morgan said, \"I think a blunting rather than a complete loss [of immunity] is the most likely outcome. The virus can\\'t possibly lose every single epitope on its surface, because if it did that spike protein couldn\\'t work any more. So, while some of the antibodies and T cell clones made against earlier versions of the virus, or against the vaccines may not be effective, there will be others, which will remain effective. (...) If half, or two-thirds, or whatever it is, of the immune response is not going to be effective, and you\\'re left with the residual half, then the more boosted that is the better.\"Professor Francois Balloux of the Genetics Institute at University College London said, \"From what we have learned so far, we can be fairly confident that – compared with other variants – Omicron tends to be better able to reinfect people who have been previously infected and received some protection against COVID-19.  That is pretty clear and was anticipated from the mutational changes we have pinpointed in its protein structure. These make it more difficult for antibodies to neutralise the virus.\"On 15 December 2021, the European Centre for Disease Prevention and Control assessed that, even if the variant turns out to be milder than Delta, its spread will very likely increase hospitalizations and fatalities due to the exponential growth in cases caused by increased transmissibility.On 23 December 2021, Nature indicates that, though Omicron likely weakens vaccine protection, reasonable effectiveness against Omicron may be maintained with currently available vaccination and boosting approaches.\\n\\n\\n=== In non-human animals ===\\nIn February 2022, the first confirmed case infecting a wild animal was confirmed by researchers at Pennsylvania State University in white-tailed deer in Staten Island, N.Y.\\n\\n\\n== Signs and symptoms ==\\n\\nA unique reported symptom of the Omicron variant is night sweats. Also, loss of taste and smell seem to be uncommon compared to other strains.A study performed between 1 and 7 December by the Center for Disease Control found that: \"The most commonly reported symptoms [were] cough, fatigue, and congestion or runny nose\" making it difficult to distinguish from a less damaging variant or other virus.Research published in London on 25 December 2021 suggested the most frequent symptoms stated by users of the Zoe Covid app were \"a running nose, headaches, fatigue, sneezing and sore throats.\"\\n\\n\\n== Characteristics ==\\nMany of the mutations to the spike protein are present in other variants of concern and are related to increased infectivity and antibody evasion. Computational modeling suggests that the variant may also escape cell-mediated immunity. On 26 November, the ECDC wrote that an evaluation of the neutralizing capacity of convalescent sera and of vaccines is urgently needed to assess possible immune escape, saying these data are expected within two to three weeks.Although transmission via fomites is rare, preliminary data indicate that the variant lasts for 194 hours on plastic surfaces and 21 hours on skin, compared with just 56 and 7 hours, respectively, for the ancestral Wuhan strain.\\n\\n\\n=== Contagiousness ===\\nIt was not known in November 2021 how the variant would spread in populations with high levels of immunity. It was also not known if the Omicron variant causes a milder or more severe COVID-19 infection. According to pharmaceutical companies, vaccines could be updated to combat the variant \"in around 100 days\" if necessary.Relating to naturally acquired immunity, Anne von Gottberg, an expert at the National Institute for Communicable Diseases, believed at the beginning of December 2021 that immunity granted by previous variants would not protect against Omicron.On 15 December 2021, Jenny Harries, head of the UK Health Security Agency, told a parliamentary committee that the doubling time of COVID-19 in most regions of the UK was now less than two days despite the country\\'s high vaccination rate. She said that the Omicron variant of COVID-19 is \"probably the most significant threat since the start of the pandemic\", and that the number of cases in the next few days would be \"quite staggering compared to the rate of growth that we\\'ve seen in cases for previous variants\".In January 2022, William Schaffner, professor of infectious diseases at Vanderbilt University Medical Center, compared the contagiousness of the Omicron variant to the contagiousness of the measles.\\n\\n\\n==== Immune evasion ====\\n\\nA study suggests that mutations that promote breakthrough infections or antibody-resistance \"like those in Omicron\" could be a new mechanism for viral evolution success of SARS-CoV-2 and that such may become a dominating mechanism of its evolution. A preprint supports such an explanation of Omicron\\'s spread, suggesting that it \"primarily can be ascribed to the immune evasiveness rather than an inherent increase in the basic transmissibility\". Studies showed the variant to escape the majority of existing SARS-CoV-2 neutralizing antibodies, including of sera from vaccinated and convalescent individuals. Nevertheless, current vaccines are expected to protect against severe illness, hospitalizations, and deaths due to Omicron and, on an individual level, the Omicron variant is milder than earlier variants that evolved when the antibody/vaccination share was lower than it was when Omicron emerged.\\n\\n\\n=== Virulence ===\\n\\nAs of 28 November 2021 the World Health Organization\\'s update states \"There is currently no information to suggest that symptoms associated with Omicron are different from ... other variants\". Increased rates of hospitalization in South Africa may be due to a higher number of cases, rather than any specific feature of the Omicron variant.On 4 December 2021, the South African Medical Research Council reported that from 14 to 29 November 2021 at a hospital complex in Tshwane, inpatients were younger than in previous waves and the ICU and oxygen therapy rates were lower than in earlier waves. These observations are not definitive and the clinical profile could change over the following two weeks, allowing for more accurate conclusions about disease severity. Excess deaths nearly doubled in the week of 28 November, suggesting under-reporting, but the level was still much lower than that seen in the second wave in mid-January 2021.\\nOn 12 December, director-general of the World Health Organization Tedros Adhanom asserted that it was wrong for people to consider Omicron as mild. This is because high exposure to previous infections in South Africa likely affects the clinical course of the new infections.On 20 December, a report by the Imperial College COVID-19 Response Team, based on data from England, found that hospitalisation and asymptomatic infection indicators were not significantly associated with Omicron infection, suggesting at most limited changes in severity compared with Delta. On 22 December, the team reported an approximately 41% (95% CI, 37–45%) lower risk of a hospitalization requiring a stay of at least 1 night compared to the Delta variant, and that the data suggest that recipients of 2 doses of the Pfizer–BioNTech, the Moderna or the Oxford–AstraZeneca vaccine remain substantially protected from hospitalization.In January, the CDC confirmed that the variant causes less severe disease than previously dominant variants. The novel Omicron subtype \\'BA.2\\' did not initially show an increase over this lower virulence, albeit one preprint found that \"infection experiments using hamsters show that BA.2 is more pathogenic than BA.1\". In March, a study from Hong Kong indicated that BA.2 may be more severe for children, compared to other variants. Nevertheless, in the U.S., the daily new COVID-19 deaths were higher during Omicron dominance than during Delta\\'s during fall and high volumes of hospitalizations due to the variant\\'s other characteristics can cause indirect harm via local health care system strains, beyond less severe but often non-mild disease effects.\\n\\n\\n== Diagnosis ==\\n\\n\\n=== PCR testing ===\\nThe FDA has published guidelines on how PCR tests will be affected by Omicron. Tests that detect multiple gene targets will continue to identify the testee as positive for COVID-19. S-gene dropout or target failure has been proposed as a shorthand way of differentiating Omicron from Delta. The variant can also be identified by sequencing and genotyping.\\n\\n\\n=== Rapid antigen testing ===\\nIn January 2022 the medicine and therapeutic regulatory agency Therapeutic Goods Administration (TGA) of the Australian Government found that only one of their 23 approved COVID-19 rapid antigen tests (RAT) stated that it detected Omicron.\\n\\n\\n== Prevention ==\\n\\nAs with other variants, the WHO recommended that people continue to keep enclosed spaces well ventilated, avoid crowding and close contact, wear well-fitting masks, clean hands frequently, and get vaccinated.\\nResponse by vaccine producersOn 26 November 2021, BioNTech said it would know in two weeks whether the current vaccine is effective against the variant and that an updated vaccine could be shipped in 100 days if necessary. AstraZeneca, Moderna and Johnson & Johnson were also studying the variant\\'s impact on the effectiveness of their vaccines. On the same day, Novavax stated that it was developing an updated vaccine requiring two doses for the Omicron variant, which the company expected to be ready for testing and manufacturing within a few weeks. On 29 November 2021, The Gamaleya Institute said that Sputnik Light should be effective against the variant, that it would begin adapting Sputnik V, and that a modified version could be ready for mass production in 45 days. Sinovac said it could quickly mass-produce an inactivated vaccine against the variant and that it was monitoring studies and collecting samples of the variant to determine if a new vaccine is needed. On 7 December 2021, at a symposium in Brazil with its partner Instituto Butantan, Sinovac said it would update its vaccine to the new variant and make it available in three months. On December 2, the Finlay Institute was already developing a version of Soberana Plus against the variant. Pfizer hopes to have a vaccine targeted to immunize against Omicron ready by March 2022.\\nWHOOn 29 November 2021, the WHO said cases and infections are expected among those vaccinated, albeit in a small and predictable proportion.\\nVaccine efficacyIn December, studies, some of which using large nationwide datasets from either Israel and Denmark, found that vaccine effectiveness of multiple common two-dosed COVID-19 vaccines is substantially lower against the Omicron variant than for other common variants including the Delta variant, and that a new (often a third) dose – a booster dose – is needed and effective, with it i.a. substantially reducing deaths from the disease compared to cohorts who received no booster but two doses.On 7 December 2021, preliminary results from a laboratory test conducted at the Africa Health Research Institute in Durban with 12 people who received the Pfizer-BioNTech vaccine found a 41-fold reduction in neutralizing antibody activity against the variant in some of the samples. This is a big reduction, but it does not mean that the variant can escape vaccines completely, so vaccination with current vaccines is still recommended. Neutralizing antibody activity against the variant was greater in those fully vaccinated after being infected about a year earlier. Effectiveness estimates will likely change as more data is collected, as antibodies generated by vaccination vary widely between individuals and the sample was small. On 8 December 2021, Pfizer and BioNTech reported that preliminary data indicated that a third dose of the vaccine would provide a similar level of neutralizing antibodies against the variant as seen against other variants after two doses.On 10 December 2021, the UK Health Security Agency reported that early data indicated a 20- to 40-fold reduction in neutralizing activity for Omicron by sera from Pfizer 2-dose vaccinees relative to earlier strains and a 20-fold reduction relative to Delta. The reduction was greater in sera from AstraZeneca 2-dose vaccinees, falling below the detectable threshold. An mRNA booster dose produced a similar increase in neutralising activity regardless of the vaccine used for primary vaccination. After a booster dose (usually with an mRNA vaccine), vaccine effectiveness against symptomatic disease was at 70%–75%, and the effectiveness against severe disease was expected to be higher.In January 2022, results from Israel suggested that a 4th dose of vaccine is only partially effective against the Omicron strain. Many cases of infection broke through, albeit \"a bit less than in the control group\", even though trial participants had higher antibody levels after the 4th dose.\\nWHO recommendations for epidemiologyOn 26 November 2021, the WHO asked nations to do the following:\\n\\nEnhance surveillance and sequencing efforts to better understand circulating SARS-CoV-2 variants.\\nSubmit complete genome sequences and associated metadata to a publicly available database, such as GISAID.\\nReport initial cases/clusters associated with virus-of-concern infection to WHO through the IHR mechanism.\\nWhere capacity exists and in coordination with the international community, perform field investigations and laboratory assessments to improve understanding of the potential impacts of the virus of concern on COVID-19 epidemiology, severity, and the effectiveness of public health and social measures, diagnostic methods, immune responses, antibody neutralization, or other relevant characteristics.\\n\\n\\n== Treatment ==\\n\\nCorticosteroids such as dexamethasone and IL6 receptor blockers such as tocilizumab (Actemra) are known to be effective for managing patients with the earlier strains of severe COVID-19.  The impact on the effectiveness of other treatments was being assessed in 2021.Relating to monoclonal antibodies (mAbs) treatments, similar testing and research is ongoing. Preclinical data on in vitro pseudotyped virus data demonstrate that some mAbs designed to use highly conserved epitopes retain neutralizing activity against key mutations of Omicron substitutions.\\nSimilar results are confirmed by cryo-electron microscopy and X-ray data, also providing the structural approach and molecular basis for the evasion of humoral immunity exhibited by Omicron antigenic shift as well as the importance of targeting conserved epitopes for vaccine and therapeutics design. While 7 clinical mAbs or mAb cocktails experienced loss of neutralizing activity of 1-2 orders of magnitude or greater relative to the prototypic virus, the S309 mAb, the parent mAb of sotrovimab, neutralized Omicron with only 2-3-fold reduced potency.\\nFurther data suggest Omicron would cause significant humoral immune evasion, while neutralizing antibodies targeting the sarbecovirus conserved region remain most effective.\\nIndeed, most receptor-binding motif (RBM)-directed monoclonal antibodies lost in vitro neutralizing activity against Omicron, with only 3 out of 29 mAbs examined in another study retaining unaltered potency. Furthermore, a fraction of broadly neutralizing sarbecovirus mAbs neutralized Omicron through recognition of antigenic sites outside the RBM, including sotrovimab (VIR-7831), S2X259 and S2H97. In particular, sotrovimab is not fully active against the BA.2 Omicron sublineage, and in early 2022 the office of the U.S. ASPR stopped distributing the antibody treatment to states where BA.2 was dominant.\\n\\n\\n== Epidemiology ==\\nOn 26 November 2021, the South African National Institute for Communicable Diseases announced that 30,904 COVID-tests (in one day) detected 2,828 new COVID infections (a 9.2% positivity rate).\\nOne week later, on 3 December 2021, the NICD announced that 65,990 COVID tests had found 16,055 new infections (5.7 times as many as seven days before; positive rate 24.3%) and that 72 percent of them were found in Gauteng.\\nThis province of South Africa is densely populated at about 850 inhabitants per km2. Gauteng\\'s capital Johannesburg is a megacity (about 5.5 million inhabitants in the city itself plus 9.5 million in the urban region).\\nIn November 2021, the transmissibility of the Omicron variant, as compared to the Delta variant or other variants of the COVID-19 virus, was still uncertain. Omicron is frequently able to infect previously Covid-positive people.It has been estimated the Omicron variant diverged in late September or early October 2021, based on Omicron genome comparisons.\\nSequencing data suggests that Omicron had become the dominant variant in South Africa by November 2021, the same month where it had been first identified in the country.Phylogeny suggests a recent emergence. Data from South Africa suggests that Omicron has a pronounced growth advantage there. However, this may be due to transmissibility or immune escape related, or both.\" Also the serial interval plays a role in the growth.\\nDetectable changes in levels of COVID-19 in wastewater samples from South Africa\\'s Gauteng province were seen as early as 17–23 October (week 42). The National Institute for Communicable Diseases reports that children under the age of 2 make up 10% of total hospital admissions in the Omicron point of discovery Tshwane in South Africa. Data on the S gene target failure (SGTF) of sampled cases in South Africa indicates a growth of 21% per day relative to Delta, generating an increased reproduction number by a factor of 2.4. Omicron became the majority strain in South Africa around 10 November. Another analysis showed 32% growth per day in Gauteng, South Africa, having become dominant there around 6 November.In the UK, the logarithmic growth rate of Omicron-associated S gene target failure (SGTF) cases over S gene target positive (SGTP) cases was estimated at 0.37 per day, which is exceptionally high. Furthermore, by 14 December it appears to have become the most dominant strain. Without presuming behavior change in response to the variant, a million infections per day by December 24 are projected for a 2.5 days doubling time. In Denmark, the growth rate has been roughly similar with a doubling time of about 2–3 days, it having become the most prevalent strain on 17 December. Switzerland is not far behind. In Germany Omicron became the most prevalent variant on January 1. In Scotland, Omicron apparently became the most prevalent variant on 17 December. In the Canadian province of Ontario it became the most prevalent strain on 13 December. In the US, the variant appears to have become the most prevalent strain on December 18, growing at 0.24 per day. In Portugal, Omicron had reached 61.5% of cases on 22 December. In Belgium, the strain has become the most prevalent on 25 December, and in the Netherlands on 28 December. In Italy, it had reached 28% of cases on 20 December and was doubling every two days, while it became the dominant variant in Norway on 25 December. In France, it made up about 15% of COVID-19 cases in mid-December, but around 27 December it had increased to more than 60%. Researchers recommend sampling at least 5% of COVID-19 patient samples in order to detect Omicron or other emerging variants.During January 2022, in Denmark the BA.2 variant grew at ~0.10 per day (+11% per day) as a ratio to BA.1 (the legacy Omicron variant), and became the dominant strain in week 2, 2022. In the United Kingdom, the BA.2 variant was growing at ~0.11 per day (+12% per day) as a ratio to BA.1.On 13 January 2022, the BBC reported that the hospitalization rate was higher in the US and Canada than in Europe and South Africa. This was attributed to a combination of a greater number of elderly people than in South Africa, greater prevalence of comorbidities such as hypertension and obesity than in Europe, higher indoor transmission due to the winter, lower vaccination rate in the US than in Europe and Canada, and a possible still high prevalence of the Delta variant, which more often leads to hospitalization.\\n\\n\\n== Statistics ==\\n\\nThe chance of detecting a case particularly depends on a country\\'s sequencing rate. For example, South Africa sequences far more samples than any other country in Africa, but at a considerably lower rate than most Western nations. Furthermore, it can take up to two weeks to return a viral sequence in places with the technical capability, hence solid statistics on confirmed cases lag the actual situation. Denmark and Norway regard cases found by their variant qPCR test, which is relatively fast and checks several genes, as sufficient for counting it as an Omicron, also before full sequencing.\\n\\n\\n== History ==\\nOn 26 November 2021, WHO designated B.1.1.529 as a variant of concern and named it \"Omicron\", after the fifteenth letter in the Greek alphabet. Omicron was first detected on 22 November 2021 in laboratories in Botswana and South Africa based on samples collected on 11–16 November. The first known sample was collected in South Africa on 8 November. The first known case, outside of South Africa, was a person arriving in Hong Kong from South Africa via Qatar on 11 November, and another person who arrived in Belgium from Egypt via Turkey on the same date. As of 7 January 2022, the variant has been confirmed in 135 countries. Omicron has an unusually large number of mutations compared to previous variants. Several of the mutations are novel and involve changes to the spike protein reducing the ability for COVID-19 vaccines to prevent symptomatic disease.A December 2021 article in Science observes Omicron did not evolve from any other variant of note, but instead on a distinct track diverging in perhaps mid-2020. The article expounds on three theories that might explain this surprising genetic lineage:\\n\\nThe virus could have circulated and evolved in a population with little surveillance and sequencing.\\nIt could have gestated in a chronically infected COVID-19 patient.\\nIt might have evolved in a nonhuman species, from which it recently spilled back into people.\\n\\n\\n=== Reported cases ===\\n\\nOn 24 November 2021, the variant was first reported to the WHO from South Africa, based on samples that had been collected from 14 to 16 November. South African scientists were first alerted by samples from the very beginning of November where the PCR tests had S gene target failure (occurs in a few variants, but not in Delta which dominated in the country in October) and by a sudden increase of COVID-19 cases in Gauteng; sequencing revealed that more than 70 percent of samples collected in the province between 14 and 23 November were a new variant. The first confirmed specimens of Omicron were collected on 8 November 2021 in South Africa, and on 9 November in Botswana. Likely Omicron (SGTF) samples had occurred  on 4 November 2021 in Pretoria, South Africa.When WHO was alerted on 24 November, Hong Kong was the only place outside Africa that had confirmed a case of Omicron; one person who traveled from South Africa on 11 November, and another traveler who was cross-infected by this case while staying in the same quarantine hotel.On 25 November, one confirmed case was identified in Israel from a traveler returning from Malawi, along with two who returned from South Africa and one from Madagascar. All four initial cases reported from Botswana occurred among fully vaccinated individuals.On 26 November, Belgium confirmed its first case; an unvaccinated person who had travelled from Egypt via Turkey on 11 November. All three initial confirmed and suspected cases reported from Israel occurred among fully vaccinated individuals, as did a single suspected case in Germany.On 27 November, two cases were detected in the United Kingdom, another two in Munich, Germany and one in Milan, Italy.On 28 November, 13 cases were confirmed in the Netherlands among the 624 airline passengers who arrived from South Africa on 26 November. Confirmation of a further 5 cases among these passengers followed later. Entry into the Netherlands generally required having been vaccinated or PCR-tested, or having recovered. The passengers of these two flights had been tested upon arrival because of the newly imposed restrictions (which were set in place during their flight), after which 61 tested positive for SARS-CoV-2. A further two cases were detected in Australia. Both people landed in Sydney the previous day, and travelled from southern Africa to Sydney Airport via Doha Airport. The two people, who were fully vaccinated, entered isolation; 12 other travellers from southern Africa also entered quarantine for fourteen days, while about 260 other passengers and crew on the flight have been directed to isolate. Two travellers from South Africa who landed in Denmark tested positive for COVID-19; it was confirmed on 28 November that both carried the Omicron variant. On the same day, Austria also confirmed its first Omicron case. A detected Omicron case was reported in the Czech Republic, from a traveler who spent time in Namibia. Canada also reported its first Omicron cases, with two from travelers from Nigeria, therefore becoming the first North American country to report an Omicron case.On 29 November, a positive case was recorded in Darwin, Australia. The person arrived in Darwin on a repatriation flight from Johannesburg, South Africa on 25 November, and was taken to a quarantine facility, where the positive test was recorded. Two more people who travelled to Sydney from southern Africa via Singapore tested positive. Portugal reported 13 Omicron cases, all of them members of a soccer club. Sweden also confirmed their first case on 29 November, as did Spain, when a traveler came from South Africa.On 30 November, the Netherlands reported that Omicron cases had been detected in two samples dating back as early as 19 November. A positive case was recorded in Sydney from a traveller who had visited southern Africa before travel restrictions were imposed, and was subsequently active in the community. Japan also confirmed its first case. Two Israeli doctors have tested positive and have entered isolation. Both of them had received three shots of the Pfizer vaccine prior to testing positive. In Brazil, three cases of the Omicron variant were confirmed in São Paulo. Another five are under suspicion. A person in Leipzig, Germany with no travel history nor contact with travellers tested positive for Omicron.On 1 December, the Omicron variant was detected in three samples in Nigeria that had been collected from travelers from South Africa within the last week. On the same day, public health authorities in the United States announced the country\\'s first confirmed Omicron case. A resident of San Francisco who had been vaccinated returned from South Africa on 22 November, began showing mild symptoms on 25 November and was confirmed to have a mild case of COVID-19 on 29 November. Ireland and South Korea also reported their first cases. South Korea reported its cases from five travelers arriving in South Korea from Nigeria.On 2 December, Dutch health authorities confirmed that all 14 passengers with confirmed Omicron infection on 26 November had been previously vaccinated. The same day, the Norwegian Institute of Public Health confirmed that 50 attendees of a company Christmas party held at a restaurant in Norway\\'s capital, Oslo, were infected with the Omicron variant. France has confirmed only 25 cases of the new Omicron variant but officials say the number could jump significantly in the coming weeks.By 6 December, Malaysia confirmed its first case of the variant. The case was a South African student entering to study at a private university. In Namibia, 18 cases out of 19 positive COVID-19 samples that had been collected between 11 and 26 November were found to be Omicron, indicating a high level of prevalence in the country. Fiji also confirmed two positive cases of the variant. They travelled from Nigeria arriving in Fiji on November 25.On 8 December, WHO announced the variant had been detected in 57 countries.On 9 December, Richard Mihigo, coordinator of the World Health Organisation\\'s Immunisation and Vaccine Development Programme for Africa, announced that Africa accounted for 46% of reported cases of the Omicron variant globally.On 13 December, the first death of a person with Omicron was reported in the UK.On 16 December, New Zealand confirmed its first case of the Omicron variant, an individual who had traveled from Germany via Dubai.The first death of a person with Omicron was reported in Germany on 23 December and in Australia on 27 December.On 3 January 2022, South Korea reported the first two deaths of people who tested positive post mortem for Omicron.On 29 March 2022, Omicron subvariant BA.2 became the dominant strain in the U.S.\\n\\n\\n=== Market reactions ===\\n\\nWorry about the potential economic impact of the Omicron variant led to a drop in global markets on 26 November, including the worst drop of the Dow Jones Industrial Average in 2021, led by travel-related stocks. The price of Brent Crude and West Texas Intermediate oil fell 10% and 11.7%, respectively. Cryptocurrency markets were also routed. The South African rand has also hit an all-time low for 2021, trading at over 16 rand to the dollar, losing 6% of its value in November.In early December 2021, the chairman of the Federal Reserve, Jerome Powell,  testified before the U.S. Senate Committee on Banking that \"The recent rise in COVID-19 cases and the emergence of the Omicron variant pose downside risks to employment and economic activity and increased uncertainty for inflation.\"\\n\\n\\n=== International response ===\\nOn 26 November 2021, WHO advised countries not to impose new restrictions on travel, instead recommending a \"risk-based and scientific\" approach to travel measures. On the same day, the European Centre for Disease Prevention and Control (ECDC) reported modeling indicating that strict travel restrictions would delay the variant\\'s impact on European countries by two weeks, possibly allowing countries to prepare for it.After the WHO announcement, on the same day, several countries announced travel bans from southern Africa in response to the identification of the variant, including the United States, which banned travel from eight African countries, although it notably did not ban travel from any European countries, Israel, Canada, or Australia where cases were also detected at the time the bans were announced. Other countries that also implemented travel bans include Japan, Canada, the European Union, Israel, Australia, the United Kingdom, Singapore, Malaysia, Indonesia, Morocco, and New Zealand.The Brazilian Health Regulatory Agency recommended flight restrictions regarding the new variant. The state of New York declared a state of emergency ahead of a potential Omicron spike, although no cases had yet been detected in the state or the rest of the United States. On 27 November, Switzerland introduced obligatory tests and quarantine for all visitors arriving from countries where the variant was detected, which originally included Belgium and Israel.In response to the various travel bans, South African Minister of Health Joe Phaahla defended his country\\'s handling of the pandemic and said that travel bans went against the \"norms and standards\" of the World Health Organization.Some speculate that travel bans could have a significant impact on South Africa\\'s economy by limiting tourism and could lead to other countries with economies that are reliant on tourism to hide the discovery of new variants of concern. Low vaccine coverage in less-developed nations could create opportunities for the emergence of new variants, and these nations also struggle to gain intellectual property to develop and produce vaccines locally. At the same time, inoculation has slowed in South Africa due to vaccine hesitancy and apathy, with a nationwide vaccination rate of only 35% as of November 2021.On 29 November, the WHO warned countries that the variant poses a very high global risk with severe consequences and that they should prepare by accelerating vaccination of high-priority groups and strengthening health systems. WHO director-general Tedros Adhanom described the global situation as dangerous and precarious and called for a new agreement on the handling of pandemics, as the current system disincentivizes countries from alerting others to threats that will inevitably land on their shores. CEPI CEO Richard Hatchett said that the variant fulfilled predictions that transmission of the virus in low-vaccination areas would accelerate its evolution.In preparation for the Omicron variant arriving in the United States, President Joe Biden has stated that the variant is \"cause for concern, not panic\" and reiterated that the government is prepared for the variant and will have it under control. He also stated that large-scale lockdowns, similar to the ones in 2020 near the beginning of the pandemic, are \"off the table for now.\"In mid-December, multiple Canadian provinces reinstated restrictions on gatherings and events such as sports tournaments, and tightened enforcement of proof of vaccination orders. British Columbia expressly prohibited any non-seated \"organized New Year\\'s Eve event\", while Quebec announced a partial lockdown on 20 December, ordering the closure of all bars, casinos, gyms, schools, and theatres, as well as imposing restrictions on the capacity and operating hours of restaurants, and the prohibition of spectators at professional sporting events.On 18 December, the Netherlands government announced a lockdown intended to prevent spread of the variant during the holiday period.In late December, some countries shortened the typical six-month interval for a booster dose of the vaccine to prepare for a wave of Omicron, as two doses are not enough to stop the infection. UK, South Korea and Thailand reduced to three months; Belgium, four months; France, Singapore, Taiwan, Italy and Australia, five months. Finland reduced it to three months for risk groups. Other countries continued with a six-month booster schedule. While antibody levels begin to drop at four months, a longer interval usually allows time for the immune system\\'s response to mature.\\n\\n\\n== See also ==\\n\\nTimeline of the SARS-CoV-2 Omicron variant\\nCOVID-19 pandemic in Africa\\nCOVID-19 vaccination in Africa\\nCOVID-19 vaccination in Botswana\\nCOVID-19 vaccination in South Africa\\nVariants of SARS-CoV-2\\nOther variants of either interest or concern: Alpha, Beta, Gamma, Delta, Epsilon, Zeta, Eta, Theta, Iota, Kappa, Lambda, Mu\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\nWHO: Update on Omicron\\nLive Omicron Variant Location Map\\nOmicron variant: What have COVID vaccine makers said and are they working on new doses?\\nCallaway E, Ledford H (December 2021). \"How bad is Omicron? What scientists know so far\". Nature. 600 (7888): 197–199. Bibcode:2021Natur.600..197C. doi:10.1038/d41586-021-03614-z. PMID 34857948. S2CID 244840636.\\n\\n\\n== External links ==\\nCorum J, Zimmer C (9 February 2021). \"Coronavirus Variant Tracker\". The New York Times. Archived from the original on 30 November 2021. Retrieved 1 December 2021. Constantly Updated\\nArchived video:\\nMohale F, Phaahla J, Oliveira T, Lessels R, Gottberg A, Moore P, Crisp N (25 November 2021). Health Department briefs media about a new so-called super-variant (Internet video live stream). South Africa: eNCA. Retrieved 27 November 2021. Archived at the Wayback Machine and Ghostarchive.\\nOliveira T, Kamwendo S (26 November 2021). Professor Tulio de Oliviera on the new COVID-19 variant (Internet video live stream). South Africa: SABC News. Archived from the original on 29 November 2021. Retrieved 27 November 2021. Archived at the Wayback Machine and Ghostarchive',\n",
              " 'A virus is a submicroscopic infectious agent that replicates only inside the living cells of an organism. Viruses infect all life forms, from animals and plants to microorganisms, including bacteria and archaea.\\nSince Dmitri Ivanovsky\\'s 1892 article describing a non-bacterial pathogen infecting tobacco plants and the discovery of the tobacco mosaic virus by Martinus Beijerinck in 1898, more than 9,000 virus species have been described in detail of the millions of types of viruses in the environment. Viruses are found in almost every ecosystem on Earth and are the most numerous type of biological entity. The study of viruses is known as virology, a subspeciality of microbiology.  \\nWhen infected, a host cell is often forced to rapidly produce thousands of copies of the original virus. When not inside an infected cell or in the process of infecting a cell, viruses exist in the form of independent particles, or virions, consisting of (i) the genetic material, i.e., long molecules of DNA or RNA that encode the structure of the proteins by which the virus acts; (ii) a protein coat, the capsid, which surrounds and protects the genetic material; and in some cases (iii) an outside envelope of lipids. The shapes of these virus particles range from simple helical and icosahedral forms to more complex structures. Most virus species have virions too small to be seen with an optical microscope and are one-hundredth the size of most bacteria.\\nThe origins of viruses in the evolutionary history of life are unclear: some may have evolved from plasmids—pieces of DNA that can move between cells—while others may have evolved from bacteria. In evolution, viruses are an important means of horizontal gene transfer, which increases genetic diversity in a way analogous to sexual reproduction. Viruses are considered by some biologists to be a life form, because they carry genetic material, reproduce, and evolve through natural selection, although they lack the key characteristics, such as cell structure, that are generally considered necessary criteria for defining life. Because they possess some but not all such qualities, viruses have been described as \"organisms at the edge of life\", and as replicators.Viruses spread in many ways. One transmission pathway is through disease-bearing organisms known as vectors: for example, viruses are often transmitted from plant to plant by insects that feed on plant sap, such as aphids; and viruses in animals can be carried by blood-sucking insects. Many viruses, including influenza viruses, SARS-CoV-2, chickenpox, smallpox, and measles, spread in the air by coughing and sneezing. Norovirus and rotavirus, common causes of viral gastroenteritis, are transmitted by the faecal–oral route, passed by hand-to-mouth contact or in food or water. The infectious dose of norovirus required to produce infection in humans is fewer than 100 particles. HIV is one of several viruses transmitted through sexual contact and by exposure to infected blood. The variety of host cells that a virus can infect is called its \"host range\". This can be narrow, meaning a virus is capable of infecting few species, or broad, meaning it is capable of infecting many.Viral infections in animals provoke an immune response that usually eliminates the infecting virus. Immune responses can also be produced by vaccines, which confer an artificially acquired immunity to the specific viral infection. Some viruses, including those that cause HIV/AIDS, HPV infection, and viral hepatitis, evade these immune responses and result in chronic infections. Several classes of antiviral drugs have been developed.\\n\\n\\n== Etymology ==\\nThe word is from the Latin neuter vīrus referring to poison and other noxious liquids, from the same Indo-European base as Sanskrit viṣa, Avestan vīša, and ancient Greek ἰός (all meaning \\'poison\\'), first attested in English in 1398 in John Trevisa\\'s translation of Bartholomeus Anglicus\\'s De Proprietatibus Rerum. Virulent, from Latin virulentus (\\'poisonous\\'), dates to c. 1400. A meaning of \\'agent that causes infectious disease\\' is first recorded in 1728, long before the discovery of viruses by Dmitri Ivanovsky in 1892. The English plural is viruses (sometimes also  vira),  whereas the Latin word is a mass noun, which has no classically attested plural (vīra is used in Neo-Latin). The adjective viral dates to 1948. The term virion (plural virions), which dates from 1959, is also used to refer to a single viral particle that is released from the cell and is capable of infecting other cells of the same type.\\n\\n\\n== History ==\\n\\nLouis Pasteur was unable to find a causative agent for rabies and speculated about a pathogen too small to be detected by microscopes. In 1884, the French microbiologist Charles Chamberland invented the Chamberland filter (or Pasteur-Chamberland filter) with pores small enough to remove all bacteria from a solution passed through it. In 1892, the Russian biologist Dmitri Ivanovsky used this filter to study what is now known as the tobacco mosaic virus: crushed leaf extracts from infected tobacco plants remained infectious even after filtration to remove bacteria. Ivanovsky suggested the infection might be caused by a toxin produced by bacteria, but he did not pursue the idea. At the time it was thought that all infectious agents could be retained by filters and grown on a nutrient medium—this was part of the germ theory of disease. In 1898, the Dutch microbiologist Martinus Beijerinck repeated the experiments and became convinced that the filtered solution contained a new form of infectious agent. He observed that the agent multiplied only in cells that were dividing, but as his experiments did not show that it was made of particles, he called it a contagium vivum fluidum (soluble living germ) and reintroduced the word virus. Beijerinck maintained that viruses were liquid in nature, a theory later discredited by Wendell Stanley, who proved they were particulate. In the same year, Friedrich Loeffler and Paul Frosch passed the first animal virus, aphthovirus (the agent of foot-and-mouth disease), through a similar filter.In the early 20th century, the English bacteriologist Frederick Twort discovered a group of viruses that infect bacteria, now called bacteriophages (or commonly \\'phages\\'), and the French-Canadian microbiologist Félix d\\'Herelle described viruses that, when added to bacteria on an agar plate, would produce areas of dead bacteria. He accurately diluted a suspension of these viruses and discovered that the highest dilutions (lowest virus concentrations), rather than killing all the bacteria, formed discrete areas of dead organisms. Counting these areas and multiplying by the dilution factor allowed him to calculate the number of viruses in the original suspension. Phages were heralded as a potential treatment for diseases such as typhoid and cholera, but their promise was forgotten with the development of penicillin. The development of bacterial resistance to antibiotics has renewed interest in the therapeutic use of bacteriophages.By the end of the 19th century, viruses were defined in terms of their infectivity, their ability to pass filters, and their requirement for living hosts. Viruses had been grown only in plants and animals. In 1906 Ross Granville Harrison invented a method for growing tissue in lymph, and in 1913 E. Steinhardt, C. Israeli, and R.A. Lambert used this method to grow vaccinia virus in fragments of guinea pig corneal tissue. In 1928, H. B. Maitland and M. C. Maitland grew vaccinia virus in suspensions of minced hens\\' kidneys. Their method was not widely adopted until the 1950s when poliovirus was grown on a large scale for vaccine production.Another breakthrough came in 1931 when the American pathologist Ernest William Goodpasture and Alice Miles Woodruff grew influenza and several other viruses in fertilised chicken eggs. In 1949, John Franklin Enders, Thomas Weller, and Frederick Robbins grew poliovirus in cultured cells from aborted human embryonic tissue, the first virus to be grown without using solid animal tissue or eggs. This work enabled Hilary Koprowski, and then Jonas Salk, to make an effective polio vaccine.The first images of viruses were obtained upon the invention of electron microscopy in 1931 by the German engineers Ernst Ruska and Max Knoll. In 1935, American biochemist and virologist Wendell Meredith Stanley examined the tobacco mosaic virus and found it was mostly made of protein. A short time later, this virus was separated into protein and RNA parts.\\nThe tobacco mosaic virus was the first to be crystallised and its structure could, therefore, be elucidated in detail. The first X-ray diffraction pictures of the crystallised virus were obtained by Bernal and Fankuchen in 1941. Based on her X-ray crystallographic pictures, Rosalind Franklin discovered the full structure of the virus in 1955. In the same year, Heinz Fraenkel-Conrat and Robley Williams showed that purified tobacco mosaic virus RNA and its protein coat can assemble by themselves to form functional viruses, suggesting that this simple mechanism was probably the means through which viruses were created within their host cells.The second half of the 20th century was the golden age of virus discovery, and most of the documented species of animal, plant, and bacterial viruses were discovered during these years. In 1957 equine arterivirus and the cause of Bovine virus diarrhoea (a pestivirus) were discovered. In 1963 the hepatitis B virus was discovered by Baruch Blumberg, and in 1965 Howard Temin described the first retrovirus. Reverse transcriptase, the enzyme that retroviruses use to make DNA copies of their RNA, was first described in 1970 by Temin and David Baltimore independently. In 1983 Luc Montagnier\\'s team at the Pasteur Institute in France, first isolated the retrovirus now called HIV. In 1989 Michael Houghton\\'s team at Chiron Corporation discovered hepatitis C.\\n\\n\\n== Origins ==\\n\\nViruses are found wherever there is life and have probably existed since living cells first evolved. The origin of viruses is unclear because they do not form fossils, so molecular techniques are used to investigate how they arose. In addition, viral genetic material occasionally integrates into the germline of the host organisms, by which they can be passed on vertically to the offspring of the host for many generations. This provides an invaluable source of information for paleovirologists to trace back ancient viruses that have existed up to millions of years ago. There are three main hypotheses that aim to explain the origins of viruses:\\nRegressive hypothesis\\nViruses may have once been small cells that parasitised larger cells. Over time, genes not required by their parasitism were lost. The bacteria rickettsia and chlamydia are living cells that, like viruses, can reproduce only inside host cells. They lend support to this hypothesis, as their dependence on parasitism is likely to have caused the loss of genes that enabled them to survive outside a cell. This is also called the \\'degeneracy hypothesis\\', or \\'reduction hypothesis\\'.\\nCellular origin hypothesis\\nSome viruses may have evolved from bits of DNA or RNA that \"escaped\" from the genes of a larger organism. The escaped DNA could have come from plasmids (pieces of naked DNA that can move between cells) or transposons (molecules of DNA that replicate and move around to different positions within the genes of the cell). Once called \"jumping genes\", transposons are examples of mobile genetic elements and could be the origin of some viruses. They were discovered in maize by Barbara McClintock in 1950. This is sometimes called the \\'vagrancy hypothesis\\', or the \\'escape hypothesis\\'.\\nCo-evolution hypothesis\\nThis is also called the \\'virus-first hypothesis\\' and proposes that viruses may have evolved from complex molecules of protein and nucleic acid at the same time that cells first appeared on Earth and would have been dependent on cellular life for billions of years. Viroids are molecules of RNA that are not classified as viruses because they lack a protein coat. They have characteristics that are common to several viruses and are often called subviral agents. Viroids are important pathogens of plants. They do not code for proteins but interact with the host cell and use the host machinery for their replication. The hepatitis delta virus of humans has an RNA genome similar to viroids but has a protein coat derived from hepatitis B virus and cannot produce one of its own. It is, therefore, a defective virus. Although hepatitis delta virus genome may replicate independently once inside a host cell, it requires the help of hepatitis B virus to provide a protein coat so that it can be transmitted to new cells. In similar manner, the sputnik virophage is dependent on mimivirus, which infects the protozoan Acanthamoeba castellanii. These viruses, which are dependent on the presence of other virus species in the host cell, are called \\'satellites\\' and may represent evolutionary intermediates of viroids and viruses.In the past, there were problems with all of these hypotheses: the regressive hypothesis did not explain why even the smallest of cellular parasites do not resemble viruses in any way. The escape hypothesis did not explain the complex capsids and other structures on virus particles. The virus-first hypothesis contravened the definition of viruses in that they require host cells. Viruses are now recognised as ancient and as having origins that pre-date the divergence of life into the three domains. This discovery has led modern virologists to reconsider and re-evaluate these three classical hypotheses.The evidence for an ancestral world of RNA cells and computer analysis of viral and host DNA sequences are giving a better understanding of the evolutionary relationships between different viruses and may help identify the ancestors of modern viruses. To date, such analyses have not proved which of these hypotheses is correct. It seems unlikely that all currently known viruses have a common ancestor, and viruses have probably arisen numerous times in the past by one or more mechanisms.\\n\\n\\n== Microbiology ==\\n\\n\\n=== Life properties ===\\nScientific opinions differ on whether viruses are a form of life or organic structures that interact with living organisms. They have been described as \"organisms at the edge of life\", since they resemble organisms in that they possess genes, evolve by natural selection, and reproduce by creating multiple copies of themselves through self-assembly. Although they have genes, they do not have a cellular structure, which is often seen as the basic unit of life. Viruses do not have their own metabolism and require a host cell to make new products. They therefore cannot naturally reproduce outside a host cell—although some bacteria such as rickettsia and chlamydia are considered living organisms despite the same limitation. Accepted forms of life use cell division to reproduce, whereas viruses spontaneously assemble within cells. They differ from autonomous growth of crystals as they inherit genetic mutations while being subject to natural selection. Virus self-assembly within host cells has implications for the study of the origin of life, as it lends further credence to the hypothesis that life could have started as self-assembling organic molecules.\\n\\n\\n=== Structure ===\\n\\nViruses display a wide diversity of sizes and shapes, called \\'morphologies\\'. In general, viruses are much smaller than bacteria and more than a thousand bacteriophage viruses would fit inside an Escherichia coli bacterium\\'s cell. Many viruses that have been studied are spherical and have a diameter between 20 and 300 nanometres. Some filoviruses, which are filaments have a total length of up to 1400 nm; their diameters are only about 80 nm. Most viruses cannot be seen with an optical microscope, so scanning and transmission electron microscopes are used to visualise them. To increase the contrast between viruses and the background, electron-dense \"stains\" are used. These are solutions of salts of heavy metals, such as tungsten, that scatter the electrons from regions covered with the stain. When virions are coated with stain (positive staining), fine detail is obscured. Negative staining overcomes this problem by staining the background only.A complete virus particle, known as a virion, consists of nucleic acid surrounded by a protective coat of protein called a capsid. These are formed from  protein subunits called capsomeres. Viruses can have a lipid \"envelope\" derived from the host cell membrane. The capsid is made from proteins encoded by the viral genome and its shape serves as the basis for morphological distinction. Virally-coded protein subunits will self-assemble to form a capsid, in general requiring the presence of the virus genome. Complex viruses code for proteins that assist in the construction of their capsid. Proteins associated with nucleic acid are known as nucleoproteins, and the association of viral capsid proteins with viral nucleic acid is called a nucleocapsid. The capsid and entire virus structure can be mechanically (physically) probed through atomic force microscopy. In general, there are five main morphological virus types:\\n\\nHelical\\nThese viruses are composed of a single type of capsomere stacked around a central axis to form a helical structure, which may have a central cavity, or tube. This arrangement results in virions which can be short and highly rigid rods, or long and very flexible filaments. The genetic material (typically single-stranded RNA, but single-stranded DNA in some cases) is bound into the protein helix by interactions between the negatively charged nucleic acid and positive charges on the protein. Overall, the length of a helical capsid is related to the length of the nucleic acid contained within it, and the diameter is dependent on the size and arrangement of capsomeres. The well-studied tobacco mosaic virus and inovirus  are examples of helical viruses.Icosahedral\\nMost animal viruses are icosahedral or near-spherical with chiral icosahedral symmetry. A regular icosahedron is the optimum way of forming a closed shell from identical subunits. The minimum number of capsomeres required for each triangular face is 3, which gives 60 for the icosahedron. Many viruses, such as rotavirus, have more than 60 capsomers and appear spherical but they retain this symmetry. To achieve this, the capsomeres at the apices are surrounded by five other capsomeres and are called pentons. Capsomeres on the triangular faces are surrounded by six others and are called hexons. Hexons are in essence flat and pentons, which form the 12 vertices, are curved. The same protein may act as the subunit of both the pentamers and hexamers or they may be composed of different proteins.Prolate\\nThis is an icosahedron elongated along the fivefold axis and is a common arrangement of the heads of bacteriophages. This structure is composed of a cylinder with a cap at either end.Enveloped\\nSome species of virus envelop themselves in a modified form of one of the cell membranes, either the outer membrane surrounding an infected host cell or internal membranes such as a nuclear membrane or endoplasmic reticulum, thus gaining an outer lipid bilayer known as a viral envelope. This membrane is studded with proteins coded for by the viral genome and host genome; the lipid membrane itself and any carbohydrates present originate entirely from the host. Influenza virus, HIV (which causes AIDS), and severe acute respiratory syndrome coronavirus 2 (which causes COVID-19) use this strategy. Most enveloped viruses are dependent on the envelope for their infectivity.Complex\\nThese viruses possess a capsid that is neither purely helical nor purely icosahedral, and that may possess extra structures such as protein tails or a complex outer wall. Some bacteriophages, such as Enterobacteria phage T4, have a complex structure consisting of an icosahedral head bound to a helical tail, which may have a hexagonal base plate with protruding protein tail fibres. This tail structure acts like a molecular syringe, attaching to the bacterial host and then injecting the viral genome into the cell.The poxviruses are large, complex viruses that have an unusual morphology. The viral genome is associated with proteins within a central disc structure known as a nucleoid. The nucleoid is surrounded by a membrane and two lateral bodies of unknown function. The virus has an outer envelope with a thick layer of protein studded over its surface. The whole virion is slightly pleomorphic, ranging from ovoid to brick-shaped.\\n\\n\\n==== Giant viruses ====\\n\\nMimivirus is one of the largest characterised viruses, with a capsid diameter of 400 nm. Protein filaments measuring 100 nm project from the surface. The capsid appears hexagonal under an electron microscope, therefore the capsid is probably icosahedral. In 2011, researchers discovered the largest then known virus in samples of water collected from the ocean floor off the coast of Las Cruces, Chile. Provisionally named Megavirus chilensis, it can be seen with a basic optical microscope. In 2013, the Pandoravirus genus was discovered in Chile and Australia, and has genomes about twice as large as Megavirus and Mimivirus. All giant viruses have dsDNA genomes and they are classified into several families: Mimiviridae, Pithoviridae, Pandoraviridae, Phycodnaviridae, and the Mollivirus genus.Some viruses that infect Archaea have complex structures unrelated to any other form of virus, with a wide variety of unusual shapes, ranging from spindle-shaped structures to viruses that resemble hooked rods, teardrops or even bottles. Other archaeal viruses resemble the tailed bacteriophages, and can have multiple tail structures.\\n\\n\\n=== Genome ===\\nAn enormous variety of genomic structures can be seen among viral species; as a group, they contain more structural genomic diversity than plants, animals, archaea, or bacteria. There are millions of different types of viruses, although fewer than 7,000 types have been described in detail. As of January 2021, the NCBI Virus genome database has more than 193,000 complete genome sequences, but there are doubtlessly many more to be discovered.A virus has either a DNA or an RNA genome and is called a DNA virus or an RNA virus, respectively. The vast majority of viruses have RNA genomes. Plant viruses tend to have single-stranded RNA genomes and bacteriophages tend to have double-stranded DNA genomes.Viral genomes are circular, as in the polyomaviruses, or linear, as in the adenoviruses. The type of nucleic acid is irrelevant to the shape of the genome. Among RNA viruses and certain DNA viruses, the genome is often divided up into separate parts, in which case it is called segmented. For RNA viruses, each segment often codes for only one protein and they are usually found together in one capsid. All segments are not required to be in the same virion for the virus to be infectious, as demonstrated by brome mosaic virus and several other plant viruses.A viral genome, irrespective of nucleic acid type, is almost always either single-stranded (ss) or double-stranded (ds). Single-stranded genomes consist of an unpaired nucleic acid, analogous to one-half of a ladder split down the middle. Double-stranded genomes consist of two complementary paired nucleic acids, analogous to a ladder. The virus particles of some virus families, such as those belonging to the Hepadnaviridae, contain a genome that is partially double-stranded and partially single-stranded.For most viruses with RNA genomes and some with single-stranded DNA (ssDNA) genomes, the single strands are said to be either positive-sense (called the \\'plus-strand\\') or negative-sense (called the \\'minus-strand\\'), depending on if they are complementary to the viral messenger RNA (mRNA). Positive-sense viral RNA is in the same sense as viral mRNA and thus at least a part of it can be immediately translated by the host cell. Negative-sense viral RNA is complementary to mRNA and thus must be converted to positive-sense RNA by an RNA-dependent RNA polymerase before translation. DNA nomenclature for viruses with genomic ssDNA is similar to RNA nomenclature, in that positive-strand viral ssDNA is identical in sequence to the viral mRNA and is thus a coding strand, while negative-sense viral ssDNA is complementary to the viral mRNA and is thus a template strand. Several types of ssDNA and ssRNA viruses have genomes that are ambisense in that transcription can occur off both strands in a double-stranded replicative intermediate. Examples include geminiviruses, which are ssDNA plant viruses and arenaviruses, which are ssRNA viruses of animals.\\n\\n\\n==== Genome size ====\\nGenome size varies greatly between species. The smallest—the ssDNA circoviruses, family Circoviridae—code for only two proteins and have a genome size of only two kilobases; the largest—the pandoraviruses—have genome sizes of around two megabases which code for about 2500 proteins. Virus genes rarely have introns and often are arranged in the genome so that they overlap.In general, RNA viruses have smaller genome sizes than DNA viruses because of a higher error-rate when replicating, and have a maximum upper size limit. Beyond this, errors when replicating render the virus useless or uncompetitive. To compensate, RNA viruses often have segmented genomes—the genome is split into smaller molecules—thus reducing the chance that an error in a single-component genome will incapacitate the entire genome. In contrast, DNA viruses generally have larger genomes because of the high fidelity of their replication enzymes. Single-strand DNA viruses are an exception to this rule, as mutation rates for these genomes can approach the extreme of the ssRNA virus case.\\n\\n\\n=== Genetic mutation ===\\n\\nViruses undergo genetic change by several mechanisms. These include a process called antigenic drift where individual bases in the DNA or RNA mutate to other bases. Most of these point mutations are \"silent\"—they do not change the protein that the gene encodes—but others can confer evolutionary advantages such as resistance to antiviral drugs. Antigenic shift occurs when there is a major change in the genome of the virus. This can be a result of recombination or reassortment. When this happens with influenza viruses, pandemics might result. RNA viruses often exist as quasispecies or swarms of viruses of the same species but with slightly different genome nucleoside sequences. Such quasispecies are a prime target for natural selection.Segmented genomes confer evolutionary advantages; different strains of a virus with a segmented genome can shuffle and combine genes and produce progeny viruses (or offspring) that have unique characteristics. This is called reassortment or \\'viral sex\\'.Genetic recombination is the process by which a strand of DNA (or RNA) is broken and then joined to the end of a different DNA (or RNA)  molecule. This can occur when viruses infect cells simultaneously and studies of viral evolution have shown that recombination has been rampant in the species studied. Recombination is common to both RNA and DNA viruses.\\n\\n\\n=== Replication cycle ===\\n\\nViral populations do not grow through cell division, because they are acellular. Instead, they use the machinery and metabolism of a host cell to produce multiple copies of themselves, and they assemble in the cell. When infected, the host cell is forced to rapidly produce thousands of  copies of the original virus.Their life cycle differs greatly between species, but there are six basic stages in their life cycle:Attachment is a specific binding between viral capsid proteins and specific receptors on the host cellular surface. This specificity determines the host range and type of host cell of a virus. For example, HIV infects a limited range of human leucocytes. This is because its surface protein, gp120, specifically interacts with the CD4 molecule—a chemokine receptor—which is most commonly found on the surface of CD4+ T-Cells. This mechanism has evolved to favour those viruses that infect only cells in which they are capable of replication. Attachment to the receptor can induce the viral envelope protein to undergo changes that result in the fusion of viral and cellular membranes, or changes of non-enveloped virus surface proteins that allow the virus to enter.Penetration or viral entry follows attachment: Virions enter the host cell through receptor-mediated endocytosis or membrane fusion. The infection of plant and fungal cells is different from that of animal cells. Plants have a rigid cell wall made of cellulose, and fungi one of chitin, so most viruses can get inside these cells only after trauma to the cell wall. Nearly all plant viruses (such as tobacco mosaic virus) can also move directly from cell to cell, in the form of single-stranded nucleoprotein complexes, through pores called plasmodesmata. Bacteria, like plants, have strong cell walls that a virus must breach to infect the cell. Given that bacterial cell walls are much thinner than plant cell walls due to their much smaller size, some viruses have evolved mechanisms that inject their genome into the bacterial cell across the cell wall, while the viral capsid remains outside.Uncoating is a process in which the viral capsid is removed: This may be by degradation by viral enzymes or host enzymes or by simple dissociation; the end-result is the releasing of the viral genomic nucleic acid.Replication of viruses involves primarily multiplication of the genome. Replication involves the synthesis of viral messenger RNA (mRNA) from \"early\" genes (with exceptions for positive-sense RNA viruses), viral protein synthesis, possible assembly of viral proteins, then viral genome replication mediated by early or regulatory protein expression. This may be followed, for complex viruses with larger genomes, by one or more further rounds of mRNA synthesis: \"late\" gene expression is, in general, of structural or virion proteins.Assembly – Following the structure-mediated self-assembly of the virus particles, some modification of the proteins often occurs. In viruses such as HIV, this modification (sometimes called maturation) occurs after the virus has been released from the host cell.Release – Viruses can be released from the host cell by lysis, a process that kills the cell by bursting its membrane and cell wall if present: this is a feature of many bacterial and some animal viruses. Some viruses undergo a lysogenic cycle where the viral genome is incorporated by genetic recombination into a specific place in the host\\'s chromosome. The viral genome is then known as a \"provirus\" or, in the case of bacteriophages a \"prophage\". Whenever the host divides, the viral genome is also replicated. The viral genome is mostly silent within the host. At some point, the provirus or prophage may give rise to the active virus, which may lyse the host cells. Enveloped viruses (e.g., HIV) typically are released from the host cell by budding. During this process, the virus acquires its envelope, which is a modified piece of the host\\'s plasma or other, internal membrane.\\n\\n\\n=== Genome replication ===\\nThe genetic material within virus particles, and the method by which the material is replicated, varies considerably between different types of viruses.\\n\\nDNA viruses\\nThe genome replication of most DNA viruses takes place in the cell\\'s nucleus. If the cell has the appropriate receptor on its surface, these viruses enter the cell either by direct fusion with the cell membrane (e.g., herpesviruses) or—more usually—by receptor-mediated endocytosis. Most DNA viruses are entirely dependent on the host cell\\'s DNA and RNA synthesising machinery and RNA processing machinery. Viruses with larger genomes may encode much of this machinery themselves. In eukaryotes, the viral genome must cross the cell\\'s nuclear membrane to access this machinery, while in bacteria it need only enter the cell.RNA viruses\\nReplication of RNA viruses usually takes place in the cytoplasm. RNA viruses can be placed into four different groups depending on their modes of replication. The polarity (whether or not it can be used directly by ribosomes to make proteins) of single-stranded RNA viruses largely determines the replicative mechanism; the other major criterion is whether the genetic material is single-stranded or double-stranded. All RNA viruses use their own RNA replicase enzymes to create copies of their genomes.Reverse transcribing viruses\\nReverse transcribing viruses have ssRNA (Retroviridae, Metaviridae, Pseudoviridae) or dsDNA (Caulimoviridae, and Hepadnaviridae) in their particles. Reverse transcribing viruses with RNA genomes (retroviruses) use a DNA intermediate to replicate, whereas those with DNA genomes (pararetroviruses) use an RNA intermediate during genome replication. Both types use a reverse transcriptase, or RNA-dependent DNA polymerase enzyme, to carry out the nucleic acid conversion. Retroviruses integrate the DNA produced by reverse transcription into the host genome as a provirus as a part of the replication process; pararetroviruses do not, although integrated genome copies of especially plant pararetroviruses can give rise to infectious virus. They are susceptible to antiviral drugs that inhibit the reverse transcriptase enzyme, e.g. zidovudine and lamivudine. An example of the first type is HIV, which is a retrovirus. Examples of the second type are the Hepadnaviridae, which includes Hepatitis B virus.\\n\\n\\n=== Cytopathic effects on the host cell ===\\nThe range of structural and biochemical effects that viruses have on the host cell is extensive. These are called \\'cytopathic effects\\'. Most virus infections eventually result in the death of the host cell. The causes of death include cell lysis, alterations to the cell\\'s surface membrane and apoptosis. Often cell death is caused by cessation of its normal activities because of suppression by virus-specific proteins, not all of which are components of the virus particle. The distinction between cytopathic and harmless is gradual. Some viruses, such as Epstein–Barr virus, can cause cells to proliferate without causing malignancy, while others, such as papillomaviruses, are established causes of cancer.\\n\\n\\n=== Dormant and latent infections ===\\nSome viruses cause no apparent changes to the infected cell. Cells in which the virus is latent and inactive show few signs of infection and often function normally. This causes persistent infections and the virus is often dormant for many months or years. This is often the case with herpes viruses.\\n\\n\\n=== Host range ===\\nViruses are by far the most abundant biological entities on Earth and they outnumber all the others put together. They infect all types of cellular life including animals, plants, bacteria and fungi. Different types of viruses can infect only a limited range of hosts and many are species-specific. Some, such as smallpox virus for example, can infect only one species—in this case humans, and are said to have a narrow host range. Other viruses, such as rabies virus, can infect different species of mammals and are said to have a broad range. The viruses that infect plants are harmless to animals, and most viruses that infect other animals are harmless to humans. The host range of some bacteriophages is limited to a single strain of bacteria and they can be used to trace the source of outbreaks of infections by a method called phage typing. The complete set of viruses in an organism or habitat is called the virome; for example, all human viruses constitute the human virome.\\n\\n\\n== Classification ==\\n\\nClassification seeks to describe the diversity of viruses by naming and grouping them on the basis of similarities. In 1962, André Lwoff, Robert Horne, and Paul Tournier were the first to develop a means of virus classification, based on the Linnaean hierarchical system. This system based classification on phylum, class, order, family, genus, and species. Viruses were grouped according to their shared properties (not those of their hosts) and the type of nucleic acid forming their genomes. In 1966, the International Committee on Taxonomy of Viruses (ICTV) was formed. The system proposed by Lwoff, Horne and Tournier was initially not accepted by the ICTV because the small genome size of viruses and their high rate of mutation made it difficult to determine their ancestry beyond order. As such, the Baltimore classification system has come to be used to supplement the more traditional hierarchy. Starting in 2018, the ICTV began to acknowledge deeper evolutionary relationships between viruses that have been discovered over time and adopted a 15-rank classification system ranging from realm to species.\\n\\n\\n=== ICTV classification ===\\nThe ICTV developed the current classification system and wrote guidelines that put a greater weight on certain virus properties to maintain family uniformity. A unified taxonomy (a universal system for classifying viruses) has been established. Only a small part of the total diversity of viruses has been studied. As of 2021, 6 realms, 10 kingdoms, 17 phyla, 2 subphyla, 39 classes, 65 orders, 8 suborders, 233 families, 168 subfamilies, 2,606 genera, 84 subgenera, and 10,434 species of viruses have been defined by the ICTV.The general taxonomic structure of taxon ranges and the suffixes used in taxonomic names are shown hereafter. As of 2020, the ranks of subrealm, subkingdom, and subclass are unused, whereas all other ranks are in use.\\n\\nRealm (-viria)\\nSubrealm (-vira)\\nKingdom (-virae)\\nSubkingdom (-virites)\\nPhylum (-viricota)\\nSubphylum (-viricotina)\\nClass (-viricetes)\\nSubclass (-viricetidae)\\nOrder (-virales)\\nSuborder (-virineae)\\nFamily (-viridae)\\nSubfamily (-virinae)\\nGenus (-virus)\\nSubgenus (-virus)\\nSpecies\\n\\n\\n=== Baltimore classification ===\\n\\nThe Nobel Prize-winning biologist David Baltimore devised the Baltimore classification system. The ICTV classification system is used in conjunction with the Baltimore classification system in modern virus classification.The Baltimore classification of viruses is based on the mechanism of mRNA production. Viruses must generate mRNAs from their genomes to produce proteins and replicate themselves, but different mechanisms are used to achieve this in each virus family. Viral genomes may be single-stranded (ss) or double-stranded (ds), RNA or DNA, and may or may not use reverse transcriptase (RT). In addition, ssRNA viruses may be either sense (+) or antisense (−). This classification places viruses into seven groups:\\n\\nI: dsDNA viruses (e.g. Adenoviruses, Herpesviruses, Poxviruses)\\nII: ssDNA viruses (+ strand or \"sense\") DNA (e.g. Parvoviruses)\\nIII: dsRNA viruses (e.g. Reoviruses)\\nIV: (+)ssRNA viruses (+ strand or sense) RNA (e.g. Coronaviruses, Picornaviruses, Togaviruses)\\nV: (−)ssRNA viruses (− strand or antisense) RNA (e.g. Orthomyxoviruses, Rhabdoviruses)\\nVI: ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses)\\nVII: dsDNA-RT viruses DNA with RNA intermediate in life-cycle (e.g. Hepadnaviruses)\\n\\n\\n== Role in human disease ==\\n\\nExamples of common human diseases caused by viruses include the common cold, influenza, chickenpox, and cold sores. Many serious diseases such as rabies, Ebola virus disease, AIDS (HIV), avian influenza, and SARS are caused by viruses. The relative ability of viruses to cause disease is described in terms of virulence. Other diseases are under investigation to discover if they have a virus as the causative agent, such as the possible connection between human herpesvirus 6 (HHV6) and neurological diseases such as multiple sclerosis and chronic fatigue syndrome. There is controversy over whether the bornavirus, previously thought to cause neurological diseases in horses, could be responsible for psychiatric illnesses in humans.Viruses have different mechanisms by which they produce disease in an organism, which depends largely on the viral species. Mechanisms at the cellular level primarily include cell lysis, the breaking open and subsequent death of the cell. In multicellular organisms, if enough cells die, the whole organism will start to suffer the effects. Although viruses cause disruption of healthy homeostasis, resulting in disease, they may exist relatively harmlessly within an organism. An example would include the ability of the herpes simplex virus, which causes cold sores, to remain in a dormant state within the human body. This is called latency and is a characteristic of the herpes viruses, including Epstein–Barr virus, which causes glandular fever, and varicella zoster virus, which causes chickenpox and shingles. Most people have been infected with at least one of these types of herpes virus. These latent viruses might sometimes be beneficial, as the presence of the virus can increase immunity against bacterial pathogens, such as Yersinia pestis.Some viruses can cause lifelong or chronic infections, where the viruses continue to replicate in the body despite the host\\'s defence mechanisms. This is common in hepatitis B virus and hepatitis C virus infections. People chronically infected are known as carriers, as they serve as reservoirs of infectious virus. In populations with a high proportion of carriers, the disease is said to be endemic.\\n\\n\\n=== Epidemiology ===\\nViral epidemiology is the branch of medical science that deals with the transmission and control of virus infections in humans. Transmission of viruses can be vertical, which means from mother to child, or horizontal, which means from person to person. Examples of vertical transmission include hepatitis B virus and HIV, where the baby is born already infected with the virus. Another, more rare, example is the varicella zoster virus, which, although causing relatively mild infections in children and adults, can be fatal to the foetus and newborn baby.Horizontal transmission is the most common mechanism of spread of viruses in populations. Horizontal transmission can occur when body fluids are exchanged during sexual activity, by exchange of saliva or when contaminated food or water is ingested. It can also occur when aerosols containing viruses are inhaled or by insect vectors such as when infected mosquitoes penetrate the skin of a host. Most types of viruses are restricted to just one or two of these mechanisms and they are referred to as \"respiratory viruses\" or \"enteric viruses\" and so forth. The rate or speed of transmission of viral infections depends on factors that include population density, the number of susceptible individuals, (i.e., those not immune), the quality of healthcare and the weather.Epidemiology is used to break the chain of infection in populations during outbreaks of viral diseases. Control measures are used that are based on knowledge of how the virus is transmitted. It is important to find the source, or sources, of the outbreak and to identify the virus. Once the virus has been identified, the chain of transmission can sometimes be broken by vaccines. When vaccines are not available, sanitation and disinfection can be effective. Often, infected people are isolated from the rest of the community, and those that have been exposed to the virus are placed in quarantine. To control the outbreak of foot-and-mouth disease in cattle in Britain in 2001, thousands of cattle were slaughtered. Most viral infections of humans and other animals have incubation periods during which the infection causes no signs or symptoms. Incubation periods for viral diseases range from a few days to weeks, but are known for most infections. Somewhat overlapping, but mainly following the incubation period, there is a period of communicability—a time when an infected individual or animal is contagious and can infect another person or animal. This, too, is known for many viral infections, and knowledge of the length of both periods is important in the control of outbreaks. When outbreaks cause an unusually high proportion of cases in a population, community, or region, they are called epidemics. If outbreaks spread worldwide, they are called pandemics.\\n\\n\\n=== Epidemics and pandemics ===\\n\\nA pandemic is a worldwide epidemic. The 1918 flu pandemic, which lasted until 1919, was a category 5 influenza pandemic caused by an unusually severe and deadly influenza A virus. The victims were often healthy young adults, in contrast to most influenza outbreaks, which predominantly affect juvenile, elderly, or otherwise-weakened patients. Older estimates say it killed 40–50 million people, while more recent research suggests that it may have killed as many as 100 million people, or 5% of the world\\'s population in 1918.Although viral pandemics are rare events, HIV—which evolved from viruses found in monkeys and chimpanzees—has been pandemic since at least the 1980s. During the 20th century there were four pandemics caused by influenza virus and those that occurred in 1918, 1957 and 1968 were severe. Most researchers believe that HIV originated in sub-Saharan Africa during the 20th century; it is now a pandemic, with an estimated 37.9 million people now living with the disease worldwide. There were about 770,000 deaths from AIDS in 2018. The Joint United Nations Programme on HIV/AIDS (UNAIDS) and the World Health Organization (WHO) estimate that AIDS has killed more than 25 million people since it was first recognised on 5 June 1981, making it one of the most destructive epidemics in recorded history. In 2007 there were 2.7 million new HIV infections and 2 million HIV-related deaths.\\n\\nSeveral highly lethal viral pathogens are members of the Filoviridae. Filoviruses are filament-like viruses that cause viral hemorrhagic fever, and include ebolaviruses and marburgviruses. Marburg virus, first discovered in 1967, attracted widespread press attention in April 2005 for an outbreak in Angola. Ebola virus disease has also caused intermittent outbreaks with high mortality rates since 1976 when it was first identified. The worst and most recent one is the 2013–2016 West Africa epidemic.Except for smallpox, most pandemics are caused by newly evolved viruses. These \"emergent\" viruses are usually mutants of less harmful viruses that have circulated previously either in humans or other animals.Severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS)  are caused by new types of coronaviruses. Other coronaviruses are known to cause mild infections in humans, so the virulence and rapid spread of SARS infections—that by July 2003 had caused around 8,000 cases and 800 deaths—was unexpected and most countries were not prepared.A related coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-Cov-2), thought to have originated in bats, emerged in Wuhan, China in November 2019 and spread rapidly around the world.  Infections with the virus caused the COVID-19 pandemic that started in 2020. Unprecedented restrictions in peacetime were placed on international travel, and curfews were imposed in several major cities worldwide in response to the pandemic.\\n\\n\\n=== Cancer ===\\n\\nViruses are an established cause of cancer in humans and other species. Viral cancers occur only in a minority of infected persons (or animals). Cancer viruses come from a range of virus families, including both RNA and DNA viruses, and so there is no single type of \"oncovirus\" (an obsolete term originally used for acutely transforming retroviruses). The development of cancer is determined by a variety of factors such as host immunity and mutations in the host. Viruses accepted to cause human cancers include some genotypes of human papillomavirus, hepatitis B virus, hepatitis C virus, Epstein–Barr virus, Kaposi\\'s sarcoma-associated herpesvirus and human T-lymphotropic virus. The most recently discovered human cancer virus is a polyomavirus (Merkel cell polyomavirus) that causes most cases of a rare form of skin cancer called Merkel cell carcinoma.\\nHepatitis viruses can develop into a chronic viral infection that leads to liver cancer. Infection by human T-lymphotropic virus can lead to tropical spastic paraparesis and adult T-cell leukaemia. Human papillomaviruses are an established cause of cancers of cervix, skin, anus, and penis. Within the Herpesviridae, Kaposi\\'s sarcoma-associated herpesvirus causes Kaposi\\'s sarcoma and body-cavity lymphoma, and Epstein–Barr virus causes Burkitt\\'s lymphoma, Hodgkin\\'s lymphoma, B lymphoproliferative disorder, and nasopharyngeal carcinoma. Merkel cell polyomavirus closely related to SV40 and mouse polyomaviruses that have been used as animal models for cancer viruses for over 50 years.\\n\\n\\n=== Host defence mechanisms ===\\n\\nThe body\\'s first line of defence against viruses is the innate immune system. This comprises cells and other mechanisms that defend the host from infection in a non-specific manner. This means that the cells of the innate system recognise, and respond to, pathogens in a generic way, but, unlike the adaptive immune system, it does not confer long-lasting or protective immunity to the host.RNA interference is an important innate defence against viruses. Many viruses have a replication strategy that involves double-stranded RNA (dsRNA). When such a virus infects a cell, it releases its RNA molecule or molecules, which immediately bind to a protein complex called a dicer that cuts the RNA into smaller pieces. A biochemical pathway—the RISC complex—is activated, which ensures cell survival by degrading the viral mRNA. Rotaviruses have evolved to avoid this defence mechanism by not uncoating fully inside the cell, and releasing newly produced mRNA through pores in the particle\\'s inner capsid. Their genomic dsRNA remains protected inside the core of the virion.When the adaptive immune system of a vertebrate encounters a virus, it produces specific antibodies that bind to the virus and often render it non-infectious. This is called humoral immunity. Two types of antibodies are important. The first, called IgM, is highly effective at neutralising viruses but is produced by the cells of the immune system only for a few weeks. The second, called IgG, is produced indefinitely. The presence of IgM in the blood of the host is used to test for acute infection, whereas IgG indicates an infection sometime in the past. IgG antibody is measured when tests for immunity are carried out.Antibodies can continue to be an effective defence mechanism even after viruses have managed to gain entry to the host cell. A protein that is in cells, called TRIM21, can attach to the antibodies on the surface of the virus particle. This primes the subsequent destruction of the virus by the enzymes of the cell\\'s proteosome system.\\n\\nA second defence of vertebrates against viruses is called cell-mediated immunity and involves immune cells known as T cells. The body\\'s cells constantly display short fragments of their proteins on the cell\\'s surface, and, if a T cell recognises a suspicious viral fragment there, the host cell is destroyed by \\'killer T\\' cells and the virus-specific T-cells proliferate. Cells such as the macrophage are specialists at this antigen presentation. The production of interferon is an important host defence mechanism. This is a hormone produced by the body when viruses are present. Its role in immunity is complex; it eventually stops the viruses from reproducing by killing the infected cell and its close neighbours.Not all virus infections produce a protective immune response in this way. HIV evades the immune system by constantly changing the amino acid sequence of the proteins on the surface of the virion. This is known as \"escape mutation\" as the viral epitopes escape recognition by the host immune response. These persistent viruses evade immune control by sequestration, blockade of antigen presentation, cytokine resistance, evasion of natural killer cell activities, escape from apoptosis, and antigenic shift. Other viruses, called \\'neurotropic viruses\\', are disseminated by neural spread where the immune system may be unable to reach them due to immune privilege.\\n\\n\\n=== Prevention and treatment ===\\nBecause viruses use vital metabolic pathways within host cells to replicate, they are difficult to eliminate without using drugs that cause toxic effects to host cells in general. The most effective medical approaches to viral diseases are vaccinations to provide immunity to infection, and antiviral drugs that selectively interfere with viral replication.\\n\\n\\n==== Vaccines ====\\n\\nVaccination is a cheap and effective way of preventing infections by viruses. Vaccines were used to prevent viral infections long before the discovery of the actual viruses. Their use has resulted in a dramatic decline in morbidity (illness) and mortality (death) associated with viral infections such as polio, measles, mumps and rubella. Smallpox infections have been eradicated. Vaccines are available to prevent over thirteen viral infections of humans, and more are used to prevent viral infections of animals. Vaccines can consist of live-attenuated or killed viruses, viral proteins (antigens), or RNA. Live vaccines contain weakened forms of the virus, which do not cause the disease but, nonetheless, confer immunity. Such viruses are called attenuated. Live vaccines can be dangerous when given to people with a weak immunity (who are described as immunocompromised), because in these people, the weakened virus can cause the original disease. Biotechnology and genetic engineering techniques are used to produce subunit vaccines. These vaccines use only the capsid proteins of the virus. Hepatitis B vaccine is an example of this type of vaccine. Subunit vaccines are safe for immunocompromised patients because they cannot cause the disease. The yellow fever virus vaccine, a live-attenuated strain called 17D, is probably the safest and most effective vaccine ever generated.\\n\\n\\n==== Antiviral drugs ====\\n\\nAntiviral drugs are often nucleoside analogues (fake DNA building-blocks), which viruses mistakenly incorporate into their genomes during replication. The life-cycle of the virus is then halted because the newly synthesised DNA is inactive. This is because these analogues lack the hydroxyl groups, which, along with phosphorus atoms, link together to form the strong \"backbone\" of the DNA molecule. This is called DNA chain termination. Examples of nucleoside analogues are aciclovir for Herpes simplex virus infections and lamivudine for HIV and hepatitis B virus infections. Aciclovir is one of the oldest and most frequently prescribed antiviral drugs.\\nOther antiviral drugs in use target different stages of the viral life cycle. HIV is dependent on a proteolytic enzyme called the HIV-1 protease for it to become fully infectious. There is a large class of drugs called protease inhibitors that inactivate this enzyme. There are around thirteen classes of antiviral drugs each targeting different viruses or stages of viral replication.Hepatitis C is caused by an RNA virus. In 80% of people infected, the disease is chronic, and without treatment, they are infected for the remainder of their lives. There are effective treatments that use direct-acting antivirals. The treatment of chronic carriers of the hepatitis B virus has also been developed by using similar strategies that include lamivudine and other anti-viral drugs.\\n\\n\\n== Infection in other species ==\\nViruses infect all cellular life and, although viruses occur universally, each cellular species has its own specific range that often infects only that species. Some viruses, called satellites, can replicate only within cells that have already been infected by another virus.\\n\\n\\n=== Animal viruses ===\\n\\nViruses are important pathogens of livestock. Diseases such as foot-and-mouth disease and bluetongue are caused by viruses. Companion animals such as cats, dogs, and horses, if not vaccinated, are susceptible to serious viral infections. Canine parvovirus is caused by a small DNA virus and infections are often fatal in pups. Like all invertebrates, the honey bee is susceptible to many viral infections. Most viruses co-exist harmlessly in their host and cause no signs or symptoms of disease.\\n\\n\\n=== Plant viruses ===\\n\\nThere are many types of plant viruses, but often they cause only a loss of yield, and it is not economically viable to try to control them. Plant viruses are often spread from plant to plant by organisms, known as vectors. These are usually insects, but some fungi, nematode worms, single-celled organisms, and parasitic plants are vectors. When control of plant virus infections is considered economical, for perennial fruits, for example, efforts are concentrated on killing the vectors and removing alternate hosts such as weeds. Plant viruses cannot infect humans and other animals because they can reproduce only in living plant cells.Originally from Peru, the potato has become a staple crop worldwide. The potato virus Y causes disease in potatoes and related species including tomatoes and peppers. In the 1980s, this virus acquired economical importance when it proved difficult to control in seed potato crops. Transmitted by aphids, this virus can reduce crop yields by up to 80 per cent, causing significant losses to potato yields.Plants have elaborate and effective defence mechanisms against viruses. One of the most effective is the presence of so-called resistance (R) genes. Each R gene confers resistance to a particular virus by triggering localised areas of cell death around the infected cell, which can often be seen with the unaided eye as large spots. This stops the infection from spreading. RNA interference is also an effective defence in plants. When they are infected, plants often produce natural disinfectants that kill viruses, such as salicylic acid, nitric oxide, and reactive oxygen molecules.Plant virus particles or virus-like particles (VLPs) have applications in both biotechnology and nanotechnology. The capsids of most plant viruses are simple and robust structures and can be produced in large quantities either by the infection of plants or by expression in a variety of heterologous systems. Plant virus particles can be modified genetically and chemically to encapsulate foreign material and can be incorporated into supramolecular structures for use in biotechnology.\\n\\n\\n=== Bacterial viruses ===\\n\\nBacteriophages are a common and diverse group of viruses and are the most abundant biological entity in aquatic environments—there are up to ten times more of these viruses in the oceans than there are bacteria, reaching levels of 250,000,000 bacteriophages per millilitre of seawater. These viruses infect specific bacteria by binding to surface receptor molecules and then entering the cell. Within a short amount of time, in some cases, just minutes, bacterial polymerase starts translating viral mRNA into protein. These proteins go on to become either new virions within the cell, helper proteins, which help assembly of new virions, or proteins involved in cell lysis. Viral enzymes aid in the breakdown of the cell membrane, and, in the case of the T4 phage, in just over twenty minutes after injection over three hundred phages could be released.The major way bacteria defend themselves from bacteriophages is by producing enzymes that destroy foreign DNA. These enzymes, called restriction endonucleases, cut up the viral DNA that bacteriophages inject into bacterial cells. Bacteria also contain a system that uses CRISPR sequences to retain fragments of the genomes of viruses that the bacteria have come into contact with in the past, which allows them to block the virus\\'s replication through a form of RNA interference. This genetic system provides bacteria with acquired immunity to infection.\\n\\n\\n=== Archaeal viruses ===\\n\\nSome viruses replicate within archaea: these are DNA viruses with unusual and sometimes unique shapes. These viruses have been studied in most detail in the thermophilic archaea, particularly the orders Sulfolobales and Thermoproteales. Defences against these viruses involve RNA interference from repetitive DNA sequences within archaean genomes that are related to the genes of the viruses. Most archaea have CRISPR–Cas systems as an adaptive defence against viruses. These enable archaea to retain sections of viral DNA, which are then used to target and eliminate subsequent infections by the virus using a process similar to RNA interference.\\n\\n\\n== Role in aquatic ecosystems ==\\n\\nViruses are the most abundant biological entity in aquatic environments. There are about ten million of them in a teaspoon of seawater. Most of these viruses are bacteriophages infecting heterotrophic bacteria and cyanophages infecting cyanobacteria and they are essential to the regulation of saltwater and freshwater ecosystems.\\nBacteriophages are harmless to plants and animals, and are essential to the regulation of marine and freshwater ecosystems  are important mortality agents of phytoplankton, the base of the foodchain in aquatic environments. They infect and destroy bacteria in aquatic microbial communities, and are one of the most important mechanisms of recycling carbon and nutrient cycling in marine environments. The organic molecules released from the dead bacterial cells stimulate fresh bacterial and algal growth, in a process known as the viral shunt. In particular, lysis of bacteria by viruses has been shown to enhance nitrogen cycling and stimulate phytoplankton growth.  Viral activity may also affect the biological pump, the process whereby carbon is sequestered in the deep ocean.Microorganisms constitute more than 90% of the biomass in the sea. It is estimated that viruses kill approximately 20% of this biomass each day and that there are 10 to 15 times as many viruses in the oceans as there are bacteria and archaea. Viruses are also major agents responsible for the destruction of phytoplankton including harmful algal blooms,\\nThe number of viruses in the oceans decreases further offshore and deeper into the water, where there are fewer host organisms.In January 2018, scientists reported that 800 million viruses, mainly of marine origin, are deposited daily from the Earth\\'s atmosphere onto every square meter of the planet\\'s surface, as the result of a global atmospheric stream of viruses, circulating above the weather system but below the altitude of usual airline travel, distributing viruses around the planet.Like any organism, marine mammals are susceptible to viral infections. In 1988 and 2002, thousands of harbour seals were killed in Europe by phocine distemper virus. Many other viruses, including caliciviruses, herpesviruses, adenoviruses and parvoviruses, circulate in marine mammal populations.\\n\\n\\n== Role in evolution ==\\n\\nViruses are an important natural means of transferring genes between different species, which increases genetic diversity and drives evolution. It is thought that viruses played a central role in early evolution, before the diversification of the last universal common ancestor into bacteria, archaea and eukaryotes. Viruses are still one of the largest reservoirs of unexplored genetic diversity on Earth.\\n\\n\\n== Applications ==\\n\\n\\n=== Life sciences and medicine ===\\n\\nViruses are important to the study of molecular and cell biology as they provide simple systems that can be used to manipulate and investigate the functions of cells. The study and use of viruses have provided valuable information about aspects of cell biology. For example, viruses have been useful in the study of genetics and helped our understanding of the basic mechanisms of molecular genetics, such as DNA replication, transcription, RNA processing, translation, protein transport, and immunology.\\nGeneticists often use viruses as vectors to introduce genes into cells that they are studying. This is useful for making the cell produce a foreign substance, or to study the effect of introducing a new gene into the genome. Similarly, virotherapy uses viruses as vectors to treat various diseases, as they can specifically target cells and DNA. It shows promising use in the treatment of cancer and in gene therapy. Eastern European scientists have used phage therapy as an alternative to antibiotics for some time, and interest in this approach is increasing, because of the high level of antibiotic resistance now found in some pathogenic bacteria.\\nThe expression of heterologous proteins by viruses is the basis of several manufacturing processes that are currently being used for the production of various proteins such as vaccine antigens and antibodies. Industrial processes have been recently developed using viral vectors and several pharmaceutical proteins are currently in pre-clinical and clinical trials.\\n\\n\\n==== Virotherapy ====\\n\\nVirotherapy involves the use of genetically modified viruses to treat diseases. Viruses have been modified by scientists to reproduce in cancer cells and destroy them but not infect healthy cells. Talimogene laherparepvec (T-VEC), for example, is a modified herpes simplex virus that has had a gene, which is required for viruses to replicate in healthy cells, deleted and replaced with a human gene (GM-CSF) that stimulates immunity. When this virus infects cancer cells, it destroys them and in doing so the presence the GM-CSF gene attracts dendritic cells from the surrounding tissues of the body. The dendritic cells process the dead cancer cells and present components of them to other cells of the immune system. Having completed successful clinical trials, the virus gained approval for the treatment of melanoma in late 2015. Viruses that have been reprogrammed to kill cancer cells are called oncolytic viruses.\\n\\n\\n=== Materials science and nanotechnology ===\\nCurrent trends in nanotechnology promise to make much more versatile use of viruses. From the viewpoint of a materials scientist, viruses can be regarded as organic nanoparticles.\\nTheir surface carries specific tools that enable them to cross the barriers of their host cells. The size and shape of viruses and the number and nature of the functional groups on their surface are precisely defined. As such, viruses are commonly used in materials science as scaffolds for covalently linked surface modifications. A particular quality of viruses is that they can be tailored by directed evolution. The powerful techniques developed by life sciences are becoming the basis of engineering approaches towards nanomaterials, opening a wide range of applications far beyond biology and medicine.Because of their size, shape, and well-defined chemical structures, viruses have been used as templates for organising materials on the nanoscale. Recent examples include work at the Naval Research Laboratory in Washington, D.C., using Cowpea mosaic virus (CPMV) particles to amplify signals in DNA microarray based sensors. In this application, the virus particles separate the fluorescent dyes used for signalling to prevent the formation of non-fluorescent dimers that act as quenchers. Another example is the use of CPMV as a nanoscale breadboard for molecular electronics.\\n\\n\\n=== Synthetic viruses ===\\nMany viruses can be synthesised de novo (\"from scratch\").  The first synthetic virus was created in 2002. Although somewhat of a misconception, it is not the actual virus that is synthesised, but rather its DNA genome (in case of a DNA virus), or a cDNA copy of its genome (in case of RNA viruses). For many virus families the naked synthetic DNA or RNA (once enzymatically converted back from the synthetic cDNA) is infectious when introduced into a cell. That is, they contain all the necessary information to produce new viruses. This technology is now being used to investigate novel vaccine strategies. The ability to synthesise viruses has far-reaching consequences, since viruses can no longer be regarded as extinct, as long as the information of their genome sequence is known and permissive cells are available. As of June 2021, the full-length genome sequences of 11,464 different viruses, including smallpox, are publicly available in an online database maintained by the National Institutes of Health.\\n\\n\\n=== Weapons ===\\n\\nThe ability of viruses to cause devastating epidemics in human societies has led to the concern that viruses could be weaponised for biological warfare. Further concern was raised by the successful recreation of the infamous 1918 influenza virus in a laboratory.\\nThe smallpox virus devastated numerous societies throughout history before its eradication. There are only two centres in the world authorised by the WHO to keep stocks of smallpox virus: the State Research Center of Virology and Biotechnology VECTOR in Russia and the Centers for Disease Control and Prevention in the United States. It may be used as a weapon, as the vaccine for smallpox sometimes had severe side-effects, it is no longer used routinely in any country. Thus, much of the modern human population has almost no established resistance to smallpox and would be vulnerable to the virus.\\n\\n\\n== See also ==\\n\\n\\n== References ==\\n\\n\\n=== Notes ===\\n\\n\\n=== Bibliography ===\\n\\n\\n== External links ==\\n Media related to Viruses at Wikimedia Commons\\n Data related to Virus at Wikispecies\\nViralZone A Swiss Institute of Bioinformatics resource for all viral families, providing general molecular and epidemiological information',\n",
              " 'There are many variants of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus that causes coronavirus disease 2019 (COVID-19). Some are believed, or have been stated, to be of particular importance due to their potential for increased transmissibility, increased virulence, or reduced effectiveness of vaccines against them. These variants contribute to the continuation of the COVID-19 pandemic.\\nFive SARS-CoV-2 variants have been designated as variants of concern by the World Health Organization: the Alpha, Beta, Gamma, Delta, and Omicron variants.\\n\\n\\n== Overview ==\\n\\nThe emergence of SARS-CoV-2 may have resulted from recombination events between a bat SARS-like coronavirus and a pangolin coronavirus through cross-species transmission. The earliest available SARS-CoV-2 viral genomes were collected from patients in December 2019, and Chinese researchers compared these early genomes with bat and pangolin coronavirus strains to estimate the ancestral human coronavirus type; the identified ancestral genome type was labeled \"S\", and its dominant derived type was labeled \"L\" to reflect the mutant amino acid changes. Independently, Western researchers carried out similar analyses but labeled the ancestral type \"A\" and the derived type \"B\". The B-type mutated into further types including B.1, which is the ancestor of the major global variants of concern, labeled in 2021 by the WHO as alpha, beta, gamma, delta and omicron variants.Early in the pandemic, the relatively low number of infections (compared with later stages of the pandemic) resulted in fewer opportunities for mutation of the viral genome and, therefore, fewer opportunities for the occurrence of differentiated variants. Since the occurrence of variants was rarer, the observation of S-protein mutations in the receptor-binding domain (RBD) region interacting with ACE2 was also not frequent.As time went on, the evolution of SARS-CoV-2\\'s genome (by means of random mutations) led to mutant specimens of the virus (i.e., genetic variants), observed to be more transmissible, to be naturally selected. Notably, both the Alpha and the Delta variants were observed to be more transmissible than previously identified viral strains.Some SARS-CoV-2 variants are considered to be of concern as they maintain (or even increase) their replication fitness in the face of rising population immunity, either by infection recovery or via vaccination. Some of the variants of concern show mutations in the RBD of the S-protein.The following table presents information and relative risk level for currently and formerly circulating variants of concern (VOC). The intervals assume a 95% confidence or credibility level, unless otherwise stated. Currently, all estimates are approximations due to the limited availability of data for studies. For Alpha, Beta, Gamma and Delta, there is no change in test accuracy, and neutralising antibody activity is retained by some monoclonal antibodies. PCR tests continue to detect the Omicron variant.\\n  Very high risk   High risk   Medium risk   Low risk   Unknown risk\\n\\n\\n== Nomenclature ==\\n\\nSARS-CoV-2 variants are grouped according to their lineage and component mutations. As of July 2021, no consistent nomenclature was established for it. Many organisations, including governments and news outlets, referred colloquially to concerning variants by the country in which they were first identified. After months of discussions, the World Health Organization announced Greek-letter names for important strains on 31 May 2021, so they could be easily referred to in a simple, easy to say, and non-stigmatising fashion. This decision may have partially been taken because of criticism from governments on using country names to refer to variants of the virus; the WHO mentioned the potential for mentioning country names to cause stigma. After using all the letters from Alpha to Mu (see below), in November 2021 the WHO skipped the next two letters of the Greek alphabet, Nu and Xi, and used Omicron, prompting speculation that Xi was skipped to avoid offending Chinese President Xi Jinping. The WHO gave as the explanation that Nu is too easily confounded with \"new\" and Xi is a common last name. In the event that the WHO uses the entirety of the Greek alphabet, the agency considered naming future variants after constellations.\\n\\n\\n=== Lineages and clades ===\\nWhile there are many thousands of variants of SARS-CoV-2, subtypes of the virus can be put into larger groupings such as lineages or clades. Three main, generally used nomenclatures have been proposed:\\n\\nAs of January 2021, GISAID—referring to SARS-CoV-2 as hCoV-19—had identified eight global clades (S, O, L, V, G, GH, GR, and GV).\\nIn 2017, Hadfield et al. announced Nextstrain, intended \"for real-time tracking of pathogen evolution\". Nextstrain has later been used for tracking SARS-CoV-2, identifying 13 major clades (19A–B, 20A–20J and 21A) as of June 2021.\\nIn 2020, Rambaut et al. of the Phylogenetic Assignment of Named Global Outbreak Lineages (PANGOLIN) software team proposed in an article \"a dynamic nomenclature for SARS-CoV-2 lineages that focuses on actively circulating virus lineages and those that spread to new locations\"; as of August 2021, 1340 lineages had been designated.Each national public health institute may also institute its own nomenclature system for the purposes of tracking specific variants. For example, Public Health England designated each tracked variant by year, month and number in the format [YYYY] [MM]/[NN], prefixing \\'VUI\\' or \\'VOC\\' for a variant under investigation or a variant of concern respectively. This system has now been modified and now uses the format [YY] [MMM]-[NN], where the month is written out using a three-letter code.\\n\\n\\n=== Classification of variants ===\\nVariants that appear to meet one or more specific criteria considered during the COVID-19 pandemic may be labeled \"variants of interest\" or \"variants under investigation\" (\\'VUI\\') pending verification and validation of these properties. Once validated, variants of interest /VUI  may be renamed \"variants of concern\" by monitoring organizations, such as the CDC in the US. A related category is \"variant of high consequence\", used by the CDC if there is clear evidence that the effectiveness of prevention or intervention measures for a particular variant is substantially reduced.\\n\\n\\n== Reference sequence ==\\nAs it is currently not known when the index case or \"patient zero\" occurred, the choice of reference sequence for a given study is relatively arbitrary, with different notable research studies\\' choices varying as follows:\\n\\nThe earliest sequence, Wuhan-1, was collected on 24 December 2019.\\nOne group (Sudhir Kumar et al.) refers extensively to an NCBI reference genome (GenBankID:NC_045512; GISAID ID: EPI_ISL_402125), this sample was collected on 26 December 2019, although they also used the WIV04 GISAID reference genome (ID: EPI_ISL_402124), in their analyses.\\nAccording to another source (Zhukova et al.), the sequence WIV04/2019, belonging to the GISAID S clade / PANGO A lineage / Nextstrain 19B clade, is thought to most closely reflect the sequence of the original virus infecting humans—known as \"sequence zero\". WIV04/2019 was sampled from a symptomatic patient on 30 December 2019 and is widely used (especially by those collaborating with GISAID) as a reference sequence.The variant first sampled and identified in Wuhan, China is considered by researchers to differ from the progenitor genome by three mutations. Subsequently, many distinct lineages of SARS-CoV-2 have evolved.\\n\\n\\n== Notability criteria ==\\nViruses generally acquire mutations over time, giving rise to new variants. When a new variant appears to be growing in a population, it can be labelled as an \"emerging variant\". In the case of SARS-CoV-2, new lineages often differ from one another by just a few nucleotides.Some of the potential consequences of emerging variants are the following:\\nIncreased transmissibility\\nIncreased morbidity\\nIncreased mortality\\nAbility to evade detection by diagnostic tests\\nDecreased susceptibility to antiviral drugs (if and when such drugs are available)\\nDecreased susceptibility to neutralising antibodies, either therapeutic (e.g., convalescent plasma or monoclonal antibodies) or in laboratory experiments\\nAbility to evade natural immunity (e.g., causing reinfections)\\nAbility to infect vaccinated individuals\\nIncreased risk of particular conditions such as multisystem inflammatory syndrome or long COVID.\\nIncreased affinity for particular demographic or clinical groups, such as children or immunocompromised individuals.Variants that appear to meet one or more of these criteria may be labelled \"variants under investigation\" or \"variants of interest\" pending verification and validation of these properties. The primary characteristic of a variant of interest is that it shows evidence that demonstrates it is the cause of an increased proportion of cases or unique outbreak clusters; however, it must also have limited prevalence or expansion at national levels, or the classification would be elevated to a \"variant of concern\". If there is clear evidence that the effectiveness of prevention or intervention measures for a particular variant is substantially reduced, that variant is termed a \"variant of high consequence\".\\n\\n\\n== Variants of concern (WHO) ==\\nListed below are the Variants of Concern (VOC) recognised by the World Health Organization as of June 2021. Other organisations such as the CDC in the United States have at times used a slightly different list. As of July 2021, their list matched that of the WHO.\\n\\n\\n=== Alpha (lineage B.1.1.7) ===\\n\\nFirst detected in October 2020 during the COVID-19 pandemic in the United Kingdom from a sample taken the previous month in Kent, lineage B.1.1.7, labelled Alpha variant by the WHO, was previously known as the first Variant Under Investigation in December 2020 (VUI – 202012/01) and later notated as VOC-202012/01. It is also known as 20I (V1), 20I/501Y.V1 (formerly 20B/501Y.V1), or 501Y.V1. From October to December 2020, its prevalence doubled every 6.5 days, the presumed generational interval. It is correlated with a significant increase in the rate of COVID-19 infection in United Kingdom, associated partly with the N501Y mutation. There was some evidence that this variant had 40–80% increased transmissibility (with most estimates lying around the middle to higher end of this range), and early analyses suggested an increase in lethality, though later work found no evidence of increased virulence. As of May 2021, the Alpha variant had been detected in some 120 countries.\\n\\n\\n==== B.1.1.7 with E484K ====\\nVariant of Concern 21FEB-02 (previously written as VOC-202102/02), described by Public Health England (PHE) as \"B.1.1.7 with E484K\" is of the same lineage in the Pango nomenclature system, but has an additional E484K mutation. As of 17 March 2021, there were 39 confirmed cases of VOC-21FEB-02 in the UK. On 4 March 2021, scientists reported B.1.1.7 with E484K mutations in the state of Oregon. In 13 test samples analysed, one had this combination, which appeared to have arisen spontaneously and locally, rather than being imported. Other names for this variant include B.1.1.7+E484K and B.1.1.7 Lineage with S:E484K.\\n\\n\\n=== Beta (lineage B.1.351) ===\\n\\nOn 18 December 2020, the 501.V2 variant, also known as 501.V2, 20H (V2), 20H/501Y.V2 (formerly 20C/501Y.V2), 501Y.V2, VOC-20DEC-02 (formerly VOC-202012/02), or lineage B.1.351, was first detected in South Africa and reported by the country\\'s health department. It has been labelled as Beta variant by WHO. Researchers and officials reported that the prevalence of the variant was higher among young people with no underlying health conditions, and by comparison with other variants it is more frequently resulting in serious illness in those cases. The South African health department also indicated that the variant may be driving the second wave of the COVID-19 epidemic in the country due to the variant spreading at a more rapid pace than other earlier variants of the virus.Scientists noted that the variant contains several mutations that allow it to attach more easily to human cells because of the following three mutations in the receptor-binding domain (RBD) in the spike glycoprotein of the virus: N501Y, K417N, and E484K. The N501Y mutation has also been detected in the United Kingdom.\\n\\n\\n=== Gamma (lineage P.1) ===\\n\\nThe Gamma variant or lineage P.1, termed Variant of Concern 21JAN-02 (formerly VOC-202101/02) by Public Health England, 20J (V3) or 20J/501Y.V3 by Nextstrain, or just 501Y.V3, was detected in Tokyo on 6 January 2021 by the National Institute of Infectious Diseases (NIID). It has been labelled as Gamma variant by WHO. The new variant was first identified in four people who arrived in Tokyo having travelled from the Brazilian Amazonas state on 2 January 2021. On 12 January 2021, the Brazil-UK CADDE Centre confirmed 13 local cases of the new Gamma variant in the Amazon rainforest. This variant of SARS-CoV-2 has been named lineage P.1 (although it is a descendant of B.1.1.28, the name B.1.1.28.1 is not permitted and thus the resultant name is P.1), and has 17 unique amino acid changes, 10 of which in its spike protein, including the three concerning mutations: N501Y, E484K and K417T.:\\u200aFigure 5\\u200aThe N501Y and E484K mutations favour the formation of a stable RBD-hACE2 complex, thus, enhancing the binding affinity of RBD to hACE2. However, the K417T mutation disfavours complex formation between RBD and hACE2, which has been demonstrated to reduce the binding affinity.The new variant was absent in samples collected from March to November 2020 in Manaus, Amazonas state, but it was detected for the same city in 42% of the samples from 15 to 23 December 2020, followed by 52.2% during 15–31 December and 85.4% during 1–9 January 2021. A study found that infections by Gamma can produce nearly ten times more viral load compared to persons infected by one of the other lineages identified in Brazil (B.1.1.28 or B.1.195). Gamma also showed 2.2 times higher transmissibility with the same ability to infect both adults and older persons, suggesting P.1 and P.1-like lineages are more successful at infecting younger humans irrespective of sex.A study of samples collected in Manaus between November 2020 and January 2021, indicated that the Gamma variant is 1.4–2.2 times more transmissible and was shown to be capable of evading 25–61% of inherited immunity from previous coronavirus diseases, leading to the possibility of reinfection after recovery from an earlier COVID-19 infection. As for the fatality ratio, infections by Gamma were also found to be 10–80% more lethal.A study found that people fully vaccinated with Pfizer or Moderna have significantly decreased neutralisation effect against Gamma, although the actual impact on the course of the disease is uncertain.\\nA pre-print study by the Oswaldo Cruz Foundation published in early April found that the real-world performance of people with the initial dose of the Sinovac\\'s Coronavac Vaccine had approximately 50% efficacy rate. They expected the efficacy to be higher after the 2nd dose. As of July 2021, the study is ongoing.Preliminary data from two studies indicate that the Oxford–AstraZeneca vaccine is effective against the Gamma variant, although the exact level of efficacy has not yet been released. Preliminary data from a study conducted by Instituto Butantan suggest that CoronaVac is effective against the Gamma variant as well, and as of July 2021 has yet to be expanded to obtain definitive data.\\n\\n\\n=== Delta (lineage B.1.617.2) ===\\n\\nThe Delta variant, also known as B.1.617.2, G/452R.V3, 21A or 21A/S:478K, is a globally dominant variant that spread to at least 185 countries. It was first discovered in India. Descendant of lineage B.1.617, which also includes the Kappa variant under investigation, it was first discovered in October 2020 and has since spread internationally. On 6 May 2021, British scientists declared B.1.617.2 (which notably lacks mutation at E484Q) as a \"variant of concern\", labelling it VOC-21APR-02, after they flagged evidence that it spreads more quickly than the original version of the virus and could spread quicker or as quickly as Alpha. It carries L452R and P681R mutations in Spike; unlike Kappa it carries T478K but not E484Q.\\nOn 3 June 2021, Public Health England reported that twelve of the 42 deaths from the Delta variant in England were among the fully vaccinated, and that it was spreading almost twice as fast as the Alpha variant. Also on 11 June, Foothills Medical Centre in Calgary, Canada reported that half of their 22 cases of the Delta variant occurred among the fully vaccinated.In June 2021, reports began to appear of a variant of Delta with the K417N mutation. The mutation, also present in the Beta and Gamma variants, raised concerns about the possibility of reduced effectiveness of vaccines and antibody treatments and increased risk of reinfection. The variant, called \"Delta with K417N\" by Public Health England, includes two clades corresponding to the Pango lineages AY.1 and AY.2. It has been nicknamed \"Delta plus\" from \"Delta plus K417N\". The name of the mutation, K417N, refers to an exchange whereby lysine (K) is replaced by asparagine (N) at position 417. On 22 June, India\\'s Ministry of Health and Family Welfare declared the \"Delta plus\" variant of COVID-19 a Variant of Concern after 22 cases of the variant were reported in India. After the announcement, leading virologists said there was insufficient data to support labelling the variant as a distinct variant of concern, pointing to the small number of patients studied. In the UK in July 2021, AY.4.2 was identified. Alongside those previously mentioned it also gained the nickname \\'Delta Plus\\', on the strength of its extra mutations, Y145H and A222V. These are not unique to it, but distinguish it from the original Delta variant.\\n\\n\\n=== Omicron ===\\n\\n\\n==== Lineage B.1.1.529 ====\\nThe Omicron variant, known as lineage B.1.1.529, was declared a variant of concern by the World Health Organization on 26 November 2021.The variant has a large number of mutations, of which some are concerning. The number of cases in the B.1.1.529 lineage is increasing in all areas of South Africa. Some evidence shows that this variant has an increased risk of reinfection. Studies are underway to evaluate the exact impact on transmissibility, mortality, and other factors.Named Omicron by the WHO, it was identified in November 2021 in Botswana and South Africa; one case had travelled to Hong Kong, one confirmed case was identified in Israel in a traveler returning from Malawi, along with two who returned from South Africa and one from Madagascar. Belgium confirmed the first detected case in Europe on 26 November 2021 in an individual who had returned from Egypt on 11 November. Indian SARS-CoV-2 Genomics Consortium (INSACOG) in its January 2022 bulletin noted that Omicron is in community transmission in India where new cases have been rising exponentially.\\n\\n\\n===== Sublineages (including BA.2) =====\\nAccording to the WHO, BA.1, BA.1.1, and BA.2 are the most common sublineages of Omicron globally as of February 2022. BA.2 contains 28 unique genetic changes, including four in its spike protein, compared to BA.1, which had already acquired 60 mutations since the ancestral Wuhan strain, including 32 in the spike protein. BA.2 is more transmissible than BA.1. It was causing most cases in England by mid-March 2022, and by the end of March, BA.2 became dominant in the US.\\n\\n\\n== Variants of interest (WHO) ==\\nListed below are the Variants of Interest (VOI) which are, as of August 2021, recognised by the World Health Organization. Other organisations such as the CDC in the United States may at times use a slightly different list.\\n\\n\\n=== Lambda (lineage C.37) ===\\n\\nThe Lambda variant, also known as lineage C.37, was first detected in Peru in August 2020 and was designated by the WHO as a variant of interest on 14 June 2021. It spread to at least 30 countries around the world and, as of July 2021, it is unknown whether it is more infectious and resistant to vaccines than other strains.\\n\\n\\n=== Mu (lineage B.1.621) ===\\n\\nThe Mu variant, also known as lineage B.1.621, was first detected in Colombia in January 2021 and was designated by the WHO as a variant of interest on 30 August 2021. There have been outbreaks in South America and Europe.\\n\\n\\n== Former variants of interest ==\\n\\n\\n=== Epsilon (lineages B.1.429, B.1.427, CAL.20C) ===\\n\\nThe Epsilon variant or lineage B.1.429, also known as CAL.20C or CA VUI1, 21C or 20C/S:452R, is defined by five distinct mutations (I4205V and D1183Y in the ORF1ab gene, and S13I, W152C, L452R in the spike protein\\'s S-gene), of which the L452R (previously also detected in other unrelated lineages) was of particular concern. From 17 March to 29 June 2021, the CDC listed B.1.429 and the related B.1.427 as \"variants of concern\". As of July 2021, Epsilon is no longer considered a variant of interest by the WHO, as it was overtaken by Alpha.From September 2020 to January 2021, it was 19% to 24% more transmissible than earlier variants in California. Neutralisation against it by antibodies from natural infections and vaccinations was moderately reduced, but it remained detectable in most diagnostic tests.Epsilon (CAL.20C) was first observed in July 2020 by researchers at the Cedars-Sinai Medical Center, California, in one of 1,230 virus samples collected in Los Angeles County since the start of the COVID-19 epidemic. It was not detected again until September when it reappeared among samples in California, but numbers remained very low until November. In November 2020, the Epsilon variant accounted for 36 per cent of samples collected at Cedars-Sinai Medical Center, and by January 2021, the Epsilon variant accounted for 50 per cent of samples. In a joint press release by University of California, San Francisco, California Department of Public Health, and Santa Clara County Public Health Department, the variant was also detected in multiple counties in Northern California. From November to December 2020, the frequency of the variant in sequenced cases from Northern California rose from 3% to 25%. In a preprint, CAL.20C is described as belonging to clade 20C and contributing approximately 36% of samples, while an emerging variant from the 20G clade accounts for some 24% of the samples in a study focused on Southern California. Note, however, that in the US as a whole, the 20G clade predominates, as of January 2021. Following the increasing numbers of Epsilon in California, the variant has been detected at varying frequencies in most US states. Small numbers have been detected in other countries in North America, and in Europe, Asia and Australia. After an initial increase, its frequency rapidly dropped from February 2021 as it was being outcompeted by the more transmissible Alpha. In April, Epsilon remained relatively frequent in parts of northern California, but it had virtually disappeared from the south of the state and had never been able to establish a foothold elsewhere; only 3.2% of all cases in the United States were Epsilon, whereas more than two-thirds were Alpha.\\n\\n\\n=== Zeta (lineage P.2) ===\\n\\nZeta variant or lineage P.2, a sub-lineage of B.1.1.28 like Gamma (P.1), was first detected in circulation in the state of Rio de Janeiro; it harbours the E484K mutation, but not the N501Y and K417T mutations. It evolved independently in Rio de Janeiro without being directly related to the Gamma variant from Manaus. Though previously Zeta was labeled a variant of interest, as of July 2021, it is no longer considered as such by the WHO.\\n\\n\\n=== Eta (lineage B.1.525) ===\\n\\nThe Eta variant or lineage B.1.525, also called VUI-21FEB-03 (previously VUI-202102/03) by Public Health England (PHE) and formerly known as UK1188, 21D or 20A/S:484K, does not carry the same N501Y mutation found in Alpha, Beta and Gamma, but carries the same E484K-mutation as found in the Gamma, Zeta, and Beta variants, and also carries the same ΔH69/ΔV70 deletion (a deletion of the amino acids histidine and valine in positions 69 and 70) as found in Alpha, N439K variant (B.1.141 and B.1.258) and Y453F variant (Cluster 5). Eta differs from all other variants by having both the E484K-mutation and a new F888L mutation (a substitution of phenylalanine (F) with leucine (L) in the S2 domain of the spike protein). As of 5 March 2021, it had been detected in 23 countries. It has also been reported in Mayotte, the overseas department/region of France. The first cases were detected in December 2020 in the UK and Nigeria, and as of 15 February 2021, it had occurred in the highest frequency among samples in the latter country. As of 24 February 56 cases were found in the UK. Denmark, which sequences all its COVID-19 cases, found 113 cases of this variant from 14 January to 21 February 2021, of which seven were directly related to foreign travel to Nigeria.As of July 2021, UK experts are studying it to ascertain how much of a risk it could be. It is currently regarded as a \"variant under investigation\", but pending further study, it may become a \"variant of concern\". Ravi Gupta, from the University of Cambridge said in a BBC interview that lineage B.1.525 appeared to have \"significant mutations\" already seen in some of the other newer variants, which means their likely effect is to some extent more predictable.\\n\\n\\n=== Theta (lineage P.3) ===\\n\\nOn 18 February 2021, the Department of Health of the Philippines confirmed the detection of two mutations of COVID-19 in Central Visayas after samples from patients were sent to undergo genome sequencing. The mutations were later named as E484K and N501Y, which were detected in 37 out of 50 samples, with both mutations co-occurrent in 29 out of these.On 13 March, the Department of Health confirmed the mutations constitutes a variant which was designated as lineage P.3. On the same day, it also confirmed the first COVID-19 case caused by the Gamma variant in the country. The Philippines had 98 cases of the Theta variant on 13 March. On 12 March it was announced that Theta had also been detected in Japan. On 17 March, the United Kingdom confirmed its first two cases, where PHE termed it VUI-21MAR-02.\\nOn 30 April 2021, Malaysia detected 8 cases of the Theta variant in Sarawak.As of July 2021, Theta is no longer considered a variant of interest by the WHO.\\n\\n\\n=== Iota (lineage B.1.526) ===\\n\\nIn November 2020, a mutant variant was discovered in New York City, which was named lineage B.1.526. As of 11 April 2021, the variant has been detected in at least 48 U.S. states and 18 countries. In a pattern mirroring Epsilon, Iota was initially able to reach relatively high levels in some states, but by May 2021 it was outcompeted by the more transmissible Delta and Alpha.\\n\\n\\n=== Kappa (lineage B.1.617.1) ===\\n\\nThe Kappa variant is one of the three sublineages of lineage B.1.617. It is also known as lineage B.1.617.1, 21B or 21A/S:154K, and was first detected in India in December 2020. By the end of March 2021, Kappa accounted for more than half of the sequences being submitted from India. On 1 April 2021, it was designated a variant under investigation (VUI-21APR-01) by Public Health England. It has the notable mutations L452R, E484Q, P681R.\\n\\n\\n== Variants under monitoring (WHO) ==\\nDefined as variants with genetic changes suspected to affect virus characteristics and some indication of posing a future risk, but with unclear evidence of phenotypic or epidemiological impact, requiring enhanced monitoring and repeat assessment after new evidence. Some former variants of interest are monitored as well.\\n\\n\\n== Formerly monitored variants (WHO) ==\\n\\n\\n== Other notable variants ==\\nLineage B.1.1.207 was first sequenced in August 2020 in Nigeria; the implications for transmission and virulence are unclear but it has been listed as an emerging variant by the US Centers for Disease Control. Sequenced by the African Centre of Excellence for Genomics of Infectious Diseases in Nigeria, this variant has a P681H mutation, shared in common with the Alpha variant. It shares no other mutations with the Alpha variant and as of late December 2020 this variant accounts for around 1% of viral genomes sequenced in Nigeria, though this may rise. As of May 2021, lineage B.1.1.207 has been detected in 10 countries.Lineage B.1.1.317, while not considered a variant of concern, is noteworthy in that Queensland Health forced 2 people undertaking hotel quarantine in Brisbane, Australia to undergo an additional 5 days\\' quarantine on top of the mandatory 14 days after it was confirmed they were infected with this variant.Lineage B.1.616, being identified in Brittany, Western France in early January 2021 and designated by WHO as \"Variant under investigation\" in March 2021, was reported to be difficult to detect from nasopharyngeal swab sampling method of coronavirus detection, and detection of the virus needs to rely on samples from lower respiratory tract.Lineage B.1.618 was first isolated in October 2020. It has the E484K mutation in common with several other variants, and showed significant spread in April 2021 in West Bengal, India. As of 23 April 2021, the PANGOLIN database showed 135 sequences detected in India, with single-figure numbers in each of eight other countries worldwide.In July 2021, scientists reported in a preprint which was published in a journal in February 2022, the detection of anomalous unnamed unknown-host SARS-CoV-2 lineages via wastewater surveillance in New York City. They hypothesized that \"these lineages are derived from unsampled human COVID-19 infections or that they indicate the presence of a non-human animal reservoir\". Lineage B.1.640.2 (also known as the IHU variant) was detected in October 2021 by researchers at the Institut Hospitalo-Universitaire (IHU) in Marseille. They found the variant in a traveler who returned to France from Cameroon and reportedly infected 12 people. The B.1.640 lineage, which includes B.1.640.2, was designated a variant under monitoring (VUM) by the World Health Organization (WHO) on 22 November 2021. However, the WHO has reported that lineage B.1.640.2 has spread much slower than the Omicron variant, and so is of relatively little concern.  According to a preprint study, lineage B.1.640.2 has two already known spike protein mutations – E484K and N501Y – among a total of 46 nucleotide substitutions and 37 deletions.\\n\\n\\n== Notable missense mutations ==\\nThere have been a number of missense mutations observed of SARS-CoV-2.\\n\\n\\n=== del 69-70 ===\\nThe name of the mutation, del 69-70, or 69-70 del, or other similar notations, refers to the deletion of amino acid at position 69 to 70. The mutation is found in the Alpha variant, and could lead to \"spike gene target failure\" and result in false negative result in PCR virus test.\\n\\n\\n=== RSYLTPGD246-253N ===\\nOtherwise referred to as del 246-252, or other various similar expression, refer to the deletion of amino acid from the position of 246 to 252, in the N-terminal domain of spike protein, accompanied with a replacement of the aspartic acid (D) at the position 253 for asparagine (N).The 7 amino acid deletion mutation is currently described as unique in the Lambda variant, and have been attributed to as one of the cause of the strain\\'s increased capability to escape from neutralizing antibodies according to preprint paper.\\n\\n\\n=== N440K ===\\nThe name of the mutation, N440K, refers to an exchange whereby the asparagine (N) is replaced by lysine (K) at position 440.This mutation has been observed in cell cultures to be 10 times more infective compared to the previously widespread A2a strain (A97V substitution in RdRP sequence) and 1000 times more in the lesser widespread A3i strain (D614G substitution in Spike and a and P323L substitution in RdRP). It was involved in rapid surges of COVID-19 cases in India in May 2021. India has the largest proportion of N440K mutated variants followed by the US and Germany.\\n\\n\\n=== G446V ===\\nThe name of the mutation, G446V, refers to an exchange whereby the glycine (G) is replaced by valine (V) at position 446.The mutation, identified in Japan among inbound travelers starting from May, and among 33 samples from individuals related to 2020 Tokyo Olympic Games and 2020 Tokyo Paralympic Games, are said to be possible to impact affinity of multiple monoclonal antibody, although its clinical impact against the use of antibody medicine is still yet to be known.\\n\\n\\n=== L452R ===\\nThe name of the mutation, L452R, refers to an exchange whereby the leucine (L) is replaced by arginine (R) at position 452.L452R is found in both the Delta and Kappa variants which first circulated in India, but have since spread around the world. L452R is a relevant mutation in this strain that enhances ACE2 receptor binding ability and can reduce vaccine-stimulated antibodies from attaching to this altered spike protein.\\nL452R, some studies show, could even make the coronavirus resistant to T cells, that are class of cells necessary to target and destroy virus-infected cells. They are different from antibodies that are useful in blocking coronavirus particles and preventing it from proliferating.\\n\\n\\n=== Y453F ===\\nThe name of the mutation, Y453F, refers to an exchange whereby the tyrosine (Y) is replaced by phenylalanine (F) at position 453. The mutation have been found potentially linked to the spread of SARS-CoV-2 among minks in the Netherlands in 2020.\\n\\n\\n=== S477G/N ===\\nA highly flexible region in the receptor binding domain (RBD) of SARS-CoV-2, starting from residue 475 and continuing up to residue 485, was identified using bioinformatics and statistical methods in several studies. The University of Graz and the Biotech Company Innophore have shown in a recent publication that structurally, the position S477 shows the highest flexibility among them.At the same time, S477 is hitherto the most frequently exchanged amino acid residue in the RBDs of SARS-CoV-2 mutants. By using molecular dynamics simulations of RBD during the binding process to hACE2, it has been shown that both S477G and S477N strengthen the binding of the SARS-COV-2 spike with the hACE2 receptor. The vaccine developer BioNTech referenced this amino acid exchange as relevant regarding future vaccine design in a preprint published in February 2021.\\n\\n\\n=== E484Q ===\\nThe name of the mutation, E484Q, refers to an exchange whereby the glutamic acid (E) is replaced by glutamine (Q) at position 484.The Kappa variant circulating in India has E484Q. These variants were initially (but misleadingly) referred to as a \"double mutant\". E484Q may enhance ACE2 receptor binding ability, and may reduce vaccine-stimulated antibodies\\' ability to attach to this altered spike protein.\\n\\n\\n=== E484K ===\\nThe name of the mutation, E484K, refers to an exchange whereby the glutamic acid (E) is replaced by lysine (K) at position 484. It is nicknamed \"Eeek\".E484K has been reported to be an escape mutation (i.e., a mutation that improves a virus\\'s ability to evade the host\\'s immune system) from at least one form of monoclonal antibody against SARS-CoV-2, indicating there may be a \"possible change in antigenicity\". The Gamma variant (lineage P.1), the Zeta variant (lineage P.2, also known as lineage B.1.1.28.2) and the Beta variant (501.V2) exhibit this mutation. A limited number of lineage B.1.1.7 genomes with E484K mutation have also been detected. Monoclonal and serum-derived antibodies are reported to be from 10 to 60 times less effective in neutralising virus bearing the E484K mutation. On 2 February 2021, medical scientists in the United Kingdom reported the detection of E484K in 11 samples (out of 214,000 samples), a mutation that may compromise current vaccine effectiveness.\\n\\n\\n=== F490S ===\\nF490S denotes a change from phenylalanine (F) to serine (S) in amino-acid position 490.It is one of the mutation found in Lambda, and have been associated with reduced susceptibility to antibody generated by those who were infected with other strains, meaning antibody treatment against people infected with strains carrying such mutation would be less effective.\\n\\n\\n=== N501Y ===\\nN501Y denotes a change from asparagine (N) to tyrosine (Y) in amino-acid position 501. N501Y has been nicknamed \"Nelly\".This change is believed by PHE to increase binding affinity because of its position inside the spike glycoprotein\\'s receptor-binding domain, which binds ACE2 in human cells; data also support the hypothesis of increased binding affinity from this change. Molecular interaction modelling and the free energy of binding calculations has demonstrated that the mutation N501Y has the highest binding affinity in variants of concern RBD to hACE2. Variants with N501Y include Gamma, Alpha (VOC 20DEC-01), Beta, and COH.20G/501Y (identified in Columbus, Ohio). This last became the dominant form of the virus in Columbus in late December 2020 and January and appears to have evolved independently of other variants.\\n\\n\\n=== N501S ===\\nN501S denotes a change from asparagine (N) to serine (S) in amino-acid position 501.As of September 2021, there are 8 cases of patients around the world infected with Delta variant which feature this N501S mutation. As it is considered a mutation similar to N501Y, it is suspected to have similar characteristics as N501Y mutation, which is believed to increase the infectivity of the virus, however the exact effect is unknown yet.\\n\\n\\n=== D614G ===\\n\\nD614G is a missense mutation that affects the spike protein of SARS-CoV-2. From early appearances in Eastern China early in 2020, the frequency of this mutation in the global viral population increased early on during the pandemic. G (glycine) quickly replaced D (aspartic acid) at position 614 in Europe, though more slowly in China and the rest of East Asia, supporting the hypothesis that G increased the transmission rate, which is consistent with higher viral titres and infectivity in vitro. Researchers with the PANGOLIN tool nicknamed this mutation \"Doug\".In July 2020, it was reported that the more infectious D614G SARS-CoV-2 variant had become the dominant form in the pandemic. PHE confirmed that the D614G mutation had a \"moderate effect on transmissibility\" and was being tracked internationally.The global prevalence of D614G correlates with the prevalence of loss of smell (anosmia) as a symptom of COVID-19, possibly mediated by higher binding of the RBD to the ACE2 receptor or higher protein stability and hence higher infectivity of the olfactory epithelium.Variants containing the D614G mutation are found in the G clade by GISAID and the B.1 clade by the PANGOLIN tool.\\n\\n\\n=== Q677P/H ===\\nThe name of the mutation, Q677P/H, refers to an exchange whereby the glutamine (Q) is replaced by proline (P) or histidine (H) at position 677. There are several sub-lineages containing the Q677P mutation; six of these, which also contain various different combinations of other mutations, are referred to by names of birds. One of the earlier ones noticed for example is known as \"Pelican,\" while the most common of these as of early 2021 was provisionally named \"Robin 1.\"The mutation has been reported in multiple lineages circulating inside the United States as of late 2020 and also some lineages outside the country. \\'Pelican\\' was first detected in Oregon, and as of early 2021 \\'Robin 1\\' was found often in the Midwestern United States, while another Q667H sub-lineage, \\'Robin 2\\', was found mostly in the southeastern United States. The frequency of such mutation being recorded has increased from late 2020 to early 2021.\\n\\n\\n=== P681H ===\\n\\nThe name of the mutation, P681H, refers to an exchange whereby the proline (P) is replaced by histidine (H) at position 681.In January 2021, scientists reported in a preprint that the mutation P681H, a characteristic feature of the Alpha variant and lineage B.1.1.207 (identified in Nigeria), is showing a significant exponential increase in worldwide frequency, thus following a trend to be expected in the lower limb of the logistics curve. This may be compared with the trend of the now globally prevalent D614G.\\n\\n\\n=== P681R ===\\nThe name of the mutation, P681R, refers to an exchange whereby the proline (P) is replaced by arginine (R) at position 681.Indian SARS-CoV-2 Genomics Consortium (INSACOG) found that other than the two mutations E484Q and L452R, there is also a third significant mutation, P681R in lineage B.1.617. All three concerning mutations are on the spike protein, the operative part of the coronavirus that binds to receptor cells of the body.\\n\\n\\n=== A701V ===\\nAccording to initial media reports, the Malaysian Ministry of Health announced on 23 December 2020 that it had discovered a mutation in the SARS-CoV-2 genome which they designated as A701B(sic), among 60 samples collected from the Benteng Lahad Datu cluster in Sabah. The mutation was characterised as being similar to the one found recently at that time in South Africa, Australia, and the Netherlands, although it was uncertain if this mutation was more infectious or aggressive than before. The provincial government of Sulu in neighbouring Philippines temporarily suspended travel to Sabah in response to the discovery of \\'A701B\\' due to uncertainty over the nature of the mutation.On 25 December 2020, the Malaysian Ministry of Health described a mutation A701V as circulating and present in 85% of cases (D614G was present in 100% of cases) in Malaysia. These reports also referred to samples collected from the Benteng Lahad Datu cluster. The text of the announcement was mirrored verbatim on the Facebook page of Noor Hisham Abdullah, Malay Director-General of Health, who was quoted in some of the news articles.The A701V mutation has the amino acid alanine (A) substituted by valine (V) at position 701 in the spike protein. Globally, South Africa, Australia, Netherlands and England also reported A701V at about the same time as Malaysia. In GISAID, the prevalence of this mutation is found to be about 0.18%. of cases.On 14 April 2021, the Malaysian Ministry of Health reported that the third wave, which had started in Sabah, has involved the introduction of variants with D614G and A701V mutations.\\n\\n\\n== Recombinant variants ==\\nThe British government has reported a number of recombinant variants of SARS-CoV-2. These recombinant lineages have been given the Pango lineage identifiers XD, XE, and XF.XE is a recombinant lineage of Pango lineages BA.1 and BA.2. As of March 2022 XE was believed to have a growth rate 9.8% greater than BA.2.\\n\\n\\n== Differential vaccine effectiveness ==\\n\\nThe interplay between the SARS-CoV-2 virus and its human hosts was initially natural but then started being altered by the rising availability of vaccines seen in 2021. The potential emergence of a SARS-CoV-2 variant that is moderately or fully resistant to the antibody response elicited by the COVID-19 vaccines may necessitate modification of the vaccines. The emergence of vaccine-resistant variants is more likely in a highly vaccinated population with uncontrolled transmission.As of February 2021, the US Food and Drug Administration believed that all FDA authorized vaccines remained effective in protecting against circulating strains of SARS-CoV-2.\\n\\n\\n== Data and methods ==\\nModern DNA sequencing, where available, may permit rapid detection (sometimes known as \\'real-time detection\\') of genetic variants that appear in pathogens during disease outbreaks. Through use of phylogenetic tree visualisation software, records of genome sequences can be clustered into groups of identical genomes all containing the same set of mutations. Each group represents a \\'variant\\', \\'clade\\', or \\'lineage\\', and comparison of the sequences allows the evolutionary path of a virus to be deduced. For SARS-CoV-2, until  March 2021, over 330,000 viral genomic sequences had been generated by molecular epidemiology studies across the world.\\n\\n\\n=== New variant detection and assessment ===\\nOn 26 January 2021, the British government said it would share its genomic sequencing capabilities with other countries in order to increase the genomic sequencing rate and trace new variants, and announced a \"New Variant Assessment Platform\". As of January 2021, more than half of all genomic sequencing of COVID-19 was carried out in the UK.Wastewater surveillance was demonstrated to be one technique to detect SARS-CoV-2 variants and to track their rise for studying related ongoing infection dynamics.\\n\\n\\n=== Testing ===\\nWhether one or more mutations visible in RT-PCR tests can be used reliably to identify a variant depends on the prevalence of other variants currently circulating in the same population.\\n\\n\\n== Incubation theory for multiple mutated variants ==\\n\\nResearchers have suggested that multiple mutations can arise in the course of the persistent infection of an immunocompromised patient, particularly when the virus develops escape mutations under the selection pressure of antibody or convalescent plasma treatment, with the same deletions in surface antigens repeatedly recurring in different patients.\\n\\n\\n== Cross-species transmission ==\\n\\nThere is a risk that COVID-19 could transfer from humans to other animal populations and could combine with other animal viruses to create yet more variants that are dangerous to humans. Reverse zoonosis spillovers may cause reservoirs for mutating variants that spill back to humans – another possible source for variants of concern, in addition to immunocompromised people.\\n\\n\\n=== Cluster 5 ===\\n\\nIn early November 2020, Cluster 5, also referred to as ΔFVI-spike by the Danish State Serum Institute (SSI), was discovered in Northern Jutland, Denmark. It is believed to have been spread from minks to humans via mink farms. On 4 November 2020, it was announced that the mink population in Denmark would be culled to prevent the possible spread of this mutation and reduce the risk of new mutations happening. A lockdown and travel restrictions were introduced in seven municipalities of Northern Jutland to prevent the mutation from spreading, which could compromise national or international responses to the COVID-19 pandemic. By 5 November 2020, some 214 mink-related human cases had been detected.The WHO stated that cluster 5 had a \"moderately decreased sensitivity to neutralising antibodies\". SSI warned that the mutation could reduce the effect of COVID-19 vaccines under development, although it was unlikely to render them useless. Following the lockdown and mass-testing, SSI announced on 19 November 2020 that cluster 5 in all probability had become extinct. As of 1 February 2021, authors to a peer-reviewed paper, all of whom were from the SSI, assessed that cluster 5 was not in circulation in the human population.\\n\\n\\n== See also ==\\n\\nRaTG13, the second closest known relative to SARS-CoV-2\\nPandemic prevention § Surveillance and mapping\\nCOVID-19 vaccine#Effectiveness\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== Further reading ==\\n\\n\\n== External links ==\\nCoVariants\\nCov-Lineages\\nGISAID – hCov19 Variants',\n",
              " 'The Gamma variant (P.1) is one of the variants of SARS-CoV-2, the virus that causes COVID-19. This variant of SARS-CoV-2 has been named lineage P.1 and has 17 amino acid substitutions, ten of which in its spike protein, including these three designated to be of particular concern: N501Y, E484K and K417T. This variant of SARS-CoV-2 was first detected by the National Institute of Infectious Diseases (NIID) of Japan, on 6 January 2021 in four people who had arrived in Tokyo having visited Amazonas, Brazil, four days earlier. It was subsequently declared to be in circulation in Brazil. Under the simplified naming scheme proposed by the World Health Organization, P.1 has been labeled Gamma variant, and is currently considered a variant of concern.Gamma caused widespread infection in early 2021 in the city of Manaus, the capital of Amazonas, although the city had already experienced widespread infection in May 2020, with a study indicating high seroprevalence of antibodies for SARS-CoV-2. A research article published in Science Journal indicate that P.1 infected people have a greater chance of transmissibility and death than B.1.1.28 infected ones.The Gamma variant comprises the two distinct subvariants 28-AM-1 and 28-AM-2, which both carry the K417T, E484K, N501Y mutations, and which both developed independently of each other within the same Brazilian Amazonas region.Gamma is notably different from the Zeta variant (lineage P.2) which also circulated strongly in Brazil. In particular, Zeta only carries the E484K mutation and has neither of the other two mutations of concern, N501Y and K417T.\\n\\n\\n== Classification ==\\nInitial reports claimed that both P.1 and P.2 were two separate and different descendants of the Brazilian lineage B.1.1.248. However, B.1.1.248 later lost its status as a distinct lineage and was reclassified to B.1.1.28. P.1 has also been called B.1.1.28.1, while P.2 has been B.1.1.28.2 or VUI-202101/01. Since only three sublevels are permitted in the PANGO Lineage system of nomenclature, hence the designation of B.1.1.28.1 to P.1 and B.1.1.28.2 to P.2.Following its detection, genome data for four samples of the new variant were shared to GISAID having been assigned the ID range: EPI_ISL_792680 to EPI_ISL_792683.\\n\\n\\n=== Mutations ===\\n\\nVariants of SARS-CoV-2\\nAs well as having eight mutations (four of these synonymous genetic mutations) in its open reading frames (ORF1a and ORF1b) – one of which is a set of deletions – Gamma has 10 defining mutations in its spike protein, including N501Y and E484K. It also has two mutations – one an insertion – in its ORF8 gene and one in its N gene.\\n\\n\\n=== Descendant and sublineages ===\\nCoronavirus lineage B.1.1.28 has originated four known lineages classified as variant of interest (VOI) or variant of concern (VOC): lineages P.1, P.2, P.3 and P.4.\\nLineage P.2 (B.1.1.28.2, Zeta variant), first detected in October 2020 in the state of Rio de Janeiro, Brazil, only shares one mutation of concern with P.1, which is the E484K. The other P.2 mutations are without concern and rarely found for other variants. The five P.2-specific mutations are: E484K in S-gene, A119S in N-gene, 5’UTR C100U, plus L3468V and synC11824U in ORF1ab-gene. Other mutations commonly found in P.2 are: 3’UTR C29754U,  F120F (synC28253U) in ORF8, M234I in the N-gene, plus L3930F and synA12964G in ORF1ab.Lineage P.3 (Theta variant) was first identified in the Philippines on 18 February 2021 when two mutations of concern were detected in Central Visayas.The remaining B.1.1.28 derivative virus is lineage P.4. Although researchers have not identified its precise origin, it was first sequenced in Itirapina, Brazil, and was already circulating in various municipalities in the state of São Paulo of the same country. It carries a mutation of concern in the spike protein called L452R which is also present in lineage B.1.617 (Delta and Kappa variants) detected in India, Epsilon variant (lineages B.1.427 and B.1.429) from California, United States. The branch of this lineage is P.4.1 (VUI-NP13L)—suspected to have arisen in Goiás, Brazil, around June–July 2020— also rapidly spread to the southeast of the country, where for example Taquara had its first genome sequence, and to the northeast of the nation. It was detected internationally, with reported cases in Japan, Netherlands and England. The P.4.1 has V1176F and D614G mutations in spike protein.\\n\\n\\n== Prevention ==\\n\\n\\n== Statistics ==\\n\\n\\n== History ==\\nOn 12 January 2021, the Brazil–United Kingdom CADDE Centre confirmed 13 local cases of lineage P.1 in Manaus, Amazonas state, the largest city of the Amazon rain forest. The new lineage was absent in 27 samples collected from March to November 2020 from Manaus, but it was identified for the same city in 42% (n=13/31) of the samples collected 15–23 December 2020, followed by 52.2% (n=35/67) during 15–31 December 2020 and 85.4% (n=41/48) during 1–9 January 2021. Most notably, the P.2 was rapidly outcompeted by P1 going from the second half of December to 1–9 January, where the lineage P.2 share for Manaus decreased from 25.4% to 6.3%.A study of 180 sequenced Brazilian samples collected in the state of Rio de Janeiro during 2020, identified emergence of the novel lineage P.2 of SARS-CoV-2 (originating from B.1.1.28). P.2 was first detected by genome sequencing in October 2020, but it was estimated to have emerged in early July 2020. As of December 2020, although having significantly increased in frequency throughout the state, it was still largely confined to the state capital Rio de Janeiro. In May 2020 the main lineages behind the COVID-19 positives were B.1.1.33 (70%) and B.1.1.28 (20%), whereas by September the main lineages were B.1.1.33 (50%) and B.1.1.28 (40%), with no detected presence of P.2, while during October and November P.2 was the most common lineage with a share close to 50% (according to the Pangolin tool). The study also found the E484K mutation as \"widely spread\" across all analysed P.2 samples (36 out of 38).Researchers at the Oswaldo Cruz Foundation published a preprint genomic epidemiology study of 250 collected genomes from different places in Amazonas and found that P.1 infections can produce nearly 10 times more viral load than in other COVID-19-infected persons involving lineages B.1.1.28 and B.1.195. The lineage also showed 2.2 times higher transmissibility with the same ability to infect both adults (18–59 years old) and older persons (60 years old and higher), suggesting P.1 and its sublineages are more successful at infecting younger humans with no gender differential.The Centre for Arbovirus Discovery, Diagnosis, Genomics and Epidemiology (CADDE) produced another journal article of samples collected in Manaus between November 2020 and January 2021. The study indicated lineage P.1 to be about 2.0 times (50% CrI, 1.7–2.4 times) more transmissible and was shown to be capable of evading about 32% (50% CrI, 21–46%) of inherited immunity from previous coronavirus diseases, leading to the possibility of reinfection. These increased statistics also had the same reflection in fatality, in that P.1 infections can be about 50% (50% CrI, 20–90%) more lethal. As part of ongoing research, the variant\\'s capacity to neutralise antibodies has been evaluated by scientists in a published preprint work demonstrating that 8 CoronaVac-immunised persons had a poor blood plasma response against lineage P.1. Since the study only had a small number of participants, it was not possible to establish any statistical conclusion as a larger number of vaccinated people would need to be studied. Scientists at MIT, Harvard and Cambridge, and hospitals physicians in Boston, corroborated that people fully vaccinated  with Pfizer and Moderna vaccines have significantly decreased neutralisation with P.1—in a preprint work.\\n\\n\\n== See also ==\\n\\nVariants of SARS-CoV-2: Alpha, Beta, Delta, Epsilon, Zeta, Eta, Theta, Iota, Kappa, Lambda, Mu, Omicron\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== External links ==\\nPANGO lineages: New variant report -Report on global distribution of 3 variants including P.1, a descendent of B.1.1.28\\nCOG-UK Report on SARS-CoV-2 Spike mutations of interest in the UK',\n",
              " 'Angiotensin-converting enzyme 2 (ACE2) is an enzyme that can be found either attached to the membrane of cells (mACE2) in the intestines, kidney, testis, gallbladder, and heart or in a soluble form (sACE2). Both membrane bound and soluble ACE2 are integral parts of the renin–angiotensin–aldosterone system (RAAS) that exists to keep the body\\'s blood pressure in check. While mACE2 does not appear to factor into the harmful phase of RAAS (the increase of blood pressure), its existence is vital in order for the enzyme ADAM17 to cleave its extracellular domain to create soluble ACE2 (sACE2). Soluble ACE2 lowers blood pressure by catalyzing the hydrolysis of angiotensin II (a vasoconstrictor peptide) into angiotensin (1–7)  (a vasodilator) which in turns binds to MasR receptors creating localized vasodilation and hence decreasing blood pressure. This decrease in blood pressure makes the entire process a promising drug target for treating cardiovascular diseases.mACE2 also serves as the entry point into cells for some coronaviruses, including  HCoV-NL63, SARS-CoV, and SARS-CoV-2. The SARS-CoV-2 spike protein itself is known to damage the endothelium via downregulation of ACE2. The human version of the enzyme can be referred to as hACE2.\\n\\n\\n== Structure ==\\nMembrane bound Angiotensin-converting enzyme 2 (mACE2) is a zinc-containing metalloenzyme located on the surface of intestinal enterocytes, renal tubular cells and other cells. mACE2 protein contains an N-terminal peptidase M2 domain and a C-terminal collectrin renal amino acid transporter domain.\\nmACE2 is a single-pass type I membrane protein, with its enzymatically active domain exposed on the surface of cells in the intestines and other tissues. The extracellular domain of mACE2 can be cleaved from the transmembrane domain by another enzyme known as ADAM17 a member of the sheddase enzyme family, during the protective phase of RAAS, the Renin Angiotension Aldosterone System which regulates our body\\'s blood pressure. The resulting cleaved protein is known as soluble ACE2 or sACE2. It is released into the bloodstream where one of sACE2\\'s functions is to turn excess angiotensin II into angiotensin 1-7 which binds to MasR receptors creating localized vasodilation and hence decreasing blood pressure. Excess sACE2 may ultimately be excreted in the urine.\\n\\n\\n== Location within the human body ==\\nmACE2 is attached to the cell membrane of mainly enterocytes of the small intestine and duodenum, proximal tubular cells of the kidneys, glandular cells of the gallbladder, as well as Sertoli cells and Leydig cells of the testis. The expression profile of mACE2 in the human body was recently thoroughly evaluated by the Human Protein Atlas team using a large-scale multiomics approach combining several different methods for analysis of gene expression, including a stringent immunohistochemical analysis using two independent antibodies. mACE2 expression was in addition to the above-mentioned tissues, also seen in endothelial cells and pericytes of blood vessels within certain tissues, cardiomyocytes in heart tissue, and a smaller subset of cells within the thyroid gland, epididymis, seminal vesicle, pancreas, liver and placenta. Despite the fact that the respiratory system is the primary route of SARS-CoV-2 infection, very limited expression is seen, both at protein and mRNA level. The expression within the respiratory system is mainly restricted to the upper bronchial and nasal epithelia, especially in the ciliated cells.\\n\\n\\n== Function ==\\nAs part of the renin–angiotensin–aldosterone system (RAAS) protective phase, soluble ACE2\\'s (sACE2) important function is to act as a counterbalance to the angiotensin-converting enzyme (ACE). ACE cleaves angiotensin I hormone into the vasoconstricting  angiotensin II which causes a cascade of hormonal reactions which is part of the body\\'s harmful phase of RAAS, which ultimately leads to an increase in the body\\'s blood pressure.  ACE2 has an opposing effect to ACE, degrading angiotensin II into angiotensin (1-7), thereby lowering blood pressure.sACE2, as part of RAAS\\'s protective phase, cleaves the carboxyl-terminal amino acid phenylalanine from angiotensin II (Asp-Arg-Val-Tyr-Ile-His-Pro-Phe) and hydrolyses it into the vasodilator angiotensin (1-7) (H-Asp-Arg-Val-Tyr-Ile-His-Pro-OH), which binds to Mas Receptors and ultimately leads to a decrease in blood pressure.  sACE2 can also cleave numerous peptides, including [des-Arg9]-bradykinin, apelin, neurotensin, dynorphin A, and ghrelin.mACE2 also regulates the membrane trafficking of the neutral amino acid transporter SLC6A19 and has been implicated in Hartnup\\'s disease.Research in mice has shown that ACE2 (whether it is the membrane bound version or soluble is inconclusive) is involved in regulation of the blood glucose level but its mechanism is yet to be confirmed.\\n\\n\\n== Coronavirus entry point ==\\nAs a transmembrane protein, mACE2 serves as the main entry point into cells for some coronaviruses, including HCoV-NL63, SARS-CoV (the virus that causes SARS), and SARS-CoV-2 (the virus that causes COVID-19).  More specifically, the binding of the spike S1 protein of SARS-CoV and SARS-CoV-2 to the enzymatic domain of mACE2 on the surface of cells results in endocytosis and translocation of both the virus and the enzyme into endosomes located within cells. The binding of the SARS-CoV-2 virus through mACE2 receptors present in heart tissue may be responsible for direct viral injury leading to myocarditis. In a study done during the SARS outbreak, SARS virus RNA was ascertained in the autopsy of heart specimens in 35% of the patients who died due to SARS. It was also observed that an already diseased heart has increased expression of mACE2 receptors contrasted to healthy individuals. This entry process also requires priming of the S protein by the host serine protease TMPRSS2, the inhibition of which is under current investigation as a potential therapeutic. It has also been shown that disruption of S-protein glycosylation significantly impairs viral entry, indicating the importance of glycan-protein interactions in the process. Amino acid variation in the ACE2 protein is able to affect the affinity of binding with the S protein of SARS-CoV-2.This has led some to hypothesize that decreasing the levels of mACE2, in cells, might help in fighting the infection.  Furthermore, according to studies conducted on mice, the interaction of the spike protein of the coronavirus with mACE2 induces a drop in the levels of mACE2 in cells through internalization and degradation of the protein and hence may contribute to lung damage.On the other hand, sACE2 has been shown to have a protective effect against virus-induced lung injury by increasing the production of the vasodilator angiotensin 1–7. Furthermore, some researchers have hypothesized that sACE2 (which is created during the Protective Phase of RAAS) is not only involved in binding to Angiotensin II to create Angiotensin I-7 which lowers blood pressure by vasodilation, but that free and soluble ACE2 may also be binding to coronavirus spike proteins, hence making those coronavirus spikes unavailable for binding to mACE-2 sites. But even with only tiny amounts of mACE2, SARS-CoV-2 virus can gain entry into cells if TMPRSS2 is present.Both ACE inhibitors and angiotensin II receptor blockers (ARBs) that are used to treat high blood pressure have been shown in rodent studies to upregulate mACE2 expression, possibly affecting the severity of coronavirus infections.However, a systematic review and meta-analysis published on July 11, 2012, found that \"use of ACE inhibitors was associated with a significant 34% reduction in risk of pneumonia compared with controls.\" Moreover, \"the risk of pneumonia was also reduced in patients treated with ACE inhibitors who were at higher risk of pneumonia, in particular those with stroke and heart failure. Use of ACE inhibitors was also associated with a reduction in pneumonia related mortality, although the results were less robust than for overall risk of pneumonia.\"  An April 2020 study of patients hospitalized in Hubei Province in China found a death rate of 3.7% for patients suffering from hypertension who were taking ACE inhibitors or ARBs. The death rate was compared with 9.8% of hospitalized patients with hypertension not taking such drugs, suggesting that ACE inhibitors and ARBs are not harmful and may help against the coronavirus.Despite lack of conclusive evidence, some have advocated for and against the cessation of ACE inhibitor or ARB treatment in COVID-19 patients with hypertension. However, multiple professional societies and regulatory bodies have recommended continuing standard ACE inhibitor and ARB therapy.Plasma ACE2 levels predict outcome of COVID-19 in hospitalized patients, with higher plasma levels being correlated with worse disease outcomes. Patients with high blood pressure or heart disease show elevated ACE2 plasma levels.\\n\\n\\n== Recombinant human ACE2 ==\\nRecombinant human ACE2 (rhACE2) is surmised to be a novel therapy for acute lung injury, and appeared to improve pulmonary blood flow and oxygen saturation in piglets with a lipopolysaccharide-induced acute respiratory distress syndrome. The half-life of rhACE2 in human beings is about 10 hours, and the onset of action is 30 minutes in addition to the course of effect (duration) of 24 hours.  Several findings suggest that rhACE2 may be a promising drug for those with intolerance to classic renin-angiotensin system inhibitors (RAS inhibitors) or in diseases where circulating angiotensin II is elevated.An in vitro study focused on the early stages of infection found that clinical-grade human recombinant soluble ACE2 (hrsACE2) reduced SARS-CoV-2 recovery from vero cells by a factor of 1,000-5,000. The equivalent mouse rsACE2 did not have such an effect. This study suggests that rhsACE2 not only restores the renin-angiotensin system to balance as in the earlier ARDS studies, but also directly slows down infection by this virus – possibly as a decoy.Infused rhACE2 has been evaluated in clinical trials for the treatment of acute respiratory distress syndrome (ARDS). rhACE2 is in phase II trial for severe COVID-19.\\n\\n\\n== See also ==\\nRenin–angiotensin system\\nAngiotensin-converting enzyme\\nSARS-CoV-2 § Structural biology\\n\\n\\n== References ==\\n\\n\\n== External links ==\\nHuman ACE2 genome location and ACE2 gene details page  in the UCSC Genome Browser.\\nAngiotensin-converting enzyme 2 in Membranome database\\n3D structure of complex of a neurotransmitter sodium symporter B(0)AT1, ACE2, and SARS-CoV-2  receptor-binding domain in OPM database\\nOverview of all the structural information available in the PDB for UniProt: Q9BYF1 (Angiotensin-converting enzyme 2) at the PDBe-KB.',\n",
              " 'The Delta variant (B.1.617.2) is a variant of SARS-CoV-2, the virus that causes COVID-19. It was first detected in India in late 2020. The Delta variant was named on 31 May 2021 and had spread to over 179 countries by 22 November 2021. The World Health Organization (WHO) indicated in June 2021 that the Delta variant was becoming the dominant strain globally.It has mutations in the gene encoding the SARS-CoV-2 spike protein causing the substitutions T478K, P681R and L452R, which are known to affect transmissibility of the virus as well as whether it can be neutralised by antibodies for previously circulating variants of the COVID-19 virus. It is thought to be one of the most transmissible respiratory viruses known. In August 2021, Public Health England (PHE) reported secondary attack rate in household contacts of non-travel or unknown cases for Delta to be 10.8% vis-à-vis 10.2% for the Alpha variant; the case fatality rate for those 386,835 people with Delta is 0.3%, where 46% of the cases and 6% of the deaths are unvaccinated and below 50 years old. Immunity from previous recovery or COVID-19 vaccines are effective in preventing severe disease or hospitalisation from infection with the variant.On 7 May 2021, PHE changed their classification of lineage B.1.617.2 from a variant under investigation (VUI) to a variant of concern (VOC) based on an assessment of transmissibility being at least equivalent to B.1.1.7 (Alpha variant); the UK\\'s SAGE using May data estimated a \"realistic\" possibility of being 50% more transmissible. On 11 May 2021, the WHO also classified this lineage VOC, and said that it showed evidence of higher transmissibility and reduced neutralisation. On 15 June 2021, the Centers for Disease Control and Prevention (CDC) declared Delta a variant of concern.The variant is thought to be partly responsible for India\\'s deadly second wave of the pandemic beginning in February 2021. It later contributed to a third wave in Fiji, the United Kingdom and South Africa, and the WHO warned in July 2021 that it could have a similar effect elsewhere in Europe and Africa. By late July, it had also driven an increase in daily infections in parts of Asia, the United States, Australia, and New Zealand. Yet another COVID variant, Omicron, followed.\\n\\n\\n== Classification ==\\nThe Delta variant has mutations in the gene encoding the SARS-CoV-2 spike protein causing the substitutions D614G, T478K, P681R and L452R. It is identified as the 21A, 21I, and 21J clades under the Nextstrain phylogenetic classification system.\\n\\n\\n=== Names ===\\nThe virus has also been referred to by the term \"Indian Variant\" as it was originally detected in India. However, the Delta variant is only one of three variants of the lineage B.1.617, all of which were first detected in India. At the end of May 2021, the WHO assigned the label Delta to lineage B.1.617.2 after introducing a new policy of using Greek letters for variants of concern and variants of interest.\\n\\n\\n=== Other sublineages of B.1.617 ===\\nThere are three sublineages of lineage B.1.617 categorised so far.\\nB.1.617.1 (Kappa variant) was designated a Variant Under Investigation in April 2021 by Public Health England. Later in April 2021, two other variants B.1.617.2 and B.1.617.3 were designated as Variants Under Investigation. While B.1.617.3 shares the L452R and E484Q mutations found in B.1.617.1, B.1.617.2 lacks the E484Q mutation. B.1.617.2 has the T478K mutation, not found in B.1.617.1 and B.1.617.3. Simultaneously, the ECDC released a brief maintaining all three sublineages of B.1.617 as VOI, estimating that a \"greater understanding of the risks related to these B.1.617 lineages is needed before any modification of current measures can be considered\".\\n\\n\\n=== Mutations ===\\n\\nThe Delta/ B.1.617.2 genome has 13 mutations (15 or 17 according to some sources, depending on whether more common mutations are included) which produce alterations in the amino-acid sequences of the proteins it encodes.The list of spike protein mutations is: 19R, (G142D), Δ156-157, R158G, L452R, T478K, D614G, P681R, D950N  according to GVN, or T19R, G142D, del 156-157, R158G, L452R, T478K, D614G, P681R according to Genscript  Four of them, all of which are in the virus\\'s spike protein code, are of particular concern:\\n\\nD614G. The substitution at position 614, an aspartic acid-to-glycine substitution, is shared with other highly transmissible variants like Alpha, Beta and Gamma.\\nT478K. The exchange at position 478 is a threonine-to-lysine substitution.\\nL452R. The substitution at position 452, a leucine-to-arginine substitution, confers stronger affinity of the spike protein for the ACE2 receptor and decreased recognition capability of the immune system.\\nP681R. The substitution at position 681, a proline-to-arginine substitution, which, according to William A. Haseltine, may boost cell-level infectivity of the variant \"by facilitating cleavage of the S precursor protein to the active S1/S2 configuration\".The E484Q mutation is not present in the B.1.617.2 genome.\\n\\n\\n=== Lineages ===\\nAs of August 2021, Delta variants have been subdivided in the Pango lineage designation system into variants from AY.1 to AY.28. However, there is no information on whether such classification correlates with biological characteristic changes of the virus. It is said that, as of August 2021, AY.4 to AY.11 are predominant in the UK, AY.12 in Israel, AY.2, AY.3, AY.13, AY.14, AY.25 in the US, AY.20 in the US and Mexico, AY.15 in Canada, AY.16 in Kenya, AY.17 in Ireland and Northern Ireland, AY.19 in South Africa, AY.21 in Italy and Switzerland, AY.22 in Portugal, AY.24 in Indonesia, and AY.23 in Indonesia, Singapore, Japan, and South Korea.\\n\\n\\n==== \"Delta plus\" variant ====\\n\\nDelta with K417N originally corresponded to lineages AY.1 and AY.2, subsequently also lineage AY.3, and has been nicknamed \"Delta plus\" or \"Nepal variant\". It has the K417N mutation, which is also present in the Beta variant. The exchange at position 417 is a lysine-to-asparagine substitution.As of mid-October 2021, the AY.3 variant accounted for a cumulative prevalence of approximately 5%  in the United States, and 2% worldwide. In mid-October the AY.4.2 Delta sublineage was expanding in England, and being monitored and assessed. It contains mutations A222V and Y145H in its spike protein, not considered of particular concern. It has been suggested that AY.4.2 might be 10-15% more transmissible than the original Delta variant. Mid-October 2021, AY.4.2 accounted for an estimated 10% of cases, and has led to an additional growth rate rising to about 1% (10% of 10%) per generational time of five days or so. This additional growth rate would grow with increasing prevalence. Without AY.4.2 and no other changes, the number of cases in the UK would have been about 10% lower. AY.4.2 grows about 15% faster per week. In the UK it was reclassified as a \"variant under investigation\" (but not \"of concern\") in late October 2021. In Denmark, after a drop in AY.4.2 cases, a new fast surge was detected and monitored, but was not yet considered a cause of concern.\\n\\n\\n== Symptoms ==\\n\\nThe most common symptoms may have changed from the most common symptoms previously associated with standard COVID-19. Infected people may mistake the symptoms for a bad cold and not realize they need to isolate. Common symptoms reported have been headaches, sore throat, a runny nose or a fever. In the United Kingdom, when the Delta variant accounted for 91 percent of new cases, one study found that the most reported symptoms were headache, sore throat, and runny nose.\\n\\n\\n== Prevention ==\\n\\nWHO has not issued preventative measures against Delta specifically; non-pharmaceutical measures recommended to prevent wild type COVID-19 should still be effective.  These would include washing hands, wearing a mask, maintaining distance from others, avoiding touching the mouth, nose or eyes, avoiding crowded indoor spaces with poor ventilation especially where people are talking, going to get tested if one develops symptoms and isolating if one becomes sick.  Public Health authorities should continue to find infected individuals using testing, trace their contacts, and isolate those who have tested positive or been exposed.  Event organizers should assess the potential risks of any mass gathering and develop a plan to mitigate these risks.  See also Non-pharmaceutical intervention (epidemiology).\\nICMR found that convalescent sera of the COVID-19 cases and recipients of Bharat Biotech\\'s BBV152 (Covaxin) were able to neutralise VUI B.1.617 although with a lower efficacy.Anurag Agrawal, the director of the Institute of Genomics and Integrative Biology (IGIB), said the study on the effectiveness of the available vaccines on lineage B.1.617 suggests that post-vaccination, the infections are milder.\\nAnthony Fauci, the Chief Medical Advisor to the President of the United States, has also expressed his confidence regarding the preliminary results. In an interview on 28 April, he said: This is something where we\\'re still gaining data daily. But the most recent data was looking at convalescent sera of COVID-19 cases and people who received the vaccine used in India, the Covaxin. It was found to neutralise the 617 variants.\\nAnother study by the Centre for Cellular and Molecular Biology (CCMB) in Hyderabad found Covishield (Oxford–AstraZeneca) vaccinated sera offers protection against lineage B.1.617.A study conducted by the Public Health England, has found that compared to those who were unvaccinated those who were vaccinated with either the Pfizer-BioNTech or AstraZeneca-Oxford had 33% less instances of symptomatic disease caused by the variant after the first dose. Among those who were two weeks after the receiving their second dose of the Pfizer-BioNTech vaccine 88% less subjects had symptomatic disease from the Delta variant versus those that were unvaccinated. Among those who were two weeks after the receiving their second dose of the AstraZeneca-Oxford vaccine 60% less subjects had symptomatic disease from the Delta variant versus those that were unvaccinated.A study by a group of researchers from the Francis Crick Institute, published in The Lancet, shows that humans fully vaccinated with the Pfizer-BioNTech vaccine are likely to have more than five times lower levels of neutralizing antibodies against the Delta variant compared to the original COVID-19 strain.In June 2021, Public Health England announced it had conducted a study which found that after two shots, the Pfizer-BioNTech vaccine and the AstraZeneca vaccine are respectively 96% and 92% effective at preventing hospitalisation from the Delta variant.On July 3, researchers from the universities of Toronto and Ottawa in Ontario, Canada, released a preprint study suggesting that the Moderna vaccine may be effective against death or hospitalization from the Delta variant.In a study of the University of Sri Jayewardenepura in July 2021 found the Sinopharm BIBP vaccine vaccine caused seroconversion in 95% of individuals studied that had received both doses of the vaccine. The rate was higher in 20-39 age group (98.9%) but slightly lower in the over 60 age group (93.3%). Neutralising antibodies were present among 81.25% of the vaccinated individuals studied.On 29 June 2021, the director of the Gamaleya Institute, Denis Logunov, said that Sputnik V is about 90% effective against the Delta variant.On July 21, researchers from Public Health England published a study finding that the Pfizer vaccine was 93.7% effective against symptomatic disease from Delta after 2 doses, while the Astrazeneca vaccine was 67% effective.On August 2, several experts expressed concern that achieving herd immunity may not currently be possible because the Delta variant is transmitted among those immunized with current vaccines.On August 10, a study showed that the full vaccination coverage rate is correlated inversely to the SARS-CoV-2 delta variant mutation frequency in 16 countries (R-squared=0.878). Data strongly indicates that full vaccination against COVID-19 may slow down virus evolution.\\n\\n\\n== Treatment ==\\n\\nIn vitro experiments suggest that bamlanivimab may not be effective against Delta on its own.  At high enough concentrations, casirivimab, etesevimab and imdevimab appear to still be effective.  A preprint study suggests that sotrovimab may also be effective against Delta.  Doctors in Singapore have been using supplemental oxygen, remdesivir and corticosteroids on more Delta patients than they did on previous variants.\\n\\n\\n== Epidemiology ==\\n\\n\\n=== Transmissibility ===\\nUK scientists have said that the Delta variant is between 40% and 60% more transmissible than the previously dominant Alpha variant, which was first identified in the UK (as the Kent variant). Given that Alpha is already 150% as transmissible as the original SARS-CoV-2 strain that emerged in late 2019 in Wuhan, and if Delta is 150% as transmissible as Alpha, then Delta may be 225% as transmissible as the original strain. BBC reported that \\n  \\n    \\n      \\n        \\n          R\\n          \\n            0\\n          \\n        \\n      \\n    \\n    {\\\\displaystyle R_{0}}\\n   – basic reproduction number, or the expected number of cases directly generated by one case in a population where all individuals are susceptible to infection – for the first detected SARS-CoV-2 virus is 2.4-2.6, whereas Alpha\\'s reproduction number is 4-5 and Delta\\'s is 5–9. These basic reproduction numbers can be compared to MERS (0.29-0.80), seasonal influenza (1.2-1.4), Ebola (1.4-1.8), common cold (2-3), SARS (2–4), smallpox (3.5-6), and chickenpox (10-12). Due to Delta\\'s high transmissibility even those that are vaccinated are vulnerable, albeit to a lesser extent.A study published online (not peer-reviewed) by Guangdong Provincial Center for Disease Control and Prevention may partly explain the increased transmissibility: people with infection caused by Delta had 1,000 times more copies of the virus in the respiratory tracts than those with infection caused by variants first identified in the beginning of the pandemic; and it took on average 4 days for people infected with Delta for the virus to be detectable compared to 6 days with initially identified variants.Surveillance data from the U.S., Germany and the Netherlands indicates the Delta variant is growing by about a factor of 4 every two weeks with respect to the Alpha variant.In India, the United Kingdom, Portugal, Russia, Mexico, Australia, Indonesia, South Africa, Germany, Luxembourg, the United States, the Netherlands, Denmark, France and probably many other countries, the Delta variant had become the dominant strain by July 2021. Depending on country, there is typically a lag from a few days to several weeks between cases and variant reporting.  As of July 20, this variant had spread to 124 countries, and WHO had indicated that it was becoming the dominant strain, if not one already.In the Netherlands, the virus was still able to propagate significantly in the population with over 93.4% of blood donors being tested positive for SARS-CoV-2 antibodies after week 28, 2021. Many people there are not fully vaccinated, so those antibodies would have been developed from exposure to the wild virus or from a vaccine. Similar high seroimmunity levels occur in the United Kingdom in blood donors and general surveillance.A preprint found that the viral load in the first positive test of infections with the variant was on average ∼1000 times higher than with compared infections during 2020. Preliminary data from a study with 100,000 volunteers in the UK from May to July 2021, when Delta was spreading rapidly, indicates that vaccinated people who test positive for COVID-19, including asymptomatic cases, have a lower viral load in average. Data from the US, UK, and Singapore indicate that vaccinated people infected by Delta may have viral loads as high as unvaccinated infected people, but might remain infectious for a shorter period.\\n\\n\\n=== Infection age groups ===\\nSurveillance data from the Indian government\\'s Integrated Disease Surveillance Programme (IDSP) shows that around 32% of patients, both hospitalised and outside hospitals, were aged below 30 in the second wave compared to 31% during the first wave, among people aged 30–40 the infection rate stayed at 21%. Hospitalisation in the 20-39 bracket increased to 25.5% from 23.7% while the 0-19 range increased to 5.8% from 4.2%. The data also showed a higher proportion of asymptomatic patients were admitted during the second wave, with more complaints of breathlessness.\\n\\n\\n=== Virulence ===\\n\\nA few early studies suggest the Delta variant causes more severe illness than other strains. On 7 June 2021, researchers at the National Centre for Infectious Diseases in Singapore posted a paper suggesting that patients testing positive for Delta are more likely to develop pneumonia and/or require oxygen than patients with wild type or Alpha. On June 11, Public Health England released a report finding that there was \"significantly increased risk of hospitalization\" from Delta as compared with Alpha; the risk was approximately twice as high for those infected with the Delta variant. On June 14, researchers from Public Health Scotland found that the risk of hospitalization from Delta was roughly double that of from Alpha. On July 7, a preprint study from epidemiologists at the University of Toronto found that Delta had a 120% greater – or more than twice as large – risk of hospitalization, 287% greater risk of ICU admission and 137% greater risk of death compared to non-variant of concern strains of SARS-COV-2. However, on July 9, Public Health England reported that the Delta variant in England had a case fatality rate (CFR) of 0.2%, while the Alpha variant\\'s case fatality rate was 1.9%, although the report warns that \"case fatality rates are not comparable across variants as they have peaked at different points in the pandemic, and so vary in background hospital pressure, vaccination availability and rates and case profiles, treatment options, and impact of reporting delay, among other factors.\" James McCreadie, a spokesperson for Public Health England, clarified \"It is too early to assess the case fatality ratio compared to other variants.\"A Canadian study released on 5 October 2021 revealed that the Delta variant caused a 108 percent rise in hospitalization, 235 percent increase in ICU admission, and a 133 percent surge in death compared to other variants. is more serious and resulted in an increased risk of death compared to previous variants, odds that are significantly decreased with immunization.\\n\\n\\n== Statistics ==\\nThe chance of detecting  a Delta case varies significantly, especially depending on a country\\'s sequencing rate (less than 0.05% of all COVID-19 cases have been sequenced in the lowest-sequencing countries to around 50 percent in the highest).By 22 June 2021, more than 4,500 sequences of the variant had been detected in about 78 countries. Reported numbers of sequences in countries with detections are:\\n\\n\\n== History ==\\nThe first cases of the variant outside India were detected in late February 2021, including the United Kingdom on 22 February, the United States on 23 February and Singapore on 26 February.British scientists at Public Health England redesignated the B.1.617.2 variant on 7 May 2021 as \"variant of concern\" (VOC-21APR-02), after they flagged evidence in May 2021 that it spreads more quickly than the original version of the virus. Another reason was that they identified 48 clusters of B.1.617.2, some of which revealed a degree of community transmission. With cases from the Delta variant having risen quickly, British scientists considered the Delta variant having overtaken the Alpha variant as the dominant variant of SARS-CoV-2 in the UK in early June 2021. Researchers at Public Health England later found that over 90% of new cases in the UK in the early part of June 2021 were the Delta variant; they also cited evidence that the Delta variant was associated with an approximately 60% increased risk of household transmission compared to the Alpha variant.Canada\\'s first confirmed case of the variant was identified in Quebec on 21 April 2021, and later the same day 39 cases of the variant were identified in British Columbia. Alberta reported a single case of the variant on 22 April 2021. Nova Scotia reported two Delta variant cases in June 2021.Fiji also confirmed its first case of the variant on 19 April 2021 in Lautoka, and has since then climbed up to 47,000 cases and counting. The variant has been identified as a super-spreader and has led to the lockdowns of five cities (Lautoka, Nadi, Suva, Lami and Nausori), an area which accounts for almost two-thirds of the country\\'s population.\\nOn 29 April 2021, health officials from Finland\\'s Ministry of Social Affairs and Health (STM) and the Finnish Institute for Health and Welfare (THL) reported that the variant had been detected in three samples dating back to March 2021.The Philippines confirmed its first two cases of the variant on 11 May 2021, despite the imposed travel ban of the country from the nations in the Indian subcontinent (except for Bhutan and Maldives). Both patients have no travel history from India for the past 14 days, but instead from Oman and UAE.North Macedonia confirmed its first case of the variant on 7 June 2021 after a person who was recovering from the virus in Iraq was transported to North Macedonia. In a laboratory test, the variant was detected in the person. On 22 June 2021, the country reported its second case of the Delta variant in a colleague of the first case who had also been in Iraq and who subsequently developed symptoms.The detection of B.1.617 was hampered in some countries by a lack of specialised kits for the variant and laboratories that can perform the genetic test. For example, as of 18 May, Pakistan had not reported any cases, but authorities noted that 15% of COVID-19 samples in the country were of an \"unknown variant\"; they could not say if it was B.1.617 because they were unable to test for it. Other countries had reported travellers arriving from Pakistan that were infected with B.1.617.In June 2021, scientist Vinod Scaria of India\\'s Institute of Genomics and Integrative Biology highlighted the existence of the B.1.617.2.1 variant, also known as AY.1 or Delta plus, which has an additional K417N mutation compared to the Delta variant. B.1.617.2.1 was detected in Europe in March 2021, and has since been detected in Asia and America.On 9 July 2021, Public Health England issued Technical Briefing 18 on SARS-CoV-2 variants, documenting 112 deaths among 45,136 UK cases of SARS-CoV-2 Delta variant with 28 days follow-up with a fatality rate of 0.2%. Briefing 16 notes that \"[M]ortality is a lagged indicator, which means that the number of cases who have completed 28 days of follow up is very low – therefore, it is too early to provide a formal assessment of the case fatality of Delta, stratified by age, compared to other variants.\"  Briefing 18 warns that \"Case fatality is not comparable across variants as they have peaked at different points in the pandemic, and so vary in background hospital pressure, vaccination availability and rates and case profiles, treatment options, and impact of reporting delay, among other factors.\"  The most concerning issue is the logistic growth rate of 0.93/week relative to Alpha. This means that per week, the number of Delta samples/cases is growing by a factor of exp (0.93)=2.5 with respect to the Alpha variant. This results, under the same infection prevention measures, in a much greater case load over time until a large fraction of people have been infected by it.\\n\\n\\n=== Government responses ===\\nAfter the rise in cases from the second wave, at least 20 countries imposed travel bans and restrictions on passengers from India in April and May. UK prime minister Boris Johnson cancelled his visit to India twice, while Japanese Prime Minister Yoshihide Suga postponed his April trip.In May 2021, residents of two tower blocks in Velbert, Germany, were quarantined after a woman in the building tested positive for the Delta variant.In May, Delhi Chief Minister Arvind Kejriwal said that a new coronavirus variant from Singapore was extremely dangerous for children and could result in a third wave in India.\\nFrom 16 May to 13 June 2021, as well as 22 July 2021 to 10 August 2021; Singapore entered lockdowns, known as \"Phase 2 Heightened Alert\", similar to 2020.\\nOn 14 June, the British prime minister Boris Johnson announced that the proposed end of all restrictions on 21 June in the United Kingdom was delayed for up to four weeks and vaccination roll-out was accelerated following concerns over the Delta variant, which accounted for the vast majority (90%) of new infections. UK scientists have said that the Delta variant is between 40% and 60% more transmissible than the previously dominant Alpha variant, which was first identified in the UK (as the Kent variant).On 23 June, the province of Ontario in Canada accelerated 2nd dose vaccine appointments for people living in Delta hot spots such as Toronto, Peel and Hamilton.On 25 June, Israel restored their mask mandate citing the threat of Delta.On 28 June, Sydney and Darwin went back into lockdown because of Delta outbreaks. South Africa banned indoor and outdoor gatherings apart from funerals, imposed a curfew, and banned the sale of alcohol.On 3 July, the islands of Bali and Java in Indonesia went into emergency lockdown.On 8 July, Japanese Prime Minister Yoshihide Suga announced that Tokyo would once again enter a state of emergency, and that most spectators would be barred from attending the Olympics set to start there on 23 July.On 9 July, Seoul, South Korea ramped up restrictions urging people to wear masks outdoors, and limiting the size of gatherings.On 12 July, French President Emmanuel Macron announced that all health care workers will need to be vaccinated by 15 September and that France will start using health passports to enter bars, cafés, restaurants and shopping centres from August.Los Angeles announced it will require masks indoors starting 17 July 2021.The United Kingdom lifted most COVID-19 restrictions on 19 July, despite a surge in cases as the Delta variant became dominant. The government cited the protection and wide coverage of the COVID-19 vaccination programme, although health experts expressed concern at the move.On 23 July, Vietnam extended its lockdown of Ho Chi Minh City to 1 August, and announced lockdown restrictions would be put in place in Hanoi, affecting a third of the country\\'s population. The Delta variant had brought upon the country\\'s largest outbreak to date, after mostly successful containment measures throughout 2020.On 17 August, New Zealand went into an alert level 4 lockdown, following a positive case being reported in Auckland. More cases soon followed in the Coromandel Peninsula. This was the first reported community transmission case in the country in 170 days (since February 2021).\\n\\n\\n== See also ==\\n\\nVariants of SARS-CoV-2: Alpha, Beta, Gamma, Epsilon, Zeta, Eta, Theta, Iota, Kappa, Lambda, Mu, Omicron, Lineage B.1.617\\n\\n\\n== Notes ==\\n\\n\\n== References ==\\n\\n\\n== External links ==\\nPublic Health England: Variants of concern or under investigation, B.1.617',\n",
              " 'There are several ongoing efforts by scientists, governments, international organisations, and others to determine the origin of SARS-CoV-2, the virus responsible for the COVID-19 pandemic. Most scientists say that as with other pandemics in human history, the virus is likely of zoonotic origin in a natural setting, and ultimately originated from a bat-borne virus. Several other explanations, including many conspiracy theories, have been proposed about the origins of the virus.SARS-CoV-2 has close genetic similarity to multiple previously identified bat coronaviruses, suggesting it crossed over into humans from bats. Research is ongoing as to whether SARS-CoV-2 came directly from bats or indirectly through any intermediate hosts. Initial genome sequences of the virus showed little genetic diversity, although subsequently a number of stable variants emerged (some spreading more vigorously), indicating that the spillover event introducing SARS-CoV-2 to humans is likely to have occurred in late 2019. Health authorities and scientists internationally state that as with the 2002–2004 outbreak of SARS-1, efforts to trace the specific geographic and taxonomic origins of SARS-CoV-2 could take years, and the results could be inconclusive.In January 2021, the World Health Assembly (decision-making body of the World Health Organization, WHO) commissioned a study on the origins of the virus, to be conducted jointly between WHO experts and Chinese scientists. In March 2021, the findings of this study were published online in a report to the WHO Commissioner-General. Echoing the assessment of most virologists, the report determined that the virus most likely had a zoonotic origin in bats, possibly transmitted through an intermediate host. It also stated that a laboratory origin for the virus was \"extremely unlikely.\"Scientists found the conclusions of the WHO report to be helpful but noted that more work was needed. In the United States, the European Union, and other countries, some criticised the lack of transparency and data access in the report\\'s formulation. The WHO issued its 30 March report alongside a statement of the WHO director-general Tedros Adhanom Ghebreyesus saying that the matter \"requires further investigation\". The U.S. government and 13 other countries and the EU issued statements on the same day, echoing Tedros\\'s critique of the report for lacking transparency and data access in its formulation. In a later press conference, the WHO director-general said it was \"premature\" for the WHO\\'s report to rule out a potential link between a laboratory leak and called on China to provide \\'raw data\\' and lab audits in a second phase of investigations. On 12 October 2021, WHO announced a new team to study the origins of the COVID-19 pandemic.Earlier, on 22 July 2021, the Chinese government had held a press conference in which Zeng Yixin, Vice Health Minister of the National Health Commission (NHC), said that China would not participate in a second phase of the WHO\\'s investigation, denouncing it as \"shocking\" and \"arrogant\".\\n\\n\\n== Scientific background ==\\n\\nCOVID-19 is caused by infection with a virus called severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). SARS-CoV-2 appears to have originated in bats and to have spread to humans by zoonotic transfer. Its exact evolutionary history, the identity and provenance of its most recent ancestors, and the place, time, and mechanism of transmission of the first human infection, remain unknown. The biology and regional distribution of other coronaviruses in southeast Asia, including SARS-CoV, help scientists understand more about the origins of SARS-CoV-2.Taxonomically, SARS-CoV-2 is a virus of the species severe acute respiratory syndrome–related coronavirus (SARSr-CoV). It is believed to have zoonotic origins and has close genetic similarity to bat coronaviruses, suggesting it emerged from a bat-borne virus. Research is ongoing as to whether SARS-CoV-2 came directly from bats or indirectly through any intermediate hosts. The virus shows little genetic diversity, indicating that the spillover event introducing SARS-CoV-2 to humans is likely to have occurred in late 2019. Ultimately, the specific evolutionary history of SARS-CoV-2 in relation to other coronaviruses will be critical to understanding how, where and when the virus spilled over into a human population.\\n\\n\\n=== Reservoir and origin ===\\n\\nThe first known infections from SARS‑CoV‑2 were discovered in Wuhan, China. The original source of viral transmission to humans remains unclear, as does whether the virus became pathogenic before or after the spillover event. Because many of the early infectees were workers at the Huanan Seafood Market, it has been suggested that the virus might have originated from the market. However, other research indicates that visitors may have introduced the virus to the market, which then facilitated rapid expansion of the infections. A March 2021 WHO-convened report stated that human spillover via an intermediate animal host was the most likely explanation, with direct spillover from bats next most likely. Introduction through the food supply chain and the Huanan Seafood Market was considered another possible, but less likely, explanation. An analysis in November 2021, however, said that the earliest-known case had been misidentified and that the preponderance of early cases linked to the Huanan Market argued for it being the source.For a virus recently acquired through a cross-species transmission, rapid evolution is expected. The mutation rate estimated from early cases of SARS-CoV-2 was of 6.54×10−4 per site per year. Coronaviruses in general have high genetic plasticity, but SARS-CoV-2\\'s viral evolution is slowed by the RNA proofreading capability of its replication machinery. For comparison, the viral mutation rate in vivo of SARS-CoV-2 has been found to be lower than that of influenza.Research into the natural reservoir of the virus that caused the 2002–2004 SARS outbreak has resulted in the discovery of many SARS-like bat coronaviruses, most originating in horseshoe bats. The closest match by far, published on Nature (journal) in February 2022, were viruses BANAL-52 (96.8% resemblance to SARS‑CoV‑2), BANAL-103 and BANAL-236, collected in three different species of bats in Feuang, Laos. An earlier source published in February 2020 identified the virus RaTG13, collected in bats in Mojiang, Yunnan, China to be the closest to SARS‑CoV‑2, with 96.1% resemblance. None of the above are its direct ancestor.\\n\\nBats are considered the most likely natural reservoir of SARS‑CoV‑2. Differences between the bat coronavirus and SARS‑CoV‑2 suggest that humans may have been infected via an intermediate host; although the source of introduction into humans remains unknown.Although the role of pangolins as an intermediate host was initially posited (a study published in July 2020 suggested that pangolins are an intermediate host of SARS‑CoV‑2-like coronaviruses), subsequent studies have not substantiated their contribution to the spillover. Evidence against this hypothesis includes the fact that pangolin virus samples are too distant to SARS-CoV-2: isolates obtained from pangolins seized in Guangdong were only 92% identical in sequence to the SARS‑CoV‑2 genome (matches above 90 percent may sound high, but in genomic terms it is a wide evolutionary gap). In addition, despite similarities in a few critical amino acids, pangolin virus samples exhibit poor binding to the human ACE2 receptor.Available scientific evidence suggests that SARS-CoV-2 has a natural zoonotic origin. Yet its origin, which remains unknown, has become debated within the context of global geopolitical tensions. Early in the pandemic, conspiracy theories spread on social media claiming that the virus was a biological weapon developed by China, amplified by echo chambers in the American far-right. Other conspiracy theories promoted misinformation that the virus is not communicable or was created to profit from new vaccines.Some politicians and scientists have speculated that the virus may have accidentally leaked from the Wuhan Institute of Virology. This has led to calls in the media for further investigations into the matter. Many virologists who have studied coronaviruses consider the possibility very remote. The WHO-convened Global Study of the Origins of SARS-CoV-2 from March 2021 stated that such an explanation is extremely unlikely. In an interview on 12 August 2021, Peter Ben Embarek, the chief investigator of the WHO team, told a Danish TV documentary that the WHO team felt pressured from Chinese scientists to put \"extremely unlikely\" as their assessment.A 2021 study found possible links between climate change and transmission of COVID-19 by bats. The authors found that climate-driven changes in the distribution and richness of bat species harboring coronaviruses may have occurred leading to a concentration of such species in eastern Asian hotspots (southern China, Myanmar and Laos). Tell conditions favors transmission of viruses. In this area live pangolins that are sold among other in the Wuhan wet market and are suspected of being the intermediate host in the transmission of SARS-COV-2 who cause the pandemic. The study was cited at 22 other studies in the National Library of Medicine of the USA.  Before the word health day 2022, the World Health Organization mentioned on its site that: \"There is growing evidence of a direct connection between environmental change and the emergence or transmission of COVID-19\".\\n\\n\\n=== Phylogenetics and taxonomy ===\\nSARS‑CoV‑2 belongs to the broad family of viruses known as coronaviruses. It is a positive-sense single-stranded RNA (+ssRNA) virus, with a single linear RNA segment. Coronaviruses infect humans, other mammals, including livestock and companion animals, and avian species. Human coronaviruses are capable of causing illnesses ranging from the common cold to more severe diseases such as Middle East respiratory syndrome (MERS, fatality rate ~34%). SARS-CoV-2 is the seventh known coronavirus to infect people, after 229E, NL63, OC43, HKU1, MERS-CoV, and the original SARS-CoV.Like the SARS-related coronavirus implicated in the 2003 SARS outbreak, SARS‑CoV‑2 is a member of the subgenus Sarbecovirus (beta-CoV lineage B). Coronaviruses undergo frequent recombination. The mechanism of recombination in unsegmented RNA viruses such as SARS-CoV-2 is generally by copy-choice replication, in which gene material switches from one RNA template molecule to another during replication. SARS-CoV-2 RNA sequence is approximately 30,000 bases in length, relatively long for a coronavirus (which in turn carry the largest genomes among all RNA families) Its genome consists nearly entirely of protein-coding sequences, a trait shared with other coronaviruses.\\n\\nA distinguishing feature of SARS‑CoV‑2 is its incorporation of a polybasic site cleaved by furin, which appears to be an important element enhancing its virulence. In SARS-CoV-2 the recognition site is formed by the incorporated 12 codon nucleotide sequence CCT CGG CGG GCA which corresponds to the amino acid sequence P RR A. This sequence is upstream of an arginine and serine which forms the S1/S2 cleavage site (P RR A R↓S) of the spike protein. Although such sites are a common naturally-occurring feature of other viruses within the Subfamily Orthocoronavirinae, it appears in few other viruses from the Beta-CoV genus, and it is unique among members of its subgenus for such a site. The furin cleavage site PRRAR↓ is identical to that of the feline coronavirus, an alphacoronavirus 1 strain.Viral genetic sequence data can provide critical information about whether viruses separated by time and space are likely to be epidemiologically linked. With a sufficient number of sequenced genomes, it is possible to reconstruct a phylogenetic tree of the mutation history of a family of viruses. By 12 January 2020, five genomes of SARS‑CoV‑2 had been isolated from Wuhan and reported by the Chinese Center for Disease Control and Prevention (CCDC) and other institutions; the number of genomes increased to 42 by 30 January 2020. A phylogenetic analysis of those samples showed they were \"highly related with at most seven mutations relative to a common ancestor\", implying that the first human infection occurred in November or December 2019. Examination of the topology of the phylogenetic tree at the start of the pandemic also found high similarities between human isolates.  As of 21 August 2021, 3,422 SARS‑CoV‑2 genomes, belonging to 19 strains, sampled on all continents except Antarctica were publicly available.On 11 February 2020, the International Committee on Taxonomy of Viruses announced that according to existing rules that compute hierarchical relationships among coronaviruses based on five conserved sequences of nucleic acids, the differences between what was then called 2019-nCoV and the virus from the 2003 SARS outbreak were insufficient to make them separate viral species. Therefore, they identified 2019-nCoV as a virus of Severe acute respiratory syndrome–related coronavirus.\\n\\n\\n== Origin scenarios ==\\nThe origin of SARS-CoV-2 has been the subject of debate. There are multiple proposed explanations for how SARS-CoV-2 was introduced into, and evolved adaptations suited to, the human population. There is significant evidence and agreement that the most likely original viral reservoir for SARS-CoV-2 is horseshoe bats, with the closest known viral relative being RaTG13. The evolutionary distance between SARS-CoV-2 and RaTG13 is estimated to be about 50 years (between 38 and 72 years). The earliest human cases of SARS-CoV-2 were identified in Wuhan, but the index case remains unknown. RaTG13 was sampled from bats in Yunnan, located roughly 1,300 km (810 mi) away from Wuhan, and there are relatively few bat coronaviruses from Hubei province. Each origin hypothesis attempts to explain this gap in virus evolution and location a different way. These scenarios continue to be investigated to identify the definitive origin of the virus.\\n\\n\\n=== Direct zoonotic transmission in a natural setting ===\\nThe most direct pathway of introduction is direct zoonotic transmission (also known as spillover) from the reservoir species to humans. Scientists consider this to be a highly likely origin of the SARS-CoV-2 virus in humans. Human contact with bats has increased as human population centers encroach on bat habitats, leading to increased opportunities for spillover. Bats are a significant reservoir species for a diverse range of coronaviruses, and humans have been found with antibodies for them suggesting that this form of direct infection by bats is common. However, in this scenario, the direct ancestor of SARS-CoV-2 remains undiscovered in bats.\\n\\n\\n=== Intermediate host ===\\nIn addition to direct spillover, another pathway, considered highly likely by scientists, is that of transmission through an intermediate host. Specifically, this implies that a cross species transmission occurred prior to the human outbreak and that it had pathogenic results on the animal. This pathway has the potential to allow for greater adaptation to human transmission via animals with more similar protein shapes to humans, though this is not required for the scenario to occur. The evolutionary separation from bat viruses is explained in this case by the virus\\' presence in an unknown species with less viral surveillance than bats. The virus\\' ability to easily infect and adapt to additional species (including mink) provides evidence that such a route of transmission is possible.\\n\\n\\n=== Cold/food chain ===\\nAnother proposed introduction to humans is through fresh or frozen food products, referred to as the cold/food chain. Scientists do not consider this to be a likely origin of SARS-CoV-2 in humans. This scenario\\'s source animal could be either a direct or intermediary species as described above. Many investigations centered around the Huanan Seafood Wholesale Market in Wuhan, which had an early cluster of cases. While there have been food-borne outbreaks of human viruses in the past, and evidence of re-introduction of SARS-CoV-2 into China through imported frozen foods, investigations found no conclusive evidence of viral contamination in products at the Huanan Market.\\n\\n\\n=== Laboratory incident ===\\n\\nA final scenario, assessed to be unlikely by most experts, is known as the lab leak hypothesis. Considered \"extremely unlikely\" by a WHO-convened study, the theory considers the possibility that the virus was introduced to humans due to an intentional or unintentional laboratory escape. This scenario would have involved laboratory staff becoming infected from contact with live bats (wild or captive), from contact with biological samples, or from contact with viruses being grown in vitro. The Wuhan Institute of Virology (WIV) has performed research into bat coronaviruses since 2005, and identified the RaTG13 virus in 2013, which is the closest known relative of SARS-CoV-2. WIV research interests have included investigations into the source of the 2002–2004 SARS outbreak and 2012 MERS outbreak, in collaboration with US researchers. The proximity of the laboratory to the Huanan seafood market has led some to speculate there may be a link between the two, and politicians, media personalities, and some scientists have called for further investigations into the matter. Experts have ruled out intentional manipulation of the virus (i.e. biological engineering) as a plausible origin, due to a lack of supporting evidence, and growing evidence in favor of a natural origin.New York-based, National Institutes of Health–funded EcoHealth Alliance has been the subject of controversy and increased scrutiny due to its ties to the Wuhan Institute of Virology. Under political pressure, the National Institutes of Health (NIH) withdrew funding to EcoHealth Alliance in July 2020.Subsequent investigations (such as the WHO-convened study) considered the possibility of a collected natural virus inadvertently infecting laboratory staff. On 15 July 2021 WHO director-general Tedros Adhanom stated in a speech that there had been a \"premature push\" to discredit the idea of a lab leak. Later on in the same speech, he expanded, \"I was a lab technician, an immunologist, and worked in the lab. And lab accidents happen.\" The WHO has planned a second phase of investigation that will include \"audits of relevant laboratories and research institutions\". However, China has rejected this plan stating \"it is impossible for us to accept such an origin-tracing plan\".An even less likely lab-leak theory has been proposed by some members of the Chinese government who have claimed that the virus came from a U.S. military laboratory. Chinese demands to investigate the U.S. lab at Fort Detrick have been said to be intended to deflect attention from Wuhan.\\n\\n\\n== Investigations ==\\n\\n\\n=== Chinese government ===\\n\\nThe first investigation conducted in China was by the Wuhan Municipal Health Commission, responding to hospitals reporting cases of pneumonia of unknown etiology, resulting in the closure of the Huanan Seafood Wholesale Market on 1 January 2020 for sanitation and disinfection. The market was originally suspected of being the source of the virus; however, the Chinese government and the WHO determined later that it was not.In April 2020, China imposed restrictions on publishing academic research on the novel coronavirus. Investigations into the origin of the virus would receive extra scrutiny and must be approved by Central Government officials. The restrictions do not ban research or publication, including with non-Chinese researchers; Ian Lipkin, a US scientist, has been working with a team of Chinese researchers under the auspices of the Chinese Center for Disease Control and Prevention, a Chinese government agency, to investigate the origin of the virus. Lipkin has long-standing relationships with Chinese officials, including premier Li Keqiang, because of his contributions to rapid testing for SARS in 2003.\\n\\n\\n=== United States government ===\\n\\n\\n==== Trump administration ====\\nOn 6 February 2020, the director of the White House\\'s Office of Science and Technology Policy requested the National Academies of Sciences, Engineering, and Medicine to convene a meeting of \"experts, world class geneticists, coronavirus experts, and evolutionary biologists\", to \"assess what data, information and samples are needed to address the unknowns, in order to understand the evolutionary origins of COVID-19 and more effectively respond to both the outbreak and any resulting information\".In April 2020, it was reported that the US intelligence community was investigating whether the virus came from an accidental leak from a Chinese lab. The hypothesis was one of several possibilities being pursued by the investigators. US Secretary of Defense Mark Esper said the results of the investigation were \"inconclusive\". By the end of April 2020, the Office of the Director of National Intelligence said the US intelligence community believed the coronavirus was not man-made or genetically modified.US officials criticised the \"terms of reference\" allowing Chinese scientists to do the first phase of preliminary research. On 15 January 2021, US Secretary of State Mike Pompeo said that to assist the WHO investigative team\\'s work and ensure a transparent, thorough investigation of COVID-19\\'s origin, the US was sharing new information and urging the WHO to press the Chinese government to address three specific issues, including the illnesses of several researchers inside the WIV in autumn 2019 \"with symptoms consistent with both COVID-19 and common seasonal illnesses\", the WIV\\'s research on \"RaTG13\" and \"gain of function\", and the WIV\\'s links to the People\\'s Liberation Army. On 18 January, the US called on China to allow the WHO\\'s expert team to interview \"care givers, former patients and lab workers\" in the city of Wuhan, drawing a rebuke from the Chinese government. Australia also called for the WHO team to have access to \"relevant data, information and key locations\".A classified report from May 2020 by the Lawrence Livermore National Laboratory, a US government national laboratory, concluded that the hypothesis that the virus leaked from the WIV \"is plausible and deserves further investigation\", although the report also notes that the virus could have developed naturally, echoing the consensus of the American intelligence community, and provides no \"smoking gun\" towards either hypothesis.\\n\\n\\n==== Biden administration ====\\nOn 13 February 2021, the White House said it had \"deep concerns\" about both the way the WHO\\'s findings were communicated and the process used to reach them. Mirroring concerns raised by the Trump administration, National Security Adviser Jake Sullivan stated that it was essential that the WHO-convened report be independent and \"free from alteration by the Chinese government\". On 14 April 2021, the Director of National Intelligence Avril Haines, along with other Biden administration officials, said that they had not ruled out the possibility of a laboratory accident as the origin of the COVID-19 virus.On 26 May 2021, President Joe Biden directed the U.S. intelligence community to produce a report within 90 days on whether the COVID-19 virus originated from a human contact with an infected animal or from an accidental lab leak, stating his national security staff said there is insufficient evidence to determine either hypothesis to be more likely. On 26 August 2021, the Office of the Director of National Intelligence released an unclassified summary of their findings, with the main point being that the report remained inconclusive as to the origin of the virus, with intelligence agencies divided on the question. The report also concluded that the virus was most likely not genetically engineered, and that China had no foreknowledge of the virus prior to the outbreak. The report concluded that a final determination of the origin was unlikely without cooperation from the Chinese government, saying their prior lack of transparency \"reflect[ed] in part China\\'s government\\'s own uncertainty about where an investigation could lead, as well as its frustration that the international community is using the issue to exert political pressure on China.\" Chinese foreign ministry spokesman Wang Wenbin said that the US intelligence report was \"unscientific and has no credibility\".On 23 May 2021, The Wall Street Journal reported that a previously undisclosed US intelligence report stated that three researchers from the Wuhan Institute of Virology became ill enough in November 2019 to seek hospital care. The report did not specify what the illness was. Officials familiar with the intelligence differed as to the strength to which it corroborates the hypothesis that the virus responsible for COVID-19 was leaked from the WIV. The WSJ report notes that it is not unusual for people in China to go to the hospital with uncomplicated influenza or common cold symptoms.Yuan Zhiming, director of the WIV\\'s Wuhan National Biosafety Laboratory, responded in the Global Times, a Chinese state media outlet, that the \"claims are groundless\". Marion Koopmans, a member of the WHO study team, described the number of flu-like illnesses at the WIV in 2019 as \"completely normal\". Workers at the WIV must provide yearly serum samples. WIV virologist Shi Zhengli said in 2020 that, based on an evaluation of those serum samples, all staff tested negative for COVID-19 antibodies.The resurgence of the theory of a laboratory accident was fueled in part by the publication, in May 2021, of early emails between Anthony Fauci and scientists discussing the issue, before deliberate manipulation was ruled out as of March 2020.On 14 July 2021, the House Committee on Science, Space and Technology held the first congressional hearing on the origins of the virus. Bill Foster, an Illinois Democrat who chaired the hearing, said the Chinese government\\'s lack of transparency is not in itself evidence of a lab leak and cautioned that answers may not be known even after the administration produces its intelligence report. Expert witnesses Stanley Perlman and David Relman presented to the congressman different proposed explanations for the origins of the virus and how to conduct further investigations.On 16 July 2021, CNN reported that Biden administration officials considered the lab leak theory \"as credible\" as the natural origins theory.\\n\\n\\n=== World Health Organization ===\\nThe World Health Organization has declared that finding where SARS-CoV-2 came from is a priority and that it is \"essential for understanding how the pandemic started.\" In May 2020, the World Health Assembly, which governs the World Health Organization (WHO), passed a motion calling for a \"comprehensive, independent and impartial\" study into the COVID-19 pandemic. A record 137 countries, including China, co-sponsored the motion, giving overwhelming international endorsement to the study. In mid 2020, the World Health Organization (WHO) began negotiations with the government of China on conducting an official study into the origins of COVID-19.\\nIn November 2020, the WHO published a two-phase study plan. The purpose of the first phase was to better understand how the virus \"might have started circulating in Wuhan\", and a second phase involves longer-term studies based on the findings of the first phase. WHO director-general Tedros Adhanom said \"We need to know the origin of this virus because it can help us to prevent future outbreaks,\" adding, \"There is nothing to hide. We want to know the origin, and that\\'s it.\" He also urged countries not to politicise the origin tracing process, saying that would only create barriers to learning the truth.\\n\\n\\n==== Phase 1 ====\\nFor the first phase, the WHO formed a team of ten researchers with expertise in virology, public health and animals to conduct a thorough study. One of the team\\'s tasks was to retrospectively ascertain what wildlife was being sold in local wet markets in Wuhan. The WHO\\'s phase one team arrived and quarantined in Wuhan, Hubei, China in January 2021.Members of the team included Thea Fisher, John Watson, Marion Koopmans, Dominic Dwyer, Vladimir Dedkov, Hung Nguyen-Viet, Fabian Leendertz, Peter Daszak, Farag El Moubasher, and Ken Maeda. The team also included five WHO experts led by Peter Ben Embarek, two Food and Agriculture Organization representatives, and two representatives from the World Organisation for Animal Health.The inclusion of Peter Daszak in the team stirred controversy. Daszak is the head of EcoHealth Alliance, a nonprofit that studies spillover events, and has been a longtime collaborator of over 15 years with Shi Zhengli, Wuhan Institute of Virology\\'s director of the Center for Emerging Infectious Diseases. While Daszak is highly knowledgeable about Chinese laboratories and the emergence of diseases in the area, his close connection with the WIV was seen by some as a conflict of interest in the WHO\\'s study. When a BBC News journalist asked about his relationship with the WIV, Daszak said, \"We file our papers, it\\'s all there for everyone to see.\"The team was denied access to raw data, including the list of early patients, swabs, and blood samples. It was allowed only a few hours of supervised access to the Wuhan Institute of Virology.\\n\\n\\n===== Findings =====\\nIn February 2021, after conducting part of their study, the WHO stated that the likely origin of COVID-19 was a zoonotic event from a virus circulating in bats, likely through another animal carrier, and that the time of transmission to humans was likely towards the end of 2019.The Chinese and the international experts who jointly carried out the WHO-convened study consider it \"extremely unlikely\" that COVID-19 leaked from a lab. No evidence of a lab leak from the Wuhan Institute of Virology was found by the WHO team, with team leader Peter Ben Embarek stating that it was \"very unlikely\" due to the safety protocols in place. During a 60 Minutes interview with Lesley Stahl, Peter Daszak, another member of the WHO team, described the investigation process to be a series of questions and answers between the WHO team and the Wuhan lab staff. Stahl made the comment that the team was \"just taking their word for it\", to which Daszak replied, \"Well, what else can we do? There\\'s a limit to what you can do and we went right up to that limit. We asked them tough questions. They weren\\'t vetted in advance. And the answers they gave, we found to be believable—correct and convincing.\"The investigation also stated that transfer from animals to humans was unlikely to have occurred at the Huanan Seafood Market, since infections without a known epidemiological link were confirmed before the outbreak around the market. In an announcement that surprised some foreign experts, the joint investigation concluded that early transmission via the cold chain of frozen products was \"possible.\"In March 2021, the WHO published a written report with the results of the study. The joint team stated that there are four scenarios for introduction:\\ndirect zoonotic transmission to humans (spillover), assessed as \"possible to likely\"\\nintroduction through an intermediate host followed by a spillover, assessed as \"likely to very likely\"\\nintroduction through the (cold) food chain, assessed as \"possible\"\\nintroduction through a laboratory incident, assessed as \"extremely unlikely\"The report mentions that direct zoonotic transmission to humans has a precedent, as most current human coronaviruses originated in animals. Zoonotic transmission is also supported by the fact that RaTG13 binds to hACE2, although the fit is not optimal.The investigative team noted the requirement for further studies, noting that these would \"potentially increase knowledge and understanding globally.\":\\u200a9\\u200a\\n\\n\\n===== Reactions =====\\nWHO director-general Tedros Adhanom, who was not directly involved with the investigation, said he was ready to dispatch additional missions involving specialist experts and that further research was required. He said in a statement, \"Some explanations may be more probable than others, but for now all possibilities remain on the table.\" He also said, \"We have not yet found the source of the virus, and we must continue to follow the science and leave no stone unturned as we do.\" Tedros called on China to provide \"more timely and comprehensive data sharing\" as part of future investigations.News outlets noted that, though it was unrealistic to expect quick and huge results from the report, it \"offered few clear-cut conclusions regarding the start of the pandemic\", \"failed to audit the Chinese official position at some parts of the report\", and was \"biased according to critics\". Other scientists praised how the report details the pathways that can shed light on the origin, if explored later.After the publication of the report, politicians, talk show hosts, journalists, and some scientists advanced unsupported claims that SARS-CoV-2 may have come from the WIV. In the United States, calls to investigate a laboratory leak reached \"fever pitch\", fueling aggressive rhetoric resulting in antipathy towards people of Asian ancestry, and the bullying of scientists. The European Union, United States, and 13 other countries criticised the WHO-convened study, calling for transparency from China and access to the raw data and original samples. Chinese officials described these criticisms as an attempt to politicise the study. Scientists involved in the WHO report, including Liang Wannian, John Watson, and Peter Daszak, objected to the criticism, and said that the report was an example of the collaboration and dialogue required to successfully continue investigations into the matter.In a letter published in Science, a number of scientists, including Ralph Baric, argued that the accidental laboratory leak hypothesis had not been sufficiently investigated and remained possible, calling for greater clarity and additional data. Their letter was criticized by some virologists and public health experts, who said that a \"hostile\" and \"divisive\" focus on the WIV was unsupported by evidence, and would cause Chinese scientists and authorities to share less, rather than more data.\\n\\n\\n==== Phase 2 ====\\n\\nOn 27 May 2021, Danish epidemiologist Tina Fischer spoke on the This Week in Virology podcast, advocating for a second phase of the study to audit blood samples for COVID-19 antibodies in China. WHO-convened study team member Marion Koopmans, on that same broadcast, advocated for WHO member states to make a decision on the second phase of the study, though she also cautioned that an investigatory audit of the laboratory itself may be inconclusive. In early July 2021, WHO emergency chief Michael Ryan said the final details of phase 2 were being worked out in negotiations between WHO and its member states, as the WHO works \"by persuasion\" and cannot compel any member state (including China) to cooperate.In July 2021 China rejected WHO requests for greater transparency, cooperation, and access to data as part of Phase 2. On 16 July 2021, Foreign Ministry spokesperson Zhao Lijian declared that China\\'s position was that future investigations should be conducted elsewhere and should focus on cold chain transmission and the US military\\'s labs. On 22 July 2021, the Chinese government held a press conference in which Zeng Yixin, Vice Health Minister of the National Health Commission (NHC), said that China would not participate in a second phase of the WHO\\'s investigation, denouncing it as \"shocking\" and \"arrogant\". He elaborated \"In some aspects, the WHO\\'s plan for next phase of investigation of the coronavirus origin doesn\\'t respect common sense, and it\\'s against science. It\\'s impossible for us to accept such a plan.\"\\n\\n\\n=== The Lancet COVID-19 Commission task force ===\\nIn November 2020, an international task force led by Peter Daszak, president of EcoHealth Alliance, was formed as part of The Lancet COVID-19 Commission, backed by the medical journal The Lancet. Daszak stated that the task force was formed to \"conduct a thorough and rigorous investigation into the origins and early spread of SARS-CoV-2\". The task force has twelve members with backgrounds in One Health, outbreak investigation, virology, lab biosecurity and disease ecology.  The task force plans to analyse scientific findings and does not plan to visit China. In June 2021, The Lancet announced that Daszak had recused himself from the commission. On 25 September 2021, the task force work was folded after procedural concerns and a need to broaden its scope to examine transparency and government regulation of risky laboratory research.\\n\\n\\n=== Independent investigations ===\\nIn June 2021, the NIH announced that a set of sequence data had been removed from the Sequence Read Archive (SRA) in June 2020. The removal was performed according to standard practice at the request of the investigators who owned the rights to the sequences, with the investigators\\' rationale that the sequences would be submitted to another database. The investigators subsequently published a paper in an academic journal the same month they were removed from the NIH database which described the sequences in detail and discussed their evolutionary relationship to other sequences, but did not include the raw data. Virologist David Robertson said that it was difficult to conclude it was a cover-up rather than the more likely explanation: a mundane deletion of data without malfeasance. The missing genetic sequence data was restored in a correction published 29 July 2021 after it was stated to be a copy-edit error.\\n\\n\\n=== International calls for investigations ===\\nIn April 2020, Australian foreign minister Marise Payne and Australian prime minister Scott Morrison called for an independent international inquiry into the origins of the coronavirus pandemic. A few days later, German chancellor Angela Merkel also pressed China for transparency about the origin of the coronavirus, following similar concerns raised by the French president Emmanuel Macron. The UK also expressed support for an investigation, although both France and UK said the priority at the time was to first fight the virus. Some public health experts have also called for an independent examination of COVID-19\\'s origins, \"arguing WHO does not have the political clout to conduct such a forensic analysis\". In May 2021, Prime Minister Justin Trudeau told reporters Canada would \"support the call by the United States and others to better understand the origins of COVID-19.\" In June 2021, at the G7 summit in Cornwall, the attending leaders issued a joint statement calling for a new investigation, citing China\\'s refusal to cooperate with certain aspects of the original WHO-convened study. This resistance to international pressure was one of the key findings of a Wall Street Journal investigation into the pandemic origin.The divisive nature of the debate has led scientists to call for less political pressure on the topic. Public health analysts have remarked that the debate over the origins of SARS-CoV-2 is fueling unnecessary confrontation, resulting in bullying and harassment of scientists, and is deepening existing geopolitical tensions and hindering collaboration at a time where such mutual cooperation is required, both to deal with the current pandemic and in preparation for future such outbreaks. This comes in the face of scientists having predicted such events for decades: according to Katie Woolaston, researcher at the Queensland University of Technology, \"The environmental drivers of pandemics are not being widely discussed\". The debate comes at a moment of difficult global relations with Chinese authorities. Researchers have noted that the politicisation of the debate is making the process more difficult, and that words are often twisted to become \"fodder for conspiracy theories.\" A letter published in The Lancet in July 2021 remarked that the atmosphere of speculation surrounding the issue was of no help in making an objective assessment of the situation. In response to this letter, in a communication published in the same journal, a small group of scientists opposed the idea that scientists should promote unity and called for openness to alternative hypotheses. Despite the unlikelihood of the event, and although definitive answers are likely to take years of research, biosecurity experts have called for a review of global biosecurity policies, citing known gaps in international standards for biosafety. The situation has also reignited a debate over gain-of-function research, although the intense political rhetoric surrounding the issue has threatened to sideline serious inquiry over policy in this domain.\\n\\n\\n== See also ==\\nScientific Advisory Group for Origins of Novel Pathogens\\nWorld Health Organization\\'s response to the COVID-19 pandemic\\n\\n\\n== References ==']"
            ]
          },
          "execution_count": 62,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "result_docs"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "xrZ6Vutb_vuq"
      },
      "source": [
        "#### **Section 4.1: Evaluation by Harmonic Mean (F1-score)**<br> \n",
        "First the model is evaluated by calculating the harmonic mean for query terms.<br> \n",
        "\n",
        "Manual method relevance judgements are used and coded to calculate F1 scores on precision-recall plane as follows:"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "0XtTr1N5FbGd"
      },
      "outputs": [],
      "source": [
        "# Inputting relevance judgement list as (query_id, document_id, judgement) with judgement 1 = relevant and 0 = not relevant \n",
        "qrels = [\n",
        "         (0,28,0),\n",
        "         (0,7,1),\n",
        "         (0,0,0),\n",
        "         (0,18,1),\n",
        "         (0,23,1),\n",
        "         (0,17,1),\n",
        "         (0,29,1),\n",
        "         (0,27,0),\n",
        "         (0,16,0),\n",
        "         (0,20,0),\n",
        "\n",
        "         (1,28,0),\n",
        "         (1,7,1),\n",
        "         (1,0,0),\n",
        "         (1,18,0),\n",
        "         (1,23,1),\n",
        "         (1,17,1),\n",
        "         (1,29,0),\n",
        "         (1,27,1),\n",
        "         (1,16,0),\n",
        "         (1,20,0),\n",
        "]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "mdQZ6N3tKnK0"
      },
      "outputs": [],
      "source": [
        "def retrieve_ranking(query, bm25_df):\n",
        "  # Creating function for ranking retrieval\n",
        "  q_terms = query.split(' ')\n",
        "  q_terms_only = bm25_df[q_terms]\n",
        "  score_q_d = q_terms_only.sum(axis=1)\n",
        "  return sorted(zip(bm25_df.index.values,score_q_d.values), key = lambda tup:tup[1], reverse=True)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "yAEdW4eTKtXD"
      },
      "outputs": [],
      "source": [
        "def precision_at_k(query_id, k=5):\n",
        "  # Creating function for calculating precision\n",
        "  doc_ranking = retrieve_ranking(queries[query_id], bm25_df)\n",
        "  retrieved = [doc[0] for doc in doc_ranking[:k]] # take only the document id, rather than score\n",
        "\n",
        "  TP = np.array([int((query_id, doc, 1) in qrels) for doc in retrieved]).sum()\n",
        "  FP = np.array([int((query_id, doc, 0) in qrels) for doc in retrieved]).sum()\n",
        "\n",
        "  precision = TP / (TP+FP)\n",
        "\n",
        "  return TP, FP, precision"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "NHydL7PfKx6K"
      },
      "outputs": [],
      "source": [
        "def f1_score_at_k(query_id, k=5):\n",
        "  # Creating function for calculating calculating F1 score\n",
        "  doc_ranking = retrieve_ranking(queries[query_id], bm25_df)\n",
        "  retrieved = [doc[0] for doc in doc_ranking[:k]] # take only the document id, rather than score\n",
        "  \n",
        "  TP, FP, precision = precision_at_k(query_id, k)\n",
        "  relevant_docs = np.array(qrels)\n",
        "  relevant_docs = relevant_docs[relevant_docs[:, 0] == query_id][:,2].sum()\n",
        "  FN = relevant_docs - TP\n",
        "\n",
        "  recall = TP / (TP+FN)\n",
        "  f1 = (2 * precision * recall) / (precision + recall)\n",
        "  \n",
        "  return f1"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Se8nG1ZMK3yZ",
        "outputId": "605f4525-e0d5-40cf-d19d-36815c235aba"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "retrieved query \"vaccine effectiveness\" with Precision@5 = 1.0 and F1-score = 0.33333333333333337\n",
            "retrieved query \"cell membrane\" with Precision@5 = 0.4 and F1-score = 0.4444444444444445\n"
          ]
        }
      ],
      "source": [
        "# Calculating accuracy metrics for each query\n",
        "k = 5\n",
        "for query_id, query in queries.items():\n",
        "  tp, fp, precision = precision_at_k(query_id, k=k)\n",
        "  f1_score = f1_score_at_k(query_id, k=k)\n",
        "  print('retrieved query \"{}\" with Precision@{} = {} and F1-score = {}'.format(query, k, precision, f1_score))"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "jG9kBx0rG9Dm"
      },
      "source": [
        "#### **Section 4.2: Evaluation by Normalized Discounted Cumulative Gain (nDCG)**<br>  \n",
        "Using nDCG method instead of the mean average precision (MAP) in the design, the model is evaluated by calculating ndcg scores for query terms as follows:"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "upvy-omDZhL8"
      },
      "outputs": [],
      "source": [
        "# Importing library\n",
        "from sklearn.metrics import ndcg_score"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xaEhjnVSFy0k",
        "outputId": "706e5ee6-0c82-4698-a915-1553b1c25254"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "retrieved for vaccine effectiveness with NGCD@5 of 0.16958010263680806\n",
            "retrieved for cell membrane with NGCD@5 of 0.4414924137367807\n"
          ]
        }
      ],
      "source": [
        "for query_id, query in queries.items():\n",
        "  # Calculate normalized dcg (ndcg) at k\n",
        "  y_score = np.array(sorted(retrieve_ranking(queries[query_id], bm25_df)))[:,1]\n",
        "  y_true = np.zeros(y_score.size)\n",
        "  np_qrels = np.array(qrels)\n",
        "\n",
        "  for data in np_qrels[np_qrels[:, 0] == query_id]:\n",
        "    y_true[data[1]] = data[2]\n",
        "\n",
        "  ndcg = ndcg_score(np.expand_dims(y_true,axis=0), np.expand_dims(y_score,axis=0), k=k)\n",
        "  print(f'retrieved for {query} with NGCD@{k} of {ndcg}')"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "fnCXu_IVaaik"
      },
      "source": [
        "### **Section 5: Discussion and Conclusions**<br> \n",
        "In this notebook, the model search engine designed in Coursework 2 is successfully built and implemented. Utilising a BM25 model, I demonstrate how a search engine works in indexing documents, retrieving documents, and ranking retrieved documents in relation to input queries. \n",
        "\n",
        "Time constraints due to single-membership in this group prevented implementation of a Divergence from Randomness (DFR) retrieval method. \n",
        "\n",
        "The model was evaluated successfully using two different methods. \n",
        "\n",
        "With the Harmonic Mean method, F1 scores of 0.333 and 0.444 were obtained with precision @5 of 1.0 and 0.4 respectively for the query terms \"vaccine effectiveness\" and \"cell membrane\". \n",
        "\n",
        "With the Normalized Discounted Cumulative Gain method, ndcg scores of 0.170 and 0.441 were obtained with precision @5 respectively for the query terms \"vaccine effectiveness\" and \"cell membrane\". \n",
        "\n",
        "Overall, this exercise is deemed an excellent success in terms of the purpose for which it is undertaken."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "b2-MDQ8Wog4q"
      },
      "source": [
        "### **<center>References</center>**\n",
        "\n",
        "1. Wang, L.L., Lo, K., Chandrasekhar, Y., Reas, R., Yang, J., Eide, D., Funk, K., Kinney, R., Liu, Z., Merrill, W. and Mooney, P., 2020, _Cord-19: The covid-19 open research dataset_, ArXiv, [online]: \n",
        "https://doi.org/10.48550/arXiv.2004.07180\n",
        "\n",
        "2. Online Reference 1, [online]: https://pypi.org/project/wikipedia/\n",
        "\n",
        "3. Robertson, S., Walker, S., Jones, S., Hancock-Beaulieu, M. and Gatford, M., 1994, Okapi at TREC-2, NIST SPECIAL PUBLICATION SP, pp. 21-21, [online]: https://books.google.co.uk/books?hl=en&lr=&id=W8MZAQAAIAAJ&oi=fnd&pg=PA21&dq=Stephen+E.+Robertson%3B+Steve+Walker%3B+Susan+Jones&ots=3WAX_FtN7f&sig=od8KRRiryjacsQxVFNkktfJd4Us&redir_esc=y#v=onepage&q&f=false\n",
        "\n",
        "4. Jones, K.S., 1999, _Information retrieval and artificial intelligence_, Artificial Intelligence, **114**, (1-2), pp. 257-281, [online]: https://reader.elsevier.com/reader/sd/pii/S0004370299000752?token=427130C67846D0099AD0B9969F30C0156F72E7A7F9EA27FFC30E89DDD8F5031A5CC69B617DA9DAC5B5FD99BDD90D0191&originRegion=eu-west-1&originCreation=20220409195834\n",
        "\n",
        "5. Kocabaş, İ., Dincer, B.T. and Karaoğlan, B., 2011, _Investigation of Luhn's claim on information retrieval_, Turkish Journal of Electrical Engineering and Computer Science, **19**, (6), pp. 993-1004, [online]: https://www.researchgate.net/publication/237266486_Investigation_of_Luhn's_claim_on_information_retrieval\n",
        "\n",
        "6. Information Retrieval Lecture, Tutorial, and Lab notes."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "SdwJn6yrVIUT"
      },
      "outputs": [],
      "source": []
    }
  ],
  "metadata": {
    "colab": {
      "collapsed_sections": [],
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.8.8"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}